<!DOCTYPE html>
<html>
<head>

<meta charset="utf-8" />
<title>MothPollen4</title>

<script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.1.10/require.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>

<style type="text/css">
    /*!
*
* Twitter Bootstrap
*
*//*! normalize.css v3.0.2 | MIT License | git.io/normalize */html{font-family:sans-serif;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,hgroup,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block;vertical-align:baseline}audio:not([controls]){display:none;height:0}[hidden],template{display:none}a{background-color:transparent}a:active,a:hover{outline:0}abbr[title]{border-bottom:1px dotted}b,strong{font-weight:bold}dfn{font-style:italic}h1{font-size:2em;margin:.67em 0}mark{background:#ff0;color:#000}small{font-size:80%}sub,sup{font-size:75%;line-height:0;position:relative;vertical-align:baseline}sup{top:-0.5em}sub{bottom:-0.25em}img{border:0}svg:not(:root){overflow:hidden}figure{margin:1em 40px}hr{-moz-box-sizing:content-box;box-sizing:content-box;height:0}pre{overflow:auto}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}button,input,optgroup,select,textarea{color:inherit;font:inherit;margin:0}button{overflow:visible}button,select{text-transform:none}button,html input[type="button"],input[type="reset"],input[type="submit"]{-webkit-appearance:button;cursor:pointer}button[disabled],html input[disabled]{cursor:default}button::-moz-focus-inner,input::-moz-focus-inner{border:0;padding:0}input{line-height:normal}input[type="checkbox"],input[type="radio"]{box-sizing:border-box;padding:0}input[type="number"]::-webkit-inner-spin-button,input[type="number"]::-webkit-outer-spin-button{height:auto}input[type="search"]{-webkit-appearance:textfield;-moz-box-sizing:content-box;-webkit-box-sizing:content-box;box-sizing:content-box}input[type="search"]::-webkit-search-cancel-button,input[type="search"]::-webkit-search-decoration{-webkit-appearance:none}fieldset{border:1px solid #c0c0c0;margin:0 2px;padding:.35em .625em .75em}legend{border:0;padding:0}textarea{overflow:auto}optgroup{font-weight:bold}table{border-collapse:collapse;border-spacing:0}td,th{padding:0}/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */@media print{*,*:before,*:after{background:transparent !important;color:#000 !important;box-shadow:none !important;text-shadow:none !important}a,a:visited{text-decoration:underline}a[href]:after{content:" (" attr(href) ")"}abbr[title]:after{content:" (" attr(title) ")"}a[href^="#"]:after,a[href^="javascript:"]:after{content:""}pre,blockquote{border:1px solid #999;page-break-inside:avoid}thead{display:table-header-group}tr,img{page-break-inside:avoid}img{max-width:100% !important}p,h2,h3{orphans:3;widows:3}h2,h3{page-break-after:avoid}select{background:#fff !important}.navbar{display:none}.btn>.caret,.dropup>.btn>.caret{border-top-color:#000 !important}.label{border:1px solid #000}.table{border-collapse:collapse !important}.table td,.table th{background-color:#fff !important}.table-bordered th,.table-bordered td{border:1px solid #ddd !important}}@font-face{font-family:'Glyphicons Halflings';src:url('../components/bootstrap/fonts/glyphicons-halflings-regular.eot');src:url('../components/bootstrap/fonts/glyphicons-halflings-regular.eot?#iefix') format('embedded-opentype'),url('../components/bootstrap/fonts/glyphicons-halflings-regular.woff') format('woff'),url('../components/bootstrap/fonts/glyphicons-halflings-regular.ttf') format('truetype'),url('../components/bootstrap/fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular') format('svg')}.glyphicon{position:relative;top:1px;display:inline-block;font-family:'Glyphicons Halflings';font-style:normal;font-weight:normal;line-height:1;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.glyphicon-asterisk:before{content:"\2a"}.glyphicon-plus:before{content:"\2b"}.glyphicon-euro:before,.glyphicon-eur:before{content:"\20ac"}.glyphicon-minus:before{content:"\2212"}.glyphicon-cloud:before{content:"\2601"}.glyphicon-envelope:before{content:"\2709"}.glyphicon-pencil:before{content:"\270f"}.glyphicon-glass:before{content:"\e001"}.glyphicon-music:before{content:"\e002"}.glyphicon-search:before{content:"\e003"}.glyphicon-heart:before{content:"\e005"}.glyphicon-star:before{content:"\e006"}.glyphicon-star-empty:before{content:"\e007"}.glyphicon-user:before{content:"\e008"}.glyphicon-film:before{content:"\e009"}.glyphicon-th-large:before{content:"\e010"}.glyphicon-th:before{content:"\e011"}.glyphicon-th-list:before{content:"\e012"}.glyphicon-ok:before{content:"\e013"}.glyphicon-remove:before{content:"\e014"}.glyphicon-zoom-in:before{content:"\e015"}.glyphicon-zoom-out:before{content:"\e016"}.glyphicon-off:before{content:"\e017"}.glyphicon-signal:before{content:"\e018"}.glyphicon-cog:before{content:"\e019"}.glyphicon-trash:before{content:"\e020"}.glyphicon-home:before{content:"\e021"}.glyphicon-file:before{content:"\e022"}.glyphicon-time:before{content:"\e023"}.glyphicon-road:before{content:"\e024"}.glyphicon-download-alt:before{content:"\e025"}.glyphicon-download:before{content:"\e026"}.glyphicon-upload:before{content:"\e027"}.glyphicon-inbox:before{content:"\e028"}.glyphicon-play-circle:before{content:"\e029"}.glyphicon-repeat:before{content:"\e030"}.glyphicon-refresh:before{content:"\e031"}.glyphicon-list-alt:before{content:"\e032"}.glyphicon-lock:before{content:"\e033"}.glyphicon-flag:before{content:"\e034"}.glyphicon-headphones:before{content:"\e035"}.glyphicon-volume-off:before{content:"\e036"}.glyphicon-volume-down:before{content:"\e037"}.glyphicon-volume-up:before{content:"\e038"}.glyphicon-qrcode:before{content:"\e039"}.glyphicon-barcode:before{content:"\e040"}.glyphicon-tag:before{content:"\e041"}.glyphicon-tags:before{content:"\e042"}.glyphicon-book:before{content:"\e043"}.glyphicon-bookmark:before{content:"\e044"}.glyphicon-print:before{content:"\e045"}.glyphicon-camera:before{content:"\e046"}.glyphicon-font:before{content:"\e047"}.glyphicon-bold:before{content:"\e048"}.glyphicon-italic:before{content:"\e049"}.glyphicon-text-height:before{content:"\e050"}.glyphicon-text-width:before{content:"\e051"}.glyphicon-align-left:before{content:"\e052"}.glyphicon-align-center:before{content:"\e053"}.glyphicon-align-right:before{content:"\e054"}.glyphicon-align-justify:before{content:"\e055"}.glyphicon-list:before{content:"\e056"}.glyphicon-indent-left:before{content:"\e057"}.glyphicon-indent-right:before{content:"\e058"}.glyphicon-facetime-video:before{content:"\e059"}.glyphicon-picture:before{content:"\e060"}.glyphicon-map-marker:before{content:"\e062"}.glyphicon-adjust:before{content:"\e063"}.glyphicon-tint:before{content:"\e064"}.glyphicon-edit:before{content:"\e065"}.glyphicon-share:before{content:"\e066"}.glyphicon-check:before{content:"\e067"}.glyphicon-move:before{content:"\e068"}.glyphicon-step-backward:before{content:"\e069"}.glyphicon-fast-backward:before{content:"\e070"}.glyphicon-backward:before{content:"\e071"}.glyphicon-play:before{content:"\e072"}.glyphicon-pause:before{content:"\e073"}.glyphicon-stop:before{content:"\e074"}.glyphicon-forward:before{content:"\e075"}.glyphicon-fast-forward:before{content:"\e076"}.glyphicon-step-forward:before{content:"\e077"}.glyphicon-eject:before{content:"\e078"}.glyphicon-chevron-left:before{content:"\e079"}.glyphicon-chevron-right:before{content:"\e080"}.glyphicon-plus-sign:before{content:"\e081"}.glyphicon-minus-sign:before{content:"\e082"}.glyphicon-remove-sign:before{content:"\e083"}.glyphicon-ok-sign:before{content:"\e084"}.glyphicon-question-sign:before{content:"\e085"}.glyphicon-info-sign:before{content:"\e086"}.glyphicon-screenshot:before{content:"\e087"}.glyphicon-remove-circle:before{content:"\e088"}.glyphicon-ok-circle:before{content:"\e089"}.glyphicon-ban-circle:before{content:"\e090"}.glyphicon-arrow-left:before{content:"\e091"}.glyphicon-arrow-right:before{content:"\e092"}.glyphicon-arrow-up:before{content:"\e093"}.glyphicon-arrow-down:before{content:"\e094"}.glyphicon-share-alt:before{content:"\e095"}.glyphicon-resize-full:before{content:"\e096"}.glyphicon-resize-small:before{content:"\e097"}.glyphicon-exclamation-sign:before{content:"\e101"}.glyphicon-gift:before{content:"\e102"}.glyphicon-leaf:before{content:"\e103"}.glyphicon-fire:before{content:"\e104"}.glyphicon-eye-open:before{content:"\e105"}.glyphicon-eye-close:before{content:"\e106"}.glyphicon-warning-sign:before{content:"\e107"}.glyphicon-plane:before{content:"\e108"}.glyphicon-calendar:before{content:"\e109"}.glyphicon-random:before{content:"\e110"}.glyphicon-comment:before{content:"\e111"}.glyphicon-magnet:before{content:"\e112"}.glyphicon-chevron-up:before{content:"\e113"}.glyphicon-chevron-down:before{content:"\e114"}.glyphicon-retweet:before{content:"\e115"}.glyphicon-shopping-cart:before{content:"\e116"}.glyphicon-folder-close:before{content:"\e117"}.glyphicon-folder-open:before{content:"\e118"}.glyphicon-resize-vertical:before{content:"\e119"}.glyphicon-resize-horizontal:before{content:"\e120"}.glyphicon-hdd:before{content:"\e121"}.glyphicon-bullhorn:before{content:"\e122"}.glyphicon-bell:before{content:"\e123"}.glyphicon-certificate:before{content:"\e124"}.glyphicon-thumbs-up:before{content:"\e125"}.glyphicon-thumbs-down:before{content:"\e126"}.glyphicon-hand-right:before{content:"\e127"}.glyphicon-hand-left:before{content:"\e128"}.glyphicon-hand-up:before{content:"\e129"}.glyphicon-hand-down:before{content:"\e130"}.glyphicon-circle-arrow-right:before{content:"\e131"}.glyphicon-circle-arrow-left:before{content:"\e132"}.glyphicon-circle-arrow-up:before{content:"\e133"}.glyphicon-circle-arrow-down:before{content:"\e134"}.glyphicon-globe:before{content:"\e135"}.glyphicon-wrench:before{content:"\e136"}.glyphicon-tasks:before{content:"\e137"}.glyphicon-filter:before{content:"\e138"}.glyphicon-briefcase:before{content:"\e139"}.glyphicon-fullscreen:before{content:"\e140"}.glyphicon-dashboard:before{content:"\e141"}.glyphicon-paperclip:before{content:"\e142"}.glyphicon-heart-empty:before{content:"\e143"}.glyphicon-link:before{content:"\e144"}.glyphicon-phone:before{content:"\e145"}.glyphicon-pushpin:before{content:"\e146"}.glyphicon-usd:before{content:"\e148"}.glyphicon-gbp:before{content:"\e149"}.glyphicon-sort:before{content:"\e150"}.glyphicon-sort-by-alphabet:before{content:"\e151"}.glyphicon-sort-by-alphabet-alt:before{content:"\e152"}.glyphicon-sort-by-order:before{content:"\e153"}.glyphicon-sort-by-order-alt:before{content:"\e154"}.glyphicon-sort-by-attributes:before{content:"\e155"}.glyphicon-sort-by-attributes-alt:before{content:"\e156"}.glyphicon-unchecked:before{content:"\e157"}.glyphicon-expand:before{content:"\e158"}.glyphicon-collapse-down:before{content:"\e159"}.glyphicon-collapse-up:before{content:"\e160"}.glyphicon-log-in:before{content:"\e161"}.glyphicon-flash:before{content:"\e162"}.glyphicon-log-out:before{content:"\e163"}.glyphicon-new-window:before{content:"\e164"}.glyphicon-record:before{content:"\e165"}.glyphicon-save:before{content:"\e166"}.glyphicon-open:before{content:"\e167"}.glyphicon-saved:before{content:"\e168"}.glyphicon-import:before{content:"\e169"}.glyphicon-export:before{content:"\e170"}.glyphicon-send:before{content:"\e171"}.glyphicon-floppy-disk:before{content:"\e172"}.glyphicon-floppy-saved:before{content:"\e173"}.glyphicon-floppy-remove:before{content:"\e174"}.glyphicon-floppy-save:before{content:"\e175"}.glyphicon-floppy-open:before{content:"\e176"}.glyphicon-credit-card:before{content:"\e177"}.glyphicon-transfer:before{content:"\e178"}.glyphicon-cutlery:before{content:"\e179"}.glyphicon-header:before{content:"\e180"}.glyphicon-compressed:before{content:"\e181"}.glyphicon-earphone:before{content:"\e182"}.glyphicon-phone-alt:before{content:"\e183"}.glyphicon-tower:before{content:"\e184"}.glyphicon-stats:before{content:"\e185"}.glyphicon-sd-video:before{content:"\e186"}.glyphicon-hd-video:before{content:"\e187"}.glyphicon-subtitles:before{content:"\e188"}.glyphicon-sound-stereo:before{content:"\e189"}.glyphicon-sound-dolby:before{content:"\e190"}.glyphicon-sound-5-1:before{content:"\e191"}.glyphicon-sound-6-1:before{content:"\e192"}.glyphicon-sound-7-1:before{content:"\e193"}.glyphicon-copyright-mark:before{content:"\e194"}.glyphicon-registration-mark:before{content:"\e195"}.glyphicon-cloud-download:before{content:"\e197"}.glyphicon-cloud-upload:before{content:"\e198"}.glyphicon-tree-conifer:before{content:"\e199"}.glyphicon-tree-deciduous:before{content:"\e200"}*{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}*:before,*:after{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}html{font-size:10px;-webkit-tap-highlight-color:rgba(0,0,0,0)}body{font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:13px;line-height:1.42857143;color:#000;background-color:#fff}input,button,select,textarea{font-family:inherit;font-size:inherit;line-height:inherit}a{color:#337ab7;text-decoration:none}a:hover,a:focus{color:#23527c;text-decoration:underline}a:focus{outline:thin dotted;outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}figure{margin:0}img{vertical-align:middle}.img-responsive,.thumbnail>img,.thumbnail a>img,.carousel-inner>.item>img,.carousel-inner>.item>a>img{display:block;max-width:100%;height:auto}.img-rounded{border-radius:3px}.img-thumbnail{padding:4px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:2px;-webkit-transition:all .2s ease-in-out;-o-transition:all .2s ease-in-out;transition:all .2s ease-in-out;display:inline-block;max-width:100%;height:auto}.img-circle{border-radius:50%}hr{margin-top:18px;margin-bottom:18px;border:0;border-top:1px solid #eee}.sr-only{position:absolute;width:1px;height:1px;margin:-1px;padding:0;overflow:hidden;clip:rect(0, 0, 0, 0);border:0}.sr-only-focusable:active,.sr-only-focusable:focus{position:static;width:auto;height:auto;margin:0;overflow:visible;clip:auto}h1,h2,h3,h4,h5,h6,.h1,.h2,.h3,.h4,.h5,.h6{font-family:inherit;font-weight:500;line-height:1.1;color:inherit}h1 small,h2 small,h3 small,h4 small,h5 small,h6 small,.h1 small,.h2 small,.h3 small,.h4 small,.h5 small,.h6 small,h1 .small,h2 .small,h3 .small,h4 .small,h5 .small,h6 .small,.h1 .small,.h2 .small,.h3 .small,.h4 .small,.h5 .small,.h6 .small{font-weight:normal;line-height:1;color:#777}h1,.h1,h2,.h2,h3,.h3{margin-top:18px;margin-bottom:9px}h1 small,.h1 small,h2 small,.h2 small,h3 small,.h3 small,h1 .small,.h1 .small,h2 .small,.h2 .small,h3 .small,.h3 .small{font-size:65%}h4,.h4,h5,.h5,h6,.h6{margin-top:9px;margin-bottom:9px}h4 small,.h4 small,h5 small,.h5 small,h6 small,.h6 small,h4 .small,.h4 .small,h5 .small,.h5 .small,h6 .small,.h6 .small{font-size:75%}h1,.h1{font-size:33px}h2,.h2{font-size:27px}h3,.h3{font-size:23px}h4,.h4{font-size:17px}h5,.h5{font-size:13px}h6,.h6{font-size:12px}p{margin:0 0 9px}.lead{margin-bottom:18px;font-size:14px;font-weight:300;line-height:1.4}@media (min-width:768px){.lead{font-size:19.5px}}small,.small{font-size:92%}mark,.mark{background-color:#fcf8e3;padding:.2em}.text-left{text-align:left}.text-right{text-align:right}.text-center{text-align:center}.text-justify{text-align:justify}.text-nowrap{white-space:nowrap}.text-lowercase{text-transform:lowercase}.text-uppercase{text-transform:uppercase}.text-capitalize{text-transform:capitalize}.text-muted{color:#777}.text-primary{color:#337ab7}a.text-primary:hover{color:#286090}.text-success{color:#3c763d}a.text-success:hover{color:#2b542c}.text-info{color:#31708f}a.text-info:hover{color:#245269}.text-warning{color:#8a6d3b}a.text-warning:hover{color:#66512c}.text-danger{color:#a94442}a.text-danger:hover{color:#843534}.bg-primary{color:#fff;background-color:#337ab7}a.bg-primary:hover{background-color:#286090}.bg-success{background-color:#dff0d8}a.bg-success:hover{background-color:#c1e2b3}.bg-info{background-color:#d9edf7}a.bg-info:hover{background-color:#afd9ee}.bg-warning{background-color:#fcf8e3}a.bg-warning:hover{background-color:#f7ecb5}.bg-danger{background-color:#f2dede}a.bg-danger:hover{background-color:#e4b9b9}.page-header{padding-bottom:8px;margin:36px 0 18px;border-bottom:1px solid #eee}ul,ol{margin-top:0;margin-bottom:9px}ul ul,ol ul,ul ol,ol ol{margin-bottom:0}.list-unstyled{padding-left:0;list-style:none}.list-inline{padding-left:0;list-style:none;margin-left:-5px}.list-inline>li{display:inline-block;padding-left:5px;padding-right:5px}dl{margin-top:0;margin-bottom:18px}dt,dd{line-height:1.42857143}dt{font-weight:bold}dd{margin-left:0}@media (min-width:541px){.dl-horizontal dt{float:left;width:160px;clear:left;text-align:right;overflow:hidden;text-overflow:ellipsis;white-space:nowrap}.dl-horizontal dd{margin-left:180px}}abbr[title],abbr[data-original-title]{cursor:help;border-bottom:1px dotted #777}.initialism{font-size:90%;text-transform:uppercase}blockquote{padding:9px 18px;margin:0 0 18px;font-size:inherit;border-left:5px solid #eee}blockquote p:last-child,blockquote ul:last-child,blockquote ol:last-child{margin-bottom:0}blockquote footer,blockquote small,blockquote .small{display:block;font-size:80%;line-height:1.42857143;color:#777}blockquote footer:before,blockquote small:before,blockquote .small:before{content:'\2014 \00A0'}.blockquote-reverse,blockquote.pull-right{padding-right:15px;padding-left:0;border-right:5px solid #eee;border-left:0;text-align:right}.blockquote-reverse footer:before,blockquote.pull-right footer:before,.blockquote-reverse small:before,blockquote.pull-right small:before,.blockquote-reverse .small:before,blockquote.pull-right .small:before{content:''}.blockquote-reverse footer:after,blockquote.pull-right footer:after,.blockquote-reverse small:after,blockquote.pull-right small:after,.blockquote-reverse .small:after,blockquote.pull-right .small:after{content:'\00A0 \2014'}address{margin-bottom:18px;font-style:normal;line-height:1.42857143}code,kbd,pre,samp{font-family:monospace}code{padding:2px 4px;font-size:90%;color:#c7254e;background-color:#f9f2f4;border-radius:2px}kbd{padding:2px 4px;font-size:90%;color:#fff;background-color:#333;border-radius:1px;box-shadow:inset 0 -1px 0 rgba(0,0,0,0.25)}kbd kbd{padding:0;font-size:100%;font-weight:bold;box-shadow:none}pre{display:block;padding:8.5px;margin:0 0 9px;font-size:12px;line-height:1.42857143;word-break:break-all;word-wrap:break-word;color:#333;background-color:#f5f5f5;border:1px solid #ccc;border-radius:2px}pre code{padding:0;font-size:inherit;color:inherit;white-space:pre-wrap;background-color:transparent;border-radius:0}.pre-scrollable{max-height:340px;overflow-y:scroll}.container{margin-right:auto;margin-left:auto;padding-left:0;padding-right:0}@media (min-width:768px){.container{width:768px}}@media (min-width:992px){.container{width:940px}}@media (min-width:1200px){.container{width:1140px}}.container-fluid{margin-right:auto;margin-left:auto;padding-left:0;padding-right:0}.row{margin-left:0;margin-right:0}.col-xs-1, .col-sm-1, .col-md-1, .col-lg-1, .col-xs-2, .col-sm-2, .col-md-2, .col-lg-2, .col-xs-3, .col-sm-3, .col-md-3, .col-lg-3, .col-xs-4, .col-sm-4, .col-md-4, .col-lg-4, .col-xs-5, .col-sm-5, .col-md-5, .col-lg-5, .col-xs-6, .col-sm-6, .col-md-6, .col-lg-6, .col-xs-7, .col-sm-7, .col-md-7, .col-lg-7, .col-xs-8, .col-sm-8, .col-md-8, .col-lg-8, .col-xs-9, .col-sm-9, .col-md-9, .col-lg-9, .col-xs-10, .col-sm-10, .col-md-10, .col-lg-10, .col-xs-11, .col-sm-11, .col-md-11, .col-lg-11, .col-xs-12, .col-sm-12, .col-md-12, .col-lg-12{position:relative;min-height:1px;padding-left:0;padding-right:0}.col-xs-1, .col-xs-2, .col-xs-3, .col-xs-4, .col-xs-5, .col-xs-6, .col-xs-7, .col-xs-8, .col-xs-9, .col-xs-10, .col-xs-11, .col-xs-12{float:left}.col-xs-12{width:100%}.col-xs-11{width:91.66666667%}.col-xs-10{width:83.33333333%}.col-xs-9{width:75%}.col-xs-8{width:66.66666667%}.col-xs-7{width:58.33333333%}.col-xs-6{width:50%}.col-xs-5{width:41.66666667%}.col-xs-4{width:33.33333333%}.col-xs-3{width:25%}.col-xs-2{width:16.66666667%}.col-xs-1{width:8.33333333%}.col-xs-pull-12{right:100%}.col-xs-pull-11{right:91.66666667%}.col-xs-pull-10{right:83.33333333%}.col-xs-pull-9{right:75%}.col-xs-pull-8{right:66.66666667%}.col-xs-pull-7{right:58.33333333%}.col-xs-pull-6{right:50%}.col-xs-pull-5{right:41.66666667%}.col-xs-pull-4{right:33.33333333%}.col-xs-pull-3{right:25%}.col-xs-pull-2{right:16.66666667%}.col-xs-pull-1{right:8.33333333%}.col-xs-pull-0{right:auto}.col-xs-push-12{left:100%}.col-xs-push-11{left:91.66666667%}.col-xs-push-10{left:83.33333333%}.col-xs-push-9{left:75%}.col-xs-push-8{left:66.66666667%}.col-xs-push-7{left:58.33333333%}.col-xs-push-6{left:50%}.col-xs-push-5{left:41.66666667%}.col-xs-push-4{left:33.33333333%}.col-xs-push-3{left:25%}.col-xs-push-2{left:16.66666667%}.col-xs-push-1{left:8.33333333%}.col-xs-push-0{left:auto}.col-xs-offset-12{margin-left:100%}.col-xs-offset-11{margin-left:91.66666667%}.col-xs-offset-10{margin-left:83.33333333%}.col-xs-offset-9{margin-left:75%}.col-xs-offset-8{margin-left:66.66666667%}.col-xs-offset-7{margin-left:58.33333333%}.col-xs-offset-6{margin-left:50%}.col-xs-offset-5{margin-left:41.66666667%}.col-xs-offset-4{margin-left:33.33333333%}.col-xs-offset-3{margin-left:25%}.col-xs-offset-2{margin-left:16.66666667%}.col-xs-offset-1{margin-left:8.33333333%}.col-xs-offset-0{margin-left:0}@media (min-width:768px){.col-sm-1, .col-sm-2, .col-sm-3, .col-sm-4, .col-sm-5, .col-sm-6, .col-sm-7, .col-sm-8, .col-sm-9, .col-sm-10, .col-sm-11, .col-sm-12{float:left}.col-sm-12{width:100%}.col-sm-11{width:91.66666667%}.col-sm-10{width:83.33333333%}.col-sm-9{width:75%}.col-sm-8{width:66.66666667%}.col-sm-7{width:58.33333333%}.col-sm-6{width:50%}.col-sm-5{width:41.66666667%}.col-sm-4{width:33.33333333%}.col-sm-3{width:25%}.col-sm-2{width:16.66666667%}.col-sm-1{width:8.33333333%}.col-sm-pull-12{right:100%}.col-sm-pull-11{right:91.66666667%}.col-sm-pull-10{right:83.33333333%}.col-sm-pull-9{right:75%}.col-sm-pull-8{right:66.66666667%}.col-sm-pull-7{right:58.33333333%}.col-sm-pull-6{right:50%}.col-sm-pull-5{right:41.66666667%}.col-sm-pull-4{right:33.33333333%}.col-sm-pull-3{right:25%}.col-sm-pull-2{right:16.66666667%}.col-sm-pull-1{right:8.33333333%}.col-sm-pull-0{right:auto}.col-sm-push-12{left:100%}.col-sm-push-11{left:91.66666667%}.col-sm-push-10{left:83.33333333%}.col-sm-push-9{left:75%}.col-sm-push-8{left:66.66666667%}.col-sm-push-7{left:58.33333333%}.col-sm-push-6{left:50%}.col-sm-push-5{left:41.66666667%}.col-sm-push-4{left:33.33333333%}.col-sm-push-3{left:25%}.col-sm-push-2{left:16.66666667%}.col-sm-push-1{left:8.33333333%}.col-sm-push-0{left:auto}.col-sm-offset-12{margin-left:100%}.col-sm-offset-11{margin-left:91.66666667%}.col-sm-offset-10{margin-left:83.33333333%}.col-sm-offset-9{margin-left:75%}.col-sm-offset-8{margin-left:66.66666667%}.col-sm-offset-7{margin-left:58.33333333%}.col-sm-offset-6{margin-left:50%}.col-sm-offset-5{margin-left:41.66666667%}.col-sm-offset-4{margin-left:33.33333333%}.col-sm-offset-3{margin-left:25%}.col-sm-offset-2{margin-left:16.66666667%}.col-sm-offset-1{margin-left:8.33333333%}.col-sm-offset-0{margin-left:0}}@media (min-width:992px){.col-md-1, .col-md-2, .col-md-3, .col-md-4, .col-md-5, .col-md-6, .col-md-7, .col-md-8, .col-md-9, .col-md-10, .col-md-11, .col-md-12{float:left}.col-md-12{width:100%}.col-md-11{width:91.66666667%}.col-md-10{width:83.33333333%}.col-md-9{width:75%}.col-md-8{width:66.66666667%}.col-md-7{width:58.33333333%}.col-md-6{width:50%}.col-md-5{width:41.66666667%}.col-md-4{width:33.33333333%}.col-md-3{width:25%}.col-md-2{width:16.66666667%}.col-md-1{width:8.33333333%}.col-md-pull-12{right:100%}.col-md-pull-11{right:91.66666667%}.col-md-pull-10{right:83.33333333%}.col-md-pull-9{right:75%}.col-md-pull-8{right:66.66666667%}.col-md-pull-7{right:58.33333333%}.col-md-pull-6{right:50%}.col-md-pull-5{right:41.66666667%}.col-md-pull-4{right:33.33333333%}.col-md-pull-3{right:25%}.col-md-pull-2{right:16.66666667%}.col-md-pull-1{right:8.33333333%}.col-md-pull-0{right:auto}.col-md-push-12{left:100%}.col-md-push-11{left:91.66666667%}.col-md-push-10{left:83.33333333%}.col-md-push-9{left:75%}.col-md-push-8{left:66.66666667%}.col-md-push-7{left:58.33333333%}.col-md-push-6{left:50%}.col-md-push-5{left:41.66666667%}.col-md-push-4{left:33.33333333%}.col-md-push-3{left:25%}.col-md-push-2{left:16.66666667%}.col-md-push-1{left:8.33333333%}.col-md-push-0{left:auto}.col-md-offset-12{margin-left:100%}.col-md-offset-11{margin-left:91.66666667%}.col-md-offset-10{margin-left:83.33333333%}.col-md-offset-9{margin-left:75%}.col-md-offset-8{margin-left:66.66666667%}.col-md-offset-7{margin-left:58.33333333%}.col-md-offset-6{margin-left:50%}.col-md-offset-5{margin-left:41.66666667%}.col-md-offset-4{margin-left:33.33333333%}.col-md-offset-3{margin-left:25%}.col-md-offset-2{margin-left:16.66666667%}.col-md-offset-1{margin-left:8.33333333%}.col-md-offset-0{margin-left:0}}@media (min-width:1200px){.col-lg-1, .col-lg-2, .col-lg-3, .col-lg-4, .col-lg-5, .col-lg-6, .col-lg-7, .col-lg-8, .col-lg-9, .col-lg-10, .col-lg-11, .col-lg-12{float:left}.col-lg-12{width:100%}.col-lg-11{width:91.66666667%}.col-lg-10{width:83.33333333%}.col-lg-9{width:75%}.col-lg-8{width:66.66666667%}.col-lg-7{width:58.33333333%}.col-lg-6{width:50%}.col-lg-5{width:41.66666667%}.col-lg-4{width:33.33333333%}.col-lg-3{width:25%}.col-lg-2{width:16.66666667%}.col-lg-1{width:8.33333333%}.col-lg-pull-12{right:100%}.col-lg-pull-11{right:91.66666667%}.col-lg-pull-10{right:83.33333333%}.col-lg-pull-9{right:75%}.col-lg-pull-8{right:66.66666667%}.col-lg-pull-7{right:58.33333333%}.col-lg-pull-6{right:50%}.col-lg-pull-5{right:41.66666667%}.col-lg-pull-4{right:33.33333333%}.col-lg-pull-3{right:25%}.col-lg-pull-2{right:16.66666667%}.col-lg-pull-1{right:8.33333333%}.col-lg-pull-0{right:auto}.col-lg-push-12{left:100%}.col-lg-push-11{left:91.66666667%}.col-lg-push-10{left:83.33333333%}.col-lg-push-9{left:75%}.col-lg-push-8{left:66.66666667%}.col-lg-push-7{left:58.33333333%}.col-lg-push-6{left:50%}.col-lg-push-5{left:41.66666667%}.col-lg-push-4{left:33.33333333%}.col-lg-push-3{left:25%}.col-lg-push-2{left:16.66666667%}.col-lg-push-1{left:8.33333333%}.col-lg-push-0{left:auto}.col-lg-offset-12{margin-left:100%}.col-lg-offset-11{margin-left:91.66666667%}.col-lg-offset-10{margin-left:83.33333333%}.col-lg-offset-9{margin-left:75%}.col-lg-offset-8{margin-left:66.66666667%}.col-lg-offset-7{margin-left:58.33333333%}.col-lg-offset-6{margin-left:50%}.col-lg-offset-5{margin-left:41.66666667%}.col-lg-offset-4{margin-left:33.33333333%}.col-lg-offset-3{margin-left:25%}.col-lg-offset-2{margin-left:16.66666667%}.col-lg-offset-1{margin-left:8.33333333%}.col-lg-offset-0{margin-left:0}}table{background-color:transparent}caption{padding-top:8px;padding-bottom:8px;color:#777;text-align:left}th{text-align:left}.table{width:100%;max-width:100%;margin-bottom:18px}.table>thead>tr>th,.table>tbody>tr>th,.table>tfoot>tr>th,.table>thead>tr>td,.table>tbody>tr>td,.table>tfoot>tr>td{padding:8px;line-height:1.42857143;vertical-align:top;border-top:1px solid #ddd}.table>thead>tr>th{vertical-align:bottom;border-bottom:2px solid #ddd}.table>caption+thead>tr:first-child>th,.table>colgroup+thead>tr:first-child>th,.table>thead:first-child>tr:first-child>th,.table>caption+thead>tr:first-child>td,.table>colgroup+thead>tr:first-child>td,.table>thead:first-child>tr:first-child>td{border-top:0}.table>tbody+tbody{border-top:2px solid #ddd}.table .table{background-color:#fff}.table-condensed>thead>tr>th,.table-condensed>tbody>tr>th,.table-condensed>tfoot>tr>th,.table-condensed>thead>tr>td,.table-condensed>tbody>tr>td,.table-condensed>tfoot>tr>td{padding:5px}.table-bordered{border:1px solid #ddd}.table-bordered>thead>tr>th,.table-bordered>tbody>tr>th,.table-bordered>tfoot>tr>th,.table-bordered>thead>tr>td,.table-bordered>tbody>tr>td,.table-bordered>tfoot>tr>td{border:1px solid #ddd}.table-bordered>thead>tr>th,.table-bordered>thead>tr>td{border-bottom-width:2px}.table-striped>tbody>tr:nth-child(odd){background-color:#f9f9f9}.table-hover>tbody>tr:hover{background-color:#f5f5f5}table col[class*="col-"]{position:static;float:none;display:table-column}table td[class*="col-"],table th[class*="col-"]{position:static;float:none;display:table-cell}.table>thead>tr>td.active,.table>tbody>tr>td.active,.table>tfoot>tr>td.active,.table>thead>tr>th.active,.table>tbody>tr>th.active,.table>tfoot>tr>th.active,.table>thead>tr.active>td,.table>tbody>tr.active>td,.table>tfoot>tr.active>td,.table>thead>tr.active>th,.table>tbody>tr.active>th,.table>tfoot>tr.active>th{background-color:#f5f5f5}.table-hover>tbody>tr>td.active:hover,.table-hover>tbody>tr>th.active:hover,.table-hover>tbody>tr.active:hover>td,.table-hover>tbody>tr:hover>.active,.table-hover>tbody>tr.active:hover>th{background-color:#e8e8e8}.table>thead>tr>td.success,.table>tbody>tr>td.success,.table>tfoot>tr>td.success,.table>thead>tr>th.success,.table>tbody>tr>th.success,.table>tfoot>tr>th.success,.table>thead>tr.success>td,.table>tbody>tr.success>td,.table>tfoot>tr.success>td,.table>thead>tr.success>th,.table>tbody>tr.success>th,.table>tfoot>tr.success>th{background-color:#dff0d8}.table-hover>tbody>tr>td.success:hover,.table-hover>tbody>tr>th.success:hover,.table-hover>tbody>tr.success:hover>td,.table-hover>tbody>tr:hover>.success,.table-hover>tbody>tr.success:hover>th{background-color:#d0e9c6}.table>thead>tr>td.info,.table>tbody>tr>td.info,.table>tfoot>tr>td.info,.table>thead>tr>th.info,.table>tbody>tr>th.info,.table>tfoot>tr>th.info,.table>thead>tr.info>td,.table>tbody>tr.info>td,.table>tfoot>tr.info>td,.table>thead>tr.info>th,.table>tbody>tr.info>th,.table>tfoot>tr.info>th{background-color:#d9edf7}.table-hover>tbody>tr>td.info:hover,.table-hover>tbody>tr>th.info:hover,.table-hover>tbody>tr.info:hover>td,.table-hover>tbody>tr:hover>.info,.table-hover>tbody>tr.info:hover>th{background-color:#c4e3f3}.table>thead>tr>td.warning,.table>tbody>tr>td.warning,.table>tfoot>tr>td.warning,.table>thead>tr>th.warning,.table>tbody>tr>th.warning,.table>tfoot>tr>th.warning,.table>thead>tr.warning>td,.table>tbody>tr.warning>td,.table>tfoot>tr.warning>td,.table>thead>tr.warning>th,.table>tbody>tr.warning>th,.table>tfoot>tr.warning>th{background-color:#fcf8e3}.table-hover>tbody>tr>td.warning:hover,.table-hover>tbody>tr>th.warning:hover,.table-hover>tbody>tr.warning:hover>td,.table-hover>tbody>tr:hover>.warning,.table-hover>tbody>tr.warning:hover>th{background-color:#faf2cc}.table>thead>tr>td.danger,.table>tbody>tr>td.danger,.table>tfoot>tr>td.danger,.table>thead>tr>th.danger,.table>tbody>tr>th.danger,.table>tfoot>tr>th.danger,.table>thead>tr.danger>td,.table>tbody>tr.danger>td,.table>tfoot>tr.danger>td,.table>thead>tr.danger>th,.table>tbody>tr.danger>th,.table>tfoot>tr.danger>th{background-color:#f2dede}.table-hover>tbody>tr>td.danger:hover,.table-hover>tbody>tr>th.danger:hover,.table-hover>tbody>tr.danger:hover>td,.table-hover>tbody>tr:hover>.danger,.table-hover>tbody>tr.danger:hover>th{background-color:#ebcccc}.table-responsive{overflow-x:auto;min-height:.01%}@media screen and (max-width:767px){.table-responsive{width:100%;margin-bottom:13.5px;overflow-y:hidden;-ms-overflow-style:-ms-autohiding-scrollbar;border:1px solid #ddd}.table-responsive>.table{margin-bottom:0}.table-responsive>.table>thead>tr>th,.table-responsive>.table>tbody>tr>th,.table-responsive>.table>tfoot>tr>th,.table-responsive>.table>thead>tr>td,.table-responsive>.table>tbody>tr>td,.table-responsive>.table>tfoot>tr>td{white-space:nowrap}.table-responsive>.table-bordered{border:0}.table-responsive>.table-bordered>thead>tr>th:first-child,.table-responsive>.table-bordered>tbody>tr>th:first-child,.table-responsive>.table-bordered>tfoot>tr>th:first-child,.table-responsive>.table-bordered>thead>tr>td:first-child,.table-responsive>.table-bordered>tbody>tr>td:first-child,.table-responsive>.table-bordered>tfoot>tr>td:first-child{border-left:0}.table-responsive>.table-bordered>thead>tr>th:last-child,.table-responsive>.table-bordered>tbody>tr>th:last-child,.table-responsive>.table-bordered>tfoot>tr>th:last-child,.table-responsive>.table-bordered>thead>tr>td:last-child,.table-responsive>.table-bordered>tbody>tr>td:last-child,.table-responsive>.table-bordered>tfoot>tr>td:last-child{border-right:0}.table-responsive>.table-bordered>tbody>tr:last-child>th,.table-responsive>.table-bordered>tfoot>tr:last-child>th,.table-responsive>.table-bordered>tbody>tr:last-child>td,.table-responsive>.table-bordered>tfoot>tr:last-child>td{border-bottom:0}}fieldset{padding:0;margin:0;border:0;min-width:0}legend{display:block;width:100%;padding:0;margin-bottom:18px;font-size:19.5px;line-height:inherit;color:#333;border:0;border-bottom:1px solid #e5e5e5}label{display:inline-block;max-width:100%;margin-bottom:5px;font-weight:bold}input[type="search"]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}input[type="radio"],input[type="checkbox"]{margin:4px 0 0;margin-top:1px \9;line-height:normal}input[type="file"]{display:block}input[type="range"]{display:block;width:100%}select[multiple],select[size]{height:auto}input[type="file"]:focus,input[type="radio"]:focus,input[type="checkbox"]:focus{outline:thin dotted;outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}output{display:block;padding-top:7px;font-size:13px;line-height:1.42857143;color:#555}.form-control{display:block;width:100%;height:32px;padding:6px 12px;font-size:13px;line-height:1.42857143;color:#555;background-color:#fff;background-image:none;border:1px solid #ccc;border-radius:2px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,0.075);box-shadow:inset 0 1px 1px rgba(0,0,0,0.075);-webkit-transition:border-color ease-in-out .15s, box-shadow ease-in-out .15s;-o-transition:border-color ease-in-out .15s, box-shadow ease-in-out .15s;transition:border-color ease-in-out .15s, box-shadow ease-in-out .15s}.form-control:focus{border-color:#66afe9;outline:0;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102, 175, 233, 0.6);box-shadow:inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102, 175, 233, 0.6)}.form-control::-moz-placeholder{color:#999;opacity:1}.form-control:-ms-input-placeholder{color:#999}.form-control::-webkit-input-placeholder{color:#999}.form-control[disabled],.form-control[readonly],fieldset[disabled] .form-control{cursor:not-allowed;background-color:#eee;opacity:1}textarea.form-control{height:auto}input[type="search"]{-webkit-appearance:none}@media screen and (-webkit-min-device-pixel-ratio:0){input[type="date"],input[type="time"],input[type="datetime-local"],input[type="month"]{line-height:32px}input[type="date"].input-sm,input[type="time"].input-sm,input[type="datetime-local"].input-sm,input[type="month"].input-sm{line-height:30px}input[type="date"].input-lg,input[type="time"].input-lg,input[type="datetime-local"].input-lg,input[type="month"].input-lg{line-height:45px}}.form-group{margin-bottom:15px}.radio,.checkbox{position:relative;display:block;margin-top:10px;margin-bottom:10px}.radio label,.checkbox label{min-height:18px;padding-left:20px;margin-bottom:0;font-weight:normal;cursor:pointer}.radio input[type="radio"],.radio-inline input[type="radio"],.checkbox input[type="checkbox"],.checkbox-inline input[type="checkbox"]{position:absolute;margin-left:-20px;margin-top:4px \9}.radio+.radio,.checkbox+.checkbox{margin-top:-5px}.radio-inline,.checkbox-inline{display:inline-block;padding-left:20px;margin-bottom:0;vertical-align:middle;font-weight:normal;cursor:pointer}.radio-inline+.radio-inline,.checkbox-inline+.checkbox-inline{margin-top:0;margin-left:10px}input[type="radio"][disabled],input[type="checkbox"][disabled],input[type="radio"].disabled,input[type="checkbox"].disabled,fieldset[disabled] input[type="radio"],fieldset[disabled] input[type="checkbox"]{cursor:not-allowed}.radio-inline.disabled,.checkbox-inline.disabled,fieldset[disabled] .radio-inline,fieldset[disabled] .checkbox-inline{cursor:not-allowed}.radio.disabled label,.checkbox.disabled label,fieldset[disabled] .radio label,fieldset[disabled] .checkbox label{cursor:not-allowed}.form-control-static{padding-top:7px;padding-bottom:7px;margin-bottom:0}.form-control-static.input-lg,.form-control-static.input-sm{padding-left:0;padding-right:0}.input-sm,.form-group-sm .form-control{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:1px}select.input-sm,select.form-group-sm .form-control{height:30px;line-height:30px}textarea.input-sm,textarea.form-group-sm .form-control,select[multiple].input-sm,select[multiple].form-group-sm .form-control{height:auto}.input-lg,.form-group-lg .form-control{height:45px;padding:10px 16px;font-size:17px;line-height:1.33;border-radius:3px}select.input-lg,select.form-group-lg .form-control{height:45px;line-height:45px}textarea.input-lg,textarea.form-group-lg .form-control,select[multiple].input-lg,select[multiple].form-group-lg .form-control{height:auto}.has-feedback{position:relative}.has-feedback .form-control{padding-right:40px}.form-control-feedback{position:absolute;top:0;right:0;z-index:2;display:block;width:32px;height:32px;line-height:32px;text-align:center;pointer-events:none}.input-lg+.form-control-feedback{width:45px;height:45px;line-height:45px}.input-sm+.form-control-feedback{width:30px;height:30px;line-height:30px}.has-success .help-block,.has-success .control-label,.has-success .radio,.has-success .checkbox,.has-success .radio-inline,.has-success .checkbox-inline,.has-success.radio label,.has-success.checkbox label,.has-success.radio-inline label,.has-success.checkbox-inline label{color:#3c763d}.has-success .form-control{border-color:#3c763d;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,0.075);box-shadow:inset 0 1px 1px rgba(0,0,0,0.075)}.has-success .form-control:focus{border-color:#2b542c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,0.075),0 0 6px #67b168;box-shadow:inset 0 1px 1px rgba(0,0,0,0.075),0 0 6px #67b168}.has-success .input-group-addon{color:#3c763d;border-color:#3c763d;background-color:#dff0d8}.has-success .form-control-feedback{color:#3c763d}.has-warning .help-block,.has-warning .control-label,.has-warning .radio,.has-warning .checkbox,.has-warning .radio-inline,.has-warning .checkbox-inline,.has-warning.radio label,.has-warning.checkbox label,.has-warning.radio-inline label,.has-warning.checkbox-inline label{color:#8a6d3b}.has-warning .form-control{border-color:#8a6d3b;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,0.075);box-shadow:inset 0 1px 1px rgba(0,0,0,0.075)}.has-warning .form-control:focus{border-color:#66512c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,0.075),0 0 6px #c0a16b;box-shadow:inset 0 1px 1px rgba(0,0,0,0.075),0 0 6px #c0a16b}.has-warning .input-group-addon{color:#8a6d3b;border-color:#8a6d3b;background-color:#fcf8e3}.has-warning .form-control-feedback{color:#8a6d3b}.has-error .help-block,.has-error .control-label,.has-error .radio,.has-error .checkbox,.has-error .radio-inline,.has-error .checkbox-inline,.has-error.radio label,.has-error.checkbox label,.has-error.radio-inline label,.has-error.checkbox-inline label{color:#a94442}.has-error .form-control{border-color:#a94442;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,0.075);box-shadow:inset 0 1px 1px rgba(0,0,0,0.075)}.has-error .form-control:focus{border-color:#843534;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,0.075),0 0 6px #ce8483;box-shadow:inset 0 1px 1px rgba(0,0,0,0.075),0 0 6px #ce8483}.has-error .input-group-addon{color:#a94442;border-color:#a94442;background-color:#f2dede}.has-error .form-control-feedback{color:#a94442}.has-feedback label~.form-control-feedback{top:23px}.has-feedback label.sr-only~.form-control-feedback{top:0}.help-block{display:block;margin-top:5px;margin-bottom:10px;color:#404040}@media (min-width:768px){.form-inline .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.form-inline .form-control{display:inline-block;width:auto;vertical-align:middle}.form-inline .form-control-static{display:inline-block}.form-inline .input-group{display:inline-table;vertical-align:middle}.form-inline .input-group .input-group-addon,.form-inline .input-group .input-group-btn,.form-inline .input-group .form-control{width:auto}.form-inline .input-group>.form-control{width:100%}.form-inline .control-label{margin-bottom:0;vertical-align:middle}.form-inline .radio,.form-inline .checkbox{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.form-inline .radio label,.form-inline .checkbox label{padding-left:0}.form-inline .radio input[type="radio"],.form-inline .checkbox input[type="checkbox"]{position:relative;margin-left:0}.form-inline .has-feedback .form-control-feedback{top:0}}.form-horizontal .radio,.form-horizontal .checkbox,.form-horizontal .radio-inline,.form-horizontal .checkbox-inline{margin-top:0;margin-bottom:0;padding-top:7px}.form-horizontal .radio,.form-horizontal .checkbox{min-height:25px}.form-horizontal .form-group{margin-left:0;margin-right:0}@media (min-width:768px){.form-horizontal .control-label{text-align:right;margin-bottom:0;padding-top:7px}}.form-horizontal .has-feedback .form-control-feedback{right:0}@media (min-width:768px){.form-horizontal .form-group-lg .control-label{padding-top:14.3px}}@media (min-width:768px){.form-horizontal .form-group-sm .control-label{padding-top:6px}}.btn{display:inline-block;margin-bottom:0;font-weight:normal;text-align:center;vertical-align:middle;touch-action:manipulation;cursor:pointer;background-image:none;border:1px solid transparent;white-space:nowrap;padding:6px 12px;font-size:13px;line-height:1.42857143;border-radius:2px;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none}.btn:focus,.btn:active:focus,.btn.active:focus,.btn.focus,.btn:active.focus,.btn.active.focus{outline:thin dotted;outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}.btn:hover,.btn:focus,.btn.focus{color:#333;text-decoration:none}.btn:active,.btn.active{outline:0;background-image:none;-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,0.125);box-shadow:inset 0 3px 5px rgba(0,0,0,0.125)}.btn.disabled,.btn[disabled],fieldset[disabled] .btn{cursor:not-allowed;pointer-events:none;opacity:.65;filter:alpha(opacity=65);-webkit-box-shadow:none;box-shadow:none}.btn-default{color:#333;background-color:#fff;border-color:#ccc}.btn-default:hover,.btn-default:focus,.btn-default.focus,.btn-default:active,.btn-default.active,.open>.dropdown-toggle.btn-default{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default:active,.btn-default.active,.open>.dropdown-toggle.btn-default{background-image:none}.btn-default.disabled,.btn-default[disabled],fieldset[disabled] .btn-default,.btn-default.disabled:hover,.btn-default[disabled]:hover,fieldset[disabled] .btn-default:hover,.btn-default.disabled:focus,.btn-default[disabled]:focus,fieldset[disabled] .btn-default:focus,.btn-default.disabled.focus,.btn-default[disabled].focus,fieldset[disabled] .btn-default.focus,.btn-default.disabled:active,.btn-default[disabled]:active,fieldset[disabled] .btn-default:active,.btn-default.disabled.active,.btn-default[disabled].active,fieldset[disabled] .btn-default.active{background-color:#fff;border-color:#ccc}.btn-default .badge{color:#fff;background-color:#333}.btn-primary{color:#fff;background-color:#337ab7;border-color:#2e6da4}.btn-primary:hover,.btn-primary:focus,.btn-primary.focus,.btn-primary:active,.btn-primary.active,.open>.dropdown-toggle.btn-primary{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary:active,.btn-primary.active,.open>.dropdown-toggle.btn-primary{background-image:none}.btn-primary.disabled,.btn-primary[disabled],fieldset[disabled] .btn-primary,.btn-primary.disabled:hover,.btn-primary[disabled]:hover,fieldset[disabled] .btn-primary:hover,.btn-primary.disabled:focus,.btn-primary[disabled]:focus,fieldset[disabled] .btn-primary:focus,.btn-primary.disabled.focus,.btn-primary[disabled].focus,fieldset[disabled] .btn-primary.focus,.btn-primary.disabled:active,.btn-primary[disabled]:active,fieldset[disabled] .btn-primary:active,.btn-primary.disabled.active,.btn-primary[disabled].active,fieldset[disabled] .btn-primary.active{background-color:#337ab7;border-color:#2e6da4}.btn-primary .badge{color:#337ab7;background-color:#fff}.btn-success{color:#fff;background-color:#5cb85c;border-color:#4cae4c}.btn-success:hover,.btn-success:focus,.btn-success.focus,.btn-success:active,.btn-success.active,.open>.dropdown-toggle.btn-success{color:#fff;background-color:#449d44;border-color:#398439}.btn-success:active,.btn-success.active,.open>.dropdown-toggle.btn-success{background-image:none}.btn-success.disabled,.btn-success[disabled],fieldset[disabled] .btn-success,.btn-success.disabled:hover,.btn-success[disabled]:hover,fieldset[disabled] .btn-success:hover,.btn-success.disabled:focus,.btn-success[disabled]:focus,fieldset[disabled] .btn-success:focus,.btn-success.disabled.focus,.btn-success[disabled].focus,fieldset[disabled] .btn-success.focus,.btn-success.disabled:active,.btn-success[disabled]:active,fieldset[disabled] .btn-success:active,.btn-success.disabled.active,.btn-success[disabled].active,fieldset[disabled] .btn-success.active{background-color:#5cb85c;border-color:#4cae4c}.btn-success .badge{color:#5cb85c;background-color:#fff}.btn-info{color:#fff;background-color:#5bc0de;border-color:#46b8da}.btn-info:hover,.btn-info:focus,.btn-info.focus,.btn-info:active,.btn-info.active,.open>.dropdown-toggle.btn-info{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info:active,.btn-info.active,.open>.dropdown-toggle.btn-info{background-image:none}.btn-info.disabled,.btn-info[disabled],fieldset[disabled] .btn-info,.btn-info.disabled:hover,.btn-info[disabled]:hover,fieldset[disabled] .btn-info:hover,.btn-info.disabled:focus,.btn-info[disabled]:focus,fieldset[disabled] .btn-info:focus,.btn-info.disabled.focus,.btn-info[disabled].focus,fieldset[disabled] .btn-info.focus,.btn-info.disabled:active,.btn-info[disabled]:active,fieldset[disabled] .btn-info:active,.btn-info.disabled.active,.btn-info[disabled].active,fieldset[disabled] .btn-info.active{background-color:#5bc0de;border-color:#46b8da}.btn-info .badge{color:#5bc0de;background-color:#fff}.btn-warning{color:#fff;background-color:#f0ad4e;border-color:#eea236}.btn-warning:hover,.btn-warning:focus,.btn-warning.focus,.btn-warning:active,.btn-warning.active,.open>.dropdown-toggle.btn-warning{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning:active,.btn-warning.active,.open>.dropdown-toggle.btn-warning{background-image:none}.btn-warning.disabled,.btn-warning[disabled],fieldset[disabled] .btn-warning,.btn-warning.disabled:hover,.btn-warning[disabled]:hover,fieldset[disabled] .btn-warning:hover,.btn-warning.disabled:focus,.btn-warning[disabled]:focus,fieldset[disabled] .btn-warning:focus,.btn-warning.disabled.focus,.btn-warning[disabled].focus,fieldset[disabled] .btn-warning.focus,.btn-warning.disabled:active,.btn-warning[disabled]:active,fieldset[disabled] .btn-warning:active,.btn-warning.disabled.active,.btn-warning[disabled].active,fieldset[disabled] .btn-warning.active{background-color:#f0ad4e;border-color:#eea236}.btn-warning .badge{color:#f0ad4e;background-color:#fff}.btn-danger{color:#fff;background-color:#d9534f;border-color:#d43f3a}.btn-danger:hover,.btn-danger:focus,.btn-danger.focus,.btn-danger:active,.btn-danger.active,.open>.dropdown-toggle.btn-danger{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger:active,.btn-danger.active,.open>.dropdown-toggle.btn-danger{background-image:none}.btn-danger.disabled,.btn-danger[disabled],fieldset[disabled] .btn-danger,.btn-danger.disabled:hover,.btn-danger[disabled]:hover,fieldset[disabled] .btn-danger:hover,.btn-danger.disabled:focus,.btn-danger[disabled]:focus,fieldset[disabled] .btn-danger:focus,.btn-danger.disabled.focus,.btn-danger[disabled].focus,fieldset[disabled] .btn-danger.focus,.btn-danger.disabled:active,.btn-danger[disabled]:active,fieldset[disabled] .btn-danger:active,.btn-danger.disabled.active,.btn-danger[disabled].active,fieldset[disabled] .btn-danger.active{background-color:#d9534f;border-color:#d43f3a}.btn-danger .badge{color:#d9534f;background-color:#fff}.btn-link{color:#337ab7;font-weight:normal;border-radius:0}.btn-link,.btn-link:active,.btn-link.active,.btn-link[disabled],fieldset[disabled] .btn-link{background-color:transparent;-webkit-box-shadow:none;box-shadow:none}.btn-link,.btn-link:hover,.btn-link:focus,.btn-link:active{border-color:transparent}.btn-link:hover,.btn-link:focus{color:#23527c;text-decoration:underline;background-color:transparent}.btn-link[disabled]:hover,fieldset[disabled] .btn-link:hover,.btn-link[disabled]:focus,fieldset[disabled] .btn-link:focus{color:#777;text-decoration:none}.btn-lg,.btn-group-lg>.btn{padding:10px 16px;font-size:17px;line-height:1.33;border-radius:3px}.btn-sm,.btn-group-sm>.btn{padding:5px 10px;font-size:12px;line-height:1.5;border-radius:1px}.btn-xs,.btn-group-xs>.btn{padding:1px 5px;font-size:12px;line-height:1.5;border-radius:1px}.btn-block{display:block;width:100%}.btn-block+.btn-block{margin-top:5px}input[type="submit"].btn-block,input[type="reset"].btn-block,input[type="button"].btn-block{width:100%}.fade{opacity:0;-webkit-transition:opacity .15s linear;-o-transition:opacity .15s linear;transition:opacity .15s linear}.fade.in{opacity:1}.collapse{display:none;visibility:hidden}.collapse.in{display:block;visibility:visible}tr.collapse.in{display:table-row}tbody.collapse.in{display:table-row-group}.collapsing{position:relative;height:0;overflow:hidden;-webkit-transition-property:height, visibility;transition-property:height, visibility;-webkit-transition-duration:.35s;transition-duration:.35s;-webkit-transition-timing-function:ease;transition-timing-function:ease}.caret{display:inline-block;width:0;height:0;margin-left:2px;vertical-align:middle;border-top:4px solid;border-right:4px solid transparent;border-left:4px solid transparent}.dropdown{position:relative}.dropdown-toggle:focus{outline:0}.dropdown-menu{position:absolute;top:100%;left:0;z-index:1000;display:none;float:left;min-width:160px;padding:5px 0;margin:2px 0 0;list-style:none;font-size:13px;text-align:left;background-color:#fff;border:1px solid #ccc;border:1px solid rgba(0,0,0,0.15);border-radius:2px;-webkit-box-shadow:0 6px 12px rgba(0,0,0,0.175);box-shadow:0 6px 12px rgba(0,0,0,0.175);background-clip:padding-box}.dropdown-menu.pull-right{right:0;left:auto}.dropdown-menu .divider{height:1px;margin:8px 0;overflow:hidden;background-color:#e5e5e5}.dropdown-menu>li>a{display:block;padding:3px 20px;clear:both;font-weight:normal;line-height:1.42857143;color:#333;white-space:nowrap}.dropdown-menu>li>a:hover,.dropdown-menu>li>a:focus{text-decoration:none;color:#262626;background-color:#f5f5f5}.dropdown-menu>.active>a,.dropdown-menu>.active>a:hover,.dropdown-menu>.active>a:focus{color:#fff;text-decoration:none;outline:0;background-color:#337ab7}.dropdown-menu>.disabled>a,.dropdown-menu>.disabled>a:hover,.dropdown-menu>.disabled>a:focus{color:#777}.dropdown-menu>.disabled>a:hover,.dropdown-menu>.disabled>a:focus{text-decoration:none;background-color:transparent;background-image:none;filter:progid:DXImageTransform.Microsoft.gradient(enabled = false);cursor:not-allowed}.open>.dropdown-menu{display:block}.open>a{outline:0}.dropdown-menu-right{left:auto;right:0}.dropdown-menu-left{left:0;right:auto}.dropdown-header{display:block;padding:3px 20px;font-size:12px;line-height:1.42857143;color:#777;white-space:nowrap}.dropdown-backdrop{position:fixed;left:0;right:0;bottom:0;top:0;z-index:990}.pull-right>.dropdown-menu{right:0;left:auto}.dropup .caret,.navbar-fixed-bottom .dropdown .caret{border-top:0;border-bottom:4px solid;content:""}.dropup .dropdown-menu,.navbar-fixed-bottom .dropdown .dropdown-menu{top:auto;bottom:100%;margin-bottom:1px}@media (min-width:541px){.navbar-right .dropdown-menu{left:auto;right:0}.navbar-right .dropdown-menu-left{left:0;right:auto}}.btn-group,.btn-group-vertical{position:relative;display:inline-block;vertical-align:middle}.btn-group>.btn,.btn-group-vertical>.btn{position:relative;float:left}.btn-group>.btn:hover,.btn-group-vertical>.btn:hover,.btn-group>.btn:focus,.btn-group-vertical>.btn:focus,.btn-group>.btn:active,.btn-group-vertical>.btn:active,.btn-group>.btn.active,.btn-group-vertical>.btn.active{z-index:2}.btn-group .btn+.btn,.btn-group .btn+.btn-group,.btn-group .btn-group+.btn,.btn-group .btn-group+.btn-group{margin-left:-1px}.btn-toolbar{margin-left:-5px}.btn-toolbar .btn-group,.btn-toolbar .input-group{float:left}.btn-toolbar>.btn,.btn-toolbar>.btn-group,.btn-toolbar>.input-group{margin-left:5px}.btn-group>.btn:not(:first-child):not(:last-child):not(.dropdown-toggle){border-radius:0}.btn-group>.btn:first-child{margin-left:0}.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle){border-bottom-right-radius:0;border-top-right-radius:0}.btn-group>.btn:last-child:not(:first-child),.btn-group>.dropdown-toggle:not(:first-child){border-bottom-left-radius:0;border-top-left-radius:0}.btn-group>.btn-group{float:left}.btn-group>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group>.btn-group:first-child>.btn:last-child,.btn-group>.btn-group:first-child>.dropdown-toggle{border-bottom-right-radius:0;border-top-right-radius:0}.btn-group>.btn-group:last-child>.btn:first-child{border-bottom-left-radius:0;border-top-left-radius:0}.btn-group .dropdown-toggle:active,.btn-group.open .dropdown-toggle{outline:0}.btn-group>.btn+.dropdown-toggle{padding-left:8px;padding-right:8px}.btn-group>.btn-lg+.dropdown-toggle{padding-left:12px;padding-right:12px}.btn-group.open .dropdown-toggle{-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,0.125);box-shadow:inset 0 3px 5px rgba(0,0,0,0.125)}.btn-group.open .dropdown-toggle.btn-link{-webkit-box-shadow:none;box-shadow:none}.btn .caret{margin-left:0}.btn-lg .caret{border-width:5px 5px 0;border-bottom-width:0}.dropup .btn-lg .caret{border-width:0 5px 5px}.btn-group-vertical>.btn,.btn-group-vertical>.btn-group,.btn-group-vertical>.btn-group>.btn{display:block;float:none;width:100%;max-width:100%}.btn-group-vertical>.btn-group>.btn{float:none}.btn-group-vertical>.btn+.btn,.btn-group-vertical>.btn+.btn-group,.btn-group-vertical>.btn-group+.btn,.btn-group-vertical>.btn-group+.btn-group{margin-top:-1px;margin-left:0}.btn-group-vertical>.btn:not(:first-child):not(:last-child){border-radius:0}.btn-group-vertical>.btn:first-child:not(:last-child){border-top-right-radius:2px;border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn:last-child:not(:first-child){border-bottom-left-radius:2px;border-top-right-radius:0;border-top-left-radius:0}.btn-group-vertical>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group-vertical>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group-vertical>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-right-radius:0;border-top-left-radius:0}.btn-group-justified{display:table;width:100%;table-layout:fixed;border-collapse:separate}.btn-group-justified>.btn,.btn-group-justified>.btn-group{float:none;display:table-cell;width:1%}.btn-group-justified>.btn-group .btn{width:100%}.btn-group-justified>.btn-group .dropdown-menu{left:auto}[data-toggle="buttons"]>.btn input[type="radio"],[data-toggle="buttons"]>.btn-group>.btn input[type="radio"],[data-toggle="buttons"]>.btn input[type="checkbox"],[data-toggle="buttons"]>.btn-group>.btn input[type="checkbox"]{position:absolute;clip:rect(0, 0, 0, 0);pointer-events:none}.input-group{position:relative;display:table;border-collapse:separate}.input-group[class*="col-"]{float:none;padding-left:0;padding-right:0}.input-group .form-control{position:relative;z-index:2;float:left;width:100%;margin-bottom:0}.input-group-lg>.form-control,.input-group-lg>.input-group-addon,.input-group-lg>.input-group-btn>.btn{height:45px;padding:10px 16px;font-size:17px;line-height:1.33;border-radius:3px}select.input-group-lg>.form-control,select.input-group-lg>.input-group-addon,select.input-group-lg>.input-group-btn>.btn{height:45px;line-height:45px}textarea.input-group-lg>.form-control,textarea.input-group-lg>.input-group-addon,textarea.input-group-lg>.input-group-btn>.btn,select[multiple].input-group-lg>.form-control,select[multiple].input-group-lg>.input-group-addon,select[multiple].input-group-lg>.input-group-btn>.btn{height:auto}.input-group-sm>.form-control,.input-group-sm>.input-group-addon,.input-group-sm>.input-group-btn>.btn{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:1px}select.input-group-sm>.form-control,select.input-group-sm>.input-group-addon,select.input-group-sm>.input-group-btn>.btn{height:30px;line-height:30px}textarea.input-group-sm>.form-control,textarea.input-group-sm>.input-group-addon,textarea.input-group-sm>.input-group-btn>.btn,select[multiple].input-group-sm>.form-control,select[multiple].input-group-sm>.input-group-addon,select[multiple].input-group-sm>.input-group-btn>.btn{height:auto}.input-group-addon,.input-group-btn,.input-group .form-control{display:table-cell}.input-group-addon:not(:first-child):not(:last-child),.input-group-btn:not(:first-child):not(:last-child),.input-group .form-control:not(:first-child):not(:last-child){border-radius:0}.input-group-addon,.input-group-btn{width:1%;white-space:nowrap;vertical-align:middle}.input-group-addon{padding:6px 12px;font-size:13px;font-weight:normal;line-height:1;color:#555;text-align:center;background-color:#eee;border:1px solid #ccc;border-radius:2px}.input-group-addon.input-sm{padding:5px 10px;font-size:12px;border-radius:1px}.input-group-addon.input-lg{padding:10px 16px;font-size:17px;border-radius:3px}.input-group-addon input[type="radio"],.input-group-addon input[type="checkbox"]{margin-top:0}.input-group .form-control:first-child,.input-group-addon:first-child,.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group>.btn,.input-group-btn:first-child>.dropdown-toggle,.input-group-btn:last-child>.btn:not(:last-child):not(.dropdown-toggle),.input-group-btn:last-child>.btn-group:not(:last-child)>.btn{border-bottom-right-radius:0;border-top-right-radius:0}.input-group-addon:first-child{border-right:0}.input-group .form-control:last-child,.input-group-addon:last-child,.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group>.btn,.input-group-btn:last-child>.dropdown-toggle,.input-group-btn:first-child>.btn:not(:first-child),.input-group-btn:first-child>.btn-group:not(:first-child)>.btn{border-bottom-left-radius:0;border-top-left-radius:0}.input-group-addon:last-child{border-left:0}.input-group-btn{position:relative;font-size:0;white-space:nowrap}.input-group-btn>.btn{position:relative}.input-group-btn>.btn+.btn{margin-left:-1px}.input-group-btn>.btn:hover,.input-group-btn>.btn:focus,.input-group-btn>.btn:active{z-index:2}.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group{margin-right:-1px}.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group{margin-left:-1px}.nav{margin-bottom:0;padding-left:0;list-style:none}.nav>li{position:relative;display:block}.nav>li>a{position:relative;display:block;padding:10px 15px}.nav>li>a:hover,.nav>li>a:focus{text-decoration:none;background-color:#eee}.nav>li.disabled>a{color:#777}.nav>li.disabled>a:hover,.nav>li.disabled>a:focus{color:#777;text-decoration:none;background-color:transparent;cursor:not-allowed}.nav .open>a,.nav .open>a:hover,.nav .open>a:focus{background-color:#eee;border-color:#337ab7}.nav .nav-divider{height:1px;margin:8px 0;overflow:hidden;background-color:#e5e5e5}.nav>li>a>img{max-width:none}.nav-tabs{border-bottom:1px solid #ddd}.nav-tabs>li{float:left;margin-bottom:-1px}.nav-tabs>li>a{margin-right:2px;line-height:1.42857143;border:1px solid transparent;border-radius:2px 2px 0 0}.nav-tabs>li>a:hover{border-color:#eee #eee #ddd}.nav-tabs>li.active>a,.nav-tabs>li.active>a:hover,.nav-tabs>li.active>a:focus{color:#555;background-color:#fff;border:1px solid #ddd;border-bottom-color:transparent;cursor:default}.nav-tabs.nav-justified{width:100%;border-bottom:0}.nav-tabs.nav-justified>li{float:none}.nav-tabs.nav-justified>li>a{text-align:center;margin-bottom:5px}.nav-tabs.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-tabs.nav-justified>li{display:table-cell;width:1%}.nav-tabs.nav-justified>li>a{margin-bottom:0}}.nav-tabs.nav-justified>li>a{margin-right:0;border-radius:2px}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:hover,.nav-tabs.nav-justified>.active>a:focus{border:1px solid #ddd}@media (min-width:768px){.nav-tabs.nav-justified>li>a{border-bottom:1px solid #ddd;border-radius:2px 2px 0 0}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:hover,.nav-tabs.nav-justified>.active>a:focus{border-bottom-color:#fff}}.nav-pills>li{float:left}.nav-pills>li>a{border-radius:2px}.nav-pills>li+li{margin-left:2px}.nav-pills>li.active>a,.nav-pills>li.active>a:hover,.nav-pills>li.active>a:focus{color:#fff;background-color:#337ab7}.nav-stacked>li{float:none}.nav-stacked>li+li{margin-top:2px;margin-left:0}.nav-justified{width:100%}.nav-justified>li{float:none}.nav-justified>li>a{text-align:center;margin-bottom:5px}.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-justified>li{display:table-cell;width:1%}.nav-justified>li>a{margin-bottom:0}}.nav-tabs-justified{border-bottom:0}.nav-tabs-justified>li>a{margin-right:0;border-radius:2px}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:hover,.nav-tabs-justified>.active>a:focus{border:1px solid #ddd}@media (min-width:768px){.nav-tabs-justified>li>a{border-bottom:1px solid #ddd;border-radius:2px 2px 0 0}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:hover,.nav-tabs-justified>.active>a:focus{border-bottom-color:#fff}}.tab-content>.tab-pane{display:none;visibility:hidden}.tab-content>.active{display:block;visibility:visible}.nav-tabs .dropdown-menu{margin-top:-1px;border-top-right-radius:0;border-top-left-radius:0}.navbar{position:relative;min-height:30px;margin-bottom:18px;border:1px solid transparent}@media (min-width:541px){.navbar{border-radius:2px}}@media (min-width:541px){.navbar-header{float:left}}.navbar-collapse{overflow-x:visible;padding-right:0;padding-left:0;border-top:1px solid transparent;box-shadow:inset 0 1px 0 rgba(255,255,255,0.1);-webkit-overflow-scrolling:touch}.navbar-collapse.in{overflow-y:auto}@media (min-width:541px){.navbar-collapse{width:auto;border-top:0;box-shadow:none}.navbar-collapse.collapse{display:block !important;visibility:visible !important;height:auto !important;padding-bottom:0;overflow:visible !important}.navbar-collapse.in{overflow-y:visible}.navbar-fixed-top .navbar-collapse,.navbar-static-top .navbar-collapse,.navbar-fixed-bottom .navbar-collapse{padding-left:0;padding-right:0}}.navbar-fixed-top .navbar-collapse,.navbar-fixed-bottom .navbar-collapse{max-height:340px}@media (max-device-width:540px) and (orientation:landscape){.navbar-fixed-top .navbar-collapse,.navbar-fixed-bottom .navbar-collapse{max-height:200px}}.container>.navbar-header,.container-fluid>.navbar-header,.container>.navbar-collapse,.container-fluid>.navbar-collapse{margin-right:0;margin-left:0}@media (min-width:541px){.container>.navbar-header,.container-fluid>.navbar-header,.container>.navbar-collapse,.container-fluid>.navbar-collapse{margin-right:0;margin-left:0}}.navbar-static-top{z-index:1000;border-width:0 0 1px}@media (min-width:541px){.navbar-static-top{border-radius:0}}.navbar-fixed-top,.navbar-fixed-bottom{position:fixed;right:0;left:0;z-index:1030}@media (min-width:541px){.navbar-fixed-top,.navbar-fixed-bottom{border-radius:0}}.navbar-fixed-top{top:0;border-width:0 0 1px}.navbar-fixed-bottom{bottom:0;margin-bottom:0;border-width:1px 0 0}.navbar-brand{float:left;padding:6px 0;font-size:17px;line-height:18px;height:30px}.navbar-brand:hover,.navbar-brand:focus{text-decoration:none}.navbar-brand>img{display:block}@media (min-width:541px){.navbar>.container .navbar-brand,.navbar>.container-fluid .navbar-brand{margin-left:0}}.navbar-toggle{position:relative;float:right;margin-right:0;padding:9px 10px;margin-top:-2px;margin-bottom:-2px;background-color:transparent;background-image:none;border:1px solid transparent;border-radius:2px}.navbar-toggle:focus{outline:0}.navbar-toggle .icon-bar{display:block;width:22px;height:2px;border-radius:1px}.navbar-toggle .icon-bar+.icon-bar{margin-top:4px}@media (min-width:541px){.navbar-toggle{display:none}}.navbar-nav{margin:3px 0}.navbar-nav>li>a{padding-top:10px;padding-bottom:10px;line-height:18px}@media (max-width:540px){.navbar-nav .open .dropdown-menu{position:static;float:none;width:auto;margin-top:0;background-color:transparent;border:0;box-shadow:none}.navbar-nav .open .dropdown-menu>li>a,.navbar-nav .open .dropdown-menu .dropdown-header{padding:5px 15px 5px 25px}.navbar-nav .open .dropdown-menu>li>a{line-height:18px}.navbar-nav .open .dropdown-menu>li>a:hover,.navbar-nav .open .dropdown-menu>li>a:focus{background-image:none}}@media (min-width:541px){.navbar-nav{float:left;margin:0}.navbar-nav>li{float:left}.navbar-nav>li>a{padding-top:6px;padding-bottom:6px}}.navbar-form{margin-left:0;margin-right:0;padding:10px 0;border-top:1px solid transparent;border-bottom:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,0.1),0 1px 0 rgba(255,255,255,0.1);box-shadow:inset 0 1px 0 rgba(255,255,255,0.1),0 1px 0 rgba(255,255,255,0.1);margin-top:-1px;margin-bottom:-1px}@media (min-width:768px){.navbar-form .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.navbar-form .form-control{display:inline-block;width:auto;vertical-align:middle}.navbar-form .form-control-static{display:inline-block}.navbar-form .input-group{display:inline-table;vertical-align:middle}.navbar-form .input-group .input-group-addon,.navbar-form .input-group .input-group-btn,.navbar-form .input-group .form-control{width:auto}.navbar-form .input-group>.form-control{width:100%}.navbar-form .control-label{margin-bottom:0;vertical-align:middle}.navbar-form .radio,.navbar-form .checkbox{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.navbar-form .radio label,.navbar-form .checkbox label{padding-left:0}.navbar-form .radio input[type="radio"],.navbar-form .checkbox input[type="checkbox"]{position:relative;margin-left:0}.navbar-form .has-feedback .form-control-feedback{top:0}}@media (max-width:540px){.navbar-form .form-group{margin-bottom:5px}.navbar-form .form-group:last-child{margin-bottom:0}}@media (min-width:541px){.navbar-form{width:auto;border:0;margin-left:0;margin-right:0;padding-top:0;padding-bottom:0;-webkit-box-shadow:none;box-shadow:none}}.navbar-nav>li>.dropdown-menu{margin-top:0;border-top-right-radius:0;border-top-left-radius:0}.navbar-fixed-bottom .navbar-nav>li>.dropdown-menu{border-top-right-radius:2px;border-top-left-radius:2px;border-bottom-right-radius:0;border-bottom-left-radius:0}.navbar-btn{margin-top:-1px;margin-bottom:-1px}.navbar-btn.btn-sm{margin-top:0;margin-bottom:0}.navbar-btn.btn-xs{margin-top:4px;margin-bottom:4px}.navbar-text{margin-top:6px;margin-bottom:6px}@media (min-width:541px){.navbar-text{float:left;margin-left:0;margin-right:0}}@media (min-width:541px){.navbar-left{float:left !important;float:left}.navbar-right{float:right !important;float:right;margin-right:0}.navbar-right~.navbar-right{margin-right:0}}.navbar-default{background-color:#f8f8f8;border-color:#e7e7e7}.navbar-default .navbar-brand{color:#777}.navbar-default .navbar-brand:hover,.navbar-default .navbar-brand:focus{color:#5e5e5e;background-color:transparent}.navbar-default .navbar-text{color:#777}.navbar-default .navbar-nav>li>a{color:#777}.navbar-default .navbar-nav>li>a:hover,.navbar-default .navbar-nav>li>a:focus{color:#333;background-color:transparent}.navbar-default .navbar-nav>.active>a,.navbar-default .navbar-nav>.active>a:hover,.navbar-default .navbar-nav>.active>a:focus{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav>.disabled>a,.navbar-default .navbar-nav>.disabled>a:hover,.navbar-default .navbar-nav>.disabled>a:focus{color:#ccc;background-color:transparent}.navbar-default .navbar-toggle{border-color:#ddd}.navbar-default .navbar-toggle:hover,.navbar-default .navbar-toggle:focus{background-color:#ddd}.navbar-default .navbar-toggle .icon-bar{background-color:#888}.navbar-default .navbar-collapse,.navbar-default .navbar-form{border-color:#e7e7e7}.navbar-default .navbar-nav>.open>a,.navbar-default .navbar-nav>.open>a:hover,.navbar-default .navbar-nav>.open>a:focus{background-color:#e7e7e7;color:#555}@media (max-width:540px){.navbar-default .navbar-nav .open .dropdown-menu>li>a{color:#777}.navbar-default .navbar-nav .open .dropdown-menu>li>a:hover,.navbar-default .navbar-nav .open .dropdown-menu>li>a:focus{color:#333;background-color:transparent}.navbar-default .navbar-nav .open .dropdown-menu>.active>a,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:hover,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:focus{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:hover,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:focus{color:#ccc;background-color:transparent}}.navbar-default .navbar-link{color:#777}.navbar-default .navbar-link:hover{color:#333}.navbar-default .btn-link{color:#777}.navbar-default .btn-link:hover,.navbar-default .btn-link:focus{color:#333}.navbar-default .btn-link[disabled]:hover,fieldset[disabled] .navbar-default .btn-link:hover,.navbar-default .btn-link[disabled]:focus,fieldset[disabled] .navbar-default .btn-link:focus{color:#ccc}.navbar-inverse{background-color:#222;border-color:#080808}.navbar-inverse .navbar-brand{color:#9d9d9d}.navbar-inverse .navbar-brand:hover,.navbar-inverse .navbar-brand:focus{color:#fff;background-color:transparent}.navbar-inverse .navbar-text{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a:hover,.navbar-inverse .navbar-nav>li>a:focus{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav>.active>a,.navbar-inverse .navbar-nav>.active>a:hover,.navbar-inverse .navbar-nav>.active>a:focus{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav>.disabled>a,.navbar-inverse .navbar-nav>.disabled>a:hover,.navbar-inverse .navbar-nav>.disabled>a:focus{color:#444;background-color:transparent}.navbar-inverse .navbar-toggle{border-color:#333}.navbar-inverse .navbar-toggle:hover,.navbar-inverse .navbar-toggle:focus{background-color:#333}.navbar-inverse .navbar-toggle .icon-bar{background-color:#fff}.navbar-inverse .navbar-collapse,.navbar-inverse .navbar-form{border-color:#101010}.navbar-inverse .navbar-nav>.open>a,.navbar-inverse .navbar-nav>.open>a:hover,.navbar-inverse .navbar-nav>.open>a:focus{background-color:#080808;color:#fff}@media (max-width:540px){.navbar-inverse .navbar-nav .open .dropdown-menu>.dropdown-header{border-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu .divider{background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:hover,.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:focus{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:hover,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:focus{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:hover,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:focus{color:#444;background-color:transparent}}.navbar-inverse .navbar-link{color:#9d9d9d}.navbar-inverse .navbar-link:hover{color:#fff}.navbar-inverse .btn-link{color:#9d9d9d}.navbar-inverse .btn-link:hover,.navbar-inverse .btn-link:focus{color:#fff}.navbar-inverse .btn-link[disabled]:hover,fieldset[disabled] .navbar-inverse .btn-link:hover,.navbar-inverse .btn-link[disabled]:focus,fieldset[disabled] .navbar-inverse .btn-link:focus{color:#444}.breadcrumb{padding:8px 15px;margin-bottom:18px;list-style:none;background-color:#f5f5f5;border-radius:2px}.breadcrumb>li{display:inline-block}.breadcrumb>li+li:before{content:"/\00a0";padding:0 5px;color:#5e5e5e}.breadcrumb>.active{color:#777}.pagination{display:inline-block;padding-left:0;margin:18px 0;border-radius:2px}.pagination>li{display:inline}.pagination>li>a,.pagination>li>span{position:relative;float:left;padding:6px 12px;line-height:1.42857143;text-decoration:none;color:#337ab7;background-color:#fff;border:1px solid #ddd;margin-left:-1px}.pagination>li:first-child>a,.pagination>li:first-child>span{margin-left:0;border-bottom-left-radius:2px;border-top-left-radius:2px}.pagination>li:last-child>a,.pagination>li:last-child>span{border-bottom-right-radius:2px;border-top-right-radius:2px}.pagination>li>a:hover,.pagination>li>span:hover,.pagination>li>a:focus,.pagination>li>span:focus{color:#23527c;background-color:#eee;border-color:#ddd}.pagination>.active>a,.pagination>.active>span,.pagination>.active>a:hover,.pagination>.active>span:hover,.pagination>.active>a:focus,.pagination>.active>span:focus{z-index:2;color:#fff;background-color:#337ab7;border-color:#337ab7;cursor:default}.pagination>.disabled>span,.pagination>.disabled>span:hover,.pagination>.disabled>span:focus,.pagination>.disabled>a,.pagination>.disabled>a:hover,.pagination>.disabled>a:focus{color:#777;background-color:#fff;border-color:#ddd;cursor:not-allowed}.pagination-lg>li>a,.pagination-lg>li>span{padding:10px 16px;font-size:17px}.pagination-lg>li:first-child>a,.pagination-lg>li:first-child>span{border-bottom-left-radius:3px;border-top-left-radius:3px}.pagination-lg>li:last-child>a,.pagination-lg>li:last-child>span{border-bottom-right-radius:3px;border-top-right-radius:3px}.pagination-sm>li>a,.pagination-sm>li>span{padding:5px 10px;font-size:12px}.pagination-sm>li:first-child>a,.pagination-sm>li:first-child>span{border-bottom-left-radius:1px;border-top-left-radius:1px}.pagination-sm>li:last-child>a,.pagination-sm>li:last-child>span{border-bottom-right-radius:1px;border-top-right-radius:1px}.pager{padding-left:0;margin:18px 0;list-style:none;text-align:center}.pager li{display:inline}.pager li>a,.pager li>span{display:inline-block;padding:5px 14px;background-color:#fff;border:1px solid #ddd;border-radius:15px}.pager li>a:hover,.pager li>a:focus{text-decoration:none;background-color:#eee}.pager .next>a,.pager .next>span{float:right}.pager .previous>a,.pager .previous>span{float:left}.pager .disabled>a,.pager .disabled>a:hover,.pager .disabled>a:focus,.pager .disabled>span{color:#777;background-color:#fff;cursor:not-allowed}.label{display:inline;padding:.2em .6em .3em;font-size:75%;font-weight:bold;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:baseline;border-radius:.25em}a.label:hover,a.label:focus{color:#fff;text-decoration:none;cursor:pointer}.label:empty{display:none}.btn .label{position:relative;top:-1px}.label-default{background-color:#777}.label-default[href]:hover,.label-default[href]:focus{background-color:#5e5e5e}.label-primary{background-color:#337ab7}.label-primary[href]:hover,.label-primary[href]:focus{background-color:#286090}.label-success{background-color:#5cb85c}.label-success[href]:hover,.label-success[href]:focus{background-color:#449d44}.label-info{background-color:#5bc0de}.label-info[href]:hover,.label-info[href]:focus{background-color:#31b0d5}.label-warning{background-color:#f0ad4e}.label-warning[href]:hover,.label-warning[href]:focus{background-color:#ec971f}.label-danger{background-color:#d9534f}.label-danger[href]:hover,.label-danger[href]:focus{background-color:#c9302c}.badge{display:inline-block;min-width:10px;padding:3px 7px;font-size:12px;font-weight:bold;color:#fff;line-height:1;vertical-align:baseline;white-space:nowrap;text-align:center;background-color:#777;border-radius:10px}.badge:empty{display:none}.btn .badge{position:relative;top:-1px}.btn-xs .badge{top:0;padding:1px 5px}a.badge:hover,a.badge:focus{color:#fff;text-decoration:none;cursor:pointer}.list-group-item.active>.badge,.nav-pills>.active>a>.badge{color:#337ab7;background-color:#fff}.list-group-item>.badge{float:right}.list-group-item>.badge+.badge{margin-right:5px}.nav-pills>li>a>.badge{margin-left:3px}.jumbotron{padding:30px 15px;margin-bottom:30px;color:inherit;background-color:#eee}.jumbotron h1,.jumbotron .h1{color:inherit}.jumbotron p{margin-bottom:15px;font-size:20px;font-weight:200}.jumbotron>hr{border-top-color:#d5d5d5}.container .jumbotron,.container-fluid .jumbotron{border-radius:3px}.jumbotron .container{max-width:100%}@media screen and (min-width:768px){.jumbotron{padding:48px 0}.container .jumbotron,.container-fluid .jumbotron{padding-left:60px;padding-right:60px}.jumbotron h1,.jumbotron .h1{font-size:58.5px}}.thumbnail{display:block;padding:4px;margin-bottom:18px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:2px;-webkit-transition:border .2s ease-in-out;-o-transition:border .2s ease-in-out;transition:border .2s ease-in-out}.thumbnail>img,.thumbnail a>img{margin-left:auto;margin-right:auto}a.thumbnail:hover,a.thumbnail:focus,a.thumbnail.active{border-color:#337ab7}.thumbnail .caption{padding:9px;color:#000}.alert{padding:15px;margin-bottom:18px;border:1px solid transparent;border-radius:2px}.alert h4{margin-top:0;color:inherit}.alert .alert-link{font-weight:bold}.alert>p,.alert>ul{margin-bottom:0}.alert>p+p{margin-top:5px}.alert-dismissable,.alert-dismissible{padding-right:35px}.alert-dismissable .close,.alert-dismissible .close{position:relative;top:-2px;right:-21px;color:inherit}.alert-success{background-color:#dff0d8;border-color:#d6e9c6;color:#3c763d}.alert-success hr{border-top-color:#c9e2b3}.alert-success .alert-link{color:#2b542c}.alert-info{background-color:#d9edf7;border-color:#bce8f1;color:#31708f}.alert-info hr{border-top-color:#a6e1ec}.alert-info .alert-link{color:#245269}.alert-warning{background-color:#fcf8e3;border-color:#faebcc;color:#8a6d3b}.alert-warning hr{border-top-color:#f7e1b5}.alert-warning .alert-link{color:#66512c}.alert-danger{background-color:#f2dede;border-color:#ebccd1;color:#a94442}.alert-danger hr{border-top-color:#e4b9c0}.alert-danger .alert-link{color:#843534}@-webkit-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}.progress{overflow:hidden;height:18px;margin-bottom:18px;background-color:#f5f5f5;border-radius:2px;-webkit-box-shadow:inset 0 1px 2px rgba(0,0,0,0.1);box-shadow:inset 0 1px 2px rgba(0,0,0,0.1)}.progress-bar{float:left;width:0;height:100%;font-size:12px;line-height:18px;color:#fff;text-align:center;background-color:#337ab7;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,0.15);box-shadow:inset 0 -1px 0 rgba(0,0,0,0.15);-webkit-transition:width .6s ease;-o-transition:width .6s ease;transition:width .6s ease}.progress-striped .progress-bar,.progress-bar-striped{background-image:-webkit-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:-o-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-size:40px 40px}.progress.active .progress-bar,.progress-bar.active{-webkit-animation:progress-bar-stripes 2s linear infinite;-o-animation:progress-bar-stripes 2s linear infinite;animation:progress-bar-stripes 2s linear infinite}.progress-bar-success{background-color:#5cb85c}.progress-striped .progress-bar-success{background-image:-webkit-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:-o-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent)}.progress-bar-info{background-color:#5bc0de}.progress-striped .progress-bar-info{background-image:-webkit-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:-o-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent)}.progress-bar-warning{background-color:#f0ad4e}.progress-striped .progress-bar-warning{background-image:-webkit-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:-o-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent)}.progress-bar-danger{background-color:#d9534f}.progress-striped .progress-bar-danger{background-image:-webkit-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:-o-linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent);background-image:linear-gradient(45deg, rgba(255,255,255,0.15) 25%, transparent 25%, transparent 50%, rgba(255,255,255,0.15) 50%, rgba(255,255,255,0.15) 75%, transparent 75%, transparent)}.media{margin-top:15px}.media:first-child{margin-top:0}.media-right,.media>.pull-right{padding-left:10px}.media-left,.media>.pull-left{padding-right:10px}.media-left,.media-right,.media-body{display:table-cell;vertical-align:top}.media-middle{vertical-align:middle}.media-bottom{vertical-align:bottom}.media-heading{margin-top:0;margin-bottom:5px}.media-list{padding-left:0;list-style:none}.list-group{margin-bottom:20px;padding-left:0}.list-group-item{position:relative;display:block;padding:10px 15px;margin-bottom:-1px;background-color:#fff;border:1px solid #ddd}.list-group-item:first-child{border-top-right-radius:2px;border-top-left-radius:2px}.list-group-item:last-child{margin-bottom:0;border-bottom-right-radius:2px;border-bottom-left-radius:2px}a.list-group-item{color:#555}a.list-group-item .list-group-item-heading{color:#333}a.list-group-item:hover,a.list-group-item:focus{text-decoration:none;color:#555;background-color:#f5f5f5}.list-group-item.disabled,.list-group-item.disabled:hover,.list-group-item.disabled:focus{background-color:#eee;color:#777;cursor:not-allowed}.list-group-item.disabled .list-group-item-heading,.list-group-item.disabled:hover .list-group-item-heading,.list-group-item.disabled:focus .list-group-item-heading{color:inherit}.list-group-item.disabled .list-group-item-text,.list-group-item.disabled:hover .list-group-item-text,.list-group-item.disabled:focus .list-group-item-text{color:#777}.list-group-item.active,.list-group-item.active:hover,.list-group-item.active:focus{z-index:2;color:#fff;background-color:#337ab7;border-color:#337ab7}.list-group-item.active .list-group-item-heading,.list-group-item.active:hover .list-group-item-heading,.list-group-item.active:focus .list-group-item-heading,.list-group-item.active .list-group-item-heading>small,.list-group-item.active:hover .list-group-item-heading>small,.list-group-item.active:focus .list-group-item-heading>small,.list-group-item.active .list-group-item-heading>.small,.list-group-item.active:hover .list-group-item-heading>.small,.list-group-item.active:focus .list-group-item-heading>.small{color:inherit}.list-group-item.active .list-group-item-text,.list-group-item.active:hover .list-group-item-text,.list-group-item.active:focus .list-group-item-text{color:#c7ddef}.list-group-item-success{color:#3c763d;background-color:#dff0d8}a.list-group-item-success{color:#3c763d}a.list-group-item-success .list-group-item-heading{color:inherit}a.list-group-item-success:hover,a.list-group-item-success:focus{color:#3c763d;background-color:#d0e9c6}a.list-group-item-success.active,a.list-group-item-success.active:hover,a.list-group-item-success.active:focus{color:#fff;background-color:#3c763d;border-color:#3c763d}.list-group-item-info{color:#31708f;background-color:#d9edf7}a.list-group-item-info{color:#31708f}a.list-group-item-info .list-group-item-heading{color:inherit}a.list-group-item-info:hover,a.list-group-item-info:focus{color:#31708f;background-color:#c4e3f3}a.list-group-item-info.active,a.list-group-item-info.active:hover,a.list-group-item-info.active:focus{color:#fff;background-color:#31708f;border-color:#31708f}.list-group-item-warning{color:#8a6d3b;background-color:#fcf8e3}a.list-group-item-warning{color:#8a6d3b}a.list-group-item-warning .list-group-item-heading{color:inherit}a.list-group-item-warning:hover,a.list-group-item-warning:focus{color:#8a6d3b;background-color:#faf2cc}a.list-group-item-warning.active,a.list-group-item-warning.active:hover,a.list-group-item-warning.active:focus{color:#fff;background-color:#8a6d3b;border-color:#8a6d3b}.list-group-item-danger{color:#a94442;background-color:#f2dede}a.list-group-item-danger{color:#a94442}a.list-group-item-danger .list-group-item-heading{color:inherit}a.list-group-item-danger:hover,a.list-group-item-danger:focus{color:#a94442;background-color:#ebcccc}a.list-group-item-danger.active,a.list-group-item-danger.active:hover,a.list-group-item-danger.active:focus{color:#fff;background-color:#a94442;border-color:#a94442}.list-group-item-heading{margin-top:0;margin-bottom:5px}.list-group-item-text{margin-bottom:0;line-height:1.3}.panel{margin-bottom:18px;background-color:#fff;border:1px solid transparent;border-radius:2px;-webkit-box-shadow:0 1px 1px rgba(0,0,0,0.05);box-shadow:0 1px 1px rgba(0,0,0,0.05)}.panel-body{padding:15px}.panel-heading{padding:10px 15px;border-bottom:1px solid transparent;border-top-right-radius:1px;border-top-left-radius:1px}.panel-heading>.dropdown .dropdown-toggle{color:inherit}.panel-title{margin-top:0;margin-bottom:0;font-size:15px;color:inherit}.panel-title>a{color:inherit}.panel-footer{padding:10px 15px;background-color:#f5f5f5;border-top:1px solid #ddd;border-bottom-right-radius:1px;border-bottom-left-radius:1px}.panel>.list-group,.panel>.panel-collapse>.list-group{margin-bottom:0}.panel>.list-group .list-group-item,.panel>.panel-collapse>.list-group .list-group-item{border-width:1px 0;border-radius:0}.panel>.list-group:first-child .list-group-item:first-child,.panel>.panel-collapse>.list-group:first-child .list-group-item:first-child{border-top:0;border-top-right-radius:1px;border-top-left-radius:1px}.panel>.list-group:last-child .list-group-item:last-child,.panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{border-bottom:0;border-bottom-right-radius:1px;border-bottom-left-radius:1px}.panel-heading+.list-group .list-group-item:first-child{border-top-width:0}.list-group+.panel-footer{border-top-width:0}.panel>.table,.panel>.table-responsive>.table,.panel>.panel-collapse>.table{margin-bottom:0}.panel>.table caption,.panel>.table-responsive>.table caption,.panel>.panel-collapse>.table caption{padding-left:15px;padding-right:15px}.panel>.table:first-child,.panel>.table-responsive:first-child>.table:first-child{border-top-right-radius:1px;border-top-left-radius:1px}.panel>.table:first-child>thead:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child{border-top-left-radius:1px;border-top-right-radius:1px}.panel>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table:first-child>thead:first-child>tr:first-child th:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:first-child{border-top-left-radius:1px}.panel>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table:first-child>thead:first-child>tr:first-child th:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:last-child{border-top-right-radius:1px}.panel>.table:last-child,.panel>.table-responsive:last-child>.table:last-child{border-bottom-right-radius:1px;border-bottom-left-radius:1px}.panel>.table:last-child>tbody:last-child>tr:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child{border-bottom-left-radius:1px;border-bottom-right-radius:1px}.panel>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:first-child{border-bottom-left-radius:1px}.panel>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:last-child{border-bottom-right-radius:1px}.panel>.panel-body+.table,.panel>.panel-body+.table-responsive,.panel>.table+.panel-body,.panel>.table-responsive+.panel-body{border-top:1px solid #ddd}.panel>.table>tbody:first-child>tr:first-child th,.panel>.table>tbody:first-child>tr:first-child td{border-top:0}.panel>.table-bordered,.panel>.table-responsive>.table-bordered{border:0}.panel>.table-bordered>thead>tr>th:first-child,.panel>.table-responsive>.table-bordered>thead>tr>th:first-child,.panel>.table-bordered>tbody>tr>th:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:first-child,.panel>.table-bordered>tfoot>tr>th:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:first-child,.panel>.table-bordered>thead>tr>td:first-child,.panel>.table-responsive>.table-bordered>thead>tr>td:first-child,.panel>.table-bordered>tbody>tr>td:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:first-child,.panel>.table-bordered>tfoot>tr>td:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:first-child{border-left:0}.panel>.table-bordered>thead>tr>th:last-child,.panel>.table-responsive>.table-bordered>thead>tr>th:last-child,.panel>.table-bordered>tbody>tr>th:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:last-child,.panel>.table-bordered>tfoot>tr>th:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:last-child,.panel>.table-bordered>thead>tr>td:last-child,.panel>.table-responsive>.table-bordered>thead>tr>td:last-child,.panel>.table-bordered>tbody>tr>td:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:last-child,.panel>.table-bordered>tfoot>tr>td:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:last-child{border-right:0}.panel>.table-bordered>thead>tr:first-child>td,.panel>.table-responsive>.table-bordered>thead>tr:first-child>td,.panel>.table-bordered>tbody>tr:first-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>td,.panel>.table-bordered>thead>tr:first-child>th,.panel>.table-responsive>.table-bordered>thead>tr:first-child>th,.panel>.table-bordered>tbody>tr:first-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>th{border-bottom:0}.panel>.table-bordered>tbody>tr:last-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>td,.panel>.table-bordered>tfoot>tr:last-child>td,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>td,.panel>.table-bordered>tbody>tr:last-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>th,.panel>.table-bordered>tfoot>tr:last-child>th,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}.panel>.table-responsive{border:0;margin-bottom:0}.panel-group{margin-bottom:18px}.panel-group .panel{margin-bottom:0;border-radius:2px}.panel-group .panel+.panel{margin-top:5px}.panel-group .panel-heading{border-bottom:0}.panel-group .panel-heading+.panel-collapse>.panel-body,.panel-group .panel-heading+.panel-collapse>.list-group{border-top:1px solid #ddd}.panel-group .panel-footer{border-top:0}.panel-group .panel-footer+.panel-collapse .panel-body{border-bottom:1px solid #ddd}.panel-default{border-color:#ddd}.panel-default>.panel-heading{color:#333;background-color:#f5f5f5;border-color:#ddd}.panel-default>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ddd}.panel-default>.panel-heading .badge{color:#f5f5f5;background-color:#333}.panel-default>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ddd}.panel-primary{border-color:#337ab7}.panel-primary>.panel-heading{color:#fff;background-color:#337ab7;border-color:#337ab7}.panel-primary>.panel-heading+.panel-collapse>.panel-body{border-top-color:#337ab7}.panel-primary>.panel-heading .badge{color:#337ab7;background-color:#fff}.panel-primary>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#337ab7}.panel-success{border-color:#d6e9c6}.panel-success>.panel-heading{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.panel-success>.panel-heading+.panel-collapse>.panel-body{border-top-color:#d6e9c6}.panel-success>.panel-heading .badge{color:#dff0d8;background-color:#3c763d}.panel-success>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#d6e9c6}.panel-info{border-color:#bce8f1}.panel-info>.panel-heading{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.panel-info>.panel-heading+.panel-collapse>.panel-body{border-top-color:#bce8f1}.panel-info>.panel-heading .badge{color:#d9edf7;background-color:#31708f}.panel-info>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#bce8f1}.panel-warning{border-color:#faebcc}.panel-warning>.panel-heading{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.panel-warning>.panel-heading+.panel-collapse>.panel-body{border-top-color:#faebcc}.panel-warning>.panel-heading .badge{color:#fcf8e3;background-color:#8a6d3b}.panel-warning>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#faebcc}.panel-danger{border-color:#ebccd1}.panel-danger>.panel-heading{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.panel-danger>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ebccd1}.panel-danger>.panel-heading .badge{color:#f2dede;background-color:#a94442}.panel-danger>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ebccd1}.embed-responsive{position:relative;display:block;height:0;padding:0;overflow:hidden}.embed-responsive .embed-responsive-item,.embed-responsive iframe,.embed-responsive embed,.embed-responsive object,.embed-responsive video{position:absolute;top:0;left:0;bottom:0;height:100%;width:100%;border:0}.embed-responsive.embed-responsive-16by9{padding-bottom:56.25%}.embed-responsive.embed-responsive-4by3{padding-bottom:75%}.well{min-height:20px;padding:19px;margin-bottom:20px;background-color:#f5f5f5;border:1px solid #e3e3e3;border-radius:2px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,0.05);box-shadow:inset 0 1px 1px rgba(0,0,0,0.05)}.well blockquote{border-color:#ddd;border-color:rgba(0,0,0,0.15)}.well-lg{padding:24px;border-radius:3px}.well-sm{padding:9px;border-radius:1px}.close{float:right;font-size:19.5px;font-weight:bold;line-height:1;color:#000;text-shadow:0 1px 0 #fff;opacity:.2;filter:alpha(opacity=20)}.close:hover,.close:focus{color:#000;text-decoration:none;cursor:pointer;opacity:.5;filter:alpha(opacity=50)}button.close{padding:0;cursor:pointer;background:transparent;border:0;-webkit-appearance:none}.modal-open{overflow:hidden}.modal{display:none;overflow:hidden;position:fixed;top:0;right:0;bottom:0;left:0;z-index:1040;-webkit-overflow-scrolling:touch;outline:0}.modal.fade .modal-dialog{-webkit-transform:translate(0, -25%);-ms-transform:translate(0, -25%);-o-transform:translate(0, -25%);transform:translate(0, -25%);-webkit-transition:-webkit-transform 0.3s ease-out;-moz-transition:-moz-transform 0.3s ease-out;-o-transition:-o-transform 0.3s ease-out;transition:transform 0.3s ease-out}.modal.in .modal-dialog{-webkit-transform:translate(0, 0);-ms-transform:translate(0, 0);-o-transform:translate(0, 0);transform:translate(0, 0)}.modal-open .modal{overflow-x:hidden;overflow-y:auto}.modal-dialog{position:relative;width:auto;margin:10px}.modal-content{position:relative;background-color:#fff;border:1px solid #999;border:1px solid rgba(0,0,0,0.2);border-radius:3px;-webkit-box-shadow:0 3px 9px rgba(0,0,0,0.5);box-shadow:0 3px 9px rgba(0,0,0,0.5);background-clip:padding-box;outline:0}.modal-backdrop{position:absolute;top:0;right:0;left:0;background-color:#000}.modal-backdrop.fade{opacity:0;filter:alpha(opacity=0)}.modal-backdrop.in{opacity:.5;filter:alpha(opacity=50)}.modal-header{padding:15px;border-bottom:1px solid #e5e5e5;min-height:16.42857143px}.modal-header .close{margin-top:-2px}.modal-title{margin:0;line-height:1.42857143}.modal-body{position:relative;padding:15px}.modal-footer{padding:15px;text-align:right;border-top:1px solid #e5e5e5}.modal-footer .btn+.btn{margin-left:5px;margin-bottom:0}.modal-footer .btn-group .btn+.btn{margin-left:-1px}.modal-footer .btn-block+.btn-block{margin-left:0}.modal-scrollbar-measure{position:absolute;top:-9999px;width:50px;height:50px;overflow:scroll}@media (min-width:768px){.modal-dialog{width:600px;margin:30px auto}.modal-content{-webkit-box-shadow:0 5px 15px rgba(0,0,0,0.5);box-shadow:0 5px 15px rgba(0,0,0,0.5)}.modal-sm{width:300px}}@media (min-width:992px){.modal-lg{width:900px}}.tooltip{position:absolute;z-index:1070;display:block;visibility:visible;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:12px;font-weight:normal;line-height:1.4;opacity:0;filter:alpha(opacity=0)}.tooltip.in{opacity:.9;filter:alpha(opacity=90)}.tooltip.top{margin-top:-3px;padding:5px 0}.tooltip.right{margin-left:3px;padding:0 5px}.tooltip.bottom{margin-top:3px;padding:5px 0}.tooltip.left{margin-left:-3px;padding:0 5px}.tooltip-inner{max-width:200px;padding:3px 8px;color:#fff;text-align:center;text-decoration:none;background-color:#000;border-radius:2px}.tooltip-arrow{position:absolute;width:0;height:0;border-color:transparent;border-style:solid}.tooltip.top .tooltip-arrow{bottom:0;left:50%;margin-left:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-left .tooltip-arrow{bottom:0;right:5px;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-right .tooltip-arrow{bottom:0;left:5px;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.right .tooltip-arrow{top:50%;left:0;margin-top:-5px;border-width:5px 5px 5px 0;border-right-color:#000}.tooltip.left .tooltip-arrow{top:50%;right:0;margin-top:-5px;border-width:5px 0 5px 5px;border-left-color:#000}.tooltip.bottom .tooltip-arrow{top:0;left:50%;margin-left:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-left .tooltip-arrow{top:0;right:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-right .tooltip-arrow{top:0;left:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.popover{position:absolute;top:0;left:0;z-index:1060;display:none;max-width:276px;padding:1px;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:13px;font-weight:normal;line-height:1.42857143;text-align:left;background-color:#fff;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,0.2);border-radius:3px;-webkit-box-shadow:0 5px 10px rgba(0,0,0,0.2);box-shadow:0 5px 10px rgba(0,0,0,0.2);white-space:normal}.popover.top{margin-top:-10px}.popover.right{margin-left:10px}.popover.bottom{margin-top:10px}.popover.left{margin-left:-10px}.popover-title{margin:0;padding:8px 14px;font-size:13px;background-color:#f7f7f7;border-bottom:1px solid #ebebeb;border-radius:2px 2px 0 0}.popover-content{padding:9px 14px}.popover>.arrow,.popover>.arrow:after{position:absolute;display:block;width:0;height:0;border-color:transparent;border-style:solid}.popover>.arrow{border-width:11px}.popover>.arrow:after{border-width:10px;content:""}.popover.top>.arrow{left:50%;margin-left:-11px;border-bottom-width:0;border-top-color:#999;border-top-color:rgba(0,0,0,0.25);bottom:-11px}.popover.top>.arrow:after{content:" ";bottom:1px;margin-left:-10px;border-bottom-width:0;border-top-color:#fff}.popover.right>.arrow{top:50%;left:-11px;margin-top:-11px;border-left-width:0;border-right-color:#999;border-right-color:rgba(0,0,0,0.25)}.popover.right>.arrow:after{content:" ";left:1px;bottom:-10px;border-left-width:0;border-right-color:#fff}.popover.bottom>.arrow{left:50%;margin-left:-11px;border-top-width:0;border-bottom-color:#999;border-bottom-color:rgba(0,0,0,0.25);top:-11px}.popover.bottom>.arrow:after{content:" ";top:1px;margin-left:-10px;border-top-width:0;border-bottom-color:#fff}.popover.left>.arrow{top:50%;right:-11px;margin-top:-11px;border-right-width:0;border-left-color:#999;border-left-color:rgba(0,0,0,0.25)}.popover.left>.arrow:after{content:" ";right:1px;border-right-width:0;border-left-color:#fff;bottom:-10px}.carousel{position:relative}.carousel-inner{position:relative;overflow:hidden;width:100%}.carousel-inner>.item{display:none;position:relative;-webkit-transition:.6s ease-in-out left;-o-transition:.6s ease-in-out left;transition:.6s ease-in-out left}.carousel-inner>.item>img,.carousel-inner>.item>a>img{line-height:1}@media all and (transform-3d),(-webkit-transform-3d){.carousel-inner>.item{transition:transform .6s ease-in-out;backface-visibility:hidden;perspective:1000}.carousel-inner>.item.next,.carousel-inner>.item.active.right{transform:translate3d(100%, 0, 0);left:0}.carousel-inner>.item.prev,.carousel-inner>.item.active.left{transform:translate3d(-100%, 0, 0);left:0}.carousel-inner>.item.next.left,.carousel-inner>.item.prev.right,.carousel-inner>.item.active{transform:translate3d(0, 0, 0);left:0}}.carousel-inner>.active,.carousel-inner>.next,.carousel-inner>.prev{display:block}.carousel-inner>.active{left:0}.carousel-inner>.next,.carousel-inner>.prev{position:absolute;top:0;width:100%}.carousel-inner>.next{left:100%}.carousel-inner>.prev{left:-100%}.carousel-inner>.next.left,.carousel-inner>.prev.right{left:0}.carousel-inner>.active.left{left:-100%}.carousel-inner>.active.right{left:100%}.carousel-control{position:absolute;top:0;left:0;bottom:0;width:15%;opacity:.5;filter:alpha(opacity=50);font-size:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,0.6)}.carousel-control.left{background-image:-webkit-linear-gradient(left, rgba(0,0,0,0.5) 0, rgba(0,0,0,0.0001) 100%);background-image:-o-linear-gradient(left, rgba(0,0,0,0.5) 0, rgba(0,0,0,0.0001) 100%);background-image:linear-gradient(to right, rgba(0,0,0,0.5) 0, rgba(0,0,0,0.0001) 100%);background-repeat:repeat-x;filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#80000000', endColorstr='#00000000', GradientType=1)}.carousel-control.right{left:auto;right:0;background-image:-webkit-linear-gradient(left, rgba(0,0,0,0.0001) 0, rgba(0,0,0,0.5) 100%);background-image:-o-linear-gradient(left, rgba(0,0,0,0.0001) 0, rgba(0,0,0,0.5) 100%);background-image:linear-gradient(to right, rgba(0,0,0,0.0001) 0, rgba(0,0,0,0.5) 100%);background-repeat:repeat-x;filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#00000000', endColorstr='#80000000', GradientType=1)}.carousel-control:hover,.carousel-control:focus{outline:0;color:#fff;text-decoration:none;opacity:.9;filter:alpha(opacity=90)}.carousel-control .icon-prev,.carousel-control .icon-next,.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right{position:absolute;top:50%;z-index:5;display:inline-block}.carousel-control .icon-prev,.carousel-control .glyphicon-chevron-left{left:50%;margin-left:-10px}.carousel-control .icon-next,.carousel-control .glyphicon-chevron-right{right:50%;margin-right:-10px}.carousel-control .icon-prev,.carousel-control .icon-next{width:20px;height:20px;margin-top:-10px;font-family:serif}.carousel-control .icon-prev:before{content:'\2039'}.carousel-control .icon-next:before{content:'\203a'}.carousel-indicators{position:absolute;bottom:10px;left:50%;z-index:15;width:60%;margin-left:-30%;padding-left:0;list-style:none;text-align:center}.carousel-indicators li{display:inline-block;width:10px;height:10px;margin:1px;text-indent:-999px;border:1px solid #fff;border-radius:10px;cursor:pointer;background-color:#000 \9;background-color:rgba(0,0,0,0)}.carousel-indicators .active{margin:0;width:12px;height:12px;background-color:#fff}.carousel-caption{position:absolute;left:15%;right:15%;bottom:20px;z-index:10;padding-top:20px;padding-bottom:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,0.6)}.carousel-caption .btn{text-shadow:none}@media screen and (min-width:768px){.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-prev,.carousel-control .icon-next{width:30px;height:30px;margin-top:-15px;font-size:30px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{margin-left:-15px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{margin-right:-15px}.carousel-caption{left:20%;right:20%;padding-bottom:30px}.carousel-indicators{bottom:20px}}.clearfix:before,.clearfix:after,.dl-horizontal dd:before,.dl-horizontal dd:after,.container:before,.container:after,.container-fluid:before,.container-fluid:after,.row:before,.row:after,.form-horizontal .form-group:before,.form-horizontal .form-group:after,.btn-toolbar:before,.btn-toolbar:after,.btn-group-vertical>.btn-group:before,.btn-group-vertical>.btn-group:after,.nav:before,.nav:after,.navbar:before,.navbar:after,.navbar-header:before,.navbar-header:after,.navbar-collapse:before,.navbar-collapse:after,.pager:before,.pager:after,.panel-body:before,.panel-body:after,.modal-footer:before,.modal-footer:after,.item_buttons:before,.item_buttons:after{content:" ";display:table}.clearfix:after,.dl-horizontal dd:after,.container:after,.container-fluid:after,.row:after,.form-horizontal .form-group:after,.btn-toolbar:after,.btn-group-vertical>.btn-group:after,.nav:after,.navbar:after,.navbar-header:after,.navbar-collapse:after,.pager:after,.panel-body:after,.modal-footer:after,.item_buttons:after{clear:both}.center-block{display:block;margin-left:auto;margin-right:auto}.pull-right{float:right !important}.pull-left{float:left !important}.hide{display:none !important}.show{display:block !important}.invisible{visibility:hidden}.text-hide{font:0/0 a;color:transparent;text-shadow:none;background-color:transparent;border:0}.hidden{display:none !important;visibility:hidden !important}.affix{position:fixed}@-ms-viewport{width:device-width}.visible-xs,.visible-sm,.visible-md,.visible-lg{display:none !important}.visible-xs-block,.visible-xs-inline,.visible-xs-inline-block,.visible-sm-block,.visible-sm-inline,.visible-sm-inline-block,.visible-md-block,.visible-md-inline,.visible-md-inline-block,.visible-lg-block,.visible-lg-inline,.visible-lg-inline-block{display:none !important}@media (max-width:767px){.visible-xs{display:block !important}table.visible-xs{display:table}tr.visible-xs{display:table-row !important}th.visible-xs,td.visible-xs{display:table-cell !important}}@media (max-width:767px){.visible-xs-block{display:block !important}}@media (max-width:767px){.visible-xs-inline{display:inline !important}}@media (max-width:767px){.visible-xs-inline-block{display:inline-block !important}}@media (min-width:768px) and (max-width:991px){.visible-sm{display:block !important}table.visible-sm{display:table}tr.visible-sm{display:table-row !important}th.visible-sm,td.visible-sm{display:table-cell !important}}@media (min-width:768px) and (max-width:991px){.visible-sm-block{display:block !important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline{display:inline !important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline-block{display:inline-block !important}}@media (min-width:992px) and (max-width:1199px){.visible-md{display:block !important}table.visible-md{display:table}tr.visible-md{display:table-row !important}th.visible-md,td.visible-md{display:table-cell !important}}@media (min-width:992px) and (max-width:1199px){.visible-md-block{display:block !important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline{display:inline !important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline-block{display:inline-block !important}}@media (min-width:1200px){.visible-lg{display:block !important}table.visible-lg{display:table}tr.visible-lg{display:table-row !important}th.visible-lg,td.visible-lg{display:table-cell !important}}@media (min-width:1200px){.visible-lg-block{display:block !important}}@media (min-width:1200px){.visible-lg-inline{display:inline !important}}@media (min-width:1200px){.visible-lg-inline-block{display:inline-block !important}}@media (max-width:767px){.hidden-xs{display:none !important}}@media (min-width:768px) and (max-width:991px){.hidden-sm{display:none !important}}@media (min-width:992px) and (max-width:1199px){.hidden-md{display:none !important}}@media (min-width:1200px){.hidden-lg{display:none !important}}.visible-print{display:none !important}@media print{.visible-print{display:block !important}table.visible-print{display:table}tr.visible-print{display:table-row !important}th.visible-print,td.visible-print{display:table-cell !important}}.visible-print-block{display:none !important}@media print{.visible-print-block{display:block !important}}.visible-print-inline{display:none !important}@media print{.visible-print-inline{display:inline !important}}.visible-print-inline-block{display:none !important}@media print{.visible-print-inline-block{display:inline-block !important}}@media print{.hidden-print{display:none !important}}/*!
*
* Font Awesome
*
*//*!
 *  Font Awesome 4.3.0 by @davegandy - http://fontawesome.io - @fontawesome
 *  License - http://fontawesome.io/license (Font: SIL OFL 1.1, CSS: MIT License)
 */@font-face{font-family:'FontAwesome';src:url('../components/font-awesome/fonts/fontawesome-webfont.eot?v=4.3.0');src:url('../components/font-awesome/fonts/fontawesome-webfont.eot?#iefix&v=4.3.0') format('embedded-opentype'),url('../components/font-awesome/fonts/fontawesome-webfont.woff2?v=4.3.0') format('woff2'),url('../components/font-awesome/fonts/fontawesome-webfont.woff?v=4.3.0') format('woff'),url('../components/font-awesome/fonts/fontawesome-webfont.ttf?v=4.3.0') format('truetype'),url('../components/font-awesome/fonts/fontawesome-webfont.svg?v=4.3.0#fontawesomeregular') format('svg');font-weight:normal;font-style:normal}.fa{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0)}.fa-lg{font-size:1.33333333em;line-height:.75em;vertical-align:-15%}.fa-2x{font-size:2em}.fa-3x{font-size:3em}.fa-4x{font-size:4em}.fa-5x{font-size:5em}.fa-fw{width:1.28571429em;text-align:center}.fa-ul{padding-left:0;margin-left:2.14285714em;list-style-type:none}.fa-ul>li{position:relative}.fa-li{position:absolute;left:-2.14285714em;width:2.14285714em;top:.14285714em;text-align:center}.fa-li.fa-lg{left:-1.85714286em}.fa-border{padding:.2em .25em .15em;border:solid .08em #eee;border-radius:.1em}.pull-right{float:right}.pull-left{float:left}.fa.pull-left{margin-right:.3em}.fa.pull-right{margin-left:.3em}.fa-spin{-webkit-animation:fa-spin 2s infinite linear;animation:fa-spin 2s infinite linear}.fa-pulse{-webkit-animation:fa-spin 1s infinite steps(8);animation:fa-spin 1s infinite steps(8)}@-webkit-keyframes fa-spin{0%{-webkit-transform:rotate(0deg);transform:rotate(0deg)}100%{-webkit-transform:rotate(359deg);transform:rotate(359deg)}}@keyframes fa-spin{0%{-webkit-transform:rotate(0deg);transform:rotate(0deg)}100%{-webkit-transform:rotate(359deg);transform:rotate(359deg)}}.fa-rotate-90{filter:progid:DXImageTransform.Microsoft.BasicImage(rotation=1);-webkit-transform:rotate(90deg);-ms-transform:rotate(90deg);transform:rotate(90deg)}.fa-rotate-180{filter:progid:DXImageTransform.Microsoft.BasicImage(rotation=2);-webkit-transform:rotate(180deg);-ms-transform:rotate(180deg);transform:rotate(180deg)}.fa-rotate-270{filter:progid:DXImageTransform.Microsoft.BasicImage(rotation=3);-webkit-transform:rotate(270deg);-ms-transform:rotate(270deg);transform:rotate(270deg)}.fa-flip-horizontal{filter:progid:DXImageTransform.Microsoft.BasicImage(rotation=0, mirror=1);-webkit-transform:scale(-1, 1);-ms-transform:scale(-1, 1);transform:scale(-1, 1)}.fa-flip-vertical{filter:progid:DXImageTransform.Microsoft.BasicImage(rotation=2, mirror=1);-webkit-transform:scale(1, -1);-ms-transform:scale(1, -1);transform:scale(1, -1)}:root .fa-rotate-90,:root .fa-rotate-180,:root .fa-rotate-270,:root .fa-flip-horizontal,:root .fa-flip-vertical{filter:none}.fa-stack{position:relative;display:inline-block;width:2em;height:2em;line-height:2em;vertical-align:middle}.fa-stack-1x,.fa-stack-2x{position:absolute;left:0;width:100%;text-align:center}.fa-stack-1x{line-height:inherit}.fa-stack-2x{font-size:2em}.fa-inverse{color:#fff}.fa-glass:before{content:"\f000"}.fa-music:before{content:"\f001"}.fa-search:before{content:"\f002"}.fa-envelope-o:before{content:"\f003"}.fa-heart:before{content:"\f004"}.fa-star:before{content:"\f005"}.fa-star-o:before{content:"\f006"}.fa-user:before{content:"\f007"}.fa-film:before{content:"\f008"}.fa-th-large:before{content:"\f009"}.fa-th:before{content:"\f00a"}.fa-th-list:before{content:"\f00b"}.fa-check:before{content:"\f00c"}.fa-remove:before,.fa-close:before,.fa-times:before{content:"\f00d"}.fa-search-plus:before{content:"\f00e"}.fa-search-minus:before{content:"\f010"}.fa-power-off:before{content:"\f011"}.fa-signal:before{content:"\f012"}.fa-gear:before,.fa-cog:before{content:"\f013"}.fa-trash-o:before{content:"\f014"}.fa-home:before{content:"\f015"}.fa-file-o:before{content:"\f016"}.fa-clock-o:before{content:"\f017"}.fa-road:before{content:"\f018"}.fa-download:before{content:"\f019"}.fa-arrow-circle-o-down:before{content:"\f01a"}.fa-arrow-circle-o-up:before{content:"\f01b"}.fa-inbox:before{content:"\f01c"}.fa-play-circle-o:before{content:"\f01d"}.fa-rotate-right:before,.fa-repeat:before{content:"\f01e"}.fa-refresh:before{content:"\f021"}.fa-list-alt:before{content:"\f022"}.fa-lock:before{content:"\f023"}.fa-flag:before{content:"\f024"}.fa-headphones:before{content:"\f025"}.fa-volume-off:before{content:"\f026"}.fa-volume-down:before{content:"\f027"}.fa-volume-up:before{content:"\f028"}.fa-qrcode:before{content:"\f029"}.fa-barcode:before{content:"\f02a"}.fa-tag:before{content:"\f02b"}.fa-tags:before{content:"\f02c"}.fa-book:before{content:"\f02d"}.fa-bookmark:before{content:"\f02e"}.fa-print:before{content:"\f02f"}.fa-camera:before{content:"\f030"}.fa-font:before{content:"\f031"}.fa-bold:before{content:"\f032"}.fa-italic:before{content:"\f033"}.fa-text-height:before{content:"\f034"}.fa-text-width:before{content:"\f035"}.fa-align-left:before{content:"\f036"}.fa-align-center:before{content:"\f037"}.fa-align-right:before{content:"\f038"}.fa-align-justify:before{content:"\f039"}.fa-list:before{content:"\f03a"}.fa-dedent:before,.fa-outdent:before{content:"\f03b"}.fa-indent:before{content:"\f03c"}.fa-video-camera:before{content:"\f03d"}.fa-photo:before,.fa-image:before,.fa-picture-o:before{content:"\f03e"}.fa-pencil:before{content:"\f040"}.fa-map-marker:before{content:"\f041"}.fa-adjust:before{content:"\f042"}.fa-tint:before{content:"\f043"}.fa-edit:before,.fa-pencil-square-o:before{content:"\f044"}.fa-share-square-o:before{content:"\f045"}.fa-check-square-o:before{content:"\f046"}.fa-arrows:before{content:"\f047"}.fa-step-backward:before{content:"\f048"}.fa-fast-backward:before{content:"\f049"}.fa-backward:before{content:"\f04a"}.fa-play:before{content:"\f04b"}.fa-pause:before{content:"\f04c"}.fa-stop:before{content:"\f04d"}.fa-forward:before{content:"\f04e"}.fa-fast-forward:before{content:"\f050"}.fa-step-forward:before{content:"\f051"}.fa-eject:before{content:"\f052"}.fa-chevron-left:before{content:"\f053"}.fa-chevron-right:before{content:"\f054"}.fa-plus-circle:before{content:"\f055"}.fa-minus-circle:before{content:"\f056"}.fa-times-circle:before{content:"\f057"}.fa-check-circle:before{content:"\f058"}.fa-question-circle:before{content:"\f059"}.fa-info-circle:before{content:"\f05a"}.fa-crosshairs:before{content:"\f05b"}.fa-times-circle-o:before{content:"\f05c"}.fa-check-circle-o:before{content:"\f05d"}.fa-ban:before{content:"\f05e"}.fa-arrow-left:before{content:"\f060"}.fa-arrow-right:before{content:"\f061"}.fa-arrow-up:before{content:"\f062"}.fa-arrow-down:before{content:"\f063"}.fa-mail-forward:before,.fa-share:before{content:"\f064"}.fa-expand:before{content:"\f065"}.fa-compress:before{content:"\f066"}.fa-plus:before{content:"\f067"}.fa-minus:before{content:"\f068"}.fa-asterisk:before{content:"\f069"}.fa-exclamation-circle:before{content:"\f06a"}.fa-gift:before{content:"\f06b"}.fa-leaf:before{content:"\f06c"}.fa-fire:before{content:"\f06d"}.fa-eye:before{content:"\f06e"}.fa-eye-slash:before{content:"\f070"}.fa-warning:before,.fa-exclamation-triangle:before{content:"\f071"}.fa-plane:before{content:"\f072"}.fa-calendar:before{content:"\f073"}.fa-random:before{content:"\f074"}.fa-comment:before{content:"\f075"}.fa-magnet:before{content:"\f076"}.fa-chevron-up:before{content:"\f077"}.fa-chevron-down:before{content:"\f078"}.fa-retweet:before{content:"\f079"}.fa-shopping-cart:before{content:"\f07a"}.fa-folder:before{content:"\f07b"}.fa-folder-open:before{content:"\f07c"}.fa-arrows-v:before{content:"\f07d"}.fa-arrows-h:before{content:"\f07e"}.fa-bar-chart-o:before,.fa-bar-chart:before{content:"\f080"}.fa-twitter-square:before{content:"\f081"}.fa-facebook-square:before{content:"\f082"}.fa-camera-retro:before{content:"\f083"}.fa-key:before{content:"\f084"}.fa-gears:before,.fa-cogs:before{content:"\f085"}.fa-comments:before{content:"\f086"}.fa-thumbs-o-up:before{content:"\f087"}.fa-thumbs-o-down:before{content:"\f088"}.fa-star-half:before{content:"\f089"}.fa-heart-o:before{content:"\f08a"}.fa-sign-out:before{content:"\f08b"}.fa-linkedin-square:before{content:"\f08c"}.fa-thumb-tack:before{content:"\f08d"}.fa-external-link:before{content:"\f08e"}.fa-sign-in:before{content:"\f090"}.fa-trophy:before{content:"\f091"}.fa-github-square:before{content:"\f092"}.fa-upload:before{content:"\f093"}.fa-lemon-o:before{content:"\f094"}.fa-phone:before{content:"\f095"}.fa-square-o:before{content:"\f096"}.fa-bookmark-o:before{content:"\f097"}.fa-phone-square:before{content:"\f098"}.fa-twitter:before{content:"\f099"}.fa-facebook-f:before,.fa-facebook:before{content:"\f09a"}.fa-github:before{content:"\f09b"}.fa-unlock:before{content:"\f09c"}.fa-credit-card:before{content:"\f09d"}.fa-rss:before{content:"\f09e"}.fa-hdd-o:before{content:"\f0a0"}.fa-bullhorn:before{content:"\f0a1"}.fa-bell:before{content:"\f0f3"}.fa-certificate:before{content:"\f0a3"}.fa-hand-o-right:before{content:"\f0a4"}.fa-hand-o-left:before{content:"\f0a5"}.fa-hand-o-up:before{content:"\f0a6"}.fa-hand-o-down:before{content:"\f0a7"}.fa-arrow-circle-left:before{content:"\f0a8"}.fa-arrow-circle-right:before{content:"\f0a9"}.fa-arrow-circle-up:before{content:"\f0aa"}.fa-arrow-circle-down:before{content:"\f0ab"}.fa-globe:before{content:"\f0ac"}.fa-wrench:before{content:"\f0ad"}.fa-tasks:before{content:"\f0ae"}.fa-filter:before{content:"\f0b0"}.fa-briefcase:before{content:"\f0b1"}.fa-arrows-alt:before{content:"\f0b2"}.fa-group:before,.fa-users:before{content:"\f0c0"}.fa-chain:before,.fa-link:before{content:"\f0c1"}.fa-cloud:before{content:"\f0c2"}.fa-flask:before{content:"\f0c3"}.fa-cut:before,.fa-scissors:before{content:"\f0c4"}.fa-copy:before,.fa-files-o:before{content:"\f0c5"}.fa-paperclip:before{content:"\f0c6"}.fa-save:before,.fa-floppy-o:before{content:"\f0c7"}.fa-square:before{content:"\f0c8"}.fa-navicon:before,.fa-reorder:before,.fa-bars:before{content:"\f0c9"}.fa-list-ul:before{content:"\f0ca"}.fa-list-ol:before{content:"\f0cb"}.fa-strikethrough:before{content:"\f0cc"}.fa-underline:before{content:"\f0cd"}.fa-table:before{content:"\f0ce"}.fa-magic:before{content:"\f0d0"}.fa-truck:before{content:"\f0d1"}.fa-pinterest:before{content:"\f0d2"}.fa-pinterest-square:before{content:"\f0d3"}.fa-google-plus-square:before{content:"\f0d4"}.fa-google-plus:before{content:"\f0d5"}.fa-money:before{content:"\f0d6"}.fa-caret-down:before{content:"\f0d7"}.fa-caret-up:before{content:"\f0d8"}.fa-caret-left:before{content:"\f0d9"}.fa-caret-right:before{content:"\f0da"}.fa-columns:before{content:"\f0db"}.fa-unsorted:before,.fa-sort:before{content:"\f0dc"}.fa-sort-down:before,.fa-sort-desc:before{content:"\f0dd"}.fa-sort-up:before,.fa-sort-asc:before{content:"\f0de"}.fa-envelope:before{content:"\f0e0"}.fa-linkedin:before{content:"\f0e1"}.fa-rotate-left:before,.fa-undo:before{content:"\f0e2"}.fa-legal:before,.fa-gavel:before{content:"\f0e3"}.fa-dashboard:before,.fa-tachometer:before{content:"\f0e4"}.fa-comment-o:before{content:"\f0e5"}.fa-comments-o:before{content:"\f0e6"}.fa-flash:before,.fa-bolt:before{content:"\f0e7"}.fa-sitemap:before{content:"\f0e8"}.fa-umbrella:before{content:"\f0e9"}.fa-paste:before,.fa-clipboard:before{content:"\f0ea"}.fa-lightbulb-o:before{content:"\f0eb"}.fa-exchange:before{content:"\f0ec"}.fa-cloud-download:before{content:"\f0ed"}.fa-cloud-upload:before{content:"\f0ee"}.fa-user-md:before{content:"\f0f0"}.fa-stethoscope:before{content:"\f0f1"}.fa-suitcase:before{content:"\f0f2"}.fa-bell-o:before{content:"\f0a2"}.fa-coffee:before{content:"\f0f4"}.fa-cutlery:before{content:"\f0f5"}.fa-file-text-o:before{content:"\f0f6"}.fa-building-o:before{content:"\f0f7"}.fa-hospital-o:before{content:"\f0f8"}.fa-ambulance:before{content:"\f0f9"}.fa-medkit:before{content:"\f0fa"}.fa-fighter-jet:before{content:"\f0fb"}.fa-beer:before{content:"\f0fc"}.fa-h-square:before{content:"\f0fd"}.fa-plus-square:before{content:"\f0fe"}.fa-angle-double-left:before{content:"\f100"}.fa-angle-double-right:before{content:"\f101"}.fa-angle-double-up:before{content:"\f102"}.fa-angle-double-down:before{content:"\f103"}.fa-angle-left:before{content:"\f104"}.fa-angle-right:before{content:"\f105"}.fa-angle-up:before{content:"\f106"}.fa-angle-down:before{content:"\f107"}.fa-desktop:before{content:"\f108"}.fa-laptop:before{content:"\f109"}.fa-tablet:before{content:"\f10a"}.fa-mobile-phone:before,.fa-mobile:before{content:"\f10b"}.fa-circle-o:before{content:"\f10c"}.fa-quote-left:before{content:"\f10d"}.fa-quote-right:before{content:"\f10e"}.fa-spinner:before{content:"\f110"}.fa-circle:before{content:"\f111"}.fa-mail-reply:before,.fa-reply:before{content:"\f112"}.fa-github-alt:before{content:"\f113"}.fa-folder-o:before{content:"\f114"}.fa-folder-open-o:before{content:"\f115"}.fa-smile-o:before{content:"\f118"}.fa-frown-o:before{content:"\f119"}.fa-meh-o:before{content:"\f11a"}.fa-gamepad:before{content:"\f11b"}.fa-keyboard-o:before{content:"\f11c"}.fa-flag-o:before{content:"\f11d"}.fa-flag-checkered:before{content:"\f11e"}.fa-terminal:before{content:"\f120"}.fa-code:before{content:"\f121"}.fa-mail-reply-all:before,.fa-reply-all:before{content:"\f122"}.fa-star-half-empty:before,.fa-star-half-full:before,.fa-star-half-o:before{content:"\f123"}.fa-location-arrow:before{content:"\f124"}.fa-crop:before{content:"\f125"}.fa-code-fork:before{content:"\f126"}.fa-unlink:before,.fa-chain-broken:before{content:"\f127"}.fa-question:before{content:"\f128"}.fa-info:before{content:"\f129"}.fa-exclamation:before{content:"\f12a"}.fa-superscript:before{content:"\f12b"}.fa-subscript:before{content:"\f12c"}.fa-eraser:before{content:"\f12d"}.fa-puzzle-piece:before{content:"\f12e"}.fa-microphone:before{content:"\f130"}.fa-microphone-slash:before{content:"\f131"}.fa-shield:before{content:"\f132"}.fa-calendar-o:before{content:"\f133"}.fa-fire-extinguisher:before{content:"\f134"}.fa-rocket:before{content:"\f135"}.fa-maxcdn:before{content:"\f136"}.fa-chevron-circle-left:before{content:"\f137"}.fa-chevron-circle-right:before{content:"\f138"}.fa-chevron-circle-up:before{content:"\f139"}.fa-chevron-circle-down:before{content:"\f13a"}.fa-html5:before{content:"\f13b"}.fa-css3:before{content:"\f13c"}.fa-anchor:before{content:"\f13d"}.fa-unlock-alt:before{content:"\f13e"}.fa-bullseye:before{content:"\f140"}.fa-ellipsis-h:before{content:"\f141"}.fa-ellipsis-v:before{content:"\f142"}.fa-rss-square:before{content:"\f143"}.fa-play-circle:before{content:"\f144"}.fa-ticket:before{content:"\f145"}.fa-minus-square:before{content:"\f146"}.fa-minus-square-o:before{content:"\f147"}.fa-level-up:before{content:"\f148"}.fa-level-down:before{content:"\f149"}.fa-check-square:before{content:"\f14a"}.fa-pencil-square:before{content:"\f14b"}.fa-external-link-square:before{content:"\f14c"}.fa-share-square:before{content:"\f14d"}.fa-compass:before{content:"\f14e"}.fa-toggle-down:before,.fa-caret-square-o-down:before{content:"\f150"}.fa-toggle-up:before,.fa-caret-square-o-up:before{content:"\f151"}.fa-toggle-right:before,.fa-caret-square-o-right:before{content:"\f152"}.fa-euro:before,.fa-eur:before{content:"\f153"}.fa-gbp:before{content:"\f154"}.fa-dollar:before,.fa-usd:before{content:"\f155"}.fa-rupee:before,.fa-inr:before{content:"\f156"}.fa-cny:before,.fa-rmb:before,.fa-yen:before,.fa-jpy:before{content:"\f157"}.fa-ruble:before,.fa-rouble:before,.fa-rub:before{content:"\f158"}.fa-won:before,.fa-krw:before{content:"\f159"}.fa-bitcoin:before,.fa-btc:before{content:"\f15a"}.fa-file:before{content:"\f15b"}.fa-file-text:before{content:"\f15c"}.fa-sort-alpha-asc:before{content:"\f15d"}.fa-sort-alpha-desc:before{content:"\f15e"}.fa-sort-amount-asc:before{content:"\f160"}.fa-sort-amount-desc:before{content:"\f161"}.fa-sort-numeric-asc:before{content:"\f162"}.fa-sort-numeric-desc:before{content:"\f163"}.fa-thumbs-up:before{content:"\f164"}.fa-thumbs-down:before{content:"\f165"}.fa-youtube-square:before{content:"\f166"}.fa-youtube:before{content:"\f167"}.fa-xing:before{content:"\f168"}.fa-xing-square:before{content:"\f169"}.fa-youtube-play:before{content:"\f16a"}.fa-dropbox:before{content:"\f16b"}.fa-stack-overflow:before{content:"\f16c"}.fa-instagram:before{content:"\f16d"}.fa-flickr:before{content:"\f16e"}.fa-adn:before{content:"\f170"}.fa-bitbucket:before{content:"\f171"}.fa-bitbucket-square:before{content:"\f172"}.fa-tumblr:before{content:"\f173"}.fa-tumblr-square:before{content:"\f174"}.fa-long-arrow-down:before{content:"\f175"}.fa-long-arrow-up:before{content:"\f176"}.fa-long-arrow-left:before{content:"\f177"}.fa-long-arrow-right:before{content:"\f178"}.fa-apple:before{content:"\f179"}.fa-windows:before{content:"\f17a"}.fa-android:before{content:"\f17b"}.fa-linux:before{content:"\f17c"}.fa-dribbble:before{content:"\f17d"}.fa-skype:before{content:"\f17e"}.fa-foursquare:before{content:"\f180"}.fa-trello:before{content:"\f181"}.fa-female:before{content:"\f182"}.fa-male:before{content:"\f183"}.fa-gittip:before,.fa-gratipay:before{content:"\f184"}.fa-sun-o:before{content:"\f185"}.fa-moon-o:before{content:"\f186"}.fa-archive:before{content:"\f187"}.fa-bug:before{content:"\f188"}.fa-vk:before{content:"\f189"}.fa-weibo:before{content:"\f18a"}.fa-renren:before{content:"\f18b"}.fa-pagelines:before{content:"\f18c"}.fa-stack-exchange:before{content:"\f18d"}.fa-arrow-circle-o-right:before{content:"\f18e"}.fa-arrow-circle-o-left:before{content:"\f190"}.fa-toggle-left:before,.fa-caret-square-o-left:before{content:"\f191"}.fa-dot-circle-o:before{content:"\f192"}.fa-wheelchair:before{content:"\f193"}.fa-vimeo-square:before{content:"\f194"}.fa-turkish-lira:before,.fa-try:before{content:"\f195"}.fa-plus-square-o:before{content:"\f196"}.fa-space-shuttle:before{content:"\f197"}.fa-slack:before{content:"\f198"}.fa-envelope-square:before{content:"\f199"}.fa-wordpress:before{content:"\f19a"}.fa-openid:before{content:"\f19b"}.fa-institution:before,.fa-bank:before,.fa-university:before{content:"\f19c"}.fa-mortar-board:before,.fa-graduation-cap:before{content:"\f19d"}.fa-yahoo:before{content:"\f19e"}.fa-google:before{content:"\f1a0"}.fa-reddit:before{content:"\f1a1"}.fa-reddit-square:before{content:"\f1a2"}.fa-stumbleupon-circle:before{content:"\f1a3"}.fa-stumbleupon:before{content:"\f1a4"}.fa-delicious:before{content:"\f1a5"}.fa-digg:before{content:"\f1a6"}.fa-pied-piper:before{content:"\f1a7"}.fa-pied-piper-alt:before{content:"\f1a8"}.fa-drupal:before{content:"\f1a9"}.fa-joomla:before{content:"\f1aa"}.fa-language:before{content:"\f1ab"}.fa-fax:before{content:"\f1ac"}.fa-building:before{content:"\f1ad"}.fa-child:before{content:"\f1ae"}.fa-paw:before{content:"\f1b0"}.fa-spoon:before{content:"\f1b1"}.fa-cube:before{content:"\f1b2"}.fa-cubes:before{content:"\f1b3"}.fa-behance:before{content:"\f1b4"}.fa-behance-square:before{content:"\f1b5"}.fa-steam:before{content:"\f1b6"}.fa-steam-square:before{content:"\f1b7"}.fa-recycle:before{content:"\f1b8"}.fa-automobile:before,.fa-car:before{content:"\f1b9"}.fa-cab:before,.fa-taxi:before{content:"\f1ba"}.fa-tree:before{content:"\f1bb"}.fa-spotify:before{content:"\f1bc"}.fa-deviantart:before{content:"\f1bd"}.fa-soundcloud:before{content:"\f1be"}.fa-database:before{content:"\f1c0"}.fa-file-pdf-o:before{content:"\f1c1"}.fa-file-word-o:before{content:"\f1c2"}.fa-file-excel-o:before{content:"\f1c3"}.fa-file-powerpoint-o:before{content:"\f1c4"}.fa-file-photo-o:before,.fa-file-picture-o:before,.fa-file-image-o:before{content:"\f1c5"}.fa-file-zip-o:before,.fa-file-archive-o:before{content:"\f1c6"}.fa-file-sound-o:before,.fa-file-audio-o:before{content:"\f1c7"}.fa-file-movie-o:before,.fa-file-video-o:before{content:"\f1c8"}.fa-file-code-o:before{content:"\f1c9"}.fa-vine:before{content:"\f1ca"}.fa-codepen:before{content:"\f1cb"}.fa-jsfiddle:before{content:"\f1cc"}.fa-life-bouy:before,.fa-life-buoy:before,.fa-life-saver:before,.fa-support:before,.fa-life-ring:before{content:"\f1cd"}.fa-circle-o-notch:before{content:"\f1ce"}.fa-ra:before,.fa-rebel:before{content:"\f1d0"}.fa-ge:before,.fa-empire:before{content:"\f1d1"}.fa-git-square:before{content:"\f1d2"}.fa-git:before{content:"\f1d3"}.fa-hacker-news:before{content:"\f1d4"}.fa-tencent-weibo:before{content:"\f1d5"}.fa-qq:before{content:"\f1d6"}.fa-wechat:before,.fa-weixin:before{content:"\f1d7"}.fa-send:before,.fa-paper-plane:before{content:"\f1d8"}.fa-send-o:before,.fa-paper-plane-o:before{content:"\f1d9"}.fa-history:before{content:"\f1da"}.fa-genderless:before,.fa-circle-thin:before{content:"\f1db"}.fa-header:before{content:"\f1dc"}.fa-paragraph:before{content:"\f1dd"}.fa-sliders:before{content:"\f1de"}.fa-share-alt:before{content:"\f1e0"}.fa-share-alt-square:before{content:"\f1e1"}.fa-bomb:before{content:"\f1e2"}.fa-soccer-ball-o:before,.fa-futbol-o:before{content:"\f1e3"}.fa-tty:before{content:"\f1e4"}.fa-binoculars:before{content:"\f1e5"}.fa-plug:before{content:"\f1e6"}.fa-slideshare:before{content:"\f1e7"}.fa-twitch:before{content:"\f1e8"}.fa-yelp:before{content:"\f1e9"}.fa-newspaper-o:before{content:"\f1ea"}.fa-wifi:before{content:"\f1eb"}.fa-calculator:before{content:"\f1ec"}.fa-paypal:before{content:"\f1ed"}.fa-google-wallet:before{content:"\f1ee"}.fa-cc-visa:before{content:"\f1f0"}.fa-cc-mastercard:before{content:"\f1f1"}.fa-cc-discover:before{content:"\f1f2"}.fa-cc-amex:before{content:"\f1f3"}.fa-cc-paypal:before{content:"\f1f4"}.fa-cc-stripe:before{content:"\f1f5"}.fa-bell-slash:before{content:"\f1f6"}.fa-bell-slash-o:before{content:"\f1f7"}.fa-trash:before{content:"\f1f8"}.fa-copyright:before{content:"\f1f9"}.fa-at:before{content:"\f1fa"}.fa-eyedropper:before{content:"\f1fb"}.fa-paint-brush:before{content:"\f1fc"}.fa-birthday-cake:before{content:"\f1fd"}.fa-area-chart:before{content:"\f1fe"}.fa-pie-chart:before{content:"\f200"}.fa-line-chart:before{content:"\f201"}.fa-lastfm:before{content:"\f202"}.fa-lastfm-square:before{content:"\f203"}.fa-toggle-off:before{content:"\f204"}.fa-toggle-on:before{content:"\f205"}.fa-bicycle:before{content:"\f206"}.fa-bus:before{content:"\f207"}.fa-ioxhost:before{content:"\f208"}.fa-angellist:before{content:"\f209"}.fa-cc:before{content:"\f20a"}.fa-shekel:before,.fa-sheqel:before,.fa-ils:before{content:"\f20b"}.fa-meanpath:before{content:"\f20c"}.fa-buysellads:before{content:"\f20d"}.fa-connectdevelop:before{content:"\f20e"}.fa-dashcube:before{content:"\f210"}.fa-forumbee:before{content:"\f211"}.fa-leanpub:before{content:"\f212"}.fa-sellsy:before{content:"\f213"}.fa-shirtsinbulk:before{content:"\f214"}.fa-simplybuilt:before{content:"\f215"}.fa-skyatlas:before{content:"\f216"}.fa-cart-plus:before{content:"\f217"}.fa-cart-arrow-down:before{content:"\f218"}.fa-diamond:before{content:"\f219"}.fa-ship:before{content:"\f21a"}.fa-user-secret:before{content:"\f21b"}.fa-motorcycle:before{content:"\f21c"}.fa-street-view:before{content:"\f21d"}.fa-heartbeat:before{content:"\f21e"}.fa-venus:before{content:"\f221"}.fa-mars:before{content:"\f222"}.fa-mercury:before{content:"\f223"}.fa-transgender:before{content:"\f224"}.fa-transgender-alt:before{content:"\f225"}.fa-venus-double:before{content:"\f226"}.fa-mars-double:before{content:"\f227"}.fa-venus-mars:before{content:"\f228"}.fa-mars-stroke:before{content:"\f229"}.fa-mars-stroke-v:before{content:"\f22a"}.fa-mars-stroke-h:before{content:"\f22b"}.fa-neuter:before{content:"\f22c"}.fa-facebook-official:before{content:"\f230"}.fa-pinterest-p:before{content:"\f231"}.fa-whatsapp:before{content:"\f232"}.fa-server:before{content:"\f233"}.fa-user-plus:before{content:"\f234"}.fa-user-times:before{content:"\f235"}.fa-hotel:before,.fa-bed:before{content:"\f236"}.fa-viacoin:before{content:"\f237"}.fa-train:before{content:"\f238"}.fa-subway:before{content:"\f239"}.fa-medium:before{content:"\f23a"}/*!
*
* IPython base
*
*/.modal.fade .modal-dialog{-webkit-transform:translate(0, 0);-ms-transform:translate(0, 0);-o-transform:translate(0, 0);transform:translate(0, 0)}code{color:#000}pre{font-size:inherit;line-height:inherit}label{font-weight:normal}.border-box-sizing{box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box}.corner-all{border-radius:2px}.no-padding{padding:0}.hbox{display:-webkit-box;-webkit-box-orient:horizontal;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:horizontal;-moz-box-align:stretch;display:box;box-orient:horizontal;box-align:stretch;display:flex;flex-direction:row;align-items:stretch}.hbox>*{-webkit-box-flex:0;-moz-box-flex:0;box-flex:0;flex:none}.vbox{display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch}.vbox>*{-webkit-box-flex:0;-moz-box-flex:0;box-flex:0;flex:none}.hbox.reverse,.vbox.reverse,.reverse{-webkit-box-direction:reverse;-moz-box-direction:reverse;box-direction:reverse;flex-direction:row-reverse}.hbox.box-flex0,.vbox.box-flex0,.box-flex0{-webkit-box-flex:0;-moz-box-flex:0;box-flex:0;flex:none;width:auto}.hbox.box-flex1,.vbox.box-flex1,.box-flex1{-webkit-box-flex:1;-moz-box-flex:1;box-flex:1;flex:1}.hbox.box-flex,.vbox.box-flex,.box-flex{-webkit-box-flex:1;-moz-box-flex:1;box-flex:1;flex:1}.hbox.box-flex2,.vbox.box-flex2,.box-flex2{-webkit-box-flex:2;-moz-box-flex:2;box-flex:2;flex:2}.box-group1{-webkit-box-flex-group:1;-moz-box-flex-group:1;box-flex-group:1}.box-group2{-webkit-box-flex-group:2;-moz-box-flex-group:2;box-flex-group:2}.hbox.start,.vbox.start,.start{-webkit-box-pack:start;-moz-box-pack:start;box-pack:start;justify-content:flex-start}.hbox.end,.vbox.end,.end{-webkit-box-pack:end;-moz-box-pack:end;box-pack:end;justify-content:flex-end}.hbox.center,.vbox.center,.center{-webkit-box-pack:center;-moz-box-pack:center;box-pack:center;justify-content:center}.hbox.baseline,.vbox.baseline,.baseline{-webkit-box-pack:baseline;-moz-box-pack:baseline;box-pack:baseline;justify-content:baseline}.hbox.stretch,.vbox.stretch,.stretch{-webkit-box-pack:stretch;-moz-box-pack:stretch;box-pack:stretch;justify-content:stretch}.hbox.align-start,.vbox.align-start,.align-start{-webkit-box-align:start;-moz-box-align:start;box-align:start;align-items:flex-start}.hbox.align-end,.vbox.align-end,.align-end{-webkit-box-align:end;-moz-box-align:end;box-align:end;align-items:flex-end}.hbox.align-center,.vbox.align-center,.align-center{-webkit-box-align:center;-moz-box-align:center;box-align:center;align-items:center}.hbox.align-baseline,.vbox.align-baseline,.align-baseline{-webkit-box-align:baseline;-moz-box-align:baseline;box-align:baseline;align-items:baseline}.hbox.align-stretch,.vbox.align-stretch,.align-stretch{-webkit-box-align:stretch;-moz-box-align:stretch;box-align:stretch;align-items:stretch}div.error{margin:2em;text-align:center}div.error>h1{font-size:500%;line-height:normal}div.error>p{font-size:200%;line-height:normal}div.traceback-wrapper{text-align:left;max-width:800px;margin:auto}body{background-color:#fff;position:absolute;left:0;right:0;top:0;bottom:0;overflow:visible}#header{display:none;background-color:#fff;position:relative;z-index:100}#header #header-container{padding-bottom:5px;padding-top:5px;box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box}#header .header-bar{width:100%;height:1px;background:#e7e7e7;margin-bottom:-1px}@media print{#header{display:none !important}}#header-spacer{width:100%;visibility:hidden}@media print{#header-spacer{display:none}}#ipython_notebook{padding-left:0;padding-top:1px;padding-bottom:1px}@media (max-width:991px){#ipython_notebook{margin-left:10px}}#noscript{width:auto;padding-top:16px;padding-bottom:16px;text-align:center;font-size:22px;color:red;font-weight:bold}#ipython_notebook img{height:28px}#site{width:100%;display:none;box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box;overflow:auto}@media print{#site{height:auto !important}}.ui-button .ui-button-text{padding:.2em .8em;font-size:77%}input.ui-button{padding:.3em .9em}span#login_widget{float:right}span#login_widget>.button,#logout{color:#333;background-color:#fff;border-color:#ccc}span#login_widget>.button:hover,#logout:hover,span#login_widget>.button:focus,#logout:focus,span#login_widget>.button.focus,#logout.focus,span#login_widget>.button:active,#logout:active,span#login_widget>.button.active,#logout.active,.open>.dropdown-togglespan#login_widget>.button,.open>.dropdown-toggle#logout{color:#333;background-color:#e6e6e6;border-color:#adadad}span#login_widget>.button:active,#logout:active,span#login_widget>.button.active,#logout.active,.open>.dropdown-togglespan#login_widget>.button,.open>.dropdown-toggle#logout{background-image:none}span#login_widget>.button.disabled,#logout.disabled,span#login_widget>.button[disabled],#logout[disabled],fieldset[disabled] span#login_widget>.button,fieldset[disabled] #logout,span#login_widget>.button.disabled:hover,#logout.disabled:hover,span#login_widget>.button[disabled]:hover,#logout[disabled]:hover,fieldset[disabled] span#login_widget>.button:hover,fieldset[disabled] #logout:hover,span#login_widget>.button.disabled:focus,#logout.disabled:focus,span#login_widget>.button[disabled]:focus,#logout[disabled]:focus,fieldset[disabled] span#login_widget>.button:focus,fieldset[disabled] #logout:focus,span#login_widget>.button.disabled.focus,#logout.disabled.focus,span#login_widget>.button[disabled].focus,#logout[disabled].focus,fieldset[disabled] span#login_widget>.button.focus,fieldset[disabled] #logout.focus,span#login_widget>.button.disabled:active,#logout.disabled:active,span#login_widget>.button[disabled]:active,#logout[disabled]:active,fieldset[disabled] span#login_widget>.button:active,fieldset[disabled] #logout:active,span#login_widget>.button.disabled.active,#logout.disabled.active,span#login_widget>.button[disabled].active,#logout[disabled].active,fieldset[disabled] span#login_widget>.button.active,fieldset[disabled] #logout.active{background-color:#fff;border-color:#ccc}span#login_widget>.button .badge,#logout .badge{color:#fff;background-color:#333}.nav-header{text-transform:none}#header>span{margin-top:10px}.modal_stretch .modal-dialog{display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch;min-height:80vh}.modal_stretch .modal-dialog .modal-body{max-height:calc(100vh - 200px);overflow:auto;flex:1}@media (min-width:768px){.modal .modal-dialog{width:700px}}@media (min-width:768px){select.form-control{margin-left:12px;margin-right:12px}}/*!
*
* IPython auth
*
*/.center-nav{display:inline-block;margin-bottom:-4px}/*!
*
* IPython tree view
*
*/.alternate_upload{background-color:none;display:inline}.alternate_upload.form{padding:0;margin:0}.alternate_upload input.fileinput{display:inline;opacity:0;z-index:2;width:12ex;margin-right:-12ex}.alternate_upload .btn-upload{height:22px}ul#tabs{margin-bottom:4px}ul#tabs a{padding-top:6px;padding-bottom:4px}ul.breadcrumb a:focus,ul.breadcrumb a:hover{text-decoration:none}ul.breadcrumb i.icon-home{font-size:16px;margin-right:4px}ul.breadcrumb span{color:#5e5e5e}.list_toolbar{padding:4px 0 4px 0;vertical-align:middle}.list_toolbar .tree-buttons{padding-top:1px}.dynamic-buttons{padding-top:3px;display:inline-block}.list_toolbar [class*="span"]{min-height:24px}.list_header{font-weight:bold;background-color:#eee}.list_placeholder{font-weight:bold;padding-top:4px;padding-bottom:4px;padding-left:7px;padding-right:7px}.list_container{margin-top:4px;margin-bottom:20px;border:1px solid #ddd;border-radius:2px}.list_container>div{border-bottom:1px solid #ddd}.list_container>div:hover .list-item{background-color:red}.list_container>div:last-child{border:none}.list_item:hover .list_item{background-color:#ddd}.list_item a{text-decoration:none}.list_item:hover{background-color:#fafafa}.action_col{text-align:right}.list_header>div,.list_item>div{padding-top:4px;padding-bottom:4px;padding-left:7px;padding-right:7px;line-height:22px}.list_header>div input,.list_item>div input{margin-right:7px;margin-left:14px;vertical-align:baseline;line-height:22px;position:relative;top:-1px}.list_header>div .item_link,.list_item>div .item_link{margin-left:-1px;vertical-align:baseline;line-height:22px}.new-file input[type=checkbox]{visibility:hidden}.item_name{line-height:22px;height:24px}.item_icon{font-size:14px;color:#5e5e5e;margin-right:7px;margin-left:7px;line-height:22px;vertical-align:baseline}.item_buttons{line-height:1em;margin-left:-5px}.item_buttons .btn-group,.item_buttons .input-group{float:left}.item_buttons>.btn,.item_buttons>.btn-group,.item_buttons>.input-group{margin-left:5px}.item_buttons .btn{min-width:13ex}.item_buttons .running-indicator{padding-top:4px;color:#5cb85c}.toolbar_info{height:24px;line-height:24px}input.nbname_input,input.engine_num_input{padding-top:3px;padding-bottom:3px;height:22px;line-height:14px;margin:0}input.engine_num_input{width:60px}.highlight_text{color:blue}#project_name{display:inline-block;padding-left:7px;margin-left:-2px}#project_name>.breadcrumb{padding:0;margin-bottom:0;background-color:transparent;font-weight:bold}#tree-selector{padding-right:0}#button-select-all{min-width:50px}#select-all{margin-left:7px;margin-right:2px}.menu_icon{margin-right:2px}.tab-content .row{margin-left:0;margin-right:0}.folder_icon:before{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);content:"\f114"}.folder_icon:before.pull-left{margin-right:.3em}.folder_icon:before.pull-right{margin-left:.3em}.notebook_icon:before{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);content:"\f02d";position:relative;top:-1px}.notebook_icon:before.pull-left{margin-right:.3em}.notebook_icon:before.pull-right{margin-left:.3em}.running_notebook_icon:before{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);content:"\f02d";position:relative;top:-1px;color:#5cb85c}.running_notebook_icon:before.pull-left{margin-right:.3em}.running_notebook_icon:before.pull-right{margin-left:.3em}.file_icon:before{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);content:"\f016";position:relative;top:-2px}.file_icon:before.pull-left{margin-right:.3em}.file_icon:before.pull-right{margin-left:.3em}#notebook_toolbar .pull-right{padding-top:0;margin-right:-1px}ul#new-menu{left:auto;right:0}.kernel-menu-icon{padding-right:12px;width:24px;content:"\f096"}.kernel-menu-icon:before{content:"\f096"}.kernel-menu-icon-current:before{content:"\f00c"}#tab_content{padding-top:20px}#running .panel-group .panel{margin-top:3px;margin-bottom:1em}#running .panel-group .panel .panel-heading{background-color:#eee;padding-top:4px;padding-bottom:4px;padding-left:7px;padding-right:7px;line-height:22px}#running .panel-group .panel .panel-heading a:focus,#running .panel-group .panel .panel-heading a:hover{text-decoration:none}#running .panel-group .panel .panel-body{padding:0}#running .panel-group .panel .panel-body .list_container{margin-top:0;margin-bottom:0;border:0;border-radius:0}#running .panel-group .panel .panel-body .list_container .list_item{border-bottom:1px solid #ddd}#running .panel-group .panel .panel-body .list_container .list_item:last-child{border-bottom:0}.delete-button{display:none}.duplicate-button{display:none}.rename-button{display:none}.shutdown-button{display:none}.dynamic-instructions{display:inline-block;padding-top:4px}/*!
*
* IPython text editor webapp
*
*/.selected-keymap i.fa{padding:0 5px}.selected-keymap i.fa:before{content:"\f00c"}#mode-menu{overflow:auto;max-height:20em}.edit_app #header{-webkit-box-shadow:0 0 12px 1px rgba(87,87,87,0.2);box-shadow:0 0 12px 1px rgba(87,87,87,0.2)}.edit_app #menubar .navbar{margin-bottom:-1px}.dirty-indicator{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);width:20px}.dirty-indicator.pull-left{margin-right:.3em}.dirty-indicator.pull-right{margin-left:.3em}.dirty-indicator-dirty{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);width:20px}.dirty-indicator-dirty.pull-left{margin-right:.3em}.dirty-indicator-dirty.pull-right{margin-left:.3em}.dirty-indicator-clean{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);width:20px}.dirty-indicator-clean.pull-left{margin-right:.3em}.dirty-indicator-clean.pull-right{margin-left:.3em}.dirty-indicator-clean:before{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);content:"\f00c"}.dirty-indicator-clean:before.pull-left{margin-right:.3em}.dirty-indicator-clean:before.pull-right{margin-left:.3em}#filename{font-size:16pt;display:table;padding:0 5px}#current-mode{padding-left:5px;padding-right:5px}#texteditor-backdrop{padding-top:20px;padding-bottom:20px}@media not print{#texteditor-backdrop{background-color:#eee}}@media print{#texteditor-backdrop #texteditor-container .CodeMirror-gutter,#texteditor-backdrop #texteditor-container .CodeMirror-gutters{background-color:#fff}}@media not print{#texteditor-backdrop #texteditor-container .CodeMirror-gutter,#texteditor-backdrop #texteditor-container .CodeMirror-gutters{background-color:#fff}}@media not print{#texteditor-backdrop #texteditor-container{padding:0;background-color:#fff;-webkit-box-shadow:0 0 12px 1px rgba(87,87,87,0.2);box-shadow:0 0 12px 1px rgba(87,87,87,0.2)}}/*!
*
* IPython notebook
*
*/.ansibold{font-weight:bold}.ansiblack{color:black}.ansired{color:darkred}.ansigreen{color:darkgreen}.ansiyellow{color:#c4a000}.ansiblue{color:darkblue}.ansipurple{color:darkviolet}.ansicyan{color:steelblue}.ansigray{color:gray}.ansibgblack{background-color:black}.ansibgred{background-color:red}.ansibggreen{background-color:green}.ansibgyellow{background-color:yellow}.ansibgblue{background-color:blue}.ansibgpurple{background-color:magenta}.ansibgcyan{background-color:cyan}.ansibggray{background-color:gray}div.cell{border:1px solid transparent;display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch;border-radius:2px;box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box;border-width:thin;border-style:solid;width:100%;padding:5px;margin:0;outline:none}div.cell.selected{border-color:#ababab}@media print{div.cell.selected{border-color:transparent}}.edit_mode div.cell.selected{border-color:green}@media print{.edit_mode div.cell.selected{border-color:transparent}}.prompt{min-width:14ex;padding:.4em;margin:0;font-family:monospace;text-align:right;line-height:1.21429em}@media (max-width:540px){.prompt{text-align:left}}div.inner_cell{display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch;-webkit-box-flex:1;-moz-box-flex:1;box-flex:1;flex:1}@-moz-document url-prefix(){div.inner_cell{overflow-x:hidden}}div.input_area{border:1px solid #cfcfcf;border-radius:2px;background:#f7f7f7;line-height:1.21429em}div.prompt:empty{padding-top:0;padding-bottom:0}div.unrecognized_cell{padding:5px 5px 5px 0;display:-webkit-box;-webkit-box-orient:horizontal;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:horizontal;-moz-box-align:stretch;display:box;box-orient:horizontal;box-align:stretch;display:flex;flex-direction:row;align-items:stretch}div.unrecognized_cell .inner_cell{border-radius:2px;padding:5px;font-weight:bold;color:red;border:1px solid #cfcfcf;background:#eaeaea}div.unrecognized_cell .inner_cell a{color:inherit;text-decoration:none}div.unrecognized_cell .inner_cell a:hover{color:inherit;text-decoration:none}@media (max-width:540px){div.unrecognized_cell>div.prompt{display:none}}@media print{div.code_cell{page-break-inside:avoid}}div.input{page-break-inside:avoid;display:-webkit-box;-webkit-box-orient:horizontal;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:horizontal;-moz-box-align:stretch;display:box;box-orient:horizontal;box-align:stretch;display:flex;flex-direction:row;align-items:stretch}@media (max-width:540px){div.input{display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch}}div.input_prompt{color:navy;border-top:1px solid transparent}div.input_area>div.highlight{margin:.4em;border:none;padding:0;background-color:transparent}div.input_area>div.highlight>pre{margin:0;border:none;padding:0;background-color:transparent}.CodeMirror{line-height:1.21429em;font-size:14px;height:auto;background:none}.CodeMirror-scroll{overflow-y:hidden;overflow-x:auto}.CodeMirror-lines{padding:.4em}.CodeMirror-linenumber{padding:0 8px 0 4px}.CodeMirror-gutters{border-bottom-left-radius:2px;border-top-left-radius:2px}.CodeMirror pre{padding:0;border:0;border-radius:0}.highlight-base{color:#000}.highlight-variable{color:#000}.highlight-variable-2{color:#1a1a1a}.highlight-variable-3{color:#333}.highlight-string{color:#ba2121}.highlight-comment{color:#408080;font-style:italic}.highlight-number{color:#080}.highlight-atom{color:#88f}.highlight-keyword{color:#008000;font-weight:bold}.highlight-builtin{color:#008000}.highlight-error{color:#f00}.highlight-operator{color:#a2f;font-weight:bold}.highlight-meta{color:#a2f}.highlight-def{color:#00f}.highlight-string-2{color:#f50}.highlight-qualifier{color:#555}.highlight-bracket{color:#997}.highlight-tag{color:#170}.highlight-attribute{color:#00c}.highlight-header{color:blue}.highlight-quote{color:#090}.highlight-link{color:#00c}.cm-s-ipython span.cm-keyword{color:#008000;font-weight:bold}.cm-s-ipython span.cm-atom{color:#88f}.cm-s-ipython span.cm-number{color:#080}.cm-s-ipython span.cm-def{color:#00f}.cm-s-ipython span.cm-variable{color:#000}.cm-s-ipython span.cm-operator{color:#a2f;font-weight:bold}.cm-s-ipython span.cm-variable-2{color:#1a1a1a}.cm-s-ipython span.cm-variable-3{color:#333}.cm-s-ipython span.cm-comment{color:#408080;font-style:italic}.cm-s-ipython span.cm-string{color:#ba2121}.cm-s-ipython span.cm-string-2{color:#f50}.cm-s-ipython span.cm-meta{color:#a2f}.cm-s-ipython span.cm-qualifier{color:#555}.cm-s-ipython span.cm-builtin{color:#008000}.cm-s-ipython span.cm-bracket{color:#997}.cm-s-ipython span.cm-tag{color:#170}.cm-s-ipython span.cm-attribute{color:#00c}.cm-s-ipython span.cm-header{color:blue}.cm-s-ipython span.cm-quote{color:#090}.cm-s-ipython span.cm-link{color:#00c}.cm-s-ipython span.cm-error{color:#f00}.cm-s-ipython span.cm-tab{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAMCAYAAAAkuj5RAAAAAXNSR0IArs4c6QAAAGFJREFUSMft1LsRQFAQheHPowAKoACx3IgEKtaEHujDjORSgWTH/ZOdnZOcM/sgk/kFFWY0qV8foQwS4MKBCS3qR6ixBJvElOobYAtivseIE120FaowJPN75GMu8j/LfMwNjh4HUpwg4LUAAAAASUVORK5CYII=);background-position:right;background-repeat:no-repeat}div.output_wrapper{position:relative;display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch;z-index:1}div.output_scroll{height:24em;width:100%;overflow:auto;border-radius:2px;-webkit-box-shadow:inset 0 2px 8px rgba(0,0,0,0.8);box-shadow:inset 0 2px 8px rgba(0,0,0,0.8);display:block}div.output_collapsed{margin:0;padding:0;display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch}div.out_prompt_overlay{height:100%;padding:0 .4em;position:absolute;border-radius:2px}div.out_prompt_overlay:hover{-webkit-box-shadow:inset 0 0 1px #000;box-shadow:inset 0 0 1px #000;background:rgba(240,240,240,0.5)}div.output_prompt{color:darkred}div.output_area{padding:0;page-break-inside:avoid;display:-webkit-box;-webkit-box-orient:horizontal;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:horizontal;-moz-box-align:stretch;display:box;box-orient:horizontal;box-align:stretch;display:flex;flex-direction:row;align-items:stretch}div.output_area .MathJax_Display{text-align:left !important}div.output_area .rendered_html table{margin-left:0;margin-right:0}div.output_area .rendered_html img{margin-left:0;margin-right:0}div.output_area img,div.output_area svg{max-width:100%;height:auto}div.output_area img.unconfined,div.output_area svg.unconfined{max-width:none}.output{display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch}@media (max-width:540px){div.output_area{display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch}}div.output_area pre{margin:0;padding:0;border:0;vertical-align:baseline;color:black;background-color:transparent;border-radius:0}div.output_subarea{overflow-x:auto;padding:.4em;-webkit-box-flex:1;-moz-box-flex:1;box-flex:1;flex:1;max-width:calc(100% - 14ex)}div.output_text{text-align:left;color:#000;line-height:1.21429em}div.output_stderr{background:#fdd}div.output_latex{text-align:left}div.output_javascript:empty{padding:0}.js-error{color:darkred}div.raw_input_container{font-family:monospace;padding-top:5px}input.raw_input{font-family:inherit;font-size:inherit;color:inherit;width:auto;vertical-align:baseline;padding:0 .25em;margin:0 .25em}input.raw_input:focus{box-shadow:none}p.p-space{margin-bottom:10px}div.output_unrecognized{padding:5px;font-weight:bold;color:red}div.output_unrecognized a{color:inherit;text-decoration:none}div.output_unrecognized a:hover{color:inherit;text-decoration:none}.rendered_html{color:#000}.rendered_html em{font-style:italic}.rendered_html strong{font-weight:bold}.rendered_html u{text-decoration:underline}.rendered_html :link{text-decoration:underline}.rendered_html :visited{text-decoration:underline}.rendered_html h1{font-size:185.7%;margin:1.08em 0 0 0;font-weight:bold;line-height:1}.rendered_html h2{font-size:157.1%;margin:1.27em 0 0 0;font-weight:bold;line-height:1}.rendered_html h3{font-size:128.6%;margin:1.55em 0 0 0;font-weight:bold;line-height:1}.rendered_html h4{font-size:100%;margin:2em 0 0 0;font-weight:bold;line-height:1}.rendered_html h5{font-size:100%;margin:2em 0 0 0;font-weight:bold;line-height:1;font-style:italic}.rendered_html h6{font-size:100%;margin:2em 0 0 0;font-weight:bold;line-height:1;font-style:italic}.rendered_html h1:first-child{margin-top:.538em}.rendered_html h2:first-child{margin-top:.636em}.rendered_html h3:first-child{margin-top:.777em}.rendered_html h4:first-child{margin-top:1em}.rendered_html h5:first-child{margin-top:1em}.rendered_html h6:first-child{margin-top:1em}.rendered_html ul{list-style:disc;margin:0 2em;padding-left:0}.rendered_html ul ul{list-style:square;margin:0 2em}.rendered_html ul ul ul{list-style:circle;margin:0 2em}.rendered_html ol{list-style:decimal;margin:0 2em;padding-left:0}.rendered_html ol ol{list-style:upper-alpha;margin:0 2em}.rendered_html ol ol ol{list-style:lower-alpha;margin:0 2em}.rendered_html ol ol ol ol{list-style:lower-roman;margin:0 2em}.rendered_html ol ol ol ol ol{list-style:decimal;margin:0 2em}.rendered_html *+ul{margin-top:1em}.rendered_html *+ol{margin-top:1em}.rendered_html hr{color:black;background-color:black}.rendered_html pre{margin:1em 2em}.rendered_html pre,.rendered_html code{border:0;background-color:#fff;color:#000;font-size:100%;padding:0}.rendered_html blockquote{margin:1em 2em}.rendered_html table{margin-left:auto;margin-right:auto;border:1px solid black;border-collapse:collapse}.rendered_html tr,.rendered_html th,.rendered_html td{border:1px solid black;border-collapse:collapse;margin:1em 2em}.rendered_html td,.rendered_html th{text-align:left;vertical-align:middle;padding:4px}.rendered_html th{font-weight:bold}.rendered_html *+table{margin-top:1em}.rendered_html p{text-align:left}.rendered_html *+p{margin-top:1em}.rendered_html img{display:block;margin-left:auto;margin-right:auto}.rendered_html *+img{margin-top:1em}.rendered_html img,.rendered_html svg{max-width:100%;height:auto}.rendered_html img.unconfined,.rendered_html svg.unconfined{max-width:none}div.text_cell{display:-webkit-box;-webkit-box-orient:horizontal;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:horizontal;-moz-box-align:stretch;display:box;box-orient:horizontal;box-align:stretch;display:flex;flex-direction:row;align-items:stretch}@media (max-width:540px){div.text_cell>div.prompt{display:none}}div.text_cell_render{outline:none;resize:none;width:inherit;border-style:none;padding:.5em .5em .5em .4em;color:#000;box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box}a.anchor-link:link{text-decoration:none;padding:0 20px;visibility:hidden}h1:hover .anchor-link,h2:hover .anchor-link,h3:hover .anchor-link,h4:hover .anchor-link,h5:hover .anchor-link,h6:hover .anchor-link{visibility:visible}.text_cell.rendered .input_area{display:none}.text_cell.rendered .rendered_html{overflow-x:auto}.text_cell.unrendered .text_cell_render{display:none}.cm-header-1,.cm-header-2,.cm-header-3,.cm-header-4,.cm-header-5,.cm-header-6{font-weight:bold;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif}.cm-header-1{font-size:185.7%}.cm-header-2{font-size:157.1%}.cm-header-3{font-size:128.6%}.cm-header-4{font-size:110%}.cm-header-5{font-size:100%;font-style:italic}.cm-header-6{font-size:100%;font-style:italic}.widget-interact>div,.widget-interact>input{padding:2.5px}.widget-area{page-break-inside:avoid;display:-webkit-box;-webkit-box-orient:horizontal;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:horizontal;-moz-box-align:stretch;display:box;box-orient:horizontal;box-align:stretch;display:flex;flex-direction:row;align-items:stretch}.widget-area .widget-subarea{padding:.44em .4em .4em 1px;margin-left:6px;box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box;display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch;-webkit-box-flex:2;-moz-box-flex:2;box-flex:2;flex:2;-webkit-box-align:start;-moz-box-align:start;box-align:start;align-items:flex-start}.widget-area.connection-problems .prompt:after{content:"\f127";font-family:'FontAwesome';color:#d9534f;font-size:14px;top:3px;padding:3px}.slide-track{border:1px solid #ccc;background:#fff;border-radius:2px}.widget-hslider{padding-left:8px;padding-right:2px;overflow:visible;width:350px;height:5px;max-height:5px;margin-top:13px;margin-bottom:10px;border:1px solid #ccc;background:#fff;border-radius:2px;display:-webkit-box;-webkit-box-orient:horizontal;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:horizontal;-moz-box-align:stretch;display:box;box-orient:horizontal;box-align:stretch;display:flex;flex-direction:row;align-items:stretch}.widget-hslider .ui-slider{border:0;background:none;display:-webkit-box;-webkit-box-orient:horizontal;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:horizontal;-moz-box-align:stretch;display:box;box-orient:horizontal;box-align:stretch;display:flex;flex-direction:row;align-items:stretch;-webkit-box-flex:1;-moz-box-flex:1;box-flex:1;flex:1}.widget-hslider .ui-slider .ui-slider-handle{width:12px;height:28px;margin-top:-8px;border-radius:2px}.widget-hslider .ui-slider .ui-slider-range{height:12px;margin-top:-4px;background:#eee}.widget-vslider{padding-bottom:5px;overflow:visible;width:5px;max-width:5px;height:250px;margin-left:12px;border:1px solid #ccc;background:#fff;border-radius:2px;display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch}.widget-vslider .ui-slider{border:0;background:none;margin-left:-4px;margin-top:5px;display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch;-webkit-box-flex:1;-moz-box-flex:1;box-flex:1;flex:1}.widget-vslider .ui-slider .ui-slider-handle{width:28px;height:12px;margin-left:-9px;border-radius:2px}.widget-vslider .ui-slider .ui-slider-range{width:12px;margin-left:-1px;background:#eee}.widget-text{width:350px;margin:0}.widget-listbox{width:350px;margin-bottom:0}.widget-numeric-text{width:150px;margin:0}.widget-progress{margin-top:6px;min-width:350px}.widget-progress .progress-bar{-webkit-transition:none;-moz-transition:none;-ms-transition:none;-o-transition:none;transition:none}.widget-combo-btn{min-width:125px}.widget_item .dropdown-menu li a{color:inherit}.widget-hbox{display:-webkit-box;-webkit-box-orient:horizontal;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:horizontal;-moz-box-align:stretch;display:box;box-orient:horizontal;box-align:stretch;display:flex;flex-direction:row;align-items:stretch}.widget-hbox input[type="checkbox"]{margin-top:9px;margin-bottom:10px}.widget-hbox .widget-label{min-width:10ex;padding-right:8px;padding-top:5px;text-align:right;vertical-align:text-top}.widget-hbox .widget-readout{padding-left:8px;padding-top:5px;text-align:left;vertical-align:text-top}.widget-vbox{display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch}.widget-vbox .widget-label{padding-bottom:5px;text-align:center;vertical-align:text-bottom}.widget-vbox .widget-readout{padding-top:5px;text-align:center;vertical-align:text-top}.widget-box{box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box;-webkit-box-align:start;-moz-box-align:start;box-align:start;align-items:flex-start}.widget-radio-box{display:-webkit-box;-webkit-box-orient:vertical;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:vertical;-moz-box-align:stretch;display:box;box-orient:vertical;box-align:stretch;display:flex;flex-direction:column;align-items:stretch;box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box;padding-top:4px}.widget-radio-box label{margin-top:0;margin-left:20px}/*!
*
* IPython notebook webapp
*
*/@media (max-width:767px){.notebook_app{padding-left:0;padding-right:0}}#ipython-main-app{box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box;height:100%}div#notebook_panel{margin:0;padding:0;box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box;height:100%}#notebook{font-size:14px;line-height:20px;overflow-y:hidden;overflow-x:auto;width:100%;padding-top:20px;margin:0;outline:none;box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box;min-height:100%}@media not print{#notebook-container{padding:15px;background-color:#fff;min-height:0;-webkit-box-shadow:0 0 12px 1px rgba(87,87,87,0.2);box-shadow:0 0 12px 1px rgba(87,87,87,0.2)}}div.ui-widget-content{border:1px solid #ababab;outline:none}pre.dialog{background-color:#f7f7f7;border:1px solid #ddd;border-radius:2px;padding:.4em;padding-left:2em}p.dialog{padding:.2em}pre,code,kbd,samp{white-space:pre-wrap}#fonttest{font-family:monospace}p{margin-bottom:0}.end_space{min-height:100px;transition:height .2s ease}.notebook_app #header{-webkit-box-shadow:0 0 12px 1px rgba(87,87,87,0.2);box-shadow:0 0 12px 1px rgba(87,87,87,0.2)}@media not print{.notebook_app{background-color:#eee}}.celltoolbar{border:thin solid #cfcfcf;border-bottom:none;background:#eee;border-radius:2px 2px 0 0;width:100%;height:29px;padding-right:4px;display:-webkit-box;-webkit-box-orient:horizontal;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:horizontal;-moz-box-align:stretch;display:box;box-orient:horizontal;box-align:stretch;display:flex;flex-direction:row;align-items:stretch;-webkit-box-pack:end;-moz-box-pack:end;box-pack:end;justify-content:flex-end}@media print{.celltoolbar{display:none}}.ctb_hideshow{display:none;vertical-align:bottom}.ctb_global_show .ctb_show.ctb_hideshow{display:block}.ctb_global_show .ctb_show+.input_area,.ctb_global_show .ctb_show+div.text_cell_input,.ctb_global_show .ctb_show~div.text_cell_render{border-top-right-radius:0;border-top-left-radius:0}.ctb_global_show .ctb_show~div.text_cell_render{border:1px solid #cfcfcf}.celltoolbar{font-size:87%;padding-top:3px}.celltoolbar select{display:block;width:100%;height:32px;padding:6px 12px;font-size:13px;line-height:1.42857143;color:#555;background-color:#fff;background-image:none;border:1px solid #ccc;border-radius:2px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,0.075);box-shadow:inset 0 1px 1px rgba(0,0,0,0.075);-webkit-transition:border-color ease-in-out .15s, box-shadow ease-in-out .15s;-o-transition:border-color ease-in-out .15s, box-shadow ease-in-out .15s;transition:border-color ease-in-out .15s, box-shadow ease-in-out .15s;height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:1px;width:inherit;font-size:inherit;height:22px;padding:0;display:inline-block}.celltoolbar select:focus{border-color:#66afe9;outline:0;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102, 175, 233, 0.6);box-shadow:inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102, 175, 233, 0.6)}.celltoolbar select::-moz-placeholder{color:#999;opacity:1}.celltoolbar select:-ms-input-placeholder{color:#999}.celltoolbar select::-webkit-input-placeholder{color:#999}.celltoolbar select[disabled],.celltoolbar select[readonly],fieldset[disabled] .celltoolbar select{cursor:not-allowed;background-color:#eee;opacity:1}textarea.celltoolbar select{height:auto}select.celltoolbar select{height:30px;line-height:30px}textarea.celltoolbar select,select[multiple].celltoolbar select{height:auto}.celltoolbar label{margin-left:5px;margin-right:5px}.completions{position:absolute;z-index:10;overflow:hidden;border:1px solid #ababab;border-radius:2px;-webkit-box-shadow:0 6px 10px -1px #adadad;box-shadow:0 6px 10px -1px #adadad}.completions select{background:white;outline:none;border:none;padding:0;margin:0;overflow:auto;font-family:monospace;font-size:110%;color:#000;width:auto}.completions select option.context{color:#286090}#kernel_logo_widget{float:right !important;float:right}#kernel_logo_widget .current_kernel_logo{display:none;margin-top:-1px;margin-bottom:-1px;width:32px;height:32px}#menubar{box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box;margin-top:1px}#menubar .navbar{border-top:1px;border-radius:0 0 2px 2px;margin-bottom:0}#menubar .navbar-toggle{float:left;padding-top:7px;padding-bottom:7px;border:none}#menubar .navbar-collapse{clear:left}.nav-wrapper{border-bottom:1px solid #e7e7e7}i.menu-icon{padding-top:4px}ul#help_menu li a{overflow:hidden;padding-right:2.2em}ul#help_menu li a i{margin-right:-1.2em}.dropdown-submenu{position:relative}.dropdown-submenu>.dropdown-menu{top:0;left:100%;margin-top:-6px;margin-left:-1px}.dropdown-submenu:hover>.dropdown-menu{display:block}.dropdown-submenu>a:after{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);display:block;content:"\f0da";float:right;color:#333;margin-top:2px;margin-right:-10px}.dropdown-submenu>a:after.pull-left{margin-right:.3em}.dropdown-submenu>a:after.pull-right{margin-left:.3em}.dropdown-submenu:hover>a:after{color:#262626}.dropdown-submenu.pull-left{float:none}.dropdown-submenu.pull-left>.dropdown-menu{left:-100%;margin-left:10px}#notification_area{float:right !important;float:right;z-index:10}.indicator_area{float:right !important;float:right;color:#777;margin-left:5px;margin-right:5px;width:11px;z-index:10;text-align:center;width:auto}#kernel_indicator{float:right !important;float:right;color:#777;margin-left:5px;margin-right:5px;width:11px;z-index:10;text-align:center;width:auto;border-left:1px solid}#kernel_indicator .kernel_indicator_name{padding-left:5px;padding-right:5px}#modal_indicator{float:right !important;float:right;color:#777;margin-left:5px;margin-right:5px;width:11px;z-index:10;text-align:center;width:auto}#readonly-indicator{float:right !important;float:right;color:#777;margin-left:5px;margin-right:5px;width:11px;z-index:10;text-align:center;width:auto;margin-top:2px;margin-bottom:0;margin-left:0;margin-right:0;display:none}.modal_indicator:before{width:1.28571429em;text-align:center}.edit_mode .modal_indicator:before{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);content:"\f040"}.edit_mode .modal_indicator:before.pull-left{margin-right:.3em}.edit_mode .modal_indicator:before.pull-right{margin-left:.3em}.command_mode .modal_indicator:before{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);content:' '}.command_mode .modal_indicator:before.pull-left{margin-right:.3em}.command_mode .modal_indicator:before.pull-right{margin-left:.3em}.kernel_idle_icon:before{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);content:"\f10c"}.kernel_idle_icon:before.pull-left{margin-right:.3em}.kernel_idle_icon:before.pull-right{margin-left:.3em}.kernel_busy_icon:before{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);content:"\f111"}.kernel_busy_icon:before.pull-left{margin-right:.3em}.kernel_busy_icon:before.pull-right{margin-left:.3em}.kernel_dead_icon:before{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);content:"\f1e2"}.kernel_dead_icon:before.pull-left{margin-right:.3em}.kernel_dead_icon:before.pull-right{margin-left:.3em}.kernel_disconnected_icon:before{display:inline-block;font:normal normal normal 14px/1 FontAwesome;font-size:inherit;text-rendering:auto;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;transform:translate(0, 0);content:"\f127"}.kernel_disconnected_icon:before.pull-left{margin-right:.3em}.kernel_disconnected_icon:before.pull-right{margin-left:.3em}.notification_widget{color:#777;z-index:10;background:rgba(240,240,240,0.5);margin-right:4px;color:#333;background-color:#fff;border-color:#ccc}.notification_widget:hover,.notification_widget:focus,.notification_widget.focus,.notification_widget:active,.notification_widget.active,.open>.dropdown-toggle.notification_widget{color:#333;background-color:#e6e6e6;border-color:#adadad}.notification_widget:active,.notification_widget.active,.open>.dropdown-toggle.notification_widget{background-image:none}.notification_widget.disabled,.notification_widget[disabled],fieldset[disabled] .notification_widget,.notification_widget.disabled:hover,.notification_widget[disabled]:hover,fieldset[disabled] .notification_widget:hover,.notification_widget.disabled:focus,.notification_widget[disabled]:focus,fieldset[disabled] .notification_widget:focus,.notification_widget.disabled.focus,.notification_widget[disabled].focus,fieldset[disabled] .notification_widget.focus,.notification_widget.disabled:active,.notification_widget[disabled]:active,fieldset[disabled] .notification_widget:active,.notification_widget.disabled.active,.notification_widget[disabled].active,fieldset[disabled] .notification_widget.active{background-color:#fff;border-color:#ccc}.notification_widget .badge{color:#fff;background-color:#333}.notification_widget.warning{color:#fff;background-color:#f0ad4e;border-color:#eea236}.notification_widget.warning:hover,.notification_widget.warning:focus,.notification_widget.warning.focus,.notification_widget.warning:active,.notification_widget.warning.active,.open>.dropdown-toggle.notification_widget.warning{color:#fff;background-color:#ec971f;border-color:#d58512}.notification_widget.warning:active,.notification_widget.warning.active,.open>.dropdown-toggle.notification_widget.warning{background-image:none}.notification_widget.warning.disabled,.notification_widget.warning[disabled],fieldset[disabled] .notification_widget.warning,.notification_widget.warning.disabled:hover,.notification_widget.warning[disabled]:hover,fieldset[disabled] .notification_widget.warning:hover,.notification_widget.warning.disabled:focus,.notification_widget.warning[disabled]:focus,fieldset[disabled] .notification_widget.warning:focus,.notification_widget.warning.disabled.focus,.notification_widget.warning[disabled].focus,fieldset[disabled] .notification_widget.warning.focus,.notification_widget.warning.disabled:active,.notification_widget.warning[disabled]:active,fieldset[disabled] .notification_widget.warning:active,.notification_widget.warning.disabled.active,.notification_widget.warning[disabled].active,fieldset[disabled] .notification_widget.warning.active{background-color:#f0ad4e;border-color:#eea236}.notification_widget.warning .badge{color:#f0ad4e;background-color:#fff}.notification_widget.success{color:#fff;background-color:#5cb85c;border-color:#4cae4c}.notification_widget.success:hover,.notification_widget.success:focus,.notification_widget.success.focus,.notification_widget.success:active,.notification_widget.success.active,.open>.dropdown-toggle.notification_widget.success{color:#fff;background-color:#449d44;border-color:#398439}.notification_widget.success:active,.notification_widget.success.active,.open>.dropdown-toggle.notification_widget.success{background-image:none}.notification_widget.success.disabled,.notification_widget.success[disabled],fieldset[disabled] .notification_widget.success,.notification_widget.success.disabled:hover,.notification_widget.success[disabled]:hover,fieldset[disabled] .notification_widget.success:hover,.notification_widget.success.disabled:focus,.notification_widget.success[disabled]:focus,fieldset[disabled] .notification_widget.success:focus,.notification_widget.success.disabled.focus,.notification_widget.success[disabled].focus,fieldset[disabled] .notification_widget.success.focus,.notification_widget.success.disabled:active,.notification_widget.success[disabled]:active,fieldset[disabled] .notification_widget.success:active,.notification_widget.success.disabled.active,.notification_widget.success[disabled].active,fieldset[disabled] .notification_widget.success.active{background-color:#5cb85c;border-color:#4cae4c}.notification_widget.success .badge{color:#5cb85c;background-color:#fff}.notification_widget.info{color:#fff;background-color:#5bc0de;border-color:#46b8da}.notification_widget.info:hover,.notification_widget.info:focus,.notification_widget.info.focus,.notification_widget.info:active,.notification_widget.info.active,.open>.dropdown-toggle.notification_widget.info{color:#fff;background-color:#31b0d5;border-color:#269abc}.notification_widget.info:active,.notification_widget.info.active,.open>.dropdown-toggle.notification_widget.info{background-image:none}.notification_widget.info.disabled,.notification_widget.info[disabled],fieldset[disabled] .notification_widget.info,.notification_widget.info.disabled:hover,.notification_widget.info[disabled]:hover,fieldset[disabled] .notification_widget.info:hover,.notification_widget.info.disabled:focus,.notification_widget.info[disabled]:focus,fieldset[disabled] .notification_widget.info:focus,.notification_widget.info.disabled.focus,.notification_widget.info[disabled].focus,fieldset[disabled] .notification_widget.info.focus,.notification_widget.info.disabled:active,.notification_widget.info[disabled]:active,fieldset[disabled] .notification_widget.info:active,.notification_widget.info.disabled.active,.notification_widget.info[disabled].active,fieldset[disabled] .notification_widget.info.active{background-color:#5bc0de;border-color:#46b8da}.notification_widget.info .badge{color:#5bc0de;background-color:#fff}.notification_widget.danger{color:#fff;background-color:#d9534f;border-color:#d43f3a}.notification_widget.danger:hover,.notification_widget.danger:focus,.notification_widget.danger.focus,.notification_widget.danger:active,.notification_widget.danger.active,.open>.dropdown-toggle.notification_widget.danger{color:#fff;background-color:#c9302c;border-color:#ac2925}.notification_widget.danger:active,.notification_widget.danger.active,.open>.dropdown-toggle.notification_widget.danger{background-image:none}.notification_widget.danger.disabled,.notification_widget.danger[disabled],fieldset[disabled] .notification_widget.danger,.notification_widget.danger.disabled:hover,.notification_widget.danger[disabled]:hover,fieldset[disabled] .notification_widget.danger:hover,.notification_widget.danger.disabled:focus,.notification_widget.danger[disabled]:focus,fieldset[disabled] .notification_widget.danger:focus,.notification_widget.danger.disabled.focus,.notification_widget.danger[disabled].focus,fieldset[disabled] .notification_widget.danger.focus,.notification_widget.danger.disabled:active,.notification_widget.danger[disabled]:active,fieldset[disabled] .notification_widget.danger:active,.notification_widget.danger.disabled.active,.notification_widget.danger[disabled].active,fieldset[disabled] .notification_widget.danger.active{background-color:#d9534f;border-color:#d43f3a}.notification_widget.danger .badge{color:#d9534f;background-color:#fff}div#pager{background-color:#fff;font-size:14px;line-height:20px;overflow:hidden;display:none;position:fixed;bottom:0;width:100%;max-height:50%;padding-top:8px;-webkit-box-shadow:0 0 12px 1px rgba(87,87,87,0.2);box-shadow:0 0 12px 1px rgba(87,87,87,0.2);z-index:100;top:auto !important}div#pager pre{line-height:1.21429em;color:#000;background-color:#f7f7f7;padding:.4em}div#pager #pager-button-area{position:absolute;top:8px;right:20px}div#pager #pager-contents{position:relative;overflow:auto;width:100%;height:100%}div#pager #pager-contents #pager-container{position:relative;padding:15px 0;box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box}div#pager .ui-resizable-handle{top:0;height:8px;background:#f7f7f7;border-top:1px solid #cfcfcf;border-bottom:1px solid #cfcfcf}div#pager .ui-resizable-handle::after{content:'';top:2px;left:50%;height:3px;width:30px;margin-left:-15px;position:absolute;border-top:1px solid #cfcfcf}.quickhelp{display:-webkit-box;-webkit-box-orient:horizontal;-webkit-box-align:stretch;display:-moz-box;-moz-box-orient:horizontal;-moz-box-align:stretch;display:box;box-orient:horizontal;box-align:stretch;display:flex;flex-direction:row;align-items:stretch}.shortcut_key{display:inline-block;width:20ex;text-align:right;font-family:monospace}.shortcut_descr{display:inline-block;-webkit-box-flex:1;-moz-box-flex:1;box-flex:1;flex:1}span.save_widget{margin-top:6px}span.save_widget span.filename{height:1em;line-height:1em;padding:3px;margin-left:16px;border:none;font-size:146.5%;border-radius:2px}span.save_widget span.filename:hover{background-color:#e6e6e6}span.checkpoint_status,span.autosave_status{font-size:small}@media (max-width:767px){span.save_widget{font-size:small}span.checkpoint_status,span.autosave_status{display:none}}@media (min-width:768px) and (max-width:991px){span.checkpoint_status{display:none}span.autosave_status{font-size:x-small}}.toolbar{padding:0;margin-left:-5px;margin-top:2px;margin-bottom:5px;box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box}.toolbar select,.toolbar label{width:auto;vertical-align:middle;margin-right:2px;margin-bottom:0;display:inline;font-size:92%;margin-left:.3em;margin-right:.3em;padding:0;padding-top:3px}.toolbar .btn{padding:2px 8px}.toolbar .btn-group{margin-top:0;margin-left:5px}#maintoolbar{margin-bottom:-3px;margin-top:-8px;border:0;min-height:27px;margin-left:0;padding-top:11px;padding-bottom:3px}#maintoolbar .navbar-text{float:none;vertical-align:middle;text-align:right;margin-left:5px;margin-right:0;margin-top:0}.select-xs{height:24px}@-moz-keyframes fadeOut{from{opacity:1}to{opacity:0}}@-webkit-keyframes fadeOut{from{opacity:1}to{opacity:0}}@-moz-keyframes fadeIn{from{opacity:0}to{opacity:1}}@-webkit-keyframes fadeIn{from{opacity:0}to{opacity:1}}.bigtooltip{overflow:auto;height:200px;-webkit-transition-property:height;-webkit-transition-duration:500ms;-moz-transition-property:height;-moz-transition-duration:500ms;transition-property:height;transition-duration:500ms}.smalltooltip{-webkit-transition-property:height;-webkit-transition-duration:500ms;-moz-transition-property:height;-moz-transition-duration:500ms;transition-property:height;transition-duration:500ms;text-overflow:ellipsis;overflow:hidden;height:80px}.tooltipbuttons{position:absolute;padding-right:15px;top:0;right:0}.tooltiptext{padding-right:30px}.ipython_tooltip{max-width:700px;-webkit-animation:fadeOut 400ms;-moz-animation:fadeOut 400ms;animation:fadeOut 400ms;-webkit-animation:fadeIn 400ms;-moz-animation:fadeIn 400ms;animation:fadeIn 400ms;vertical-align:middle;background-color:#f7f7f7;overflow:visible;border:#ababab 1px solid;outline:none;padding:3px;margin:0;padding-left:7px;font-family:monospace;min-height:50px;-moz-box-shadow:0 6px 10px -1px #adadad;-webkit-box-shadow:0 6px 10px -1px #adadad;box-shadow:0 6px 10px -1px #adadad;border-radius:2px;position:absolute;z-index:1000}.ipython_tooltip a{float:right}.ipython_tooltip .tooltiptext pre{border:0;border-radius:0;font-size:100%;background-color:#f7f7f7}.pretooltiparrow{left:0;margin:0;top:-16px;width:40px;height:16px;overflow:hidden;position:absolute}.pretooltiparrow:before{background-color:#f7f7f7;border:1px #ababab solid;z-index:11;content:"";position:absolute;left:15px;top:10px;width:25px;height:25px;-webkit-transform:rotate(45deg);-moz-transform:rotate(45deg);-ms-transform:rotate(45deg);-o-transform:rotate(45deg)}.terminal-app{background:#eee}.terminal-app #header{background:#fff;-webkit-box-shadow:0 0 12px 1px rgba(87,87,87,0.2);box-shadow:0 0 12px 1px rgba(87,87,87,0.2)}.terminal-app .terminal{float:left;font-family:monospace;color:white;background:black;padding:.4em;border-radius:2px;-webkit-box-shadow:0 0 12px 1px rgba(87,87,87,0.4);box-shadow:0 0 12px 1px rgba(87,87,87,0.4)}.terminal-app .terminal,.terminal-app .terminal dummy-screen{line-height:1em;font-size:14px}.terminal-app .terminal-cursor{color:black;background:white}.terminal-app #terminado-container{margin-top:20px}/*# sourceMappingURL=style.min.css.map */
    </style>
<style type="text/css">
    .highlight .hll { background-color: #ffffcc }
.highlight  { background: #f8f8f8; }
.highlight .c { color: #408080; font-style: italic } /* Comment */
.highlight .err { border: 1px solid #FF0000 } /* Error */
.highlight .k { color: #008000; font-weight: bold } /* Keyword */
.highlight .o { color: #666666 } /* Operator */
.highlight .cm { color: #408080; font-style: italic } /* Comment.Multiline */
.highlight .cp { color: #BC7A00 } /* Comment.Preproc */
.highlight .c1 { color: #408080; font-style: italic } /* Comment.Single */
.highlight .cs { color: #408080; font-style: italic } /* Comment.Special */
.highlight .gd { color: #A00000 } /* Generic.Deleted */
.highlight .ge { font-style: italic } /* Generic.Emph */
.highlight .gr { color: #FF0000 } /* Generic.Error */
.highlight .gh { color: #000080; font-weight: bold } /* Generic.Heading */
.highlight .gi { color: #00A000 } /* Generic.Inserted */
.highlight .go { color: #888888 } /* Generic.Output */
.highlight .gp { color: #000080; font-weight: bold } /* Generic.Prompt */
.highlight .gs { font-weight: bold } /* Generic.Strong */
.highlight .gu { color: #800080; font-weight: bold } /* Generic.Subheading */
.highlight .gt { color: #0044DD } /* Generic.Traceback */
.highlight .kc { color: #008000; font-weight: bold } /* Keyword.Constant */
.highlight .kd { color: #008000; font-weight: bold } /* Keyword.Declaration */
.highlight .kn { color: #008000; font-weight: bold } /* Keyword.Namespace */
.highlight .kp { color: #008000 } /* Keyword.Pseudo */
.highlight .kr { color: #008000; font-weight: bold } /* Keyword.Reserved */
.highlight .kt { color: #B00040 } /* Keyword.Type */
.highlight .m { color: #666666 } /* Literal.Number */
.highlight .s { color: #BA2121 } /* Literal.String */
.highlight .na { color: #7D9029 } /* Name.Attribute */
.highlight .nb { color: #008000 } /* Name.Builtin */
.highlight .nc { color: #0000FF; font-weight: bold } /* Name.Class */
.highlight .no { color: #880000 } /* Name.Constant */
.highlight .nd { color: #AA22FF } /* Name.Decorator */
.highlight .ni { color: #999999; font-weight: bold } /* Name.Entity */
.highlight .ne { color: #D2413A; font-weight: bold } /* Name.Exception */
.highlight .nf { color: #0000FF } /* Name.Function */
.highlight .nl { color: #A0A000 } /* Name.Label */
.highlight .nn { color: #0000FF; font-weight: bold } /* Name.Namespace */
.highlight .nt { color: #008000; font-weight: bold } /* Name.Tag */
.highlight .nv { color: #19177C } /* Name.Variable */
.highlight .ow { color: #AA22FF; font-weight: bold } /* Operator.Word */
.highlight .w { color: #bbbbbb } /* Text.Whitespace */
.highlight .mf { color: #666666 } /* Literal.Number.Float */
.highlight .mh { color: #666666 } /* Literal.Number.Hex */
.highlight .mi { color: #666666 } /* Literal.Number.Integer */
.highlight .mo { color: #666666 } /* Literal.Number.Oct */
.highlight .sb { color: #BA2121 } /* Literal.String.Backtick */
.highlight .sc { color: #BA2121 } /* Literal.String.Char */
.highlight .sd { color: #BA2121; font-style: italic } /* Literal.String.Doc */
.highlight .s2 { color: #BA2121 } /* Literal.String.Double */
.highlight .se { color: #BB6622; font-weight: bold } /* Literal.String.Escape */
.highlight .sh { color: #BA2121 } /* Literal.String.Heredoc */
.highlight .si { color: #BB6688; font-weight: bold } /* Literal.String.Interpol */
.highlight .sx { color: #008000 } /* Literal.String.Other */
.highlight .sr { color: #BB6688 } /* Literal.String.Regex */
.highlight .s1 { color: #BA2121 } /* Literal.String.Single */
.highlight .ss { color: #19177C } /* Literal.String.Symbol */
.highlight .bp { color: #008000 } /* Name.Builtin.Pseudo */
.highlight .vc { color: #19177C } /* Name.Variable.Class */
.highlight .vg { color: #19177C } /* Name.Variable.Global */
.highlight .vi { color: #19177C } /* Name.Variable.Instance */
.highlight .il { color: #666666 } /* Literal.Number.Integer.Long */
    </style>


<style type="text/css">
/* Overrides of notebook CSS for static HTML export */
body {
  overflow: visible;
  padding: 8px;
}

div#notebook {
  overflow: visible;
  border-top: none;
}

@media print {
  div.cell {
    display: block;
    page-break-inside: avoid;
  } 
  div.output_wrapper { 
    display: block;
    page-break-inside: avoid; 
  }
  div.output { 
    display: block;
    page-break-inside: avoid; 
  }
}
</style>

<!-- Custom stylesheet, it must be in the same directory as the html file -->
<link rel="stylesheet" href="custom.css">

<!-- Loading mathjax macro -->
<!-- Load mathjax -->
    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML"></script>
    <!-- MathJax configuration -->
    <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        tex2jax: {
            inlineMath: [ ['$','$'], ["\\(","\\)"] ],
            displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
            processEscapes: true,
            processEnvironments: true
        },
        // Center justify equations in code and markdown cells. Elsewhere
        // we use CSS to left justify single line equations in code cells.
        displayAlign: 'center',
        "HTML-CSS": {
            styles: {'.MathJax_Display': {"margin": 0}},
            linebreaks: { automatic: true }
        }
    });
    </script>
    <!-- End of mathjax configuration -->

</head>
<body>
  <div tabindex="-1" id="notebook" class="border-box-sizing">
    <div class="container" id="notebook-container">

<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="metaBEAT-analysis-of-Illumina-seq-output-for-pollen-transported-by-moths">metaBEAT analysis of Illumina seq output for pollen transported by moths<a class="anchor-link" href="#metaBEAT-analysis-of-Illumina-seq-output-for-pollen-transported-by-moths">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>The first step will be to <strong>trim/clean our raw Illumina data</strong>.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Prepare a text file specifying the samples to be processed including the format and location of the reads.</p>
<p>The below command expects the Illumina data to be present in 2 fastq files (forward and reverse reads) per sample in a directory ./raw_data/. It expects the files to be named 'plateID_L001', followed by 'R1' or 'R2' to identify the forward/reverse read file respectively.</p>
<p>We need a query map which lists these files along with the primer combinations for each well in each file, and the length of the primer sequence (including heterogeneity spacers, but NOT including any tags) to be trimmed off.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[1]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>head Querymap_global.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>









</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[3]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span> <span class="n">raw_data</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>Moth1_S1_L001_R1_001.fastq.gz  Moth3_S3_L001_R1_001.fastq.gz
Moth1_S1_L001_R2_001.fastq.gz  Moth3_S3_L001_R2_001.fastq.gz
Moth2_S2_L001_R1_001.fastq.gz  Moth4_S4_L001_R1_001.fastq.gz
Moth2_S2_L001_R2_001.fastq.gz  Moth4_S4_L001_R2_001.fastq.gz
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[5]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py -h
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>usage: metaBEAT.py [-h] [-Q &lt;FILE&gt;] [-B &lt;FILE&gt;] [--g_queries &lt;FILE&gt;] [-v] [-s]
                   [-f] [-p] [-k] [-t] [-b] [-m &lt;string&gt;] [-n &lt;INT&gt;] [-E] [-e]
                   [--read_stats_off] [--PCR_primer &lt;FILE&gt;]
                   [--trim_adapter &lt;FILE&gt;] [--trim_qual &lt;INT&gt;] [--phred &lt;INT&gt;]
                   [--trim_window &lt;INT&gt;] [--read_crop &lt;INT&gt;]
                   [--trim_minlength &lt;INT&gt;] [--merge] [--product_length &lt;INT&gt;]
                   [--merged_only] [--forward_only] [--length_filter &lt;INT&gt;]
                   [--length_deviation &lt;FLOAT&gt;] [-R &lt;FILE&gt;] [--gb_out &lt;FILE&gt;]
                   [--rec_check] [--gi_to_taxid &lt;FILE&gt;] [--cluster]
                   [--clust_match &lt;FLOAT&gt;] [--clust_cov &lt;INT&gt;]
                   [--blast_db &lt;PATH&gt;] [--blast_xml &lt;PATH&gt;]
                   [--min_ident &lt;FLOAT&gt;] [--min_ali_length &lt;FLOAT&gt;]
                   [--min_bit &lt;INT&gt;] [--refpkg &lt;DIR&gt;] [--jplace &lt;FILE&gt;]
                   [--kraken_db &lt;DIR&gt;] [--rm_kraken_db] [-o OUTPUT_PREFIX]
                   [--metadata METADATA] [--mock_meta_data]
                   [-@ &lt;email-address&gt;] [--version]

metaBEAT - metaBarcoding and Environmental DNA Analyses tool version:
v.0.97.4-global

optional arguments:
  -h, --help            show this help message and exit
  -Q &lt;FILE&gt;, --querylist &lt;FILE&gt;
                        file containing a list of query files
  -B &lt;FILE&gt;, --BIOM_input &lt;FILE&gt;
                        OTU table in BIOM format
  --g_queries &lt;FILE&gt;    fasta file containing query sequences (in combination
                        with &apos;-B&apos; sequence headers are expected to match)
  -v, --verbose         turn verbose output on
  -s, --seqinfo         write out seq_info.csv file
  -f, --fasta           write out ref.fasta file
  -p, --pplace          perform phylogenetic placement
  -k, --kraken          perform phylogenetic placement
  -t, --taxids          write out taxid.txt file
  -b, --blast           compile local blast db and blast queries
  -m &lt;string&gt;, --marker &lt;string&gt;
                        marker ID (default: marker)
  -n &lt;INT&gt;, --n_threads &lt;INT&gt;
                        Number of threads (default: 1)
  -E, --extract_centroid_reads
                        extract centroid reads to files
  -e, --extract_all_reads
                        extract reads to files
  --read_stats_off      ommit writing read stats to file
  --version             show program&apos;s version number and exit

Query preprocessing:
  The parameters in this group affect how the query sequences are processed

  --PCR_primer &lt;FILE&gt;   PCR primers (provided in fasta file) to be clipped
                        from reads
  --trim_adapter &lt;FILE&gt;
                        trim adapters provided in file
  --trim_qual &lt;INT&gt;     minimum phred quality score (default: 30)
  --phred &lt;INT&gt;         phred quality score offset - 33 or 64 (default: 33)
  --trim_window &lt;INT&gt;   sliding window size (default: 5) for trimming; if
                        average quality drops below the specified minimum
                        quality all subsequent bases are removed from the
                        reads
  --read_crop &lt;INT&gt;     Crop reads to this length if they are longer than that
                        (default: off)
  --trim_minlength &lt;INT&gt;
                        minimum length of reads to be retained after trimming
                        (default: 50)
  --merge               attempt to merge paired-end reads
  --product_length &lt;INT&gt;
                        estimated length of PCR product (specifying this
                        option increases merging efficiency)
  --merged_only         only process successfully merged read-pairs
  --forward_only        only process sequences that contain forward reads
                        (i.e. unmerged forward reads and merged reads)
  --length_filter &lt;INT&gt;
                        only process reads, which are within +/- 10 percent of
                        this length
  --length_deviation &lt;FLOAT&gt;
                        allowed deviation (in percent) from length specified
                        by --length_filter (default=0.1)

Reference:
  The parameters in this group affect the reference to be used in the
  analyses

  -R &lt;FILE&gt;, --REFlist &lt;FILE&gt;
                        file containing a list of files to be used as
                        reference sequences
  --gb_out &lt;FILE&gt;       output the corrected gb file
  --rec_check           check records to be used as reference
  --gi_to_taxid &lt;FILE&gt;  comma delimited file containing &apos;gi accession,taxid&apos;
                        for a list of taxa

Query clustering options:
  The parameters in this group affect read clustering

  --cluster             perform clustering of query sequences using vsearch
  --clust_match &lt;FLOAT&gt;
                        identity threshold for clustering in percent (default:
                        1)
  --clust_cov &lt;INT&gt;     minimum number of records in cluster (default: 1)

BLAST search:
  The parameters in this group affect BLAST search and BLAST based taxonomic
  assignment

  --blast_db &lt;PATH&gt;     path to precompiled blast database
  --blast_xml &lt;PATH&gt;    path to Blast result in xml format
  --min_ident &lt;FLOAT&gt;   minimum identity threshold in percent (default: 0.80)
  --min_ali_length &lt;FLOAT&gt;
                        minimum alignment length in percent of total query
                        length (default: 0.95)
  --min_bit &lt;INT&gt;       minimum bitscore (default: 80)

Phylogenetic placement:
  The parameters in this group affect phylogenetic placement

  --refpkg &lt;DIR&gt;        PATH to refpkg for pplacer
  --jplace &lt;FILE&gt;       phylogenetic placement result from prefious pplacer
                        run in *.jplace format

Kraken:
  The parameters in this group affect taxonomic assignment using Kraken

  --kraken_db &lt;DIR&gt;     PATH to a Kraken database
  --rm_kraken_db        Remove Kraken database after successful completion

BIOM OUTPUT:
  The arguments in this groups affect the output in BIOM format

  -o OUTPUT_PREFIX, --output_prefix OUTPUT_PREFIX
                        prefix for BIOM output files (default=&apos;metaBEAT&apos;)
  --metadata METADATA   comma delimited file containing metadata (optional)
  --mock_meta_data      add mock metadata to the samples in the BIOM output

Entrez identification:
  metaBEAT is querying the NCBI Entrez databases, please provide an email
  address for identification

  -@ &lt;email-address&gt;, --email &lt;email-address&gt;
                        provide your email address for identification to NCBI
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Now we run the trimming and clustering; note that there is no assignment method selected at this stage so metaBEAT will not attempt to make any assignments</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[6]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-Q Querymap_global.txt <span class="se">\</span>
--trim_qual 30 <span class="se">\</span>
--trim_minlength 90 <span class="se">\</span>
--merge <span class="se">\</span>
--product_length 350 <span class="se">\</span>
--merged_only <span class="se">\</span>
-R REFmap.txt <span class="se">\</span>
--cluster <span class="se">\</span>
--clust_match 1 <span class="se">\</span>
--clust_cov 5 <span class="se">\</span>
--min_ident 0.90 <span class="se">\</span>
-o MothPollenRestart <span class="se">\</span>
-@ callumjmacgregor@gmail.com
-n 5 -v &amp;&gt; log_restart
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>
metaBEAT - metaBarcoding and Environmental DNA Analyses tool
version: v.0.97.4-global


Wed Oct 19 16:43:52 2016

/usr/bin/metaBEAT_global.py -Q Querymap_global.txt --trim_qual 30 --trim_minlength 90 --merge --product_length 350 --merged_only -R REFmap.txt --cluster --clust_match 1 --clust_cov 5 --min_ident 0.90 -o MothPollenRestart -@ callumjmacgregor@gmail.com


metaBEAT may be querying NCBI&apos;s Entrez databases to fetch/verify taxonomic ids. Entrez User requirements state that you need to identify yourself by providing an email address so that NCBI can contact you in case there is a problem.

You have specified: &apos;callumjmacgregor@gmail.com&apos;

taxonomy.db found at /usr/bin/taxonomy.db

Parsing querylist file

Number of samples to process: 317
Sequence input format: defaultdict(&lt;type &apos;int&apos;&gt;, {&apos;fastq&apos;: 336})
Barcodes for demultiplexing provided for 336 samples
Cropping instructions provided for 336 samples


######## PROCESSING REFERENCE DATA ########


processing reference/Delosperma.gb (containing 58 records)

total number of valid records: 58


processing reference/Crassulaceae.gb (containing 330 records)

total number of valid records: 388


processing reference/yorks_flora.gb (containing 9405 records)

total number of valid records: 9793


processing reference/Dracaena.gb (containing 37 records)

total number of valid records: 9830


Wed Oct 19 16:43:59 2016


### DEMULTIPLEXING ###

assessing basic characteristics
data comes as paired end - ok
process_shortreads -1 /home/working/raw_data/Moth1_S1_L001_R1_001.fastq.gz -2 /home/working/raw_data/Moth1_S1_L001_R2_001.fastq.gz -i gzfastq -o . -b barcodes --inline_inline -y fastq
Using Phred+33 encoding for quality scores.
Reads trimmed shorter than 31 nucleotides will be discarded.
Found 1 paired input file(s).
Searching for single and paired-end, inlined barcodes.
Loaded 96 barcodes (8bp / 8bp).
Processing file 1 of 1 [Moth1_S1_L001_R1_001.fastq.gz]
  Reading data from
    /home/working/raw_data/Moth1_S1_L001_R1_001.fastq.gz and
    /home/working/raw_data/Moth1_S1_L001_R2_001.fastq.gz
  5976182 total reads; -1259396 ambiguous barcodes; +0 recovered; -0 low quality reads; 4716786 retained reads.
    0 trimmed reads; 0 orphaned paired-ends.
Closing files, flushing buffers...
Outputing details to log: &apos;./process_shortreads.log&apos;

5976182 total sequences;
  1259396 ambiguous barcode drops;
  0 low quality read drops;
  0 trimmed reads;
  0 orphaned paired-end reads;
4716786 retained reads.

assessing basic characteristics
data comes as paired end - ok
process_shortreads -1 /home/working/raw_data/Moth4_S4_L001_R1_001.fastq.gz -2 /home/working/raw_data/Moth4_S4_L001_R2_001.fastq.gz -i gzfastq -o . -b barcodes --inline_inline -y fastq
Using Phred+33 encoding for quality scores.
Reads trimmed shorter than 31 nucleotides will be discarded.
Found 1 paired input file(s).
Searching for single and paired-end, inlined barcodes.
Loaded 48 barcodes (8bp / 8bp).
Processing file 1 of 1 [Moth4_S4_L001_R1_001.fastq.gz]
  Reading data from
    /home/working/raw_data/Moth4_S4_L001_R1_001.fastq.gz and
    /home/working/raw_data/Moth4_S4_L001_R2_001.fastq.gz
  8464282 total reads; -1099580 ambiguous barcodes; +0 recovered; -0 low quality reads; 7364702 retained reads.
    0 trimmed reads; 0 orphaned paired-ends.
Closing files, flushing buffers...
Outputing details to log: &apos;./process_shortreads.log&apos;

8464282 total sequences;
  1099580 ambiguous barcode drops;
  0 low quality read drops;
  0 trimmed reads;
  0 orphaned paired-end reads;
7364702 retained reads.

assessing basic characteristics
data comes as paired end - ok
process_shortreads -1 /home/working/raw_data/Moth3_S3_L001_R1_001.fastq.gz -2 /home/working/raw_data/Moth3_S3_L001_R2_001.fastq.gz -i gzfastq -o . -b barcodes --inline_inline -y fastq
Using Phred+33 encoding for quality scores.
Reads trimmed shorter than 31 nucleotides will be discarded.
Found 1 paired input file(s).
Searching for single and paired-end, inlined barcodes.
Loaded 96 barcodes (8bp / 8bp).
Processing file 1 of 1 [Moth3_S3_L001_R1_001.fastq.gz]
  Reading data from
    /home/working/raw_data/Moth3_S3_L001_R1_001.fastq.gz and
    /home/working/raw_data/Moth3_S3_L001_R2_001.fastq.gz
  6757328 total reads; -1226048 ambiguous barcodes; +0 recovered; -0 low quality reads; 5531280 retained reads.
    0 trimmed reads; 0 orphaned paired-ends.
Closing files, flushing buffers...
Outputing details to log: &apos;./process_shortreads.log&apos;

6757328 total sequences;
  1226048 ambiguous barcode drops;
  0 low quality read drops;
  0 trimmed reads;
  0 orphaned paired-end reads;
5531280 retained reads.

assessing basic characteristics
data comes as paired end - ok
process_shortreads -1 /home/working/raw_data/Moth2_S2_L001_R1_001.fastq.gz -2 /home/working/raw_data/Moth2_S2_L001_R2_001.fastq.gz -i gzfastq -o . -b barcodes --inline_inline -y fastq
Using Phred+33 encoding for quality scores.
Reads trimmed shorter than 31 nucleotides will be discarded.
Found 1 paired input file(s).
Searching for single and paired-end, inlined barcodes.
Loaded 95 barcodes (8bp / 8bp).
Processing file 1 of 1 [Moth2_S2_L001_R1_001.fastq.gz]
  Reading data from
    /home/working/raw_data/Moth2_S2_L001_R1_001.fastq.gz and
    /home/working/raw_data/Moth2_S2_L001_R2_001.fastq.gz
  4193542 total reads; -826944 ambiguous barcodes; +0 recovered; -0 low quality reads; 3366598 retained reads.
    0 trimmed reads; 0 orphaned paired-ends.
Closing files, flushing buffers...
Outputing details to log: &apos;./process_shortreads.log&apos;

4193542 total sequences;
  826944 ambiguous barcode drops;
  0 low quality read drops;
  0 trimmed reads;
  0 orphaned paired-end reads;
3366598 retained reads.


##### processing query ID: Lg27 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg27_1.fastq /home/working/demultiplexed/Lg27_2.fastq Lg27_forward.paired.fastq.gz Lg27_forward.singletons.fastq.gz Lg27_reverse.paired.fastq.gz Lg27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg27_1.fastq /home/working/demultiplexed/Lg27_2.fastq Lg27_forward.paired.fastq.gz Lg27_forward.singletons.fastq.gz Lg27_reverse.paired.fastq.gz Lg27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:45:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/Lg27/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26514 Both Surviving: 6740 (25.42%) Forward Only Surviving: 3298 (12.44%) Reverse Only Surviving: 209 (0.79%) Dropped: 16267 (61.35%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: Lg27_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: Lg27_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: Lg27_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: Lg27_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg27_forward.paired.headclipped.fastq.gz Lg27_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg27 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg27_forward.paired.headclipped.fastq.gz
[FLASH]     Lg27_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg27.extendedFrags.fastq.gz
[FLASH]     ./Lg27.notCombined_1.fastq.gz
[FLASH]     ./Lg27.notCombined_2.fastq.gz
[FLASH]     ./Lg27.hist
[FLASH]     ./Lg27.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6740 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6740
[FLASH]     Combined pairs:   5348
[FLASH]     Uncombined pairs: 1392
[FLASH]     Percent combined: 79.35%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.147 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5348 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/Lg27/Lg27_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg27_centroids.fasta --uc Lg27.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5348 sequences and identified 2955 clusters (clustering threshold 1.00) - 72 clusters (representing 2021 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Lg37 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg37_1.fastq /home/working/demultiplexed/Lg37_2.fastq Lg37_forward.paired.fastq.gz Lg37_forward.singletons.fastq.gz Lg37_reverse.paired.fastq.gz Lg37_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg37_1.fastq /home/working/demultiplexed/Lg37_2.fastq Lg37_forward.paired.fastq.gz Lg37_forward.singletons.fastq.gz Lg37_reverse.paired.fastq.gz Lg37_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:45:53 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/Lg37/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31577 Both Surviving: 247 (0.78%) Forward Only Surviving: 489 (1.55%) Reverse Only Surviving: 197 (0.62%) Dropped: 30644 (97.05%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: Lg37_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: Lg37_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: Lg37_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: Lg37_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg37_forward.paired.headclipped.fastq.gz Lg37_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg37 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg37_forward.paired.headclipped.fastq.gz
[FLASH]     Lg37_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg37.extendedFrags.fastq.gz
[FLASH]     ./Lg37.notCombined_1.fastq.gz
[FLASH]     ./Lg37.notCombined_2.fastq.gz
[FLASH]     ./Lg37.hist
[FLASH]     ./Lg37.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 247 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      247
[FLASH]     Combined pairs:   28
[FLASH]     Uncombined pairs: 219
[FLASH]     Percent combined: 11.34%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

28 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/Lg37/Lg37_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg37_centroids.fasta --uc Lg37.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 28 sequences and identified 26 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Lg45 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg45_1.fastq /home/working/demultiplexed/Lg45_2.fastq Lg45_forward.paired.fastq.gz Lg45_forward.singletons.fastq.gz Lg45_reverse.paired.fastq.gz Lg45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg45_1.fastq /home/working/demultiplexed/Lg45_2.fastq Lg45_forward.paired.fastq.gz Lg45_forward.singletons.fastq.gz Lg45_reverse.paired.fastq.gz Lg45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:45:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/Lg45/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9421 Both Surviving: 2219 (23.55%) Forward Only Surviving: 590 (6.26%) Reverse Only Surviving: 112 (1.19%) Dropped: 6500 (68.99%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: Lg45_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: Lg45_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: Lg45_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: Lg45_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg45_forward.paired.headclipped.fastq.gz Lg45_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg45 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg45_forward.paired.headclipped.fastq.gz
[FLASH]     Lg45_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg45.extendedFrags.fastq.gz
[FLASH]     ./Lg45.notCombined_1.fastq.gz
[FLASH]     ./Lg45.notCombined_2.fastq.gz
[FLASH]     ./Lg45.hist
[FLASH]     ./Lg45.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2219 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2219
[FLASH]     Combined pairs:   2093
[FLASH]     Uncombined pairs: 126
[FLASH]     Percent combined: 94.32%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.059 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2093 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/Lg45/Lg45_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg45_centroids.fasta --uc Lg45.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2093 sequences and identified 1056 clusters (clustering threshold 1.00) - 21 clusters (representing 859 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Lg53 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg53_1.fastq /home/working/demultiplexed/Lg53_2.fastq Lg53_forward.paired.fastq.gz Lg53_forward.singletons.fastq.gz Lg53_reverse.paired.fastq.gz Lg53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg53_1.fastq /home/working/demultiplexed/Lg53_2.fastq Lg53_forward.paired.fastq.gz Lg53_forward.singletons.fastq.gz Lg53_reverse.paired.fastq.gz Lg53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:46:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/Lg53/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22402 Both Surviving: 5675 (25.33%) Forward Only Surviving: 1586 (7.08%) Reverse Only Surviving: 283 (1.26%) Dropped: 14858 (66.32%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: Lg53_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: Lg53_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: Lg53_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: Lg53_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg53_forward.paired.headclipped.fastq.gz Lg53_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg53 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg53_forward.paired.headclipped.fastq.gz
[FLASH]     Lg53_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg53.extendedFrags.fastq.gz
[FLASH]     ./Lg53.notCombined_1.fastq.gz
[FLASH]     ./Lg53.notCombined_2.fastq.gz
[FLASH]     ./Lg53.hist
[FLASH]     ./Lg53.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5675 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5675
[FLASH]     Combined pairs:   877
[FLASH]     Uncombined pairs: 4798
[FLASH]     Percent combined: 15.45%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.032 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

877 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/Lg53/Lg53_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg53_centroids.fasta --uc Lg53.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 877 sequences and identified 582 clusters (clustering threshold 1.00) - 15 clusters (representing 245 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Lg57 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg57_1.fastq /home/working/demultiplexed/Lg57_2.fastq Lg57_forward.paired.fastq.gz Lg57_forward.singletons.fastq.gz Lg57_reverse.paired.fastq.gz Lg57_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg57_1.fastq /home/working/demultiplexed/Lg57_2.fastq Lg57_forward.paired.fastq.gz Lg57_forward.singletons.fastq.gz Lg57_reverse.paired.fastq.gz Lg57_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:46:06 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/Lg57/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15465 Both Surviving: 4506 (29.14%) Forward Only Surviving: 1245 (8.05%) Reverse Only Surviving: 171 (1.11%) Dropped: 9543 (61.71%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: Lg57_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: Lg57_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: Lg57_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: Lg57_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg57_forward.paired.headclipped.fastq.gz Lg57_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg57 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg57_forward.paired.headclipped.fastq.gz
[FLASH]     Lg57_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg57.extendedFrags.fastq.gz
[FLASH]     ./Lg57.notCombined_1.fastq.gz
[FLASH]     ./Lg57.notCombined_2.fastq.gz
[FLASH]     ./Lg57.hist
[FLASH]     ./Lg57.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4506 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4506
[FLASH]     Combined pairs:   399
[FLASH]     Uncombined pairs: 4107
[FLASH]     Percent combined: 8.85%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.030 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

399 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/Lg57/Lg57_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg57_centroids.fasta --uc Lg57.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 399 sequences and identified 318 clusters (clustering threshold 1.00) - 7 clusters (representing 56 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Lg65 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg65_1.fastq /home/working/demultiplexed/Lg65_2.fastq Lg65_forward.paired.fastq.gz Lg65_forward.singletons.fastq.gz Lg65_reverse.paired.fastq.gz Lg65_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg65_1.fastq /home/working/demultiplexed/Lg65_2.fastq Lg65_forward.paired.fastq.gz Lg65_forward.singletons.fastq.gz Lg65_reverse.paired.fastq.gz Lg65_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:46:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/Lg65/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37917 Both Surviving: 804 (2.12%) Forward Only Surviving: 550 (1.45%) Reverse Only Surviving: 304 (0.80%) Dropped: 36259 (95.63%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: Lg65_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: Lg65_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: Lg65_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: Lg65_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg65_forward.paired.headclipped.fastq.gz Lg65_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg65 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg65_forward.paired.headclipped.fastq.gz
[FLASH]     Lg65_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg65.extendedFrags.fastq.gz
[FLASH]     ./Lg65.notCombined_1.fastq.gz
[FLASH]     ./Lg65.notCombined_2.fastq.gz
[FLASH]     ./Lg65.hist
[FLASH]     ./Lg65.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 804 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      804
[FLASH]     Combined pairs:   84
[FLASH]     Uncombined pairs: 720
[FLASH]     Percent combined: 10.45%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.006 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

84 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/Lg65/Lg65_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg65_centroids.fasta --uc Lg65.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 84 sequences and identified 61 clusters (clustering threshold 1.00) - 1 clusters (representing 22 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Lg9 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg9_1.fastq /home/working/demultiplexed/Lg9_2.fastq Lg9_forward.paired.fastq.gz Lg9_forward.singletons.fastq.gz Lg9_reverse.paired.fastq.gz Lg9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg9_1.fastq /home/working/demultiplexed/Lg9_2.fastq Lg9_forward.paired.fastq.gz Lg9_forward.singletons.fastq.gz Lg9_reverse.paired.fastq.gz Lg9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:46:13 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/Lg9/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 19281 Both Surviving: 90 (0.47%) Forward Only Surviving: 335 (1.74%) Reverse Only Surviving: 199 (1.03%) Dropped: 18657 (96.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: Lg9_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: Lg9_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: Lg9_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: Lg9_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg9_forward.paired.headclipped.fastq.gz Lg9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg9 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg9_forward.paired.headclipped.fastq.gz
[FLASH]     Lg9_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg9.extendedFrags.fastq.gz
[FLASH]     ./Lg9.notCombined_1.fastq.gz
[FLASH]     ./Lg9.notCombined_2.fastq.gz
[FLASH]     ./Lg9.hist
[FLASH]     ./Lg9.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 90 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      90
[FLASH]     Combined pairs:   67
[FLASH]     Uncombined pairs: 23
[FLASH]     Percent combined: 74.44%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

67 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/Lg9/Lg9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg9_centroids.fasta --uc Lg9.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 67 sequences and identified 52 clusters (clustering threshold 1.00) - 2 clusters (representing 13 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: NEG #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/NEG_1.fastq /home/working/demultiplexed/NEG_2.fastq NEG_forward.paired.fastq.gz NEG_forward.singletons.fastq.gz NEG_reverse.paired.fastq.gz NEG_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/NEG_1.fastq /home/working/demultiplexed/NEG_2.fastq NEG_forward.paired.fastq.gz NEG_forward.singletons.fastq.gz NEG_reverse.paired.fastq.gz NEG_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:46:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/NEG/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 3505 Both Surviving: 402 (11.47%) Forward Only Surviving: 167 (4.76%) Reverse Only Surviving: 54 (1.54%) Dropped: 2882 (82.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: NEG_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: NEG_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: NEG_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: NEG_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash NEG_forward.paired.headclipped.fastq.gz NEG_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o NEG -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     NEG_forward.paired.headclipped.fastq.gz
[FLASH]     NEG_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./NEG.extendedFrags.fastq.gz
[FLASH]     ./NEG.notCombined_1.fastq.gz
[FLASH]     ./NEG.notCombined_2.fastq.gz
[FLASH]     ./NEG.hist
[FLASH]     ./NEG.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 402 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      402
[FLASH]     Combined pairs:   139
[FLASH]     Uncombined pairs: 263
[FLASH]     Percent combined: 34.58%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.005 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

139 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/NEG/NEG_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids NEG_centroids.fasta --uc NEG.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 139 sequences and identified 115 clusters (clustering threshold 1.00) - 2 clusters (representing 15 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: O84 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/O84_1.fastq /home/working/demultiplexed/O84_2.fastq O84_forward.paired.fastq.gz O84_forward.singletons.fastq.gz O84_reverse.paired.fastq.gz O84_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/O84_1.fastq /home/working/demultiplexed/O84_2.fastq O84_forward.paired.fastq.gz O84_forward.singletons.fastq.gz O84_reverse.paired.fastq.gz O84_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:46:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/O84/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 36398 Both Surviving: 15690 (43.11%) Forward Only Surviving: 5948 (16.34%) Reverse Only Surviving: 326 (0.90%) Dropped: 14434 (39.66%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: O84_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: O84_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: O84_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: O84_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash O84_forward.paired.headclipped.fastq.gz O84_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o O84 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     O84_forward.paired.headclipped.fastq.gz
[FLASH]     O84_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./O84.extendedFrags.fastq.gz
[FLASH]     ./O84.notCombined_1.fastq.gz
[FLASH]     ./O84.notCombined_2.fastq.gz
[FLASH]     ./O84.hist
[FLASH]     ./O84.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 15690 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      15690
[FLASH]     Combined pairs:   13492
[FLASH]     Uncombined pairs: 2198
[FLASH]     Percent combined: 85.99%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.355 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

13492 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/O84/O84_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids O84_centroids.fasta --uc O84.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 13492 sequences and identified 4050 clusters (clustering threshold 1.00) - 312 clusters (representing 9371 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: O96 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/O96_1.fastq /home/working/demultiplexed/O96_2.fastq O96_forward.paired.fastq.gz O96_forward.singletons.fastq.gz O96_reverse.paired.fastq.gz O96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/O96_1.fastq /home/working/demultiplexed/O96_2.fastq O96_forward.paired.fastq.gz O96_forward.singletons.fastq.gz O96_reverse.paired.fastq.gz O96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:46:42 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/O96/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14802 Both Surviving: 5346 (36.12%) Forward Only Surviving: 1579 (10.67%) Reverse Only Surviving: 210 (1.42%) Dropped: 7667 (51.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: O96_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: O96_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: O96_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: O96_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash O96_forward.paired.headclipped.fastq.gz O96_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o O96 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     O96_forward.paired.headclipped.fastq.gz
[FLASH]     O96_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./O96.extendedFrags.fastq.gz
[FLASH]     ./O96.notCombined_1.fastq.gz
[FLASH]     ./O96.notCombined_2.fastq.gz
[FLASH]     ./O96.hist
[FLASH]     ./O96.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5346 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5346
[FLASH]     Combined pairs:   1914
[FLASH]     Uncombined pairs: 3432
[FLASH]     Percent combined: 35.80%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.058 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1914 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/O96/O96_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids O96_centroids.fasta --uc O96.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1914 sequences and identified 1228 clusters (clustering threshold 1.00) - 28 clusters (representing 583 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: P18 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P18_1.fastq /home/working/demultiplexed/P18_2.fastq P18_forward.paired.fastq.gz P18_forward.singletons.fastq.gz P18_reverse.paired.fastq.gz P18_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P18_1.fastq /home/working/demultiplexed/P18_2.fastq P18_forward.paired.fastq.gz P18_forward.singletons.fastq.gz P18_reverse.paired.fastq.gz P18_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:46:48 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/P18/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25495 Both Surviving: 5531 (21.69%) Forward Only Surviving: 1954 (7.66%) Reverse Only Surviving: 326 (1.28%) Dropped: 17684 (69.36%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: P18_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: P18_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: P18_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: P18_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash P18_forward.paired.headclipped.fastq.gz P18_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P18 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     P18_forward.paired.headclipped.fastq.gz
[FLASH]     P18_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./P18.extendedFrags.fastq.gz
[FLASH]     ./P18.notCombined_1.fastq.gz
[FLASH]     ./P18.notCombined_2.fastq.gz
[FLASH]     ./P18.hist
[FLASH]     ./P18.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5531 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5531
[FLASH]     Combined pairs:   3010
[FLASH]     Uncombined pairs: 2521
[FLASH]     Percent combined: 54.42%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.083 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3010 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/P18/P18_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P18_centroids.fasta --uc P18.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3010 sequences and identified 1116 clusters (clustering threshold 1.00) - 67 clusters (representing 1723 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: P54 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P54_1.fastq /home/working/demultiplexed/P54_2.fastq P54_forward.paired.fastq.gz P54_forward.singletons.fastq.gz P54_reverse.paired.fastq.gz P54_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P54_1.fastq /home/working/demultiplexed/P54_2.fastq P54_forward.paired.fastq.gz P54_forward.singletons.fastq.gz P54_reverse.paired.fastq.gz P54_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:46:57 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/P54/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 50485 Both Surviving: 15012 (29.74%) Forward Only Surviving: 3807 (7.54%) Reverse Only Surviving: 1539 (3.05%) Dropped: 30127 (59.68%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: P54_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: P54_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: P54_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: P54_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash P54_forward.paired.headclipped.fastq.gz P54_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P54 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     P54_forward.paired.headclipped.fastq.gz
[FLASH]     P54_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./P54.extendedFrags.fastq.gz
[FLASH]     ./P54.notCombined_1.fastq.gz
[FLASH]     ./P54.notCombined_2.fastq.gz
[FLASH]     ./P54.hist
[FLASH]     ./P54.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 15012 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      15012
[FLASH]     Combined pairs:   5359
[FLASH]     Uncombined pairs: 9653
[FLASH]     Percent combined: 35.70%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.172 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5359 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/P54/P54_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P54_centroids.fasta --uc P54.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5359 sequences and identified 2153 clusters (clustering threshold 1.00) - 86 clusters (representing 2811 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: P59 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P59_1.fastq /home/working/demultiplexed/P59_2.fastq P59_forward.paired.fastq.gz P59_forward.singletons.fastq.gz P59_reverse.paired.fastq.gz P59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P59_1.fastq /home/working/demultiplexed/P59_2.fastq P59_forward.paired.fastq.gz P59_forward.singletons.fastq.gz P59_reverse.paired.fastq.gz P59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:47:13 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/P59/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 10337 Both Surviving: 1710 (16.54%) Forward Only Surviving: 420 (4.06%) Reverse Only Surviving: 272 (2.63%) Dropped: 7935 (76.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: P59_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: P59_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: P59_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: P59_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash P59_forward.paired.headclipped.fastq.gz P59_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P59 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     P59_forward.paired.headclipped.fastq.gz
[FLASH]     P59_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./P59.extendedFrags.fastq.gz
[FLASH]     ./P59.notCombined_1.fastq.gz
[FLASH]     ./P59.notCombined_2.fastq.gz
[FLASH]     ./P59.hist
[FLASH]     ./P59.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1710 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1710
[FLASH]     Combined pairs:   29
[FLASH]     Uncombined pairs: 1681
[FLASH]     Percent combined: 1.70%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.013 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

29 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/P59/P59_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P59_centroids.fasta --uc P59.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 29 sequences and identified 28 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: P60 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P60_1.fastq /home/working/demultiplexed/P60_2.fastq P60_forward.paired.fastq.gz P60_forward.singletons.fastq.gz P60_reverse.paired.fastq.gz P60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P60_1.fastq /home/working/demultiplexed/P60_2.fastq P60_forward.paired.fastq.gz P60_forward.singletons.fastq.gz P60_reverse.paired.fastq.gz P60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:47:16 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/P60/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26955 Both Surviving: 10127 (37.57%) Forward Only Surviving: 3468 (12.87%) Reverse Only Surviving: 309 (1.15%) Dropped: 13051 (48.42%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: P60_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: P60_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: P60_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: P60_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash P60_forward.paired.headclipped.fastq.gz P60_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P60 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     P60_forward.paired.headclipped.fastq.gz
[FLASH]     P60_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./P60.extendedFrags.fastq.gz
[FLASH]     ./P60.notCombined_1.fastq.gz
[FLASH]     ./P60.notCombined_2.fastq.gz
[FLASH]     ./P60.hist
[FLASH]     ./P60.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10127 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10127
[FLASH]     Combined pairs:   5608
[FLASH]     Uncombined pairs: 4519
[FLASH]     Percent combined: 55.38%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.155 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5608 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/P60/P60_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P60_centroids.fasta --uc P60.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5608 sequences and identified 3054 clusters (clustering threshold 1.00) - 92 clusters (representing 2206 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: P73 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P73_1.fastq /home/working/demultiplexed/P73_2.fastq P73_forward.paired.fastq.gz P73_forward.singletons.fastq.gz P73_reverse.paired.fastq.gz P73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P73_1.fastq /home/working/demultiplexed/P73_2.fastq P73_forward.paired.fastq.gz P73_forward.singletons.fastq.gz P73_reverse.paired.fastq.gz P73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:47:28 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/P73/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: P73_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: P73_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: P73_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: P73_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash P73_forward.paired.headclipped.fastq.gz P73_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P73 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     P73_forward.paired.headclipped.fastq.gz
[FLASH]     P73_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./P73.extendedFrags.fastq.gz
[FLASH]     ./P73.notCombined_1.fastq.gz
[FLASH]     ./P73.notCombined_2.fastq.gz
[FLASH]     ./P73.hist
[FLASH]     ./P73.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/P73/P73_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P73_centroids.fasta --uc P73.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: P98 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P98_1.fastq /home/working/demultiplexed/P98_2.fastq P98_forward.paired.fastq.gz P98_forward.singletons.fastq.gz P98_reverse.paired.fastq.gz P98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P98_1.fastq /home/working/demultiplexed/P98_2.fastq P98_forward.paired.fastq.gz P98_forward.singletons.fastq.gz P98_reverse.paired.fastq.gz P98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:47:29 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/P98/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 4 Both Surviving: 1 (25.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 3 (75.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: P98_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: P98_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: P98_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: P98_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash P98_forward.paired.headclipped.fastq.gz P98_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P98 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     P98_forward.paired.headclipped.fastq.gz
[FLASH]     P98_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./P98.extendedFrags.fastq.gz
[FLASH]     ./P98.notCombined_1.fastq.gz
[FLASH]     ./P98.notCombined_2.fastq.gz
[FLASH]     ./P98.hist
[FLASH]     ./P98.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1
[FLASH]     Combined pairs:   1
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 100.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/P98/P98_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P98_centroids.fasta --uc P98.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1 sequences and identified 1 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS1 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS1_1.fastq /home/working/demultiplexed/POS1_2.fastq POS1_forward.paired.fastq.gz POS1_forward.singletons.fastq.gz POS1_reverse.paired.fastq.gz POS1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS1_1.fastq /home/working/demultiplexed/POS1_2.fastq POS1_forward.paired.fastq.gz POS1_forward.singletons.fastq.gz POS1_reverse.paired.fastq.gz POS1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:47:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/POS1/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28492 Both Surviving: 10615 (37.26%) Forward Only Surviving: 3913 (13.73%) Reverse Only Surviving: 296 (1.04%) Dropped: 13668 (47.97%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: POS1_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS1_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: POS1_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: POS1_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS1_forward.paired.headclipped.fastq.gz POS1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS1 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS1_forward.paired.headclipped.fastq.gz
[FLASH]     POS1_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS1.extendedFrags.fastq.gz
[FLASH]     ./POS1.notCombined_1.fastq.gz
[FLASH]     ./POS1.notCombined_2.fastq.gz
[FLASH]     ./POS1.hist
[FLASH]     ./POS1.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10615 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10615
[FLASH]     Combined pairs:   6159
[FLASH]     Uncombined pairs: 4456
[FLASH]     Percent combined: 58.02%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.171 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6159 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/POS1/POS1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS1_centroids.fasta --uc POS1.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6159 sequences and identified 4116 clusters (clustering threshold 1.00) - 88 clusters (representing 1672 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS2 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS2_1.fastq /home/working/demultiplexed/POS2_2.fastq POS2_forward.paired.fastq.gz POS2_forward.singletons.fastq.gz POS2_reverse.paired.fastq.gz POS2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS2_1.fastq /home/working/demultiplexed/POS2_2.fastq POS2_forward.paired.fastq.gz POS2_forward.singletons.fastq.gz POS2_reverse.paired.fastq.gz POS2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:47:44 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/POS2/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11207 Both Surviving: 1086 (9.69%) Forward Only Surviving: 317 (2.83%) Reverse Only Surviving: 109 (0.97%) Dropped: 9695 (86.51%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: POS2_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: POS2_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: POS2_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: POS2_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS2_forward.paired.headclipped.fastq.gz POS2_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS2 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS2_forward.paired.headclipped.fastq.gz
[FLASH]     POS2_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS2.extendedFrags.fastq.gz
[FLASH]     ./POS2.notCombined_1.fastq.gz
[FLASH]     ./POS2.notCombined_2.fastq.gz
[FLASH]     ./POS2.hist
[FLASH]     ./POS2.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1086 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1086
[FLASH]     Combined pairs:   356
[FLASH]     Uncombined pairs: 730
[FLASH]     Percent combined: 32.78%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.011 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

356 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/POS2/POS2_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS2_centroids.fasta --uc POS2.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 356 sequences and identified 221 clusters (clustering threshold 1.00) - 3 clusters (representing 121 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS3 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS3_1.fastq /home/working/demultiplexed/POS3_2.fastq POS3_forward.paired.fastq.gz POS3_forward.singletons.fastq.gz POS3_reverse.paired.fastq.gz POS3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS3_1.fastq /home/working/demultiplexed/POS3_2.fastq POS3_forward.paired.fastq.gz POS3_forward.singletons.fastq.gz POS3_reverse.paired.fastq.gz POS3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:47:46 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/POS3/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 10258 Both Surviving: 3469 (33.82%) Forward Only Surviving: 838 (8.17%) Reverse Only Surviving: 138 (1.35%) Dropped: 5813 (56.67%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: POS3_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS3_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: POS3_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS3_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS3_forward.paired.headclipped.fastq.gz POS3_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS3 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS3_forward.paired.headclipped.fastq.gz
[FLASH]     POS3_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS3.extendedFrags.fastq.gz
[FLASH]     ./POS3.notCombined_1.fastq.gz
[FLASH]     ./POS3.notCombined_2.fastq.gz
[FLASH]     ./POS3.hist
[FLASH]     ./POS3.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3469 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3469
[FLASH]     Combined pairs:   2854
[FLASH]     Uncombined pairs: 615
[FLASH]     Percent combined: 82.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.076 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2854 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/POS3/POS3_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS3_centroids.fasta --uc POS3.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2854 sequences and identified 1775 clusters (clustering threshold 1.00) - 27 clusters (representing 794 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS4 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS4_1.fastq /home/working/demultiplexed/POS4_2.fastq POS4_forward.paired.fastq.gz POS4_forward.singletons.fastq.gz POS4_reverse.paired.fastq.gz POS4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS4_1.fastq /home/working/demultiplexed/POS4_2.fastq POS4_forward.paired.fastq.gz POS4_forward.singletons.fastq.gz POS4_reverse.paired.fastq.gz POS4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:47:53 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/POS4/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11301 Both Surviving: 1999 (17.69%) Forward Only Surviving: 548 (4.85%) Reverse Only Surviving: 109 (0.96%) Dropped: 8645 (76.50%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: POS4_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: POS4_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: POS4_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: POS4_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS4_forward.paired.headclipped.fastq.gz POS4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS4 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS4_forward.paired.headclipped.fastq.gz
[FLASH]     POS4_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS4.extendedFrags.fastq.gz
[FLASH]     ./POS4.notCombined_1.fastq.gz
[FLASH]     ./POS4.notCombined_2.fastq.gz
[FLASH]     ./POS4.hist
[FLASH]     ./POS4.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1999 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1999
[FLASH]     Combined pairs:   1192
[FLASH]     Uncombined pairs: 807
[FLASH]     Percent combined: 59.63%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.032 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1192 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/POS4/POS4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS4_centroids.fasta --uc POS4.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1192 sequences and identified 786 clusters (clustering threshold 1.00) - 13 clusters (representing 296 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: PROC #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/PROC_1.fastq /home/working/demultiplexed/PROC_2.fastq PROC_forward.paired.fastq.gz PROC_forward.singletons.fastq.gz PROC_reverse.paired.fastq.gz PROC_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/PROC_1.fastq /home/working/demultiplexed/PROC_2.fastq PROC_forward.paired.fastq.gz PROC_forward.singletons.fastq.gz PROC_reverse.paired.fastq.gz PROC_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:47:57 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/PROC/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16777 Both Surviving: 3210 (19.13%) Forward Only Surviving: 860 (5.13%) Reverse Only Surviving: 167 (1.00%) Dropped: 12540 (74.75%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: PROC_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: PROC_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: PROC_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: PROC_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash PROC_forward.paired.headclipped.fastq.gz PROC_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o PROC -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     PROC_forward.paired.headclipped.fastq.gz
[FLASH]     PROC_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./PROC.extendedFrags.fastq.gz
[FLASH]     ./PROC.notCombined_1.fastq.gz
[FLASH]     ./PROC.notCombined_2.fastq.gz
[FLASH]     ./PROC.hist
[FLASH]     ./PROC.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3210 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3210
[FLASH]     Combined pairs:   155
[FLASH]     Uncombined pairs: 3055
[FLASH]     Percent combined: 4.83%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.021 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

155 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/PROC/PROC_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids PROC_centroids.fasta --uc PROC.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 155 sequences and identified 116 clusters (clustering threshold 1.00) - 4 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Q10 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q10_1.fastq /home/working/demultiplexed/Q10_2.fastq Q10_forward.paired.fastq.gz Q10_forward.singletons.fastq.gz Q10_reverse.paired.fastq.gz Q10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q10_1.fastq /home/working/demultiplexed/Q10_2.fastq Q10_forward.paired.fastq.gz Q10_forward.singletons.fastq.gz Q10_reverse.paired.fastq.gz Q10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:48:00 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/Q10/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23215 Both Surviving: 1090 (4.70%) Forward Only Surviving: 694 (2.99%) Reverse Only Surviving: 300 (1.29%) Dropped: 21131 (91.02%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: Q10_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: Q10_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: Q10_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: Q10_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Q10_forward.paired.headclipped.fastq.gz Q10_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q10 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Q10_forward.paired.headclipped.fastq.gz
[FLASH]     Q10_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Q10.extendedFrags.fastq.gz
[FLASH]     ./Q10.notCombined_1.fastq.gz
[FLASH]     ./Q10.notCombined_2.fastq.gz
[FLASH]     ./Q10.hist
[FLASH]     ./Q10.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1090 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1090
[FLASH]     Combined pairs:   251
[FLASH]     Uncombined pairs: 839
[FLASH]     Percent combined: 23.03%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.012 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

251 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/Q10/Q10_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q10_centroids.fasta --uc Q10.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 251 sequences and identified 203 clusters (clustering threshold 1.00) - 4 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Q30 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q30_1.fastq /home/working/demultiplexed/Q30_2.fastq Q30_forward.paired.fastq.gz Q30_forward.singletons.fastq.gz Q30_reverse.paired.fastq.gz Q30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q30_1.fastq /home/working/demultiplexed/Q30_2.fastq Q30_forward.paired.fastq.gz Q30_forward.singletons.fastq.gz Q30_reverse.paired.fastq.gz Q30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:48:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/Q30/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 2 Both Surviving: 2 (100.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 0 (0.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: Q30_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: Q30_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: Q30_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: Q30_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Q30_forward.paired.headclipped.fastq.gz Q30_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q30 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Q30_forward.paired.headclipped.fastq.gz
[FLASH]     Q30_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Q30.extendedFrags.fastq.gz
[FLASH]     ./Q30.notCombined_1.fastq.gz
[FLASH]     ./Q30.notCombined_2.fastq.gz
[FLASH]     ./Q30.hist
[FLASH]     ./Q30.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2
[FLASH]     Combined pairs:   2
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 100.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/Q30/Q30_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q30_centroids.fasta --uc Q30.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2 sequences and identified 2 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Q43 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q43_1.fastq /home/working/demultiplexed/Q43_2.fastq Q43_forward.paired.fastq.gz Q43_forward.singletons.fastq.gz Q43_reverse.paired.fastq.gz Q43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q43_1.fastq /home/working/demultiplexed/Q43_2.fastq Q43_forward.paired.fastq.gz Q43_forward.singletons.fastq.gz Q43_reverse.paired.fastq.gz Q43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:48:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/Q43/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 1 Both Surviving: 0 (0.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 1 (100.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: Q43_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: Q43_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: Q43_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: Q43_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Q43_forward.paired.headclipped.fastq.gz Q43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q43 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Q43_forward.paired.headclipped.fastq.gz
[FLASH]     Q43_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Q43.extendedFrags.fastq.gz
[FLASH]     ./Q43.notCombined_1.fastq.gz
[FLASH]     ./Q43.notCombined_2.fastq.gz
[FLASH]     ./Q43.hist
[FLASH]     ./Q43.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/Q43/Q43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q43_centroids.fasta --uc Q43.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Q64 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q64_1.fastq /home/working/demultiplexed/Q64_2.fastq Q64_forward.paired.fastq.gz Q64_forward.singletons.fastq.gz Q64_reverse.paired.fastq.gz Q64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q64_1.fastq /home/working/demultiplexed/Q64_2.fastq Q64_forward.paired.fastq.gz Q64_forward.singletons.fastq.gz Q64_reverse.paired.fastq.gz Q64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:48:03 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/Q64/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 2 Both Surviving: 1 (50.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 1 (50.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: Q64_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: Q64_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: Q64_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: Q64_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Q64_forward.paired.headclipped.fastq.gz Q64_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q64 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Q64_forward.paired.headclipped.fastq.gz
[FLASH]     Q64_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Q64.extendedFrags.fastq.gz
[FLASH]     ./Q64.notCombined_1.fastq.gz
[FLASH]     ./Q64.notCombined_2.fastq.gz
[FLASH]     ./Q64.hist
[FLASH]     ./Q64.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1
[FLASH]     Combined pairs:   1
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 100.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/Q64/Q64_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q64_centroids.fasta --uc Q64.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1 sequences and identified 1 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Q69 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q69_1.fastq /home/working/demultiplexed/Q69_2.fastq Q69_forward.paired.fastq.gz Q69_forward.singletons.fastq.gz Q69_reverse.paired.fastq.gz Q69_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q69_1.fastq /home/working/demultiplexed/Q69_2.fastq Q69_forward.paired.fastq.gz Q69_forward.singletons.fastq.gz Q69_reverse.paired.fastq.gz Q69_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:48:03 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/Q69/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 4825 Both Surviving: 51 (1.06%) Forward Only Surviving: 97 (2.01%) Reverse Only Surviving: 34 (0.70%) Dropped: 4643 (96.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: Q69_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: Q69_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: Q69_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: Q69_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Q69_forward.paired.headclipped.fastq.gz Q69_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q69 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Q69_forward.paired.headclipped.fastq.gz
[FLASH]     Q69_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Q69.extendedFrags.fastq.gz
[FLASH]     ./Q69.notCombined_1.fastq.gz
[FLASH]     ./Q69.notCombined_2.fastq.gz
[FLASH]     ./Q69.hist
[FLASH]     ./Q69.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 51 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      51
[FLASH]     Combined pairs:   21
[FLASH]     Uncombined pairs: 30
[FLASH]     Percent combined: 41.18%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

21 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/Q69/Q69_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q69_centroids.fasta --uc Q69.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 21 sequences and identified 20 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Q83 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q83_1.fastq /home/working/demultiplexed/Q83_2.fastq Q83_forward.paired.fastq.gz Q83_forward.singletons.fastq.gz Q83_reverse.paired.fastq.gz Q83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q83_1.fastq /home/working/demultiplexed/Q83_2.fastq Q83_forward.paired.fastq.gz Q83_forward.singletons.fastq.gz Q83_reverse.paired.fastq.gz Q83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:48:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/Q83/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29958 Both Surviving: 5419 (18.09%) Forward Only Surviving: 1242 (4.15%) Reverse Only Surviving: 466 (1.56%) Dropped: 22831 (76.21%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: Q83_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: Q83_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: Q83_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: Q83_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Q83_forward.paired.headclipped.fastq.gz Q83_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q83 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Q83_forward.paired.headclipped.fastq.gz
[FLASH]     Q83_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Q83.extendedFrags.fastq.gz
[FLASH]     ./Q83.notCombined_1.fastq.gz
[FLASH]     ./Q83.notCombined_2.fastq.gz
[FLASH]     ./Q83.hist
[FLASH]     ./Q83.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5419 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5419
[FLASH]     Combined pairs:   1324
[FLASH]     Uncombined pairs: 4095
[FLASH]     Percent combined: 24.43%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.054 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1324 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/Q83/Q83_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q83_centroids.fasta --uc Q83.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1324 sequences and identified 822 clusters (clustering threshold 1.00) - 17 clusters (representing 405 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R1 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R1_1.fastq /home/working/demultiplexed/R1_2.fastq R1_forward.paired.fastq.gz R1_forward.singletons.fastq.gz R1_reverse.paired.fastq.gz R1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R1_1.fastq /home/working/demultiplexed/R1_2.fastq R1_forward.paired.fastq.gz R1_forward.singletons.fastq.gz R1_reverse.paired.fastq.gz R1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:48:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R1/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16656 Both Surviving: 5127 (30.78%) Forward Only Surviving: 1237 (7.43%) Reverse Only Surviving: 238 (1.43%) Dropped: 10054 (60.36%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: R1_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: R1_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: R1_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: R1_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R1_forward.paired.headclipped.fastq.gz R1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R1 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R1_forward.paired.headclipped.fastq.gz
[FLASH]     R1_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R1.extendedFrags.fastq.gz
[FLASH]     ./R1.notCombined_1.fastq.gz
[FLASH]     ./R1.notCombined_2.fastq.gz
[FLASH]     ./R1.hist
[FLASH]     ./R1.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5127 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5127
[FLASH]     Combined pairs:   2511
[FLASH]     Uncombined pairs: 2616
[FLASH]     Percent combined: 48.98%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.070 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2511 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R1/R1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R1_centroids.fasta --uc R1.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2511 sequences and identified 872 clusters (clustering threshold 1.00) - 44 clusters (representing 1438 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R10 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R10_1.fastq /home/working/demultiplexed/R10_2.fastq R10_forward.paired.fastq.gz R10_forward.singletons.fastq.gz R10_reverse.paired.fastq.gz R10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R10_1.fastq /home/working/demultiplexed/R10_2.fastq R10_forward.paired.fastq.gz R10_forward.singletons.fastq.gz R10_reverse.paired.fastq.gz R10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:48:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R10/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: R10_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: R10_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R10_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R10_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R10_forward.paired.headclipped.fastq.gz R10_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R10 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R10_forward.paired.headclipped.fastq.gz
[FLASH]     R10_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R10.extendedFrags.fastq.gz
[FLASH]     ./R10.notCombined_1.fastq.gz
[FLASH]     ./R10.notCombined_2.fastq.gz
[FLASH]     ./R10.hist
[FLASH]     ./R10.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R10/R10_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R10_centroids.fasta --uc R10.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R102 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R102_1.fastq /home/working/demultiplexed/R102_2.fastq R102_forward.paired.fastq.gz R102_forward.singletons.fastq.gz R102_reverse.paired.fastq.gz R102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R102_1.fastq /home/working/demultiplexed/R102_2.fastq R102_forward.paired.fastq.gz R102_forward.singletons.fastq.gz R102_reverse.paired.fastq.gz R102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:48:21 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R102/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 62773 Both Surviving: 13515 (21.53%) Forward Only Surviving: 3949 (6.29%) Reverse Only Surviving: 1852 (2.95%) Dropped: 43457 (69.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: R102_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: R102_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R102_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R102_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R102_forward.paired.headclipped.fastq.gz R102_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R102 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R102_forward.paired.headclipped.fastq.gz
[FLASH]     R102_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R102.extendedFrags.fastq.gz
[FLASH]     ./R102.notCombined_1.fastq.gz
[FLASH]     ./R102.notCombined_2.fastq.gz
[FLASH]     ./R102.hist
[FLASH]     ./R102.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 13515 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      13515
[FLASH]     Combined pairs:   765
[FLASH]     Uncombined pairs: 12750
[FLASH]     Percent combined: 5.66%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.092 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

765 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R102/R102_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R102_centroids.fasta --uc R102.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 765 sequences and identified 582 clusters (clustering threshold 1.00) - 14 clusters (representing 145 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R13 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R13_1.fastq /home/working/demultiplexed/R13_2.fastq R13_forward.paired.fastq.gz R13_forward.singletons.fastq.gz R13_reverse.paired.fastq.gz R13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R13_1.fastq /home/working/demultiplexed/R13_2.fastq R13_forward.paired.fastq.gz R13_forward.singletons.fastq.gz R13_reverse.paired.fastq.gz R13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:48:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R13/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 19033 Both Surviving: 4660 (24.48%) Forward Only Surviving: 1222 (6.42%) Reverse Only Surviving: 224 (1.18%) Dropped: 12927 (67.92%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: R13_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: R13_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R13_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R13_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R13_forward.paired.headclipped.fastq.gz R13_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R13 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R13_forward.paired.headclipped.fastq.gz
[FLASH]     R13_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R13.extendedFrags.fastq.gz
[FLASH]     ./R13.notCombined_1.fastq.gz
[FLASH]     ./R13.notCombined_2.fastq.gz
[FLASH]     ./R13.hist
[FLASH]     ./R13.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4660 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4660
[FLASH]     Combined pairs:   629
[FLASH]     Uncombined pairs: 4031
[FLASH]     Percent combined: 13.50%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.036 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

629 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R13/R13_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R13_centroids.fasta --uc R13.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 629 sequences and identified 406 clusters (clustering threshold 1.00) - 12 clusters (representing 203 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R15 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R15_1.fastq /home/working/demultiplexed/R15_2.fastq R15_forward.paired.fastq.gz R15_forward.singletons.fastq.gz R15_reverse.paired.fastq.gz R15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R15_1.fastq /home/working/demultiplexed/R15_2.fastq R15_forward.paired.fastq.gz R15_forward.singletons.fastq.gz R15_reverse.paired.fastq.gz R15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:48:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R15/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 65631 Both Surviving: 3298 (5.03%) Forward Only Surviving: 2188 (3.33%) Reverse Only Surviving: 804 (1.23%) Dropped: 59341 (90.42%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: R15_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: R15_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: R15_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: R15_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R15_forward.paired.headclipped.fastq.gz R15_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R15 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R15_forward.paired.headclipped.fastq.gz
[FLASH]     R15_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R15.extendedFrags.fastq.gz
[FLASH]     ./R15.notCombined_1.fastq.gz
[FLASH]     ./R15.notCombined_2.fastq.gz
[FLASH]     ./R15.hist
[FLASH]     ./R15.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3298 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3298
[FLASH]     Combined pairs:   2086
[FLASH]     Uncombined pairs: 1212
[FLASH]     Percent combined: 63.25%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.061 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2086 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R15/R15_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R15_centroids.fasta --uc R15.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2086 sequences and identified 1204 clusters (clustering threshold 1.00) - 32 clusters (representing 807 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R16 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R16_1.fastq /home/working/demultiplexed/R16_2.fastq R16_forward.paired.fastq.gz R16_forward.singletons.fastq.gz R16_reverse.paired.fastq.gz R16_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R16_1.fastq /home/working/demultiplexed/R16_2.fastq R16_forward.paired.fastq.gz R16_forward.singletons.fastq.gz R16_reverse.paired.fastq.gz R16_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:48:45 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R16/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25112 Both Surviving: 8135 (32.39%) Forward Only Surviving: 3246 (12.93%) Reverse Only Surviving: 283 (1.13%) Dropped: 13448 (53.55%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R16_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R16_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: R16_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: R16_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R16_forward.paired.headclipped.fastq.gz R16_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R16 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R16_forward.paired.headclipped.fastq.gz
[FLASH]     R16_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R16.extendedFrags.fastq.gz
[FLASH]     ./R16.notCombined_1.fastq.gz
[FLASH]     ./R16.notCombined_2.fastq.gz
[FLASH]     ./R16.hist
[FLASH]     ./R16.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8135 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8135
[FLASH]     Combined pairs:   4002
[FLASH]     Uncombined pairs: 4133
[FLASH]     Percent combined: 49.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.114 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4002 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R16/R16_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R16_centroids.fasta --uc R16.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4002 sequences and identified 2239 clusters (clustering threshold 1.00) - 59 clusters (representing 1497 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R20 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R20_1.fastq /home/working/demultiplexed/R20_2.fastq R20_forward.paired.fastq.gz R20_forward.singletons.fastq.gz R20_reverse.paired.fastq.gz R20_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R20_1.fastq /home/working/demultiplexed/R20_2.fastq R20_forward.paired.fastq.gz R20_forward.singletons.fastq.gz R20_reverse.paired.fastq.gz R20_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:48:56 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R20/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41196 Both Surviving: 10311 (25.03%) Forward Only Surviving: 6801 (16.51%) Reverse Only Surviving: 389 (0.94%) Dropped: 23695 (57.52%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R20_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R20_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: R20_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: R20_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R20_forward.paired.headclipped.fastq.gz R20_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R20 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R20_forward.paired.headclipped.fastq.gz
[FLASH]     R20_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R20.extendedFrags.fastq.gz
[FLASH]     ./R20.notCombined_1.fastq.gz
[FLASH]     ./R20.notCombined_2.fastq.gz
[FLASH]     ./R20.hist
[FLASH]     ./R20.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10311 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10311
[FLASH]     Combined pairs:   8629
[FLASH]     Uncombined pairs: 1682
[FLASH]     Percent combined: 83.69%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.230 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

8629 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R20/R20_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R20_centroids.fasta --uc R20.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 8629 sequences and identified 3449 clusters (clustering threshold 1.00) - 186 clusters (representing 4843 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R22 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R22_1.fastq /home/working/demultiplexed/R22_2.fastq R22_forward.paired.fastq.gz R22_forward.singletons.fastq.gz R22_reverse.paired.fastq.gz R22_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R22_1.fastq /home/working/demultiplexed/R22_2.fastq R22_forward.paired.fastq.gz R22_forward.singletons.fastq.gz R22_reverse.paired.fastq.gz R22_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:49:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R22/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 40727 Both Surviving: 5266 (12.93%) Forward Only Surviving: 1531 (3.76%) Reverse Only Surviving: 618 (1.52%) Dropped: 33312 (81.79%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R22_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R22_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: R22_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: R22_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R22_forward.paired.headclipped.fastq.gz R22_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R22 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R22_forward.paired.headclipped.fastq.gz
[FLASH]     R22_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R22.extendedFrags.fastq.gz
[FLASH]     ./R22.notCombined_1.fastq.gz
[FLASH]     ./R22.notCombined_2.fastq.gz
[FLASH]     ./R22.hist
[FLASH]     ./R22.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5266 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5266
[FLASH]     Combined pairs:   1400
[FLASH]     Uncombined pairs: 3866
[FLASH]     Percent combined: 26.59%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.047 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1400 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R22/R22_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R22_centroids.fasta --uc R22.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1400 sequences and identified 870 clusters (clustering threshold 1.00) - 21 clusters (representing 458 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R23 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R23_1.fastq /home/working/demultiplexed/R23_2.fastq R23_forward.paired.fastq.gz R23_forward.singletons.fastq.gz R23_reverse.paired.fastq.gz R23_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R23_1.fastq /home/working/demultiplexed/R23_2.fastq R23_forward.paired.fastq.gz R23_forward.singletons.fastq.gz R23_reverse.paired.fastq.gz R23_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:49:21 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R23/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 27238 Both Surviving: 5046 (18.53%) Forward Only Surviving: 994 (3.65%) Reverse Only Surviving: 810 (2.97%) Dropped: 20388 (74.85%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: R23_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: R23_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R23_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R23_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R23_forward.paired.headclipped.fastq.gz R23_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R23 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R23_forward.paired.headclipped.fastq.gz
[FLASH]     R23_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R23.extendedFrags.fastq.gz
[FLASH]     ./R23.notCombined_1.fastq.gz
[FLASH]     ./R23.notCombined_2.fastq.gz
[FLASH]     ./R23.hist
[FLASH]     ./R23.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5046 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5046
[FLASH]     Combined pairs:   642
[FLASH]     Uncombined pairs: 4404
[FLASH]     Percent combined: 12.72%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.036 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

642 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R23/R23_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R23_centroids.fasta --uc R23.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 642 sequences and identified 364 clusters (clustering threshold 1.00) - 13 clusters (representing 256 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R27 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R27_1.fastq /home/working/demultiplexed/R27_2.fastq R27_forward.paired.fastq.gz R27_forward.singletons.fastq.gz R27_reverse.paired.fastq.gz R27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R27_1.fastq /home/working/demultiplexed/R27_2.fastq R27_forward.paired.fastq.gz R27_forward.singletons.fastq.gz R27_reverse.paired.fastq.gz R27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:49:28 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R27/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41488 Both Surviving: 9286 (22.38%) Forward Only Surviving: 3270 (7.88%) Reverse Only Surviving: 522 (1.26%) Dropped: 28410 (68.48%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: R27_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: R27_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R27_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R27_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R27_forward.paired.headclipped.fastq.gz R27_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R27 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R27_forward.paired.headclipped.fastq.gz
[FLASH]     R27_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R27.extendedFrags.fastq.gz
[FLASH]     ./R27.notCombined_1.fastq.gz
[FLASH]     ./R27.notCombined_2.fastq.gz
[FLASH]     ./R27.hist
[FLASH]     ./R27.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9286 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9286
[FLASH]     Combined pairs:   7926
[FLASH]     Uncombined pairs: 1360
[FLASH]     Percent combined: 85.35%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.209 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

7926 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R27/R27_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R27_centroids.fasta --uc R27.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 7926 sequences and identified 2335 clusters (clustering threshold 1.00) - 172 clusters (representing 5330 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R29 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R29_1.fastq /home/working/demultiplexed/R29_2.fastq R29_forward.paired.fastq.gz R29_forward.singletons.fastq.gz R29_reverse.paired.fastq.gz R29_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R29_1.fastq /home/working/demultiplexed/R29_2.fastq R29_forward.paired.fastq.gz R29_forward.singletons.fastq.gz R29_reverse.paired.fastq.gz R29_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:49:43 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R29/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 33797 Both Surviving: 8652 (25.60%) Forward Only Surviving: 2104 (6.23%) Reverse Only Surviving: 517 (1.53%) Dropped: 22524 (66.64%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: R29_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: R29_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: R29_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: R29_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R29_forward.paired.headclipped.fastq.gz R29_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R29 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R29_forward.paired.headclipped.fastq.gz
[FLASH]     R29_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R29.extendedFrags.fastq.gz
[FLASH]     ./R29.notCombined_1.fastq.gz
[FLASH]     ./R29.notCombined_2.fastq.gz
[FLASH]     ./R29.hist
[FLASH]     ./R29.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8652 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8652
[FLASH]     Combined pairs:   2616
[FLASH]     Uncombined pairs: 6036
[FLASH]     Percent combined: 30.24%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.085 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2616 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R29/R29_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R29_centroids.fasta --uc R29.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2616 sequences and identified 1564 clusters (clustering threshold 1.00) - 49 clusters (representing 816 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R30 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R30_1.fastq /home/working/demultiplexed/R30_2.fastq R30_forward.paired.fastq.gz R30_forward.singletons.fastq.gz R30_reverse.paired.fastq.gz R30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R30_1.fastq /home/working/demultiplexed/R30_2.fastq R30_forward.paired.fastq.gz R30_forward.singletons.fastq.gz R30_reverse.paired.fastq.gz R30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:49:52 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R30/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16649 Both Surviving: 2332 (14.01%) Forward Only Surviving: 703 (4.22%) Reverse Only Surviving: 184 (1.11%) Dropped: 13430 (80.67%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: R30_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: R30_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: R30_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: R30_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R30_forward.paired.headclipped.fastq.gz R30_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R30 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R30_forward.paired.headclipped.fastq.gz
[FLASH]     R30_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R30.extendedFrags.fastq.gz
[FLASH]     ./R30.notCombined_1.fastq.gz
[FLASH]     ./R30.notCombined_2.fastq.gz
[FLASH]     ./R30.hist
[FLASH]     ./R30.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2332 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2332
[FLASH]     Combined pairs:   531
[FLASH]     Uncombined pairs: 1801
[FLASH]     Percent combined: 22.77%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.029 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

531 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R30/R30_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R30_centroids.fasta --uc R30.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 531 sequences and identified 277 clusters (clustering threshold 1.00) - 5 clusters (representing 225 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R31 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R31_1.fastq /home/working/demultiplexed/R31_2.fastq R31_forward.paired.fastq.gz R31_forward.singletons.fastq.gz R31_reverse.paired.fastq.gz R31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R31_1.fastq /home/working/demultiplexed/R31_2.fastq R31_forward.paired.fastq.gz R31_forward.singletons.fastq.gz R31_reverse.paired.fastq.gz R31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:49:56 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R31/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26755 Both Surviving: 5140 (19.21%) Forward Only Surviving: 1230 (4.60%) Reverse Only Surviving: 349 (1.30%) Dropped: 20036 (74.89%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: R31_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: R31_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R31_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R31_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R31_forward.paired.headclipped.fastq.gz R31_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R31 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R31_forward.paired.headclipped.fastq.gz
[FLASH]     R31_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R31.extendedFrags.fastq.gz
[FLASH]     ./R31.notCombined_1.fastq.gz
[FLASH]     ./R31.notCombined_2.fastq.gz
[FLASH]     ./R31.hist
[FLASH]     ./R31.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5140 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5140
[FLASH]     Combined pairs:   592
[FLASH]     Uncombined pairs: 4548
[FLASH]     Percent combined: 11.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.044 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

592 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R31/R31_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R31_centroids.fasta --uc R31.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 592 sequences and identified 414 clusters (clustering threshold 1.00) - 10 clusters (representing 132 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R32 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R32_1.fastq /home/working/demultiplexed/R32_2.fastq R32_forward.paired.fastq.gz R32_forward.singletons.fastq.gz R32_reverse.paired.fastq.gz R32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R32_1.fastq /home/working/demultiplexed/R32_2.fastq R32_forward.paired.fastq.gz R32_forward.singletons.fastq.gz R32_reverse.paired.fastq.gz R32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:50:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R32/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 32624 Both Surviving: 1639 (5.02%) Forward Only Surviving: 1353 (4.15%) Reverse Only Surviving: 415 (1.27%) Dropped: 29217 (89.56%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: R32_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: R32_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R32_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R32_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R32_forward.paired.headclipped.fastq.gz R32_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R32 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R32_forward.paired.headclipped.fastq.gz
[FLASH]     R32_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R32.extendedFrags.fastq.gz
[FLASH]     ./R32.notCombined_1.fastq.gz
[FLASH]     ./R32.notCombined_2.fastq.gz
[FLASH]     ./R32.hist
[FLASH]     ./R32.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1639 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1639
[FLASH]     Combined pairs:   328
[FLASH]     Uncombined pairs: 1311
[FLASH]     Percent combined: 20.01%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.024 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

328 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R32/R32_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R32_centroids.fasta --uc R32.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 328 sequences and identified 263 clusters (clustering threshold 1.00) - 5 clusters (representing 44 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R34 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R34_1.fastq /home/working/demultiplexed/R34_2.fastq R34_forward.paired.fastq.gz R34_forward.singletons.fastq.gz R34_reverse.paired.fastq.gz R34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R34_1.fastq /home/working/demultiplexed/R34_2.fastq R34_forward.paired.fastq.gz R34_forward.singletons.fastq.gz R34_reverse.paired.fastq.gz R34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:50:06 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R34/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9121 Both Surviving: 2265 (24.83%) Forward Only Surviving: 636 (6.97%) Reverse Only Surviving: 93 (1.02%) Dropped: 6127 (67.17%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: R34_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: R34_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: R34_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: R34_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R34_forward.paired.headclipped.fastq.gz R34_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R34 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R34_forward.paired.headclipped.fastq.gz
[FLASH]     R34_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R34.extendedFrags.fastq.gz
[FLASH]     ./R34.notCombined_1.fastq.gz
[FLASH]     ./R34.notCombined_2.fastq.gz
[FLASH]     ./R34.hist
[FLASH]     ./R34.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2265 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2265
[FLASH]     Combined pairs:   74
[FLASH]     Uncombined pairs: 2191
[FLASH]     Percent combined: 3.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.022 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

74 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R34/R34_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R34_centroids.fasta --uc R34.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 74 sequences and identified 66 clusters (clustering threshold 1.00) - 1 clusters (representing 7 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R36 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R36_1.fastq /home/working/demultiplexed/R36_2.fastq R36_forward.paired.fastq.gz R36_forward.singletons.fastq.gz R36_reverse.paired.fastq.gz R36_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R36_1.fastq /home/working/demultiplexed/R36_2.fastq R36_forward.paired.fastq.gz R36_forward.singletons.fastq.gz R36_reverse.paired.fastq.gz R36_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:50:08 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R36/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 12398 Both Surviving: 2888 (23.29%) Forward Only Surviving: 1053 (8.49%) Reverse Only Surviving: 127 (1.02%) Dropped: 8330 (67.19%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: R36_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: R36_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R36_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R36_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R36_forward.paired.headclipped.fastq.gz R36_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R36 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R36_forward.paired.headclipped.fastq.gz
[FLASH]     R36_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R36.extendedFrags.fastq.gz
[FLASH]     ./R36.notCombined_1.fastq.gz
[FLASH]     ./R36.notCombined_2.fastq.gz
[FLASH]     ./R36.hist
[FLASH]     ./R36.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2888 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2888
[FLASH]     Combined pairs:   2562
[FLASH]     Uncombined pairs: 326
[FLASH]     Percent combined: 88.71%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.074 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2562 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R36/R36_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R36_centroids.fasta --uc R36.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2562 sequences and identified 1528 clusters (clustering threshold 1.00) - 16 clusters (representing 849 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R39 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R39_1.fastq /home/working/demultiplexed/R39_2.fastq R39_forward.paired.fastq.gz R39_forward.singletons.fastq.gz R39_reverse.paired.fastq.gz R39_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R39_1.fastq /home/working/demultiplexed/R39_2.fastq R39_forward.paired.fastq.gz R39_forward.singletons.fastq.gz R39_reverse.paired.fastq.gz R39_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:50:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R39/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 30931 Both Surviving: 8970 (29.00%) Forward Only Surviving: 3918 (12.67%) Reverse Only Surviving: 373 (1.21%) Dropped: 17670 (57.13%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: R39_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: R39_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: R39_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: R39_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R39_forward.paired.headclipped.fastq.gz R39_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R39 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R39_forward.paired.headclipped.fastq.gz
[FLASH]     R39_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R39.extendedFrags.fastq.gz
[FLASH]     ./R39.notCombined_1.fastq.gz
[FLASH]     ./R39.notCombined_2.fastq.gz
[FLASH]     ./R39.hist
[FLASH]     ./R39.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8970 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8970
[FLASH]     Combined pairs:   3486
[FLASH]     Uncombined pairs: 5484
[FLASH]     Percent combined: 38.86%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.116 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3486 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R39/R39_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R39_centroids.fasta --uc R39.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3486 sequences and identified 1707 clusters (clustering threshold 1.00) - 62 clusters (representing 1535 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R4 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R4_1.fastq /home/working/demultiplexed/R4_2.fastq R4_forward.paired.fastq.gz R4_forward.singletons.fastq.gz R4_reverse.paired.fastq.gz R4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R4_1.fastq /home/working/demultiplexed/R4_2.fastq R4_forward.paired.fastq.gz R4_forward.singletons.fastq.gz R4_reverse.paired.fastq.gz R4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:50:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R4/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 12106 Both Surviving: 3362 (27.77%) Forward Only Surviving: 786 (6.49%) Reverse Only Surviving: 225 (1.86%) Dropped: 7733 (63.88%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: R4_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: R4_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R4_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R4_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R4_forward.paired.headclipped.fastq.gz R4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R4 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R4_forward.paired.headclipped.fastq.gz
[FLASH]     R4_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R4.extendedFrags.fastq.gz
[FLASH]     ./R4.notCombined_1.fastq.gz
[FLASH]     ./R4.notCombined_2.fastq.gz
[FLASH]     ./R4.hist
[FLASH]     ./R4.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3362 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3362
[FLASH]     Combined pairs:   1701
[FLASH]     Uncombined pairs: 1661
[FLASH]     Percent combined: 50.59%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.071 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1701 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R4/R4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R4_centroids.fasta --uc R4.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1701 sequences and identified 1031 clusters (clustering threshold 1.00) - 31 clusters (representing 573 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R40 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R40_1.fastq /home/working/demultiplexed/R40_2.fastq R40_forward.paired.fastq.gz R40_forward.singletons.fastq.gz R40_reverse.paired.fastq.gz R40_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R40_1.fastq /home/working/demultiplexed/R40_2.fastq R40_forward.paired.fastq.gz R40_forward.singletons.fastq.gz R40_reverse.paired.fastq.gz R40_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:50:31 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R40/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 10491 Both Surviving: 1666 (15.88%) Forward Only Surviving: 412 (3.93%) Reverse Only Surviving: 153 (1.46%) Dropped: 8260 (78.73%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: R40_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: R40_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R40_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R40_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R40_forward.paired.headclipped.fastq.gz R40_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R40 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R40_forward.paired.headclipped.fastq.gz
[FLASH]     R40_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R40.extendedFrags.fastq.gz
[FLASH]     ./R40.notCombined_1.fastq.gz
[FLASH]     ./R40.notCombined_2.fastq.gz
[FLASH]     ./R40.hist
[FLASH]     ./R40.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1666 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1666
[FLASH]     Combined pairs:   349
[FLASH]     Uncombined pairs: 1317
[FLASH]     Percent combined: 20.95%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.027 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

349 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R40/R40_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R40_centroids.fasta --uc R40.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 349 sequences and identified 246 clusters (clustering threshold 1.00) - 3 clusters (representing 91 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R43 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R43_1.fastq /home/working/demultiplexed/R43_2.fastq R43_forward.paired.fastq.gz R43_forward.singletons.fastq.gz R43_reverse.paired.fastq.gz R43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R43_1.fastq /home/working/demultiplexed/R43_2.fastq R43_forward.paired.fastq.gz R43_forward.singletons.fastq.gz R43_reverse.paired.fastq.gz R43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:50:39 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R43/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 102059 Both Surviving: 26955 (26.41%) Forward Only Surviving: 7384 (7.24%) Reverse Only Surviving: 1624 (1.59%) Dropped: 66096 (64.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: R43_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: R43_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: R43_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: R43_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R43_forward.paired.headclipped.fastq.gz R43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R43 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R43_forward.paired.headclipped.fastq.gz
[FLASH]     R43_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R43.extendedFrags.fastq.gz
[FLASH]     ./R43.notCombined_1.fastq.gz
[FLASH]     ./R43.notCombined_2.fastq.gz
[FLASH]     ./R43.hist
[FLASH]     ./R43.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 26955 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      26955
[FLASH]     Combined pairs:   5230
[FLASH]     Uncombined pairs: 21725
[FLASH]     Percent combined: 19.40%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.253 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5230 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R43/R43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R43_centroids.fasta --uc R43.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5230 sequences and identified 2691 clusters (clustering threshold 1.00) - 84 clusters (representing 2203 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R44 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R44_1.fastq /home/working/demultiplexed/R44_2.fastq R44_forward.paired.fastq.gz R44_forward.singletons.fastq.gz R44_reverse.paired.fastq.gz R44_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R44_1.fastq /home/working/demultiplexed/R44_2.fastq R44_forward.paired.fastq.gz R44_forward.singletons.fastq.gz R44_reverse.paired.fastq.gz R44_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:51:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R44/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 59807 Both Surviving: 3915 (6.55%) Forward Only Surviving: 1319 (2.21%) Reverse Only Surviving: 818 (1.37%) Dropped: 53755 (89.88%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R44_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R44_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: R44_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: R44_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R44_forward.paired.headclipped.fastq.gz R44_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R44 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R44_forward.paired.headclipped.fastq.gz
[FLASH]     R44_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R44.extendedFrags.fastq.gz
[FLASH]     ./R44.notCombined_1.fastq.gz
[FLASH]     ./R44.notCombined_2.fastq.gz
[FLASH]     ./R44.hist
[FLASH]     ./R44.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3915 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3915
[FLASH]     Combined pairs:   1308
[FLASH]     Uncombined pairs: 2607
[FLASH]     Percent combined: 33.41%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.040 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1308 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R44/R44_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R44_centroids.fasta --uc R44.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1308 sequences and identified 566 clusters (clustering threshold 1.00) - 13 clusters (representing 617 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R45 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R45_1.fastq /home/working/demultiplexed/R45_2.fastq R45_forward.paired.fastq.gz R45_forward.singletons.fastq.gz R45_reverse.paired.fastq.gz R45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R45_1.fastq /home/working/demultiplexed/R45_2.fastq R45_forward.paired.fastq.gz R45_forward.singletons.fastq.gz R45_reverse.paired.fastq.gz R45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:51:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R45/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 52759 Both Surviving: 17663 (33.48%) Forward Only Surviving: 10402 (19.72%) Reverse Only Surviving: 418 (0.79%) Dropped: 24276 (46.01%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R45_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R45_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: R45_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: R45_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R45_forward.paired.headclipped.fastq.gz R45_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R45 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R45_forward.paired.headclipped.fastq.gz
[FLASH]     R45_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R45.extendedFrags.fastq.gz
[FLASH]     ./R45.notCombined_1.fastq.gz
[FLASH]     ./R45.notCombined_2.fastq.gz
[FLASH]     ./R45.hist
[FLASH]     ./R45.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 17663 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      17663
[FLASH]     Combined pairs:   14829
[FLASH]     Uncombined pairs: 2834
[FLASH]     Percent combined: 83.96%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.418 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

14829 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R45/R45_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R45_centroids.fasta --uc R45.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 14829 sequences and identified 6262 clusters (clustering threshold 1.00) - 316 clusters (representing 8149 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R48 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R48_1.fastq /home/working/demultiplexed/R48_2.fastq R48_forward.paired.fastq.gz R48_forward.singletons.fastq.gz R48_reverse.paired.fastq.gz R48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R48_1.fastq /home/working/demultiplexed/R48_2.fastq R48_forward.paired.fastq.gz R48_forward.singletons.fastq.gz R48_reverse.paired.fastq.gz R48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:51:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R48/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20979 Both Surviving: 5392 (25.70%) Forward Only Surviving: 1126 (5.37%) Reverse Only Surviving: 316 (1.51%) Dropped: 14145 (67.42%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: R48_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: R48_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: R48_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: R48_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R48_forward.paired.headclipped.fastq.gz R48_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R48 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R48_forward.paired.headclipped.fastq.gz
[FLASH]     R48_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R48.extendedFrags.fastq.gz
[FLASH]     ./R48.notCombined_1.fastq.gz
[FLASH]     ./R48.notCombined_2.fastq.gz
[FLASH]     ./R48.hist
[FLASH]     ./R48.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5392 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5392
[FLASH]     Combined pairs:   2042
[FLASH]     Uncombined pairs: 3350
[FLASH]     Percent combined: 37.87%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.066 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2042 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R48/R48_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R48_centroids.fasta --uc R48.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2042 sequences and identified 1303 clusters (clustering threshold 1.00) - 26 clusters (representing 655 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R49 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R49_1.fastq /home/working/demultiplexed/R49_2.fastq R49_forward.paired.fastq.gz R49_forward.singletons.fastq.gz R49_reverse.paired.fastq.gz R49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R49_1.fastq /home/working/demultiplexed/R49_2.fastq R49_forward.paired.fastq.gz R49_forward.singletons.fastq.gz R49_reverse.paired.fastq.gz R49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:51:47 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R49/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9114 Both Surviving: 1505 (16.51%) Forward Only Surviving: 341 (3.74%) Reverse Only Surviving: 295 (3.24%) Dropped: 6973 (76.51%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: R49_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: R49_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R49_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R49_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R49_forward.paired.headclipped.fastq.gz R49_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R49 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R49_forward.paired.headclipped.fastq.gz
[FLASH]     R49_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R49.extendedFrags.fastq.gz
[FLASH]     ./R49.notCombined_1.fastq.gz
[FLASH]     ./R49.notCombined_2.fastq.gz
[FLASH]     ./R49.hist
[FLASH]     ./R49.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1505 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1505
[FLASH]     Combined pairs:   358
[FLASH]     Uncombined pairs: 1147
[FLASH]     Percent combined: 23.79%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.015 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

358 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R49/R49_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R49_centroids.fasta --uc R49.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 358 sequences and identified 195 clusters (clustering threshold 1.00) - 6 clusters (representing 156 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R59 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R59_1.fastq /home/working/demultiplexed/R59_2.fastq R59_forward.paired.fastq.gz R59_forward.singletons.fastq.gz R59_reverse.paired.fastq.gz R59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R59_1.fastq /home/working/demultiplexed/R59_2.fastq R59_forward.paired.fastq.gz R59_forward.singletons.fastq.gz R59_reverse.paired.fastq.gz R59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:51:50 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R59/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 35926 Both Surviving: 3298 (9.18%) Forward Only Surviving: 970 (2.70%) Reverse Only Surviving: 529 (1.47%) Dropped: 31129 (86.65%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R59_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R59_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: R59_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: R59_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R59_forward.paired.headclipped.fastq.gz R59_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R59 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R59_forward.paired.headclipped.fastq.gz
[FLASH]     R59_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R59.extendedFrags.fastq.gz
[FLASH]     ./R59.notCombined_1.fastq.gz
[FLASH]     ./R59.notCombined_2.fastq.gz
[FLASH]     ./R59.hist
[FLASH]     ./R59.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3298 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3298
[FLASH]     Combined pairs:   129
[FLASH]     Uncombined pairs: 3169
[FLASH]     Percent combined: 3.91%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.024 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

129 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R59/R59_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R59_centroids.fasta --uc R59.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 129 sequences and identified 87 clusters (clustering threshold 1.00) - 3 clusters (representing 34 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R60 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R60_1.fastq /home/working/demultiplexed/R60_2.fastq R60_forward.paired.fastq.gz R60_forward.singletons.fastq.gz R60_reverse.paired.fastq.gz R60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R60_1.fastq /home/working/demultiplexed/R60_2.fastq R60_forward.paired.fastq.gz R60_forward.singletons.fastq.gz R60_reverse.paired.fastq.gz R60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:51:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R60/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 39486 Both Surviving: 5392 (13.66%) Forward Only Surviving: 1158 (2.93%) Reverse Only Surviving: 443 (1.12%) Dropped: 32493 (82.29%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: R60_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: R60_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R60_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R60_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R60_forward.paired.headclipped.fastq.gz R60_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R60 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R60_forward.paired.headclipped.fastq.gz
[FLASH]     R60_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R60.extendedFrags.fastq.gz
[FLASH]     ./R60.notCombined_1.fastq.gz
[FLASH]     ./R60.notCombined_2.fastq.gz
[FLASH]     ./R60.hist
[FLASH]     ./R60.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5392 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5392
[FLASH]     Combined pairs:   47
[FLASH]     Uncombined pairs: 5345
[FLASH]     Percent combined: 0.87%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.049 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

47 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R60/R60_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R60_centroids.fasta --uc R60.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 47 sequences and identified 42 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R67 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R67_1.fastq /home/working/demultiplexed/R67_2.fastq R67_forward.paired.fastq.gz R67_forward.singletons.fastq.gz R67_reverse.paired.fastq.gz R67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R67_1.fastq /home/working/demultiplexed/R67_2.fastq R67_forward.paired.fastq.gz R67_forward.singletons.fastq.gz R67_reverse.paired.fastq.gz R67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:52:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R67/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 27457 Both Surviving: 6301 (22.95%) Forward Only Surviving: 4040 (14.71%) Reverse Only Surviving: 307 (1.12%) Dropped: 16809 (61.22%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: R67_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: R67_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: R67_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: R67_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R67_forward.paired.headclipped.fastq.gz R67_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R67 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R67_forward.paired.headclipped.fastq.gz
[FLASH]     R67_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R67.extendedFrags.fastq.gz
[FLASH]     ./R67.notCombined_1.fastq.gz
[FLASH]     ./R67.notCombined_2.fastq.gz
[FLASH]     ./R67.hist
[FLASH]     ./R67.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6301 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6301
[FLASH]     Combined pairs:   4584
[FLASH]     Uncombined pairs: 1717
[FLASH]     Percent combined: 72.75%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.127 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4584 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R67/R67_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R67_centroids.fasta --uc R67.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4584 sequences and identified 2358 clusters (clustering threshold 1.00) - 60 clusters (representing 1875 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R68 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R68_1.fastq /home/working/demultiplexed/R68_2.fastq R68_forward.paired.fastq.gz R68_forward.singletons.fastq.gz R68_reverse.paired.fastq.gz R68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R68_1.fastq /home/working/demultiplexed/R68_2.fastq R68_forward.paired.fastq.gz R68_forward.singletons.fastq.gz R68_reverse.paired.fastq.gz R68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:52:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R68/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 12701 Both Surviving: 2577 (20.29%) Forward Only Surviving: 1071 (8.43%) Reverse Only Surviving: 201 (1.58%) Dropped: 8852 (69.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: R68_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: R68_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R68_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R68_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R68_forward.paired.headclipped.fastq.gz R68_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R68 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R68_forward.paired.headclipped.fastq.gz
[FLASH]     R68_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R68.extendedFrags.fastq.gz
[FLASH]     ./R68.notCombined_1.fastq.gz
[FLASH]     ./R68.notCombined_2.fastq.gz
[FLASH]     ./R68.hist
[FLASH]     ./R68.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2577 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2577
[FLASH]     Combined pairs:   2078
[FLASH]     Uncombined pairs: 499
[FLASH]     Percent combined: 80.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.057 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2078 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R68/R68_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R68_centroids.fasta --uc R68.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2078 sequences and identified 912 clusters (clustering threshold 1.00) - 34 clusters (representing 1014 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R71 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R71_1.fastq /home/working/demultiplexed/R71_2.fastq R71_forward.paired.fastq.gz R71_forward.singletons.fastq.gz R71_reverse.paired.fastq.gz R71_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R71_1.fastq /home/working/demultiplexed/R71_2.fastq R71_forward.paired.fastq.gz R71_forward.singletons.fastq.gz R71_reverse.paired.fastq.gz R71_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:52:17 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R71/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 35268 Both Surviving: 4575 (12.97%) Forward Only Surviving: 1404 (3.98%) Reverse Only Surviving: 472 (1.34%) Dropped: 28817 (81.71%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: R71_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: R71_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R71_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R71_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R71_forward.paired.headclipped.fastq.gz R71_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R71 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R71_forward.paired.headclipped.fastq.gz
[FLASH]     R71_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R71.extendedFrags.fastq.gz
[FLASH]     ./R71.notCombined_1.fastq.gz
[FLASH]     ./R71.notCombined_2.fastq.gz
[FLASH]     ./R71.hist
[FLASH]     ./R71.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4575 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4575
[FLASH]     Combined pairs:   262
[FLASH]     Uncombined pairs: 4313
[FLASH]     Percent combined: 5.73%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.036 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

262 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R71/R71_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R71_centroids.fasta --uc R71.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 262 sequences and identified 189 clusters (clustering threshold 1.00) - 7 clusters (representing 55 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R74 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R74_1.fastq /home/working/demultiplexed/R74_2.fastq R74_forward.paired.fastq.gz R74_forward.singletons.fastq.gz R74_reverse.paired.fastq.gz R74_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R74_1.fastq /home/working/demultiplexed/R74_2.fastq R74_forward.paired.fastq.gz R74_forward.singletons.fastq.gz R74_reverse.paired.fastq.gz R74_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:52:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R74/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37447 Both Surviving: 185 (0.49%) Forward Only Surviving: 449 (1.20%) Reverse Only Surviving: 251 (0.67%) Dropped: 36562 (97.64%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: R74_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: R74_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R74_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R74_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R74_forward.paired.headclipped.fastq.gz R74_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R74 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R74_forward.paired.headclipped.fastq.gz
[FLASH]     R74_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R74.extendedFrags.fastq.gz
[FLASH]     ./R74.notCombined_1.fastq.gz
[FLASH]     ./R74.notCombined_2.fastq.gz
[FLASH]     ./R74.hist
[FLASH]     ./R74.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 185 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      185
[FLASH]     Combined pairs:   128
[FLASH]     Uncombined pairs: 57
[FLASH]     Percent combined: 69.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

128 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R74/R74_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R74_centroids.fasta --uc R74.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 128 sequences and identified 73 clusters (clustering threshold 1.00) - 3 clusters (representing 52 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R8 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R8_1.fastq /home/working/demultiplexed/R8_2.fastq R8_forward.paired.fastq.gz R8_forward.singletons.fastq.gz R8_reverse.paired.fastq.gz R8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R8_1.fastq /home/working/demultiplexed/R8_2.fastq R8_forward.paired.fastq.gz R8_forward.singletons.fastq.gz R8_reverse.paired.fastq.gz R8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:52:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R8/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29705 Both Surviving: 2428 (8.17%) Forward Only Surviving: 762 (2.57%) Reverse Only Surviving: 299 (1.01%) Dropped: 26216 (88.25%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R8_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R8_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R8_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R8_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R8_forward.paired.headclipped.fastq.gz R8_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R8 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R8_forward.paired.headclipped.fastq.gz
[FLASH]     R8_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R8.extendedFrags.fastq.gz
[FLASH]     ./R8.notCombined_1.fastq.gz
[FLASH]     ./R8.notCombined_2.fastq.gz
[FLASH]     ./R8.hist
[FLASH]     ./R8.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2428 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2428
[FLASH]     Combined pairs:   2143
[FLASH]     Uncombined pairs: 285
[FLASH]     Percent combined: 88.26%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.062 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2143 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R8/R8_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R8_centroids.fasta --uc R8.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2143 sequences and identified 1180 clusters (clustering threshold 1.00) - 27 clusters (representing 850 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R85 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R85_1.fastq /home/working/demultiplexed/R85_2.fastq R85_forward.paired.fastq.gz R85_forward.singletons.fastq.gz R85_reverse.paired.fastq.gz R85_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R85_1.fastq /home/working/demultiplexed/R85_2.fastq R85_forward.paired.fastq.gz R85_forward.singletons.fastq.gz R85_reverse.paired.fastq.gz R85_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:52:32 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R85/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18834 Both Surviving: 3948 (20.96%) Forward Only Surviving: 1269 (6.74%) Reverse Only Surviving: 242 (1.28%) Dropped: 13375 (71.02%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R85_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R85_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: R85_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: R85_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R85_forward.paired.headclipped.fastq.gz R85_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R85 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R85_forward.paired.headclipped.fastq.gz
[FLASH]     R85_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R85.extendedFrags.fastq.gz
[FLASH]     ./R85.notCombined_1.fastq.gz
[FLASH]     ./R85.notCombined_2.fastq.gz
[FLASH]     ./R85.hist
[FLASH]     ./R85.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3948 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3948
[FLASH]     Combined pairs:   1219
[FLASH]     Uncombined pairs: 2729
[FLASH]     Percent combined: 30.88%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.043 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1219 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R85/R85_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R85_centroids.fasta --uc R85.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1219 sequences and identified 733 clusters (clustering threshold 1.00) - 16 clusters (representing 439 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R9 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R9_1.fastq /home/working/demultiplexed/R9_2.fastq R9_forward.paired.fastq.gz R9_forward.singletons.fastq.gz R9_reverse.paired.fastq.gz R9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R9_1.fastq /home/working/demultiplexed/R9_2.fastq R9_forward.paired.fastq.gz R9_forward.singletons.fastq.gz R9_reverse.paired.fastq.gz R9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:52:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R9/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22136 Both Surviving: 5623 (25.40%) Forward Only Surviving: 1911 (8.63%) Reverse Only Surviving: 301 (1.36%) Dropped: 14301 (64.61%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: R9_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: R9_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: R9_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: R9_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R9_forward.paired.headclipped.fastq.gz R9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R9 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R9_forward.paired.headclipped.fastq.gz
[FLASH]     R9_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R9.extendedFrags.fastq.gz
[FLASH]     ./R9.notCombined_1.fastq.gz
[FLASH]     ./R9.notCombined_2.fastq.gz
[FLASH]     ./R9.hist
[FLASH]     ./R9.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5623 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5623
[FLASH]     Combined pairs:   251
[FLASH]     Uncombined pairs: 5372
[FLASH]     Percent combined: 4.46%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.041 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

251 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R9/R9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R9_centroids.fasta --uc R9.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 251 sequences and identified 219 clusters (clustering threshold 1.00) - 2 clusters (representing 16 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R90 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R90_1.fastq /home/working/demultiplexed/R90_2.fastq R90_forward.paired.fastq.gz R90_forward.singletons.fastq.gz R90_reverse.paired.fastq.gz R90_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R90_1.fastq /home/working/demultiplexed/R90_2.fastq R90_forward.paired.fastq.gz R90_forward.singletons.fastq.gz R90_reverse.paired.fastq.gz R90_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:52:42 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R90/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15190 Both Surviving: 328 (2.16%) Forward Only Surviving: 380 (2.50%) Reverse Only Surviving: 160 (1.05%) Dropped: 14322 (94.29%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: R90_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: R90_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: R90_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: R90_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R90_forward.paired.headclipped.fastq.gz R90_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R90 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R90_forward.paired.headclipped.fastq.gz
[FLASH]     R90_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R90.extendedFrags.fastq.gz
[FLASH]     ./R90.notCombined_1.fastq.gz
[FLASH]     ./R90.notCombined_2.fastq.gz
[FLASH]     ./R90.hist
[FLASH]     ./R90.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 328 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      328
[FLASH]     Combined pairs:   231
[FLASH]     Uncombined pairs: 97
[FLASH]     Percent combined: 70.43%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.007 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

231 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R90/R90_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R90_centroids.fasta --uc R90.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 231 sequences and identified 178 clusters (clustering threshold 1.00) - 3 clusters (representing 45 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R95 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R95_1.fastq /home/working/demultiplexed/R95_2.fastq R95_forward.paired.fastq.gz R95_forward.singletons.fastq.gz R95_reverse.paired.fastq.gz R95_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R95_1.fastq /home/working/demultiplexed/R95_2.fastq R95_forward.paired.fastq.gz R95_forward.singletons.fastq.gz R95_reverse.paired.fastq.gz R95_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:52:44 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R95/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15768 Both Surviving: 300 (1.90%) Forward Only Surviving: 279 (1.77%) Reverse Only Surviving: 149 (0.94%) Dropped: 15040 (95.38%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: R95_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: R95_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: R95_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: R95_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R95_forward.paired.headclipped.fastq.gz R95_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R95 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R95_forward.paired.headclipped.fastq.gz
[FLASH]     R95_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R95.extendedFrags.fastq.gz
[FLASH]     ./R95.notCombined_1.fastq.gz
[FLASH]     ./R95.notCombined_2.fastq.gz
[FLASH]     ./R95.hist
[FLASH]     ./R95.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 300 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      300
[FLASH]     Combined pairs:   60
[FLASH]     Uncombined pairs: 240
[FLASH]     Percent combined: 20.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.005 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

60 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R95/R95_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R95_centroids.fasta --uc R95.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 60 sequences and identified 53 clusters (clustering threshold 1.00) - 1 clusters (representing 8 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R98 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R98_1.fastq /home/working/demultiplexed/R98_2.fastq R98_forward.paired.fastq.gz R98_forward.singletons.fastq.gz R98_reverse.paired.fastq.gz R98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R98_1.fastq /home/working/demultiplexed/R98_2.fastq R98_forward.paired.fastq.gz R98_forward.singletons.fastq.gz R98_reverse.paired.fastq.gz R98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:52:48 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R98/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 47670 Both Surviving: 8088 (16.97%) Forward Only Surviving: 2258 (4.74%) Reverse Only Surviving: 560 (1.17%) Dropped: 36764 (77.12%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: R98_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: R98_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R98_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R98_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R98_forward.paired.headclipped.fastq.gz R98_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R98 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R98_forward.paired.headclipped.fastq.gz
[FLASH]     R98_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R98.extendedFrags.fastq.gz
[FLASH]     ./R98.notCombined_1.fastq.gz
[FLASH]     ./R98.notCombined_2.fastq.gz
[FLASH]     ./R98.hist
[FLASH]     ./R98.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8088 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8088
[FLASH]     Combined pairs:   3682
[FLASH]     Uncombined pairs: 4406
[FLASH]     Percent combined: 45.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.116 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3682 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R98/R98_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R98_centroids.fasta --uc R98.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3682 sequences and identified 1948 clusters (clustering threshold 1.00) - 49 clusters (representing 1523 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R99 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R99_1.fastq /home/working/demultiplexed/R99_2.fastq R99_forward.paired.fastq.gz R99_forward.singletons.fastq.gz R99_reverse.paired.fastq.gz R99_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R99_1.fastq /home/working/demultiplexed/R99_2.fastq R99_forward.paired.fastq.gz R99_forward.singletons.fastq.gz R99_reverse.paired.fastq.gz R99_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:53:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/R99/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 82372 Both Surviving: 1529 (1.86%) Forward Only Surviving: 2684 (3.26%) Reverse Only Surviving: 882 (1.07%) Dropped: 77277 (93.81%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: R99_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: R99_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: R99_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: R99_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R99_forward.paired.headclipped.fastq.gz R99_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R99 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R99_forward.paired.headclipped.fastq.gz
[FLASH]     R99_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R99.extendedFrags.fastq.gz
[FLASH]     ./R99.notCombined_1.fastq.gz
[FLASH]     ./R99.notCombined_2.fastq.gz
[FLASH]     ./R99.hist
[FLASH]     ./R99.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1529 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1529
[FLASH]     Combined pairs:   736
[FLASH]     Uncombined pairs: 793
[FLASH]     Percent combined: 48.14%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.020 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

736 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/R99/R99_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R99_centroids.fasta --uc R99.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 736 sequences and identified 559 clusters (clustering threshold 1.00) - 10 clusters (representing 117 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S100 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S100_1.fastq /home/working/demultiplexed/S100_2.fastq S100_forward.paired.fastq.gz S100_forward.singletons.fastq.gz S100_reverse.paired.fastq.gz S100_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S100_1.fastq /home/working/demultiplexed/S100_2.fastq S100_forward.paired.fastq.gz S100_forward.singletons.fastq.gz S100_reverse.paired.fastq.gz S100_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:53:08 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S100/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15016 Both Surviving: 1579 (10.52%) Forward Only Surviving: 676 (4.50%) Reverse Only Surviving: 115 (0.77%) Dropped: 12646 (84.22%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S100_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S100_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: S100_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: S100_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S100_forward.paired.headclipped.fastq.gz S100_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S100 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S100_forward.paired.headclipped.fastq.gz
[FLASH]     S100_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S100.extendedFrags.fastq.gz
[FLASH]     ./S100.notCombined_1.fastq.gz
[FLASH]     ./S100.notCombined_2.fastq.gz
[FLASH]     ./S100.hist
[FLASH]     ./S100.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1579 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1579
[FLASH]     Combined pairs:   1345
[FLASH]     Uncombined pairs: 234
[FLASH]     Percent combined: 85.18%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.037 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1345 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S100/S100_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S100_centroids.fasta --uc S100.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1345 sequences and identified 747 clusters (clustering threshold 1.00) - 22 clusters (representing 525 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S101 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S101_1.fastq /home/working/demultiplexed/S101_2.fastq S101_forward.paired.fastq.gz S101_forward.singletons.fastq.gz S101_reverse.paired.fastq.gz S101_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S101_1.fastq /home/working/demultiplexed/S101_2.fastq S101_forward.paired.fastq.gz S101_forward.singletons.fastq.gz S101_reverse.paired.fastq.gz S101_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:53:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S101/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9613 Both Surviving: 328 (3.41%) Forward Only Surviving: 199 (2.07%) Reverse Only Surviving: 243 (2.53%) Dropped: 8843 (91.99%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: S101_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: S101_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S101_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S101_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S101_forward.paired.headclipped.fastq.gz S101_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S101 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S101_forward.paired.headclipped.fastq.gz
[FLASH]     S101_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S101.extendedFrags.fastq.gz
[FLASH]     ./S101.notCombined_1.fastq.gz
[FLASH]     ./S101.notCombined_2.fastq.gz
[FLASH]     ./S101.hist
[FLASH]     ./S101.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 328 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      328
[FLASH]     Combined pairs:   59
[FLASH]     Uncombined pairs: 269
[FLASH]     Percent combined: 17.99%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.005 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

59 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S101/S101_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S101_centroids.fasta --uc S101.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 59 sequences and identified 50 clusters (clustering threshold 1.00) - 1 clusters (representing 10 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S102 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S102_1.fastq /home/working/demultiplexed/S102_2.fastq S102_forward.paired.fastq.gz S102_forward.singletons.fastq.gz S102_reverse.paired.fastq.gz S102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S102_1.fastq /home/working/demultiplexed/S102_2.fastq S102_forward.paired.fastq.gz S102_forward.singletons.fastq.gz S102_reverse.paired.fastq.gz S102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:53:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S102/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17129 Both Surviving: 3440 (20.08%) Forward Only Surviving: 1158 (6.76%) Reverse Only Surviving: 149 (0.87%) Dropped: 12382 (72.29%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: S102_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: S102_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: S102_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: S102_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S102_forward.paired.headclipped.fastq.gz S102_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S102 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S102_forward.paired.headclipped.fastq.gz
[FLASH]     S102_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S102.extendedFrags.fastq.gz
[FLASH]     ./S102.notCombined_1.fastq.gz
[FLASH]     ./S102.notCombined_2.fastq.gz
[FLASH]     ./S102.hist
[FLASH]     ./S102.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3440 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3440
[FLASH]     Combined pairs:   1551
[FLASH]     Uncombined pairs: 1889
[FLASH]     Percent combined: 45.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.051 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1551 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S102/S102_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S102_centroids.fasta --uc S102.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1551 sequences and identified 816 clusters (clustering threshold 1.00) - 14 clusters (representing 650 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S11 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S11_1.fastq /home/working/demultiplexed/S11_2.fastq S11_forward.paired.fastq.gz S11_forward.singletons.fastq.gz S11_reverse.paired.fastq.gz S11_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S11_1.fastq /home/working/demultiplexed/S11_2.fastq S11_forward.paired.fastq.gz S11_forward.singletons.fastq.gz S11_reverse.paired.fastq.gz S11_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:53:20 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S11/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41543 Both Surviving: 671 (1.62%) Forward Only Surviving: 1052 (2.53%) Reverse Only Surviving: 567 (1.36%) Dropped: 39253 (94.49%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: S11_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: S11_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S11_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S11_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S11_forward.paired.headclipped.fastq.gz S11_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S11 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S11_forward.paired.headclipped.fastq.gz
[FLASH]     S11_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S11.extendedFrags.fastq.gz
[FLASH]     ./S11.notCombined_1.fastq.gz
[FLASH]     ./S11.notCombined_2.fastq.gz
[FLASH]     ./S11.hist
[FLASH]     ./S11.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 671 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      671
[FLASH]     Combined pairs:   345
[FLASH]     Uncombined pairs: 326
[FLASH]     Percent combined: 51.42%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.010 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

345 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S11/S11_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S11_centroids.fasta --uc S11.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 345 sequences and identified 185 clusters (clustering threshold 1.00) - 5 clusters (representing 141 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S13 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S13_1.fastq /home/working/demultiplexed/S13_2.fastq S13_forward.paired.fastq.gz S13_forward.singletons.fastq.gz S13_reverse.paired.fastq.gz S13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S13_1.fastq /home/working/demultiplexed/S13_2.fastq S13_forward.paired.fastq.gz S13_forward.singletons.fastq.gz S13_reverse.paired.fastq.gz S13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:53:24 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S13/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 21977 Both Surviving: 3585 (16.31%) Forward Only Surviving: 869 (3.95%) Reverse Only Surviving: 356 (1.62%) Dropped: 17167 (78.11%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: S13_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: S13_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: S13_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: S13_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S13_forward.paired.headclipped.fastq.gz S13_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S13 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S13_forward.paired.headclipped.fastq.gz
[FLASH]     S13_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S13.extendedFrags.fastq.gz
[FLASH]     ./S13.notCombined_1.fastq.gz
[FLASH]     ./S13.notCombined_2.fastq.gz
[FLASH]     ./S13.hist
[FLASH]     ./S13.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3585 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3585
[FLASH]     Combined pairs:   904
[FLASH]     Uncombined pairs: 2681
[FLASH]     Percent combined: 25.22%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.033 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

904 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S13/S13_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S13_centroids.fasta --uc S13.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 904 sequences and identified 511 clusters (clustering threshold 1.00) - 9 clusters (representing 347 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S18 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S18_1.fastq /home/working/demultiplexed/S18_2.fastq S18_forward.paired.fastq.gz S18_forward.singletons.fastq.gz S18_reverse.paired.fastq.gz S18_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S18_1.fastq /home/working/demultiplexed/S18_2.fastq S18_forward.paired.fastq.gz S18_forward.singletons.fastq.gz S18_reverse.paired.fastq.gz S18_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:53:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S18/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 45130 Both Surviving: 6734 (14.92%) Forward Only Surviving: 1728 (3.83%) Reverse Only Surviving: 829 (1.84%) Dropped: 35839 (79.41%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: S18_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: S18_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: S18_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: S18_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S18_forward.paired.headclipped.fastq.gz S18_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S18 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S18_forward.paired.headclipped.fastq.gz
[FLASH]     S18_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S18.extendedFrags.fastq.gz
[FLASH]     ./S18.notCombined_1.fastq.gz
[FLASH]     ./S18.notCombined_2.fastq.gz
[FLASH]     ./S18.hist
[FLASH]     ./S18.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6734 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6734
[FLASH]     Combined pairs:   71
[FLASH]     Uncombined pairs: 6663
[FLASH]     Percent combined: 1.05%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.049 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

71 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S18/S18_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S18_centroids.fasta --uc S18.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 71 sequences and identified 61 clusters (clustering threshold 1.00) - 1 clusters (representing 9 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S19 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S19_1.fastq /home/working/demultiplexed/S19_2.fastq S19_forward.paired.fastq.gz S19_forward.singletons.fastq.gz S19_reverse.paired.fastq.gz S19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S19_1.fastq /home/working/demultiplexed/S19_2.fastq S19_forward.paired.fastq.gz S19_forward.singletons.fastq.gz S19_reverse.paired.fastq.gz S19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:53:35 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S19/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14524 Both Surviving: 3946 (27.17%) Forward Only Surviving: 1547 (10.65%) Reverse Only Surviving: 301 (2.07%) Dropped: 8730 (60.11%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: S19_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: S19_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S19_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S19_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S19_forward.paired.headclipped.fastq.gz S19_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S19 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S19_forward.paired.headclipped.fastq.gz
[FLASH]     S19_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S19.extendedFrags.fastq.gz
[FLASH]     ./S19.notCombined_1.fastq.gz
[FLASH]     ./S19.notCombined_2.fastq.gz
[FLASH]     ./S19.hist
[FLASH]     ./S19.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3946 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3946
[FLASH]     Combined pairs:   2350
[FLASH]     Uncombined pairs: 1596
[FLASH]     Percent combined: 59.55%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.070 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2350 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S19/S19_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S19_centroids.fasta --uc S19.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2350 sequences and identified 1203 clusters (clustering threshold 1.00) - 28 clusters (representing 980 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S24 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S24_1.fastq /home/working/demultiplexed/S24_2.fastq S24_forward.paired.fastq.gz S24_forward.singletons.fastq.gz S24_reverse.paired.fastq.gz S24_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S24_1.fastq /home/working/demultiplexed/S24_2.fastq S24_forward.paired.fastq.gz S24_forward.singletons.fastq.gz S24_reverse.paired.fastq.gz S24_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:53:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S24/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9162 Both Surviving: 875 (9.55%) Forward Only Surviving: 690 (7.53%) Reverse Only Surviving: 70 (0.76%) Dropped: 7527 (82.15%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S24_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S24_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S24_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S24_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S24_forward.paired.headclipped.fastq.gz S24_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S24 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S24_forward.paired.headclipped.fastq.gz
[FLASH]     S24_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S24.extendedFrags.fastq.gz
[FLASH]     ./S24.notCombined_1.fastq.gz
[FLASH]     ./S24.notCombined_2.fastq.gz
[FLASH]     ./S24.hist
[FLASH]     ./S24.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 875 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      875
[FLASH]     Combined pairs:   288
[FLASH]     Uncombined pairs: 587
[FLASH]     Percent combined: 32.91%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.010 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

288 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S24/S24_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S24_centroids.fasta --uc S24.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 288 sequences and identified 240 clusters (clustering threshold 1.00) - 2 clusters (representing 24 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S30 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S30_1.fastq /home/working/demultiplexed/S30_2.fastq S30_forward.paired.fastq.gz S30_forward.singletons.fastq.gz S30_reverse.paired.fastq.gz S30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S30_1.fastq /home/working/demultiplexed/S30_2.fastq S30_forward.paired.fastq.gz S30_forward.singletons.fastq.gz S30_reverse.paired.fastq.gz S30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:53:43 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S30/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11465 Both Surviving: 2208 (19.26%) Forward Only Surviving: 583 (5.09%) Reverse Only Surviving: 164 (1.43%) Dropped: 8510 (74.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: S30_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: S30_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S30_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S30_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S30_forward.paired.headclipped.fastq.gz S30_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S30 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S30_forward.paired.headclipped.fastq.gz
[FLASH]     S30_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S30.extendedFrags.fastq.gz
[FLASH]     ./S30.notCombined_1.fastq.gz
[FLASH]     ./S30.notCombined_2.fastq.gz
[FLASH]     ./S30.hist
[FLASH]     ./S30.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2208 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2208
[FLASH]     Combined pairs:   32
[FLASH]     Uncombined pairs: 2176
[FLASH]     Percent combined: 1.45%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.019 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

32 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S30/S30_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S30_centroids.fasta --uc S30.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 32 sequences and identified 29 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S31 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S31_1.fastq /home/working/demultiplexed/S31_2.fastq S31_forward.paired.fastq.gz S31_forward.singletons.fastq.gz S31_reverse.paired.fastq.gz S31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S31_1.fastq /home/working/demultiplexed/S31_2.fastq S31_forward.paired.fastq.gz S31_forward.singletons.fastq.gz S31_reverse.paired.fastq.gz S31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:53:45 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S31/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 8667 Both Surviving: 111 (1.28%) Forward Only Surviving: 129 (1.49%) Reverse Only Surviving: 100 (1.15%) Dropped: 8327 (96.08%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S31_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S31_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: S31_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: S31_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S31_forward.paired.headclipped.fastq.gz S31_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S31 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S31_forward.paired.headclipped.fastq.gz
[FLASH]     S31_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S31.extendedFrags.fastq.gz
[FLASH]     ./S31.notCombined_1.fastq.gz
[FLASH]     ./S31.notCombined_2.fastq.gz
[FLASH]     ./S31.hist
[FLASH]     ./S31.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 111 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      111
[FLASH]     Combined pairs:   34
[FLASH]     Uncombined pairs: 77
[FLASH]     Percent combined: 30.63%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

34 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S31/S31_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S31_centroids.fasta --uc S31.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 34 sequences and identified 33 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S33 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S33_1.fastq /home/working/demultiplexed/S33_2.fastq S33_forward.paired.fastq.gz S33_forward.singletons.fastq.gz S33_reverse.paired.fastq.gz S33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S33_1.fastq /home/working/demultiplexed/S33_2.fastq S33_forward.paired.fastq.gz S33_forward.singletons.fastq.gz S33_reverse.paired.fastq.gz S33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:53:47 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S33/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22355 Both Surviving: 761 (3.40%) Forward Only Surviving: 773 (3.46%) Reverse Only Surviving: 235 (1.05%) Dropped: 20586 (92.09%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: S33_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: S33_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: S33_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: S33_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S33_forward.paired.headclipped.fastq.gz S33_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S33 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S33_forward.paired.headclipped.fastq.gz
[FLASH]     S33_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S33.extendedFrags.fastq.gz
[FLASH]     ./S33.notCombined_1.fastq.gz
[FLASH]     ./S33.notCombined_2.fastq.gz
[FLASH]     ./S33.hist
[FLASH]     ./S33.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 761 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      761
[FLASH]     Combined pairs:   432
[FLASH]     Uncombined pairs: 329
[FLASH]     Percent combined: 56.77%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.013 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

432 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S33/S33_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S33_centroids.fasta --uc S33.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 432 sequences and identified 320 clusters (clustering threshold 1.00) - 3 clusters (representing 89 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S35 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S35_1.fastq /home/working/demultiplexed/S35_2.fastq S35_forward.paired.fastq.gz S35_forward.singletons.fastq.gz S35_reverse.paired.fastq.gz S35_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S35_1.fastq /home/working/demultiplexed/S35_2.fastq S35_forward.paired.fastq.gz S35_forward.singletons.fastq.gz S35_reverse.paired.fastq.gz S35_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:53:53 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S35/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 77091 Both Surviving: 5196 (6.74%) Forward Only Surviving: 3552 (4.61%) Reverse Only Surviving: 988 (1.28%) Dropped: 67355 (87.37%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S35_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S35_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: S35_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: S35_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S35_forward.paired.headclipped.fastq.gz S35_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S35 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S35_forward.paired.headclipped.fastq.gz
[FLASH]     S35_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S35.extendedFrags.fastq.gz
[FLASH]     ./S35.notCombined_1.fastq.gz
[FLASH]     ./S35.notCombined_2.fastq.gz
[FLASH]     ./S35.hist
[FLASH]     ./S35.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5196 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5196
[FLASH]     Combined pairs:   2485
[FLASH]     Uncombined pairs: 2711
[FLASH]     Percent combined: 47.83%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.075 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2485 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S35/S35_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S35_centroids.fasta --uc S35.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2485 sequences and identified 1438 clusters (clustering threshold 1.00) - 40 clusters (representing 905 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S39 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S39_1.fastq /home/working/demultiplexed/S39_2.fastq S39_forward.paired.fastq.gz S39_forward.singletons.fastq.gz S39_reverse.paired.fastq.gz S39_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S39_1.fastq /home/working/demultiplexed/S39_2.fastq S39_forward.paired.fastq.gz S39_forward.singletons.fastq.gz S39_reverse.paired.fastq.gz S39_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:54:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S39/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16769 Both Surviving: 3591 (21.41%) Forward Only Surviving: 1075 (6.41%) Reverse Only Surviving: 215 (1.28%) Dropped: 11888 (70.89%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: S39_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: S39_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S39_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S39_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S39_forward.paired.headclipped.fastq.gz S39_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S39 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S39_forward.paired.headclipped.fastq.gz
[FLASH]     S39_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S39.extendedFrags.fastq.gz
[FLASH]     ./S39.notCombined_1.fastq.gz
[FLASH]     ./S39.notCombined_2.fastq.gz
[FLASH]     ./S39.hist
[FLASH]     ./S39.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3591 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3591
[FLASH]     Combined pairs:   482
[FLASH]     Uncombined pairs: 3109
[FLASH]     Percent combined: 13.42%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.028 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

482 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S39/S39_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S39_centroids.fasta --uc S39.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 482 sequences and identified 345 clusters (clustering threshold 1.00) - 12 clusters (representing 128 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S43 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S43_1.fastq /home/working/demultiplexed/S43_2.fastq S43_forward.paired.fastq.gz S43_forward.singletons.fastq.gz S43_reverse.paired.fastq.gz S43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S43_1.fastq /home/working/demultiplexed/S43_2.fastq S43_forward.paired.fastq.gz S43_forward.singletons.fastq.gz S43_reverse.paired.fastq.gz S43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:54:06 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S43/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9106 Both Surviving: 1630 (17.90%) Forward Only Surviving: 825 (9.06%) Reverse Only Surviving: 113 (1.24%) Dropped: 6538 (71.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: S43_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: S43_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S43_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S43_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S43_forward.paired.headclipped.fastq.gz S43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S43 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S43_forward.paired.headclipped.fastq.gz
[FLASH]     S43_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S43.extendedFrags.fastq.gz
[FLASH]     ./S43.notCombined_1.fastq.gz
[FLASH]     ./S43.notCombined_2.fastq.gz
[FLASH]     ./S43.hist
[FLASH]     ./S43.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1630 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1630
[FLASH]     Combined pairs:   1138
[FLASH]     Uncombined pairs: 492
[FLASH]     Percent combined: 69.82%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.032 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1138 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S43/S43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S43_centroids.fasta --uc S43.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1138 sequences and identified 567 clusters (clustering threshold 1.00) - 8 clusters (representing 479 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S50 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S50_1.fastq /home/working/demultiplexed/S50_2.fastq S50_forward.paired.fastq.gz S50_forward.singletons.fastq.gz S50_reverse.paired.fastq.gz S50_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S50_1.fastq /home/working/demultiplexed/S50_2.fastq S50_forward.paired.fastq.gz S50_forward.singletons.fastq.gz S50_reverse.paired.fastq.gz S50_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:54:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S50/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23841 Both Surviving: 5537 (23.22%) Forward Only Surviving: 1563 (6.56%) Reverse Only Surviving: 325 (1.36%) Dropped: 16416 (68.86%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: S50_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: S50_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: S50_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: S50_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S50_forward.paired.headclipped.fastq.gz S50_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S50 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S50_forward.paired.headclipped.fastq.gz
[FLASH]     S50_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S50.extendedFrags.fastq.gz
[FLASH]     ./S50.notCombined_1.fastq.gz
[FLASH]     ./S50.notCombined_2.fastq.gz
[FLASH]     ./S50.hist
[FLASH]     ./S50.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5537 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5537
[FLASH]     Combined pairs:   2823
[FLASH]     Uncombined pairs: 2714
[FLASH]     Percent combined: 50.98%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.080 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2823 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S50/S50_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S50_centroids.fasta --uc S50.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2823 sequences and identified 1523 clusters (clustering threshold 1.00) - 37 clusters (representing 1105 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S56 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S56_1.fastq /home/working/demultiplexed/S56_2.fastq S56_forward.paired.fastq.gz S56_forward.singletons.fastq.gz S56_reverse.paired.fastq.gz S56_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S56_1.fastq /home/working/demultiplexed/S56_2.fastq S56_forward.paired.fastq.gz S56_forward.singletons.fastq.gz S56_reverse.paired.fastq.gz S56_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:54:16 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S56/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: S56_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: S56_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S56_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S56_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S56_forward.paired.headclipped.fastq.gz S56_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S56 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S56_forward.paired.headclipped.fastq.gz
[FLASH]     S56_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S56.extendedFrags.fastq.gz
[FLASH]     ./S56.notCombined_1.fastq.gz
[FLASH]     ./S56.notCombined_2.fastq.gz
[FLASH]     ./S56.hist
[FLASH]     ./S56.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S56/S56_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S56_centroids.fasta --uc S56.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S59 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S59_1.fastq /home/working/demultiplexed/S59_2.fastq S59_forward.paired.fastq.gz S59_forward.singletons.fastq.gz S59_reverse.paired.fastq.gz S59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S59_1.fastq /home/working/demultiplexed/S59_2.fastq S59_forward.paired.fastq.gz S59_forward.singletons.fastq.gz S59_reverse.paired.fastq.gz S59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:54:17 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S59/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18121 Both Surviving: 3365 (18.57%) Forward Only Surviving: 1089 (6.01%) Reverse Only Surviving: 238 (1.31%) Dropped: 13429 (74.11%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: S59_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: S59_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: S59_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: S59_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S59_forward.paired.headclipped.fastq.gz S59_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S59 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S59_forward.paired.headclipped.fastq.gz
[FLASH]     S59_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S59.extendedFrags.fastq.gz
[FLASH]     ./S59.notCombined_1.fastq.gz
[FLASH]     ./S59.notCombined_2.fastq.gz
[FLASH]     ./S59.hist
[FLASH]     ./S59.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3365 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3365
[FLASH]     Combined pairs:   312
[FLASH]     Uncombined pairs: 3053
[FLASH]     Percent combined: 9.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.024 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

312 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S59/S59_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S59_centroids.fasta --uc S59.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 312 sequences and identified 237 clusters (clustering threshold 1.00) - 4 clusters (representing 65 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S62 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S62_1.fastq /home/working/demultiplexed/S62_2.fastq S62_forward.paired.fastq.gz S62_forward.singletons.fastq.gz S62_reverse.paired.fastq.gz S62_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S62_1.fastq /home/working/demultiplexed/S62_2.fastq S62_forward.paired.fastq.gz S62_forward.singletons.fastq.gz S62_reverse.paired.fastq.gz S62_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:54:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S62/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 39931 Both Surviving: 4343 (10.88%) Forward Only Surviving: 1632 (4.09%) Reverse Only Surviving: 405 (1.01%) Dropped: 33551 (84.02%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: S62_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: S62_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S62_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S62_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S62_forward.paired.headclipped.fastq.gz S62_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S62 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S62_forward.paired.headclipped.fastq.gz
[FLASH]     S62_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S62.extendedFrags.fastq.gz
[FLASH]     ./S62.notCombined_1.fastq.gz
[FLASH]     ./S62.notCombined_2.fastq.gz
[FLASH]     ./S62.hist
[FLASH]     ./S62.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4343 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4343
[FLASH]     Combined pairs:   1611
[FLASH]     Uncombined pairs: 2732
[FLASH]     Percent combined: 37.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.050 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1611 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S62/S62_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S62_centroids.fasta --uc S62.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1611 sequences and identified 697 clusters (clustering threshold 1.00) - 26 clusters (representing 796 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S63 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S63_1.fastq /home/working/demultiplexed/S63_2.fastq S63_forward.paired.fastq.gz S63_forward.singletons.fastq.gz S63_reverse.paired.fastq.gz S63_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S63_1.fastq /home/working/demultiplexed/S63_2.fastq S63_forward.paired.fastq.gz S63_forward.singletons.fastq.gz S63_reverse.paired.fastq.gz S63_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:54:27 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S63/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 6702 Both Surviving: 610 (9.10%) Forward Only Surviving: 253 (3.77%) Reverse Only Surviving: 80 (1.19%) Dropped: 5759 (85.93%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S63_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S63_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S63_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S63_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S63_forward.paired.headclipped.fastq.gz S63_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S63 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S63_forward.paired.headclipped.fastq.gz
[FLASH]     S63_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S63.extendedFrags.fastq.gz
[FLASH]     ./S63.notCombined_1.fastq.gz
[FLASH]     ./S63.notCombined_2.fastq.gz
[FLASH]     ./S63.hist
[FLASH]     ./S63.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 610 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      610
[FLASH]     Combined pairs:   540
[FLASH]     Uncombined pairs: 70
[FLASH]     Percent combined: 88.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.016 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

540 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S63/S63_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S63_centroids.fasta --uc S63.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 540 sequences and identified 341 clusters (clustering threshold 1.00) - 6 clusters (representing 176 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S65 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S65_1.fastq /home/working/demultiplexed/S65_2.fastq S65_forward.paired.fastq.gz S65_forward.singletons.fastq.gz S65_reverse.paired.fastq.gz S65_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S65_1.fastq /home/working/demultiplexed/S65_2.fastq S65_forward.paired.fastq.gz S65_forward.singletons.fastq.gz S65_reverse.paired.fastq.gz S65_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:54:29 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S65/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 7804 Both Surviving: 2569 (32.92%) Forward Only Surviving: 1071 (13.72%) Reverse Only Surviving: 88 (1.13%) Dropped: 4076 (52.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S65_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S65_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: S65_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: S65_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S65_forward.paired.headclipped.fastq.gz S65_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S65 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S65_forward.paired.headclipped.fastq.gz
[FLASH]     S65_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S65.extendedFrags.fastq.gz
[FLASH]     ./S65.notCombined_1.fastq.gz
[FLASH]     ./S65.notCombined_2.fastq.gz
[FLASH]     ./S65.hist
[FLASH]     ./S65.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2569 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2569
[FLASH]     Combined pairs:   2216
[FLASH]     Uncombined pairs: 353
[FLASH]     Percent combined: 86.26%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.060 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2216 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S65/S65_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S65_centroids.fasta --uc S65.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2216 sequences and identified 1018 clusters (clustering threshold 1.00) - 46 clusters (representing 1026 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S67 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S67_1.fastq /home/working/demultiplexed/S67_2.fastq S67_forward.paired.fastq.gz S67_forward.singletons.fastq.gz S67_reverse.paired.fastq.gz S67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S67_1.fastq /home/working/demultiplexed/S67_2.fastq S67_forward.paired.fastq.gz S67_forward.singletons.fastq.gz S67_reverse.paired.fastq.gz S67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:54:34 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S67/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18116 Both Surviving: 4019 (22.18%) Forward Only Surviving: 945 (5.22%) Reverse Only Surviving: 327 (1.81%) Dropped: 12825 (70.79%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: S67_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: S67_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S67_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S67_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S67_forward.paired.headclipped.fastq.gz S67_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S67 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S67_forward.paired.headclipped.fastq.gz
[FLASH]     S67_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S67.extendedFrags.fastq.gz
[FLASH]     ./S67.notCombined_1.fastq.gz
[FLASH]     ./S67.notCombined_2.fastq.gz
[FLASH]     ./S67.hist
[FLASH]     ./S67.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4019 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4019
[FLASH]     Combined pairs:   224
[FLASH]     Uncombined pairs: 3795
[FLASH]     Percent combined: 5.57%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.026 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

224 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S67/S67_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S67_centroids.fasta --uc S67.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 224 sequences and identified 174 clusters (clustering threshold 1.00) - 3 clusters (representing 37 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S7 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S7_1.fastq /home/working/demultiplexed/S7_2.fastq S7_forward.paired.fastq.gz S7_forward.singletons.fastq.gz S7_reverse.paired.fastq.gz S7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S7_1.fastq /home/working/demultiplexed/S7_2.fastq S7_forward.paired.fastq.gz S7_forward.singletons.fastq.gz S7_reverse.paired.fastq.gz S7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:54:39 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S7/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 32963 Both Surviving: 4645 (14.09%) Forward Only Surviving: 1505 (4.57%) Reverse Only Surviving: 245 (0.74%) Dropped: 26568 (80.60%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: S7_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: S7_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: S7_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: S7_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S7_forward.paired.headclipped.fastq.gz S7_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S7 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S7_forward.paired.headclipped.fastq.gz
[FLASH]     S7_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S7.extendedFrags.fastq.gz
[FLASH]     ./S7.notCombined_1.fastq.gz
[FLASH]     ./S7.notCombined_2.fastq.gz
[FLASH]     ./S7.hist
[FLASH]     ./S7.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4645 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4645
[FLASH]     Combined pairs:   4316
[FLASH]     Uncombined pairs: 329
[FLASH]     Percent combined: 92.92%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.115 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4316 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S7/S7_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S7_centroids.fasta --uc S7.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4316 sequences and identified 2427 clusters (clustering threshold 1.00) - 46 clusters (representing 1620 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S72 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S72_1.fastq /home/working/demultiplexed/S72_2.fastq S72_forward.paired.fastq.gz S72_forward.singletons.fastq.gz S72_reverse.paired.fastq.gz S72_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S72_1.fastq /home/working/demultiplexed/S72_2.fastq S72_forward.paired.fastq.gz S72_forward.singletons.fastq.gz S72_reverse.paired.fastq.gz S72_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:54:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S72/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 38851 Both Surviving: 115 (0.30%) Forward Only Surviving: 569 (1.46%) Reverse Only Surviving: 398 (1.02%) Dropped: 37769 (97.22%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S72_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S72_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: S72_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: S72_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S72_forward.paired.headclipped.fastq.gz S72_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S72 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S72_forward.paired.headclipped.fastq.gz
[FLASH]     S72_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S72.extendedFrags.fastq.gz
[FLASH]     ./S72.notCombined_1.fastq.gz
[FLASH]     ./S72.notCombined_2.fastq.gz
[FLASH]     ./S72.hist
[FLASH]     ./S72.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 115 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      115
[FLASH]     Combined pairs:   80
[FLASH]     Uncombined pairs: 35
[FLASH]     Percent combined: 69.57%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

80 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S72/S72_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S72_centroids.fasta --uc S72.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 80 sequences and identified 71 clusters (clustering threshold 1.00) - 1 clusters (representing 6 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S77 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S77_1.fastq /home/working/demultiplexed/S77_2.fastq S77_forward.paired.fastq.gz S77_forward.singletons.fastq.gz S77_reverse.paired.fastq.gz S77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S77_1.fastq /home/working/demultiplexed/S77_2.fastq S77_forward.paired.fastq.gz S77_forward.singletons.fastq.gz S77_reverse.paired.fastq.gz S77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:54:52 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S77/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14430 Both Surviving: 1819 (12.61%) Forward Only Surviving: 613 (4.25%) Reverse Only Surviving: 134 (0.93%) Dropped: 11864 (82.22%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: S77_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: S77_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S77_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S77_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S77_forward.paired.headclipped.fastq.gz S77_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S77 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S77_forward.paired.headclipped.fastq.gz
[FLASH]     S77_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S77.extendedFrags.fastq.gz
[FLASH]     ./S77.notCombined_1.fastq.gz
[FLASH]     ./S77.notCombined_2.fastq.gz
[FLASH]     ./S77.hist
[FLASH]     ./S77.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1819 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1819
[FLASH]     Combined pairs:   115
[FLASH]     Uncombined pairs: 1704
[FLASH]     Percent combined: 6.32%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.012 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

115 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S77/S77_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S77_centroids.fasta --uc S77.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 115 sequences and identified 101 clusters (clustering threshold 1.00) - 1 clusters (representing 10 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S79 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S79_1.fastq /home/working/demultiplexed/S79_2.fastq S79_forward.paired.fastq.gz S79_forward.singletons.fastq.gz S79_reverse.paired.fastq.gz S79_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S79_1.fastq /home/working/demultiplexed/S79_2.fastq S79_forward.paired.fastq.gz S79_forward.singletons.fastq.gz S79_reverse.paired.fastq.gz S79_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:55:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S79/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 143319 Both Surviving: 50529 (35.26%) Forward Only Surviving: 11833 (8.26%) Reverse Only Surviving: 2377 (1.66%) Dropped: 78580 (54.83%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: S79_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: S79_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: S79_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: S79_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S79_forward.paired.headclipped.fastq.gz S79_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S79 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S79_forward.paired.headclipped.fastq.gz
[FLASH]     S79_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S79.extendedFrags.fastq.gz
[FLASH]     ./S79.notCombined_1.fastq.gz
[FLASH]     ./S79.notCombined_2.fastq.gz
[FLASH]     ./S79.hist
[FLASH]     ./S79.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 50000 read pairs
[FLASH] Processed 50529 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      50529
[FLASH]     Combined pairs:   32082
[FLASH]     Uncombined pairs: 18447
[FLASH]     Percent combined: 63.49%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.861 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

32082 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S79/S79_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S79_centroids.fasta --uc S79.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 32082 sequences and identified 9396 clusters (clustering threshold 1.00) - 592 clusters (representing 21271 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S8 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S8_1.fastq /home/working/demultiplexed/S8_2.fastq S8_forward.paired.fastq.gz S8_forward.singletons.fastq.gz S8_reverse.paired.fastq.gz S8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S8_1.fastq /home/working/demultiplexed/S8_2.fastq S8_forward.paired.fastq.gz S8_forward.singletons.fastq.gz S8_reverse.paired.fastq.gz S8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:56:07 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S8/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26392 Both Surviving: 9081 (34.41%) Forward Only Surviving: 3915 (14.83%) Reverse Only Surviving: 236 (0.89%) Dropped: 13160 (49.86%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: S8_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: S8_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S8_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S8_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S8_forward.paired.headclipped.fastq.gz S8_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S8 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S8_forward.paired.headclipped.fastq.gz
[FLASH]     S8_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S8.extendedFrags.fastq.gz
[FLASH]     ./S8.notCombined_1.fastq.gz
[FLASH]     ./S8.notCombined_2.fastq.gz
[FLASH]     ./S8.hist
[FLASH]     ./S8.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9081 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9081
[FLASH]     Combined pairs:   5726
[FLASH]     Uncombined pairs: 3355
[FLASH]     Percent combined: 63.05%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.195 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5726 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S8/S8_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S8_centroids.fasta --uc S8.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5726 sequences and identified 2705 clusters (clustering threshold 1.00) - 81 clusters (representing 2588 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S80 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S80_1.fastq /home/working/demultiplexed/S80_2.fastq S80_forward.paired.fastq.gz S80_forward.singletons.fastq.gz S80_reverse.paired.fastq.gz S80_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S80_1.fastq /home/working/demultiplexed/S80_2.fastq S80_forward.paired.fastq.gz S80_forward.singletons.fastq.gz S80_reverse.paired.fastq.gz S80_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:56:19 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S80/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 10756 Both Surviving: 1785 (16.60%) Forward Only Surviving: 535 (4.97%) Reverse Only Surviving: 148 (1.38%) Dropped: 8288 (77.05%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: S80_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: S80_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S80_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S80_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S80_forward.paired.headclipped.fastq.gz S80_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S80 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S80_forward.paired.headclipped.fastq.gz
[FLASH]     S80_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S80.extendedFrags.fastq.gz
[FLASH]     ./S80.notCombined_1.fastq.gz
[FLASH]     ./S80.notCombined_2.fastq.gz
[FLASH]     ./S80.hist
[FLASH]     ./S80.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1785 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1785
[FLASH]     Combined pairs:   362
[FLASH]     Uncombined pairs: 1423
[FLASH]     Percent combined: 20.28%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.022 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

362 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S80/S80_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S80_centroids.fasta --uc S80.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 362 sequences and identified 234 clusters (clustering threshold 1.00) - 6 clusters (representing 121 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S81 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S81_1.fastq /home/working/demultiplexed/S81_2.fastq S81_forward.paired.fastq.gz S81_forward.singletons.fastq.gz S81_reverse.paired.fastq.gz S81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S81_1.fastq /home/working/demultiplexed/S81_2.fastq S81_forward.paired.fastq.gz S81_forward.singletons.fastq.gz S81_reverse.paired.fastq.gz S81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:56:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S81/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20291 Both Surviving: 373 (1.84%) Forward Only Surviving: 524 (2.58%) Reverse Only Surviving: 177 (0.87%) Dropped: 19217 (94.71%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S81_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S81_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S81_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S81_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S81_forward.paired.headclipped.fastq.gz S81_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S81 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S81_forward.paired.headclipped.fastq.gz
[FLASH]     S81_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S81.extendedFrags.fastq.gz
[FLASH]     ./S81.notCombined_1.fastq.gz
[FLASH]     ./S81.notCombined_2.fastq.gz
[FLASH]     ./S81.hist
[FLASH]     ./S81.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 373 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      373
[FLASH]     Combined pairs:   240
[FLASH]     Uncombined pairs: 133
[FLASH]     Percent combined: 64.34%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.008 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

240 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S81/S81_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S81_centroids.fasta --uc S81.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 240 sequences and identified 190 clusters (clustering threshold 1.00) - 3 clusters (representing 43 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S83 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S83_1.fastq /home/working/demultiplexed/S83_2.fastq S83_forward.paired.fastq.gz S83_forward.singletons.fastq.gz S83_reverse.paired.fastq.gz S83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S83_1.fastq /home/working/demultiplexed/S83_2.fastq S83_forward.paired.fastq.gz S83_forward.singletons.fastq.gz S83_reverse.paired.fastq.gz S83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:56:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S83/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 40119 Both Surviving: 17744 (44.23%) Forward Only Surviving: 8944 (22.29%) Reverse Only Surviving: 329 (0.82%) Dropped: 13102 (32.66%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: S83_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: S83_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S83_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S83_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S83_forward.paired.headclipped.fastq.gz S83_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S83 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S83_forward.paired.headclipped.fastq.gz
[FLASH]     S83_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S83.extendedFrags.fastq.gz
[FLASH]     ./S83.notCombined_1.fastq.gz
[FLASH]     ./S83.notCombined_2.fastq.gz
[FLASH]     ./S83.hist
[FLASH]     ./S83.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 17744 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      17744
[FLASH]     Combined pairs:   13157
[FLASH]     Uncombined pairs: 4587
[FLASH]     Percent combined: 74.15%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.365 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

13157 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S83/S83_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S83_centroids.fasta --uc S83.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 13157 sequences and identified 3794 clusters (clustering threshold 1.00) - 314 clusters (representing 9363 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S9 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S9_1.fastq /home/working/demultiplexed/S9_2.fastq S9_forward.paired.fastq.gz S9_forward.singletons.fastq.gz S9_reverse.paired.fastq.gz S9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S9_1.fastq /home/working/demultiplexed/S9_2.fastq S9_forward.paired.fastq.gz S9_forward.singletons.fastq.gz S9_reverse.paired.fastq.gz S9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:56:53 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S9/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41954 Both Surviving: 13198 (31.46%) Forward Only Surviving: 4239 (10.10%) Reverse Only Surviving: 607 (1.45%) Dropped: 23910 (56.99%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: S9_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: S9_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: S9_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: S9_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S9_forward.paired.headclipped.fastq.gz S9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S9 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S9_forward.paired.headclipped.fastq.gz
[FLASH]     S9_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S9.extendedFrags.fastq.gz
[FLASH]     ./S9.notCombined_1.fastq.gz
[FLASH]     ./S9.notCombined_2.fastq.gz
[FLASH]     ./S9.hist
[FLASH]     ./S9.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 13198 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      13198
[FLASH]     Combined pairs:   8987
[FLASH]     Uncombined pairs: 4211
[FLASH]     Percent combined: 68.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.414 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

8987 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S9/S9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S9_centroids.fasta --uc S9.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 8987 sequences and identified 2196 clusters (clustering threshold 1.00) - 235 clusters (representing 6715 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S90 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S90_1.fastq /home/working/demultiplexed/S90_2.fastq S90_forward.paired.fastq.gz S90_forward.singletons.fastq.gz S90_reverse.paired.fastq.gz S90_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S90_1.fastq /home/working/demultiplexed/S90_2.fastq S90_forward.paired.fastq.gz S90_forward.singletons.fastq.gz S90_reverse.paired.fastq.gz S90_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:57:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S90/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18970 Both Surviving: 5970 (31.47%) Forward Only Surviving: 1659 (8.75%) Reverse Only Surviving: 249 (1.31%) Dropped: 11092 (58.47%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: S90_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: S90_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S90_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S90_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S90_forward.paired.headclipped.fastq.gz S90_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S90 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S90_forward.paired.headclipped.fastq.gz
[FLASH]     S90_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S90.extendedFrags.fastq.gz
[FLASH]     ./S90.notCombined_1.fastq.gz
[FLASH]     ./S90.notCombined_2.fastq.gz
[FLASH]     ./S90.hist
[FLASH]     ./S90.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5970 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5970
[FLASH]     Combined pairs:   3361
[FLASH]     Uncombined pairs: 2609
[FLASH]     Percent combined: 56.30%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.129 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3361 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S90/S90_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S90_centroids.fasta --uc S90.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3361 sequences and identified 2119 clusters (clustering threshold 1.00) - 41 clusters (representing 1066 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S92 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S92_1.fastq /home/working/demultiplexed/S92_2.fastq S92_forward.paired.fastq.gz S92_forward.singletons.fastq.gz S92_reverse.paired.fastq.gz S92_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S92_1.fastq /home/working/demultiplexed/S92_2.fastq S92_forward.paired.fastq.gz S92_forward.singletons.fastq.gz S92_reverse.paired.fastq.gz S92_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:57:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S92/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 58656 Both Surviving: 3312 (5.65%) Forward Only Surviving: 2833 (4.83%) Reverse Only Surviving: 755 (1.29%) Dropped: 51756 (88.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S92_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S92_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S92_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S92_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S92_forward.paired.headclipped.fastq.gz S92_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S92 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S92_forward.paired.headclipped.fastq.gz
[FLASH]     S92_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S92.extendedFrags.fastq.gz
[FLASH]     ./S92.notCombined_1.fastq.gz
[FLASH]     ./S92.notCombined_2.fastq.gz
[FLASH]     ./S92.hist
[FLASH]     ./S92.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3312 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3312
[FLASH]     Combined pairs:   805
[FLASH]     Uncombined pairs: 2507
[FLASH]     Percent combined: 24.31%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.051 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

805 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S92/S92_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S92_centroids.fasta --uc S92.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 805 sequences and identified 575 clusters (clustering threshold 1.00) - 13 clusters (representing 167 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S96 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S96_1.fastq /home/working/demultiplexed/S96_2.fastq S96_forward.paired.fastq.gz S96_forward.singletons.fastq.gz S96_reverse.paired.fastq.gz S96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S96_1.fastq /home/working/demultiplexed/S96_2.fastq S96_forward.paired.fastq.gz S96_forward.singletons.fastq.gz S96_reverse.paired.fastq.gz S96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:57:33 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S96/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17331 Both Surviving: 2217 (12.79%) Forward Only Surviving: 807 (4.66%) Reverse Only Surviving: 207 (1.19%) Dropped: 14100 (81.36%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S96_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S96_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: S96_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: S96_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S96_forward.paired.headclipped.fastq.gz S96_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S96 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S96_forward.paired.headclipped.fastq.gz
[FLASH]     S96_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S96.extendedFrags.fastq.gz
[FLASH]     ./S96.notCombined_1.fastq.gz
[FLASH]     ./S96.notCombined_2.fastq.gz
[FLASH]     ./S96.hist
[FLASH]     ./S96.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2217 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2217
[FLASH]     Combined pairs:   2021
[FLASH]     Uncombined pairs: 196
[FLASH]     Percent combined: 91.16%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.056 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2021 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S96/S96_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S96_centroids.fasta --uc S96.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2021 sequences and identified 1156 clusters (clustering threshold 1.00) - 26 clusters (representing 799 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S97 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S97_1.fastq /home/working/demultiplexed/S97_2.fastq S97_forward.paired.fastq.gz S97_forward.singletons.fastq.gz S97_reverse.paired.fastq.gz S97_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S97_1.fastq /home/working/demultiplexed/S97_2.fastq S97_forward.paired.fastq.gz S97_forward.singletons.fastq.gz S97_reverse.paired.fastq.gz S97_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:57:39 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/S97/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 19611 Both Surviving: 5305 (27.05%) Forward Only Surviving: 1691 (8.62%) Reverse Only Surviving: 253 (1.29%) Dropped: 12362 (63.04%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: S97_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: S97_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S97_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S97_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S97_forward.paired.headclipped.fastq.gz S97_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S97 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S97_forward.paired.headclipped.fastq.gz
[FLASH]     S97_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S97.extendedFrags.fastq.gz
[FLASH]     ./S97.notCombined_1.fastq.gz
[FLASH]     ./S97.notCombined_2.fastq.gz
[FLASH]     ./S97.hist
[FLASH]     ./S97.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5305 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5305
[FLASH]     Combined pairs:   276
[FLASH]     Uncombined pairs: 5029
[FLASH]     Percent combined: 5.20%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.052 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

276 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/S97/S97_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S97_centroids.fasta --uc S97.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 276 sequences and identified 211 clusters (clustering threshold 1.00) - 5 clusters (representing 37 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA1 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA1_1.fastq /home/working/demultiplexed/SA1_2.fastq SA1_forward.paired.fastq.gz SA1_forward.singletons.fastq.gz SA1_reverse.paired.fastq.gz SA1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA1_1.fastq /home/working/demultiplexed/SA1_2.fastq SA1_forward.paired.fastq.gz SA1_forward.singletons.fastq.gz SA1_reverse.paired.fastq.gz SA1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:57:43 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA1/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 8479 Both Surviving: 396 (4.67%) Forward Only Surviving: 277 (3.27%) Reverse Only Surviving: 151 (1.78%) Dropped: 7655 (90.28%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA1_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA1_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA1_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA1_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA1_forward.paired.headclipped.fastq.gz SA1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA1 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA1_forward.paired.headclipped.fastq.gz
[FLASH]     SA1_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA1.extendedFrags.fastq.gz
[FLASH]     ./SA1.notCombined_1.fastq.gz
[FLASH]     ./SA1.notCombined_2.fastq.gz
[FLASH]     ./SA1.hist
[FLASH]     ./SA1.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 396 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      396
[FLASH]     Combined pairs:   329
[FLASH]     Uncombined pairs: 67
[FLASH]     Percent combined: 83.08%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.011 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

329 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA1/SA1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA1_centroids.fasta --uc SA1.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 329 sequences and identified 217 clusters (clustering threshold 1.00) - 10 clusters (representing 103 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA104 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA104_1.fastq /home/working/demultiplexed/SA104_2.fastq SA104_forward.paired.fastq.gz SA104_forward.singletons.fastq.gz SA104_reverse.paired.fastq.gz SA104_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA104_1.fastq /home/working/demultiplexed/SA104_2.fastq SA104_forward.paired.fastq.gz SA104_forward.singletons.fastq.gz SA104_reverse.paired.fastq.gz SA104_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:57:45 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA104/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 24043 Both Surviving: 3279 (13.64%) Forward Only Surviving: 915 (3.81%) Reverse Only Surviving: 431 (1.79%) Dropped: 19418 (80.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA104_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA104_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA104_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA104_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA104_forward.paired.headclipped.fastq.gz SA104_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA104 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA104_forward.paired.headclipped.fastq.gz
[FLASH]     SA104_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA104.extendedFrags.fastq.gz
[FLASH]     ./SA104.notCombined_1.fastq.gz
[FLASH]     ./SA104.notCombined_2.fastq.gz
[FLASH]     ./SA104.hist
[FLASH]     ./SA104.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3279 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3279
[FLASH]     Combined pairs:   82
[FLASH]     Uncombined pairs: 3197
[FLASH]     Percent combined: 2.50%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.041 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

82 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA104/SA104_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA104_centroids.fasta --uc SA104.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 82 sequences and identified 62 clusters (clustering threshold 1.00) - 2 clusters (representing 10 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA105 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA105_1.fastq /home/working/demultiplexed/SA105_2.fastq SA105_forward.paired.fastq.gz SA105_forward.singletons.fastq.gz SA105_reverse.paired.fastq.gz SA105_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA105_1.fastq /home/working/demultiplexed/SA105_2.fastq SA105_forward.paired.fastq.gz SA105_forward.singletons.fastq.gz SA105_reverse.paired.fastq.gz SA105_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:57:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA105/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23190 Both Surviving: 114 (0.49%) Forward Only Surviving: 242 (1.04%) Reverse Only Surviving: 165 (0.71%) Dropped: 22669 (97.75%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA105_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA105_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA105_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA105_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA105_forward.paired.headclipped.fastq.gz SA105_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA105 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA105_forward.paired.headclipped.fastq.gz
[FLASH]     SA105_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA105.extendedFrags.fastq.gz
[FLASH]     ./SA105.notCombined_1.fastq.gz
[FLASH]     ./SA105.notCombined_2.fastq.gz
[FLASH]     ./SA105.hist
[FLASH]     ./SA105.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 114 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      114
[FLASH]     Combined pairs:   85
[FLASH]     Uncombined pairs: 29
[FLASH]     Percent combined: 74.56%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.005 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

85 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA105/SA105_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA105_centroids.fasta --uc SA105.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 85 sequences and identified 59 clusters (clustering threshold 1.00) - 2 clusters (representing 24 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA109 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA109_1.fastq /home/working/demultiplexed/SA109_2.fastq SA109_forward.paired.fastq.gz SA109_forward.singletons.fastq.gz SA109_reverse.paired.fastq.gz SA109_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA109_1.fastq /home/working/demultiplexed/SA109_2.fastq SA109_forward.paired.fastq.gz SA109_forward.singletons.fastq.gz SA109_reverse.paired.fastq.gz SA109_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:57:51 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA109/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 6 Both Surviving: 1 (16.67%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 5 (83.33%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SA109_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA109_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA109_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA109_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA109_forward.paired.headclipped.fastq.gz SA109_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA109 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA109_forward.paired.headclipped.fastq.gz
[FLASH]     SA109_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA109.extendedFrags.fastq.gz
[FLASH]     ./SA109.notCombined_1.fastq.gz
[FLASH]     ./SA109.notCombined_2.fastq.gz
[FLASH]     ./SA109.hist
[FLASH]     ./SA109.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1
[FLASH]     Combined pairs:   1
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 100.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA109/SA109_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA109_centroids.fasta --uc SA109.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1 sequences and identified 1 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA111 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA111_1.fastq /home/working/demultiplexed/SA111_2.fastq SA111_forward.paired.fastq.gz SA111_forward.singletons.fastq.gz SA111_reverse.paired.fastq.gz SA111_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA111_1.fastq /home/working/demultiplexed/SA111_2.fastq SA111_forward.paired.fastq.gz SA111_forward.singletons.fastq.gz SA111_reverse.paired.fastq.gz SA111_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:57:52 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA111/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16872 Both Surviving: 3412 (20.22%) Forward Only Surviving: 1618 (9.59%) Reverse Only Surviving: 186 (1.10%) Dropped: 11656 (69.08%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA111_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA111_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA111_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA111_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA111_forward.paired.headclipped.fastq.gz SA111_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA111 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA111_forward.paired.headclipped.fastq.gz
[FLASH]     SA111_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA111.extendedFrags.fastq.gz
[FLASH]     ./SA111.notCombined_1.fastq.gz
[FLASH]     ./SA111.notCombined_2.fastq.gz
[FLASH]     ./SA111.hist
[FLASH]     ./SA111.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3412 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3412
[FLASH]     Combined pairs:   412
[FLASH]     Uncombined pairs: 3000
[FLASH]     Percent combined: 12.08%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.032 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

412 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA111/SA111_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA111_centroids.fasta --uc SA111.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 412 sequences and identified 319 clusters (clustering threshold 1.00) - 8 clusters (representing 56 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA112 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA112_1.fastq /home/working/demultiplexed/SA112_2.fastq SA112_forward.paired.fastq.gz SA112_forward.singletons.fastq.gz SA112_reverse.paired.fastq.gz SA112_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA112_1.fastq /home/working/demultiplexed/SA112_2.fastq SA112_forward.paired.fastq.gz SA112_forward.singletons.fastq.gz SA112_reverse.paired.fastq.gz SA112_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:57:59 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA112/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 86230 Both Surviving: 26880 (31.17%) Forward Only Surviving: 5576 (6.47%) Reverse Only Surviving: 3113 (3.61%) Dropped: 50661 (58.75%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SA112_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA112_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA112_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA112_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA112_forward.paired.headclipped.fastq.gz SA112_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA112 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA112_forward.paired.headclipped.fastq.gz
[FLASH]     SA112_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA112.extendedFrags.fastq.gz
[FLASH]     ./SA112.notCombined_1.fastq.gz
[FLASH]     ./SA112.notCombined_2.fastq.gz
[FLASH]     ./SA112.hist
[FLASH]     ./SA112.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 26880 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      26880
[FLASH]     Combined pairs:   5185
[FLASH]     Uncombined pairs: 21695
[FLASH]     Percent combined: 19.29%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.245 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5185 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA112/SA112_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA112_centroids.fasta --uc SA112.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5185 sequences and identified 2926 clusters (clustering threshold 1.00) - 76 clusters (representing 1962 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA113 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA113_1.fastq /home/working/demultiplexed/SA113_2.fastq SA113_forward.paired.fastq.gz SA113_forward.singletons.fastq.gz SA113_reverse.paired.fastq.gz SA113_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA113_1.fastq /home/working/demultiplexed/SA113_2.fastq SA113_forward.paired.fastq.gz SA113_forward.singletons.fastq.gz SA113_reverse.paired.fastq.gz SA113_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:58:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA113/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31186 Both Surviving: 6006 (19.26%) Forward Only Surviving: 1650 (5.29%) Reverse Only Surviving: 427 (1.37%) Dropped: 23103 (74.08%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA113_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA113_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA113_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA113_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA113_forward.paired.headclipped.fastq.gz SA113_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA113 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA113_forward.paired.headclipped.fastq.gz
[FLASH]     SA113_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA113.extendedFrags.fastq.gz
[FLASH]     ./SA113.notCombined_1.fastq.gz
[FLASH]     ./SA113.notCombined_2.fastq.gz
[FLASH]     ./SA113.hist
[FLASH]     ./SA113.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6006 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6006
[FLASH]     Combined pairs:   500
[FLASH]     Uncombined pairs: 5506
[FLASH]     Percent combined: 8.33%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.050 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

500 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA113/SA113_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA113_centroids.fasta --uc SA113.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 500 sequences and identified 338 clusters (clustering threshold 1.00) - 13 clusters (representing 149 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA114 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA114_1.fastq /home/working/demultiplexed/SA114_2.fastq SA114_forward.paired.fastq.gz SA114_forward.singletons.fastq.gz SA114_reverse.paired.fastq.gz SA114_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA114_1.fastq /home/working/demultiplexed/SA114_2.fastq SA114_forward.paired.fastq.gz SA114_forward.singletons.fastq.gz SA114_reverse.paired.fastq.gz SA114_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:58:33 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA114/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 133965 Both Surviving: 33203 (24.78%) Forward Only Surviving: 12322 (9.20%) Reverse Only Surviving: 1774 (1.32%) Dropped: 86666 (64.69%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SA114_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SA114_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SA114_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SA114_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA114_forward.paired.headclipped.fastq.gz SA114_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA114 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA114_forward.paired.headclipped.fastq.gz
[FLASH]     SA114_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA114.extendedFrags.fastq.gz
[FLASH]     ./SA114.notCombined_1.fastq.gz
[FLASH]     ./SA114.notCombined_2.fastq.gz
[FLASH]     ./SA114.hist
[FLASH]     ./SA114.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 33203 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      33203
[FLASH]     Combined pairs:   28702
[FLASH]     Uncombined pairs: 4501
[FLASH]     Percent combined: 86.44%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.749 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

28702 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA114/SA114_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA114_centroids.fasta --uc SA114.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 28702 sequences and identified 11395 clusters (clustering threshold 1.00) - 476 clusters (representing 16481 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA115 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA115_1.fastq /home/working/demultiplexed/SA115_2.fastq SA115_forward.paired.fastq.gz SA115_forward.singletons.fastq.gz SA115_reverse.paired.fastq.gz SA115_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA115_1.fastq /home/working/demultiplexed/SA115_2.fastq SA115_forward.paired.fastq.gz SA115_forward.singletons.fastq.gz SA115_reverse.paired.fastq.gz SA115_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:59:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA115/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 21323 Both Surviving: 323 (1.51%) Forward Only Surviving: 345 (1.62%) Reverse Only Surviving: 232 (1.09%) Dropped: 20423 (95.78%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA115_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA115_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA115_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA115_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA115_forward.paired.headclipped.fastq.gz SA115_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA115 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA115_forward.paired.headclipped.fastq.gz
[FLASH]     SA115_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA115.extendedFrags.fastq.gz
[FLASH]     ./SA115.notCombined_1.fastq.gz
[FLASH]     ./SA115.notCombined_2.fastq.gz
[FLASH]     ./SA115.hist
[FLASH]     ./SA115.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 323 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      323
[FLASH]     Combined pairs:   79
[FLASH]     Uncombined pairs: 244
[FLASH]     Percent combined: 24.46%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

79 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA115/SA115_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA115_centroids.fasta --uc SA115.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 79 sequences and identified 61 clusters (clustering threshold 1.00) - 2 clusters (representing 15 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA116 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA116_1.fastq /home/working/demultiplexed/SA116_2.fastq SA116_forward.paired.fastq.gz SA116_forward.singletons.fastq.gz SA116_reverse.paired.fastq.gz SA116_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA116_1.fastq /home/working/demultiplexed/SA116_2.fastq SA116_forward.paired.fastq.gz SA116_forward.singletons.fastq.gz SA116_reverse.paired.fastq.gz SA116_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:59:33 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA116/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31818 Both Surviving: 2035 (6.40%) Forward Only Surviving: 881 (2.77%) Reverse Only Surviving: 589 (1.85%) Dropped: 28313 (88.98%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA116_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA116_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA116_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA116_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA116_forward.paired.headclipped.fastq.gz SA116_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA116 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA116_forward.paired.headclipped.fastq.gz
[FLASH]     SA116_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA116.extendedFrags.fastq.gz
[FLASH]     ./SA116.notCombined_1.fastq.gz
[FLASH]     ./SA116.notCombined_2.fastq.gz
[FLASH]     ./SA116.hist
[FLASH]     ./SA116.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2035 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2035
[FLASH]     Combined pairs:   13
[FLASH]     Uncombined pairs: 2022
[FLASH]     Percent combined: 0.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.020 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

13 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA116/SA116_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA116_centroids.fasta --uc SA116.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 13 sequences and identified 13 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA117 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA117_1.fastq /home/working/demultiplexed/SA117_2.fastq SA117_forward.paired.fastq.gz SA117_forward.singletons.fastq.gz SA117_reverse.paired.fastq.gz SA117_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA117_1.fastq /home/working/demultiplexed/SA117_2.fastq SA117_forward.paired.fastq.gz SA117_forward.singletons.fastq.gz SA117_reverse.paired.fastq.gz SA117_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:59:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA117/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37714 Both Surviving: 7464 (19.79%) Forward Only Surviving: 1783 (4.73%) Reverse Only Surviving: 467 (1.24%) Dropped: 28000 (74.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA117_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA117_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA117_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA117_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA117_forward.paired.headclipped.fastq.gz SA117_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA117 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA117_forward.paired.headclipped.fastq.gz
[FLASH]     SA117_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA117.extendedFrags.fastq.gz
[FLASH]     ./SA117.notCombined_1.fastq.gz
[FLASH]     ./SA117.notCombined_2.fastq.gz
[FLASH]     ./SA117.hist
[FLASH]     ./SA117.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7464 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7464
[FLASH]     Combined pairs:   1516
[FLASH]     Uncombined pairs: 5948
[FLASH]     Percent combined: 20.31%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.059 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1516 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA117/SA117_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA117_centroids.fasta --uc SA117.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1516 sequences and identified 932 clusters (clustering threshold 1.00) - 19 clusters (representing 523 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA119 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA119_1.fastq /home/working/demultiplexed/SA119_2.fastq SA119_forward.paired.fastq.gz SA119_forward.singletons.fastq.gz SA119_reverse.paired.fastq.gz SA119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA119_1.fastq /home/working/demultiplexed/SA119_2.fastq SA119_forward.paired.fastq.gz SA119_forward.singletons.fastq.gz SA119_reverse.paired.fastq.gz SA119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:59:46 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA119/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 76452 Both Surviving: 12333 (16.13%) Forward Only Surviving: 3713 (4.86%) Reverse Only Surviving: 2374 (3.11%) Dropped: 58032 (75.91%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SA119_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA119_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SA119_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA119_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA119_forward.paired.headclipped.fastq.gz SA119_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA119 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA119_forward.paired.headclipped.fastq.gz
[FLASH]     SA119_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA119.extendedFrags.fastq.gz
[FLASH]     ./SA119.notCombined_1.fastq.gz
[FLASH]     ./SA119.notCombined_2.fastq.gz
[FLASH]     ./SA119.hist
[FLASH]     ./SA119.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 12333 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      12333
[FLASH]     Combined pairs:   548
[FLASH]     Uncombined pairs: 11785
[FLASH]     Percent combined: 4.44%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.087 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

548 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA119/SA119_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA119_centroids.fasta --uc SA119.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 548 sequences and identified 429 clusters (clustering threshold 1.00) - 3 clusters (representing 86 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA120 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA120_1.fastq /home/working/demultiplexed/SA120_2.fastq SA120_forward.paired.fastq.gz SA120_forward.singletons.fastq.gz SA120_reverse.paired.fastq.gz SA120_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA120_1.fastq /home/working/demultiplexed/SA120_2.fastq SA120_forward.paired.fastq.gz SA120_forward.singletons.fastq.gz SA120_reverse.paired.fastq.gz SA120_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 4:59:56 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA120/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26355 Both Surviving: 872 (3.31%) Forward Only Surviving: 734 (2.79%) Reverse Only Surviving: 223 (0.85%) Dropped: 24526 (93.06%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA120_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA120_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA120_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA120_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA120_forward.paired.headclipped.fastq.gz SA120_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA120 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA120_forward.paired.headclipped.fastq.gz
[FLASH]     SA120_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA120.extendedFrags.fastq.gz
[FLASH]     ./SA120.notCombined_1.fastq.gz
[FLASH]     ./SA120.notCombined_2.fastq.gz
[FLASH]     ./SA120.hist
[FLASH]     ./SA120.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 872 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      872
[FLASH]     Combined pairs:   669
[FLASH]     Uncombined pairs: 203
[FLASH]     Percent combined: 76.72%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.018 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

669 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA120/SA120_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA120_centroids.fasta --uc SA120.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 669 sequences and identified 419 clusters (clustering threshold 1.00) - 15 clusters (representing 234 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA121 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA121_1.fastq /home/working/demultiplexed/SA121_2.fastq SA121_forward.paired.fastq.gz SA121_forward.singletons.fastq.gz SA121_reverse.paired.fastq.gz SA121_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA121_1.fastq /home/working/demultiplexed/SA121_2.fastq SA121_forward.paired.fastq.gz SA121_forward.singletons.fastq.gz SA121_reverse.paired.fastq.gz SA121_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:00:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA121/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 50866 Both Surviving: 10952 (21.53%) Forward Only Surviving: 3441 (6.76%) Reverse Only Surviving: 729 (1.43%) Dropped: 35744 (70.27%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SA121_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SA121_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA121_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA121_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA121_forward.paired.headclipped.fastq.gz SA121_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA121 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA121_forward.paired.headclipped.fastq.gz
[FLASH]     SA121_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA121.extendedFrags.fastq.gz
[FLASH]     ./SA121.notCombined_1.fastq.gz
[FLASH]     ./SA121.notCombined_2.fastq.gz
[FLASH]     ./SA121.hist
[FLASH]     ./SA121.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10952 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10952
[FLASH]     Combined pairs:   1060
[FLASH]     Uncombined pairs: 9892
[FLASH]     Percent combined: 9.68%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.070 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1060 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA121/SA121_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA121_centroids.fasta --uc SA121.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1060 sequences and identified 715 clusters (clustering threshold 1.00) - 17 clusters (representing 283 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA122 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA122_1.fastq /home/working/demultiplexed/SA122_2.fastq SA122_forward.paired.fastq.gz SA122_forward.singletons.fastq.gz SA122_reverse.paired.fastq.gz SA122_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA122_1.fastq /home/working/demultiplexed/SA122_2.fastq SA122_forward.paired.fastq.gz SA122_forward.singletons.fastq.gz SA122_reverse.paired.fastq.gz SA122_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:00:09 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA122/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 13391 Both Surviving: 4648 (34.71%) Forward Only Surviving: 940 (7.02%) Reverse Only Surviving: 201 (1.50%) Dropped: 7602 (56.77%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SA122_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SA122_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA122_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA122_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA122_forward.paired.headclipped.fastq.gz SA122_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA122 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA122_forward.paired.headclipped.fastq.gz
[FLASH]     SA122_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA122.extendedFrags.fastq.gz
[FLASH]     ./SA122.notCombined_1.fastq.gz
[FLASH]     ./SA122.notCombined_2.fastq.gz
[FLASH]     ./SA122.hist
[FLASH]     ./SA122.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4648 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4648
[FLASH]     Combined pairs:   2464
[FLASH]     Uncombined pairs: 2184
[FLASH]     Percent combined: 53.01%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.074 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2464 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA122/SA122_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA122_centroids.fasta --uc SA122.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2464 sequences and identified 1373 clusters (clustering threshold 1.00) - 56 clusters (representing 941 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA124 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA124_1.fastq /home/working/demultiplexed/SA124_2.fastq SA124_forward.paired.fastq.gz SA124_forward.singletons.fastq.gz SA124_reverse.paired.fastq.gz SA124_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA124_1.fastq /home/working/demultiplexed/SA124_2.fastq SA124_forward.paired.fastq.gz SA124_forward.singletons.fastq.gz SA124_reverse.paired.fastq.gz SA124_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:00:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA124/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SA124_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA124_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA124_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA124_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA124_forward.paired.headclipped.fastq.gz SA124_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA124 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA124_forward.paired.headclipped.fastq.gz
[FLASH]     SA124_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA124.extendedFrags.fastq.gz
[FLASH]     ./SA124.notCombined_1.fastq.gz
[FLASH]     ./SA124.notCombined_2.fastq.gz
[FLASH]     ./SA124.hist
[FLASH]     ./SA124.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA124/SA124_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA124_centroids.fasta --uc SA124.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA125 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA125_1.fastq /home/working/demultiplexed/SA125_2.fastq SA125_forward.paired.fastq.gz SA125_forward.singletons.fastq.gz SA125_reverse.paired.fastq.gz SA125_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA125_1.fastq /home/working/demultiplexed/SA125_2.fastq SA125_forward.paired.fastq.gz SA125_forward.singletons.fastq.gz SA125_reverse.paired.fastq.gz SA125_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:00:19 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA125/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 84537 Both Surviving: 26226 (31.02%) Forward Only Surviving: 8061 (9.54%) Reverse Only Surviving: 1377 (1.63%) Dropped: 48873 (57.81%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA125_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA125_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA125_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA125_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA125_forward.paired.headclipped.fastq.gz SA125_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA125 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA125_forward.paired.headclipped.fastq.gz
[FLASH]     SA125_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA125.extendedFrags.fastq.gz
[FLASH]     ./SA125.notCombined_1.fastq.gz
[FLASH]     ./SA125.notCombined_2.fastq.gz
[FLASH]     ./SA125.hist
[FLASH]     ./SA125.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 26226 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      26226
[FLASH]     Combined pairs:   6724
[FLASH]     Uncombined pairs: 19502
[FLASH]     Percent combined: 25.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.252 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6724 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA125/SA125_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA125_centroids.fasta --uc SA125.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6724 sequences and identified 2696 clusters (clustering threshold 1.00) - 109 clusters (representing 3473 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA126 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA126_1.fastq /home/working/demultiplexed/SA126_2.fastq SA126_forward.paired.fastq.gz SA126_forward.singletons.fastq.gz SA126_reverse.paired.fastq.gz SA126_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA126_1.fastq /home/working/demultiplexed/SA126_2.fastq SA126_forward.paired.fastq.gz SA126_forward.singletons.fastq.gz SA126_reverse.paired.fastq.gz SA126_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:00:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA126/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 43041 Both Surviving: 12938 (30.06%) Forward Only Surviving: 3091 (7.18%) Reverse Only Surviving: 878 (2.04%) Dropped: 26134 (60.72%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SA126_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SA126_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA126_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA126_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA126_forward.paired.headclipped.fastq.gz SA126_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA126 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA126_forward.paired.headclipped.fastq.gz
[FLASH]     SA126_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA126.extendedFrags.fastq.gz
[FLASH]     ./SA126.notCombined_1.fastq.gz
[FLASH]     ./SA126.notCombined_2.fastq.gz
[FLASH]     ./SA126.hist
[FLASH]     ./SA126.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 12938 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      12938
[FLASH]     Combined pairs:   568
[FLASH]     Uncombined pairs: 12370
[FLASH]     Percent combined: 4.39%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.074 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

568 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA126/SA126_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA126_centroids.fasta --uc SA126.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 568 sequences and identified 417 clusters (clustering threshold 1.00) - 7 clusters (representing 105 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA128 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA128_1.fastq /home/working/demultiplexed/SA128_2.fastq SA128_forward.paired.fastq.gz SA128_forward.singletons.fastq.gz SA128_reverse.paired.fastq.gz SA128_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA128_1.fastq /home/working/demultiplexed/SA128_2.fastq SA128_forward.paired.fastq.gz SA128_forward.singletons.fastq.gz SA128_reverse.paired.fastq.gz SA128_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:00:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA128/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 10561 Both Surviving: 3860 (36.55%) Forward Only Surviving: 809 (7.66%) Reverse Only Surviving: 188 (1.78%) Dropped: 5704 (54.01%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA128_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA128_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA128_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA128_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA128_forward.paired.headclipped.fastq.gz SA128_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA128 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA128_forward.paired.headclipped.fastq.gz
[FLASH]     SA128_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA128.extendedFrags.fastq.gz
[FLASH]     ./SA128.notCombined_1.fastq.gz
[FLASH]     ./SA128.notCombined_2.fastq.gz
[FLASH]     ./SA128.hist
[FLASH]     ./SA128.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3860 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3860
[FLASH]     Combined pairs:   2075
[FLASH]     Uncombined pairs: 1785
[FLASH]     Percent combined: 53.76%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.060 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2075 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA128/SA128_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA128_centroids.fasta --uc SA128.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2075 sequences and identified 1191 clusters (clustering threshold 1.00) - 23 clusters (representing 769 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA13 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA13_1.fastq /home/working/demultiplexed/SA13_2.fastq SA13_forward.paired.fastq.gz SA13_forward.singletons.fastq.gz SA13_reverse.paired.fastq.gz SA13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA13_1.fastq /home/working/demultiplexed/SA13_2.fastq SA13_forward.paired.fastq.gz SA13_forward.singletons.fastq.gz SA13_reverse.paired.fastq.gz SA13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:00:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA13/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31253 Both Surviving: 10666 (34.13%) Forward Only Surviving: 2268 (7.26%) Reverse Only Surviving: 484 (1.55%) Dropped: 17835 (57.07%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA13_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA13_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA13_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA13_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA13_forward.paired.headclipped.fastq.gz SA13_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA13 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA13_forward.paired.headclipped.fastq.gz
[FLASH]     SA13_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA13.extendedFrags.fastq.gz
[FLASH]     ./SA13.notCombined_1.fastq.gz
[FLASH]     ./SA13.notCombined_2.fastq.gz
[FLASH]     ./SA13.hist
[FLASH]     ./SA13.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10666 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10666
[FLASH]     Combined pairs:   6139
[FLASH]     Uncombined pairs: 4527
[FLASH]     Percent combined: 57.56%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.172 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6139 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA13/SA13_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA13_centroids.fasta --uc SA13.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6139 sequences and identified 3472 clusters (clustering threshold 1.00) - 98 clusters (representing 2365 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA130 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA130_1.fastq /home/working/demultiplexed/SA130_2.fastq SA130_forward.paired.fastq.gz SA130_forward.singletons.fastq.gz SA130_reverse.paired.fastq.gz SA130_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA130_1.fastq /home/working/demultiplexed/SA130_2.fastq SA130_forward.paired.fastq.gz SA130_forward.singletons.fastq.gz SA130_reverse.paired.fastq.gz SA130_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:01:08 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA130/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20540 Both Surviving: 3652 (17.78%) Forward Only Surviving: 800 (3.89%) Reverse Only Surviving: 646 (3.15%) Dropped: 15442 (75.18%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SA130_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA130_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA130_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA130_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA130_forward.paired.headclipped.fastq.gz SA130_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA130 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA130_forward.paired.headclipped.fastq.gz
[FLASH]     SA130_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA130.extendedFrags.fastq.gz
[FLASH]     ./SA130.notCombined_1.fastq.gz
[FLASH]     ./SA130.notCombined_2.fastq.gz
[FLASH]     ./SA130.hist
[FLASH]     ./SA130.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3652 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3652
[FLASH]     Combined pairs:   514
[FLASH]     Uncombined pairs: 3138
[FLASH]     Percent combined: 14.07%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.025 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

514 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA130/SA130_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA130_centroids.fasta --uc SA130.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 514 sequences and identified 284 clusters (clustering threshold 1.00) - 7 clusters (representing 218 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA132 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA132_1.fastq /home/working/demultiplexed/SA132_2.fastq SA132_forward.paired.fastq.gz SA132_forward.singletons.fastq.gz SA132_reverse.paired.fastq.gz SA132_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA132_1.fastq /home/working/demultiplexed/SA132_2.fastq SA132_forward.paired.fastq.gz SA132_forward.singletons.fastq.gz SA132_reverse.paired.fastq.gz SA132_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:01:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA132/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 5677 Both Surviving: 1186 (20.89%) Forward Only Surviving: 263 (4.63%) Reverse Only Surviving: 195 (3.43%) Dropped: 4033 (71.04%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SA132_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA132_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA132_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA132_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA132_forward.paired.headclipped.fastq.gz SA132_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA132 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA132_forward.paired.headclipped.fastq.gz
[FLASH]     SA132_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA132.extendedFrags.fastq.gz
[FLASH]     ./SA132.notCombined_1.fastq.gz
[FLASH]     ./SA132.notCombined_2.fastq.gz
[FLASH]     ./SA132.hist
[FLASH]     ./SA132.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1186 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1186
[FLASH]     Combined pairs:   455
[FLASH]     Uncombined pairs: 731
[FLASH]     Percent combined: 38.36%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.014 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

455 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA132/SA132_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA132_centroids.fasta --uc SA132.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 455 sequences and identified 274 clusters (clustering threshold 1.00) - 7 clusters (representing 159 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA138 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA138_1.fastq /home/working/demultiplexed/SA138_2.fastq SA138_forward.paired.fastq.gz SA138_forward.singletons.fastq.gz SA138_reverse.paired.fastq.gz SA138_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA138_1.fastq /home/working/demultiplexed/SA138_2.fastq SA138_forward.paired.fastq.gz SA138_forward.singletons.fastq.gz SA138_reverse.paired.fastq.gz SA138_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:01:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA138/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16117 Both Surviving: 648 (4.02%) Forward Only Surviving: 393 (2.44%) Reverse Only Surviving: 272 (1.69%) Dropped: 14804 (91.85%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA138_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA138_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA138_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA138_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA138_forward.paired.headclipped.fastq.gz SA138_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA138 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA138_forward.paired.headclipped.fastq.gz
[FLASH]     SA138_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA138.extendedFrags.fastq.gz
[FLASH]     ./SA138.notCombined_1.fastq.gz
[FLASH]     ./SA138.notCombined_2.fastq.gz
[FLASH]     ./SA138.hist
[FLASH]     ./SA138.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 648 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      648
[FLASH]     Combined pairs:   332
[FLASH]     Uncombined pairs: 316
[FLASH]     Percent combined: 51.23%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.010 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

332 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA138/SA138_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA138_centroids.fasta --uc SA138.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 332 sequences and identified 196 clusters (clustering threshold 1.00) - 7 clusters (representing 135 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA140 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA140_1.fastq /home/working/demultiplexed/SA140_2.fastq SA140_forward.paired.fastq.gz SA140_forward.singletons.fastq.gz SA140_reverse.paired.fastq.gz SA140_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA140_1.fastq /home/working/demultiplexed/SA140_2.fastq SA140_forward.paired.fastq.gz SA140_forward.singletons.fastq.gz SA140_reverse.paired.fastq.gz SA140_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:01:20 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA140/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 81551 Both Surviving: 8759 (10.74%) Forward Only Surviving: 2778 (3.41%) Reverse Only Surviving: 1121 (1.37%) Dropped: 68893 (84.48%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA140_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA140_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA140_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA140_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA140_forward.paired.headclipped.fastq.gz SA140_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA140 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA140_forward.paired.headclipped.fastq.gz
[FLASH]     SA140_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA140.extendedFrags.fastq.gz
[FLASH]     ./SA140.notCombined_1.fastq.gz
[FLASH]     ./SA140.notCombined_2.fastq.gz
[FLASH]     ./SA140.hist
[FLASH]     ./SA140.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8759 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8759
[FLASH]     Combined pairs:   242
[FLASH]     Uncombined pairs: 8517
[FLASH]     Percent combined: 2.76%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.050 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

242 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA140/SA140_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA140_centroids.fasta --uc SA140.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 242 sequences and identified 192 clusters (clustering threshold 1.00) - 3 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA144 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA144_1.fastq /home/working/demultiplexed/SA144_2.fastq SA144_forward.paired.fastq.gz SA144_forward.singletons.fastq.gz SA144_reverse.paired.fastq.gz SA144_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA144_1.fastq /home/working/demultiplexed/SA144_2.fastq SA144_forward.paired.fastq.gz SA144_forward.singletons.fastq.gz SA144_reverse.paired.fastq.gz SA144_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:01:27 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA144/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28135 Both Surviving: 7735 (27.49%) Forward Only Surviving: 2906 (10.33%) Reverse Only Surviving: 328 (1.17%) Dropped: 17166 (61.01%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA144_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA144_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA144_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA144_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA144_forward.paired.headclipped.fastq.gz SA144_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA144 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA144_forward.paired.headclipped.fastq.gz
[FLASH]     SA144_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA144.extendedFrags.fastq.gz
[FLASH]     ./SA144.notCombined_1.fastq.gz
[FLASH]     ./SA144.notCombined_2.fastq.gz
[FLASH]     ./SA144.hist
[FLASH]     ./SA144.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7735 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7735
[FLASH]     Combined pairs:   6523
[FLASH]     Uncombined pairs: 1212
[FLASH]     Percent combined: 84.33%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.181 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6523 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA144/SA144_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA144_centroids.fasta --uc SA144.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6523 sequences and identified 3140 clusters (clustering threshold 1.00) - 131 clusters (representing 3022 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA147 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA147_1.fastq /home/working/demultiplexed/SA147_2.fastq SA147_forward.paired.fastq.gz SA147_forward.singletons.fastq.gz SA147_reverse.paired.fastq.gz SA147_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA147_1.fastq /home/working/demultiplexed/SA147_2.fastq SA147_forward.paired.fastq.gz SA147_forward.singletons.fastq.gz SA147_reverse.paired.fastq.gz SA147_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:01:42 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA147/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 52843 Both Surviving: 9208 (17.43%) Forward Only Surviving: 2868 (5.43%) Reverse Only Surviving: 630 (1.19%) Dropped: 40137 (75.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA147_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA147_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA147_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA147_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA147_forward.paired.headclipped.fastq.gz SA147_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA147 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA147_forward.paired.headclipped.fastq.gz
[FLASH]     SA147_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA147.extendedFrags.fastq.gz
[FLASH]     ./SA147.notCombined_1.fastq.gz
[FLASH]     ./SA147.notCombined_2.fastq.gz
[FLASH]     ./SA147.hist
[FLASH]     ./SA147.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9208 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9208
[FLASH]     Combined pairs:   3170
[FLASH]     Uncombined pairs: 6038
[FLASH]     Percent combined: 34.43%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.108 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3170 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA147/SA147_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA147_centroids.fasta --uc SA147.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3170 sequences and identified 1383 clusters (clustering threshold 1.00) - 67 clusters (representing 1555 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA15 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA15_1.fastq /home/working/demultiplexed/SA15_2.fastq SA15_forward.paired.fastq.gz SA15_forward.singletons.fastq.gz SA15_reverse.paired.fastq.gz SA15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA15_1.fastq /home/working/demultiplexed/SA15_2.fastq SA15_forward.paired.fastq.gz SA15_forward.singletons.fastq.gz SA15_reverse.paired.fastq.gz SA15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:01:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA15/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 86082 Both Surviving: 26672 (30.98%) Forward Only Surviving: 6078 (7.06%) Reverse Only Surviving: 1725 (2.00%) Dropped: 51607 (59.95%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA15_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA15_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA15_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA15_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA15_forward.paired.headclipped.fastq.gz SA15_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA15 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA15_forward.paired.headclipped.fastq.gz
[FLASH]     SA15_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA15.extendedFrags.fastq.gz
[FLASH]     ./SA15.notCombined_1.fastq.gz
[FLASH]     ./SA15.notCombined_2.fastq.gz
[FLASH]     ./SA15.hist
[FLASH]     ./SA15.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 26672 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      26672
[FLASH]     Combined pairs:   889
[FLASH]     Uncombined pairs: 25783
[FLASH]     Percent combined: 3.33%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.153 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

889 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA15/SA15_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA15_centroids.fasta --uc SA15.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 889 sequences and identified 586 clusters (clustering threshold 1.00) - 20 clusters (representing 252 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA153 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA153_1.fastq /home/working/demultiplexed/SA153_2.fastq SA153_forward.paired.fastq.gz SA153_forward.singletons.fastq.gz SA153_reverse.paired.fastq.gz SA153_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA153_1.fastq /home/working/demultiplexed/SA153_2.fastq SA153_forward.paired.fastq.gz SA153_forward.singletons.fastq.gz SA153_reverse.paired.fastq.gz SA153_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:02:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA153/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 54014 Both Surviving: 11939 (22.10%) Forward Only Surviving: 2234 (4.14%) Reverse Only Surviving: 992 (1.84%) Dropped: 38849 (71.92%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA153_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA153_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA153_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA153_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA153_forward.paired.headclipped.fastq.gz SA153_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA153 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA153_forward.paired.headclipped.fastq.gz
[FLASH]     SA153_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA153.extendedFrags.fastq.gz
[FLASH]     ./SA153.notCombined_1.fastq.gz
[FLASH]     ./SA153.notCombined_2.fastq.gz
[FLASH]     ./SA153.hist
[FLASH]     ./SA153.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11939 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11939
[FLASH]     Combined pairs:   1679
[FLASH]     Uncombined pairs: 10260
[FLASH]     Percent combined: 14.06%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.072 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1679 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA153/SA153_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA153_centroids.fasta --uc SA153.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1679 sequences and identified 1046 clusters (clustering threshold 1.00) - 29 clusters (representing 564 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA155 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA155_1.fastq /home/working/demultiplexed/SA155_2.fastq SA155_forward.paired.fastq.gz SA155_forward.singletons.fastq.gz SA155_reverse.paired.fastq.gz SA155_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA155_1.fastq /home/working/demultiplexed/SA155_2.fastq SA155_forward.paired.fastq.gz SA155_forward.singletons.fastq.gz SA155_reverse.paired.fastq.gz SA155_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:02:20 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA155/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31128 Both Surviving: 752 (2.42%) Forward Only Surviving: 717 (2.30%) Reverse Only Surviving: 480 (1.54%) Dropped: 29179 (93.74%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA155_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA155_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA155_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA155_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA155_forward.paired.headclipped.fastq.gz SA155_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA155 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA155_forward.paired.headclipped.fastq.gz
[FLASH]     SA155_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA155.extendedFrags.fastq.gz
[FLASH]     ./SA155.notCombined_1.fastq.gz
[FLASH]     ./SA155.notCombined_2.fastq.gz
[FLASH]     ./SA155.hist
[FLASH]     ./SA155.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 752 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      752
[FLASH]     Combined pairs:   561
[FLASH]     Uncombined pairs: 191
[FLASH]     Percent combined: 74.60%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.023 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

561 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA155/SA155_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA155_centroids.fasta --uc SA155.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 561 sequences and identified 321 clusters (clustering threshold 1.00) - 8 clusters (representing 201 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA156 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA156_1.fastq /home/working/demultiplexed/SA156_2.fastq SA156_forward.paired.fastq.gz SA156_forward.singletons.fastq.gz SA156_reverse.paired.fastq.gz SA156_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA156_1.fastq /home/working/demultiplexed/SA156_2.fastq SA156_forward.paired.fastq.gz SA156_forward.singletons.fastq.gz SA156_reverse.paired.fastq.gz SA156_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:02:25 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA156/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 53417 Both Surviving: 2365 (4.43%) Forward Only Surviving: 1836 (3.44%) Reverse Only Surviving: 713 (1.33%) Dropped: 48503 (90.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA156_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA156_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA156_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA156_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA156_forward.paired.headclipped.fastq.gz SA156_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA156 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA156_forward.paired.headclipped.fastq.gz
[FLASH]     SA156_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA156.extendedFrags.fastq.gz
[FLASH]     ./SA156.notCombined_1.fastq.gz
[FLASH]     ./SA156.notCombined_2.fastq.gz
[FLASH]     ./SA156.hist
[FLASH]     ./SA156.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2365 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2365
[FLASH]     Combined pairs:   312
[FLASH]     Uncombined pairs: 2053
[FLASH]     Percent combined: 13.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.021 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

312 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA156/SA156_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA156_centroids.fasta --uc SA156.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 312 sequences and identified 258 clusters (clustering threshold 1.00) - 3 clusters (representing 21 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA157 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA157_1.fastq /home/working/demultiplexed/SA157_2.fastq SA157_forward.paired.fastq.gz SA157_forward.singletons.fastq.gz SA157_reverse.paired.fastq.gz SA157_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA157_1.fastq /home/working/demultiplexed/SA157_2.fastq SA157_forward.paired.fastq.gz SA157_forward.singletons.fastq.gz SA157_reverse.paired.fastq.gz SA157_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:02:31 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA157/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37950 Both Surviving: 6388 (16.83%) Forward Only Surviving: 1776 (4.68%) Reverse Only Surviving: 666 (1.75%) Dropped: 29120 (76.73%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA157_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA157_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA157_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA157_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA157_forward.paired.headclipped.fastq.gz SA157_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA157 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA157_forward.paired.headclipped.fastq.gz
[FLASH]     SA157_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA157.extendedFrags.fastq.gz
[FLASH]     ./SA157.notCombined_1.fastq.gz
[FLASH]     ./SA157.notCombined_2.fastq.gz
[FLASH]     ./SA157.hist
[FLASH]     ./SA157.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6388 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6388
[FLASH]     Combined pairs:   41
[FLASH]     Uncombined pairs: 6347
[FLASH]     Percent combined: 0.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.040 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

41 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA157/SA157_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA157_centroids.fasta --uc SA157.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 41 sequences and identified 35 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA160 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA160_1.fastq /home/working/demultiplexed/SA160_2.fastq SA160_forward.paired.fastq.gz SA160_forward.singletons.fastq.gz SA160_reverse.paired.fastq.gz SA160_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA160_1.fastq /home/working/demultiplexed/SA160_2.fastq SA160_forward.paired.fastq.gz SA160_forward.singletons.fastq.gz SA160_reverse.paired.fastq.gz SA160_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:02:36 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA160/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 39762 Both Surviving: 6875 (17.29%) Forward Only Surviving: 1725 (4.34%) Reverse Only Surviving: 618 (1.55%) Dropped: 30544 (76.82%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA160_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA160_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA160_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA160_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA160_forward.paired.headclipped.fastq.gz SA160_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA160 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA160_forward.paired.headclipped.fastq.gz
[FLASH]     SA160_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA160.extendedFrags.fastq.gz
[FLASH]     ./SA160.notCombined_1.fastq.gz
[FLASH]     ./SA160.notCombined_2.fastq.gz
[FLASH]     ./SA160.hist
[FLASH]     ./SA160.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6875 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6875
[FLASH]     Combined pairs:   758
[FLASH]     Uncombined pairs: 6117
[FLASH]     Percent combined: 11.03%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.041 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

758 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA160/SA160_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA160_centroids.fasta --uc SA160.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 758 sequences and identified 367 clusters (clustering threshold 1.00) - 14 clusters (representing 367 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA162 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA162_1.fastq /home/working/demultiplexed/SA162_2.fastq SA162_forward.paired.fastq.gz SA162_forward.singletons.fastq.gz SA162_reverse.paired.fastq.gz SA162_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA162_1.fastq /home/working/demultiplexed/SA162_2.fastq SA162_forward.paired.fastq.gz SA162_forward.singletons.fastq.gz SA162_reverse.paired.fastq.gz SA162_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:02:43 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA162/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37273 Both Surviving: 11447 (30.71%) Forward Only Surviving: 3157 (8.47%) Reverse Only Surviving: 439 (1.18%) Dropped: 22230 (59.64%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA162_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA162_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SA162_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA162_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA162_forward.paired.headclipped.fastq.gz SA162_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA162 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA162_forward.paired.headclipped.fastq.gz
[FLASH]     SA162_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA162.extendedFrags.fastq.gz
[FLASH]     ./SA162.notCombined_1.fastq.gz
[FLASH]     ./SA162.notCombined_2.fastq.gz
[FLASH]     ./SA162.hist
[FLASH]     ./SA162.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11447 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11447
[FLASH]     Combined pairs:   5798
[FLASH]     Uncombined pairs: 5649
[FLASH]     Percent combined: 50.65%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.164 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5798 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA162/SA162_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA162_centroids.fasta --uc SA162.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5798 sequences and identified 2921 clusters (clustering threshold 1.00) - 79 clusters (representing 2543 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA164 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA164_1.fastq /home/working/demultiplexed/SA164_2.fastq SA164_forward.paired.fastq.gz SA164_forward.singletons.fastq.gz SA164_reverse.paired.fastq.gz SA164_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA164_1.fastq /home/working/demultiplexed/SA164_2.fastq SA164_forward.paired.fastq.gz SA164_forward.singletons.fastq.gz SA164_reverse.paired.fastq.gz SA164_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:02:56 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA164/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SA164_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA164_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA164_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA164_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA164_forward.paired.headclipped.fastq.gz SA164_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA164 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA164_forward.paired.headclipped.fastq.gz
[FLASH]     SA164_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA164.extendedFrags.fastq.gz
[FLASH]     ./SA164.notCombined_1.fastq.gz
[FLASH]     ./SA164.notCombined_2.fastq.gz
[FLASH]     ./SA164.hist
[FLASH]     ./SA164.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA164/SA164_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA164_centroids.fasta --uc SA164.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA165 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA165_1.fastq /home/working/demultiplexed/SA165_2.fastq SA165_forward.paired.fastq.gz SA165_forward.singletons.fastq.gz SA165_reverse.paired.fastq.gz SA165_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA165_1.fastq /home/working/demultiplexed/SA165_2.fastq SA165_forward.paired.fastq.gz SA165_forward.singletons.fastq.gz SA165_reverse.paired.fastq.gz SA165_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:02:58 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA165/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31845 Both Surviving: 9829 (30.87%) Forward Only Surviving: 2198 (6.90%) Reverse Only Surviving: 609 (1.91%) Dropped: 19209 (60.32%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA165_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA165_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA165_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA165_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA165_forward.paired.headclipped.fastq.gz SA165_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA165 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA165_forward.paired.headclipped.fastq.gz
[FLASH]     SA165_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA165.extendedFrags.fastq.gz
[FLASH]     ./SA165.notCombined_1.fastq.gz
[FLASH]     ./SA165.notCombined_2.fastq.gz
[FLASH]     ./SA165.hist
[FLASH]     ./SA165.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9829 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9829
[FLASH]     Combined pairs:   4629
[FLASH]     Uncombined pairs: 5200
[FLASH]     Percent combined: 47.10%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.144 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4629 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA165/SA165_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA165_centroids.fasta --uc SA165.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4629 sequences and identified 2550 clusters (clustering threshold 1.00) - 59 clusters (representing 1820 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA166 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA166_1.fastq /home/working/demultiplexed/SA166_2.fastq SA166_forward.paired.fastq.gz SA166_forward.singletons.fastq.gz SA166_reverse.paired.fastq.gz SA166_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA166_1.fastq /home/working/demultiplexed/SA166_2.fastq SA166_forward.paired.fastq.gz SA166_forward.singletons.fastq.gz SA166_reverse.paired.fastq.gz SA166_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:03:09 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA166/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22760 Both Surviving: 9143 (40.17%) Forward Only Surviving: 1993 (8.76%) Reverse Only Surviving: 322 (1.41%) Dropped: 11302 (49.66%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA166_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA166_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA166_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA166_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA166_forward.paired.headclipped.fastq.gz SA166_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA166 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA166_forward.paired.headclipped.fastq.gz
[FLASH]     SA166_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA166.extendedFrags.fastq.gz
[FLASH]     ./SA166.notCombined_1.fastq.gz
[FLASH]     ./SA166.notCombined_2.fastq.gz
[FLASH]     ./SA166.hist
[FLASH]     ./SA166.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9143 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9143
[FLASH]     Combined pairs:   4628
[FLASH]     Uncombined pairs: 4515
[FLASH]     Percent combined: 50.62%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.140 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4628 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA166/SA166_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA166_centroids.fasta --uc SA166.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4628 sequences and identified 2605 clusters (clustering threshold 1.00) - 110 clusters (representing 1757 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA171 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA171_1.fastq /home/working/demultiplexed/SA171_2.fastq SA171_forward.paired.fastq.gz SA171_forward.singletons.fastq.gz SA171_reverse.paired.fastq.gz SA171_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA171_1.fastq /home/working/demultiplexed/SA171_2.fastq SA171_forward.paired.fastq.gz SA171_forward.singletons.fastq.gz SA171_reverse.paired.fastq.gz SA171_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:03:19 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA171/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 7014 Both Surviving: 2316 (33.02%) Forward Only Surviving: 562 (8.01%) Reverse Only Surviving: 193 (2.75%) Dropped: 3943 (56.22%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SA171_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA171_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA171_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA171_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA171_forward.paired.headclipped.fastq.gz SA171_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA171 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA171_forward.paired.headclipped.fastq.gz
[FLASH]     SA171_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA171.extendedFrags.fastq.gz
[FLASH]     ./SA171.notCombined_1.fastq.gz
[FLASH]     ./SA171.notCombined_2.fastq.gz
[FLASH]     ./SA171.hist
[FLASH]     ./SA171.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2316 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2316
[FLASH]     Combined pairs:   456
[FLASH]     Uncombined pairs: 1860
[FLASH]     Percent combined: 19.69%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.023 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

456 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA171/SA171_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA171_centroids.fasta --uc SA171.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 456 sequences and identified 336 clusters (clustering threshold 1.00) - 8 clusters (representing 110 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA172 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA172_1.fastq /home/working/demultiplexed/SA172_2.fastq SA172_forward.paired.fastq.gz SA172_forward.singletons.fastq.gz SA172_reverse.paired.fastq.gz SA172_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA172_1.fastq /home/working/demultiplexed/SA172_2.fastq SA172_forward.paired.fastq.gz SA172_forward.singletons.fastq.gz SA172_reverse.paired.fastq.gz SA172_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:03:23 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA172/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29978 Both Surviving: 9181 (30.63%) Forward Only Surviving: 3718 (12.40%) Reverse Only Surviving: 329 (1.10%) Dropped: 16750 (55.87%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA172_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA172_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA172_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA172_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA172_forward.paired.headclipped.fastq.gz SA172_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA172 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA172_forward.paired.headclipped.fastq.gz
[FLASH]     SA172_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA172.extendedFrags.fastq.gz
[FLASH]     ./SA172.notCombined_1.fastq.gz
[FLASH]     ./SA172.notCombined_2.fastq.gz
[FLASH]     ./SA172.hist
[FLASH]     ./SA172.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9181 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9181
[FLASH]     Combined pairs:   4464
[FLASH]     Uncombined pairs: 4717
[FLASH]     Percent combined: 48.62%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.128 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4464 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA172/SA172_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA172_centroids.fasta --uc SA172.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4464 sequences and identified 1339 clusters (clustering threshold 1.00) - 127 clusters (representing 2965 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA174 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA174_1.fastq /home/working/demultiplexed/SA174_2.fastq SA174_forward.paired.fastq.gz SA174_forward.singletons.fastq.gz SA174_reverse.paired.fastq.gz SA174_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA174_1.fastq /home/working/demultiplexed/SA174_2.fastq SA174_forward.paired.fastq.gz SA174_forward.singletons.fastq.gz SA174_reverse.paired.fastq.gz SA174_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:03:33 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA174/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 24531 Both Surviving: 7385 (30.10%) Forward Only Surviving: 1759 (7.17%) Reverse Only Surviving: 279 (1.14%) Dropped: 15108 (61.59%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA174_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA174_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA174_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA174_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA174_forward.paired.headclipped.fastq.gz SA174_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA174 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA174_forward.paired.headclipped.fastq.gz
[FLASH]     SA174_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA174.extendedFrags.fastq.gz
[FLASH]     ./SA174.notCombined_1.fastq.gz
[FLASH]     ./SA174.notCombined_2.fastq.gz
[FLASH]     ./SA174.hist
[FLASH]     ./SA174.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7385 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7385
[FLASH]     Combined pairs:   2705
[FLASH]     Uncombined pairs: 4680
[FLASH]     Percent combined: 36.63%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.082 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2705 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA174/SA174_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA174_centroids.fasta --uc SA174.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2705 sequences and identified 1744 clusters (clustering threshold 1.00) - 31 clusters (representing 808 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA177 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA177_1.fastq /home/working/demultiplexed/SA177_2.fastq SA177_forward.paired.fastq.gz SA177_forward.singletons.fastq.gz SA177_reverse.paired.fastq.gz SA177_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA177_1.fastq /home/working/demultiplexed/SA177_2.fastq SA177_forward.paired.fastq.gz SA177_forward.singletons.fastq.gz SA177_reverse.paired.fastq.gz SA177_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:03:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA177/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18782 Both Surviving: 385 (2.05%) Forward Only Surviving: 385 (2.05%) Reverse Only Surviving: 127 (0.68%) Dropped: 17885 (95.22%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA177_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA177_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA177_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA177_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA177_forward.paired.headclipped.fastq.gz SA177_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA177 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA177_forward.paired.headclipped.fastq.gz
[FLASH]     SA177_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA177.extendedFrags.fastq.gz
[FLASH]     ./SA177.notCombined_1.fastq.gz
[FLASH]     ./SA177.notCombined_2.fastq.gz
[FLASH]     ./SA177.hist
[FLASH]     ./SA177.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 385 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      385
[FLASH]     Combined pairs:   258
[FLASH]     Uncombined pairs: 127
[FLASH]     Percent combined: 67.01%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.008 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

258 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA177/SA177_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA177_centroids.fasta --uc SA177.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 258 sequences and identified 176 clusters (clustering threshold 1.00) - 3 clusters (representing 63 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA180 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA180_1.fastq /home/working/demultiplexed/SA180_2.fastq SA180_forward.paired.fastq.gz SA180_forward.singletons.fastq.gz SA180_reverse.paired.fastq.gz SA180_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA180_1.fastq /home/working/demultiplexed/SA180_2.fastq SA180_forward.paired.fastq.gz SA180_forward.singletons.fastq.gz SA180_reverse.paired.fastq.gz SA180_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:03:43 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA180/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15643 Both Surviving: 717 (4.58%) Forward Only Surviving: 310 (1.98%) Reverse Only Surviving: 175 (1.12%) Dropped: 14441 (92.32%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA180_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA180_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA180_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA180_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA180_forward.paired.headclipped.fastq.gz SA180_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA180 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA180_forward.paired.headclipped.fastq.gz
[FLASH]     SA180_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA180.extendedFrags.fastq.gz
[FLASH]     ./SA180.notCombined_1.fastq.gz
[FLASH]     ./SA180.notCombined_2.fastq.gz
[FLASH]     ./SA180.hist
[FLASH]     ./SA180.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 717 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      717
[FLASH]     Combined pairs:   223
[FLASH]     Uncombined pairs: 494
[FLASH]     Percent combined: 31.10%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.007 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

223 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA180/SA180_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA180_centroids.fasta --uc SA180.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 223 sequences and identified 142 clusters (clustering threshold 1.00) - 2 clusters (representing 70 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA182 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA182_1.fastq /home/working/demultiplexed/SA182_2.fastq SA182_forward.paired.fastq.gz SA182_forward.singletons.fastq.gz SA182_reverse.paired.fastq.gz SA182_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA182_1.fastq /home/working/demultiplexed/SA182_2.fastq SA182_forward.paired.fastq.gz SA182_forward.singletons.fastq.gz SA182_reverse.paired.fastq.gz SA182_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:03:45 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA182/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14801 Both Surviving: 2276 (15.38%) Forward Only Surviving: 487 (3.29%) Reverse Only Surviving: 441 (2.98%) Dropped: 11597 (78.35%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SA182_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA182_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA182_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA182_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA182_forward.paired.headclipped.fastq.gz SA182_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA182 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA182_forward.paired.headclipped.fastq.gz
[FLASH]     SA182_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA182.extendedFrags.fastq.gz
[FLASH]     ./SA182.notCombined_1.fastq.gz
[FLASH]     ./SA182.notCombined_2.fastq.gz
[FLASH]     ./SA182.hist
[FLASH]     ./SA182.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2276 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2276
[FLASH]     Combined pairs:   1194
[FLASH]     Uncombined pairs: 1082
[FLASH]     Percent combined: 52.46%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.033 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1194 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA182/SA182_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA182_centroids.fasta --uc SA182.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1194 sequences and identified 669 clusters (clustering threshold 1.00) - 16 clusters (representing 465 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA189 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA189_1.fastq /home/working/demultiplexed/SA189_2.fastq SA189_forward.paired.fastq.gz SA189_forward.singletons.fastq.gz SA189_reverse.paired.fastq.gz SA189_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA189_1.fastq /home/working/demultiplexed/SA189_2.fastq SA189_forward.paired.fastq.gz SA189_forward.singletons.fastq.gz SA189_reverse.paired.fastq.gz SA189_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:03:51 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA189/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 40281 Both Surviving: 6626 (16.45%) Forward Only Surviving: 1736 (4.31%) Reverse Only Surviving: 589 (1.46%) Dropped: 31330 (77.78%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA189_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA189_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA189_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA189_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA189_forward.paired.headclipped.fastq.gz SA189_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA189 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA189_forward.paired.headclipped.fastq.gz
[FLASH]     SA189_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA189.extendedFrags.fastq.gz
[FLASH]     ./SA189.notCombined_1.fastq.gz
[FLASH]     ./SA189.notCombined_2.fastq.gz
[FLASH]     ./SA189.hist
[FLASH]     ./SA189.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6626 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6626
[FLASH]     Combined pairs:   1042
[FLASH]     Uncombined pairs: 5584
[FLASH]     Percent combined: 15.73%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.042 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1042 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA189/SA189_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA189_centroids.fasta --uc SA189.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1042 sequences and identified 562 clusters (clustering threshold 1.00) - 14 clusters (representing 431 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA19 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA19_1.fastq /home/working/demultiplexed/SA19_2.fastq SA19_forward.paired.fastq.gz SA19_forward.singletons.fastq.gz SA19_reverse.paired.fastq.gz SA19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA19_1.fastq /home/working/demultiplexed/SA19_2.fastq SA19_forward.paired.fastq.gz SA19_forward.singletons.fastq.gz SA19_reverse.paired.fastq.gz SA19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:04:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA19/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 97463 Both Surviving: 21686 (22.25%) Forward Only Surviving: 6419 (6.59%) Reverse Only Surviving: 1547 (1.59%) Dropped: 67811 (69.58%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA19_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA19_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA19_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA19_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA19_forward.paired.headclipped.fastq.gz SA19_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA19 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA19_forward.paired.headclipped.fastq.gz
[FLASH]     SA19_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA19.extendedFrags.fastq.gz
[FLASH]     ./SA19.notCombined_1.fastq.gz
[FLASH]     ./SA19.notCombined_2.fastq.gz
[FLASH]     ./SA19.hist
[FLASH]     ./SA19.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 21686 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      21686
[FLASH]     Combined pairs:   1454
[FLASH]     Uncombined pairs: 20232
[FLASH]     Percent combined: 6.70%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.124 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1454 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA19/SA19_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA19_centroids.fasta --uc SA19.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1454 sequences and identified 925 clusters (clustering threshold 1.00) - 33 clusters (representing 447 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA190 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA190_1.fastq /home/working/demultiplexed/SA190_2.fastq SA190_forward.paired.fastq.gz SA190_forward.singletons.fastq.gz SA190_reverse.paired.fastq.gz SA190_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA190_1.fastq /home/working/demultiplexed/SA190_2.fastq SA190_forward.paired.fastq.gz SA190_forward.singletons.fastq.gz SA190_reverse.paired.fastq.gz SA190_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:04:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA190/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 57953 Both Surviving: 708 (1.22%) Forward Only Surviving: 1425 (2.46%) Reverse Only Surviving: 656 (1.13%) Dropped: 55164 (95.19%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA190_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA190_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA190_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA190_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA190_forward.paired.headclipped.fastq.gz SA190_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA190 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA190_forward.paired.headclipped.fastq.gz
[FLASH]     SA190_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA190.extendedFrags.fastq.gz
[FLASH]     ./SA190.notCombined_1.fastq.gz
[FLASH]     ./SA190.notCombined_2.fastq.gz
[FLASH]     ./SA190.hist
[FLASH]     ./SA190.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 708 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      708
[FLASH]     Combined pairs:   189
[FLASH]     Uncombined pairs: 519
[FLASH]     Percent combined: 26.69%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.010 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

189 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA190/SA190_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA190_centroids.fasta --uc SA190.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 189 sequences and identified 153 clusters (clustering threshold 1.00) - 1 clusters (representing 8 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA196 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA196_1.fastq /home/working/demultiplexed/SA196_2.fastq SA196_forward.paired.fastq.gz SA196_forward.singletons.fastq.gz SA196_reverse.paired.fastq.gz SA196_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA196_1.fastq /home/working/demultiplexed/SA196_2.fastq SA196_forward.paired.fastq.gz SA196_forward.singletons.fastq.gz SA196_reverse.paired.fastq.gz SA196_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:04:20 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA196/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 30293 Both Surviving: 431 (1.42%) Forward Only Surviving: 604 (1.99%) Reverse Only Surviving: 436 (1.44%) Dropped: 28822 (95.14%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA196_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA196_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA196_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA196_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA196_forward.paired.headclipped.fastq.gz SA196_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA196 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA196_forward.paired.headclipped.fastq.gz
[FLASH]     SA196_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA196.extendedFrags.fastq.gz
[FLASH]     ./SA196.notCombined_1.fastq.gz
[FLASH]     ./SA196.notCombined_2.fastq.gz
[FLASH]     ./SA196.hist
[FLASH]     ./SA196.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 431 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      431
[FLASH]     Combined pairs:   304
[FLASH]     Uncombined pairs: 127
[FLASH]     Percent combined: 70.53%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.016 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

304 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA196/SA196_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA196_centroids.fasta --uc SA196.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 304 sequences and identified 156 clusters (clustering threshold 1.00) - 6 clusters (representing 143 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA198 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA198_1.fastq /home/working/demultiplexed/SA198_2.fastq SA198_forward.paired.fastq.gz SA198_forward.singletons.fastq.gz SA198_reverse.paired.fastq.gz SA198_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA198_1.fastq /home/working/demultiplexed/SA198_2.fastq SA198_forward.paired.fastq.gz SA198_forward.singletons.fastq.gz SA198_reverse.paired.fastq.gz SA198_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:04:23 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA198/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14709 Both Surviving: 3771 (25.64%) Forward Only Surviving: 1102 (7.49%) Reverse Only Surviving: 201 (1.37%) Dropped: 9635 (65.50%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA198_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA198_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA198_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA198_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA198_forward.paired.headclipped.fastq.gz SA198_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA198 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA198_forward.paired.headclipped.fastq.gz
[FLASH]     SA198_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA198.extendedFrags.fastq.gz
[FLASH]     ./SA198.notCombined_1.fastq.gz
[FLASH]     ./SA198.notCombined_2.fastq.gz
[FLASH]     ./SA198.hist
[FLASH]     ./SA198.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3771 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3771
[FLASH]     Combined pairs:   660
[FLASH]     Uncombined pairs: 3111
[FLASH]     Percent combined: 17.50%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.027 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

660 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA198/SA198_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA198_centroids.fasta --uc SA198.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 660 sequences and identified 452 clusters (clustering threshold 1.00) - 12 clusters (representing 189 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA199 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA199_1.fastq /home/working/demultiplexed/SA199_2.fastq SA199_forward.paired.fastq.gz SA199_forward.singletons.fastq.gz SA199_reverse.paired.fastq.gz SA199_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA199_1.fastq /home/working/demultiplexed/SA199_2.fastq SA199_forward.paired.fastq.gz SA199_forward.singletons.fastq.gz SA199_reverse.paired.fastq.gz SA199_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:04:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA199/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 1 Both Surviving: 0 (0.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 1 (100.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SA199_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA199_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA199_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA199_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA199_forward.paired.headclipped.fastq.gz SA199_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA199 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA199_forward.paired.headclipped.fastq.gz
[FLASH]     SA199_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA199.extendedFrags.fastq.gz
[FLASH]     ./SA199.notCombined_1.fastq.gz
[FLASH]     ./SA199.notCombined_2.fastq.gz
[FLASH]     ./SA199.hist
[FLASH]     ./SA199.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA199/SA199_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA199_centroids.fasta --uc SA199.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA2 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA2_1.fastq /home/working/demultiplexed/SA2_2.fastq SA2_forward.paired.fastq.gz SA2_forward.singletons.fastq.gz SA2_reverse.paired.fastq.gz SA2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA2_1.fastq /home/working/demultiplexed/SA2_2.fastq SA2_forward.paired.fastq.gz SA2_forward.singletons.fastq.gz SA2_reverse.paired.fastq.gz SA2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:04:32 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA2/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 96310 Both Surviving: 30413 (31.58%) Forward Only Surviving: 7502 (7.79%) Reverse Only Surviving: 1434 (1.49%) Dropped: 56961 (59.14%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA2_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA2_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA2_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA2_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA2_forward.paired.headclipped.fastq.gz SA2_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA2 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA2_forward.paired.headclipped.fastq.gz
[FLASH]     SA2_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA2.extendedFrags.fastq.gz
[FLASH]     ./SA2.notCombined_1.fastq.gz
[FLASH]     ./SA2.notCombined_2.fastq.gz
[FLASH]     ./SA2.hist
[FLASH]     ./SA2.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 30413 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      30413
[FLASH]     Combined pairs:   18029
[FLASH]     Uncombined pairs: 12384
[FLASH]     Percent combined: 59.28%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.478 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

18029 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA2/SA2_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA2_centroids.fasta --uc SA2.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 18029 sequences and identified 5296 clusters (clustering threshold 1.00) - 434 clusters (representing 12156 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA202 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA202_1.fastq /home/working/demultiplexed/SA202_2.fastq SA202_forward.paired.fastq.gz SA202_forward.singletons.fastq.gz SA202_reverse.paired.fastq.gz SA202_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA202_1.fastq /home/working/demultiplexed/SA202_2.fastq SA202_forward.paired.fastq.gz SA202_forward.singletons.fastq.gz SA202_reverse.paired.fastq.gz SA202_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:05:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA202/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 102107 Both Surviving: 26482 (25.94%) Forward Only Surviving: 7417 (7.26%) Reverse Only Surviving: 1638 (1.60%) Dropped: 66570 (65.20%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA202_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA202_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SA202_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA202_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA202_forward.paired.headclipped.fastq.gz SA202_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA202 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA202_forward.paired.headclipped.fastq.gz
[FLASH]     SA202_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA202.extendedFrags.fastq.gz
[FLASH]     ./SA202.notCombined_1.fastq.gz
[FLASH]     ./SA202.notCombined_2.fastq.gz
[FLASH]     ./SA202.hist
[FLASH]     ./SA202.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 26482 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      26482
[FLASH]     Combined pairs:   2971
[FLASH]     Uncombined pairs: 23511
[FLASH]     Percent combined: 11.22%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.207 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2971 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA202/SA202_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA202_centroids.fasta --uc SA202.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2971 sequences and identified 1577 clusters (clustering threshold 1.00) - 41 clusters (representing 1228 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA21 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA21_1.fastq /home/working/demultiplexed/SA21_2.fastq SA21_forward.paired.fastq.gz SA21_forward.singletons.fastq.gz SA21_reverse.paired.fastq.gz SA21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA21_1.fastq /home/working/demultiplexed/SA21_2.fastq SA21_forward.paired.fastq.gz SA21_forward.singletons.fastq.gz SA21_reverse.paired.fastq.gz SA21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:05:31 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA21/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37070 Both Surviving: 5135 (13.85%) Forward Only Surviving: 1558 (4.20%) Reverse Only Surviving: 623 (1.68%) Dropped: 29754 (80.26%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA21_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA21_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA21_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA21_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA21_forward.paired.headclipped.fastq.gz SA21_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA21 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA21_forward.paired.headclipped.fastq.gz
[FLASH]     SA21_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA21.extendedFrags.fastq.gz
[FLASH]     ./SA21.notCombined_1.fastq.gz
[FLASH]     ./SA21.notCombined_2.fastq.gz
[FLASH]     ./SA21.hist
[FLASH]     ./SA21.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5135 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5135
[FLASH]     Combined pairs:   56
[FLASH]     Uncombined pairs: 5079
[FLASH]     Percent combined: 1.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.049 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

56 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA21/SA21_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA21_centroids.fasta --uc SA21.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 56 sequences and identified 49 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA28 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA28_1.fastq /home/working/demultiplexed/SA28_2.fastq SA28_forward.paired.fastq.gz SA28_forward.singletons.fastq.gz SA28_reverse.paired.fastq.gz SA28_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA28_1.fastq /home/working/demultiplexed/SA28_2.fastq SA28_forward.paired.fastq.gz SA28_forward.singletons.fastq.gz SA28_reverse.paired.fastq.gz SA28_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:05:38 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA28/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 55630 Both Surviving: 13372 (24.04%) Forward Only Surviving: 4577 (8.23%) Reverse Only Surviving: 769 (1.38%) Dropped: 36912 (66.35%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA28_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA28_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA28_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA28_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA28_forward.paired.headclipped.fastq.gz SA28_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA28 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA28_forward.paired.headclipped.fastq.gz
[FLASH]     SA28_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA28.extendedFrags.fastq.gz
[FLASH]     ./SA28.notCombined_1.fastq.gz
[FLASH]     ./SA28.notCombined_2.fastq.gz
[FLASH]     ./SA28.hist
[FLASH]     ./SA28.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 13372 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      13372
[FLASH]     Combined pairs:   2894
[FLASH]     Uncombined pairs: 10478
[FLASH]     Percent combined: 21.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.123 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2894 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA28/SA28_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA28_centroids.fasta --uc SA28.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2894 sequences and identified 1162 clusters (clustering threshold 1.00) - 33 clusters (representing 1432 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA31 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA31_1.fastq /home/working/demultiplexed/SA31_2.fastq SA31_forward.paired.fastq.gz SA31_forward.singletons.fastq.gz SA31_reverse.paired.fastq.gz SA31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA31_1.fastq /home/working/demultiplexed/SA31_2.fastq SA31_forward.paired.fastq.gz SA31_forward.singletons.fastq.gz SA31_reverse.paired.fastq.gz SA31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:05:50 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA31/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 33983 Both Surviving: 11512 (33.88%) Forward Only Surviving: 2671 (7.86%) Reverse Only Surviving: 509 (1.50%) Dropped: 19291 (56.77%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA31_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA31_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA31_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA31_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA31_forward.paired.headclipped.fastq.gz SA31_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA31 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA31_forward.paired.headclipped.fastq.gz
[FLASH]     SA31_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA31.extendedFrags.fastq.gz
[FLASH]     ./SA31.notCombined_1.fastq.gz
[FLASH]     ./SA31.notCombined_2.fastq.gz
[FLASH]     ./SA31.hist
[FLASH]     ./SA31.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11512 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11512
[FLASH]     Combined pairs:   3713
[FLASH]     Uncombined pairs: 7799
[FLASH]     Percent combined: 32.25%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.160 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3713 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA31/SA31_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA31_centroids.fasta --uc SA31.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3713 sequences and identified 1957 clusters (clustering threshold 1.00) - 100 clusters (representing 1543 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA32 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA32_1.fastq /home/working/demultiplexed/SA32_2.fastq SA32_forward.paired.fastq.gz SA32_forward.singletons.fastq.gz SA32_reverse.paired.fastq.gz SA32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA32_1.fastq /home/working/demultiplexed/SA32_2.fastq SA32_forward.paired.fastq.gz SA32_forward.singletons.fastq.gz SA32_reverse.paired.fastq.gz SA32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:06:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA32/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 21785 Both Surviving: 2978 (13.67%) Forward Only Surviving: 887 (4.07%) Reverse Only Surviving: 276 (1.27%) Dropped: 17644 (80.99%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA32_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA32_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA32_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA32_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA32_forward.paired.headclipped.fastq.gz SA32_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA32 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA32_forward.paired.headclipped.fastq.gz
[FLASH]     SA32_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA32.extendedFrags.fastq.gz
[FLASH]     ./SA32.notCombined_1.fastq.gz
[FLASH]     ./SA32.notCombined_2.fastq.gz
[FLASH]     ./SA32.hist
[FLASH]     ./SA32.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2978 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2978
[FLASH]     Combined pairs:   1249
[FLASH]     Uncombined pairs: 1729
[FLASH]     Percent combined: 41.94%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.056 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1249 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA32/SA32_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA32_centroids.fasta --uc SA32.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1249 sequences and identified 682 clusters (clustering threshold 1.00) - 21 clusters (representing 527 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA33 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA33_1.fastq /home/working/demultiplexed/SA33_2.fastq SA33_forward.paired.fastq.gz SA33_forward.singletons.fastq.gz SA33_reverse.paired.fastq.gz SA33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA33_1.fastq /home/working/demultiplexed/SA33_2.fastq SA33_forward.paired.fastq.gz SA33_forward.singletons.fastq.gz SA33_reverse.paired.fastq.gz SA33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:06:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA33/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 98419 Both Surviving: 3584 (3.64%) Forward Only Surviving: 3007 (3.06%) Reverse Only Surviving: 995 (1.01%) Dropped: 90833 (92.29%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA33_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA33_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SA33_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA33_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA33_forward.paired.headclipped.fastq.gz SA33_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA33 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA33_forward.paired.headclipped.fastq.gz
[FLASH]     SA33_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA33.extendedFrags.fastq.gz
[FLASH]     ./SA33.notCombined_1.fastq.gz
[FLASH]     ./SA33.notCombined_2.fastq.gz
[FLASH]     ./SA33.hist
[FLASH]     ./SA33.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3584 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3584
[FLASH]     Combined pairs:   2545
[FLASH]     Uncombined pairs: 1039
[FLASH]     Percent combined: 71.01%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.091 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2545 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA33/SA33_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA33_centroids.fasta --uc SA33.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2545 sequences and identified 1255 clusters (clustering threshold 1.00) - 41 clusters (representing 1163 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA36 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA36_1.fastq /home/working/demultiplexed/SA36_2.fastq SA36_forward.paired.fastq.gz SA36_forward.singletons.fastq.gz SA36_reverse.paired.fastq.gz SA36_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA36_1.fastq /home/working/demultiplexed/SA36_2.fastq SA36_forward.paired.fastq.gz SA36_forward.singletons.fastq.gz SA36_reverse.paired.fastq.gz SA36_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:06:24 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA36/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 51138 Both Surviving: 7411 (14.49%) Forward Only Surviving: 1797 (3.51%) Reverse Only Surviving: 663 (1.30%) Dropped: 41267 (80.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA36_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA36_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SA36_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SA36_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA36_forward.paired.headclipped.fastq.gz SA36_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA36 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA36_forward.paired.headclipped.fastq.gz
[FLASH]     SA36_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA36.extendedFrags.fastq.gz
[FLASH]     ./SA36.notCombined_1.fastq.gz
[FLASH]     ./SA36.notCombined_2.fastq.gz
[FLASH]     ./SA36.hist
[FLASH]     ./SA36.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7411 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7411
[FLASH]     Combined pairs:   200
[FLASH]     Uncombined pairs: 7211
[FLASH]     Percent combined: 2.70%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.087 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

200 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA36/SA36_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA36_centroids.fasta --uc SA36.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 200 sequences and identified 135 clusters (clustering threshold 1.00) - 6 clusters (representing 43 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA43 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA43_1.fastq /home/working/demultiplexed/SA43_2.fastq SA43_forward.paired.fastq.gz SA43_forward.singletons.fastq.gz SA43_reverse.paired.fastq.gz SA43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA43_1.fastq /home/working/demultiplexed/SA43_2.fastq SA43_forward.paired.fastq.gz SA43_forward.singletons.fastq.gz SA43_reverse.paired.fastq.gz SA43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:06:32 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA43/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22712 Both Surviving: 5221 (22.99%) Forward Only Surviving: 1302 (5.73%) Reverse Only Surviving: 488 (2.15%) Dropped: 15701 (69.13%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA43_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA43_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA43_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA43_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA43_forward.paired.headclipped.fastq.gz SA43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA43 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA43_forward.paired.headclipped.fastq.gz
[FLASH]     SA43_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA43.extendedFrags.fastq.gz
[FLASH]     ./SA43.notCombined_1.fastq.gz
[FLASH]     ./SA43.notCombined_2.fastq.gz
[FLASH]     ./SA43.hist
[FLASH]     ./SA43.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5221 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5221
[FLASH]     Combined pairs:   57
[FLASH]     Uncombined pairs: 5164
[FLASH]     Percent combined: 1.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.088 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

57 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA43/SA43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA43_centroids.fasta --uc SA43.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 57 sequences and identified 48 clusters (clustering threshold 1.00) - 1 clusters (representing 6 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA46 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA46_1.fastq /home/working/demultiplexed/SA46_2.fastq SA46_forward.paired.fastq.gz SA46_forward.singletons.fastq.gz SA46_reverse.paired.fastq.gz SA46_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA46_1.fastq /home/working/demultiplexed/SA46_2.fastq SA46_forward.paired.fastq.gz SA46_forward.singletons.fastq.gz SA46_reverse.paired.fastq.gz SA46_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:06:39 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA46/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 62158 Both Surviving: 14590 (23.47%) Forward Only Surviving: 4249 (6.84%) Reverse Only Surviving: 926 (1.49%) Dropped: 42393 (68.20%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA46_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA46_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA46_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA46_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA46_forward.paired.headclipped.fastq.gz SA46_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA46 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA46_forward.paired.headclipped.fastq.gz
[FLASH]     SA46_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA46.extendedFrags.fastq.gz
[FLASH]     ./SA46.notCombined_1.fastq.gz
[FLASH]     ./SA46.notCombined_2.fastq.gz
[FLASH]     ./SA46.hist
[FLASH]     ./SA46.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 14590 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      14590
[FLASH]     Combined pairs:   1007
[FLASH]     Uncombined pairs: 13583
[FLASH]     Percent combined: 6.90%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.284 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1007 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA46/SA46_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA46_centroids.fasta --uc SA46.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1007 sequences and identified 562 clusters (clustering threshold 1.00) - 13 clusters (representing 376 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA47 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA47_1.fastq /home/working/demultiplexed/SA47_2.fastq SA47_forward.paired.fastq.gz SA47_forward.singletons.fastq.gz SA47_reverse.paired.fastq.gz SA47_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA47_1.fastq /home/working/demultiplexed/SA47_2.fastq SA47_forward.paired.fastq.gz SA47_forward.singletons.fastq.gz SA47_reverse.paired.fastq.gz SA47_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:06:51 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA47/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17517 Both Surviving: 7452 (42.54%) Forward Only Surviving: 3387 (19.34%) Reverse Only Surviving: 204 (1.16%) Dropped: 6474 (36.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA47_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA47_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA47_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA47_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA47_forward.paired.headclipped.fastq.gz SA47_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA47 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA47_forward.paired.headclipped.fastq.gz
[FLASH]     SA47_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA47.extendedFrags.fastq.gz
[FLASH]     ./SA47.notCombined_1.fastq.gz
[FLASH]     ./SA47.notCombined_2.fastq.gz
[FLASH]     ./SA47.hist
[FLASH]     ./SA47.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7452 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7452
[FLASH]     Combined pairs:   5420
[FLASH]     Uncombined pairs: 2032
[FLASH]     Percent combined: 72.73%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.267 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5420 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA47/SA47_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA47_centroids.fasta --uc SA47.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5420 sequences and identified 1636 clusters (clustering threshold 1.00) - 162 clusters (representing 3625 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA48 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA48_1.fastq /home/working/demultiplexed/SA48_2.fastq SA48_forward.paired.fastq.gz SA48_forward.singletons.fastq.gz SA48_reverse.paired.fastq.gz SA48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA48_1.fastq /home/working/demultiplexed/SA48_2.fastq SA48_forward.paired.fastq.gz SA48_forward.singletons.fastq.gz SA48_reverse.paired.fastq.gz SA48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:07:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA48/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11761 Both Surviving: 145 (1.23%) Forward Only Surviving: 144 (1.22%) Reverse Only Surviving: 68 (0.58%) Dropped: 11404 (96.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA48_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA48_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA48_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA48_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA48_forward.paired.headclipped.fastq.gz SA48_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA48 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA48_forward.paired.headclipped.fastq.gz
[FLASH]     SA48_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA48.extendedFrags.fastq.gz
[FLASH]     ./SA48.notCombined_1.fastq.gz
[FLASH]     ./SA48.notCombined_2.fastq.gz
[FLASH]     ./SA48.hist
[FLASH]     ./SA48.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 145 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      145
[FLASH]     Combined pairs:   12
[FLASH]     Uncombined pairs: 133
[FLASH]     Percent combined: 8.28%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

12 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA48/SA48_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA48_centroids.fasta --uc SA48.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 12 sequences and identified 11 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA49 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA49_1.fastq /home/working/demultiplexed/SA49_2.fastq SA49_forward.paired.fastq.gz SA49_forward.singletons.fastq.gz SA49_reverse.paired.fastq.gz SA49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA49_1.fastq /home/working/demultiplexed/SA49_2.fastq SA49_forward.paired.fastq.gz SA49_forward.singletons.fastq.gz SA49_reverse.paired.fastq.gz SA49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:07:08 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA49/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23373 Both Surviving: 3250 (13.90%) Forward Only Surviving: 989 (4.23%) Reverse Only Surviving: 238 (1.02%) Dropped: 18896 (80.85%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA49_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA49_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA49_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA49_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA49_forward.paired.headclipped.fastq.gz SA49_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA49 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA49_forward.paired.headclipped.fastq.gz
[FLASH]     SA49_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA49.extendedFrags.fastq.gz
[FLASH]     ./SA49.notCombined_1.fastq.gz
[FLASH]     ./SA49.notCombined_2.fastq.gz
[FLASH]     ./SA49.hist
[FLASH]     ./SA49.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3250 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3250
[FLASH]     Combined pairs:   2976
[FLASH]     Uncombined pairs: 274
[FLASH]     Percent combined: 91.57%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.107 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2976 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA49/SA49_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA49_centroids.fasta --uc SA49.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2976 sequences and identified 1540 clusters (clustering threshold 1.00) - 77 clusters (representing 1271 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA5 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA5_1.fastq /home/working/demultiplexed/SA5_2.fastq SA5_forward.paired.fastq.gz SA5_forward.singletons.fastq.gz SA5_reverse.paired.fastq.gz SA5_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA5_1.fastq /home/working/demultiplexed/SA5_2.fastq SA5_forward.paired.fastq.gz SA5_forward.singletons.fastq.gz SA5_reverse.paired.fastq.gz SA5_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:07:19 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA5/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 39236 Both Surviving: 902 (2.30%) Forward Only Surviving: 560 (1.43%) Reverse Only Surviving: 619 (1.58%) Dropped: 37155 (94.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA5_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA5_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA5_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA5_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA5_forward.paired.headclipped.fastq.gz SA5_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA5 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA5_forward.paired.headclipped.fastq.gz
[FLASH]     SA5_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA5.extendedFrags.fastq.gz
[FLASH]     ./SA5.notCombined_1.fastq.gz
[FLASH]     ./SA5.notCombined_2.fastq.gz
[FLASH]     ./SA5.hist
[FLASH]     ./SA5.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 902 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      902
[FLASH]     Combined pairs:   137
[FLASH]     Uncombined pairs: 765
[FLASH]     Percent combined: 15.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.013 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

137 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA5/SA5_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA5_centroids.fasta --uc SA5.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 137 sequences and identified 80 clusters (clustering threshold 1.00) - 3 clusters (representing 53 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA50 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA50_1.fastq /home/working/demultiplexed/SA50_2.fastq SA50_forward.paired.fastq.gz SA50_forward.singletons.fastq.gz SA50_reverse.paired.fastq.gz SA50_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA50_1.fastq /home/working/demultiplexed/SA50_2.fastq SA50_forward.paired.fastq.gz SA50_forward.singletons.fastq.gz SA50_reverse.paired.fastq.gz SA50_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:07:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA50/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 8879 Both Surviving: 3139 (35.35%) Forward Only Surviving: 843 (9.49%) Reverse Only Surviving: 139 (1.57%) Dropped: 4758 (53.59%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA50_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA50_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA50_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA50_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA50_forward.paired.headclipped.fastq.gz SA50_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA50 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA50_forward.paired.headclipped.fastq.gz
[FLASH]     SA50_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA50.extendedFrags.fastq.gz
[FLASH]     ./SA50.notCombined_1.fastq.gz
[FLASH]     ./SA50.notCombined_2.fastq.gz
[FLASH]     ./SA50.hist
[FLASH]     ./SA50.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3139 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3139
[FLASH]     Combined pairs:   918
[FLASH]     Uncombined pairs: 2221
[FLASH]     Percent combined: 29.24%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.062 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

918 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA50/SA50_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA50_centroids.fasta --uc SA50.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 918 sequences and identified 566 clusters (clustering threshold 1.00) - 11 clusters (representing 313 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA51 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA51_1.fastq /home/working/demultiplexed/SA51_2.fastq SA51_forward.paired.fastq.gz SA51_forward.singletons.fastq.gz SA51_reverse.paired.fastq.gz SA51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA51_1.fastq /home/working/demultiplexed/SA51_2.fastq SA51_forward.paired.fastq.gz SA51_forward.singletons.fastq.gz SA51_reverse.paired.fastq.gz SA51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:07:28 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA51/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 24102 Both Surviving: 5856 (24.30%) Forward Only Surviving: 1209 (5.02%) Reverse Only Surviving: 454 (1.88%) Dropped: 16583 (68.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA51_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA51_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA51_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA51_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA51_forward.paired.headclipped.fastq.gz SA51_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA51 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA51_forward.paired.headclipped.fastq.gz
[FLASH]     SA51_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA51.extendedFrags.fastq.gz
[FLASH]     ./SA51.notCombined_1.fastq.gz
[FLASH]     ./SA51.notCombined_2.fastq.gz
[FLASH]     ./SA51.hist
[FLASH]     ./SA51.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5856 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5856
[FLASH]     Combined pairs:   1967
[FLASH]     Uncombined pairs: 3889
[FLASH]     Percent combined: 33.59%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.119 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1967 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA51/SA51_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA51_centroids.fasta --uc SA51.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1967 sequences and identified 858 clusters (clustering threshold 1.00) - 27 clusters (representing 976 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA53 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA53_1.fastq /home/working/demultiplexed/SA53_2.fastq SA53_forward.paired.fastq.gz SA53_forward.singletons.fastq.gz SA53_reverse.paired.fastq.gz SA53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA53_1.fastq /home/working/demultiplexed/SA53_2.fastq SA53_forward.paired.fastq.gz SA53_forward.singletons.fastq.gz SA53_reverse.paired.fastq.gz SA53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:07:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA53/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 73318 Both Surviving: 20776 (28.34%) Forward Only Surviving: 5766 (7.86%) Reverse Only Surviving: 1405 (1.92%) Dropped: 45371 (61.88%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA53_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA53_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA53_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA53_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA53_forward.paired.headclipped.fastq.gz SA53_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA53 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA53_forward.paired.headclipped.fastq.gz
[FLASH]     SA53_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA53.extendedFrags.fastq.gz
[FLASH]     ./SA53.notCombined_1.fastq.gz
[FLASH]     ./SA53.notCombined_2.fastq.gz
[FLASH]     ./SA53.hist
[FLASH]     ./SA53.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 20776 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      20776
[FLASH]     Combined pairs:   9111
[FLASH]     Uncombined pairs: 11665
[FLASH]     Percent combined: 43.85%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.546 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

9111 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA53/SA53_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA53_centroids.fasta --uc SA53.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 9111 sequences and identified 3677 clusters (clustering threshold 1.00) - 97 clusters (representing 4680 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA55 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA55_1.fastq /home/working/demultiplexed/SA55_2.fastq SA55_forward.paired.fastq.gz SA55_forward.singletons.fastq.gz SA55_reverse.paired.fastq.gz SA55_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA55_1.fastq /home/working/demultiplexed/SA55_2.fastq SA55_forward.paired.fastq.gz SA55_forward.singletons.fastq.gz SA55_reverse.paired.fastq.gz SA55_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:08:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA55/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 49553 Both Surviving: 11984 (24.18%) Forward Only Surviving: 2637 (5.32%) Reverse Only Surviving: 826 (1.67%) Dropped: 34106 (68.83%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA55_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA55_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SA55_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA55_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA55_forward.paired.headclipped.fastq.gz SA55_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA55 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA55_forward.paired.headclipped.fastq.gz
[FLASH]     SA55_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA55.extendedFrags.fastq.gz
[FLASH]     ./SA55.notCombined_1.fastq.gz
[FLASH]     ./SA55.notCombined_2.fastq.gz
[FLASH]     ./SA55.hist
[FLASH]     ./SA55.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11984 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11984
[FLASH]     Combined pairs:   3330
[FLASH]     Uncombined pairs: 8654
[FLASH]     Percent combined: 27.79%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.158 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3330 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA55/SA55_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA55_centroids.fasta --uc SA55.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3330 sequences and identified 1674 clusters (clustering threshold 1.00) - 47 clusters (representing 1432 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA58 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA58_1.fastq /home/working/demultiplexed/SA58_2.fastq SA58_forward.paired.fastq.gz SA58_forward.singletons.fastq.gz SA58_reverse.paired.fastq.gz SA58_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA58_1.fastq /home/working/demultiplexed/SA58_2.fastq SA58_forward.paired.fastq.gz SA58_forward.singletons.fastq.gz SA58_reverse.paired.fastq.gz SA58_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:08:28 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA58/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 43499 Both Surviving: 10937 (25.14%) Forward Only Surviving: 2426 (5.58%) Reverse Only Surviving: 671 (1.54%) Dropped: 29465 (67.74%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA58_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA58_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA58_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA58_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA58_forward.paired.headclipped.fastq.gz SA58_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA58 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA58_forward.paired.headclipped.fastq.gz
[FLASH]     SA58_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA58.extendedFrags.fastq.gz
[FLASH]     ./SA58.notCombined_1.fastq.gz
[FLASH]     ./SA58.notCombined_2.fastq.gz
[FLASH]     ./SA58.hist
[FLASH]     ./SA58.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10937 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10937
[FLASH]     Combined pairs:   2439
[FLASH]     Uncombined pairs: 8498
[FLASH]     Percent combined: 22.30%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.119 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2439 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA58/SA58_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA58_centroids.fasta --uc SA58.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2439 sequences and identified 1500 clusters (clustering threshold 1.00) - 29 clusters (representing 816 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA62 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA62_1.fastq /home/working/demultiplexed/SA62_2.fastq SA62_forward.paired.fastq.gz SA62_forward.singletons.fastq.gz SA62_reverse.paired.fastq.gz SA62_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA62_1.fastq /home/working/demultiplexed/SA62_2.fastq SA62_forward.paired.fastq.gz SA62_forward.singletons.fastq.gz SA62_reverse.paired.fastq.gz SA62_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:08:40 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA62/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 39926 Both Surviving: 104 (0.26%) Forward Only Surviving: 753 (1.89%) Reverse Only Surviving: 484 (1.21%) Dropped: 38585 (96.64%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA62_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA62_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA62_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA62_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA62_forward.paired.headclipped.fastq.gz SA62_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA62 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA62_forward.paired.headclipped.fastq.gz
[FLASH]     SA62_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA62.extendedFrags.fastq.gz
[FLASH]     ./SA62.notCombined_1.fastq.gz
[FLASH]     ./SA62.notCombined_2.fastq.gz
[FLASH]     ./SA62.hist
[FLASH]     ./SA62.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 104 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      104
[FLASH]     Combined pairs:   45
[FLASH]     Uncombined pairs: 59
[FLASH]     Percent combined: 43.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

45 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA62/SA62_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA62_centroids.fasta --uc SA62.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 45 sequences and identified 37 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA64 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA64_1.fastq /home/working/demultiplexed/SA64_2.fastq SA64_forward.paired.fastq.gz SA64_forward.singletons.fastq.gz SA64_reverse.paired.fastq.gz SA64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA64_1.fastq /home/working/demultiplexed/SA64_2.fastq SA64_forward.paired.fastq.gz SA64_forward.singletons.fastq.gz SA64_reverse.paired.fastq.gz SA64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:08:44 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA64/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28581 Both Surviving: 5231 (18.30%) Forward Only Surviving: 1867 (6.53%) Reverse Only Surviving: 380 (1.33%) Dropped: 21103 (73.84%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA64_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA64_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA64_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA64_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA64_forward.paired.headclipped.fastq.gz SA64_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA64 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA64_forward.paired.headclipped.fastq.gz
[FLASH]     SA64_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA64.extendedFrags.fastq.gz
[FLASH]     ./SA64.notCombined_1.fastq.gz
[FLASH]     ./SA64.notCombined_2.fastq.gz
[FLASH]     ./SA64.hist
[FLASH]     ./SA64.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5231 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5231
[FLASH]     Combined pairs:   366
[FLASH]     Uncombined pairs: 4865
[FLASH]     Percent combined: 7.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.057 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

366 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA64/SA64_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA64_centroids.fasta --uc SA64.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 366 sequences and identified 262 clusters (clustering threshold 1.00) - 8 clusters (representing 78 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA67 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA67_1.fastq /home/working/demultiplexed/SA67_2.fastq SA67_forward.paired.fastq.gz SA67_forward.singletons.fastq.gz SA67_reverse.paired.fastq.gz SA67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA67_1.fastq /home/working/demultiplexed/SA67_2.fastq SA67_forward.paired.fastq.gz SA67_forward.singletons.fastq.gz SA67_reverse.paired.fastq.gz SA67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:08:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA67/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29766 Both Surviving: 11022 (37.03%) Forward Only Surviving: 3974 (13.35%) Reverse Only Surviving: 376 (1.26%) Dropped: 14394 (48.36%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA67_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA67_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA67_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA67_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA67_forward.paired.headclipped.fastq.gz SA67_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA67 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA67_forward.paired.headclipped.fastq.gz
[FLASH]     SA67_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA67.extendedFrags.fastq.gz
[FLASH]     ./SA67.notCombined_1.fastq.gz
[FLASH]     ./SA67.notCombined_2.fastq.gz
[FLASH]     ./SA67.hist
[FLASH]     ./SA67.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11022 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11022
[FLASH]     Combined pairs:   4029
[FLASH]     Uncombined pairs: 6993
[FLASH]     Percent combined: 36.55%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.159 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4029 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA67/SA67_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA67_centroids.fasta --uc SA67.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4029 sequences and identified 2678 clusters (clustering threshold 1.00) - 77 clusters (representing 1169 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA68 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA68_1.fastq /home/working/demultiplexed/SA68_2.fastq SA68_forward.paired.fastq.gz SA68_forward.singletons.fastq.gz SA68_reverse.paired.fastq.gz SA68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA68_1.fastq /home/working/demultiplexed/SA68_2.fastq SA68_forward.paired.fastq.gz SA68_forward.singletons.fastq.gz SA68_reverse.paired.fastq.gz SA68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:09:03 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA68/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 36383 Both Surviving: 2264 (6.22%) Forward Only Surviving: 770 (2.12%) Reverse Only Surviving: 425 (1.17%) Dropped: 32924 (90.49%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA68_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA68_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA68_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA68_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA68_forward.paired.headclipped.fastq.gz SA68_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA68 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA68_forward.paired.headclipped.fastq.gz
[FLASH]     SA68_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA68.extendedFrags.fastq.gz
[FLASH]     ./SA68.notCombined_1.fastq.gz
[FLASH]     ./SA68.notCombined_2.fastq.gz
[FLASH]     ./SA68.hist
[FLASH]     ./SA68.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2264 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2264
[FLASH]     Combined pairs:   1962
[FLASH]     Uncombined pairs: 302
[FLASH]     Percent combined: 86.66%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.056 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1962 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA68/SA68_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA68_centroids.fasta --uc SA68.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1962 sequences and identified 891 clusters (clustering threshold 1.00) - 32 clusters (representing 959 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA7 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA7_1.fastq /home/working/demultiplexed/SA7_2.fastq SA7_forward.paired.fastq.gz SA7_forward.singletons.fastq.gz SA7_reverse.paired.fastq.gz SA7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA7_1.fastq /home/working/demultiplexed/SA7_2.fastq SA7_forward.paired.fastq.gz SA7_forward.singletons.fastq.gz SA7_reverse.paired.fastq.gz SA7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:09:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA7/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 30674 Both Surviving: 2927 (9.54%) Forward Only Surviving: 1329 (4.33%) Reverse Only Surviving: 307 (1.00%) Dropped: 26111 (85.12%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA7_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA7_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SA7_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA7_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA7_forward.paired.headclipped.fastq.gz SA7_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA7 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA7_forward.paired.headclipped.fastq.gz
[FLASH]     SA7_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA7.extendedFrags.fastq.gz
[FLASH]     ./SA7.notCombined_1.fastq.gz
[FLASH]     ./SA7.notCombined_2.fastq.gz
[FLASH]     ./SA7.hist
[FLASH]     ./SA7.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2927 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2927
[FLASH]     Combined pairs:   2420
[FLASH]     Uncombined pairs: 507
[FLASH]     Percent combined: 82.68%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.091 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2420 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA7/SA7_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA7_centroids.fasta --uc SA7.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2420 sequences and identified 1319 clusters (clustering threshold 1.00) - 49 clusters (representing 954 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA70 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA70_1.fastq /home/working/demultiplexed/SA70_2.fastq SA70_forward.paired.fastq.gz SA70_forward.singletons.fastq.gz SA70_reverse.paired.fastq.gz SA70_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA70_1.fastq /home/working/demultiplexed/SA70_2.fastq SA70_forward.paired.fastq.gz SA70_forward.singletons.fastq.gz SA70_reverse.paired.fastq.gz SA70_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:09:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA70/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 48118 Both Surviving: 417 (0.87%) Forward Only Surviving: 633 (1.32%) Reverse Only Surviving: 620 (1.29%) Dropped: 46448 (96.53%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA70_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA70_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA70_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA70_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA70_forward.paired.headclipped.fastq.gz SA70_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA70 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA70_forward.paired.headclipped.fastq.gz
[FLASH]     SA70_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA70.extendedFrags.fastq.gz
[FLASH]     ./SA70.notCombined_1.fastq.gz
[FLASH]     ./SA70.notCombined_2.fastq.gz
[FLASH]     ./SA70.hist
[FLASH]     ./SA70.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 417 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      417
[FLASH]     Combined pairs:   267
[FLASH]     Uncombined pairs: 150
[FLASH]     Percent combined: 64.03%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.009 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

267 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA70/SA70_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA70_centroids.fasta --uc SA70.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 267 sequences and identified 163 clusters (clustering threshold 1.00) - 7 clusters (representing 97 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA73 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA73_1.fastq /home/working/demultiplexed/SA73_2.fastq SA73_forward.paired.fastq.gz SA73_forward.singletons.fastq.gz SA73_reverse.paired.fastq.gz SA73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA73_1.fastq /home/working/demultiplexed/SA73_2.fastq SA73_forward.paired.fastq.gz SA73_forward.singletons.fastq.gz SA73_reverse.paired.fastq.gz SA73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:09:24 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA73/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 45415 Both Surviving: 9585 (21.11%) Forward Only Surviving: 2169 (4.78%) Reverse Only Surviving: 756 (1.66%) Dropped: 32905 (72.45%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA73_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA73_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA73_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA73_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA73_forward.paired.headclipped.fastq.gz SA73_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA73 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA73_forward.paired.headclipped.fastq.gz
[FLASH]     SA73_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA73.extendedFrags.fastq.gz
[FLASH]     ./SA73.notCombined_1.fastq.gz
[FLASH]     ./SA73.notCombined_2.fastq.gz
[FLASH]     ./SA73.hist
[FLASH]     ./SA73.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9585 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9585
[FLASH]     Combined pairs:   2029
[FLASH]     Uncombined pairs: 7556
[FLASH]     Percent combined: 21.17%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.096 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2029 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA73/SA73_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA73_centroids.fasta --uc SA73.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2029 sequences and identified 1038 clusters (clustering threshold 1.00) - 37 clusters (representing 840 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA75 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA75_1.fastq /home/working/demultiplexed/SA75_2.fastq SA75_forward.paired.fastq.gz SA75_forward.singletons.fastq.gz SA75_reverse.paired.fastq.gz SA75_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA75_1.fastq /home/working/demultiplexed/SA75_2.fastq SA75_forward.paired.fastq.gz SA75_forward.singletons.fastq.gz SA75_reverse.paired.fastq.gz SA75_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:09:33 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA75/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17374 Both Surviving: 1201 (6.91%) Forward Only Surviving: 573 (3.30%) Reverse Only Surviving: 227 (1.31%) Dropped: 15373 (88.48%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA75_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA75_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA75_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA75_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA75_forward.paired.headclipped.fastq.gz SA75_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA75 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA75_forward.paired.headclipped.fastq.gz
[FLASH]     SA75_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA75.extendedFrags.fastq.gz
[FLASH]     ./SA75.notCombined_1.fastq.gz
[FLASH]     ./SA75.notCombined_2.fastq.gz
[FLASH]     ./SA75.hist
[FLASH]     ./SA75.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1201 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1201
[FLASH]     Combined pairs:   421
[FLASH]     Uncombined pairs: 780
[FLASH]     Percent combined: 35.05%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.021 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

421 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA75/SA75_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA75_centroids.fasta --uc SA75.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 421 sequences and identified 300 clusters (clustering threshold 1.00) - 6 clusters (representing 110 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA76 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA76_1.fastq /home/working/demultiplexed/SA76_2.fastq SA76_forward.paired.fastq.gz SA76_forward.singletons.fastq.gz SA76_reverse.paired.fastq.gz SA76_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA76_1.fastq /home/working/demultiplexed/SA76_2.fastq SA76_forward.paired.fastq.gz SA76_forward.singletons.fastq.gz SA76_reverse.paired.fastq.gz SA76_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:09:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA76/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20250 Both Surviving: 7598 (37.52%) Forward Only Surviving: 3215 (15.88%) Reverse Only Surviving: 243 (1.20%) Dropped: 9194 (45.40%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA76_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA76_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA76_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA76_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA76_forward.paired.headclipped.fastq.gz SA76_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA76 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA76_forward.paired.headclipped.fastq.gz
[FLASH]     SA76_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA76.extendedFrags.fastq.gz
[FLASH]     ./SA76.notCombined_1.fastq.gz
[FLASH]     ./SA76.notCombined_2.fastq.gz
[FLASH]     ./SA76.hist
[FLASH]     ./SA76.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7598 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7598
[FLASH]     Combined pairs:   6303
[FLASH]     Uncombined pairs: 1295
[FLASH]     Percent combined: 82.96%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.176 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6303 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA76/SA76_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA76_centroids.fasta --uc SA76.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6303 sequences and identified 2397 clusters (clustering threshold 1.00) - 142 clusters (representing 3603 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA81 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA81_1.fastq /home/working/demultiplexed/SA81_2.fastq SA81_forward.paired.fastq.gz SA81_forward.singletons.fastq.gz SA81_reverse.paired.fastq.gz SA81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA81_1.fastq /home/working/demultiplexed/SA81_2.fastq SA81_forward.paired.fastq.gz SA81_forward.singletons.fastq.gz SA81_reverse.paired.fastq.gz SA81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:09:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA81/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 30631 Both Surviving: 585 (1.91%) Forward Only Surviving: 387 (1.26%) Reverse Only Surviving: 304 (0.99%) Dropped: 29355 (95.83%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA81_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA81_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SA81_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA81_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA81_forward.paired.headclipped.fastq.gz SA81_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA81 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA81_forward.paired.headclipped.fastq.gz
[FLASH]     SA81_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA81.extendedFrags.fastq.gz
[FLASH]     ./SA81.notCombined_1.fastq.gz
[FLASH]     ./SA81.notCombined_2.fastq.gz
[FLASH]     ./SA81.hist
[FLASH]     ./SA81.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 585 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      585
[FLASH]     Combined pairs:   47
[FLASH]     Uncombined pairs: 538
[FLASH]     Percent combined: 8.03%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.007 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

47 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA81/SA81_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA81_centroids.fasta --uc SA81.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 47 sequences and identified 42 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA86 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA86_1.fastq /home/working/demultiplexed/SA86_2.fastq SA86_forward.paired.fastq.gz SA86_forward.singletons.fastq.gz SA86_reverse.paired.fastq.gz SA86_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA86_1.fastq /home/working/demultiplexed/SA86_2.fastq SA86_forward.paired.fastq.gz SA86_forward.singletons.fastq.gz SA86_reverse.paired.fastq.gz SA86_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:09:54 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA86/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 60032 Both Surviving: 8569 (14.27%) Forward Only Surviving: 2738 (4.56%) Reverse Only Surviving: 656 (1.09%) Dropped: 48069 (80.07%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA86_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA86_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA86_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA86_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA86_forward.paired.headclipped.fastq.gz SA86_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA86 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA86_forward.paired.headclipped.fastq.gz
[FLASH]     SA86_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA86.extendedFrags.fastq.gz
[FLASH]     ./SA86.notCombined_1.fastq.gz
[FLASH]     ./SA86.notCombined_2.fastq.gz
[FLASH]     ./SA86.hist
[FLASH]     ./SA86.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8569 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8569
[FLASH]     Combined pairs:   216
[FLASH]     Uncombined pairs: 8353
[FLASH]     Percent combined: 2.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.072 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

216 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA86/SA86_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA86_centroids.fasta --uc SA86.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 216 sequences and identified 181 clusters (clustering threshold 1.00) - 2 clusters (representing 28 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA87 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA87_1.fastq /home/working/demultiplexed/SA87_2.fastq SA87_forward.paired.fastq.gz SA87_forward.singletons.fastq.gz SA87_reverse.paired.fastq.gz SA87_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA87_1.fastq /home/working/demultiplexed/SA87_2.fastq SA87_forward.paired.fastq.gz SA87_forward.singletons.fastq.gz SA87_reverse.paired.fastq.gz SA87_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:10:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA87/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31996 Both Surviving: 1905 (5.95%) Forward Only Surviving: 1110 (3.47%) Reverse Only Surviving: 484 (1.51%) Dropped: 28497 (89.06%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA87_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA87_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA87_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA87_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA87_forward.paired.headclipped.fastq.gz SA87_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA87 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA87_forward.paired.headclipped.fastq.gz
[FLASH]     SA87_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA87.extendedFrags.fastq.gz
[FLASH]     ./SA87.notCombined_1.fastq.gz
[FLASH]     ./SA87.notCombined_2.fastq.gz
[FLASH]     ./SA87.hist
[FLASH]     ./SA87.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1905 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1905
[FLASH]     Combined pairs:   643
[FLASH]     Uncombined pairs: 1262
[FLASH]     Percent combined: 33.75%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.025 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

643 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA87/SA87_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA87_centroids.fasta --uc SA87.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 643 sequences and identified 404 clusters (clustering threshold 1.00) - 9 clusters (representing 198 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA88 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA88_1.fastq /home/working/demultiplexed/SA88_2.fastq SA88_forward.paired.fastq.gz SA88_forward.singletons.fastq.gz SA88_reverse.paired.fastq.gz SA88_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA88_1.fastq /home/working/demultiplexed/SA88_2.fastq SA88_forward.paired.fastq.gz SA88_forward.singletons.fastq.gz SA88_reverse.paired.fastq.gz SA88_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:10:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA88/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 95056 Both Surviving: 32779 (34.48%) Forward Only Surviving: 7861 (8.27%) Reverse Only Surviving: 2531 (2.66%) Dropped: 51885 (54.58%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA88_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA88_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA88_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA88_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA88_forward.paired.headclipped.fastq.gz SA88_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA88 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA88_forward.paired.headclipped.fastq.gz
[FLASH]     SA88_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA88.extendedFrags.fastq.gz
[FLASH]     ./SA88.notCombined_1.fastq.gz
[FLASH]     ./SA88.notCombined_2.fastq.gz
[FLASH]     ./SA88.hist
[FLASH]     ./SA88.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 32779 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      32779
[FLASH]     Combined pairs:   1124
[FLASH]     Uncombined pairs: 31655
[FLASH]     Percent combined: 3.43%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.370 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1124 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA88/SA88_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA88_centroids.fasta --uc SA88.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1124 sequences and identified 882 clusters (clustering threshold 1.00) - 17 clusters (representing 150 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA89 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA89_1.fastq /home/working/demultiplexed/SA89_2.fastq SA89_forward.paired.fastq.gz SA89_forward.singletons.fastq.gz SA89_reverse.paired.fastq.gz SA89_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA89_1.fastq /home/working/demultiplexed/SA89_2.fastq SA89_forward.paired.fastq.gz SA89_forward.singletons.fastq.gz SA89_reverse.paired.fastq.gz SA89_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:10:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA89/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20721 Both Surviving: 4774 (23.04%) Forward Only Surviving: 992 (4.79%) Reverse Only Surviving: 412 (1.99%) Dropped: 14543 (70.18%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA89_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA89_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA89_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA89_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA89_forward.paired.headclipped.fastq.gz SA89_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA89 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA89_forward.paired.headclipped.fastq.gz
[FLASH]     SA89_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA89.extendedFrags.fastq.gz
[FLASH]     ./SA89.notCombined_1.fastq.gz
[FLASH]     ./SA89.notCombined_2.fastq.gz
[FLASH]     ./SA89.hist
[FLASH]     ./SA89.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4774 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4774
[FLASH]     Combined pairs:   177
[FLASH]     Uncombined pairs: 4597
[FLASH]     Percent combined: 3.71%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.053 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

177 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA89/SA89_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA89_centroids.fasta --uc SA89.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 177 sequences and identified 130 clusters (clustering threshold 1.00) - 2 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA9 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA9_1.fastq /home/working/demultiplexed/SA9_2.fastq SA9_forward.paired.fastq.gz SA9_forward.singletons.fastq.gz SA9_reverse.paired.fastq.gz SA9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA9_1.fastq /home/working/demultiplexed/SA9_2.fastq SA9_forward.paired.fastq.gz SA9_forward.singletons.fastq.gz SA9_reverse.paired.fastq.gz SA9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:10:35 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA9/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 27387 Both Surviving: 3248 (11.86%) Forward Only Surviving: 1038 (3.79%) Reverse Only Surviving: 271 (0.99%) Dropped: 22830 (83.36%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA9_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA9_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA9_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA9_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA9_forward.paired.headclipped.fastq.gz SA9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA9 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA9_forward.paired.headclipped.fastq.gz
[FLASH]     SA9_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA9.extendedFrags.fastq.gz
[FLASH]     ./SA9.notCombined_1.fastq.gz
[FLASH]     ./SA9.notCombined_2.fastq.gz
[FLASH]     ./SA9.hist
[FLASH]     ./SA9.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3248 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3248
[FLASH]     Combined pairs:   2832
[FLASH]     Uncombined pairs: 416
[FLASH]     Percent combined: 87.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.087 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2832 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA9/SA9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA9_centroids.fasta --uc SA9.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2832 sequences and identified 1263 clusters (clustering threshold 1.00) - 32 clusters (representing 1368 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA91 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA91_1.fastq /home/working/demultiplexed/SA91_2.fastq SA91_forward.paired.fastq.gz SA91_forward.singletons.fastq.gz SA91_reverse.paired.fastq.gz SA91_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA91_1.fastq /home/working/demultiplexed/SA91_2.fastq SA91_forward.paired.fastq.gz SA91_forward.singletons.fastq.gz SA91_reverse.paired.fastq.gz SA91_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:10:45 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA91/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 58616 Both Surviving: 9782 (16.69%) Forward Only Surviving: 3409 (5.82%) Reverse Only Surviving: 846 (1.44%) Dropped: 44579 (76.05%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA91_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA91_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA91_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA91_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA91_forward.paired.headclipped.fastq.gz SA91_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA91 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA91_forward.paired.headclipped.fastq.gz
[FLASH]     SA91_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA91.extendedFrags.fastq.gz
[FLASH]     ./SA91.notCombined_1.fastq.gz
[FLASH]     ./SA91.notCombined_2.fastq.gz
[FLASH]     ./SA91.hist
[FLASH]     ./SA91.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9782 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9782
[FLASH]     Combined pairs:   1474
[FLASH]     Uncombined pairs: 8308
[FLASH]     Percent combined: 15.07%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.092 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1474 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA91/SA91_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA91_centroids.fasta --uc SA91.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1474 sequences and identified 745 clusters (clustering threshold 1.00) - 16 clusters (representing 648 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA93 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA93_1.fastq /home/working/demultiplexed/SA93_2.fastq SA93_forward.paired.fastq.gz SA93_forward.singletons.fastq.gz SA93_reverse.paired.fastq.gz SA93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA93_1.fastq /home/working/demultiplexed/SA93_2.fastq SA93_forward.paired.fastq.gz SA93_forward.singletons.fastq.gz SA93_reverse.paired.fastq.gz SA93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:10:53 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA93/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20338 Both Surviving: 3625 (17.82%) Forward Only Surviving: 818 (4.02%) Reverse Only Surviving: 288 (1.42%) Dropped: 15607 (76.74%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA93_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA93_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA93_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA93_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA93_forward.paired.headclipped.fastq.gz SA93_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA93 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA93_forward.paired.headclipped.fastq.gz
[FLASH]     SA93_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA93.extendedFrags.fastq.gz
[FLASH]     ./SA93.notCombined_1.fastq.gz
[FLASH]     ./SA93.notCombined_2.fastq.gz
[FLASH]     ./SA93.hist
[FLASH]     ./SA93.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3625 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3625
[FLASH]     Combined pairs:   1692
[FLASH]     Uncombined pairs: 1933
[FLASH]     Percent combined: 46.68%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.059 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1692 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA93/SA93_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA93_centroids.fasta --uc SA93.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1692 sequences and identified 1004 clusters (clustering threshold 1.00) - 20 clusters (representing 607 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA94 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA94_1.fastq /home/working/demultiplexed/SA94_2.fastq SA94_forward.paired.fastq.gz SA94_forward.singletons.fastq.gz SA94_reverse.paired.fastq.gz SA94_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA94_1.fastq /home/working/demultiplexed/SA94_2.fastq SA94_forward.paired.fastq.gz SA94_forward.singletons.fastq.gz SA94_reverse.paired.fastq.gz SA94_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:10:59 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA94/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25533 Both Surviving: 139 (0.54%) Forward Only Surviving: 402 (1.57%) Reverse Only Surviving: 269 (1.05%) Dropped: 24723 (96.83%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA94_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA94_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA94_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA94_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA94_forward.paired.headclipped.fastq.gz SA94_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA94 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA94_forward.paired.headclipped.fastq.gz
[FLASH]     SA94_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA94.extendedFrags.fastq.gz
[FLASH]     ./SA94.notCombined_1.fastq.gz
[FLASH]     ./SA94.notCombined_2.fastq.gz
[FLASH]     ./SA94.hist
[FLASH]     ./SA94.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 139 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      139
[FLASH]     Combined pairs:   72
[FLASH]     Uncombined pairs: 67
[FLASH]     Percent combined: 51.80%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

72 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA94/SA94_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA94_centroids.fasta --uc SA94.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 72 sequences and identified 44 clusters (clustering threshold 1.00) - 1 clusters (representing 26 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA95 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA95_1.fastq /home/working/demultiplexed/SA95_2.fastq SA95_forward.paired.fastq.gz SA95_forward.singletons.fastq.gz SA95_reverse.paired.fastq.gz SA95_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA95_1.fastq /home/working/demultiplexed/SA95_2.fastq SA95_forward.paired.fastq.gz SA95_forward.singletons.fastq.gz SA95_reverse.paired.fastq.gz SA95_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:11:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA95/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11287 Both Surviving: 698 (6.18%) Forward Only Surviving: 258 (2.29%) Reverse Only Surviving: 124 (1.10%) Dropped: 10207 (90.43%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA95_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA95_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA95_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA95_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA95_forward.paired.headclipped.fastq.gz SA95_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA95 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA95_forward.paired.headclipped.fastq.gz
[FLASH]     SA95_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA95.extendedFrags.fastq.gz
[FLASH]     ./SA95.notCombined_1.fastq.gz
[FLASH]     ./SA95.notCombined_2.fastq.gz
[FLASH]     ./SA95.hist
[FLASH]     ./SA95.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 698 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      698
[FLASH]     Combined pairs:   526
[FLASH]     Uncombined pairs: 172
[FLASH]     Percent combined: 75.36%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.017 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

526 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA95/SA95_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA95_centroids.fasta --uc SA95.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 526 sequences and identified 290 clusters (clustering threshold 1.00) - 8 clusters (representing 216 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA96 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA96_1.fastq /home/working/demultiplexed/SA96_2.fastq SA96_forward.paired.fastq.gz SA96_forward.singletons.fastq.gz SA96_reverse.paired.fastq.gz SA96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA96_1.fastq /home/working/demultiplexed/SA96_2.fastq SA96_forward.paired.fastq.gz SA96_forward.singletons.fastq.gz SA96_reverse.paired.fastq.gz SA96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:11:04 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA96/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15602 Both Surviving: 4680 (30.00%) Forward Only Surviving: 1124 (7.20%) Reverse Only Surviving: 244 (1.56%) Dropped: 9554 (61.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA96_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA96_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA96_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA96_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA96_forward.paired.headclipped.fastq.gz SA96_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA96 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA96_forward.paired.headclipped.fastq.gz
[FLASH]     SA96_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA96.extendedFrags.fastq.gz
[FLASH]     ./SA96.notCombined_1.fastq.gz
[FLASH]     ./SA96.notCombined_2.fastq.gz
[FLASH]     ./SA96.hist
[FLASH]     ./SA96.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4680 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4680
[FLASH]     Combined pairs:   1353
[FLASH]     Uncombined pairs: 3327
[FLASH]     Percent combined: 28.91%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.049 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1353 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA96/SA96_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA96_centroids.fasta --uc SA96.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1353 sequences and identified 828 clusters (clustering threshold 1.00) - 22 clusters (representing 483 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA97 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA97_1.fastq /home/working/demultiplexed/SA97_2.fastq SA97_forward.paired.fastq.gz SA97_forward.singletons.fastq.gz SA97_reverse.paired.fastq.gz SA97_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA97_1.fastq /home/working/demultiplexed/SA97_2.fastq SA97_forward.paired.fastq.gz SA97_forward.singletons.fastq.gz SA97_reverse.paired.fastq.gz SA97_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:11:09 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA97/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11026 Both Surviving: 2186 (19.83%) Forward Only Surviving: 409 (3.71%) Reverse Only Surviving: 184 (1.67%) Dropped: 8247 (74.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA97_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA97_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA97_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA97_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA97_forward.paired.headclipped.fastq.gz SA97_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA97 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA97_forward.paired.headclipped.fastq.gz
[FLASH]     SA97_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA97.extendedFrags.fastq.gz
[FLASH]     ./SA97.notCombined_1.fastq.gz
[FLASH]     ./SA97.notCombined_2.fastq.gz
[FLASH]     ./SA97.hist
[FLASH]     ./SA97.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2186 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2186
[FLASH]     Combined pairs:   816
[FLASH]     Uncombined pairs: 1370
[FLASH]     Percent combined: 37.33%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.028 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

816 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA97/SA97_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA97_centroids.fasta --uc SA97.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 816 sequences and identified 476 clusters (clustering threshold 1.00) - 9 clusters (representing 299 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA98 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA98_1.fastq /home/working/demultiplexed/SA98_2.fastq SA98_forward.paired.fastq.gz SA98_forward.singletons.fastq.gz SA98_reverse.paired.fastq.gz SA98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA98_1.fastq /home/working/demultiplexed/SA98_2.fastq SA98_forward.paired.fastq.gz SA98_forward.singletons.fastq.gz SA98_reverse.paired.fastq.gz SA98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:11:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SA98/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 42857 Both Surviving: 5781 (13.49%) Forward Only Surviving: 2218 (5.18%) Reverse Only Surviving: 333 (0.78%) Dropped: 34525 (80.56%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA98_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA98_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA98_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA98_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA98_forward.paired.headclipped.fastq.gz SA98_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA98 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA98_forward.paired.headclipped.fastq.gz
[FLASH]     SA98_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA98.extendedFrags.fastq.gz
[FLASH]     ./SA98.notCombined_1.fastq.gz
[FLASH]     ./SA98.notCombined_2.fastq.gz
[FLASH]     ./SA98.hist
[FLASH]     ./SA98.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5781 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5781
[FLASH]     Combined pairs:   5329
[FLASH]     Uncombined pairs: 452
[FLASH]     Percent combined: 92.18%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.139 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5329 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SA98/SA98_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA98_centroids.fasta --uc SA98.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5329 sequences and identified 1260 clusters (clustering threshold 1.00) - 143 clusters (representing 3925 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB102 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB102_1.fastq /home/working/demultiplexed/SB102_2.fastq SB102_forward.paired.fastq.gz SB102_forward.singletons.fastq.gz SB102_reverse.paired.fastq.gz SB102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB102_1.fastq /home/working/demultiplexed/SB102_2.fastq SB102_forward.paired.fastq.gz SB102_forward.singletons.fastq.gz SB102_reverse.paired.fastq.gz SB102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:11:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB102/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 95589 Both Surviving: 18233 (19.07%) Forward Only Surviving: 5549 (5.81%) Reverse Only Surviving: 1427 (1.49%) Dropped: 70380 (73.63%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB102_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB102_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB102_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB102_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB102_forward.paired.headclipped.fastq.gz SB102_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB102 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB102_forward.paired.headclipped.fastq.gz
[FLASH]     SB102_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB102.extendedFrags.fastq.gz
[FLASH]     ./SB102.notCombined_1.fastq.gz
[FLASH]     ./SB102.notCombined_2.fastq.gz
[FLASH]     ./SB102.hist
[FLASH]     ./SB102.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 18233 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      18233
[FLASH]     Combined pairs:   980
[FLASH]     Uncombined pairs: 17253
[FLASH]     Percent combined: 5.37%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.111 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

980 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB102/SB102_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB102_centroids.fasta --uc SB102.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 980 sequences and identified 623 clusters (clustering threshold 1.00) - 18 clusters (representing 291 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB104 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB104_1.fastq /home/working/demultiplexed/SB104_2.fastq SB104_forward.paired.fastq.gz SB104_forward.singletons.fastq.gz SB104_reverse.paired.fastq.gz SB104_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB104_1.fastq /home/working/demultiplexed/SB104_2.fastq SB104_forward.paired.fastq.gz SB104_forward.singletons.fastq.gz SB104_reverse.paired.fastq.gz SB104_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:11:42 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB104/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 21182 Both Surviving: 9019 (42.58%) Forward Only Surviving: 1940 (9.16%) Reverse Only Surviving: 659 (3.11%) Dropped: 9564 (45.15%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SB104_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB104_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB104_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB104_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB104_forward.paired.headclipped.fastq.gz SB104_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB104 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB104_forward.paired.headclipped.fastq.gz
[FLASH]     SB104_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB104.extendedFrags.fastq.gz
[FLASH]     ./SB104.notCombined_1.fastq.gz
[FLASH]     ./SB104.notCombined_2.fastq.gz
[FLASH]     ./SB104.hist
[FLASH]     ./SB104.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9019 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9019
[FLASH]     Combined pairs:   5159
[FLASH]     Uncombined pairs: 3860
[FLASH]     Percent combined: 57.20%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.146 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5159 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB104/SB104_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB104_centroids.fasta --uc SB104.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5159 sequences and identified 3829 clusters (clustering threshold 1.00) - 65 clusters (representing 924 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB106 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB106_1.fastq /home/working/demultiplexed/SB106_2.fastq SB106_forward.paired.fastq.gz SB106_forward.singletons.fastq.gz SB106_reverse.paired.fastq.gz SB106_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB106_1.fastq /home/working/demultiplexed/SB106_2.fastq SB106_forward.paired.fastq.gz SB106_forward.singletons.fastq.gz SB106_reverse.paired.fastq.gz SB106_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:11:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB106/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 49076 Both Surviving: 8570 (17.46%) Forward Only Surviving: 2504 (5.10%) Reverse Only Surviving: 465 (0.95%) Dropped: 37537 (76.49%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB106_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB106_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB106_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB106_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB106_forward.paired.headclipped.fastq.gz SB106_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB106 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB106_forward.paired.headclipped.fastq.gz
[FLASH]     SB106_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB106.extendedFrags.fastq.gz
[FLASH]     ./SB106.notCombined_1.fastq.gz
[FLASH]     ./SB106.notCombined_2.fastq.gz
[FLASH]     ./SB106.hist
[FLASH]     ./SB106.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8570 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8570
[FLASH]     Combined pairs:   7840
[FLASH]     Uncombined pairs: 730
[FLASH]     Percent combined: 91.48%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.226 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

7840 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB106/SB106_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB106_centroids.fasta --uc SB106.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 7840 sequences and identified 3631 clusters (clustering threshold 1.00) - 111 clusters (representing 3692 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB111 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB111_1.fastq /home/working/demultiplexed/SB111_2.fastq SB111_forward.paired.fastq.gz SB111_forward.singletons.fastq.gz SB111_reverse.paired.fastq.gz SB111_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB111_1.fastq /home/working/demultiplexed/SB111_2.fastq SB111_forward.paired.fastq.gz SB111_forward.singletons.fastq.gz SB111_reverse.paired.fastq.gz SB111_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:12:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB111/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31510 Both Surviving: 103 (0.33%) Forward Only Surviving: 529 (1.68%) Reverse Only Surviving: 306 (0.97%) Dropped: 30572 (97.02%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB111_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB111_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB111_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB111_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB111_forward.paired.headclipped.fastq.gz SB111_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB111 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB111_forward.paired.headclipped.fastq.gz
[FLASH]     SB111_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB111.extendedFrags.fastq.gz
[FLASH]     ./SB111.notCombined_1.fastq.gz
[FLASH]     ./SB111.notCombined_2.fastq.gz
[FLASH]     ./SB111.hist
[FLASH]     ./SB111.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 103 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      103
[FLASH]     Combined pairs:   27
[FLASH]     Uncombined pairs: 76
[FLASH]     Percent combined: 26.21%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

27 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB111/SB111_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB111_centroids.fasta --uc SB111.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 27 sequences and identified 26 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB116 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB116_1.fastq /home/working/demultiplexed/SB116_2.fastq SB116_forward.paired.fastq.gz SB116_forward.singletons.fastq.gz SB116_reverse.paired.fastq.gz SB116_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB116_1.fastq /home/working/demultiplexed/SB116_2.fastq SB116_forward.paired.fastq.gz SB116_forward.singletons.fastq.gz SB116_reverse.paired.fastq.gz SB116_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:12:13 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB116/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 12335 Both Surviving: 2168 (17.58%) Forward Only Surviving: 604 (4.90%) Reverse Only Surviving: 215 (1.74%) Dropped: 9348 (75.78%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB116_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB116_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB116_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB116_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB116_forward.paired.headclipped.fastq.gz SB116_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB116 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB116_forward.paired.headclipped.fastq.gz
[FLASH]     SB116_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB116.extendedFrags.fastq.gz
[FLASH]     ./SB116.notCombined_1.fastq.gz
[FLASH]     ./SB116.notCombined_2.fastq.gz
[FLASH]     ./SB116.hist
[FLASH]     ./SB116.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2168 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2168
[FLASH]     Combined pairs:   333
[FLASH]     Uncombined pairs: 1835
[FLASH]     Percent combined: 15.36%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.015 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

333 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB116/SB116_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB116_centroids.fasta --uc SB116.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 333 sequences and identified 189 clusters (clustering threshold 1.00) - 3 clusters (representing 124 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB119 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB119_1.fastq /home/working/demultiplexed/SB119_2.fastq SB119_forward.paired.fastq.gz SB119_forward.singletons.fastq.gz SB119_reverse.paired.fastq.gz SB119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB119_1.fastq /home/working/demultiplexed/SB119_2.fastq SB119_forward.paired.fastq.gz SB119_forward.singletons.fastq.gz SB119_reverse.paired.fastq.gz SB119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:12:17 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB119/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 27754 Both Surviving: 5093 (18.35%) Forward Only Surviving: 1118 (4.03%) Reverse Only Surviving: 486 (1.75%) Dropped: 21057 (75.87%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB119_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB119_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB119_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB119_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB119_forward.paired.headclipped.fastq.gz SB119_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB119 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB119_forward.paired.headclipped.fastq.gz
[FLASH]     SB119_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB119.extendedFrags.fastq.gz
[FLASH]     ./SB119.notCombined_1.fastq.gz
[FLASH]     ./SB119.notCombined_2.fastq.gz
[FLASH]     ./SB119.hist
[FLASH]     ./SB119.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5093 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5093
[FLASH]     Combined pairs:   106
[FLASH]     Uncombined pairs: 4987
[FLASH]     Percent combined: 2.08%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.032 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

106 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB119/SB119_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB119_centroids.fasta --uc SB119.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 106 sequences and identified 77 clusters (clustering threshold 1.00) - 1 clusters (representing 27 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB120 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB120_1.fastq /home/working/demultiplexed/SB120_2.fastq SB120_forward.paired.fastq.gz SB120_forward.singletons.fastq.gz SB120_reverse.paired.fastq.gz SB120_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB120_1.fastq /home/working/demultiplexed/SB120_2.fastq SB120_forward.paired.fastq.gz SB120_forward.singletons.fastq.gz SB120_reverse.paired.fastq.gz SB120_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:12:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB120/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 60339 Both Surviving: 12705 (21.06%) Forward Only Surviving: 3855 (6.39%) Reverse Only Surviving: 963 (1.60%) Dropped: 42816 (70.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB120_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB120_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB120_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB120_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB120_forward.paired.headclipped.fastq.gz SB120_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB120 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB120_forward.paired.headclipped.fastq.gz
[FLASH]     SB120_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB120.extendedFrags.fastq.gz
[FLASH]     ./SB120.notCombined_1.fastq.gz
[FLASH]     ./SB120.notCombined_2.fastq.gz
[FLASH]     ./SB120.hist
[FLASH]     ./SB120.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 12705 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      12705
[FLASH]     Combined pairs:   2445
[FLASH]     Uncombined pairs: 10260
[FLASH]     Percent combined: 19.24%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.095 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2445 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB120/SB120_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB120_centroids.fasta --uc SB120.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2445 sequences and identified 1432 clusters (clustering threshold 1.00) - 38 clusters (representing 850 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB127 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB127_1.fastq /home/working/demultiplexed/SB127_2.fastq SB127_forward.paired.fastq.gz SB127_forward.singletons.fastq.gz SB127_reverse.paired.fastq.gz SB127_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB127_1.fastq /home/working/demultiplexed/SB127_2.fastq SB127_forward.paired.fastq.gz SB127_forward.singletons.fastq.gz SB127_reverse.paired.fastq.gz SB127_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:12:34 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB127/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 38679 Both Surviving: 1764 (4.56%) Forward Only Surviving: 696 (1.80%) Reverse Only Surviving: 470 (1.22%) Dropped: 35749 (92.42%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB127_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB127_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB127_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB127_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB127_forward.paired.headclipped.fastq.gz SB127_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB127 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB127_forward.paired.headclipped.fastq.gz
[FLASH]     SB127_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB127.extendedFrags.fastq.gz
[FLASH]     ./SB127.notCombined_1.fastq.gz
[FLASH]     ./SB127.notCombined_2.fastq.gz
[FLASH]     ./SB127.hist
[FLASH]     ./SB127.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1764 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1764
[FLASH]     Combined pairs:   1660
[FLASH]     Uncombined pairs: 104
[FLASH]     Percent combined: 94.10%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.042 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1660 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB127/SB127_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB127_centroids.fasta --uc SB127.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1660 sequences and identified 765 clusters (clustering threshold 1.00) - 24 clusters (representing 749 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB128 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB128_1.fastq /home/working/demultiplexed/SB128_2.fastq SB128_forward.paired.fastq.gz SB128_forward.singletons.fastq.gz SB128_reverse.paired.fastq.gz SB128_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB128_1.fastq /home/working/demultiplexed/SB128_2.fastq SB128_forward.paired.fastq.gz SB128_forward.singletons.fastq.gz SB128_reverse.paired.fastq.gz SB128_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:12:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB128/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 54931 Both Surviving: 9843 (17.92%) Forward Only Surviving: 2614 (4.76%) Reverse Only Surviving: 903 (1.64%) Dropped: 41571 (75.68%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB128_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB128_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB128_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB128_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB128_forward.paired.headclipped.fastq.gz SB128_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB128 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB128_forward.paired.headclipped.fastq.gz
[FLASH]     SB128_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB128.extendedFrags.fastq.gz
[FLASH]     ./SB128.notCombined_1.fastq.gz
[FLASH]     ./SB128.notCombined_2.fastq.gz
[FLASH]     ./SB128.hist
[FLASH]     ./SB128.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9843 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9843
[FLASH]     Combined pairs:   1131
[FLASH]     Uncombined pairs: 8712
[FLASH]     Percent combined: 11.49%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.057 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1131 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB128/SB128_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB128_centroids.fasta --uc SB128.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1131 sequences and identified 596 clusters (clustering threshold 1.00) - 23 clusters (representing 488 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB129 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB129_1.fastq /home/working/demultiplexed/SB129_2.fastq SB129_forward.paired.fastq.gz SB129_forward.singletons.fastq.gz SB129_reverse.paired.fastq.gz SB129_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB129_1.fastq /home/working/demultiplexed/SB129_2.fastq SB129_forward.paired.fastq.gz SB129_forward.singletons.fastq.gz SB129_reverse.paired.fastq.gz SB129_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:12:50 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB129/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41461 Both Surviving: 12224 (29.48%) Forward Only Surviving: 2762 (6.66%) Reverse Only Surviving: 712 (1.72%) Dropped: 25763 (62.14%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB129_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB129_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SB129_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB129_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB129_forward.paired.headclipped.fastq.gz SB129_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB129 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB129_forward.paired.headclipped.fastq.gz
[FLASH]     SB129_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB129.extendedFrags.fastq.gz
[FLASH]     ./SB129.notCombined_1.fastq.gz
[FLASH]     ./SB129.notCombined_2.fastq.gz
[FLASH]     ./SB129.hist
[FLASH]     ./SB129.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 12224 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      12224
[FLASH]     Combined pairs:   4737
[FLASH]     Uncombined pairs: 7487
[FLASH]     Percent combined: 38.75%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.141 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4737 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB129/SB129_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB129_centroids.fasta --uc SB129.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4737 sequences and identified 2274 clusters (clustering threshold 1.00) - 71 clusters (representing 2123 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB13 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB13_1.fastq /home/working/demultiplexed/SB13_2.fastq SB13_forward.paired.fastq.gz SB13_forward.singletons.fastq.gz SB13_reverse.paired.fastq.gz SB13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB13_1.fastq /home/working/demultiplexed/SB13_2.fastq SB13_forward.paired.fastq.gz SB13_forward.singletons.fastq.gz SB13_reverse.paired.fastq.gz SB13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:13:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB13/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 34681 Both Surviving: 6218 (17.93%) Forward Only Surviving: 1604 (4.63%) Reverse Only Surviving: 580 (1.67%) Dropped: 26279 (75.77%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB13_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB13_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB13_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB13_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB13_forward.paired.headclipped.fastq.gz SB13_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB13 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB13_forward.paired.headclipped.fastq.gz
[FLASH]     SB13_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB13.extendedFrags.fastq.gz
[FLASH]     ./SB13.notCombined_1.fastq.gz
[FLASH]     ./SB13.notCombined_2.fastq.gz
[FLASH]     ./SB13.hist
[FLASH]     ./SB13.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6218 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6218
[FLASH]     Combined pairs:   40
[FLASH]     Uncombined pairs: 6178
[FLASH]     Percent combined: 0.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.040 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

40 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB13/SB13_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB13_centroids.fasta --uc SB13.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 40 sequences and identified 38 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB131 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB131_1.fastq /home/working/demultiplexed/SB131_2.fastq SB131_forward.paired.fastq.gz SB131_forward.singletons.fastq.gz SB131_reverse.paired.fastq.gz SB131_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB131_1.fastq /home/working/demultiplexed/SB131_2.fastq SB131_forward.paired.fastq.gz SB131_forward.singletons.fastq.gz SB131_reverse.paired.fastq.gz SB131_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:13:08 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB131/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 40610 Both Surviving: 3564 (8.78%) Forward Only Surviving: 1088 (2.68%) Reverse Only Surviving: 579 (1.43%) Dropped: 35379 (87.12%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB131_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB131_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB131_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB131_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB131_forward.paired.headclipped.fastq.gz SB131_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB131 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB131_forward.paired.headclipped.fastq.gz
[FLASH]     SB131_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB131.extendedFrags.fastq.gz
[FLASH]     ./SB131.notCombined_1.fastq.gz
[FLASH]     ./SB131.notCombined_2.fastq.gz
[FLASH]     ./SB131.hist
[FLASH]     ./SB131.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3564 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3564
[FLASH]     Combined pairs:   95
[FLASH]     Uncombined pairs: 3469
[FLASH]     Percent combined: 2.67%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.022 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

95 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB131/SB131_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB131_centroids.fasta --uc SB131.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 95 sequences and identified 82 clusters (clustering threshold 1.00) - 1 clusters (representing 11 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB135 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB135_1.fastq /home/working/demultiplexed/SB135_2.fastq SB135_forward.paired.fastq.gz SB135_forward.singletons.fastq.gz SB135_reverse.paired.fastq.gz SB135_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB135_1.fastq /home/working/demultiplexed/SB135_2.fastq SB135_forward.paired.fastq.gz SB135_forward.singletons.fastq.gz SB135_reverse.paired.fastq.gz SB135_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:13:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB135/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29244 Both Surviving: 2587 (8.85%) Forward Only Surviving: 787 (2.69%) Reverse Only Surviving: 356 (1.22%) Dropped: 25514 (87.25%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SB135_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SB135_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB135_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB135_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB135_forward.paired.headclipped.fastq.gz SB135_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB135 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB135_forward.paired.headclipped.fastq.gz
[FLASH]     SB135_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB135.extendedFrags.fastq.gz
[FLASH]     ./SB135.notCombined_1.fastq.gz
[FLASH]     ./SB135.notCombined_2.fastq.gz
[FLASH]     ./SB135.hist
[FLASH]     ./SB135.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2587 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2587
[FLASH]     Combined pairs:   277
[FLASH]     Uncombined pairs: 2310
[FLASH]     Percent combined: 10.71%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.015 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

277 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB135/SB135_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB135_centroids.fasta --uc SB135.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 277 sequences and identified 185 clusters (clustering threshold 1.00) - 3 clusters (representing 80 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB136 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB136_1.fastq /home/working/demultiplexed/SB136_2.fastq SB136_forward.paired.fastq.gz SB136_forward.singletons.fastq.gz SB136_reverse.paired.fastq.gz SB136_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB136_1.fastq /home/working/demultiplexed/SB136_2.fastq SB136_forward.paired.fastq.gz SB136_forward.singletons.fastq.gz SB136_reverse.paired.fastq.gz SB136_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:13:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB136/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 43996 Both Surviving: 15162 (34.46%) Forward Only Surviving: 3126 (7.11%) Reverse Only Surviving: 704 (1.60%) Dropped: 25004 (56.83%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB136_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB136_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB136_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB136_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB136_forward.paired.headclipped.fastq.gz SB136_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB136 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB136_forward.paired.headclipped.fastq.gz
[FLASH]     SB136_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB136.extendedFrags.fastq.gz
[FLASH]     ./SB136.notCombined_1.fastq.gz
[FLASH]     ./SB136.notCombined_2.fastq.gz
[FLASH]     ./SB136.hist
[FLASH]     ./SB136.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 15162 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      15162
[FLASH]     Combined pairs:   8678
[FLASH]     Uncombined pairs: 6484
[FLASH]     Percent combined: 57.24%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.237 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

8678 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB136/SB136_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB136_centroids.fasta --uc SB136.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 8678 sequences and identified 3769 clusters (clustering threshold 1.00) - 160 clusters (representing 4524 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB138 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB138_1.fastq /home/working/demultiplexed/SB138_2.fastq SB138_forward.paired.fastq.gz SB138_forward.singletons.fastq.gz SB138_reverse.paired.fastq.gz SB138_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB138_1.fastq /home/working/demultiplexed/SB138_2.fastq SB138_forward.paired.fastq.gz SB138_forward.singletons.fastq.gz SB138_reverse.paired.fastq.gz SB138_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:13:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB138/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 35038 Both Surviving: 11925 (34.03%) Forward Only Surviving: 4745 (13.54%) Reverse Only Surviving: 416 (1.19%) Dropped: 17952 (51.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB138_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB138_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB138_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB138_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB138_forward.paired.headclipped.fastq.gz SB138_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB138 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB138_forward.paired.headclipped.fastq.gz
[FLASH]     SB138_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB138.extendedFrags.fastq.gz
[FLASH]     ./SB138.notCombined_1.fastq.gz
[FLASH]     ./SB138.notCombined_2.fastq.gz
[FLASH]     ./SB138.hist
[FLASH]     ./SB138.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11925 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11925
[FLASH]     Combined pairs:   7192
[FLASH]     Uncombined pairs: 4733
[FLASH]     Percent combined: 60.31%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.205 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

7192 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB138/SB138_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB138_centroids.fasta --uc SB138.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 7192 sequences and identified 2542 clusters (clustering threshold 1.00) - 182 clusters (representing 4448 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB139 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB139_1.fastq /home/working/demultiplexed/SB139_2.fastq SB139_forward.paired.fastq.gz SB139_forward.singletons.fastq.gz SB139_reverse.paired.fastq.gz SB139_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB139_1.fastq /home/working/demultiplexed/SB139_2.fastq SB139_forward.paired.fastq.gz SB139_forward.singletons.fastq.gz SB139_reverse.paired.fastq.gz SB139_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:13:52 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB139/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 32952 Both Surviving: 6139 (18.63%) Forward Only Surviving: 1682 (5.10%) Reverse Only Surviving: 480 (1.46%) Dropped: 24651 (74.81%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB139_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB139_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB139_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB139_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB139_forward.paired.headclipped.fastq.gz SB139_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB139 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB139_forward.paired.headclipped.fastq.gz
[FLASH]     SB139_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB139.extendedFrags.fastq.gz
[FLASH]     ./SB139.notCombined_1.fastq.gz
[FLASH]     ./SB139.notCombined_2.fastq.gz
[FLASH]     ./SB139.hist
[FLASH]     ./SB139.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6139 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6139
[FLASH]     Combined pairs:   3251
[FLASH]     Uncombined pairs: 2888
[FLASH]     Percent combined: 52.96%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.101 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3251 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB139/SB139_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB139_centroids.fasta --uc SB139.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3251 sequences and identified 1707 clusters (clustering threshold 1.00) - 31 clusters (representing 1401 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB140 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB140_1.fastq /home/working/demultiplexed/SB140_2.fastq SB140_forward.paired.fastq.gz SB140_forward.singletons.fastq.gz SB140_reverse.paired.fastq.gz SB140_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB140_1.fastq /home/working/demultiplexed/SB140_2.fastq SB140_forward.paired.fastq.gz SB140_forward.singletons.fastq.gz SB140_reverse.paired.fastq.gz SB140_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:14:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB140/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11730 Both Surviving: 4648 (39.62%) Forward Only Surviving: 718 (6.12%) Reverse Only Surviving: 205 (1.75%) Dropped: 6159 (52.51%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB140_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB140_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB140_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB140_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB140_forward.paired.headclipped.fastq.gz SB140_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB140 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB140_forward.paired.headclipped.fastq.gz
[FLASH]     SB140_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB140.extendedFrags.fastq.gz
[FLASH]     ./SB140.notCombined_1.fastq.gz
[FLASH]     ./SB140.notCombined_2.fastq.gz
[FLASH]     ./SB140.hist
[FLASH]     ./SB140.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4648 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4648
[FLASH]     Combined pairs:   4230
[FLASH]     Uncombined pairs: 418
[FLASH]     Percent combined: 91.01%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.112 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4230 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB140/SB140_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB140_centroids.fasta --uc SB140.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4230 sequences and identified 1791 clusters (clustering threshold 1.00) - 140 clusters (representing 2275 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB141 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB141_1.fastq /home/working/demultiplexed/SB141_2.fastq SB141_forward.paired.fastq.gz SB141_forward.singletons.fastq.gz SB141_reverse.paired.fastq.gz SB141_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB141_1.fastq /home/working/demultiplexed/SB141_2.fastq SB141_forward.paired.fastq.gz SB141_forward.singletons.fastq.gz SB141_reverse.paired.fastq.gz SB141_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:14:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB141/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9355 Both Surviving: 173 (1.85%) Forward Only Surviving: 141 (1.51%) Reverse Only Surviving: 88 (0.94%) Dropped: 8953 (95.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB141_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB141_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SB141_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SB141_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB141_forward.paired.headclipped.fastq.gz SB141_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB141 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB141_forward.paired.headclipped.fastq.gz
[FLASH]     SB141_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB141.extendedFrags.fastq.gz
[FLASH]     ./SB141.notCombined_1.fastq.gz
[FLASH]     ./SB141.notCombined_2.fastq.gz
[FLASH]     ./SB141.hist
[FLASH]     ./SB141.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 173 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      173
[FLASH]     Combined pairs:   44
[FLASH]     Uncombined pairs: 129
[FLASH]     Percent combined: 25.43%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

44 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB141/SB141_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB141_centroids.fasta --uc SB141.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 44 sequences and identified 39 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB142 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB142_1.fastq /home/working/demultiplexed/SB142_2.fastq SB142_forward.paired.fastq.gz SB142_forward.singletons.fastq.gz SB142_reverse.paired.fastq.gz SB142_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB142_1.fastq /home/working/demultiplexed/SB142_2.fastq SB142_forward.paired.fastq.gz SB142_forward.singletons.fastq.gz SB142_reverse.paired.fastq.gz SB142_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:14:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB142/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22361 Both Surviving: 5354 (23.94%) Forward Only Surviving: 1256 (5.62%) Reverse Only Surviving: 779 (3.48%) Dropped: 14972 (66.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SB142_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB142_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB142_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB142_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB142_forward.paired.headclipped.fastq.gz SB142_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB142 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB142_forward.paired.headclipped.fastq.gz
[FLASH]     SB142_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB142.extendedFrags.fastq.gz
[FLASH]     ./SB142.notCombined_1.fastq.gz
[FLASH]     ./SB142.notCombined_2.fastq.gz
[FLASH]     ./SB142.hist
[FLASH]     ./SB142.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5354 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5354
[FLASH]     Combined pairs:   293
[FLASH]     Uncombined pairs: 5061
[FLASH]     Percent combined: 5.47%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.041 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

293 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB142/SB142_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB142_centroids.fasta --uc SB142.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 293 sequences and identified 239 clusters (clustering threshold 1.00) - 5 clusters (representing 36 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB146 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB146_1.fastq /home/working/demultiplexed/SB146_2.fastq SB146_forward.paired.fastq.gz SB146_forward.singletons.fastq.gz SB146_reverse.paired.fastq.gz SB146_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB146_1.fastq /home/working/demultiplexed/SB146_2.fastq SB146_forward.paired.fastq.gz SB146_forward.singletons.fastq.gz SB146_reverse.paired.fastq.gz SB146_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:14:17 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB146/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31692 Both Surviving: 5584 (17.62%) Forward Only Surviving: 1183 (3.73%) Reverse Only Surviving: 530 (1.67%) Dropped: 24395 (76.98%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB146_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB146_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB146_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB146_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB146_forward.paired.headclipped.fastq.gz SB146_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB146 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB146_forward.paired.headclipped.fastq.gz
[FLASH]     SB146_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB146.extendedFrags.fastq.gz
[FLASH]     ./SB146.notCombined_1.fastq.gz
[FLASH]     ./SB146.notCombined_2.fastq.gz
[FLASH]     ./SB146.hist
[FLASH]     ./SB146.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5584 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5584
[FLASH]     Combined pairs:   208
[FLASH]     Uncombined pairs: 5376
[FLASH]     Percent combined: 3.72%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.045 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

208 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB146/SB146_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB146_centroids.fasta --uc SB146.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 208 sequences and identified 163 clusters (clustering threshold 1.00) - 1 clusters (representing 32 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB147 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB147_1.fastq /home/working/demultiplexed/SB147_2.fastq SB147_forward.paired.fastq.gz SB147_forward.singletons.fastq.gz SB147_reverse.paired.fastq.gz SB147_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB147_1.fastq /home/working/demultiplexed/SB147_2.fastq SB147_forward.paired.fastq.gz SB147_forward.singletons.fastq.gz SB147_reverse.paired.fastq.gz SB147_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:14:23 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB147/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 32021 Both Surviving: 8758 (27.35%) Forward Only Surviving: 2026 (6.33%) Reverse Only Surviving: 496 (1.55%) Dropped: 20741 (64.77%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB147_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB147_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB147_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB147_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB147_forward.paired.headclipped.fastq.gz SB147_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB147 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB147_forward.paired.headclipped.fastq.gz
[FLASH]     SB147_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB147.extendedFrags.fastq.gz
[FLASH]     ./SB147.notCombined_1.fastq.gz
[FLASH]     ./SB147.notCombined_2.fastq.gz
[FLASH]     ./SB147.hist
[FLASH]     ./SB147.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8758 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8758
[FLASH]     Combined pairs:   3053
[FLASH]     Uncombined pairs: 5705
[FLASH]     Percent combined: 34.86%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.117 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3053 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB147/SB147_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB147_centroids.fasta --uc SB147.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3053 sequences and identified 1676 clusters (clustering threshold 1.00) - 32 clusters (representing 1242 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB15 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB15_1.fastq /home/working/demultiplexed/SB15_2.fastq SB15_forward.paired.fastq.gz SB15_forward.singletons.fastq.gz SB15_reverse.paired.fastq.gz SB15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB15_1.fastq /home/working/demultiplexed/SB15_2.fastq SB15_forward.paired.fastq.gz SB15_forward.singletons.fastq.gz SB15_reverse.paired.fastq.gz SB15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:14:35 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB15/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 65483 Both Surviving: 3843 (5.87%) Forward Only Surviving: 1745 (2.66%) Reverse Only Surviving: 903 (1.38%) Dropped: 58992 (90.09%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SB15_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SB15_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB15_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB15_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB15_forward.paired.headclipped.fastq.gz SB15_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB15 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB15_forward.paired.headclipped.fastq.gz
[FLASH]     SB15_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB15.extendedFrags.fastq.gz
[FLASH]     ./SB15.notCombined_1.fastq.gz
[FLASH]     ./SB15.notCombined_2.fastq.gz
[FLASH]     ./SB15.hist
[FLASH]     ./SB15.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3843 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3843
[FLASH]     Combined pairs:   500
[FLASH]     Uncombined pairs: 3343
[FLASH]     Percent combined: 13.01%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.049 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

500 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB15/SB15_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB15_centroids.fasta --uc SB15.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 500 sequences and identified 264 clusters (clustering threshold 1.00) - 8 clusters (representing 214 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB150 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB150_1.fastq /home/working/demultiplexed/SB150_2.fastq SB150_forward.paired.fastq.gz SB150_forward.singletons.fastq.gz SB150_reverse.paired.fastq.gz SB150_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB150_1.fastq /home/working/demultiplexed/SB150_2.fastq SB150_forward.paired.fastq.gz SB150_forward.singletons.fastq.gz SB150_reverse.paired.fastq.gz SB150_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:14:42 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB150/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 36268 Both Surviving: 8292 (22.86%) Forward Only Surviving: 2052 (5.66%) Reverse Only Surviving: 552 (1.52%) Dropped: 25372 (69.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB150_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB150_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB150_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB150_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB150_forward.paired.headclipped.fastq.gz SB150_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB150 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB150_forward.paired.headclipped.fastq.gz
[FLASH]     SB150_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB150.extendedFrags.fastq.gz
[FLASH]     ./SB150.notCombined_1.fastq.gz
[FLASH]     ./SB150.notCombined_2.fastq.gz
[FLASH]     ./SB150.hist
[FLASH]     ./SB150.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8292 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8292
[FLASH]     Combined pairs:   3195
[FLASH]     Uncombined pairs: 5097
[FLASH]     Percent combined: 38.53%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.123 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3195 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB150/SB150_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB150_centroids.fasta --uc SB150.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3195 sequences and identified 1886 clusters (clustering threshold 1.00) - 37 clusters (representing 1107 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB153 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB153_1.fastq /home/working/demultiplexed/SB153_2.fastq SB153_forward.paired.fastq.gz SB153_forward.singletons.fastq.gz SB153_reverse.paired.fastq.gz SB153_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB153_1.fastq /home/working/demultiplexed/SB153_2.fastq SB153_forward.paired.fastq.gz SB153_forward.singletons.fastq.gz SB153_reverse.paired.fastq.gz SB153_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:14:54 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB153/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41820 Both Surviving: 11773 (28.15%) Forward Only Surviving: 3169 (7.58%) Reverse Only Surviving: 853 (2.04%) Dropped: 26025 (62.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB153_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB153_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB153_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB153_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB153_forward.paired.headclipped.fastq.gz SB153_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB153 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB153_forward.paired.headclipped.fastq.gz
[FLASH]     SB153_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB153.extendedFrags.fastq.gz
[FLASH]     ./SB153.notCombined_1.fastq.gz
[FLASH]     ./SB153.notCombined_2.fastq.gz
[FLASH]     ./SB153.hist
[FLASH]     ./SB153.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11773 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11773
[FLASH]     Combined pairs:   1450
[FLASH]     Uncombined pairs: 10323
[FLASH]     Percent combined: 12.32%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.114 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1450 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB153/SB153_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB153_centroids.fasta --uc SB153.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1450 sequences and identified 839 clusters (clustering threshold 1.00) - 18 clusters (representing 558 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB159 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB159_1.fastq /home/working/demultiplexed/SB159_2.fastq SB159_forward.paired.fastq.gz SB159_forward.singletons.fastq.gz SB159_reverse.paired.fastq.gz SB159_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB159_1.fastq /home/working/demultiplexed/SB159_2.fastq SB159_forward.paired.fastq.gz SB159_forward.singletons.fastq.gz SB159_reverse.paired.fastq.gz SB159_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:15:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB159/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37399 Both Surviving: 7205 (19.27%) Forward Only Surviving: 1797 (4.80%) Reverse Only Surviving: 683 (1.83%) Dropped: 27714 (74.10%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB159_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB159_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB159_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB159_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB159_forward.paired.headclipped.fastq.gz SB159_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB159 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB159_forward.paired.headclipped.fastq.gz
[FLASH]     SB159_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB159.extendedFrags.fastq.gz
[FLASH]     ./SB159.notCombined_1.fastq.gz
[FLASH]     ./SB159.notCombined_2.fastq.gz
[FLASH]     ./SB159.hist
[FLASH]     ./SB159.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7205 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7205
[FLASH]     Combined pairs:   60
[FLASH]     Uncombined pairs: 7145
[FLASH]     Percent combined: 0.83%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.057 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

60 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB159/SB159_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB159_centroids.fasta --uc SB159.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 60 sequences and identified 51 clusters (clustering threshold 1.00) - 1 clusters (representing 9 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB163 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB163_1.fastq /home/working/demultiplexed/SB163_2.fastq SB163_forward.paired.fastq.gz SB163_forward.singletons.fastq.gz SB163_reverse.paired.fastq.gz SB163_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB163_1.fastq /home/working/demultiplexed/SB163_2.fastq SB163_forward.paired.fastq.gz SB163_forward.singletons.fastq.gz SB163_reverse.paired.fastq.gz SB163_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:15:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB163/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 80933 Both Surviving: 14908 (18.42%) Forward Only Surviving: 5632 (6.96%) Reverse Only Surviving: 1070 (1.32%) Dropped: 59323 (73.30%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SB163_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SB163_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB163_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB163_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB163_forward.paired.headclipped.fastq.gz SB163_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB163 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB163_forward.paired.headclipped.fastq.gz
[FLASH]     SB163_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB163.extendedFrags.fastq.gz
[FLASH]     ./SB163.notCombined_1.fastq.gz
[FLASH]     ./SB163.notCombined_2.fastq.gz
[FLASH]     ./SB163.hist
[FLASH]     ./SB163.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 14908 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      14908
[FLASH]     Combined pairs:   8755
[FLASH]     Uncombined pairs: 6153
[FLASH]     Percent combined: 58.73%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.346 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

8755 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB163/SB163_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB163_centroids.fasta --uc SB163.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 8755 sequences and identified 2594 clusters (clustering threshold 1.00) - 208 clusters (representing 5810 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB166 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB166_1.fastq /home/working/demultiplexed/SB166_2.fastq SB166_forward.paired.fastq.gz SB166_forward.singletons.fastq.gz SB166_reverse.paired.fastq.gz SB166_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB166_1.fastq /home/working/demultiplexed/SB166_2.fastq SB166_forward.paired.fastq.gz SB166_forward.singletons.fastq.gz SB166_reverse.paired.fastq.gz SB166_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:15:40 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB166/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 24390 Both Surviving: 5441 (22.31%) Forward Only Surviving: 1502 (6.16%) Reverse Only Surviving: 319 (1.31%) Dropped: 17128 (70.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB166_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB166_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB166_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB166_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB166_forward.paired.headclipped.fastq.gz SB166_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB166 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB166_forward.paired.headclipped.fastq.gz
[FLASH]     SB166_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB166.extendedFrags.fastq.gz
[FLASH]     ./SB166.notCombined_1.fastq.gz
[FLASH]     ./SB166.notCombined_2.fastq.gz
[FLASH]     ./SB166.hist
[FLASH]     ./SB166.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5441 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5441
[FLASH]     Combined pairs:   2584
[FLASH]     Uncombined pairs: 2857
[FLASH]     Percent combined: 47.49%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.092 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2584 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB166/SB166_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB166_centroids.fasta --uc SB166.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2584 sequences and identified 1272 clusters (clustering threshold 1.00) - 24 clusters (representing 1096 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB170 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB170_1.fastq /home/working/demultiplexed/SB170_2.fastq SB170_forward.paired.fastq.gz SB170_forward.singletons.fastq.gz SB170_reverse.paired.fastq.gz SB170_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB170_1.fastq /home/working/demultiplexed/SB170_2.fastq SB170_forward.paired.fastq.gz SB170_forward.singletons.fastq.gz SB170_reverse.paired.fastq.gz SB170_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:15:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB170/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 40667 Both Surviving: 9713 (23.88%) Forward Only Surviving: 2448 (6.02%) Reverse Only Surviving: 653 (1.61%) Dropped: 27853 (68.49%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB170_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB170_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB170_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB170_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB170_forward.paired.headclipped.fastq.gz SB170_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB170 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB170_forward.paired.headclipped.fastq.gz
[FLASH]     SB170_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB170.extendedFrags.fastq.gz
[FLASH]     ./SB170.notCombined_1.fastq.gz
[FLASH]     ./SB170.notCombined_2.fastq.gz
[FLASH]     ./SB170.hist
[FLASH]     ./SB170.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9713 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9713
[FLASH]     Combined pairs:   2788
[FLASH]     Uncombined pairs: 6925
[FLASH]     Percent combined: 28.70%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.119 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2788 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB170/SB170_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB170_centroids.fasta --uc SB170.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2788 sequences and identified 1482 clusters (clustering threshold 1.00) - 26 clusters (representing 1114 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB171 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB171_1.fastq /home/working/demultiplexed/SB171_2.fastq SB171_forward.paired.fastq.gz SB171_forward.singletons.fastq.gz SB171_reverse.paired.fastq.gz SB171_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB171_1.fastq /home/working/demultiplexed/SB171_2.fastq SB171_forward.paired.fastq.gz SB171_forward.singletons.fastq.gz SB171_reverse.paired.fastq.gz SB171_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:16:00 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB171/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14032 Both Surviving: 3915 (27.90%) Forward Only Surviving: 1302 (9.28%) Reverse Only Surviving: 152 (1.08%) Dropped: 8663 (61.74%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB171_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB171_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SB171_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SB171_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB171_forward.paired.headclipped.fastq.gz SB171_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB171 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB171_forward.paired.headclipped.fastq.gz
[FLASH]     SB171_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB171.extendedFrags.fastq.gz
[FLASH]     ./SB171.notCombined_1.fastq.gz
[FLASH]     ./SB171.notCombined_2.fastq.gz
[FLASH]     ./SB171.hist
[FLASH]     ./SB171.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3915 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3915
[FLASH]     Combined pairs:   265
[FLASH]     Uncombined pairs: 3650
[FLASH]     Percent combined: 6.77%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.046 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

265 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB171/SB171_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB171_centroids.fasta --uc SB171.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 265 sequences and identified 233 clusters (clustering threshold 1.00) - 2 clusters (representing 17 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB172 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB172_1.fastq /home/working/demultiplexed/SB172_2.fastq SB172_forward.paired.fastq.gz SB172_forward.singletons.fastq.gz SB172_reverse.paired.fastq.gz SB172_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB172_1.fastq /home/working/demultiplexed/SB172_2.fastq SB172_forward.paired.fastq.gz SB172_forward.singletons.fastq.gz SB172_reverse.paired.fastq.gz SB172_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:16:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB172/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 43254 Both Surviving: 10559 (24.41%) Forward Only Surviving: 2599 (6.01%) Reverse Only Surviving: 1370 (3.17%) Dropped: 28726 (66.41%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SB172_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB172_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB172_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB172_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB172_forward.paired.headclipped.fastq.gz SB172_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB172 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB172_forward.paired.headclipped.fastq.gz
[FLASH]     SB172_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB172.extendedFrags.fastq.gz
[FLASH]     ./SB172.notCombined_1.fastq.gz
[FLASH]     ./SB172.notCombined_2.fastq.gz
[FLASH]     ./SB172.hist
[FLASH]     ./SB172.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10559 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10559
[FLASH]     Combined pairs:   737
[FLASH]     Uncombined pairs: 9822
[FLASH]     Percent combined: 6.98%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.100 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

737 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB172/SB172_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB172_centroids.fasta --uc SB172.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 737 sequences and identified 480 clusters (clustering threshold 1.00) - 11 clusters (representing 227 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB180 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB180_1.fastq /home/working/demultiplexed/SB180_2.fastq SB180_forward.paired.fastq.gz SB180_forward.singletons.fastq.gz SB180_reverse.paired.fastq.gz SB180_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB180_1.fastq /home/working/demultiplexed/SB180_2.fastq SB180_forward.paired.fastq.gz SB180_forward.singletons.fastq.gz SB180_reverse.paired.fastq.gz SB180_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:16:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB180/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 34105 Both Surviving: 6549 (19.20%) Forward Only Surviving: 1906 (5.59%) Reverse Only Surviving: 465 (1.36%) Dropped: 25185 (73.85%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB180_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB180_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB180_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB180_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB180_forward.paired.headclipped.fastq.gz SB180_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB180 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB180_forward.paired.headclipped.fastq.gz
[FLASH]     SB180_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB180.extendedFrags.fastq.gz
[FLASH]     ./SB180.notCombined_1.fastq.gz
[FLASH]     ./SB180.notCombined_2.fastq.gz
[FLASH]     ./SB180.hist
[FLASH]     ./SB180.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6549 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6549
[FLASH]     Combined pairs:   2670
[FLASH]     Uncombined pairs: 3879
[FLASH]     Percent combined: 40.77%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.091 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2670 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB180/SB180_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB180_centroids.fasta --uc SB180.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2670 sequences and identified 1420 clusters (clustering threshold 1.00) - 24 clusters (representing 1121 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB182 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB182_1.fastq /home/working/demultiplexed/SB182_2.fastq SB182_forward.paired.fastq.gz SB182_forward.singletons.fastq.gz SB182_reverse.paired.fastq.gz SB182_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB182_1.fastq /home/working/demultiplexed/SB182_2.fastq SB182_forward.paired.fastq.gz SB182_forward.singletons.fastq.gz SB182_reverse.paired.fastq.gz SB182_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:16:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB182/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 49576 Both Surviving: 154 (0.31%) Forward Only Surviving: 517 (1.04%) Reverse Only Surviving: 627 (1.26%) Dropped: 48278 (97.38%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB182_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB182_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB182_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB182_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB182_forward.paired.headclipped.fastq.gz SB182_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB182 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB182_forward.paired.headclipped.fastq.gz
[FLASH]     SB182_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB182.extendedFrags.fastq.gz
[FLASH]     ./SB182.notCombined_1.fastq.gz
[FLASH]     ./SB182.notCombined_2.fastq.gz
[FLASH]     ./SB182.hist
[FLASH]     ./SB182.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 154 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      154
[FLASH]     Combined pairs:   54
[FLASH]     Uncombined pairs: 100
[FLASH]     Percent combined: 35.06%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

54 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB182/SB182_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB182_centroids.fasta --uc SB182.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 54 sequences and identified 45 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB183 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB183_1.fastq /home/working/demultiplexed/SB183_2.fastq SB183_forward.paired.fastq.gz SB183_forward.singletons.fastq.gz SB183_reverse.paired.fastq.gz SB183_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB183_1.fastq /home/working/demultiplexed/SB183_2.fastq SB183_forward.paired.fastq.gz SB183_forward.singletons.fastq.gz SB183_reverse.paired.fastq.gz SB183_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:16:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB183/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14364 Both Surviving: 316 (2.20%) Forward Only Surviving: 343 (2.39%) Reverse Only Surviving: 130 (0.91%) Dropped: 13575 (94.51%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB183_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB183_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB183_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB183_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB183_forward.paired.headclipped.fastq.gz SB183_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB183 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB183_forward.paired.headclipped.fastq.gz
[FLASH]     SB183_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB183.extendedFrags.fastq.gz
[FLASH]     ./SB183.notCombined_1.fastq.gz
[FLASH]     ./SB183.notCombined_2.fastq.gz
[FLASH]     ./SB183.hist
[FLASH]     ./SB183.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 316 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      316
[FLASH]     Combined pairs:   237
[FLASH]     Uncombined pairs: 79
[FLASH]     Percent combined: 75.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.008 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

237 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB183/SB183_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB183_centroids.fasta --uc SB183.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 237 sequences and identified 140 clusters (clustering threshold 1.00) - 7 clusters (representing 91 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB186 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB186_1.fastq /home/working/demultiplexed/SB186_2.fastq SB186_forward.paired.fastq.gz SB186_forward.singletons.fastq.gz SB186_reverse.paired.fastq.gz SB186_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB186_1.fastq /home/working/demultiplexed/SB186_2.fastq SB186_forward.paired.fastq.gz SB186_forward.singletons.fastq.gz SB186_reverse.paired.fastq.gz SB186_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:16:35 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB186/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 72012 Both Surviving: 3107 (4.31%) Forward Only Surviving: 2444 (3.39%) Reverse Only Surviving: 755 (1.05%) Dropped: 65706 (91.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB186_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB186_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB186_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB186_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB186_forward.paired.headclipped.fastq.gz SB186_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB186 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB186_forward.paired.headclipped.fastq.gz
[FLASH]     SB186_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB186.extendedFrags.fastq.gz
[FLASH]     ./SB186.notCombined_1.fastq.gz
[FLASH]     ./SB186.notCombined_2.fastq.gz
[FLASH]     ./SB186.hist
[FLASH]     ./SB186.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3107 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3107
[FLASH]     Combined pairs:   2217
[FLASH]     Uncombined pairs: 890
[FLASH]     Percent combined: 71.36%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.059 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2217 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB186/SB186_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB186_centroids.fasta --uc SB186.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2217 sequences and identified 829 clusters (clustering threshold 1.00) - 35 clusters (representing 1212 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB194 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB194_1.fastq /home/working/demultiplexed/SB194_2.fastq SB194_forward.paired.fastq.gz SB194_forward.singletons.fastq.gz SB194_reverse.paired.fastq.gz SB194_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB194_1.fastq /home/working/demultiplexed/SB194_2.fastq SB194_forward.paired.fastq.gz SB194_forward.singletons.fastq.gz SB194_reverse.paired.fastq.gz SB194_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:16:47 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB194/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 97194 Both Surviving: 24529 (25.24%) Forward Only Surviving: 8080 (8.31%) Reverse Only Surviving: 1643 (1.69%) Dropped: 62942 (64.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB194_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB194_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB194_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB194_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB194_forward.paired.headclipped.fastq.gz SB194_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB194 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB194_forward.paired.headclipped.fastq.gz
[FLASH]     SB194_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB194.extendedFrags.fastq.gz
[FLASH]     ./SB194.notCombined_1.fastq.gz
[FLASH]     ./SB194.notCombined_2.fastq.gz
[FLASH]     ./SB194.hist
[FLASH]     ./SB194.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 24529 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      24529
[FLASH]     Combined pairs:   2260
[FLASH]     Uncombined pairs: 22269
[FLASH]     Percent combined: 9.21%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.183 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2260 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB194/SB194_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB194_centroids.fasta --uc SB194.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2260 sequences and identified 1427 clusters (clustering threshold 1.00) - 32 clusters (representing 629 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB195 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB195_1.fastq /home/working/demultiplexed/SB195_2.fastq SB195_forward.paired.fastq.gz SB195_forward.singletons.fastq.gz SB195_reverse.paired.fastq.gz SB195_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB195_1.fastq /home/working/demultiplexed/SB195_2.fastq SB195_forward.paired.fastq.gz SB195_forward.singletons.fastq.gz SB195_reverse.paired.fastq.gz SB195_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:17:04 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB195/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25672 Both Surviving: 6139 (23.91%) Forward Only Surviving: 920 (3.58%) Reverse Only Surviving: 954 (3.72%) Dropped: 17659 (68.79%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SB195_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB195_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB195_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB195_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB195_forward.paired.headclipped.fastq.gz SB195_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB195 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB195_forward.paired.headclipped.fastq.gz
[FLASH]     SB195_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB195.extendedFrags.fastq.gz
[FLASH]     ./SB195.notCombined_1.fastq.gz
[FLASH]     ./SB195.notCombined_2.fastq.gz
[FLASH]     ./SB195.hist
[FLASH]     ./SB195.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6139 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6139
[FLASH]     Combined pairs:   198
[FLASH]     Uncombined pairs: 5941
[FLASH]     Percent combined: 3.23%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.057 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

198 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB195/SB195_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB195_centroids.fasta --uc SB195.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 198 sequences and identified 155 clusters (clustering threshold 1.00) - 3 clusters (representing 27 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB197 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB197_1.fastq /home/working/demultiplexed/SB197_2.fastq SB197_forward.paired.fastq.gz SB197_forward.singletons.fastq.gz SB197_reverse.paired.fastq.gz SB197_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB197_1.fastq /home/working/demultiplexed/SB197_2.fastq SB197_forward.paired.fastq.gz SB197_forward.singletons.fastq.gz SB197_reverse.paired.fastq.gz SB197_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:17:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB197/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 53032 Both Surviving: 13391 (25.25%) Forward Only Surviving: 3997 (7.54%) Reverse Only Surviving: 831 (1.57%) Dropped: 34813 (65.65%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB197_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB197_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB197_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB197_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB197_forward.paired.headclipped.fastq.gz SB197_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB197 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB197_forward.paired.headclipped.fastq.gz
[FLASH]     SB197_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB197.extendedFrags.fastq.gz
[FLASH]     ./SB197.notCombined_1.fastq.gz
[FLASH]     ./SB197.notCombined_2.fastq.gz
[FLASH]     ./SB197.hist
[FLASH]     ./SB197.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 13391 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      13391
[FLASH]     Combined pairs:   461
[FLASH]     Uncombined pairs: 12930
[FLASH]     Percent combined: 3.44%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.092 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

461 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB197/SB197_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB197_centroids.fasta --uc SB197.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 461 sequences and identified 319 clusters (clustering threshold 1.00) - 9 clusters (representing 105 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB199 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB199_1.fastq /home/working/demultiplexed/SB199_2.fastq SB199_forward.paired.fastq.gz SB199_forward.singletons.fastq.gz SB199_reverse.paired.fastq.gz SB199_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB199_1.fastq /home/working/demultiplexed/SB199_2.fastq SB199_forward.paired.fastq.gz SB199_forward.singletons.fastq.gz SB199_reverse.paired.fastq.gz SB199_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:17:19 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB199/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20753 Both Surviving: 2795 (13.47%) Forward Only Surviving: 931 (4.49%) Reverse Only Surviving: 313 (1.51%) Dropped: 16714 (80.54%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB199_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB199_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB199_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB199_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB199_forward.paired.headclipped.fastq.gz SB199_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB199 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB199_forward.paired.headclipped.fastq.gz
[FLASH]     SB199_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB199.extendedFrags.fastq.gz
[FLASH]     ./SB199.notCombined_1.fastq.gz
[FLASH]     ./SB199.notCombined_2.fastq.gz
[FLASH]     ./SB199.hist
[FLASH]     ./SB199.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2795 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2795
[FLASH]     Combined pairs:   56
[FLASH]     Uncombined pairs: 2739
[FLASH]     Percent combined: 2.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.025 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

56 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB199/SB199_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB199_centroids.fasta --uc SB199.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 56 sequences and identified 49 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB200 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB200_1.fastq /home/working/demultiplexed/SB200_2.fastq SB200_forward.paired.fastq.gz SB200_forward.singletons.fastq.gz SB200_reverse.paired.fastq.gz SB200_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB200_1.fastq /home/working/demultiplexed/SB200_2.fastq SB200_forward.paired.fastq.gz SB200_forward.singletons.fastq.gz SB200_reverse.paired.fastq.gz SB200_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:17:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB200/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SB200_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB200_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB200_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB200_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB200_forward.paired.headclipped.fastq.gz SB200_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB200 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB200_forward.paired.headclipped.fastq.gz
[FLASH]     SB200_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB200.extendedFrags.fastq.gz
[FLASH]     ./SB200.notCombined_1.fastq.gz
[FLASH]     ./SB200.notCombined_2.fastq.gz
[FLASH]     ./SB200.hist
[FLASH]     ./SB200.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB200/SB200_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB200_centroids.fasta --uc SB200.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB201 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB201_1.fastq /home/working/demultiplexed/SB201_2.fastq SB201_forward.paired.fastq.gz SB201_forward.singletons.fastq.gz SB201_reverse.paired.fastq.gz SB201_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB201_1.fastq /home/working/demultiplexed/SB201_2.fastq SB201_forward.paired.fastq.gz SB201_forward.singletons.fastq.gz SB201_reverse.paired.fastq.gz SB201_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:17:23 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB201/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 33178 Both Surviving: 1958 (5.90%) Forward Only Surviving: 1408 (4.24%) Reverse Only Surviving: 279 (0.84%) Dropped: 29533 (89.01%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB201_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB201_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB201_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB201_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB201_forward.paired.headclipped.fastq.gz SB201_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB201 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB201_forward.paired.headclipped.fastq.gz
[FLASH]     SB201_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB201.extendedFrags.fastq.gz
[FLASH]     ./SB201.notCombined_1.fastq.gz
[FLASH]     ./SB201.notCombined_2.fastq.gz
[FLASH]     ./SB201.hist
[FLASH]     ./SB201.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1958 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1958
[FLASH]     Combined pairs:   826
[FLASH]     Uncombined pairs: 1132
[FLASH]     Percent combined: 42.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.029 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

826 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB201/SB201_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB201_centroids.fasta --uc SB201.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 826 sequences and identified 569 clusters (clustering threshold 1.00) - 19 clusters (representing 228 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB204 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB204_1.fastq /home/working/demultiplexed/SB204_2.fastq SB204_forward.paired.fastq.gz SB204_forward.singletons.fastq.gz SB204_reverse.paired.fastq.gz SB204_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB204_1.fastq /home/working/demultiplexed/SB204_2.fastq SB204_forward.paired.fastq.gz SB204_forward.singletons.fastq.gz SB204_reverse.paired.fastq.gz SB204_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:17:32 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB204/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 99793 Both Surviving: 5432 (5.44%) Forward Only Surviving: 2232 (2.24%) Reverse Only Surviving: 1231 (1.23%) Dropped: 90898 (91.09%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB204_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB204_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB204_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB204_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB204_forward.paired.headclipped.fastq.gz SB204_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB204 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB204_forward.paired.headclipped.fastq.gz
[FLASH]     SB204_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB204.extendedFrags.fastq.gz
[FLASH]     ./SB204.notCombined_1.fastq.gz
[FLASH]     ./SB204.notCombined_2.fastq.gz
[FLASH]     ./SB204.hist
[FLASH]     ./SB204.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5432 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5432
[FLASH]     Combined pairs:   4767
[FLASH]     Uncombined pairs: 665
[FLASH]     Percent combined: 87.76%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.131 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4767 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB204/SB204_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB204_centroids.fasta --uc SB204.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4767 sequences and identified 1938 clusters (clustering threshold 1.00) - 70 clusters (representing 2414 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB21 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB21_1.fastq /home/working/demultiplexed/SB21_2.fastq SB21_forward.paired.fastq.gz SB21_forward.singletons.fastq.gz SB21_reverse.paired.fastq.gz SB21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB21_1.fastq /home/working/demultiplexed/SB21_2.fastq SB21_forward.paired.fastq.gz SB21_forward.singletons.fastq.gz SB21_reverse.paired.fastq.gz SB21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:17:47 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB21/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25058 Both Surviving: 5378 (21.46%) Forward Only Surviving: 1236 (4.93%) Reverse Only Surviving: 413 (1.65%) Dropped: 18031 (71.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB21_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB21_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB21_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB21_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB21_forward.paired.headclipped.fastq.gz SB21_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB21 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB21_forward.paired.headclipped.fastq.gz
[FLASH]     SB21_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB21.extendedFrags.fastq.gz
[FLASH]     ./SB21.notCombined_1.fastq.gz
[FLASH]     ./SB21.notCombined_2.fastq.gz
[FLASH]     ./SB21.hist
[FLASH]     ./SB21.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5378 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5378
[FLASH]     Combined pairs:   3058
[FLASH]     Uncombined pairs: 2320
[FLASH]     Percent combined: 56.86%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.114 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3058 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB21/SB21_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB21_centroids.fasta --uc SB21.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3058 sequences and identified 1907 clusters (clustering threshold 1.00) - 45 clusters (representing 1006 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB22 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB22_1.fastq /home/working/demultiplexed/SB22_2.fastq SB22_forward.paired.fastq.gz SB22_forward.singletons.fastq.gz SB22_reverse.paired.fastq.gz SB22_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB22_1.fastq /home/working/demultiplexed/SB22_2.fastq SB22_forward.paired.fastq.gz SB22_forward.singletons.fastq.gz SB22_reverse.paired.fastq.gz SB22_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:17:57 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB22/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37803 Both Surviving: 9518 (25.18%) Forward Only Surviving: 2446 (6.47%) Reverse Only Surviving: 592 (1.57%) Dropped: 25247 (66.79%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB22_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB22_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB22_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB22_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB22_forward.paired.headclipped.fastq.gz SB22_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB22 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB22_forward.paired.headclipped.fastq.gz
[FLASH]     SB22_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB22.extendedFrags.fastq.gz
[FLASH]     ./SB22.notCombined_1.fastq.gz
[FLASH]     ./SB22.notCombined_2.fastq.gz
[FLASH]     ./SB22.hist
[FLASH]     ./SB22.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9518 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9518
[FLASH]     Combined pairs:   4105
[FLASH]     Uncombined pairs: 5413
[FLASH]     Percent combined: 43.13%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.143 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4105 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB22/SB22_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB22_centroids.fasta --uc SB22.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4105 sequences and identified 1997 clusters (clustering threshold 1.00) - 40 clusters (representing 1786 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB23 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB23_1.fastq /home/working/demultiplexed/SB23_2.fastq SB23_forward.paired.fastq.gz SB23_forward.singletons.fastq.gz SB23_reverse.paired.fastq.gz SB23_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB23_1.fastq /home/working/demultiplexed/SB23_2.fastq SB23_forward.paired.fastq.gz SB23_forward.singletons.fastq.gz SB23_reverse.paired.fastq.gz SB23_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:18:08 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB23/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SB23_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB23_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB23_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB23_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB23_forward.paired.headclipped.fastq.gz SB23_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB23 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB23_forward.paired.headclipped.fastq.gz
[FLASH]     SB23_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB23.extendedFrags.fastq.gz
[FLASH]     ./SB23.notCombined_1.fastq.gz
[FLASH]     ./SB23.notCombined_2.fastq.gz
[FLASH]     ./SB23.hist
[FLASH]     ./SB23.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB23/SB23_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB23_centroids.fasta --uc SB23.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB24 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB24_1.fastq /home/working/demultiplexed/SB24_2.fastq SB24_forward.paired.fastq.gz SB24_forward.singletons.fastq.gz SB24_reverse.paired.fastq.gz SB24_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB24_1.fastq /home/working/demultiplexed/SB24_2.fastq SB24_forward.paired.fastq.gz SB24_forward.singletons.fastq.gz SB24_reverse.paired.fastq.gz SB24_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:18:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB24/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28519 Both Surviving: 11971 (41.98%) Forward Only Surviving: 2534 (8.89%) Reverse Only Surviving: 494 (1.73%) Dropped: 13520 (47.41%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB24_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB24_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB24_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB24_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB24_forward.paired.headclipped.fastq.gz SB24_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB24 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB24_forward.paired.headclipped.fastq.gz
[FLASH]     SB24_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB24.extendedFrags.fastq.gz
[FLASH]     ./SB24.notCombined_1.fastq.gz
[FLASH]     ./SB24.notCombined_2.fastq.gz
[FLASH]     ./SB24.hist
[FLASH]     ./SB24.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11971 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11971
[FLASH]     Combined pairs:   6050
[FLASH]     Uncombined pairs: 5921
[FLASH]     Percent combined: 50.54%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.168 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6050 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB24/SB24_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB24_centroids.fasta --uc SB24.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6050 sequences and identified 3589 clusters (clustering threshold 1.00) - 138 clusters (representing 2114 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB27 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB27_1.fastq /home/working/demultiplexed/SB27_2.fastq SB27_forward.paired.fastq.gz SB27_forward.singletons.fastq.gz SB27_reverse.paired.fastq.gz SB27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB27_1.fastq /home/working/demultiplexed/SB27_2.fastq SB27_forward.paired.fastq.gz SB27_forward.singletons.fastq.gz SB27_reverse.paired.fastq.gz SB27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:18:24 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB27/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15596 Both Surviving: 2718 (17.43%) Forward Only Surviving: 745 (4.78%) Reverse Only Surviving: 204 (1.31%) Dropped: 11929 (76.49%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB27_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB27_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB27_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB27_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB27_forward.paired.headclipped.fastq.gz SB27_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB27 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB27_forward.paired.headclipped.fastq.gz
[FLASH]     SB27_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB27.extendedFrags.fastq.gz
[FLASH]     ./SB27.notCombined_1.fastq.gz
[FLASH]     ./SB27.notCombined_2.fastq.gz
[FLASH]     ./SB27.hist
[FLASH]     ./SB27.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2718 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2718
[FLASH]     Combined pairs:   55
[FLASH]     Uncombined pairs: 2663
[FLASH]     Percent combined: 2.02%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.025 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

55 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB27/SB27_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB27_centroids.fasta --uc SB27.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 55 sequences and identified 46 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB29 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB29_1.fastq /home/working/demultiplexed/SB29_2.fastq SB29_forward.paired.fastq.gz SB29_forward.singletons.fastq.gz SB29_reverse.paired.fastq.gz SB29_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB29_1.fastq /home/working/demultiplexed/SB29_2.fastq SB29_forward.paired.fastq.gz SB29_forward.singletons.fastq.gz SB29_reverse.paired.fastq.gz SB29_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:18:27 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB29/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37392 Both Surviving: 1224 (3.27%) Forward Only Surviving: 976 (2.61%) Reverse Only Surviving: 315 (0.84%) Dropped: 34877 (93.27%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB29_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB29_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB29_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB29_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB29_forward.paired.headclipped.fastq.gz SB29_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB29 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB29_forward.paired.headclipped.fastq.gz
[FLASH]     SB29_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB29.extendedFrags.fastq.gz
[FLASH]     ./SB29.notCombined_1.fastq.gz
[FLASH]     ./SB29.notCombined_2.fastq.gz
[FLASH]     ./SB29.hist
[FLASH]     ./SB29.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1224 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1224
[FLASH]     Combined pairs:   867
[FLASH]     Uncombined pairs: 357
[FLASH]     Percent combined: 70.83%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.022 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

867 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB29/SB29_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB29_centroids.fasta --uc SB29.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 867 sequences and identified 475 clusters (clustering threshold 1.00) - 12 clusters (representing 341 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB3 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB3_1.fastq /home/working/demultiplexed/SB3_2.fastq SB3_forward.paired.fastq.gz SB3_forward.singletons.fastq.gz SB3_reverse.paired.fastq.gz SB3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB3_1.fastq /home/working/demultiplexed/SB3_2.fastq SB3_forward.paired.fastq.gz SB3_forward.singletons.fastq.gz SB3_reverse.paired.fastq.gz SB3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:18:36 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB3/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 107968 Both Surviving: 4015 (3.72%) Forward Only Surviving: 5331 (4.94%) Reverse Only Surviving: 1056 (0.98%) Dropped: 97566 (90.37%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB3_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB3_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB3_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB3_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB3_forward.paired.headclipped.fastq.gz SB3_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB3 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB3_forward.paired.headclipped.fastq.gz
[FLASH]     SB3_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB3.extendedFrags.fastq.gz
[FLASH]     ./SB3.notCombined_1.fastq.gz
[FLASH]     ./SB3.notCombined_2.fastq.gz
[FLASH]     ./SB3.hist
[FLASH]     ./SB3.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4015 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4015
[FLASH]     Combined pairs:   1099
[FLASH]     Uncombined pairs: 2916
[FLASH]     Percent combined: 27.37%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.033 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1099 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB3/SB3_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB3_centroids.fasta --uc SB3.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1099 sequences and identified 670 clusters (clustering threshold 1.00) - 31 clusters (representing 390 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB33 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB33_1.fastq /home/working/demultiplexed/SB33_2.fastq SB33_forward.paired.fastq.gz SB33_forward.singletons.fastq.gz SB33_reverse.paired.fastq.gz SB33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB33_1.fastq /home/working/demultiplexed/SB33_2.fastq SB33_forward.paired.fastq.gz SB33_forward.singletons.fastq.gz SB33_reverse.paired.fastq.gz SB33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:18:59 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB33/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 250333 Both Surviving: 102563 (40.97%) Forward Only Surviving: 23564 (9.41%) Reverse Only Surviving: 4893 (1.95%) Dropped: 119313 (47.66%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB33_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB33_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB33_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB33_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB33_forward.paired.headclipped.fastq.gz SB33_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB33 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB33_forward.paired.headclipped.fastq.gz
[FLASH]     SB33_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB33.extendedFrags.fastq.gz
[FLASH]     ./SB33.notCombined_1.fastq.gz
[FLASH]     ./SB33.notCombined_2.fastq.gz
[FLASH]     ./SB33.hist
[FLASH]     ./SB33.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 50000 read pairs
[FLASH] Processed 75000 read pairs
[FLASH] Processed 100000 read pairs
[FLASH] Processed 102563 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      102563
[FLASH]     Combined pairs:   60103
[FLASH]     Uncombined pairs: 42460
[FLASH]     Percent combined: 58.60%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 1.585 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

60103 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB33/SB33_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB33_centroids.fasta --uc SB33.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 60103 sequences and identified 22427 clusters (clustering threshold 1.00) - 747 clusters (representing 35660 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB34 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB34_1.fastq /home/working/demultiplexed/SB34_2.fastq SB34_forward.paired.fastq.gz SB34_forward.singletons.fastq.gz SB34_reverse.paired.fastq.gz SB34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB34_1.fastq /home/working/demultiplexed/SB34_2.fastq SB34_forward.paired.fastq.gz SB34_forward.singletons.fastq.gz SB34_reverse.paired.fastq.gz SB34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:21:23 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB34/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 5099 Both Surviving: 1755 (34.42%) Forward Only Surviving: 415 (8.14%) Reverse Only Surviving: 67 (1.31%) Dropped: 2862 (56.13%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB34_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB34_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB34_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB34_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB34_forward.paired.headclipped.fastq.gz SB34_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB34 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB34_forward.paired.headclipped.fastq.gz
[FLASH]     SB34_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB34.extendedFrags.fastq.gz
[FLASH]     ./SB34.notCombined_1.fastq.gz
[FLASH]     ./SB34.notCombined_2.fastq.gz
[FLASH]     ./SB34.hist
[FLASH]     ./SB34.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1755 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1755
[FLASH]     Combined pairs:   721
[FLASH]     Uncombined pairs: 1034
[FLASH]     Percent combined: 41.08%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.023 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

721 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB34/SB34_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB34_centroids.fasta --uc SB34.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 721 sequences and identified 509 clusters (clustering threshold 1.00) - 13 clusters (representing 166 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB37 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB37_1.fastq /home/working/demultiplexed/SB37_2.fastq SB37_forward.paired.fastq.gz SB37_forward.singletons.fastq.gz SB37_reverse.paired.fastq.gz SB37_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB37_1.fastq /home/working/demultiplexed/SB37_2.fastq SB37_forward.paired.fastq.gz SB37_forward.singletons.fastq.gz SB37_reverse.paired.fastq.gz SB37_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:21:25 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB37/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15290 Both Surviving: 2458 (16.08%) Forward Only Surviving: 686 (4.49%) Reverse Only Surviving: 189 (1.24%) Dropped: 11957 (78.20%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB37_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB37_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB37_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB37_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB37_forward.paired.headclipped.fastq.gz SB37_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB37 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB37_forward.paired.headclipped.fastq.gz
[FLASH]     SB37_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB37.extendedFrags.fastq.gz
[FLASH]     ./SB37.notCombined_1.fastq.gz
[FLASH]     ./SB37.notCombined_2.fastq.gz
[FLASH]     ./SB37.hist
[FLASH]     ./SB37.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2458 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2458
[FLASH]     Combined pairs:   62
[FLASH]     Uncombined pairs: 2396
[FLASH]     Percent combined: 2.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.024 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

62 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB37/SB37_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB37_centroids.fasta --uc SB37.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 62 sequences and identified 44 clusters (clustering threshold 1.00) - 1 clusters (representing 7 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB4 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB4_1.fastq /home/working/demultiplexed/SB4_2.fastq SB4_forward.paired.fastq.gz SB4_forward.singletons.fastq.gz SB4_reverse.paired.fastq.gz SB4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB4_1.fastq /home/working/demultiplexed/SB4_2.fastq SB4_forward.paired.fastq.gz SB4_forward.singletons.fastq.gz SB4_reverse.paired.fastq.gz SB4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:21:28 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB4/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17746 Both Surviving: 2361 (13.30%) Forward Only Surviving: 616 (3.47%) Reverse Only Surviving: 545 (3.07%) Dropped: 14224 (80.15%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SB4_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB4_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB4_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB4_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB4_forward.paired.headclipped.fastq.gz SB4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB4 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB4_forward.paired.headclipped.fastq.gz
[FLASH]     SB4_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB4.extendedFrags.fastq.gz
[FLASH]     ./SB4.notCombined_1.fastq.gz
[FLASH]     ./SB4.notCombined_2.fastq.gz
[FLASH]     ./SB4.hist
[FLASH]     ./SB4.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2361 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2361
[FLASH]     Combined pairs:   110
[FLASH]     Uncombined pairs: 2251
[FLASH]     Percent combined: 4.66%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.024 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

110 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB4/SB4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB4_centroids.fasta --uc SB4.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 110 sequences and identified 104 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB45 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB45_1.fastq /home/working/demultiplexed/SB45_2.fastq SB45_forward.paired.fastq.gz SB45_forward.singletons.fastq.gz SB45_reverse.paired.fastq.gz SB45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB45_1.fastq /home/working/demultiplexed/SB45_2.fastq SB45_forward.paired.fastq.gz SB45_forward.singletons.fastq.gz SB45_reverse.paired.fastq.gz SB45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:21:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB45/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 114934 Both Surviving: 33820 (29.43%) Forward Only Surviving: 8649 (7.53%) Reverse Only Surviving: 1860 (1.62%) Dropped: 70605 (61.43%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB45_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB45_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB45_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB45_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB45_forward.paired.headclipped.fastq.gz SB45_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB45 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB45_forward.paired.headclipped.fastq.gz
[FLASH]     SB45_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB45.extendedFrags.fastq.gz
[FLASH]     ./SB45.notCombined_1.fastq.gz
[FLASH]     ./SB45.notCombined_2.fastq.gz
[FLASH]     ./SB45.hist
[FLASH]     ./SB45.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 33820 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      33820
[FLASH]     Combined pairs:   16731
[FLASH]     Uncombined pairs: 17089
[FLASH]     Percent combined: 49.47%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.544 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

16731 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB45/SB45_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB45_centroids.fasta --uc SB45.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 16731 sequences and identified 5203 clusters (clustering threshold 1.00) - 315 clusters (representing 10908 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB48 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB48_1.fastq /home/working/demultiplexed/SB48_2.fastq SB48_forward.paired.fastq.gz SB48_forward.singletons.fastq.gz SB48_reverse.paired.fastq.gz SB48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB48_1.fastq /home/working/demultiplexed/SB48_2.fastq SB48_forward.paired.fastq.gz SB48_forward.singletons.fastq.gz SB48_reverse.paired.fastq.gz SB48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:22:16 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB48/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 38763 Both Surviving: 7572 (19.53%) Forward Only Surviving: 1864 (4.81%) Reverse Only Surviving: 658 (1.70%) Dropped: 28669 (73.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB48_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB48_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB48_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB48_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB48_forward.paired.headclipped.fastq.gz SB48_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB48 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB48_forward.paired.headclipped.fastq.gz
[FLASH]     SB48_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB48.extendedFrags.fastq.gz
[FLASH]     ./SB48.notCombined_1.fastq.gz
[FLASH]     ./SB48.notCombined_2.fastq.gz
[FLASH]     ./SB48.hist
[FLASH]     ./SB48.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7572 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7572
[FLASH]     Combined pairs:   837
[FLASH]     Uncombined pairs: 6735
[FLASH]     Percent combined: 11.05%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.066 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

837 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB48/SB48_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB48_centroids.fasta --uc SB48.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 837 sequences and identified 474 clusters (clustering threshold 1.00) - 20 clusters (representing 329 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB51 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB51_1.fastq /home/working/demultiplexed/SB51_2.fastq SB51_forward.paired.fastq.gz SB51_forward.singletons.fastq.gz SB51_reverse.paired.fastq.gz SB51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB51_1.fastq /home/working/demultiplexed/SB51_2.fastq SB51_forward.paired.fastq.gz SB51_forward.singletons.fastq.gz SB51_reverse.paired.fastq.gz SB51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:22:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB51/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 2 Both Surviving: 2 (100.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 0 (0.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SB51_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB51_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB51_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB51_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB51_forward.paired.headclipped.fastq.gz SB51_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB51 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB51_forward.paired.headclipped.fastq.gz
[FLASH]     SB51_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB51.extendedFrags.fastq.gz
[FLASH]     ./SB51.notCombined_1.fastq.gz
[FLASH]     ./SB51.notCombined_2.fastq.gz
[FLASH]     ./SB51.hist
[FLASH]     ./SB51.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2
[FLASH]     Combined pairs:   1
[FLASH]     Uncombined pairs: 1
[FLASH]     Percent combined: 50.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB51/SB51_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB51_centroids.fasta --uc SB51.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1 sequences and identified 1 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB53 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB53_1.fastq /home/working/demultiplexed/SB53_2.fastq SB53_forward.paired.fastq.gz SB53_forward.singletons.fastq.gz SB53_reverse.paired.fastq.gz SB53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB53_1.fastq /home/working/demultiplexed/SB53_2.fastq SB53_forward.paired.fastq.gz SB53_forward.singletons.fastq.gz SB53_reverse.paired.fastq.gz SB53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:22:24 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB53/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 34274 Both Surviving: 8781 (25.62%) Forward Only Surviving: 2596 (7.57%) Reverse Only Surviving: 500 (1.46%) Dropped: 22397 (65.35%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB53_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB53_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB53_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB53_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB53_forward.paired.headclipped.fastq.gz SB53_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB53 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB53_forward.paired.headclipped.fastq.gz
[FLASH]     SB53_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB53.extendedFrags.fastq.gz
[FLASH]     ./SB53.notCombined_1.fastq.gz
[FLASH]     ./SB53.notCombined_2.fastq.gz
[FLASH]     ./SB53.hist
[FLASH]     ./SB53.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8781 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8781
[FLASH]     Combined pairs:   2543
[FLASH]     Uncombined pairs: 6238
[FLASH]     Percent combined: 28.96%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.093 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2543 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB53/SB53_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB53_centroids.fasta --uc SB53.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2543 sequences and identified 1393 clusters (clustering threshold 1.00) - 20 clusters (representing 1010 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB59 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB59_1.fastq /home/working/demultiplexed/SB59_2.fastq SB59_forward.paired.fastq.gz SB59_forward.singletons.fastq.gz SB59_reverse.paired.fastq.gz SB59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB59_1.fastq /home/working/demultiplexed/SB59_2.fastq SB59_forward.paired.fastq.gz SB59_forward.singletons.fastq.gz SB59_reverse.paired.fastq.gz SB59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:22:34 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB59/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37151 Both Surviving: 7895 (21.25%) Forward Only Surviving: 2714 (7.31%) Reverse Only Surviving: 494 (1.33%) Dropped: 26048 (70.11%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB59_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB59_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB59_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB59_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB59_forward.paired.headclipped.fastq.gz SB59_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB59 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB59_forward.paired.headclipped.fastq.gz
[FLASH]     SB59_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB59.extendedFrags.fastq.gz
[FLASH]     ./SB59.notCombined_1.fastq.gz
[FLASH]     ./SB59.notCombined_2.fastq.gz
[FLASH]     ./SB59.hist
[FLASH]     ./SB59.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7895 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7895
[FLASH]     Combined pairs:   2360
[FLASH]     Uncombined pairs: 5535
[FLASH]     Percent combined: 29.89%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.101 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2360 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB59/SB59_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB59_centroids.fasta --uc SB59.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2360 sequences and identified 1147 clusters (clustering threshold 1.00) - 32 clusters (representing 1081 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB60 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB60_1.fastq /home/working/demultiplexed/SB60_2.fastq SB60_forward.paired.fastq.gz SB60_forward.singletons.fastq.gz SB60_reverse.paired.fastq.gz SB60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB60_1.fastq /home/working/demultiplexed/SB60_2.fastq SB60_forward.paired.fastq.gz SB60_forward.singletons.fastq.gz SB60_reverse.paired.fastq.gz SB60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:22:42 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB60/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26851 Both Surviving: 9687 (36.08%) Forward Only Surviving: 2603 (9.69%) Reverse Only Surviving: 322 (1.20%) Dropped: 14239 (53.03%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB60_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB60_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SB60_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SB60_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB60_forward.paired.headclipped.fastq.gz SB60_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB60 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB60_forward.paired.headclipped.fastq.gz
[FLASH]     SB60_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB60.extendedFrags.fastq.gz
[FLASH]     ./SB60.notCombined_1.fastq.gz
[FLASH]     ./SB60.notCombined_2.fastq.gz
[FLASH]     ./SB60.hist
[FLASH]     ./SB60.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9687 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9687
[FLASH]     Combined pairs:   4979
[FLASH]     Uncombined pairs: 4708
[FLASH]     Percent combined: 51.40%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.161 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4979 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB60/SB60_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB60_centroids.fasta --uc SB60.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4979 sequences and identified 2895 clusters (clustering threshold 1.00) - 123 clusters (representing 1906 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB63 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB63_1.fastq /home/working/demultiplexed/SB63_2.fastq SB63_forward.paired.fastq.gz SB63_forward.singletons.fastq.gz SB63_reverse.paired.fastq.gz SB63_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB63_1.fastq /home/working/demultiplexed/SB63_2.fastq SB63_forward.paired.fastq.gz SB63_forward.singletons.fastq.gz SB63_reverse.paired.fastq.gz SB63_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:22:54 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB63/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15119 Both Surviving: 3736 (24.71%) Forward Only Surviving: 845 (5.59%) Reverse Only Surviving: 227 (1.50%) Dropped: 10311 (68.20%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB63_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB63_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SB63_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SB63_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB63_forward.paired.headclipped.fastq.gz SB63_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB63 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB63_forward.paired.headclipped.fastq.gz
[FLASH]     SB63_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB63.extendedFrags.fastq.gz
[FLASH]     ./SB63.notCombined_1.fastq.gz
[FLASH]     ./SB63.notCombined_2.fastq.gz
[FLASH]     ./SB63.hist
[FLASH]     ./SB63.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3736 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3736
[FLASH]     Combined pairs:   1303
[FLASH]     Uncombined pairs: 2433
[FLASH]     Percent combined: 34.88%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.040 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1303 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB63/SB63_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB63_centroids.fasta --uc SB63.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1303 sequences and identified 675 clusters (clustering threshold 1.00) - 25 clusters (representing 552 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB64 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB64_1.fastq /home/working/demultiplexed/SB64_2.fastq SB64_forward.paired.fastq.gz SB64_forward.singletons.fastq.gz SB64_reverse.paired.fastq.gz SB64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB64_1.fastq /home/working/demultiplexed/SB64_2.fastq SB64_forward.paired.fastq.gz SB64_forward.singletons.fastq.gz SB64_reverse.paired.fastq.gz SB64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:22:58 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB64/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20694 Both Surviving: 1095 (5.29%) Forward Only Surviving: 979 (4.73%) Reverse Only Surviving: 253 (1.22%) Dropped: 18367 (88.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB64_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB64_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB64_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB64_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB64_forward.paired.headclipped.fastq.gz SB64_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB64 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB64_forward.paired.headclipped.fastq.gz
[FLASH]     SB64_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB64.extendedFrags.fastq.gz
[FLASH]     ./SB64.notCombined_1.fastq.gz
[FLASH]     ./SB64.notCombined_2.fastq.gz
[FLASH]     ./SB64.hist
[FLASH]     ./SB64.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1095 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1095
[FLASH]     Combined pairs:   180
[FLASH]     Uncombined pairs: 915
[FLASH]     Percent combined: 16.44%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.011 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

180 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB64/SB64_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB64_centroids.fasta --uc SB64.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 180 sequences and identified 149 clusters (clustering threshold 1.00) - 2 clusters (representing 12 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB67 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB67_1.fastq /home/working/demultiplexed/SB67_2.fastq SB67_forward.paired.fastq.gz SB67_forward.singletons.fastq.gz SB67_reverse.paired.fastq.gz SB67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB67_1.fastq /home/working/demultiplexed/SB67_2.fastq SB67_forward.paired.fastq.gz SB67_forward.singletons.fastq.gz SB67_reverse.paired.fastq.gz SB67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:23:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB67/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25560 Both Surviving: 302 (1.18%) Forward Only Surviving: 566 (2.21%) Reverse Only Surviving: 267 (1.04%) Dropped: 24425 (95.56%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SB67_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SB67_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB67_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB67_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB67_forward.paired.headclipped.fastq.gz SB67_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB67 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB67_forward.paired.headclipped.fastq.gz
[FLASH]     SB67_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB67.extendedFrags.fastq.gz
[FLASH]     ./SB67.notCombined_1.fastq.gz
[FLASH]     ./SB67.notCombined_2.fastq.gz
[FLASH]     ./SB67.hist
[FLASH]     ./SB67.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 302 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      302
[FLASH]     Combined pairs:   211
[FLASH]     Uncombined pairs: 91
[FLASH]     Percent combined: 69.87%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.007 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

211 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB67/SB67_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB67_centroids.fasta --uc SB67.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 211 sequences and identified 136 clusters (clustering threshold 1.00) - 4 clusters (representing 77 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB71 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB71_1.fastq /home/working/demultiplexed/SB71_2.fastq SB71_forward.paired.fastq.gz SB71_forward.singletons.fastq.gz SB71_reverse.paired.fastq.gz SB71_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB71_1.fastq /home/working/demultiplexed/SB71_2.fastq SB71_forward.paired.fastq.gz SB71_forward.singletons.fastq.gz SB71_reverse.paired.fastq.gz SB71_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:23:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB71/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29730 Both Surviving: 5425 (18.25%) Forward Only Surviving: 1311 (4.41%) Reverse Only Surviving: 389 (1.31%) Dropped: 22605 (76.03%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB71_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB71_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB71_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB71_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB71_forward.paired.headclipped.fastq.gz SB71_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB71 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB71_forward.paired.headclipped.fastq.gz
[FLASH]     SB71_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB71.extendedFrags.fastq.gz
[FLASH]     ./SB71.notCombined_1.fastq.gz
[FLASH]     ./SB71.notCombined_2.fastq.gz
[FLASH]     ./SB71.hist
[FLASH]     ./SB71.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5425 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5425
[FLASH]     Combined pairs:   5019
[FLASH]     Uncombined pairs: 406
[FLASH]     Percent combined: 92.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.129 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5019 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB71/SB71_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB71_centroids.fasta --uc SB71.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5019 sequences and identified 2229 clusters (clustering threshold 1.00) - 89 clusters (representing 2479 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB73 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB73_1.fastq /home/working/demultiplexed/SB73_2.fastq SB73_forward.paired.fastq.gz SB73_forward.singletons.fastq.gz SB73_reverse.paired.fastq.gz SB73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB73_1.fastq /home/working/demultiplexed/SB73_2.fastq SB73_forward.paired.fastq.gz SB73_forward.singletons.fastq.gz SB73_reverse.paired.fastq.gz SB73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:23:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB73/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 30141 Both Surviving: 1194 (3.96%) Forward Only Surviving: 905 (3.00%) Reverse Only Surviving: 309 (1.03%) Dropped: 27733 (92.01%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB73_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB73_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB73_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB73_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB73_forward.paired.headclipped.fastq.gz SB73_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB73 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB73_forward.paired.headclipped.fastq.gz
[FLASH]     SB73_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB73.extendedFrags.fastq.gz
[FLASH]     ./SB73.notCombined_1.fastq.gz
[FLASH]     ./SB73.notCombined_2.fastq.gz
[FLASH]     ./SB73.hist
[FLASH]     ./SB73.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1194 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1194
[FLASH]     Combined pairs:   944
[FLASH]     Uncombined pairs: 250
[FLASH]     Percent combined: 79.06%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.025 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

944 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB73/SB73_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB73_centroids.fasta --uc SB73.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 944 sequences and identified 464 clusters (clustering threshold 1.00) - 8 clusters (representing 438 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB76 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB76_1.fastq /home/working/demultiplexed/SB76_2.fastq SB76_forward.paired.fastq.gz SB76_forward.singletons.fastq.gz SB76_reverse.paired.fastq.gz SB76_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB76_1.fastq /home/working/demultiplexed/SB76_2.fastq SB76_forward.paired.fastq.gz SB76_forward.singletons.fastq.gz SB76_reverse.paired.fastq.gz SB76_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:23:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB76/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 118828 Both Surviving: 47500 (39.97%) Forward Only Surviving: 12180 (10.25%) Reverse Only Surviving: 2009 (1.69%) Dropped: 57139 (48.09%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB76_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB76_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB76_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB76_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB76_forward.paired.headclipped.fastq.gz SB76_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB76 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB76_forward.paired.headclipped.fastq.gz
[FLASH]     SB76_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB76.extendedFrags.fastq.gz
[FLASH]     ./SB76.notCombined_1.fastq.gz
[FLASH]     ./SB76.notCombined_2.fastq.gz
[FLASH]     ./SB76.hist
[FLASH]     ./SB76.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 47500 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      47500
[FLASH]     Combined pairs:   24052
[FLASH]     Uncombined pairs: 23448
[FLASH]     Percent combined: 50.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.721 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

24052 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB76/SB76_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB76_centroids.fasta --uc SB76.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 24052 sequences and identified 9072 clusters (clustering threshold 1.00) - 456 clusters (representing 14503 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB77 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB77_1.fastq /home/working/demultiplexed/SB77_2.fastq SB77_forward.paired.fastq.gz SB77_forward.singletons.fastq.gz SB77_reverse.paired.fastq.gz SB77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB77_1.fastq /home/working/demultiplexed/SB77_2.fastq SB77_forward.paired.fastq.gz SB77_forward.singletons.fastq.gz SB77_reverse.paired.fastq.gz SB77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:24:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB77/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 24892 Both Surviving: 4667 (18.75%) Forward Only Surviving: 1168 (4.69%) Reverse Only Surviving: 427 (1.72%) Dropped: 18630 (74.84%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB77_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB77_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB77_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB77_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB77_forward.paired.headclipped.fastq.gz SB77_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB77 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB77_forward.paired.headclipped.fastq.gz
[FLASH]     SB77_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB77.extendedFrags.fastq.gz
[FLASH]     ./SB77.notCombined_1.fastq.gz
[FLASH]     ./SB77.notCombined_2.fastq.gz
[FLASH]     ./SB77.hist
[FLASH]     ./SB77.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4667 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4667
[FLASH]     Combined pairs:   53
[FLASH]     Uncombined pairs: 4614
[FLASH]     Percent combined: 1.14%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.041 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

53 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB77/SB77_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB77_centroids.fasta --uc SB77.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 53 sequences and identified 48 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB8 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB8_1.fastq /home/working/demultiplexed/SB8_2.fastq SB8_forward.paired.fastq.gz SB8_forward.singletons.fastq.gz SB8_reverse.paired.fastq.gz SB8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB8_1.fastq /home/working/demultiplexed/SB8_2.fastq SB8_forward.paired.fastq.gz SB8_forward.singletons.fastq.gz SB8_reverse.paired.fastq.gz SB8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:24:23 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB8/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26459 Both Surviving: 2673 (10.10%) Forward Only Surviving: 944 (3.57%) Reverse Only Surviving: 335 (1.27%) Dropped: 22507 (85.06%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB8_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB8_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB8_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB8_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB8_forward.paired.headclipped.fastq.gz SB8_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB8 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB8_forward.paired.headclipped.fastq.gz
[FLASH]     SB8_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB8.extendedFrags.fastq.gz
[FLASH]     ./SB8.notCombined_1.fastq.gz
[FLASH]     ./SB8.notCombined_2.fastq.gz
[FLASH]     ./SB8.hist
[FLASH]     ./SB8.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2673 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2673
[FLASH]     Combined pairs:   2308
[FLASH]     Uncombined pairs: 365
[FLASH]     Percent combined: 86.34%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.062 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2308 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB8/SB8_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB8_centroids.fasta --uc SB8.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2308 sequences and identified 1361 clusters (clustering threshold 1.00) - 24 clusters (representing 806 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB80 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB80_1.fastq /home/working/demultiplexed/SB80_2.fastq SB80_forward.paired.fastq.gz SB80_forward.singletons.fastq.gz SB80_reverse.paired.fastq.gz SB80_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB80_1.fastq /home/working/demultiplexed/SB80_2.fastq SB80_forward.paired.fastq.gz SB80_forward.singletons.fastq.gz SB80_reverse.paired.fastq.gz SB80_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:24:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB80/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 34212 Both Surviving: 398 (1.16%) Forward Only Surviving: 465 (1.36%) Reverse Only Surviving: 237 (0.69%) Dropped: 33112 (96.78%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB80_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB80_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB80_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB80_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB80_forward.paired.headclipped.fastq.gz SB80_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB80 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB80_forward.paired.headclipped.fastq.gz
[FLASH]     SB80_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB80.extendedFrags.fastq.gz
[FLASH]     ./SB80.notCombined_1.fastq.gz
[FLASH]     ./SB80.notCombined_2.fastq.gz
[FLASH]     ./SB80.hist
[FLASH]     ./SB80.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 398 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      398
[FLASH]     Combined pairs:   15
[FLASH]     Uncombined pairs: 383
[FLASH]     Percent combined: 3.77%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

15 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB80/SB80_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB80_centroids.fasta --uc SB80.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 15 sequences and identified 14 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB82 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB82_1.fastq /home/working/demultiplexed/SB82_2.fastq SB82_forward.paired.fastq.gz SB82_forward.singletons.fastq.gz SB82_reverse.paired.fastq.gz SB82_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB82_1.fastq /home/working/demultiplexed/SB82_2.fastq SB82_forward.paired.fastq.gz SB82_forward.singletons.fastq.gz SB82_reverse.paired.fastq.gz SB82_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:24:35 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB82/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 56602 Both Surviving: 19825 (35.03%) Forward Only Surviving: 3783 (6.68%) Reverse Only Surviving: 937 (1.66%) Dropped: 32057 (56.64%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB82_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB82_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB82_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB82_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB82_forward.paired.headclipped.fastq.gz SB82_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB82 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB82_forward.paired.headclipped.fastq.gz
[FLASH]     SB82_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB82.extendedFrags.fastq.gz
[FLASH]     ./SB82.notCombined_1.fastq.gz
[FLASH]     ./SB82.notCombined_2.fastq.gz
[FLASH]     ./SB82.hist
[FLASH]     ./SB82.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 19825 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      19825
[FLASH]     Combined pairs:   10734
[FLASH]     Uncombined pairs: 9091
[FLASH]     Percent combined: 54.14%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.307 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

10734 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB82/SB82_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB82_centroids.fasta --uc SB82.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 10734 sequences and identified 5324 clusters (clustering threshold 1.00) - 215 clusters (representing 4986 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB83 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB83_1.fastq /home/working/demultiplexed/SB83_2.fastq SB83_forward.paired.fastq.gz SB83_forward.singletons.fastq.gz SB83_reverse.paired.fastq.gz SB83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB83_1.fastq /home/working/demultiplexed/SB83_2.fastq SB83_forward.paired.fastq.gz SB83_forward.singletons.fastq.gz SB83_reverse.paired.fastq.gz SB83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:24:57 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB83/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 13296 Both Surviving: 2639 (19.85%) Forward Only Surviving: 502 (3.78%) Reverse Only Surviving: 223 (1.68%) Dropped: 9932 (74.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB83_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB83_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB83_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB83_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB83_forward.paired.headclipped.fastq.gz SB83_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB83 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB83_forward.paired.headclipped.fastq.gz
[FLASH]     SB83_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB83.extendedFrags.fastq.gz
[FLASH]     ./SB83.notCombined_1.fastq.gz
[FLASH]     ./SB83.notCombined_2.fastq.gz
[FLASH]     ./SB83.hist
[FLASH]     ./SB83.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2639 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2639
[FLASH]     Combined pairs:   195
[FLASH]     Uncombined pairs: 2444
[FLASH]     Percent combined: 7.39%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.018 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

195 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB83/SB83_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB83_centroids.fasta --uc SB83.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 195 sequences and identified 88 clusters (clustering threshold 1.00) - 2 clusters (representing 98 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB85 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB85_1.fastq /home/working/demultiplexed/SB85_2.fastq SB85_forward.paired.fastq.gz SB85_forward.singletons.fastq.gz SB85_reverse.paired.fastq.gz SB85_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB85_1.fastq /home/working/demultiplexed/SB85_2.fastq SB85_forward.paired.fastq.gz SB85_forward.singletons.fastq.gz SB85_reverse.paired.fastq.gz SB85_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:25:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB85/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28986 Both Surviving: 5672 (19.57%) Forward Only Surviving: 1435 (4.95%) Reverse Only Surviving: 464 (1.60%) Dropped: 21415 (73.88%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB85_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB85_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB85_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB85_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB85_forward.paired.headclipped.fastq.gz SB85_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB85 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB85_forward.paired.headclipped.fastq.gz
[FLASH]     SB85_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB85.extendedFrags.fastq.gz
[FLASH]     ./SB85.notCombined_1.fastq.gz
[FLASH]     ./SB85.notCombined_2.fastq.gz
[FLASH]     ./SB85.hist
[FLASH]     ./SB85.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5672 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5672
[FLASH]     Combined pairs:   702
[FLASH]     Uncombined pairs: 4970
[FLASH]     Percent combined: 12.38%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.038 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

702 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB85/SB85_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB85_centroids.fasta --uc SB85.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 702 sequences and identified 426 clusters (clustering threshold 1.00) - 13 clusters (representing 266 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB86 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB86_1.fastq /home/working/demultiplexed/SB86_2.fastq SB86_forward.paired.fastq.gz SB86_forward.singletons.fastq.gz SB86_reverse.paired.fastq.gz SB86_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB86_1.fastq /home/working/demultiplexed/SB86_2.fastq SB86_forward.paired.fastq.gz SB86_forward.singletons.fastq.gz SB86_reverse.paired.fastq.gz SB86_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:25:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB86/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 10822 Both Surviving: 3987 (36.84%) Forward Only Surviving: 1134 (10.48%) Reverse Only Surviving: 161 (1.49%) Dropped: 5540 (51.19%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB86_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB86_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB86_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB86_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB86_forward.paired.headclipped.fastq.gz SB86_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB86 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB86_forward.paired.headclipped.fastq.gz
[FLASH]     SB86_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB86.extendedFrags.fastq.gz
[FLASH]     ./SB86.notCombined_1.fastq.gz
[FLASH]     ./SB86.notCombined_2.fastq.gz
[FLASH]     ./SB86.hist
[FLASH]     ./SB86.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3987 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3987
[FLASH]     Combined pairs:   1868
[FLASH]     Uncombined pairs: 2119
[FLASH]     Percent combined: 46.85%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.056 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1868 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB86/SB86_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB86_centroids.fasta --uc SB86.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1868 sequences and identified 1174 clusters (clustering threshold 1.00) - 27 clusters (representing 600 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB87 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB87_1.fastq /home/working/demultiplexed/SB87_2.fastq SB87_forward.paired.fastq.gz SB87_forward.singletons.fastq.gz SB87_reverse.paired.fastq.gz SB87_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB87_1.fastq /home/working/demultiplexed/SB87_2.fastq SB87_forward.paired.fastq.gz SB87_forward.singletons.fastq.gz SB87_reverse.paired.fastq.gz SB87_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:25:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB87/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SB87_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB87_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB87_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB87_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB87_forward.paired.headclipped.fastq.gz SB87_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB87 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB87_forward.paired.headclipped.fastq.gz
[FLASH]     SB87_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB87.extendedFrags.fastq.gz
[FLASH]     ./SB87.notCombined_1.fastq.gz
[FLASH]     ./SB87.notCombined_2.fastq.gz
[FLASH]     ./SB87.hist
[FLASH]     ./SB87.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB87/SB87_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB87_centroids.fasta --uc SB87.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB91 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB91_1.fastq /home/working/demultiplexed/SB91_2.fastq SB91_forward.paired.fastq.gz SB91_forward.singletons.fastq.gz SB91_reverse.paired.fastq.gz SB91_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB91_1.fastq /home/working/demultiplexed/SB91_2.fastq SB91_forward.paired.fastq.gz SB91_forward.singletons.fastq.gz SB91_reverse.paired.fastq.gz SB91_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:25:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB91/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 35326 Both Surviving: 2322 (6.57%) Forward Only Surviving: 1198 (3.39%) Reverse Only Surviving: 322 (0.91%) Dropped: 31484 (89.12%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB91_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB91_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB91_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB91_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB91_forward.paired.headclipped.fastq.gz SB91_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB91 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB91_forward.paired.headclipped.fastq.gz
[FLASH]     SB91_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB91.extendedFrags.fastq.gz
[FLASH]     ./SB91.notCombined_1.fastq.gz
[FLASH]     ./SB91.notCombined_2.fastq.gz
[FLASH]     ./SB91.hist
[FLASH]     ./SB91.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2322 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2322
[FLASH]     Combined pairs:   2028
[FLASH]     Uncombined pairs: 294
[FLASH]     Percent combined: 87.34%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.055 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2028 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB91/SB91_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB91_centroids.fasta --uc SB91.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2028 sequences and identified 1108 clusters (clustering threshold 1.00) - 24 clusters (representing 820 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB93 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB93_1.fastq /home/working/demultiplexed/SB93_2.fastq SB93_forward.paired.fastq.gz SB93_forward.singletons.fastq.gz SB93_reverse.paired.fastq.gz SB93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB93_1.fastq /home/working/demultiplexed/SB93_2.fastq SB93_forward.paired.fastq.gz SB93_forward.singletons.fastq.gz SB93_reverse.paired.fastq.gz SB93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:25:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB93/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20814 Both Surviving: 2455 (11.79%) Forward Only Surviving: 977 (4.69%) Reverse Only Surviving: 388 (1.86%) Dropped: 16994 (81.65%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB93_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB93_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB93_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB93_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB93_forward.paired.headclipped.fastq.gz SB93_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB93 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB93_forward.paired.headclipped.fastq.gz
[FLASH]     SB93_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB93.extendedFrags.fastq.gz
[FLASH]     ./SB93.notCombined_1.fastq.gz
[FLASH]     ./SB93.notCombined_2.fastq.gz
[FLASH]     ./SB93.hist
[FLASH]     ./SB93.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2455 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2455
[FLASH]     Combined pairs:   365
[FLASH]     Uncombined pairs: 2090
[FLASH]     Percent combined: 14.87%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.019 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

365 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB93/SB93_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB93_centroids.fasta --uc SB93.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 365 sequences and identified 188 clusters (clustering threshold 1.00) - 4 clusters (representing 165 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB99 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB99_1.fastq /home/working/demultiplexed/SB99_2.fastq SB99_forward.paired.fastq.gz SB99_forward.singletons.fastq.gz SB99_reverse.paired.fastq.gz SB99_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB99_1.fastq /home/working/demultiplexed/SB99_2.fastq SB99_forward.paired.fastq.gz SB99_forward.singletons.fastq.gz SB99_reverse.paired.fastq.gz SB99_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:25:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SB99/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22690 Both Surviving: 2824 (12.45%) Forward Only Surviving: 846 (3.73%) Reverse Only Surviving: 417 (1.84%) Dropped: 18603 (81.99%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB99_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB99_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB99_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB99_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB99_forward.paired.headclipped.fastq.gz SB99_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB99 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB99_forward.paired.headclipped.fastq.gz
[FLASH]     SB99_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB99.extendedFrags.fastq.gz
[FLASH]     ./SB99.notCombined_1.fastq.gz
[FLASH]     ./SB99.notCombined_2.fastq.gz
[FLASH]     ./SB99.hist
[FLASH]     ./SB99.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2824 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2824
[FLASH]     Combined pairs:   746
[FLASH]     Uncombined pairs: 2078
[FLASH]     Percent combined: 26.42%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.028 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

746 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SB99/SB99_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB99_centroids.fasta --uc SB99.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 746 sequences and identified 394 clusters (clustering threshold 1.00) - 8 clusters (representing 317 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC10 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC10_1.fastq /home/working/demultiplexed/SC10_2.fastq SC10_forward.paired.fastq.gz SC10_forward.singletons.fastq.gz SC10_reverse.paired.fastq.gz SC10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC10_1.fastq /home/working/demultiplexed/SC10_2.fastq SC10_forward.paired.fastq.gz SC10_forward.singletons.fastq.gz SC10_reverse.paired.fastq.gz SC10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:25:27 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC10/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 39166 Both Surviving: 16252 (41.50%) Forward Only Surviving: 3651 (9.32%) Reverse Only Surviving: 596 (1.52%) Dropped: 18667 (47.66%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC10_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC10_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SC10_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC10_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC10_forward.paired.headclipped.fastq.gz SC10_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC10 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC10_forward.paired.headclipped.fastq.gz
[FLASH]     SC10_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC10.extendedFrags.fastq.gz
[FLASH]     ./SC10.notCombined_1.fastq.gz
[FLASH]     ./SC10.notCombined_2.fastq.gz
[FLASH]     ./SC10.hist
[FLASH]     ./SC10.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 16252 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      16252
[FLASH]     Combined pairs:   9440
[FLASH]     Uncombined pairs: 6812
[FLASH]     Percent combined: 58.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.263 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

9440 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC10/SC10_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC10_centroids.fasta --uc SC10.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 9440 sequences and identified 4841 clusters (clustering threshold 1.00) - 261 clusters (representing 4381 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC101 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC101_1.fastq /home/working/demultiplexed/SC101_2.fastq SC101_forward.paired.fastq.gz SC101_forward.singletons.fastq.gz SC101_reverse.paired.fastq.gz SC101_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC101_1.fastq /home/working/demultiplexed/SC101_2.fastq SC101_forward.paired.fastq.gz SC101_forward.singletons.fastq.gz SC101_reverse.paired.fastq.gz SC101_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:25:47 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC101/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29995 Both Surviving: 6087 (20.29%) Forward Only Surviving: 1131 (3.77%) Reverse Only Surviving: 494 (1.65%) Dropped: 22283 (74.29%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC101_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC101_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC101_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC101_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC101_forward.paired.headclipped.fastq.gz SC101_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC101 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC101_forward.paired.headclipped.fastq.gz
[FLASH]     SC101_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC101.extendedFrags.fastq.gz
[FLASH]     ./SC101.notCombined_1.fastq.gz
[FLASH]     ./SC101.notCombined_2.fastq.gz
[FLASH]     ./SC101.hist
[FLASH]     ./SC101.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6087 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6087
[FLASH]     Combined pairs:   11
[FLASH]     Uncombined pairs: 6076
[FLASH]     Percent combined: 0.18%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.048 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

11 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC101/SC101_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC101_centroids.fasta --uc SC101.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 11 sequences and identified 11 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC106 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC106_1.fastq /home/working/demultiplexed/SC106_2.fastq SC106_forward.paired.fastq.gz SC106_forward.singletons.fastq.gz SC106_reverse.paired.fastq.gz SC106_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC106_1.fastq /home/working/demultiplexed/SC106_2.fastq SC106_forward.paired.fastq.gz SC106_forward.singletons.fastq.gz SC106_reverse.paired.fastq.gz SC106_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:25:50 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC106/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 1 Both Surviving: 0 (0.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 1 (100.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SC106_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC106_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC106_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC106_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC106_forward.paired.headclipped.fastq.gz SC106_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC106 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC106_forward.paired.headclipped.fastq.gz
[FLASH]     SC106_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC106.extendedFrags.fastq.gz
[FLASH]     ./SC106.notCombined_1.fastq.gz
[FLASH]     ./SC106.notCombined_2.fastq.gz
[FLASH]     ./SC106.hist
[FLASH]     ./SC106.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC106/SC106_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC106_centroids.fasta --uc SC106.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC107 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC107_1.fastq /home/working/demultiplexed/SC107_2.fastq SC107_forward.paired.fastq.gz SC107_forward.singletons.fastq.gz SC107_reverse.paired.fastq.gz SC107_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC107_1.fastq /home/working/demultiplexed/SC107_2.fastq SC107_forward.paired.fastq.gz SC107_forward.singletons.fastq.gz SC107_reverse.paired.fastq.gz SC107_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:25:51 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC107/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16670 Both Surviving: 3240 (19.44%) Forward Only Surviving: 944 (5.66%) Reverse Only Surviving: 191 (1.15%) Dropped: 12295 (73.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC107_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC107_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC107_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC107_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC107_forward.paired.headclipped.fastq.gz SC107_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC107 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC107_forward.paired.headclipped.fastq.gz
[FLASH]     SC107_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC107.extendedFrags.fastq.gz
[FLASH]     ./SC107.notCombined_1.fastq.gz
[FLASH]     ./SC107.notCombined_2.fastq.gz
[FLASH]     ./SC107.hist
[FLASH]     ./SC107.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3240 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3240
[FLASH]     Combined pairs:   268
[FLASH]     Uncombined pairs: 2972
[FLASH]     Percent combined: 8.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.020 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

268 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC107/SC107_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC107_centroids.fasta --uc SC107.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 268 sequences and identified 195 clusters (clustering threshold 1.00) - 1 clusters (representing 51 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC110 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC110_1.fastq /home/working/demultiplexed/SC110_2.fastq SC110_forward.paired.fastq.gz SC110_forward.singletons.fastq.gz SC110_reverse.paired.fastq.gz SC110_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC110_1.fastq /home/working/demultiplexed/SC110_2.fastq SC110_forward.paired.fastq.gz SC110_forward.singletons.fastq.gz SC110_reverse.paired.fastq.gz SC110_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:25:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC110/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 19004 Both Surviving: 3316 (17.45%) Forward Only Surviving: 1648 (8.67%) Reverse Only Surviving: 250 (1.32%) Dropped: 13790 (72.56%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC110_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC110_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC110_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC110_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC110_forward.paired.headclipped.fastq.gz SC110_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC110 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC110_forward.paired.headclipped.fastq.gz
[FLASH]     SC110_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC110.extendedFrags.fastq.gz
[FLASH]     ./SC110.notCombined_1.fastq.gz
[FLASH]     ./SC110.notCombined_2.fastq.gz
[FLASH]     ./SC110.hist
[FLASH]     ./SC110.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3316 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3316
[FLASH]     Combined pairs:   2849
[FLASH]     Uncombined pairs: 467
[FLASH]     Percent combined: 85.92%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.092 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2849 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC110/SC110_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC110_centroids.fasta --uc SC110.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2849 sequences and identified 1498 clusters (clustering threshold 1.00) - 27 clusters (representing 1109 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC113 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC113_1.fastq /home/working/demultiplexed/SC113_2.fastq SC113_forward.paired.fastq.gz SC113_forward.singletons.fastq.gz SC113_reverse.paired.fastq.gz SC113_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC113_1.fastq /home/working/demultiplexed/SC113_2.fastq SC113_forward.paired.fastq.gz SC113_forward.singletons.fastq.gz SC113_reverse.paired.fastq.gz SC113_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC113/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14610 Both Surviving: 351 (2.40%) Forward Only Surviving: 423 (2.90%) Reverse Only Surviving: 117 (0.80%) Dropped: 13719 (93.90%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC113_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC113_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC113_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC113_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC113_forward.paired.headclipped.fastq.gz SC113_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC113 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC113_forward.paired.headclipped.fastq.gz
[FLASH]     SC113_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC113.extendedFrags.fastq.gz
[FLASH]     ./SC113.notCombined_1.fastq.gz
[FLASH]     ./SC113.notCombined_2.fastq.gz
[FLASH]     ./SC113.hist
[FLASH]     ./SC113.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 351 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      351
[FLASH]     Combined pairs:   263
[FLASH]     Uncombined pairs: 88
[FLASH]     Percent combined: 74.93%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.012 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

263 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC113/SC113_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC113_centroids.fasta --uc SC113.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 263 sequences and identified 176 clusters (clustering threshold 1.00) - 4 clusters (representing 68 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC117 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC117_1.fastq /home/working/demultiplexed/SC117_2.fastq SC117_forward.paired.fastq.gz SC117_forward.singletons.fastq.gz SC117_reverse.paired.fastq.gz SC117_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC117_1.fastq /home/working/demultiplexed/SC117_2.fastq SC117_forward.paired.fastq.gz SC117_forward.singletons.fastq.gz SC117_reverse.paired.fastq.gz SC117_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC117/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 34241 Both Surviving: 7288 (21.28%) Forward Only Surviving: 1972 (5.76%) Reverse Only Surviving: 492 (1.44%) Dropped: 24489 (71.52%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC117_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC117_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SC117_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC117_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC117_forward.paired.headclipped.fastq.gz SC117_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC117 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC117_forward.paired.headclipped.fastq.gz
[FLASH]     SC117_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC117.extendedFrags.fastq.gz
[FLASH]     ./SC117.notCombined_1.fastq.gz
[FLASH]     ./SC117.notCombined_2.fastq.gz
[FLASH]     ./SC117.hist
[FLASH]     ./SC117.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7288 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7288
[FLASH]     Combined pairs:   423
[FLASH]     Uncombined pairs: 6865
[FLASH]     Percent combined: 5.80%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.046 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

423 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC117/SC117_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC117_centroids.fasta --uc SC117.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 423 sequences and identified 260 clusters (clustering threshold 1.00) - 2 clusters (representing 138 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC119 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC119_1.fastq /home/working/demultiplexed/SC119_2.fastq SC119_forward.paired.fastq.gz SC119_forward.singletons.fastq.gz SC119_reverse.paired.fastq.gz SC119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC119_1.fastq /home/working/demultiplexed/SC119_2.fastq SC119_forward.paired.fastq.gz SC119_forward.singletons.fastq.gz SC119_reverse.paired.fastq.gz SC119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC119/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16249 Both Surviving: 4595 (28.28%) Forward Only Surviving: 1155 (7.11%) Reverse Only Surviving: 242 (1.49%) Dropped: 10257 (63.12%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC119_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC119_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SC119_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SC119_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC119_forward.paired.headclipped.fastq.gz SC119_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC119 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC119_forward.paired.headclipped.fastq.gz
[FLASH]     SC119_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC119.extendedFrags.fastq.gz
[FLASH]     ./SC119.notCombined_1.fastq.gz
[FLASH]     ./SC119.notCombined_2.fastq.gz
[FLASH]     ./SC119.hist
[FLASH]     ./SC119.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4595 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4595
[FLASH]     Combined pairs:   827
[FLASH]     Uncombined pairs: 3768
[FLASH]     Percent combined: 18.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.038 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

827 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC119/SC119_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC119_centroids.fasta --uc SC119.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 827 sequences and identified 414 clusters (clustering threshold 1.00) - 7 clusters (representing 376 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC122 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC122_1.fastq /home/working/demultiplexed/SC122_2.fastq SC122_forward.paired.fastq.gz SC122_forward.singletons.fastq.gz SC122_reverse.paired.fastq.gz SC122_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC122_1.fastq /home/working/demultiplexed/SC122_2.fastq SC122_forward.paired.fastq.gz SC122_forward.singletons.fastq.gz SC122_reverse.paired.fastq.gz SC122_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC122/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 32336 Both Surviving: 1273 (3.94%) Forward Only Surviving: 724 (2.24%) Reverse Only Surviving: 364 (1.13%) Dropped: 29975 (92.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SC122_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SC122_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SC122_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SC122_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC122_forward.paired.headclipped.fastq.gz SC122_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC122 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC122_forward.paired.headclipped.fastq.gz
[FLASH]     SC122_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC122.extendedFrags.fastq.gz
[FLASH]     ./SC122.notCombined_1.fastq.gz
[FLASH]     ./SC122.notCombined_2.fastq.gz
[FLASH]     ./SC122.hist
[FLASH]     ./SC122.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1273 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1273
[FLASH]     Combined pairs:   1029
[FLASH]     Uncombined pairs: 244
[FLASH]     Percent combined: 80.83%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.025 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1029 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC122/SC122_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC122_centroids.fasta --uc SC122.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1029 sequences and identified 653 clusters (clustering threshold 1.00) - 17 clusters (representing 338 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC124 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC124_1.fastq /home/working/demultiplexed/SC124_2.fastq SC124_forward.paired.fastq.gz SC124_forward.singletons.fastq.gz SC124_reverse.paired.fastq.gz SC124_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC124_1.fastq /home/working/demultiplexed/SC124_2.fastq SC124_forward.paired.fastq.gz SC124_forward.singletons.fastq.gz SC124_reverse.paired.fastq.gz SC124_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:19 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC124/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16653 Both Surviving: 2962 (17.79%) Forward Only Surviving: 779 (4.68%) Reverse Only Surviving: 233 (1.40%) Dropped: 12679 (76.14%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC124_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC124_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC124_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC124_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC124_forward.paired.headclipped.fastq.gz SC124_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC124 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC124_forward.paired.headclipped.fastq.gz
[FLASH]     SC124_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC124.extendedFrags.fastq.gz
[FLASH]     ./SC124.notCombined_1.fastq.gz
[FLASH]     ./SC124.notCombined_2.fastq.gz
[FLASH]     ./SC124.hist
[FLASH]     ./SC124.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2962 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2962
[FLASH]     Combined pairs:   56
[FLASH]     Uncombined pairs: 2906
[FLASH]     Percent combined: 1.89%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.021 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

56 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC124/SC124_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC124_centroids.fasta --uc SC124.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 56 sequences and identified 54 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC125 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC125_1.fastq /home/working/demultiplexed/SC125_2.fastq SC125_forward.paired.fastq.gz SC125_forward.singletons.fastq.gz SC125_reverse.paired.fastq.gz SC125_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC125_1.fastq /home/working/demultiplexed/SC125_2.fastq SC125_forward.paired.fastq.gz SC125_forward.singletons.fastq.gz SC125_reverse.paired.fastq.gz SC125_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC125/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18973 Both Surviving: 2226 (11.73%) Forward Only Surviving: 628 (3.31%) Reverse Only Surviving: 189 (1.00%) Dropped: 15930 (83.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SC125_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SC125_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC125_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC125_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC125_forward.paired.headclipped.fastq.gz SC125_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC125 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC125_forward.paired.headclipped.fastq.gz
[FLASH]     SC125_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC125.extendedFrags.fastq.gz
[FLASH]     ./SC125.notCombined_1.fastq.gz
[FLASH]     ./SC125.notCombined_2.fastq.gz
[FLASH]     ./SC125.hist
[FLASH]     ./SC125.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2226 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2226
[FLASH]     Combined pairs:   37
[FLASH]     Uncombined pairs: 2189
[FLASH]     Percent combined: 1.66%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.030 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

37 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC125/SC125_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC125_centroids.fasta --uc SC125.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 37 sequences and identified 30 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC141 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC141_1.fastq /home/working/demultiplexed/SC141_2.fastq SC141_forward.paired.fastq.gz SC141_forward.singletons.fastq.gz SC141_reverse.paired.fastq.gz SC141_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC141_1.fastq /home/working/demultiplexed/SC141_2.fastq SC141_forward.paired.fastq.gz SC141_forward.singletons.fastq.gz SC141_reverse.paired.fastq.gz SC141_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:25 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC141/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 12531 Both Surviving: 2206 (17.60%) Forward Only Surviving: 549 (4.38%) Reverse Only Surviving: 337 (2.69%) Dropped: 9439 (75.33%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SC141_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC141_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC141_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC141_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC141_forward.paired.headclipped.fastq.gz SC141_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC141 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC141_forward.paired.headclipped.fastq.gz
[FLASH]     SC141_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC141.extendedFrags.fastq.gz
[FLASH]     ./SC141.notCombined_1.fastq.gz
[FLASH]     ./SC141.notCombined_2.fastq.gz
[FLASH]     ./SC141.hist
[FLASH]     ./SC141.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2206 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2206
[FLASH]     Combined pairs:   57
[FLASH]     Uncombined pairs: 2149
[FLASH]     Percent combined: 2.58%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.017 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

57 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC141/SC141_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC141_centroids.fasta --uc SC141.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 57 sequences and identified 45 clusters (clustering threshold 1.00) - 1 clusters (representing 6 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC142 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC142_1.fastq /home/working/demultiplexed/SC142_2.fastq SC142_forward.paired.fastq.gz SC142_forward.singletons.fastq.gz SC142_reverse.paired.fastq.gz SC142_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC142_1.fastq /home/working/demultiplexed/SC142_2.fastq SC142_forward.paired.fastq.gz SC142_forward.singletons.fastq.gz SC142_reverse.paired.fastq.gz SC142_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:27 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC142/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17791 Both Surviving: 3480 (19.56%) Forward Only Surviving: 964 (5.42%) Reverse Only Surviving: 223 (1.25%) Dropped: 13124 (73.77%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC142_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC142_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC142_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC142_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC142_forward.paired.headclipped.fastq.gz SC142_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC142 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC142_forward.paired.headclipped.fastq.gz
[FLASH]     SC142_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC142.extendedFrags.fastq.gz
[FLASH]     ./SC142.notCombined_1.fastq.gz
[FLASH]     ./SC142.notCombined_2.fastq.gz
[FLASH]     ./SC142.hist
[FLASH]     ./SC142.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3480 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3480
[FLASH]     Combined pairs:   55
[FLASH]     Uncombined pairs: 3425
[FLASH]     Percent combined: 1.58%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.023 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

55 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC142/SC142_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC142_centroids.fasta --uc SC142.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 55 sequences and identified 50 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC143 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC143_1.fastq /home/working/demultiplexed/SC143_2.fastq SC143_forward.paired.fastq.gz SC143_forward.singletons.fastq.gz SC143_reverse.paired.fastq.gz SC143_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC143_1.fastq /home/working/demultiplexed/SC143_2.fastq SC143_forward.paired.fastq.gz SC143_forward.singletons.fastq.gz SC143_reverse.paired.fastq.gz SC143_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:31 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC143/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 27590 Both Surviving: 2582 (9.36%) Forward Only Surviving: 805 (2.92%) Reverse Only Surviving: 426 (1.54%) Dropped: 23777 (86.18%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC143_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC143_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SC143_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC143_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC143_forward.paired.headclipped.fastq.gz SC143_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC143 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC143_forward.paired.headclipped.fastq.gz
[FLASH]     SC143_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC143.extendedFrags.fastq.gz
[FLASH]     ./SC143.notCombined_1.fastq.gz
[FLASH]     ./SC143.notCombined_2.fastq.gz
[FLASH]     ./SC143.hist
[FLASH]     ./SC143.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2582 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2582
[FLASH]     Combined pairs:   122
[FLASH]     Uncombined pairs: 2460
[FLASH]     Percent combined: 4.73%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.032 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

122 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC143/SC143_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC143_centroids.fasta --uc SC143.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 122 sequences and identified 91 clusters (clustering threshold 1.00) - 2 clusters (representing 20 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC157 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC157_1.fastq /home/working/demultiplexed/SC157_2.fastq SC157_forward.paired.fastq.gz SC157_forward.singletons.fastq.gz SC157_reverse.paired.fastq.gz SC157_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC157_1.fastq /home/working/demultiplexed/SC157_2.fastq SC157_forward.paired.fastq.gz SC157_forward.singletons.fastq.gz SC157_reverse.paired.fastq.gz SC157_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:34 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC157/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 4782 Both Surviving: 123 (2.57%) Forward Only Surviving: 143 (2.99%) Reverse Only Surviving: 56 (1.17%) Dropped: 4460 (93.27%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC157_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC157_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC157_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC157_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC157_forward.paired.headclipped.fastq.gz SC157_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC157 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC157_forward.paired.headclipped.fastq.gz
[FLASH]     SC157_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC157.extendedFrags.fastq.gz
[FLASH]     ./SC157.notCombined_1.fastq.gz
[FLASH]     ./SC157.notCombined_2.fastq.gz
[FLASH]     ./SC157.hist
[FLASH]     ./SC157.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 123 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      123
[FLASH]     Combined pairs:   34
[FLASH]     Uncombined pairs: 89
[FLASH]     Percent combined: 27.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

34 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC157/SC157_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC157_centroids.fasta --uc SC157.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 34 sequences and identified 30 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC16 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC16_1.fastq /home/working/demultiplexed/SC16_2.fastq SC16_forward.paired.fastq.gz SC16_forward.singletons.fastq.gz SC16_reverse.paired.fastq.gz SC16_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC16_1.fastq /home/working/demultiplexed/SC16_2.fastq SC16_forward.paired.fastq.gz SC16_forward.singletons.fastq.gz SC16_reverse.paired.fastq.gz SC16_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:36 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC16/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 33389 Both Surviving: 9086 (27.21%) Forward Only Surviving: 3296 (9.87%) Reverse Only Surviving: 356 (1.07%) Dropped: 20651 (61.85%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC16_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC16_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC16_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC16_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC16_forward.paired.headclipped.fastq.gz SC16_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC16 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC16_forward.paired.headclipped.fastq.gz
[FLASH]     SC16_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC16.extendedFrags.fastq.gz
[FLASH]     ./SC16.notCombined_1.fastq.gz
[FLASH]     ./SC16.notCombined_2.fastq.gz
[FLASH]     ./SC16.hist
[FLASH]     ./SC16.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9086 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9086
[FLASH]     Combined pairs:   8024
[FLASH]     Uncombined pairs: 1062
[FLASH]     Percent combined: 88.31%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.246 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

8024 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC16/SC16_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC16_centroids.fasta --uc SC16.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 8024 sequences and identified 3299 clusters (clustering threshold 1.00) - 194 clusters (representing 4308 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC168 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC168_1.fastq /home/working/demultiplexed/SC168_2.fastq SC168_forward.paired.fastq.gz SC168_forward.singletons.fastq.gz SC168_reverse.paired.fastq.gz SC168_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC168_1.fastq /home/working/demultiplexed/SC168_2.fastq SC168_forward.paired.fastq.gz SC168_forward.singletons.fastq.gz SC168_reverse.paired.fastq.gz SC168_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:54 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC168/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37551 Both Surviving: 5209 (13.87%) Forward Only Surviving: 1233 (3.28%) Reverse Only Surviving: 602 (1.60%) Dropped: 30507 (81.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC168_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC168_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC168_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC168_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC168_forward.paired.headclipped.fastq.gz SC168_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC168 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC168_forward.paired.headclipped.fastq.gz
[FLASH]     SC168_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC168.extendedFrags.fastq.gz
[FLASH]     ./SC168.notCombined_1.fastq.gz
[FLASH]     ./SC168.notCombined_2.fastq.gz
[FLASH]     ./SC168.hist
[FLASH]     ./SC168.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5209 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5209
[FLASH]     Combined pairs:   19
[FLASH]     Uncombined pairs: 5190
[FLASH]     Percent combined: 0.36%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.039 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

19 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC168/SC168_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC168_centroids.fasta --uc SC168.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 19 sequences and identified 19 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC178 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC178_1.fastq /home/working/demultiplexed/SC178_2.fastq SC178_forward.paired.fastq.gz SC178_forward.singletons.fastq.gz SC178_reverse.paired.fastq.gz SC178_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC178_1.fastq /home/working/demultiplexed/SC178_2.fastq SC178_forward.paired.fastq.gz SC178_forward.singletons.fastq.gz SC178_reverse.paired.fastq.gz SC178_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:58 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC178/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SC178_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC178_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC178_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC178_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC178_forward.paired.headclipped.fastq.gz SC178_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC178 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC178_forward.paired.headclipped.fastq.gz
[FLASH]     SC178_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC178.extendedFrags.fastq.gz
[FLASH]     ./SC178.notCombined_1.fastq.gz
[FLASH]     ./SC178.notCombined_2.fastq.gz
[FLASH]     ./SC178.hist
[FLASH]     ./SC178.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC178/SC178_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC178_centroids.fasta --uc SC178.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC187 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC187_1.fastq /home/working/demultiplexed/SC187_2.fastq SC187_forward.paired.fastq.gz SC187_forward.singletons.fastq.gz SC187_reverse.paired.fastq.gz SC187_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC187_1.fastq /home/working/demultiplexed/SC187_2.fastq SC187_forward.paired.fastq.gz SC187_forward.singletons.fastq.gz SC187_reverse.paired.fastq.gz SC187_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:26:59 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC187/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20627 Both Surviving: 1801 (8.73%) Forward Only Surviving: 617 (2.99%) Reverse Only Surviving: 312 (1.51%) Dropped: 17897 (86.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC187_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC187_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC187_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC187_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC187_forward.paired.headclipped.fastq.gz SC187_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC187 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC187_forward.paired.headclipped.fastq.gz
[FLASH]     SC187_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC187.extendedFrags.fastq.gz
[FLASH]     ./SC187.notCombined_1.fastq.gz
[FLASH]     ./SC187.notCombined_2.fastq.gz
[FLASH]     ./SC187.hist
[FLASH]     ./SC187.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1801 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1801
[FLASH]     Combined pairs:   22
[FLASH]     Uncombined pairs: 1779
[FLASH]     Percent combined: 1.22%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.014 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

22 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC187/SC187_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC187_centroids.fasta --uc SC187.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 22 sequences and identified 17 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC198 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC198_1.fastq /home/working/demultiplexed/SC198_2.fastq SC198_forward.paired.fastq.gz SC198_forward.singletons.fastq.gz SC198_reverse.paired.fastq.gz SC198_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC198_1.fastq /home/working/demultiplexed/SC198_2.fastq SC198_forward.paired.fastq.gz SC198_forward.singletons.fastq.gz SC198_reverse.paired.fastq.gz SC198_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:27:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC198/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17339 Both Surviving: 3301 (19.04%) Forward Only Surviving: 903 (5.21%) Reverse Only Surviving: 242 (1.40%) Dropped: 12893 (74.36%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC198_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC198_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC198_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC198_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC198_forward.paired.headclipped.fastq.gz SC198_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC198 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC198_forward.paired.headclipped.fastq.gz
[FLASH]     SC198_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC198.extendedFrags.fastq.gz
[FLASH]     ./SC198.notCombined_1.fastq.gz
[FLASH]     ./SC198.notCombined_2.fastq.gz
[FLASH]     ./SC198.hist
[FLASH]     ./SC198.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3301 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3301
[FLASH]     Combined pairs:   51
[FLASH]     Uncombined pairs: 3250
[FLASH]     Percent combined: 1.54%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.019 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

51 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC198/SC198_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC198_centroids.fasta --uc SC198.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 51 sequences and identified 44 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC19A #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC19A_1.fastq /home/working/demultiplexed/SC19A_2.fastq SC19A_forward.paired.fastq.gz SC19A_forward.singletons.fastq.gz SC19A_reverse.paired.fastq.gz SC19A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC19A_1.fastq /home/working/demultiplexed/SC19A_2.fastq SC19A_forward.paired.fastq.gz SC19A_forward.singletons.fastq.gz SC19A_reverse.paired.fastq.gz SC19A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:27:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC19A/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31576 Both Surviving: 2907 (9.21%) Forward Only Surviving: 964 (3.05%) Reverse Only Surviving: 422 (1.34%) Dropped: 27283 (86.40%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SC19A_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SC19A_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SC19A_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SC19A_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC19A_forward.paired.headclipped.fastq.gz SC19A_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC19A -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC19A_forward.paired.headclipped.fastq.gz
[FLASH]     SC19A_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC19A.extendedFrags.fastq.gz
[FLASH]     ./SC19A.notCombined_1.fastq.gz
[FLASH]     ./SC19A.notCombined_2.fastq.gz
[FLASH]     ./SC19A.hist
[FLASH]     ./SC19A.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2907 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2907
[FLASH]     Combined pairs:   2641
[FLASH]     Uncombined pairs: 266
[FLASH]     Percent combined: 90.85%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.104 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2641 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC19A/SC19A_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC19A_centroids.fasta --uc SC19A.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2641 sequences and identified 1242 clusters (clustering threshold 1.00) - 23 clusters (representing 1210 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC202 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC202_1.fastq /home/working/demultiplexed/SC202_2.fastq SC202_forward.paired.fastq.gz SC202_forward.singletons.fastq.gz SC202_reverse.paired.fastq.gz SC202_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC202_1.fastq /home/working/demultiplexed/SC202_2.fastq SC202_forward.paired.fastq.gz SC202_forward.singletons.fastq.gz SC202_reverse.paired.fastq.gz SC202_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:27:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC202/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20207 Both Surviving: 377 (1.87%) Forward Only Surviving: 334 (1.65%) Reverse Only Surviving: 179 (0.89%) Dropped: 19317 (95.60%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC202_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC202_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC202_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC202_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC202_forward.paired.headclipped.fastq.gz SC202_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC202 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC202_forward.paired.headclipped.fastq.gz
[FLASH]     SC202_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC202.extendedFrags.fastq.gz
[FLASH]     ./SC202.notCombined_1.fastq.gz
[FLASH]     ./SC202.notCombined_2.fastq.gz
[FLASH]     ./SC202.hist
[FLASH]     ./SC202.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 377 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      377
[FLASH]     Combined pairs:   7
[FLASH]     Uncombined pairs: 370
[FLASH]     Percent combined: 1.86%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

7 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC202/SC202_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC202_centroids.fasta --uc SC202.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 7 sequences and identified 6 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC26 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC26_1.fastq /home/working/demultiplexed/SC26_2.fastq SC26_forward.paired.fastq.gz SC26_forward.singletons.fastq.gz SC26_reverse.paired.fastq.gz SC26_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC26_1.fastq /home/working/demultiplexed/SC26_2.fastq SC26_forward.paired.fastq.gz SC26_forward.singletons.fastq.gz SC26_reverse.paired.fastq.gz SC26_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:27:16 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC26/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 38586 Both Surviving: 14376 (37.26%) Forward Only Surviving: 3099 (8.03%) Reverse Only Surviving: 648 (1.68%) Dropped: 20463 (53.03%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC26_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC26_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SC26_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SC26_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC26_forward.paired.headclipped.fastq.gz SC26_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC26 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC26_forward.paired.headclipped.fastq.gz
[FLASH]     SC26_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC26.extendedFrags.fastq.gz
[FLASH]     ./SC26.notCombined_1.fastq.gz
[FLASH]     ./SC26.notCombined_2.fastq.gz
[FLASH]     ./SC26.hist
[FLASH]     ./SC26.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 14376 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      14376
[FLASH]     Combined pairs:   7773
[FLASH]     Uncombined pairs: 6603
[FLASH]     Percent combined: 54.07%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.213 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

7773 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC26/SC26_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC26_centroids.fasta --uc SC26.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 7773 sequences and identified 3818 clusters (clustering threshold 1.00) - 209 clusters (representing 3667 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC28 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC28_1.fastq /home/working/demultiplexed/SC28_2.fastq SC28_forward.paired.fastq.gz SC28_forward.singletons.fastq.gz SC28_reverse.paired.fastq.gz SC28_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC28_1.fastq /home/working/demultiplexed/SC28_2.fastq SC28_forward.paired.fastq.gz SC28_forward.singletons.fastq.gz SC28_reverse.paired.fastq.gz SC28_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:27:31 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC28/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25262 Both Surviving: 3707 (14.67%) Forward Only Surviving: 1212 (4.80%) Reverse Only Surviving: 310 (1.23%) Dropped: 20033 (79.30%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC28_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC28_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC28_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC28_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC28_forward.paired.headclipped.fastq.gz SC28_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC28 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC28_forward.paired.headclipped.fastq.gz
[FLASH]     SC28_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC28.extendedFrags.fastq.gz
[FLASH]     ./SC28.notCombined_1.fastq.gz
[FLASH]     ./SC28.notCombined_2.fastq.gz
[FLASH]     ./SC28.hist
[FLASH]     ./SC28.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3707 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3707
[FLASH]     Combined pairs:   410
[FLASH]     Uncombined pairs: 3297
[FLASH]     Percent combined: 11.06%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.024 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

410 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC28/SC28_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC28_centroids.fasta --uc SC28.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 410 sequences and identified 243 clusters (clustering threshold 1.00) - 8 clusters (representing 159 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC34 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC34_1.fastq /home/working/demultiplexed/SC34_2.fastq SC34_forward.paired.fastq.gz SC34_forward.singletons.fastq.gz SC34_reverse.paired.fastq.gz SC34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC34_1.fastq /home/working/demultiplexed/SC34_2.fastq SC34_forward.paired.fastq.gz SC34_forward.singletons.fastq.gz SC34_reverse.paired.fastq.gz SC34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:27:36 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC34/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28305 Both Surviving: 36 (0.13%) Forward Only Surviving: 490 (1.73%) Reverse Only Surviving: 366 (1.29%) Dropped: 27413 (96.85%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SC34_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SC34_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC34_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC34_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC34_forward.paired.headclipped.fastq.gz SC34_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC34 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC34_forward.paired.headclipped.fastq.gz
[FLASH]     SC34_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC34.extendedFrags.fastq.gz
[FLASH]     ./SC34.notCombined_1.fastq.gz
[FLASH]     ./SC34.notCombined_2.fastq.gz
[FLASH]     ./SC34.hist
[FLASH]     ./SC34.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 36 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      36
[FLASH]     Combined pairs:   12
[FLASH]     Uncombined pairs: 24
[FLASH]     Percent combined: 33.33%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

12 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC34/SC34_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC34_centroids.fasta --uc SC34.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 12 sequences and identified 10 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC35 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC35_1.fastq /home/working/demultiplexed/SC35_2.fastq SC35_forward.paired.fastq.gz SC35_forward.singletons.fastq.gz SC35_reverse.paired.fastq.gz SC35_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC35_1.fastq /home/working/demultiplexed/SC35_2.fastq SC35_forward.paired.fastq.gz SC35_forward.singletons.fastq.gz SC35_reverse.paired.fastq.gz SC35_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:27:39 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC35/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22422 Both Surviving: 6879 (30.68%) Forward Only Surviving: 2026 (9.04%) Reverse Only Surviving: 280 (1.25%) Dropped: 13237 (59.04%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC35_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC35_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC35_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC35_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC35_forward.paired.headclipped.fastq.gz SC35_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC35 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC35_forward.paired.headclipped.fastq.gz
[FLASH]     SC35_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC35.extendedFrags.fastq.gz
[FLASH]     ./SC35.notCombined_1.fastq.gz
[FLASH]     ./SC35.notCombined_2.fastq.gz
[FLASH]     ./SC35.hist
[FLASH]     ./SC35.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6879 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6879
[FLASH]     Combined pairs:   1981
[FLASH]     Uncombined pairs: 4898
[FLASH]     Percent combined: 28.80%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.063 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1981 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC35/SC35_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC35_centroids.fasta --uc SC35.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1981 sequences and identified 1341 clusters (clustering threshold 1.00) - 41 clusters (representing 505 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC43 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC43_1.fastq /home/working/demultiplexed/SC43_2.fastq SC43_forward.paired.fastq.gz SC43_forward.singletons.fastq.gz SC43_reverse.paired.fastq.gz SC43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC43_1.fastq /home/working/demultiplexed/SC43_2.fastq SC43_forward.paired.fastq.gz SC43_forward.singletons.fastq.gz SC43_reverse.paired.fastq.gz SC43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:27:46 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC43/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16887 Both Surviving: 490 (2.90%) Forward Only Surviving: 524 (3.10%) Reverse Only Surviving: 123 (0.73%) Dropped: 15750 (93.27%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC43_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC43_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC43_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC43_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC43_forward.paired.headclipped.fastq.gz SC43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC43 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC43_forward.paired.headclipped.fastq.gz
[FLASH]     SC43_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC43.extendedFrags.fastq.gz
[FLASH]     ./SC43.notCombined_1.fastq.gz
[FLASH]     ./SC43.notCombined_2.fastq.gz
[FLASH]     ./SC43.hist
[FLASH]     ./SC43.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 490 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      490
[FLASH]     Combined pairs:   330
[FLASH]     Uncombined pairs: 160
[FLASH]     Percent combined: 67.35%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.013 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

330 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC43/SC43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC43_centroids.fasta --uc SC43.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 330 sequences and identified 253 clusters (clustering threshold 1.00) - 10 clusters (representing 65 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC44 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC44_1.fastq /home/working/demultiplexed/SC44_2.fastq SC44_forward.paired.fastq.gz SC44_forward.singletons.fastq.gz SC44_reverse.paired.fastq.gz SC44_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC44_1.fastq /home/working/demultiplexed/SC44_2.fastq SC44_forward.paired.fastq.gz SC44_forward.singletons.fastq.gz SC44_reverse.paired.fastq.gz SC44_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:27:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC44/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18958 Both Surviving: 4960 (26.16%) Forward Only Surviving: 1556 (8.21%) Reverse Only Surviving: 276 (1.46%) Dropped: 12166 (64.17%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC44_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC44_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC44_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC44_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC44_forward.paired.headclipped.fastq.gz SC44_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC44 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC44_forward.paired.headclipped.fastq.gz
[FLASH]     SC44_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC44.extendedFrags.fastq.gz
[FLASH]     ./SC44.notCombined_1.fastq.gz
[FLASH]     ./SC44.notCombined_2.fastq.gz
[FLASH]     ./SC44.hist
[FLASH]     ./SC44.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4960 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4960
[FLASH]     Combined pairs:   146
[FLASH]     Uncombined pairs: 4814
[FLASH]     Percent combined: 2.94%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.044 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

146 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC44/SC44_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC44_centroids.fasta --uc SC44.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 146 sequences and identified 137 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC48 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC48_1.fastq /home/working/demultiplexed/SC48_2.fastq SC48_forward.paired.fastq.gz SC48_forward.singletons.fastq.gz SC48_reverse.paired.fastq.gz SC48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC48_1.fastq /home/working/demultiplexed/SC48_2.fastq SC48_forward.paired.fastq.gz SC48_forward.singletons.fastq.gz SC48_reverse.paired.fastq.gz SC48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:27:53 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC48/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20800 Both Surviving: 72 (0.35%) Forward Only Surviving: 318 (1.53%) Reverse Only Surviving: 200 (0.96%) Dropped: 20210 (97.16%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SC48_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SC48_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC48_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC48_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC48_forward.paired.headclipped.fastq.gz SC48_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC48 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC48_forward.paired.headclipped.fastq.gz
[FLASH]     SC48_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC48.extendedFrags.fastq.gz
[FLASH]     ./SC48.notCombined_1.fastq.gz
[FLASH]     ./SC48.notCombined_2.fastq.gz
[FLASH]     ./SC48.hist
[FLASH]     ./SC48.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 72 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      72
[FLASH]     Combined pairs:   24
[FLASH]     Uncombined pairs: 48
[FLASH]     Percent combined: 33.33%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

24 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC48/SC48_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC48_centroids.fasta --uc SC48.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 24 sequences and identified 20 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC49 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC49_1.fastq /home/working/demultiplexed/SC49_2.fastq SC49_forward.paired.fastq.gz SC49_forward.singletons.fastq.gz SC49_reverse.paired.fastq.gz SC49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC49_1.fastq /home/working/demultiplexed/SC49_2.fastq SC49_forward.paired.fastq.gz SC49_forward.singletons.fastq.gz SC49_reverse.paired.fastq.gz SC49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:27:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC49/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31250 Both Surviving: 6674 (21.36%) Forward Only Surviving: 2063 (6.60%) Reverse Only Surviving: 491 (1.57%) Dropped: 22022 (70.47%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC49_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC49_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SC49_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC49_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC49_forward.paired.headclipped.fastq.gz SC49_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC49 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC49_forward.paired.headclipped.fastq.gz
[FLASH]     SC49_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC49.extendedFrags.fastq.gz
[FLASH]     ./SC49.notCombined_1.fastq.gz
[FLASH]     ./SC49.notCombined_2.fastq.gz
[FLASH]     ./SC49.hist
[FLASH]     ./SC49.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6674 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6674
[FLASH]     Combined pairs:   899
[FLASH]     Uncombined pairs: 5775
[FLASH]     Percent combined: 13.47%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.046 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

899 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC49/SC49_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC49_centroids.fasta --uc SC49.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 899 sequences and identified 406 clusters (clustering threshold 1.00) - 9 clusters (representing 426 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC51 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC51_1.fastq /home/working/demultiplexed/SC51_2.fastq SC51_forward.paired.fastq.gz SC51_forward.singletons.fastq.gz SC51_reverse.paired.fastq.gz SC51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC51_1.fastq /home/working/demultiplexed/SC51_2.fastq SC51_forward.paired.fastq.gz SC51_forward.singletons.fastq.gz SC51_reverse.paired.fastq.gz SC51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC51/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23723 Both Surviving: 886 (3.73%) Forward Only Surviving: 537 (2.26%) Reverse Only Surviving: 282 (1.19%) Dropped: 22018 (92.81%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC51_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC51_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SC51_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SC51_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC51_forward.paired.headclipped.fastq.gz SC51_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC51 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC51_forward.paired.headclipped.fastq.gz
[FLASH]     SC51_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC51.extendedFrags.fastq.gz
[FLASH]     ./SC51.notCombined_1.fastq.gz
[FLASH]     ./SC51.notCombined_2.fastq.gz
[FLASH]     ./SC51.hist
[FLASH]     ./SC51.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 886 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      886
[FLASH]     Combined pairs:   396
[FLASH]     Uncombined pairs: 490
[FLASH]     Percent combined: 44.70%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.017 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

396 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC51/SC51_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC51_centroids.fasta --uc SC51.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 396 sequences and identified 187 clusters (clustering threshold 1.00) - 2 clusters (representing 184 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC51A #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC51A_1.fastq /home/working/demultiplexed/SC51A_2.fastq SC51A_forward.paired.fastq.gz SC51A_forward.singletons.fastq.gz SC51A_reverse.paired.fastq.gz SC51A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC51A_1.fastq /home/working/demultiplexed/SC51A_2.fastq SC51A_forward.paired.fastq.gz SC51A_forward.singletons.fastq.gz SC51A_reverse.paired.fastq.gz SC51A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:04 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC51A/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14869 Both Surviving: 83 (0.56%) Forward Only Surviving: 247 (1.66%) Reverse Only Surviving: 92 (0.62%) Dropped: 14447 (97.16%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC51A_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC51A_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC51A_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC51A_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC51A_forward.paired.headclipped.fastq.gz SC51A_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC51A -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC51A_forward.paired.headclipped.fastq.gz
[FLASH]     SC51A_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC51A.extendedFrags.fastq.gz
[FLASH]     ./SC51A.notCombined_1.fastq.gz
[FLASH]     ./SC51A.notCombined_2.fastq.gz
[FLASH]     ./SC51A.hist
[FLASH]     ./SC51A.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 83 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      83
[FLASH]     Combined pairs:   40
[FLASH]     Uncombined pairs: 43
[FLASH]     Percent combined: 48.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

40 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC51A/SC51A_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC51A_centroids.fasta --uc SC51A.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 40 sequences and identified 31 clusters (clustering threshold 1.00) - 2 clusters (representing 10 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC54A #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC54A_1.fastq /home/working/demultiplexed/SC54A_2.fastq SC54A_forward.paired.fastq.gz SC54A_forward.singletons.fastq.gz SC54A_reverse.paired.fastq.gz SC54A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC54A_1.fastq /home/working/demultiplexed/SC54A_2.fastq SC54A_forward.paired.fastq.gz SC54A_forward.singletons.fastq.gz SC54A_reverse.paired.fastq.gz SC54A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:07 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC54A/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 21305 Both Surviving: 2339 (10.98%) Forward Only Surviving: 692 (3.25%) Reverse Only Surviving: 326 (1.53%) Dropped: 17948 (84.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC54A_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC54A_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC54A_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC54A_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC54A_forward.paired.headclipped.fastq.gz SC54A_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC54A -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC54A_forward.paired.headclipped.fastq.gz
[FLASH]     SC54A_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC54A.extendedFrags.fastq.gz
[FLASH]     ./SC54A.notCombined_1.fastq.gz
[FLASH]     ./SC54A.notCombined_2.fastq.gz
[FLASH]     ./SC54A.hist
[FLASH]     ./SC54A.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2339 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2339
[FLASH]     Combined pairs:   270
[FLASH]     Uncombined pairs: 2069
[FLASH]     Percent combined: 11.54%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.020 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

270 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC54A/SC54A_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC54A_centroids.fasta --uc SC54A.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 270 sequences and identified 148 clusters (clustering threshold 1.00) - 3 clusters (representing 113 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC55 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC55_1.fastq /home/working/demultiplexed/SC55_2.fastq SC55_forward.paired.fastq.gz SC55_forward.singletons.fastq.gz SC55_reverse.paired.fastq.gz SC55_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC55_1.fastq /home/working/demultiplexed/SC55_2.fastq SC55_forward.paired.fastq.gz SC55_forward.singletons.fastq.gz SC55_reverse.paired.fastq.gz SC55_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC55/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22557 Both Surviving: 87 (0.39%) Forward Only Surviving: 392 (1.74%) Reverse Only Surviving: 196 (0.87%) Dropped: 21882 (97.01%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC55_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC55_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC55_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC55_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC55_forward.paired.headclipped.fastq.gz SC55_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC55 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC55_forward.paired.headclipped.fastq.gz
[FLASH]     SC55_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC55.extendedFrags.fastq.gz
[FLASH]     ./SC55.notCombined_1.fastq.gz
[FLASH]     ./SC55.notCombined_2.fastq.gz
[FLASH]     ./SC55.hist
[FLASH]     ./SC55.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 87 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      87
[FLASH]     Combined pairs:   27
[FLASH]     Uncombined pairs: 60
[FLASH]     Percent combined: 31.03%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

27 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC55/SC55_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC55_centroids.fasta --uc SC55.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 27 sequences and identified 22 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC57 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC57_1.fastq /home/working/demultiplexed/SC57_2.fastq SC57_forward.paired.fastq.gz SC57_forward.singletons.fastq.gz SC57_reverse.paired.fastq.gz SC57_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC57_1.fastq /home/working/demultiplexed/SC57_2.fastq SC57_forward.paired.fastq.gz SC57_forward.singletons.fastq.gz SC57_reverse.paired.fastq.gz SC57_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC57/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41048 Both Surviving: 3355 (8.17%) Forward Only Surviving: 1317 (3.21%) Reverse Only Surviving: 345 (0.84%) Dropped: 36031 (87.78%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC57_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC57_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC57_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC57_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC57_forward.paired.headclipped.fastq.gz SC57_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC57 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC57_forward.paired.headclipped.fastq.gz
[FLASH]     SC57_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC57.extendedFrags.fastq.gz
[FLASH]     ./SC57.notCombined_1.fastq.gz
[FLASH]     ./SC57.notCombined_2.fastq.gz
[FLASH]     ./SC57.hist
[FLASH]     ./SC57.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3355 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3355
[FLASH]     Combined pairs:   2894
[FLASH]     Uncombined pairs: 461
[FLASH]     Percent combined: 86.26%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.086 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2894 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC57/SC57_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC57_centroids.fasta --uc SC57.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2894 sequences and identified 1559 clusters (clustering threshold 1.00) - 25 clusters (representing 1111 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC60 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC60_1.fastq /home/working/demultiplexed/SC60_2.fastq SC60_forward.paired.fastq.gz SC60_forward.singletons.fastq.gz SC60_reverse.paired.fastq.gz SC60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC60_1.fastq /home/working/demultiplexed/SC60_2.fastq SC60_forward.paired.fastq.gz SC60_forward.singletons.fastq.gz SC60_reverse.paired.fastq.gz SC60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC60/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37624 Both Surviving: 4932 (13.11%) Forward Only Surviving: 1535 (4.08%) Reverse Only Surviving: 322 (0.86%) Dropped: 30835 (81.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC60_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC60_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC60_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC60_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC60_forward.paired.headclipped.fastq.gz SC60_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC60 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC60_forward.paired.headclipped.fastq.gz
[FLASH]     SC60_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC60.extendedFrags.fastq.gz
[FLASH]     ./SC60.notCombined_1.fastq.gz
[FLASH]     ./SC60.notCombined_2.fastq.gz
[FLASH]     ./SC60.hist
[FLASH]     ./SC60.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4932 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4932
[FLASH]     Combined pairs:   4428
[FLASH]     Uncombined pairs: 504
[FLASH]     Percent combined: 89.78%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.121 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4428 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC60/SC60_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC60_centroids.fasta --uc SC60.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4428 sequences and identified 2621 clusters (clustering threshold 1.00) - 51 clusters (representing 1597 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC61 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC61_1.fastq /home/working/demultiplexed/SC61_2.fastq SC61_forward.paired.fastq.gz SC61_forward.singletons.fastq.gz SC61_reverse.paired.fastq.gz SC61_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC61_1.fastq /home/working/demultiplexed/SC61_2.fastq SC61_forward.paired.fastq.gz SC61_forward.singletons.fastq.gz SC61_reverse.paired.fastq.gz SC61_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:32 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC61/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18563 Both Surviving: 312 (1.68%) Forward Only Surviving: 310 (1.67%) Reverse Only Surviving: 307 (1.65%) Dropped: 17634 (95.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC61_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC61_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC61_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC61_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC61_forward.paired.headclipped.fastq.gz SC61_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC61 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC61_forward.paired.headclipped.fastq.gz
[FLASH]     SC61_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC61.extendedFrags.fastq.gz
[FLASH]     ./SC61.notCombined_1.fastq.gz
[FLASH]     ./SC61.notCombined_2.fastq.gz
[FLASH]     ./SC61.hist
[FLASH]     ./SC61.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 312 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      312
[FLASH]     Combined pairs:   4
[FLASH]     Uncombined pairs: 308
[FLASH]     Percent combined: 1.28%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC61/SC61_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC61_centroids.fasta --uc SC61.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4 sequences and identified 4 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC66 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC66_1.fastq /home/working/demultiplexed/SC66_2.fastq SC66_forward.paired.fastq.gz SC66_forward.singletons.fastq.gz SC66_reverse.paired.fastq.gz SC66_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC66_1.fastq /home/working/demultiplexed/SC66_2.fastq SC66_forward.paired.fastq.gz SC66_forward.singletons.fastq.gz SC66_reverse.paired.fastq.gz SC66_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:34 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC66/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 8462 Both Surviving: 1017 (12.02%) Forward Only Surviving: 497 (5.87%) Reverse Only Surviving: 76 (0.90%) Dropped: 6872 (81.21%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC66_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC66_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC66_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC66_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC66_forward.paired.headclipped.fastq.gz SC66_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC66 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC66_forward.paired.headclipped.fastq.gz
[FLASH]     SC66_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC66.extendedFrags.fastq.gz
[FLASH]     ./SC66.notCombined_1.fastq.gz
[FLASH]     ./SC66.notCombined_2.fastq.gz
[FLASH]     ./SC66.hist
[FLASH]     ./SC66.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1017 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1017
[FLASH]     Combined pairs:   811
[FLASH]     Uncombined pairs: 206
[FLASH]     Percent combined: 79.74%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.022 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

811 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC66/SC66_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC66_centroids.fasta --uc SC66.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 811 sequences and identified 360 clusters (clustering threshold 1.00) - 7 clusters (representing 410 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC7 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC7_1.fastq /home/working/demultiplexed/SC7_2.fastq SC7_forward.paired.fastq.gz SC7_forward.singletons.fastq.gz SC7_reverse.paired.fastq.gz SC7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC7_1.fastq /home/working/demultiplexed/SC7_2.fastq SC7_forward.paired.fastq.gz SC7_forward.singletons.fastq.gz SC7_reverse.paired.fastq.gz SC7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:36 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC7/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 4 Both Surviving: 0 (0.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 1 (25.00%) Dropped: 3 (75.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SC7_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC7_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SC7_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC7_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC7_forward.paired.headclipped.fastq.gz SC7_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC7 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC7_forward.paired.headclipped.fastq.gz
[FLASH]     SC7_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC7.extendedFrags.fastq.gz
[FLASH]     ./SC7.notCombined_1.fastq.gz
[FLASH]     ./SC7.notCombined_2.fastq.gz
[FLASH]     ./SC7.hist
[FLASH]     ./SC7.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC7/SC7_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC7_centroids.fasta --uc SC7.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC72 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC72_1.fastq /home/working/demultiplexed/SC72_2.fastq SC72_forward.paired.fastq.gz SC72_forward.singletons.fastq.gz SC72_reverse.paired.fastq.gz SC72_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC72_1.fastq /home/working/demultiplexed/SC72_2.fastq SC72_forward.paired.fastq.gz SC72_forward.singletons.fastq.gz SC72_reverse.paired.fastq.gz SC72_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC72/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9942 Both Surviving: 120 (1.21%) Forward Only Surviving: 158 (1.59%) Reverse Only Surviving: 139 (1.40%) Dropped: 9525 (95.81%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC72_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC72_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC72_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC72_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC72_forward.paired.headclipped.fastq.gz SC72_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC72 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC72_forward.paired.headclipped.fastq.gz
[FLASH]     SC72_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC72.extendedFrags.fastq.gz
[FLASH]     ./SC72.notCombined_1.fastq.gz
[FLASH]     ./SC72.notCombined_2.fastq.gz
[FLASH]     ./SC72.hist
[FLASH]     ./SC72.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 120 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      120
[FLASH]     Combined pairs:   9
[FLASH]     Uncombined pairs: 111
[FLASH]     Percent combined: 7.50%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

9 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC72/SC72_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC72_centroids.fasta --uc SC72.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 9 sequences and identified 9 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC74 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC74_1.fastq /home/working/demultiplexed/SC74_2.fastq SC74_forward.paired.fastq.gz SC74_forward.singletons.fastq.gz SC74_reverse.paired.fastq.gz SC74_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC74_1.fastq /home/working/demultiplexed/SC74_2.fastq SC74_forward.paired.fastq.gz SC74_forward.singletons.fastq.gz SC74_reverse.paired.fastq.gz SC74_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:38 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC74/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14010 Both Surviving: 31 (0.22%) Forward Only Surviving: 250 (1.78%) Reverse Only Surviving: 64 (0.46%) Dropped: 13665 (97.54%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC74_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC74_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC74_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC74_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC74_forward.paired.headclipped.fastq.gz SC74_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC74 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC74_forward.paired.headclipped.fastq.gz
[FLASH]     SC74_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC74.extendedFrags.fastq.gz
[FLASH]     ./SC74.notCombined_1.fastq.gz
[FLASH]     ./SC74.notCombined_2.fastq.gz
[FLASH]     ./SC74.hist
[FLASH]     ./SC74.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 31 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      31
[FLASH]     Combined pairs:   20
[FLASH]     Uncombined pairs: 11
[FLASH]     Percent combined: 64.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

20 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC74/SC74_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC74_centroids.fasta --uc SC74.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 20 sequences and identified 19 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC77 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC77_1.fastq /home/working/demultiplexed/SC77_2.fastq SC77_forward.paired.fastq.gz SC77_forward.singletons.fastq.gz SC77_reverse.paired.fastq.gz SC77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC77_1.fastq /home/working/demultiplexed/SC77_2.fastq SC77_forward.paired.fastq.gz SC77_forward.singletons.fastq.gz SC77_reverse.paired.fastq.gz SC77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC77/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28064 Both Surviving: 1951 (6.95%) Forward Only Surviving: 902 (3.21%) Reverse Only Surviving: 222 (0.79%) Dropped: 24989 (89.04%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC77_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC77_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SC77_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC77_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC77_forward.paired.headclipped.fastq.gz SC77_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC77 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC77_forward.paired.headclipped.fastq.gz
[FLASH]     SC77_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC77.extendedFrags.fastq.gz
[FLASH]     ./SC77.notCombined_1.fastq.gz
[FLASH]     ./SC77.notCombined_2.fastq.gz
[FLASH]     ./SC77.hist
[FLASH]     ./SC77.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1951 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1951
[FLASH]     Combined pairs:   1605
[FLASH]     Uncombined pairs: 346
[FLASH]     Percent combined: 82.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.042 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1605 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC77/SC77_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC77_centroids.fasta --uc SC77.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1605 sequences and identified 885 clusters (clustering threshold 1.00) - 18 clusters (representing 637 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC78 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC78_1.fastq /home/working/demultiplexed/SC78_2.fastq SC78_forward.paired.fastq.gz SC78_forward.singletons.fastq.gz SC78_reverse.paired.fastq.gz SC78_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC78_1.fastq /home/working/demultiplexed/SC78_2.fastq SC78_forward.paired.fastq.gz SC78_forward.singletons.fastq.gz SC78_reverse.paired.fastq.gz SC78_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:46 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC78/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14701 Both Surviving: 2518 (17.13%) Forward Only Surviving: 803 (5.46%) Reverse Only Surviving: 238 (1.62%) Dropped: 11142 (75.79%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC78_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC78_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC78_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC78_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC78_forward.paired.headclipped.fastq.gz SC78_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC78 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC78_forward.paired.headclipped.fastq.gz
[FLASH]     SC78_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC78.extendedFrags.fastq.gz
[FLASH]     ./SC78.notCombined_1.fastq.gz
[FLASH]     ./SC78.notCombined_2.fastq.gz
[FLASH]     ./SC78.hist
[FLASH]     ./SC78.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2518 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2518
[FLASH]     Combined pairs:   102
[FLASH]     Uncombined pairs: 2416
[FLASH]     Percent combined: 4.05%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.016 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

102 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC78/SC78_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC78_centroids.fasta --uc SC78.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 102 sequences and identified 82 clusters (clustering threshold 1.00) - 1 clusters (representing 12 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC81 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC81_1.fastq /home/working/demultiplexed/SC81_2.fastq SC81_forward.paired.fastq.gz SC81_forward.singletons.fastq.gz SC81_reverse.paired.fastq.gz SC81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC81_1.fastq /home/working/demultiplexed/SC81_2.fastq SC81_forward.paired.fastq.gz SC81_forward.singletons.fastq.gz SC81_reverse.paired.fastq.gz SC81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC81/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 24476 Both Surviving: 4241 (17.33%) Forward Only Surviving: 1081 (4.42%) Reverse Only Surviving: 382 (1.56%) Dropped: 18772 (76.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC81_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC81_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SC81_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC81_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC81_forward.paired.headclipped.fastq.gz SC81_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC81 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC81_forward.paired.headclipped.fastq.gz
[FLASH]     SC81_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC81.extendedFrags.fastq.gz
[FLASH]     ./SC81.notCombined_1.fastq.gz
[FLASH]     ./SC81.notCombined_2.fastq.gz
[FLASH]     ./SC81.hist
[FLASH]     ./SC81.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4241 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4241
[FLASH]     Combined pairs:   3838
[FLASH]     Uncombined pairs: 403
[FLASH]     Percent combined: 90.50%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.122 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3838 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC81/SC81_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC81_centroids.fasta --uc SC81.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3838 sequences and identified 2236 clusters (clustering threshold 1.00) - 47 clusters (representing 1401 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC82 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC82_1.fastq /home/working/demultiplexed/SC82_2.fastq SC82_forward.paired.fastq.gz SC82_forward.singletons.fastq.gz SC82_reverse.paired.fastq.gz SC82_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC82_1.fastq /home/working/demultiplexed/SC82_2.fastq SC82_forward.paired.fastq.gz SC82_forward.singletons.fastq.gz SC82_reverse.paired.fastq.gz SC82_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:28:57 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC82/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11078 Both Surviving: 1658 (14.97%) Forward Only Surviving: 712 (6.43%) Reverse Only Surviving: 156 (1.41%) Dropped: 8552 (77.20%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC82_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC82_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC82_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC82_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC82_forward.paired.headclipped.fastq.gz SC82_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC82 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC82_forward.paired.headclipped.fastq.gz
[FLASH]     SC82_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC82.extendedFrags.fastq.gz
[FLASH]     ./SC82.notCombined_1.fastq.gz
[FLASH]     ./SC82.notCombined_2.fastq.gz
[FLASH]     ./SC82.hist
[FLASH]     ./SC82.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1658 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1658
[FLASH]     Combined pairs:   228
[FLASH]     Uncombined pairs: 1430
[FLASH]     Percent combined: 13.75%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.014 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

228 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC82/SC82_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC82_centroids.fasta --uc SC82.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 228 sequences and identified 168 clusters (clustering threshold 1.00) - 6 clusters (representing 56 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC86A #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC86A_1.fastq /home/working/demultiplexed/SC86A_2.fastq SC86A_forward.paired.fastq.gz SC86A_forward.singletons.fastq.gz SC86A_reverse.paired.fastq.gz SC86A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC86A_1.fastq /home/working/demultiplexed/SC86A_2.fastq SC86A_forward.paired.fastq.gz SC86A_forward.singletons.fastq.gz SC86A_reverse.paired.fastq.gz SC86A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:29:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC86A/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 33212 Both Surviving: 5816 (17.51%) Forward Only Surviving: 1743 (5.25%) Reverse Only Surviving: 506 (1.52%) Dropped: 25147 (75.72%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC86A_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC86A_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SC86A_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC86A_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC86A_forward.paired.headclipped.fastq.gz SC86A_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC86A -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC86A_forward.paired.headclipped.fastq.gz
[FLASH]     SC86A_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC86A.extendedFrags.fastq.gz
[FLASH]     ./SC86A.notCombined_1.fastq.gz
[FLASH]     ./SC86A.notCombined_2.fastq.gz
[FLASH]     ./SC86A.hist
[FLASH]     ./SC86A.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5816 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5816
[FLASH]     Combined pairs:   1295
[FLASH]     Uncombined pairs: 4521
[FLASH]     Percent combined: 22.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.046 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1295 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC86A/SC86A_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC86A_centroids.fasta --uc SC86A.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1295 sequences and identified 585 clusters (clustering threshold 1.00) - 9 clusters (representing 583 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC88 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC88_1.fastq /home/working/demultiplexed/SC88_2.fastq SC88_forward.paired.fastq.gz SC88_forward.singletons.fastq.gz SC88_reverse.paired.fastq.gz SC88_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC88_1.fastq /home/working/demultiplexed/SC88_2.fastq SC88_forward.paired.fastq.gz SC88_forward.singletons.fastq.gz SC88_reverse.paired.fastq.gz SC88_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:29:07 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC88/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22460 Both Surviving: 3099 (13.80%) Forward Only Surviving: 1398 (6.22%) Reverse Only Surviving: 318 (1.42%) Dropped: 17645 (78.56%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC88_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC88_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC88_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC88_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC88_forward.paired.headclipped.fastq.gz SC88_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC88 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC88_forward.paired.headclipped.fastq.gz
[FLASH]     SC88_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC88.extendedFrags.fastq.gz
[FLASH]     ./SC88.notCombined_1.fastq.gz
[FLASH]     ./SC88.notCombined_2.fastq.gz
[FLASH]     ./SC88.hist
[FLASH]     ./SC88.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3099 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3099
[FLASH]     Combined pairs:   1576
[FLASH]     Uncombined pairs: 1523
[FLASH]     Percent combined: 50.86%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.046 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1576 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC88/SC88_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC88_centroids.fasta --uc SC88.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1576 sequences and identified 873 clusters (clustering threshold 1.00) - 22 clusters (representing 660 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC9 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC9_1.fastq /home/working/demultiplexed/SC9_2.fastq SC9_forward.paired.fastq.gz SC9_forward.singletons.fastq.gz SC9_reverse.paired.fastq.gz SC9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC9_1.fastq /home/working/demultiplexed/SC9_2.fastq SC9_forward.paired.fastq.gz SC9_forward.singletons.fastq.gz SC9_reverse.paired.fastq.gz SC9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:29:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC9/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 19434 Both Surviving: 882 (4.54%) Forward Only Surviving: 689 (3.55%) Reverse Only Surviving: 234 (1.20%) Dropped: 17629 (90.71%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC9_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC9_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC9_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC9_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC9_forward.paired.headclipped.fastq.gz SC9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC9 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC9_forward.paired.headclipped.fastq.gz
[FLASH]     SC9_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC9.extendedFrags.fastq.gz
[FLASH]     ./SC9.notCombined_1.fastq.gz
[FLASH]     ./SC9.notCombined_2.fastq.gz
[FLASH]     ./SC9.hist
[FLASH]     ./SC9.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 882 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      882
[FLASH]     Combined pairs:   665
[FLASH]     Uncombined pairs: 217
[FLASH]     Percent combined: 75.40%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.018 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

665 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC9/SC9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC9_centroids.fasta --uc SC9.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 665 sequences and identified 445 clusters (clustering threshold 1.00) - 15 clusters (representing 211 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC93 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC93_1.fastq /home/working/demultiplexed/SC93_2.fastq SC93_forward.paired.fastq.gz SC93_forward.singletons.fastq.gz SC93_reverse.paired.fastq.gz SC93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC93_1.fastq /home/working/demultiplexed/SC93_2.fastq SC93_forward.paired.fastq.gz SC93_forward.singletons.fastq.gz SC93_reverse.paired.fastq.gz SC93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:29:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/SC93/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 13069 Both Surviving: 496 (3.80%) Forward Only Surviving: 247 (1.89%) Reverse Only Surviving: 106 (0.81%) Dropped: 12220 (93.50%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SC93_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SC93_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC93_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC93_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC93_forward.paired.headclipped.fastq.gz SC93_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC93 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC93_forward.paired.headclipped.fastq.gz
[FLASH]     SC93_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC93.extendedFrags.fastq.gz
[FLASH]     ./SC93.notCombined_1.fastq.gz
[FLASH]     ./SC93.notCombined_2.fastq.gz
[FLASH]     ./SC93.hist
[FLASH]     ./SC93.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 496 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      496
[FLASH]     Combined pairs:   37
[FLASH]     Uncombined pairs: 459
[FLASH]     Percent combined: 7.46%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.005 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

37 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/SC93/SC93_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC93_centroids.fasta --uc SC93.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 37 sequences and identified 31 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: T19 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T19_1.fastq /home/working/demultiplexed/T19_2.fastq T19_forward.paired.fastq.gz T19_forward.singletons.fastq.gz T19_reverse.paired.fastq.gz T19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T19_1.fastq /home/working/demultiplexed/T19_2.fastq T19_forward.paired.fastq.gz T19_forward.singletons.fastq.gz T19_reverse.paired.fastq.gz T19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:29:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/T19/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29655 Both Surviving: 29 (0.10%) Forward Only Surviving: 265 (0.89%) Reverse Only Surviving: 90 (0.30%) Dropped: 29271 (98.71%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: T19_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: T19_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: T19_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: T19_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash T19_forward.paired.headclipped.fastq.gz T19_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T19 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     T19_forward.paired.headclipped.fastq.gz
[FLASH]     T19_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./T19.extendedFrags.fastq.gz
[FLASH]     ./T19.notCombined_1.fastq.gz
[FLASH]     ./T19.notCombined_2.fastq.gz
[FLASH]     ./T19.hist
[FLASH]     ./T19.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 29 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      29
[FLASH]     Combined pairs:   18
[FLASH]     Uncombined pairs: 11
[FLASH]     Percent combined: 62.07%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

18 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/T19/T19_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T19_centroids.fasta --uc T19.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 18 sequences and identified 16 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: T21 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T21_1.fastq /home/working/demultiplexed/T21_2.fastq T21_forward.paired.fastq.gz T21_forward.singletons.fastq.gz T21_reverse.paired.fastq.gz T21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T21_1.fastq /home/working/demultiplexed/T21_2.fastq T21_forward.paired.fastq.gz T21_forward.singletons.fastq.gz T21_reverse.paired.fastq.gz T21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:29:20 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/T21/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 8196 Both Surviving: 288 (3.51%) Forward Only Surviving: 162 (1.98%) Reverse Only Surviving: 140 (1.71%) Dropped: 7606 (92.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: T21_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: T21_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: T21_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: T21_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash T21_forward.paired.headclipped.fastq.gz T21_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T21 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     T21_forward.paired.headclipped.fastq.gz
[FLASH]     T21_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./T21.extendedFrags.fastq.gz
[FLASH]     ./T21.notCombined_1.fastq.gz
[FLASH]     ./T21.notCombined_2.fastq.gz
[FLASH]     ./T21.hist
[FLASH]     ./T21.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 288 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      288
[FLASH]     Combined pairs:   28
[FLASH]     Uncombined pairs: 260
[FLASH]     Percent combined: 9.72%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

28 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/T21/T21_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T21_centroids.fasta --uc T21.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 28 sequences and identified 24 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: T32 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T32_1.fastq /home/working/demultiplexed/T32_2.fastq T32_forward.paired.fastq.gz T32_forward.singletons.fastq.gz T32_reverse.paired.fastq.gz T32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T32_1.fastq /home/working/demultiplexed/T32_2.fastq T32_forward.paired.fastq.gz T32_forward.singletons.fastq.gz T32_reverse.paired.fastq.gz T32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:29:21 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/T32/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 13172 Both Surviving: 1614 (12.25%) Forward Only Surviving: 386 (2.93%) Reverse Only Surviving: 163 (1.24%) Dropped: 11009 (83.58%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: T32_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: T32_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: T32_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: T32_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash T32_forward.paired.headclipped.fastq.gz T32_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T32 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     T32_forward.paired.headclipped.fastq.gz
[FLASH]     T32_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./T32.extendedFrags.fastq.gz
[FLASH]     ./T32.notCombined_1.fastq.gz
[FLASH]     ./T32.notCombined_2.fastq.gz
[FLASH]     ./T32.hist
[FLASH]     ./T32.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1614 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1614
[FLASH]     Combined pairs:   285
[FLASH]     Uncombined pairs: 1329
[FLASH]     Percent combined: 17.66%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.013 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

285 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/T32/T32_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T32_centroids.fasta --uc T32.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 285 sequences and identified 187 clusters (clustering threshold 1.00) - 3 clusters (representing 87 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: T49 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T49_1.fastq /home/working/demultiplexed/T49_2.fastq T49_forward.paired.fastq.gz T49_forward.singletons.fastq.gz T49_reverse.paired.fastq.gz T49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T49_1.fastq /home/working/demultiplexed/T49_2.fastq T49_forward.paired.fastq.gz T49_forward.singletons.fastq.gz T49_reverse.paired.fastq.gz T49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:29:24 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/T49/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9868 Both Surviving: 421 (4.27%) Forward Only Surviving: 275 (2.79%) Reverse Only Surviving: 113 (1.15%) Dropped: 9059 (91.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: T49_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: T49_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: T49_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: T49_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash T49_forward.paired.headclipped.fastq.gz T49_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T49 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     T49_forward.paired.headclipped.fastq.gz
[FLASH]     T49_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./T49.extendedFrags.fastq.gz
[FLASH]     ./T49.notCombined_1.fastq.gz
[FLASH]     ./T49.notCombined_2.fastq.gz
[FLASH]     ./T49.hist
[FLASH]     ./T49.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 421 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      421
[FLASH]     Combined pairs:   339
[FLASH]     Uncombined pairs: 82
[FLASH]     Percent combined: 80.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.009 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

339 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/T49/T49_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T49_centroids.fasta --uc T49.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 339 sequences and identified 210 clusters (clustering threshold 1.00) - 10 clusters (representing 127 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: T68 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T68_1.fastq /home/working/demultiplexed/T68_2.fastq T68_forward.paired.fastq.gz T68_forward.singletons.fastq.gz T68_reverse.paired.fastq.gz T68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T68_1.fastq /home/working/demultiplexed/T68_2.fastq T68_forward.paired.fastq.gz T68_forward.singletons.fastq.gz T68_reverse.paired.fastq.gz T68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:29:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/T68/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23250 Both Surviving: 3693 (15.88%) Forward Only Surviving: 778 (3.35%) Reverse Only Surviving: 388 (1.67%) Dropped: 18391 (79.10%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: T68_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: T68_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: T68_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: T68_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash T68_forward.paired.headclipped.fastq.gz T68_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T68 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     T68_forward.paired.headclipped.fastq.gz
[FLASH]     T68_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./T68.extendedFrags.fastq.gz
[FLASH]     ./T68.notCombined_1.fastq.gz
[FLASH]     ./T68.notCombined_2.fastq.gz
[FLASH]     ./T68.hist
[FLASH]     ./T68.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3693 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3693
[FLASH]     Combined pairs:   151
[FLASH]     Uncombined pairs: 3542
[FLASH]     Percent combined: 4.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.022 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

151 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/T68/T68_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T68_centroids.fasta --uc T68.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 151 sequences and identified 99 clusters (clustering threshold 1.00) - 2 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: T78 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T78_1.fastq /home/working/demultiplexed/T78_2.fastq T78_forward.paired.fastq.gz T78_forward.singletons.fastq.gz T78_reverse.paired.fastq.gz T78_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T78_1.fastq /home/working/demultiplexed/T78_2.fastq T78_forward.paired.fastq.gz T78_forward.singletons.fastq.gz T78_reverse.paired.fastq.gz T78_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Oct 19, 2016 5:29:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/T78/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23555 Both Surviving: 6269 (26.61%) Forward Only Surviving: 1325 (5.63%) Reverse Only Surviving: 408 (1.73%) Dropped: 15553 (66.03%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: T78_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: T78_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: T78_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: T78_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash T78_forward.paired.headclipped.fastq.gz T78_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T78 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     T78_forward.paired.headclipped.fastq.gz
[FLASH]     T78_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./T78.extendedFrags.fastq.gz
[FLASH]     ./T78.notCombined_1.fastq.gz
[FLASH]     ./T78.notCombined_2.fastq.gz
[FLASH]     ./T78.hist
[FLASH]     ./T78.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6269 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6269
[FLASH]     Combined pairs:   2603
[FLASH]     Uncombined pairs: 3666
[FLASH]     Percent combined: 41.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.082 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2603 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/T78/T78_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T78_centroids.fasta --uc T78.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2603 sequences and identified 1486 clusters (clustering threshold 1.00) - 60 clusters (representing 972 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


### GLOBAL CLUSTERING ###


Concatenating for global clustering


Performing global clustering

vsearch --cluster_fast /home/working/GLOBAL/global_queries.fasta --id 1.00 --strand both --threads 1 --centroids global_centroids.fasta --uc global.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


writing denovo OTU table


Please select a method for taxonomic assignment:
--blast
--pplace
--kraken


##### DONE! #####


Wed Oct 19 17:29:49 2016

</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Trimming, pairing and clustering (within samples) has been successful. Next step is to try <strong>identification using BLAST</strong> and <strong>clustering across samples</strong>.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="BLAST">BLAST<a class="anchor-link" href="#BLAST">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Produce new querymap based on the one from the previous runs. Specify the clustering results from the last run as input.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[7]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

<span class="k">for </span>sample in <span class="k">$(</span>cat Querymap_global.txt | cut -f 1<span class="k">)</span>
<span class="k">do</span>
<span class="k">    </span><span class="nv">fasta</span><span class="o">=</span><span class="k">$(</span>ls -1 <span class="nv">$sample</span>/<span class="nv">$sample</span><span class="se">\_</span>trimmed.fasta<span class="k">)</span>
    <span class="nb">echo</span> -e <span class="s2">&quot;$sample\tfasta\t$fasta&quot;</span>
<span class="k">done</span> &gt; Querymap.txt

head Querymap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>SA177	fasta	SA177/SA177_trimmed.fasta
SB140	fasta	SB140/SB140_trimmed.fasta
T19	fasta	T19/T19_trimmed.fasta
Lg45	fasta	Lg45/Lg45_trimmed.fasta
Q69	fasta	Q69/Q69_trimmed.fasta
S72	fasta	S72/S72_trimmed.fasta
SC66	fasta	SC66/SC66_trimmed.fasta
S31	fasta	S31/S31_trimmed.fasta
SC74	fasta	SC74/SC74_trimmed.fasta
SA68	fasta	SA68/SA68_trimmed.fasta
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="Blast-against-Genbank">Blast against Genbank<a class="anchor-link" href="#Blast-against-Genbank">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Now try doing a BLAST against the entirety of Genbank, using a recently downloaded local version</p>

</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>N.b. sometimes this will fail due to a dodgy GI/TaxID combination. In this case you need to: (1) figure out the GI of the search that failed; (2) go to NCBI and find out the TaxID for that GI; (3) create a file called gi_to_taxid.csv; (4) write the first line of that file as gi,taxid. If this still fails you may not have chosen the right GI, or there may be several dodgy ones.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[13]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-Q Querymap.txt <span class="se">\</span>
--cluster <span class="se">\</span>
--clust_match 1 <span class="se">\</span>
--clust_cov 5 <span class="se">\</span>
--blast <span class="se">\</span>
--blast_db Final/nt/nt <span class="se">\</span>
--min_ident 0.95 <span class="se">\</span>
-n 5 <span class="se">\</span>
-v <span class="se">\</span>
-@ callumjmacgregor@gmail.com <span class="se">\</span>
-o MothPollenGenbankRestart &gt; log
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>We want to inspect the outputs of this on a case by case basis. The file we need for this is a .xml file and it would be easier to have it as a .txt file, so we partially rerun the above query to convert it.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[14]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash
blastn -query /home/working/GLOBAL/global_centroids.fasta <span class="se">\</span>
-db /home/working/Final/nt/nt <span class="se">\</span>
-out GLOBAL/BLAST_0.95/global_blastn.out.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="Blast-against-reference-database">Blast against reference database<a class="anchor-link" href="#Blast-against-reference-database">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Now we want to download our curated reference list (with help from inspecting the outputs of the above, and a list of East Yorkshire flora from Dick Middleton).</p>
<p>The list of binomial species names was prepared in a simple text file: FinalRefList.txt.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[15]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>head FinalRefList.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>









</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[11]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>fetch_from_db.py -t FinalRefList.txt -m rbcl -o eyorks_flora_curated -@ callumjmacgregor@gmail.com
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>
metaBEAT may be querying NCBI&apos;s Entrez databases to fetch/verify taxonomic ids. Entrez User requirements state that you need to identify yourself by providing an email address so that NCBI can contact you in case there is a problem.

You have specified: &apos;callumjmacgregor@gmail.com&apos;


QUERYING GENBANK

check for synonyms for &quot;rbcl&quot; (this is relevant only for Genbank searches)

fetching accessions ..

#	Abies alba	8
#	Abies cephalonica	2
#	Abies cilicica	1
#	Abies concolor	6
#	Abies delavayi	6
#	Abies firma	6
#	Abies grandis	4
#	Abies homolepis	4
#	Abies lasiocarpa	5
#	Abies nordmanniana	4
#	Abies pinsapo	5
#	Abies procera	1
#	Abies veitchii	4
#	Acaena novae-zelandiae	3
#	Acer campestre	17
#	Acer platanoides	6
#	Acer pseudoplatanus	16
#	Achillea alpina	0
#	Achillea distans	0
#	Achillea ligustica	0
#	Achillea millefolium	39
#	Achillea ptarmica	4
#	Aconitum napellus	5
#	Acorus calamus	33
#	Actaea spicata	5
#	Adonis annua	2
#	Adoxa moschatellina	7
#	Aegopodium podagraria	7
#	Aesculus carnea	0
#	Aesculus hippocastanum	11
#	Aethusa cynapium	5
#	Agrimonia eupatoria	5
#	Agrimonia procera	5
#	Agrostemma githago	7
#	Agrostis canina	3
#	Agrostis capillaris	5
#	Agrostis castellana	0
#	Agrostis gigantea	11
#	Agrostis scabra	6
#	Agrostis stolonifera	20
#	Agrostis vinealis	3
#	Aira caryophyllea	6
#	Aira praecox	6
#	Ajuga reptans	10
#	Alcea rosea	2
#	Alchemilla glabra	5
#	Alchemilla mollis	2
#	Alchemilla vulgaris	2
#	Alchemilla xanthochlora	5
#	Alisma lanceolatum	4
#	Alisma plantago-aquatica	17
#	Alliaria petiolata	21
#	Allium carinatum	1
#	Allium moly	3
#	Allium neapolitanum	9
#	Allium oleraceum	3
#	Allium schoenoprasum	3
#	Allium scorodoprasum	2
#	Allium triquetrum	6
#	Allium ursinum	10
#	Allium vineale	3
#	Alnus cordata	1
#	Alnus glutinosa	17
#	Alnus incana	11
#	Alopecurus aequalis	16
#	Alopecurus geniculatus	4
#	Alopecurus myosuroides	4
#	Alopecurus pratensis	9
#	Alyssum alyssoides	0
#	Amaranthus albus	5
#	Amaranthus hybridus	18
#	Amaranthus retroflexus	16
#	Amaranthus thunbergii	0
#	Ambrosia artemisiifolia	19
#	Ammi majus	1
#	Ammi visnaga	1
#	Amsinckia lycopsoides	0
#	Amsinckia micrantha	1
#	Anacamptis morio	1
#	Anacamptis pyramidalis	2
#	Anagallis arvensis	9
#	Anagallis tenella	5
#	Anaphalis margaritacea	7
#	Anchusa arvensis	4
#	Andromeda polifolia	15
#	Anemone apennina	0
#	Anemone nemorosa	6
#	Anemone ranunculoides	1
#	Anethum graveolens	6
#	Angelica sylvestris	4
#	Anisantha diandra	0
#	Anisantha madritensis	0
#	Anisantha rigida	7
#	Anisantha rubens	0
#	Anisantha sterilis	7
#	Anisantha tectorum	0
#	Antennaria dioica	4
#	Anthemis arvensis	2
#	Anthemis cotula	2
#	Anthemis tinctoria	1
#	Anthoxanthum aristatum	0
#	Anthoxanthum odoratum	14
#	Anthriscus caucalis	2
#	Anthriscus sylvestris	9
#	Anthyllis vulneraria	8
#	Antirrhinum majus	7
#	Apera interrupta	1
#	Apera spica-venti	1
#	Aphanes arvensis	4
#	Aphanes australis	2
#	Apium graveolens	10
#	Apium inundatum	2
#	Apium nodiflorum	6
#	Apium repens	0
#	Aquilegia vulgaris	7
#	Arabidopsis thaliana	187
#	Arabis caucasica	1
#	Arabis hirsuta	7
#	Araucaria araucana	5
#	Arbutus unedo	3
#	Arctium lappa	11
#	Arctium minus	15
#	Arctium nemorosum	0
#	Arenaria leptoclados	0
#	Arenaria serpyllifolia	15
#	Armeria maritima	6
#	Armoracia rusticana	5
#	Arnoseris minima	0
#	Arrhenatherum elatius	11
#	Artemisia absinthium	8
#	Artemisia campestris	3
#	Artemisia maritima	2
#	Artemisia verlotiorum	0
#	Artemisia vulgaris	21
#	Arum italicum	9
#	Arum maculatum	7
#	Asparagus officinalis	17
#	Asperugo procumbens	1
#	Asperula arvensis	0
#	Asperula cynanchica	6
#	Asplenium adiantum-nigrum	15
#	Asplenium ceterach	9
#	Asplenium marinum	2
#	Asplenium ruta-muraria	23
#	Asplenium scolopendrium	8
#	Asplenium trichomanes	20
#	Aster lanceolatus	7
#	Aster novi-belgii	1
#	Aster tripolium	3
#	Astragalus danicus	1
#	Astragalus glycyphyllos	3
#	Astrantia major	3
#	Athyrium filix-femina	17
#	Atriplex littoralis	4
#	Atriplex patula	13
#	Atriplex prostrata	5
#	Atropa belladonna	13
#	Aubrieta deltoidea	1
#	Avena fatua	15
#	Avena sativa	10
#	Avenula pratensis	2
#	Avenula pubescens	5
#	Azolla filiculoides	6
#	Baldellia ranunculoides	4
#	Ballota nigra	6
#	Barbarea intermedia	1
#	Barbarea stricta	0
#	Barbarea verna	4
#	Barbarea vulgaris	16
#	Bassia scoparia	4
#	Bellis perennis	12
#	Berberis darwinii	0
#	Berberis vulgaris	3
#	Berteroa incana	3
#	Berula erecta	5
#	Beta trigyna	1
#	Beta vulgaris	121
#	Betonica officinalis	8
#	Betula pendula	18
#	Betula pubescens	8
#	Bidens cernua	12
#	Bidens tripartita	7
#	Blackstonia perfoliata	5
#	Blechnum spicant	4
#	Blysmus compressus	8
#	Bolboschoenus maritimus	7
#	Borago officinalis	9
#	Botrychium lunaria	11
#	Brachypodium pinnatum	7
#	Brachypodium rupestre	3
#	Brachypodium sylvaticum	9
#	Brassica juncea	16
#	Brassica napus	290
#	Brassica nigra	11
#	Brassica oleracea	102
#	Brassica rapa	82
#	Briza maxima	5
#	Briza media	7
#	Bromopsis benekenii	2
#	Bromopsis erecta	9
#	Bromopsis inermis	22
#	Bromopsis ramosa	3
#	Bromus arvensis	1
#	Bromus commutatus	4
#	Bromus hordeaceus	8
#	Bromus lanceolatus	0
#	Bromus lepidus	0
#	Bromus racemosus	3
#	Bromus secalinus	2
#	Bryonia dioica	5
#	Buddleja davidii	6
#	Buddleja globosa	0
#	Buddleja officinalis	1
#	Bunias orientalis	1
#	Bupleurum baldense	0
#	Bupleurum rotundifolium	0
#	Bupleurum subovatum	0
#	Bupleurum tenuissimum	2
#	Butomus umbellatus	7
#	Buxus sempervirens	13
#	Cakile maritima	4
#	Calamagrostis canescens	5
#	Calamagrostis epigejos	13
#	Calamagrostis gracilescens	0
#	Calendula officinalis	5
#	Calla palustris	5
#	Callitriche hermaphroditica	6
#	Callitriche obtusangula	3
#	Callitriche platycarpa	5
#	Callitriche stagnalis	7
#	Calluna vulgaris	11
#	Caltha palustris	32
#	Calystegia pulchra	2
#	Calystegia sepium	18
#	Calystegia silvatica	4
#	Calystegia soldanella	6
#	Camelina sativa	156
#	Campanula glomerata	11
#	Campanula latifolia	6
#	Campanula medium	2
#	Campanula patula	10
#	Campanula persicifolia	2
#	Campanula portenschlagiana	2
#	Campanula poscharskyana	1
#	Campanula rapunculoides	10
#	Campanula rapunculus	5
#	Campanula rotundifolia	22
#	Campanula trachelium	9
#	Cannabis sativa	26
#	Capsella bursa-pastoris	24
#	Capsicum annuum	137
#	Cardamine amara	2
#	Cardamine flexuosa	5
#	Cardamine hirsuta	13
#	Cardamine impatiens	10
#	Cardamine pratensis	24
#	Carduus crispus	3
#	Carduus nutans	6
#	Carduus pycnocephalus	0
#	Carduus tenuiflorus	4
#	Carex acuta	3
#	Carex acutiformis	3
#	Carex appropinquata	0
#	Carex arenaria	5
#	Carex binervis	3
#	Carex canescens	8
#	Carex caryophyllea	4
#	Carex demissa	0
#	Carex diandra	5
#	Carex dioica	4
#	Carex distans	4
#	Carex disticha	3
#	Carex divisa	3
#	Carex divulsa	5
#	Carex echinata	3
#	Carex elata	3
#	Carex elongata	2
#	Carex extensa	5
#	Carex flacca	8
#	Carex hirta	4
#	Carex hostiana	5
#	Carex lasiocarpa	4
#	Carex lepidocarpa	0
#	Carex leporina	0
#	Carex muricata	4
#	Carex nigra	7
#	Carex oederi	0
#	Carex otrubae	5
#	Carex pallescens	5
#	Carex panicea	7
#	Carex paniculata	4
#	Carex pendula	9
#	Carex pilulifera	5
#	Carex pseudocyperus	6
#	Carex pulicaris	5
#	Carex remota	4
#	Carex riparia	3
#	Carex rostrata	4
#	Carex spicata	4
#	Carex sylvatica	5
#	Carex vesicaria	3
#	Carex viridula	4
#	Carlina vulgaris	5
#	Carpinus betulus	8
#	Carthamus tinctorius	18
#	Carum carvi	6
#	Castanea sativa	12
#	Catabrosa aquatica	6
#	Catapodium marinum	3
#	Catapodium rigidum	6
#	Cedrus atlantica	6
#	Cedrus deodara	17
#	Cedrus libani	4
#	Centaurea aspera	0
#	Centaurea calcitrapa	0
#	Centaurea cyanus	5
#	Centaurea diluta	0
#	Centaurea montana	1
#	Centaurea nigra	6
#	Centaurea scabiosa	5
#	Centaurea solstitialis	0
#	Centaurium erythraea	7
#	Centaurium littorale	1
#	Centaurium pulchellum	5
#	Centranthus ruber	7
#	Centunculus minimus	0
#	Cephalanthera damasonium	1
#	Cephalaria gigantea	1
#	Cerastium arvense	7
#	Cerastium diffusum	4
#	Cerastium fontanum	15
#	Cerastium glomeratum	6
#	Cerastium semidecandrum	2
#	Cerastium tomentosum	2
#	Ceratocapnos claviculata	4
#	Ceratochloa carinata	2
#	Ceratochloa cathartica	12
#	Ceratophyllum demersum	15
#	Ceratophyllum submersum	4
#	Chaenorhinum minus	7
#	Chaerophyllum temulum	9
#	Chamaecyparis lawsoniana	8
#	Chamaecyparis obtusa	7
#	Chamaecyparis pisifera	7
#	Chamaemelum nobile	5
#	Chamerion angustifolium	29
#	Chara vulgaris	8
#	Chelidonium majus	14
#	Chenopodium album	46
#	Chenopodium bonus-henricus	6
#	Chenopodium ficifolium	5
#	Chenopodium glaucum	7
#	Chenopodium hybridum	0
#	Chenopodium opulifolium	2
#	Chenopodium polyspermum	4
#	Chenopodium pratericola	0
#	Chenopodium quinoa	13
#	Chenopodium rubrum	4
#	Chenopodium urbicum	4
#	Chenopodium vulvaria	4
#	Chrysosplenium alternifolium	4
#	Chrysosplenium oppositifolium	4
#	Cicerbita macrophylla	3
#	Cichorium intybus	17
#	Cicuta virosa	7
#	Circaea lutetiana	15
#	Cirsium acaule	4
#	Cirsium arvense	14
#	Cirsium dissectum	3
#	Cirsium eriophorum	3
#	Cirsium palustre	8
#	Cirsium vulgare	18
#	Cladium mariscus	7
#	Claytonia perfoliata	1
#	Claytonia sibirica	2
#	Clematis montana	2
#	Clematis tangutica	1
#	Clematis vitalba	8
#	Clinopodium acinos	5
#	Clinopodium ascendens	1
#	Clinopodium vulgare	17
#	Cochlearia anglica	4
#	Cochlearia danica	4
#	Cochlearia officinalis	3
#	Coeloglossum viride	6
#	Coincya monensis	2
#	Colchicum autumnale	14
#	Comarum palustre	8
#	Conium maculatum	8
#	Conopodium majus	5
#	Conringia orientalis	0
#	Consolida ajacis	3
#	Convallaria majalis	15
#	Convolvulus arvensis	20
#	Conyza canadensis	9
#	Conyza floribunda	0
#	Coriandrum sativum	7
#	Cornus alba	1
#	Cornus sanguinea	11
#	Coronilla scorpioides	0
#	Coronilla valentina	0
#	Corrigiola litoralis	1
#	Cortaderia selloana	3
#	Corylus avellana	22
#	Cotoneaster dielsianus	3
#	Cotoneaster horizontalis	5
#	Cotoneaster integrifolius	0
#	Cotoneaster lacteus	2
#	Cotoneaster rehderi	2
#	Cotoneaster simonsii	1
#	Cotoneaster splendens	0
#	Crambe hispanica	0
#	Crambe maritima	3
#	Crassula helmsii	4
#	Crataegus laevigata	9
#	Crataegus monogyna	36
#	Crepis biennis	2
#	Crepis capillaris	6
#	Crepis nicaeensis	0
#	Crepis paludosa	3
#	Crepis vesicaria	2
#	Crocus vernus	2
#	Cruciata laevipes	6
#	Cupressus duclouxiana	8
#	Cupressus goveniana	4
#	Cupressus macrocarpa	8
#	Cupressus sempervirens	10
#	Cuscuta campestris	5
#	Cuscuta epithymum	7
#	Cuscuta europaea	5
#	Cuscuta suaveolens	2
#	Cymbalaria muralis	2
#	Cymbalaria pallida	0
#	Cynoglossum officinale	7
#	Cynosurus cristatus	8
#	Cynosurus echinatus	2
#	Cyperus eragrostis	4
#	Cyperus longus	7
#	Cyrtomium falcatum	13
#	Cyrtomium fortunei	21
#	Cystopteris fragilis	19
#	Cytisus scoparius	8
#	Dactylis glomerata	32
#	Dactylorhiza fuchsii	4
#	Dactylorhiza incarnata	4
#	Dactylorhiza maculata	5
#	Dactylorhiza praetermissa	3
#	Dactylorhiza purpurella	4
#	Damasonium alisma	1
#	Danthonia decumbens	6
#	Daphne laureola	4
#	Datura stramonium	31
#	Daucus carota	42
#	Deschampsia cespitosa	53
#	Deschampsia flexuosa	5
#	Deschampsia setacea	2
#	Descurainia sophia	5
#	Dianthus barbatus	1
#	Dianthus deltoides	8
#	Digitalis purpurea	14
#	Digitaria sanguinalis	9
#	Diplotaxis muralis	1
#	Diplotaxis tenuifolia	9
#	Dipsacus fullonum	13
#	Dipsacus pilosus	4
#	Dipsacus sativus	1
#	Doronicum pardalianches	1
#	Drosera intermedia	1
#	Drosera rotundifolia	10
#	Dryopteris affinis	4
#	Dryopteris borreri	4
#	Dryopteris cambrensis	0
#	Dryopteris carthusiana	22
#	Dryopteris cristata	20
#	Dryopteris dilatata	11
#	Dryopteris filix-mas	15
#	Dysphania pumilio	0
#	Echinochloa colona	9
#	Echinochloa crus-galli	33
#	Echium plantagineum	3
#	Echium vulgare	5
#	Eleocharis acicularis	11
#	Eleocharis multicaulis	3
#	Eleocharis palustris	13
#	Eleocharis quinqueflora	5
#	Eleocharis uniglumis	4
#	Eleogiton fluitans	9
#	Eleusine indica	18
#	Elodea canadensis	7
#	Elodea nuttallii	8
#	Elymus caninus	9
#	Elytrigia atherica	2
#	Elytrigia juncea	3
#	Elytrigia repens	25
#	Ensete ventricosum	4
#	Epilobium ciliatum	4
#	Epilobium hirsutum	8
#	Epilobium komarovianum	0
#	Epilobium montanum	7
#	Epilobium obscurum	3
#	Epilobium palustre	18
#	Epilobium parviflorum	13
#	Epilobium roseum	3
#	Epilobium tetragonum	6
#	Epipactis dunensis	1
#	Epipactis helleborine	21
#	Epipactis leptochila	3
#	Epipactis palustris	5
#	Epipactis phyllanthes	2
#	Equisetum arvense	44
#	Equisetum fluviatile	10
#	Equisetum hyemale	23
#	Equisetum palustre	7
#	Equisetum telmateia	3
#	Eragrostis cilianensis	2
#	Eragrostis pectinacea	13
#	Eranthis hyemalis	2
#	Erica cinerea	5
#	Erica tetralix	5
#	Erigeron acris	4
#	Erigeron karvinskianus	2
#	Eriophorum angustifolium	23
#	Eriophorum latifolium	4
#	Eriophorum vaginatum	19
#	Erodium cicutarium	6
#	Erodium moschatum	5
#	Erophila verna	10
#	Eryngium maritimum	6
#	Erysimum cheiranthoides	11
#	Erysimum cheiri	3
#	Escallonia macrantha	1
#	Eschscholzia californica	7
#	Euonymus europaeus	8
#	Eupatorium cannabinum	3
#	Euphorbia cyparissias	13
#	Euphorbia esula	8
#	Euphorbia exigua	6
#	Euphorbia helioscopia	10
#	Euphorbia lathyris	5
#	Euphorbia peplus	13
#	Euphorbia platyphyllos	3
#	Euphrasia micrantha	3
#	Euphrasia nemorosa	3
#	Euphrasia officinalis	2
#	Fagopyrum esculentum	12
#	Fagus sylvatica	14
#	Falcaria vulgaris	1
#	Fallopia baldschuanica	2
#	Fallopia convolvulus	13
#	Fallopia japonica	114
#	Fallopia sachalinensis	191
#	Festuca arenaria	3
#	Festuca brevipila	0
#	Festuca filiformis	2
#	Festuca ovina	10
#	Festuca rubra	52
#	Ficaria verna	6
#	Ficus carica	12
#	Filago lutescens	0
#	Filago minima	2
#	Filago pyramidata	1
#	Filago vulgaris	3
#	Filipendula ulmaria	9
#	Filipendula vulgaris	8
#	Foeniculum vulgare	17
#	Fragaria ananassa	5
#	Fragaria moschata	2
#	Fragaria vesca	69
#	Frangula alnus	12
#	Fraxinus excelsior	21
#	Fritillaria meleagris	4
#	Fumaria densiflora	2
#	Fumaria muralis	5
#	Fumaria officinalis	4
#	Fumaria parviflora	0
#	Fumaria purpurea	2
#	Fumaria vaillantii	0
#	Gagea bohemica	3
#	Gagea lutea	7
#	Galanthus nivalis	2
#	Galanthus plicatus	1
#	Galanthus woronowii	0
#	Galega officinalis	2
#	Galeopsis angustifolia	3
#	Galeopsis bifida	5
#	Galeopsis speciosa	3
#	Galeopsis tetrahit	8
#	Galinsoga parviflora	3
#	Galinsoga quadriradiata	11
#	Galium aparine	25
#	Galium odoratum	5
#	Galium palustre	14
#	Galium saxatile	5
#	Galium uliginosum	3
#	Galium verum	14
#	Gentiana pneumonanthe	2
#	Gentianella amarella	7
#	Gentianella campestris	4
#	Geranium columbinum	4
#	Geranium dissectum	7
#	Geranium endressii	3
#	Geranium lucidum	4
#	Geranium molle	9
#	Geranium nodosum	3
#	Geranium phaeum	6
#	Geranium pratense	4
#	Geranium pusillum	3
#	Geranium pyrenaicum	2
#	Geranium robertianum	13
#	Geranium rotundifolium	6
#	Geranium sanguineum	10
#	Geranium sylvaticum	6
#	Geranium versicolor	0
#	Geum rivale	5
#	Geum urbanum	8
#	Ginkgo biloba	30
#	Gladiolus communis	1
#	Glaucium flavum	6
#	Glaux maritima	5
#	Glebionis segetum	6
#	Glechoma hederacea	19
#	Glyceria declinata	5
#	Glyceria fluitans	7
#	Glyceria maxima	3
#	Glyceria notata	5
#	Glycine max	159
#	Gnaphalium luteoalbum	3
#	Gnaphalium sylvaticum	1
#	Gnaphalium uliginosum	6
#	Goodyera repens	9
#	Groenlandia densa	6
#	Guizotia abyssinica	3
#	Gymnadenia conopsea	9
#	Gymnadenia densiflora	0
#	Gymnocarpium dryopteris	17
#	Gymnocarpium robertianum	3
#	Hedera helix	26
#	Hedera hibernica	2
#	Helianthemum nummularium	9
#	Helianthus annuus	14
#	Helianthus decapetalus	7
#	Helianthus tuberosus	11
#	Helleborus foetidus	3
#	Helleborus viridis	4
#	Helminthotheca echioides	6
#	Helosciadium nodiflorum	6
#	Hemizonia pungens	0
#	Hepatica nobilis	8
#	Heracleum mantegazzianum	0
#	Heracleum sphondylium	6
#	Herniaria glabra	2
#	Hesperis matronalis	7
#	Hieracium acuminatum	0
#	Hieracium aggregatum	0
#	Hieracium calcaricola	0
#	Hieracium decolor	0
#	Hieracium diaphanoides	0
#	Hieracium diaphanum	0
#	Hieracium grandidens	0
#	Hieracium lepidulum	1
#	Hieracium sabaudum	0
#	Hieracium umbellatum	6
#	Hieracium vagum	0
#	Hieracium vulgatum	0
#	Hippocrepis comosa	7
#	Hippophae rhamnoides	14
#	Hippuris vulgaris	21
#	Hirschfeldia incana	2
#	Holcus lanatus	9
#	Holcus mollis	3
#	Honckenya peploides	21
#	Hordeum distichon	0
#	Hordeum jubatum	13
#	Hordeum marinum	22
#	Hordeum murinum	19
#	Hordeum secalinum	7
#	Hordeum vulgare	34
#	Hottonia palustris	3
#	Humulus lupulus	20
#	Hyacinthoides hispanica	28
#	Hyacinthoides non-scripta	15
#	Hydrangea anomala	5
#	Hydrangea arborescens	9
#	Hydrangea aspera	2
#	Hydrocharis morsus-ranae	4
#	Hydrocotyle ranunculoides	1
#	Hydrocotyle vulgaris	5
#	Hyoscyamus niger	7
#	Hypericum androsaemum	5
#	Hypericum calycinum	1
#	Hypericum elodes	7
#	Hypericum hirsutum	3
#	Hypericum humifusum	4
#	Hypericum montanum	2
#	Hypericum perforatum	23
#	Hypericum pulchrum	4
#	Hypericum tetrapterum	3
#	Hypochaeris glabra	5
#	Hypochaeris maculata	1
#	Hypochaeris radicata	8
#	Hypopitys monotropa	1
#	Iberis amara	3
#	Ilex aquifolium	19
#	Impatiens capensis	17
#	Impatiens glandulifera	2
#	Impatiens noli-tangere	5
#	Impatiens parviflora	2
#	Inula conyzae	3
#	Inula helenium	11
#	Iris foetidissima	3
#	Iris germanica	6
#	Iris pseudacorus	12
#	Isoetes lacustris	2
#	Isolepis setacea	11
#	Iva xanthiifolia	0
#	Jacobaea aquatica	5
#	Jacobaea vulgaris	7
#	Jasione montana	6
#	Juglans nigra	10
#	Juglans regia	8
#	Juncus acutiflorus	3
#	Juncus articulatus	8
#	Juncus bufonius	12
#	Juncus bulbosus	7
#	Juncus compressus	5
#	Juncus conglomeratus	6
#	Juncus effusus	20
#	Juncus gerardii	6
#	Juncus inflexus	5
#	Juncus maritimus	7
#	Juncus ranarius	1
#	Juncus squarrosus	5
#	Juncus subnodulosus	3
#	Juncus tenuis	7
#	Kickxia elatine	4
#	Knautia arvensis	8
#	Koeleria macrantha	7
#	Laburnum anagyroides	5
#	Lactuca sativa	12
#	Lactuca serriola	14
#	Lactuca virosa	3
#	Lagarosiphon major	14
#	Lamiastrum galeobdolon	7
#	Lamium album	9
#	Lamium amplexicaule	10
#	Lamium confertum	4
#	Lamium hybridum	3
#	Lamium maculatum	5
#	Lamium purpureum	14
#	Lappula squarrosa	2
#	Lapsana communis	17
#	Larix decidua	9
#	Larix kaempferi	11
#	Lathraea clandestina	12
#	Lathraea squamaria	5
#	Lathyrus aphaca	14
#	Lathyrus grandiflorus	0
#	Lathyrus hirsutus	2
#	Lathyrus latifolius	3
#	Lathyrus linifolius	3
#	Lathyrus nissolia	2
#	Lathyrus palustris	6
#	Lathyrus pratensis	12
#	Lathyrus sativus	4
#	Lathyrus sylvestris	10
#	Lathyrus tuberosus	11
#	Legousia hybrida	3
#	Lemna gibba	15
#	Lemna minor	21
#	Lemna minuta	6
#	Lemna trisulca	11
#	Leontodon hispidus	6
#	Leontodon saxatilis	6
#	Leonurus cardiaca	8
#	Lepidium campestre	12
#	Lepidium coronopus	4
#	Lepidium didymum	2
#	Lepidium draba	3
#	Lepidium heterophyllum	4
#	Lepidium latifolium	3
#	Lepidium ruderale	6
#	Lepidium sativum	1
#	Lepidium virginicum	4
#	Leucanthemum vulgare	17
#	Leucojum aestivum	2
#	Leycesteria formosa	4
#	Leymus arenarius	10
#	Ligularia dentata	1
#	Ligustrum ovalifolium	13
#	Ligustrum vulgare	15
#	Lilium martagon	5
#	Lilium pyrenaicum	2
#	Limnanthes douglasii	1
#	Limonium vulgare	5
#	Limosella aquatica	6
#	Linaria purpurea	1
#	Linaria repens	3
#	Linaria vulgaris	17
#	Linum bienne	7
#	Linum catharticum	5
#	Linum perenne	8
#	Linum usitatissimum	2
#	Lithospermum arvense	4
#	Lithospermum officinale	7
#	Littorella uniflora	4
#	Lobelia erinus	2
#	Lobularia maritima	4
#	Lolium multiflorum	17
#	Lolium perenne	28
#	Lolium rigidum	9
#	Lolium temulentum	8
#	Lonicera caprifolium	0
#	Lonicera involucrata	0
#	Lonicera periclymenum	6
#	Lonicera xylosteum	1
#	Lotus corniculatus	21
#	Lotus pedunculatus	6
#	Lotus tenuis	3
#	Lunaria annua	1
#	Lupinus arboreus	1
#	Lupinus polyphyllus	2
#	Luzula campestris	9
#	Luzula luzuloides	0
#	Luzula multiflora	9
#	Luzula pilosa	6
#	Luzula sylvatica	6
#	Lycium barbarum	6
#	Lycium chinense	10
#	Lycopodiella inundata	2
#	Lycopodium clavatum	13
#	Lycopus europaeus	3
#	Lysimachia nemorum	3
#	Lysimachia nummularia	6
#	Lysimachia punctata	1
#	Lysimachia thyrsiflora	0
#	Lysimachia vulgaris	5
#	Lythrum hyssopifolia	2
#	Lythrum portula	4
#	Lythrum salicaria	16
#	Mahonia aquifolium	2
#	Maianthemum bifolium	15
#	Malus pumila	16
#	Malus sylvestris	5
#	Malva arborea	3
#	Malva moschata	5
#	Malva neglecta	8
#	Malva parviflora	6
#	Malva pseudolavatera	1
#	Malva pusilla	3
#	Malva setigera	0
#	Malva sylvestris	14
#	Marrubium vulgare	8
#	Matricaria chamomilla	34
#	Matricaria discoidea	6
#	Matthiola sinuata	3
#	Meconopsis cambrica	6
#	Medicago arabica	5
#	Medicago lupulina	20
#	Medicago polymorpha	7
#	Medicago sativa	18
#	Melampyrum pratense	7
#	Melica uniflora	9
#	Melilotus albus	15
#	Melilotus altissimus	6
#	Melilotus indicus	11
#	Melilotus officinalis	15
#	Melissa officinalis	10
#	Mentha aquatica	6
#	Mentha arvensis	7
#	Mentha pulegium	10
#	Mentha spicata	16
#	Menyanthes trifoliata	17
#	Mercurialis annua	9
#	Mercurialis perennis	10
#	Metasequoia glyptostroboides	13
#	Milium effusum	8
#	Mimulus guttatus	89
#	Mimulus moschatus	3
#	Minuartia hybrida	3
#	Misopates orontium	4
#	Moehringia trinervia	6
#	Molinia caerulea	9
#	Montia fontana	10
#	Musa acuminata	160
#	Musa balbisiana	29
#	Musa ornata	9
#	Musa velutina	6
#	Muscari armeniacum	1
#	Myagrum perfoliatum	0
#	Mycelis muralis	6
#	Myosotis arvensis	5
#	Myosotis discolor	5
#	Myosotis laxa	8
#	Myosotis ramosissima	4
#	Myosotis scorpioides	4
#	Myosotis secunda	5
#	Myosotis sylvatica	6
#	Myosoton aquaticum	4
#	Myosurus minimus	3
#	Myrica gale	12
#	Myriophyllum alterniflorum	2
#	Myriophyllum aquaticum	10
#	Myriophyllum spicatum	11
#	Myriophyllum verticillatum	8
#	Myrrhis odorata	2
#	Narcissus poeticus	0
#	Narcissus pseudonarcissus	1
#	Nardus stricta	7
#	Narthecium ossifragum	9
#	Nasturtium microphyllum	7
#	Nasturtium officinale	13
#	Neotinea ustulata	1
#	Neottia cordata	2
#	Neottia nidus-avis	7
#	Neottia ovata	7
#	Nepeta cataria	12
#	Neslia paniculata	2
#	Nicandra physalodes	4
#	Nicotiana tabacum	127
#	Nigella damascena	3
#	Nuphar lutea	5
#	Nymphaea alba	12
#	Nymphoides peltata	2
#	Odontites vernus	6
#	Oenanthe aquatica	3
#	Oenanthe crocata	3
#	Oenanthe fistulosa	2
#	Oenanthe fluviatilis	0
#	Oenanthe lachenalii	4
#	Oenanthe pimpinelloides	3
#	Oenanthe silaifolia	0
#	Oenothera biennis	17
#	Oenothera glazioviana	7
#	Omphalodes verna	1
#	Onobrychis viciifolia	3
#	Onoclea sensibilis	18
#	Ononis repens	6
#	Ononis spinosa	7
#	Onopordum acanthium	4
#	Ophioglossum vulgatum	8
#	Ophrys apifera	8
#	Ophrys insectifera	3
#	Orchis mascula	4
#	Orchis simia	1
#	Oreopteris limbosperma	3
#	Origanum vulgare	20
#	Ornithogalum nutans	1
#	Ornithopus perpusillus	5
#	Orobanche elatior	1
#	Orobanche hederae	11
#	Orobanche minor	46
#	Orobanche rapum-genistae	8
#	Orobanche reticulata	0
#	Osmunda regalis	41
#	Oxalis acetosella	18
#	Oxalis articulata	3
#	Oxalis corniculata	13
#	Oxalis exilis	1
#	Oxalis incarnata	1
#	Oxalis pes-caprae	3
#	Oxalis stricta	13
#	Panicum capillare	13
#	Panicum miliaceum	8
#	Papaver argemone	2
#	Papaver dubium	5
#	Papaver hybridum	3
#	Papaver lecoqii	0
#	Papaver rhoeas	15
#	Papaver somniferum	10
#	Parapholis incurva	5
#	Parapholis strigosa	4
#	Parentucellia viscosa	7
#	Parietaria judaica	8
#	Paris quadrifolia	8
#	Parnassia palustris	29
#	Pastinaca sativa	7
#	Pedicularis palustris	3
#	Pedicularis sylvatica	5
#	Pentaglottis sempervirens	3
#	Persicaria amphibia	14
#	Persicaria bistorta	6
#	Persicaria capitata	5
#	Persicaria hydropiper	28
#	Persicaria lapathifolia	17
#	Persicaria maculosa	13
#	Persicaria minor	4
#	Persicaria pensylvanica	10
#	Persicaria wallichii	6
#	Petasites albus	2
#	Petasites fragrans	1
#	Petasites hybridus	5
#	Petroselinum crispum	8
#	Petroselinum segetum	2
#	Phacelia ciliata	0
#	Phacelia parviflora	0
#	Phacelia tanacetifolia	1
#	Phalaris aquatica	0
#	Phalaris arundinacea	26
#	Phalaris canariensis	1
#	Phalaris paradoxa	1
#	Phegopteris connectilis	12
#	Philadelphus coronarius	2
#	Phleum arenarium	3
#	Phleum bertolonii	4
#	Phleum pratense	23
#	Phragmites australis	45
#	Physalis alkekengi	3
#	Phytolacca acinosa	0
#	Picea abies	33
#	Picea sitchensis	10
#	Picris hieracioides	11
#	Pilosella aurantiaca	8
#	Pilosella officinarum	10
#	Pilularia globulifera	5
#	Pimpinella major	1
#	Pimpinella saxifraga	8
#	Pinguicula vulgaris	21
#	Pinus contorta	9
#	Pinus nigra	15
#	Pinus sylvestris	27
#	Pisum sativum	31
#	Plantago arenaria	0
#	Plantago coronopus	12
#	Plantago lanceolata	18
#	Plantago major	24
#	Plantago maritima	9
#	Plantago media	10
#	Platanthera bifolia	2
#	Platanthera chlorantha	8
#	Poa angustifolia	4
#	Poa annua	27
#	Poa compressa	6
#	Poa humilis	5
#	Poa nemoralis	6
#	Poa pratensis	89
#	Poa trivialis	11
#	Polemonium caeruleum	3
#	Polygala calcarea	1
#	Polygala serpyllifolia	7
#	Polygala vulgaris	7
#	Polygonatum multiflorum	6
#	Polygonum arenarium	0
#	Polygonum arenastrum	4
#	Polygonum aviculare	21
#	Polygonum rurivagum	4
#	Polypodium interjectum	3
#	Polypodium vulgare	6
#	Polypogon monspeliensis	4
#	Polystichum aculeatum	2
#	Polystichum setiferum	3
#	Pontederia cordata	4
#	Populus alba	22
#	Populus nigra	164
#	Populus tremula	17
#	Populus trichocarpa	85
#	Portulaca oleracea	17
#	Potamogeton acutifolius	0
#	Potamogeton alpinus	9
#	Potamogeton berchtoldii	4
#	Potamogeton coloratus	3
#	Potamogeton compressus	6
#	Potamogeton crispus	18
#	Potamogeton friesii	2
#	Potamogeton gramineus	11
#	Potamogeton lucens	8
#	Potamogeton natans	16
#	Potamogeton obtusifolius	6
#	Potamogeton pectinatus	13
#	Potamogeton perfoliatus	22
#	Potamogeton polygonifolius	4
#	Potamogeton pusillus	17
#	Potamogeton trichoides	3
#	Potentilla anglica	4
#	Potentilla anserina	8
#	Potentilla argentea	4
#	Potentilla erecta	5
#	Potentilla intermedia	0
#	Potentilla norvegica	4
#	Potentilla recta	4
#	Potentilla reptans	9
#	Potentilla sterilis	5
#	Potentilla tabernaemontani	0
#	Poterium sanguisorba	9
#	Primula auricula	0
#	Primula pulverulenta	3
#	Primula veris	10
#	Primula vulgaris	7
#	Prunella vulgaris	33
#	Prunus avium	11
#	Prunus cerasifera	13
#	Prunus cerasus	11
#	Prunus domestica	17
#	Prunus dulcis	8
#	Prunus laurocerasus	4
#	Prunus padus	15
#	Prunus spinosa	22
#	Pseudofumaria lutea	4
#	Pseudotsuga menziesii	9
#	Pteridium aquilinum	20
#	Pteris cretica	21
#	Pteris incompleta	1
#	Pteris tremula	5
#	Pteris vittata	47
#	Puccinellia distans	11
#	Puccinellia maritima	4
#	Puccinellia rupestris	2
#	Pulicaria dysenterica	4
#	Pulmonaria officinalis	2
#	Pyracantha coccinea	10
#	Pyrola minor	4
#	Pyrus communis	17
#	Pyrus pyraster	2
#	Quercus cerris	4
#	Quercus ilex	63
#	Quercus petraea	11
#	Quercus robur	19
#	Quercus rubra	19
#	Radiola linoides	6
#	Ranunculus acris	18
#	Ranunculus aquatilis	9
#	Ranunculus arvensis	1
#	Ranunculus auricomus	4
#	Ranunculus baudotii	2
#	Ranunculus bulbosus	10
#	Ranunculus circinatus	3
#	Ranunculus flammula	7
#	Ranunculus fluitans	1
#	Ranunculus hederaceus	1
#	Ranunculus lingua	5
#	Ranunculus parviflorus	3
#	Ranunculus peltatus	3
#	Ranunculus penicillatus	2
#	Ranunculus repens	10
#	Ranunculus sardous	3
#	Ranunculus sceleratus	8
#	Ranunculus trichophyllus	5
#	Ranunculus water-crowfoot	0
#	Raphanus raphanistrum	12
#	Raphanus sativus	35
#	Rapistrum rugosum	4
#	Reseda alba	3
#	Reseda lutea	8
#	Reseda luteola	5
#	Rhamnus cathartica	23
#	Rhinanthus angustifolius	0
#	Rhinanthus minor	8
#	Rhododendron ponticum	4
#	Rhus typhina	8
#	Ribes alpinum	5
#	Ribes nigrum	1
#	Ribes rubrum	8
#	Ribes sanguineum	0
#	Ribes uva-crispa	1
#	Ricinus communis	111
#	Robinia pseudoacacia	14
#	Rorippa amphibia	3
#	Rorippa palustris	5
#	Rorippa sylvestris	5
#	Rosa arvensis	5
#	Rosa caesia	5
#	Rosa canina	10
#	Rosa micrantha	4
#	Rosa mollis	2
#	Rosa multiflora	12
#	Rosa obtusifolia	3
#	Rosa rubiginosa	4
#	Rosa rugosa	6
#	Rosa sherardii	4
#	Rosa spinosissima	3
#	Rosmarinus officinalis	12
#	Rubus armeniacus	0
#	Rubus caesius	5
#	Rubus dasyphyllus	0
#	Rubus eboracensis	0
#	Rubus echinatoides	0
#	Rubus fruticosus	4
#	Rubus halsteadensis	0
#	Rubus idaeus	20
#	Rubus nemoralis	0
#	Rubus polyanthemus	0
#	Rubus saxatilis	1
#	Rubus spectabilis	1
#	Rubus sprengelii	0
#	Rubus ulmifolius	4
#	Rubus warrenii	0
#	Rudbeckia laciniata	5
#	Rumex acetosa	20
#	Rumex acetosella	12
#	Rumex conglomeratus	6
#	Rumex crispus	22
#	Rumex dentatus	0
#	Rumex hydrolapathum	5
#	Rumex longifolius	1
#	Rumex maritimus	3
#	Rumex obtusifolius	18
#	Rumex palustris	3
#	Rumex patientia	9
#	Rumex salicifolius	1
#	Rumex sanguineus	4
#	Ruppia cirrhosa	8
#	Ruppia maritima	48
#	Ruscus aculeatus	12
#	Sagina apetala	5
#	Sagina filicaulis	0
#	Sagina maritima	3
#	Sagina nodosa	3
#	Sagina procumbens	4
#	Sagittaria sagittifolia	2
#	Salicornia dolichostachya	4
#	Salicornia europaea	8
#	Salicornia fragilis	1
#	Salicornia ramosissima	3
#	Salix alba	19
#	Salix aurita	6
#	Salix caprea	13
#	Salix cinerea	10
#	Salix daphnoides	3
#	Salix pentandra	10
#	Salix purpurea	12
#	Salix repens	6
#	Salix triandra	11
#	Salix viminalis	8
#	Salsola kali	8
#	Salvia pratensis	8
#	Salvia reflexa	0
#	Salvia verbenaca	5
#	Salvia verticillata	1
#	Sambucus nigra	35
#	Samolus valerandi	6
#	Sanguisorba officinalis	15
#	Sanicula europaea	5
#	Saponaria officinalis	14
#	Saxifraga granulata	2
#	Saxifraga hirsuta	2
#	Saxifraga hypnoides	3
#	Saxifraga tridactylites	3
#	Scabiosa atropurpurea	1
#	Scabiosa columbaria	7
#	Scandix pecten-veneris	3
#	Schedonorus arundinaceus	30
#	Schedonorus giganteus	6
#	Schedonorus pratensis	16
#	Schoenoplectus lacustris	6
#	Schoenoplectus tabernaemontani	12
#	Schoenus nigricans	7
#	Scilla bifolia	1
#	Scilla forbesii	1
#	Scirpus sylvaticus	8
#	Scleranthus annuus	4
#	Scolymus hispanicus	1
#	Scorzoneroides autumnalis	6
#	Scrophularia auriculata	5
#	Scrophularia nodosa	2
#	Scutellaria altissima	1
#	Scutellaria galericulata	6
#	Scutellaria minor	4
#	Securigera varia	5
#	Sedum acre	9
#	Sedum album	5
#	Sedum anglicum	3
#	Sedum forsterianum	4
#	Sedum rupestre	5
#	Sedum telephium	4
#	Selaginella selaginoides	3
#	Sempervivum tectorum	2
#	Senecio erucifolius	5
#	Senecio inaequidens	3
#	Senecio squalidus	0
#	Senecio sylvaticus	3
#	Senecio viscosus	0
#	Senecio vulgaris	11
#	Serratula tinctoria	3
#	Sesleria caerulea	3
#	Setaria italica	83
#	Setaria pumila	12
#	Setaria verticillata	9
#	Setaria viridis	16
#	Sherardia arvensis	9
#	Sidalcea candida	0
#	Sidalcea malviflora	0
#	Silaum silaus	3
#	Silene dioica	22
#	Silene flos-cuculi	7
#	Silene gallica	5
#	Silene latifolia	73
#	Silene noctiflora	5
#	Silene vulgaris	17
#	Silybum marianum	10
#	Sinapis alba	6
#	Sinapis arvensis	10
#	Sison amomum	5
#	Sisymbrium altissimum	2
#	Sisymbrium irio	1
#	Sisymbrium loeselii	1
#	Sisymbrium officinale	12
#	Sisymbrium orientale	3
#	Sisymbrium strictissimum	0
#	Sisymbrium volgense	0
#	Sisyrinchium montanum	10
#	Sium latifolium	3
#	Smyrnium olusatrum	2
#	Solanum chenopodioides	3
#	Solanum dulcamara	16
#	Solanum lycopersicum	138
#	Solanum nigrum	31
#	Solanum physalifolium	2
#	Solanum rostratum	1
#	Solanum sarachoides	0
#	Solanum sisymbriifolium	1
#	Solanum tuberosum	89
#	Soleirolia soleirolii	5
#	Solidago canadensis	18
#	Solidago gigantea	7
#	Solidago graminifolia	7
#	Solidago virgaurea	6
#	Sonchus arvensis	11
#	Sonchus asper	13
#	Sonchus oleraceus	15
#	Sonchus palustris	1
#	Sorbus aria	8
#	Sorbus aucuparia	16
#	Sorbus intermedia	7
#	Sorbus latifolia	0
#	Sorbus torminalis	7
#	Sparganium emersum	12
#	Sparganium erectum	6
#	Sparganium natans	7
#	Spartina anglica	6
#	Spergula arvensis	5
#	Spergularia marina	5
#	Spergularia media	3
#	Spergularia rubra	4
#	Spilanthes decumbens	0
#	Spiraea alba	2
#	Spiraea douglasii	0
#	Spiraea salicifolia	5
#	Spiranthes spiralis	3
#	Spirodela polyrhiza	21
#	Stachys annua	0
#	Stachys arvensis	5
#	Stachys byzantina	4
#	Stachys palustris	5
#	Stachys sylvatica	11
#	Stellaria alsine	3
#	Stellaria graminea	6
#	Stellaria holostea	5
#	Stellaria media	16
#	Stellaria neglecta	2
#	Stellaria nemorum	3
#	Stellaria pallida	3
#	Stellaria palustris	4
#	Stratiotes aloides	5
#	Suaeda maritima	13
#	Succisa pratensis	4
#	Symphoricarpos albus	1
#	Symphoricarpos orbiculatus	2
#	Symphytum bulbosum	0
#	Symphytum officinale	10
#	Symphytum orientale	1
#	Syringa vulgaris	9
#	Tagetes minuta	4
#	Tamarix gallica	2
#	Tamus communis	15
#	Tanacetum parthenium	4
#	Tanacetum vulgare	7
#	Taraxacum aequilobum	0
#	Taraxacum alatum	0
#	Taraxacum ancistrolobum	0
#	Taraxacum arenastrum	0
#	Taraxacum argutum	0
#	Taraxacum aurosulum	0
#	Taraxacum boekmanii	0
#	Taraxacum brachyglossum	0
#	Taraxacum bracteatum	0
#	Taraxacum cordatum	0
#	Taraxacum croceiflorum	0
#	Taraxacum dilatatum	0
#	Taraxacum duplidentifrons	0
#	Taraxacum ekmanii	1
#	Taraxacum euryphyllum	0
#	Taraxacum exacutum	0
#	Taraxacum expallidiforme	0
#	Taraxacum faeroense	0
#	Taraxacum fasciatum	0
#	Taraxacum fulgidum	0
#	Taraxacum fulviforme	0
#	Taraxacum haematicum	0
#	Taraxacum hamatiforme	0
#	Taraxacum hamatum	0
#	Taraxacum huelphersianum	0
#	Taraxacum insigne	0
#	Taraxacum interveniens	0
#	Taraxacum kernianum	0
#	Taraxacum laciniosifrons	0
#	Taraxacum lacistophyllum	0
#	Taraxacum lamprophyllum	0
#	Taraxacum latisectum	0
#	Taraxacum lingulatum	0
#	Taraxacum maculosum	0
#	Taraxacum marklundii	0
#	Taraxacum nordstedtii	0
#	Taraxacum ochrochlorum	0
#	Taraxacum ostenfeldii	0
#	Taraxacum oxoniense	0
#	Taraxacum pachymerum	0
#	Taraxacum pallescens	0
#	Taraxacum palustre	0
#	Taraxacum pannucium	0
#	Taraxacum polyodon	0
#	Taraxacum proximum	0
#	Taraxacum pseudohamatum	0
#	Taraxacum pseudolarssonii	0
#	Taraxacum quadrans	0
#	Taraxacum richardsianum	0
#	Taraxacum rubicundum	0
#	Taraxacum sagittipotens	0
#	Taraxacum sellandii	0
#	Taraxacum sinuatum	0
#	Taraxacum stenacrum	0
#	Taraxacum stictophyllum	0
#	Taraxacum subcyanolepis	0
#	Taraxacum subhamatum	0
#	Taraxacum sublaeticolor	0
#	Taraxacum subundulatum	0
#	Taraxacum trilobatum	0
#	Taraxacum undulatiflorum	0
#	Taraxacum xanthostigma	0
#	Taxus baccata	24
#	Teesdalia nudicaulis	1
#	Tellima grandiflora	2
#	Teucrium scorodonia	7
#	Thalictrum flavum	3
#	Thelypteris palustris	14
#	Thlaspi arvense	13
#	Thuja plicata	4
#	Thymus polytrichus	8
#	Thymus pulegioides	10
#	Thyselium palustre	0
#	Tilia cordata	6
#	Tilia platyphyllos	6
#	Tolmiea menziesii	1
#	Torilis arvensis	7
#	Torilis japonica	8
#	Torilis nodosa	4
#	Tragopogon porrifolius	2
#	Tragopogon pratensis	11
#	Trichophorum germanicum	0
#	Trifolium arvense	8
#	Trifolium campestre	11
#	Trifolium dubium	9
#	Trifolium fragiferum	4
#	Trifolium hybridum	7
#	Trifolium medium	4
#	Trifolium micranthum	3
#	Trifolium pratense	30
#	Trifolium repens	24
#	Triglochin maritima	21
#	Triglochin palustris	14
#	Tripleurospermum inodorum	4
#	Tripleurospermum maritimum	26
#	Trisetum flavescens	7
#	Triticum aestivum	70
#	Triticum spelta	9
#	Tropaeolum majus	6
#	Tsuga heterophylla	1
#	Tulipa gesneriana	1
#	Turgenia latifolia	1
#	Tussilago farfara	15
#	Typha angustifolia	12
#	Typha latifolia	14
#	Ulex europaeus	5
#	Ulmus glabra	12
#	Ulmus minor	8
#	Ulmus plotii	0
#	Ulmus procera	6
#	Urtica dioica	16
#	Urtica urens	8
#	Utricularia minor	5
#	Utricularia vulgaris	2
#	Vaccaria hispanica	3
#	Vaccinium myrtillus	11
#	Vaccinium oxycoccos	4
#	Vaccinium vitis-idaea	24
#	Valeriana dioica	7
#	Valeriana officinalis	16
#	Valeriana pyrenaica	3
#	Valerianella carinata	3
#	Valerianella dentata	4
#	Valerianella eriocarpa	3
#	Valerianella locusta	7
#	Valerianella rimosa	1
#	Verbascum blattaria	0
#	Verbascum nigrum	5
#	Verbascum phlomoides	1
#	Verbascum thapsus	14
#	Verbascum virgatum	3
#	Verbena bonariensis	7
#	Verbena hastata	7
#	Verbena officinalis	13
#	Veronica agrestis	5
#	Veronica anagallis-aquatica	6
#	Veronica arvensis	9
#	Veronica beccabunga	4
#	Veronica catenata	4
#	Veronica chamaedrys	8
#	Veronica filiformis	1
#	Veronica hederifolia	3
#	Veronica longifolia	0
#	Veronica montana	4
#	Veronica officinalis	13
#	Veronica persica	9
#	Veronica polita	2
#	Veronica scutellata	4
#	Veronica serpyllifolia	14
#	Veronica spicata	5
#	Viburnum lantana	9
#	Viburnum opulus	16
#	Vicia cracca	27
#	Vicia faba	18
#	Vicia hirsuta	8
#	Vicia lathyroides	2
#	Vicia lutea	4
#	Vicia sativa	35
#	Vicia sepium	6
#	Vicia sylvatica	5
#	Vicia tetrasperma	17
#	Vinca major	1
#	Vinca minor	12
#	Viola arvensis	9
#	Viola canina	5
#	Viola hirta	4
#	Viola odorata	6
#	Viola palustris	5
#	Viola reichenbachiana	5
#	Viola riviniana	10
#	Viola tricolor	4
#	Viscum album	8
#	Vitis vinifera	129
#	Vulpia bromoides	7
#	Vulpia myuros	14
#	Xanthium spinosum	1
#	Xanthium strumarium	3
#	Zannichellia palustris	13
#	Zantedeschia aethiopica	13
#	Zea mays	113
#	Zostera marina	33
#	Zostera noltei	5

total number of accessions fetched: 13583


downloading 13583 records .. processing 1000 accessions per batch

[Wed Oct 19 2016 20:58:24]	batch: 0000001	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000001.gb	(total: 1000 (7.36 %); remaining: 12583 (92.64 %) -&gt; eyorks_flora_curated.accessions.txt)
[Wed Oct 19 2016 20:59:16]	batch: 0000002	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000002.gb	(total: 2000 (14.72 %); remaining: 11583 (85.28 %) -&gt; eyorks_flora_curated.accessions.txt)
[Wed Oct 19 2016 21:00:58]	batch: 0000003	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000003.gb	(total: 3000 (22.09 %); remaining: 10583 (77.91 %) -&gt; eyorks_flora_curated.accessions.txt)
[Wed Oct 19 2016 21:02:08]	batch: 0000004	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000004.gb	(total: 4000 (29.45 %); remaining: 9583 (70.55 %) -&gt; eyorks_flora_curated.accessions.txt)
[Wed Oct 19 2016 21:02:51]	batch: 0000005	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000005.gb	(total: 5000 (36.81 %); remaining: 8583 (63.19 %) -&gt; eyorks_flora_curated.accessions.txt)
[Wed Oct 19 2016 21:03:56]	batch: 0000006	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000006.gb	(total: 6000 (44.17 %); remaining: 7583 (55.83 %) -&gt; eyorks_flora_curated.accessions.txt)
[Wed Oct 19 2016 21:04:52]	batch: 0000007	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000007.gb	(total: 7000 (51.54 %); remaining: 6583 (48.46 %) -&gt; eyorks_flora_curated.accessions.txt)
[Wed Oct 19 2016 21:05:58]	batch: 0000008	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000008.gb	(total: 8000 (58.90 %); remaining: 5583 (41.10 %) -&gt; eyorks_flora_curated.accessions.txt)
[Wed Oct 19 2016 21:07:18]	batch: 0000009	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000009.gb	(total: 9000 (66.26 %); remaining: 4583 (33.74 %) -&gt; eyorks_flora_curated.accessions.txt)
[Wed Oct 19 2016 21:08:51]	batch: 0000010	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000010.gb	(total: 10000 (73.62 %); remaining: 3583 (26.38 %) -&gt; eyorks_flora_curated.accessions.txt)
[Wed Oct 19 2016 21:10:24]	batch: 0000011	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000011.gb	(total: 11000 (80.98 %); remaining: 2583 (19.02 %) -&gt; eyorks_flora_curated.accessions.txt)
[Wed Oct 19 2016 21:11:15]	batch: 0000012	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000012.gb	(total: 12000 (88.35 %); remaining: 1583 (11.65 %) -&gt; eyorks_flora_curated.accessions.txt)
[Wed Oct 19 2016 21:12:14]	batch: 0000013	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000013.gb	(total: 13000 (95.71 %); remaining: 583 (4.29 %) -&gt; eyorks_flora_curated.accessions.txt)
[Wed Oct 19 2016 21:39:22]	batch: 0000014	583 unique records downloaded	-&gt; eyorks_flora_curated.0000014.gb	(total: 13583 (100.00 %); remaining: 0 -&gt; DONE!)

Concatenate all files	-&gt; eyorks_flora_curated.gb

</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Now we want an additional reference file containing the positive controls (unfortunately this may be less successful as several are unexpectedly missing - so I've included a number of congenerics for those that I know are missing, in the hope of a hit)</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[11]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>fetch_from_db.py -t PosList.txt -m rbcl -o positives_curated -@ callumjmacgregor@gmail.com
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>
metaBEAT may be querying NCBI&apos;s Entrez databases to fetch/verify taxonomic ids. Entrez User requirements state that you need to identify yourself by providing an email address so that NCBI can contact you in case there is a problem.

You have specified: &apos;callumjmacgregor@gmail.com&apos;


QUERYING GENBANK

check for synonyms for &quot;rbcl&quot; (this is relevant only for Genbank searches)

fetching accessions ..

#	Bryophyllum pinnatum	0
#	Crassula capitella	0
#	Crassula columnaris	0
#	Crassula deceptor	0
#	Crassula lactea	0
#	Crassula nudicaulis	1
#	Crassula ovata	0
#	Crassula perforata	1
#	Crassula socialis	0
#	Delosperma cooperi	0
#	Delosperma jansei	0
#	Delosperma sutherlandii	0
#	Delosperma tradescantioides	0
#	Dracaena aletriformis	2
#	Dracaena draco	3
#	Dracaena fragrans	1
#	Dracaena mannii	1
#	Dracaena marginata	0
#	Dracaena transvaalensis	1
#	Kalanchoe pinnata	7

total number of accessions fetched: 17


downloading 17 records .. processing 1000 accessions per batch

[Mon Oct 24 2016 11:40:38]	batch: 0000001	17 unique records downloaded	-&gt; positives_curated.0000001.gb	(total: 17 (100.00 %); remaining: 0 -&gt; DONE!)

Concatenate all files	-&gt; positives_curated.gb

</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Now run another BLAST with the same parameters as before, but the curated reference database instead of the local Genbank copy</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[2]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cd</span> <span class="n">AgainstRefDB</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>/home/working/AgainstRefDB
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Create a new Querymap.txt, telling it to go up one directory first (in order to find the trimmed, tag-assigned sequences)</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[23]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

<span class="k">for </span>sample in <span class="k">$(</span>cat ../Querymap_global.txt | cut -f 1<span class="k">)</span>
<span class="k">do</span>
<span class="k">    </span><span class="nv">fasta</span><span class="o">=</span><span class="k">$(</span>ls -1 ../<span class="nv">$sample</span>/<span class="nv">$sample</span><span class="se">\_</span>trimmed.fasta<span class="k">)</span>
    <span class="nb">echo</span> -e <span class="s2">&quot;$sample\tfasta\t$fasta&quot;</span>
<span class="k">done</span> &gt; Querymap.txt

head Querymap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>SA177	fasta	../SA177/SA177_trimmed.fasta
SB140	fasta	../SB140/SB140_trimmed.fasta
T19	fasta	../T19/T19_trimmed.fasta
Lg45	fasta	../Lg45/Lg45_trimmed.fasta
Q69	fasta	../Q69/Q69_trimmed.fasta
S72	fasta	../S72/S72_trimmed.fasta
SC66	fasta	../SC66/SC66_trimmed.fasta
S31	fasta	../S31/S31_trimmed.fasta
SC74	fasta	../SC74/SC74_trimmed.fasta
SA68	fasta	../SA68/SA68_trimmed.fasta
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[44]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-Q Querymap.txt <span class="se">\</span>
--cluster <span class="se">\</span>
--clust_match 1 <span class="se">\</span>
--clust_cov 5 <span class="se">\</span>
--blast <span class="se">\</span>
--REFlist REFmap.txt <span class="se">\</span>
--min_ident 0.95 <span class="se">\</span>
-n 5 <span class="se">\</span>
-v <span class="se">\</span>
-@ callumjmacgregor@gmail.com <span class="se">\</span>
-o MothPollenReflistRestart &gt; log_reflist
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Again we want a .txt file to inspect</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[51]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash
blastn -query /home/working/AgainstRefDB/GLOBAL/global_centroids.fasta <span class="se">\</span>
-db /home/working/AgainstRefDB/GLOBAL/BLAST_0.95/marker_blast_db <span class="se">\</span>
-out GLOBAL/BLAST_0.95/global_blastn.out.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="Blast-against-curated-reference-list">Blast against curated reference list<a class="anchor-link" href="#Blast-against-curated-reference-list">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>I now want to curate the reference list in order to improve the quality of the assignments, and to get as many of them as possible to genus or species level. To do this, I inspect the outputs of the previous BLAST on a case-by-case basis, and determine where there are hits to species that are not plausibly correct. Remove these species from the reference list and repeat the previous steps in a new sub-folder, first copying a couple of unchanged files to the new location.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[9]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="n">positives_curated</span><span class="o">.</span><span class="n">gb</span> <span class="n">Curat1</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[10]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="n">REFmap</span><span class="o">.</span><span class="n">txt</span> <span class="n">Curat1</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[2]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cd</span> <span class="n">Curat1</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>/home/working/AgainstRefDB/Curat1
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[3]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre><span class="ansibold ansigreen">AccessionBlacklist.txt</span>*                 <span class="ansibold ansiblue">S96</span>/    <span class="ansibold ansiblue">SB15</span>/
eyorks_flora_curat1.fasta               <span class="ansibold ansiblue">S97</span>/    <span class="ansibold ansiblue">SB150</span>/
eyorks_flora_curat1.gb                  <span class="ansibold ansiblue">SA1</span>/    <span class="ansibold ansiblue">SB153</span>/
eyorks_flora_curated.fasta              <span class="ansibold ansiblue">SA104</span>/  <span class="ansibold ansiblue">SB159</span>/
eyorks_flora_curated.gb                 <span class="ansibold ansiblue">SA105</span>/  <span class="ansibold ansiblue">SB163</span>/
<span class="ansibold ansiblue">GLOBAL</span>/                                 <span class="ansibold ansiblue">SA109</span>/  <span class="ansibold ansiblue">SB166</span>/
<span class="ansibold ansiblue">Lg27</span>/                                   <span class="ansibold ansiblue">SA111</span>/  <span class="ansibold ansiblue">SB170</span>/
<span class="ansibold ansiblue">Lg37</span>/                                   <span class="ansibold ansiblue">SA112</span>/  <span class="ansibold ansiblue">SB171</span>/
<span class="ansibold ansiblue">Lg45</span>/                                   <span class="ansibold ansiblue">SA113</span>/  <span class="ansibold ansiblue">SB172</span>/
<span class="ansibold ansiblue">Lg53</span>/                                   <span class="ansibold ansiblue">SA114</span>/  <span class="ansibold ansiblue">SB180</span>/
<span class="ansibold ansiblue">Lg57</span>/                                   <span class="ansibold ansiblue">SA115</span>/  <span class="ansibold ansiblue">SB182</span>/
<span class="ansibold ansiblue">Lg65</span>/                                   <span class="ansibold ansiblue">SA116</span>/  <span class="ansibold ansiblue">SB183</span>/
<span class="ansibold ansiblue">Lg9</span>/                                    <span class="ansibold ansiblue">SA117</span>/  <span class="ansibold ansiblue">SB186</span>/
loci_counts.txt                         <span class="ansibold ansiblue">SA119</span>/  <span class="ansibold ansiblue">SB194</span>/
loci.csv                                <span class="ansibold ansiblue">SA120</span>/  <span class="ansibold ansiblue">SB195</span>/
log_curat1                              <span class="ansibold ansiblue">SA121</span>/  <span class="ansibold ansiblue">SB197</span>/
MothPollenReflistCurat1_read_stats.csv  <span class="ansibold ansiblue">SA122</span>/  <span class="ansibold ansiblue">SB199</span>/
<span class="ansibold ansiblue">NEG</span>/                                    <span class="ansibold ansiblue">SA124</span>/  <span class="ansibold ansiblue">SB200</span>/
<span class="ansibold ansiblue">O84</span>/                                    <span class="ansibold ansiblue">SA125</span>/  <span class="ansibold ansiblue">SB201</span>/
<span class="ansibold ansiblue">O96</span>/                                    <span class="ansibold ansiblue">SA126</span>/  <span class="ansibold ansiblue">SB204</span>/
<span class="ansibold ansiblue">P18</span>/                                    <span class="ansibold ansiblue">SA128</span>/  <span class="ansibold ansiblue">SB21</span>/
<span class="ansibold ansiblue">P54</span>/                                    <span class="ansibold ansiblue">SA13</span>/   <span class="ansibold ansiblue">SB22</span>/
<span class="ansibold ansiblue">P59</span>/                                    <span class="ansibold ansiblue">SA130</span>/  <span class="ansibold ansiblue">SB23</span>/
<span class="ansibold ansiblue">P60</span>/                                    <span class="ansibold ansiblue">SA132</span>/  <span class="ansibold ansiblue">SB24</span>/
<span class="ansibold ansiblue">P73</span>/                                    <span class="ansibold ansiblue">SA138</span>/  <span class="ansibold ansiblue">SB27</span>/
<span class="ansibold ansiblue">P98</span>/                                    <span class="ansibold ansiblue">SA140</span>/  <span class="ansibold ansiblue">SB29</span>/
<span class="ansibold ansiblue">POS1</span>/                                   <span class="ansibold ansiblue">SA144</span>/  <span class="ansibold ansiblue">SB3</span>/
<span class="ansibold ansiblue">POS2</span>/                                   <span class="ansibold ansiblue">SA147</span>/  <span class="ansibold ansiblue">SB33</span>/
<span class="ansibold ansiblue">POS3</span>/                                   <span class="ansibold ansiblue">SA15</span>/   <span class="ansibold ansiblue">SB34</span>/
<span class="ansibold ansiblue">POS4</span>/                                   <span class="ansibold ansiblue">SA153</span>/  <span class="ansibold ansiblue">SB37</span>/
positives_curated.gb                    <span class="ansibold ansiblue">SA155</span>/  <span class="ansibold ansiblue">SB4</span>/
<span class="ansibold ansiblue">PROC</span>/                                   <span class="ansibold ansiblue">SA156</span>/  <span class="ansibold ansiblue">SB45</span>/
<span class="ansibold ansiblue">Q10</span>/                                    <span class="ansibold ansiblue">SA157</span>/  <span class="ansibold ansiblue">SB48</span>/
<span class="ansibold ansiblue">Q30</span>/                                    <span class="ansibold ansiblue">SA160</span>/  <span class="ansibold ansiblue">SB51</span>/
<span class="ansibold ansiblue">Q43</span>/                                    <span class="ansibold ansiblue">SA162</span>/  <span class="ansibold ansiblue">SB53</span>/
<span class="ansibold ansiblue">Q64</span>/                                    <span class="ansibold ansiblue">SA164</span>/  <span class="ansibold ansiblue">SB59</span>/
<span class="ansibold ansiblue">Q69</span>/                                    <span class="ansibold ansiblue">SA165</span>/  <span class="ansibold ansiblue">SB60</span>/
<span class="ansibold ansiblue">Q83</span>/                                    <span class="ansibold ansiblue">SA166</span>/  <span class="ansibold ansiblue">SB63</span>/
Querymap.txt                            <span class="ansibold ansiblue">SA171</span>/  <span class="ansibold ansiblue">SB64</span>/
<span class="ansibold ansiblue">R1</span>/                                     <span class="ansibold ansiblue">SA172</span>/  <span class="ansibold ansiblue">SB67</span>/
<span class="ansibold ansiblue">R10</span>/                                    <span class="ansibold ansiblue">SA174</span>/  <span class="ansibold ansiblue">SB71</span>/
<span class="ansibold ansiblue">R102</span>/                                   <span class="ansibold ansiblue">SA177</span>/  <span class="ansibold ansiblue">SB73</span>/
<span class="ansibold ansiblue">R13</span>/                                    <span class="ansibold ansiblue">SA180</span>/  <span class="ansibold ansiblue">SB76</span>/
<span class="ansibold ansiblue">R15</span>/                                    <span class="ansibold ansiblue">SA182</span>/  <span class="ansibold ansiblue">SB77</span>/
<span class="ansibold ansiblue">R16</span>/                                    <span class="ansibold ansiblue">SA189</span>/  <span class="ansibold ansiblue">SB8</span>/
<span class="ansibold ansiblue">R20</span>/                                    <span class="ansibold ansiblue">SA19</span>/   <span class="ansibold ansiblue">SB80</span>/
<span class="ansibold ansiblue">R22</span>/                                    <span class="ansibold ansiblue">SA190</span>/  <span class="ansibold ansiblue">SB82</span>/
<span class="ansibold ansiblue">R23</span>/                                    <span class="ansibold ansiblue">SA196</span>/  <span class="ansibold ansiblue">SB83</span>/
<span class="ansibold ansiblue">R27</span>/                                    <span class="ansibold ansiblue">SA198</span>/  <span class="ansibold ansiblue">SB85</span>/
<span class="ansibold ansiblue">R29</span>/                                    <span class="ansibold ansiblue">SA199</span>/  <span class="ansibold ansiblue">SB86</span>/
<span class="ansibold ansiblue">R30</span>/                                    <span class="ansibold ansiblue">SA2</span>/    <span class="ansibold ansiblue">SB87</span>/
<span class="ansibold ansiblue">R31</span>/                                    <span class="ansibold ansiblue">SA202</span>/  <span class="ansibold ansiblue">SB91</span>/
<span class="ansibold ansiblue">R32</span>/                                    <span class="ansibold ansiblue">SA21</span>/   <span class="ansibold ansiblue">SB93</span>/
<span class="ansibold ansiblue">R34</span>/                                    <span class="ansibold ansiblue">SA28</span>/   <span class="ansibold ansiblue">SB99</span>/
<span class="ansibold ansiblue">R36</span>/                                    <span class="ansibold ansiblue">SA31</span>/   <span class="ansibold ansiblue">SC10</span>/
<span class="ansibold ansiblue">R39</span>/                                    <span class="ansibold ansiblue">SA32</span>/   <span class="ansibold ansiblue">SC101</span>/
<span class="ansibold ansiblue">R4</span>/                                     <span class="ansibold ansiblue">SA33</span>/   <span class="ansibold ansiblue">SC106</span>/
<span class="ansibold ansiblue">R40</span>/                                    <span class="ansibold ansiblue">SA36</span>/   <span class="ansibold ansiblue">SC107</span>/
<span class="ansibold ansiblue">R43</span>/                                    <span class="ansibold ansiblue">SA43</span>/   <span class="ansibold ansiblue">SC110</span>/
<span class="ansibold ansiblue">R44</span>/                                    <span class="ansibold ansiblue">SA46</span>/   <span class="ansibold ansiblue">SC113</span>/
<span class="ansibold ansiblue">R45</span>/                                    <span class="ansibold ansiblue">SA47</span>/   <span class="ansibold ansiblue">SC117</span>/
<span class="ansibold ansiblue">R48</span>/                                    <span class="ansibold ansiblue">SA48</span>/   <span class="ansibold ansiblue">SC119</span>/
<span class="ansibold ansiblue">R49</span>/                                    <span class="ansibold ansiblue">SA49</span>/   <span class="ansibold ansiblue">SC122</span>/
<span class="ansibold ansiblue">R59</span>/                                    <span class="ansibold ansiblue">SA5</span>/    <span class="ansibold ansiblue">SC124</span>/
<span class="ansibold ansiblue">R60</span>/                                    <span class="ansibold ansiblue">SA50</span>/   <span class="ansibold ansiblue">SC125</span>/
<span class="ansibold ansiblue">R67</span>/                                    <span class="ansibold ansiblue">SA51</span>/   <span class="ansibold ansiblue">SC141</span>/
<span class="ansibold ansiblue">R68</span>/                                    <span class="ansibold ansiblue">SA53</span>/   <span class="ansibold ansiblue">SC142</span>/
<span class="ansibold ansiblue">R71</span>/                                    <span class="ansibold ansiblue">SA55</span>/   <span class="ansibold ansiblue">SC143</span>/
<span class="ansibold ansiblue">R74</span>/                                    <span class="ansibold ansiblue">SA58</span>/   <span class="ansibold ansiblue">SC157</span>/
<span class="ansibold ansiblue">R8</span>/                                     <span class="ansibold ansiblue">SA62</span>/   <span class="ansibold ansiblue">SC16</span>/
<span class="ansibold ansiblue">R85</span>/                                    <span class="ansibold ansiblue">SA64</span>/   <span class="ansibold ansiblue">SC168</span>/
<span class="ansibold ansiblue">R9</span>/                                     <span class="ansibold ansiblue">SA67</span>/   <span class="ansibold ansiblue">SC178</span>/
<span class="ansibold ansiblue">R90</span>/                                    <span class="ansibold ansiblue">SA68</span>/   <span class="ansibold ansiblue">SC187</span>/
<span class="ansibold ansiblue">R95</span>/                                    <span class="ansibold ansiblue">SA7</span>/    <span class="ansibold ansiblue">SC198</span>/
<span class="ansibold ansiblue">R98</span>/                                    <span class="ansibold ansiblue">SA70</span>/   <span class="ansibold ansiblue">SC19A</span>/
<span class="ansibold ansiblue">R99</span>/                                    <span class="ansibold ansiblue">SA73</span>/   <span class="ansibold ansiblue">SC202</span>/
raw.pkl                                 <span class="ansibold ansiblue">SA75</span>/   <span class="ansibold ansiblue">SC26</span>/
rbcl_synonyms.txt                       <span class="ansibold ansiblue">SA76</span>/   <span class="ansibold ansiblue">SC28</span>/
<span class="ansibold ansigreen">RefListCurat1.txt</span>*                      <span class="ansibold ansiblue">SA81</span>/   <span class="ansibold ansiblue">SC34</span>/
REFmap.txt                              <span class="ansibold ansiblue">SA86</span>/   <span class="ansibold ansiblue">SC35</span>/
<span class="ansibold ansiblue">S100</span>/                                   <span class="ansibold ansiblue">SA87</span>/   <span class="ansibold ansiblue">SC43</span>/
<span class="ansibold ansiblue">S101</span>/                                   <span class="ansibold ansiblue">SA88</span>/   <span class="ansibold ansiblue">SC44</span>/
<span class="ansibold ansiblue">S102</span>/                                   <span class="ansibold ansiblue">SA89</span>/   <span class="ansibold ansiblue">SC48</span>/
<span class="ansibold ansiblue">S11</span>/                                    <span class="ansibold ansiblue">SA9</span>/    <span class="ansibold ansiblue">SC49</span>/
<span class="ansibold ansiblue">S13</span>/                                    <span class="ansibold ansiblue">SA91</span>/   <span class="ansibold ansiblue">SC51</span>/
<span class="ansibold ansiblue">S18</span>/                                    <span class="ansibold ansiblue">SA93</span>/   <span class="ansibold ansiblue">SC51A</span>/
<span class="ansibold ansiblue">S19</span>/                                    <span class="ansibold ansiblue">SA94</span>/   <span class="ansibold ansiblue">SC54A</span>/
<span class="ansibold ansiblue">S24</span>/                                    <span class="ansibold ansiblue">SA95</span>/   <span class="ansibold ansiblue">SC55</span>/
<span class="ansibold ansiblue">S30</span>/                                    <span class="ansibold ansiblue">SA96</span>/   <span class="ansibold ansiblue">SC57</span>/
<span class="ansibold ansiblue">S31</span>/                                    <span class="ansibold ansiblue">SA97</span>/   <span class="ansibold ansiblue">SC60</span>/
<span class="ansibold ansiblue">S33</span>/                                    <span class="ansibold ansiblue">SA98</span>/   <span class="ansibold ansiblue">SC61</span>/
<span class="ansibold ansiblue">S35</span>/                                    <span class="ansibold ansiblue">SB102</span>/  <span class="ansibold ansiblue">SC66</span>/
<span class="ansibold ansiblue">S39</span>/                                    <span class="ansibold ansiblue">SB104</span>/  <span class="ansibold ansiblue">SC7</span>/
<span class="ansibold ansiblue">S43</span>/                                    <span class="ansibold ansiblue">SB106</span>/  <span class="ansibold ansiblue">SC72</span>/
<span class="ansibold ansiblue">S50</span>/                                    <span class="ansibold ansiblue">SB111</span>/  <span class="ansibold ansiblue">SC74</span>/
<span class="ansibold ansiblue">S56</span>/                                    <span class="ansibold ansiblue">SB116</span>/  <span class="ansibold ansiblue">SC77</span>/
<span class="ansibold ansiblue">S59</span>/                                    <span class="ansibold ansiblue">SB119</span>/  <span class="ansibold ansiblue">SC78</span>/
<span class="ansibold ansiblue">S62</span>/                                    <span class="ansibold ansiblue">SB120</span>/  <span class="ansibold ansiblue">SC81</span>/
<span class="ansibold ansiblue">S63</span>/                                    <span class="ansibold ansiblue">SB127</span>/  <span class="ansibold ansiblue">SC82</span>/
<span class="ansibold ansiblue">S65</span>/                                    <span class="ansibold ansiblue">SB128</span>/  <span class="ansibold ansiblue">SC86A</span>/
<span class="ansibold ansiblue">S67</span>/                                    <span class="ansibold ansiblue">SB129</span>/  <span class="ansibold ansiblue">SC88</span>/
<span class="ansibold ansiblue">S7</span>/                                     <span class="ansibold ansiblue">SB13</span>/   <span class="ansibold ansiblue">SC9</span>/
<span class="ansibold ansiblue">S72</span>/                                    <span class="ansibold ansiblue">SB131</span>/  <span class="ansibold ansiblue">SC93</span>/
<span class="ansibold ansiblue">S77</span>/                                    <span class="ansibold ansiblue">SB135</span>/  species_vs_locus_raw.csv
<span class="ansibold ansiblue">S79</span>/                                    <span class="ansibold ansiblue">SB136</span>/  <span class="ansibold ansiblue">T19</span>/
<span class="ansibold ansiblue">S8</span>/                                     <span class="ansibold ansiblue">SB138</span>/  <span class="ansibold ansiblue">T21</span>/
<span class="ansibold ansiblue">S80</span>/                                    <span class="ansibold ansiblue">SB139</span>/  <span class="ansibold ansiblue">T32</span>/
<span class="ansibold ansiblue">S81</span>/                                    <span class="ansibold ansiblue">SB140</span>/  <span class="ansibold ansiblue">T49</span>/
<span class="ansibold ansiblue">S83</span>/                                    <span class="ansibold ansiblue">SB141</span>/  <span class="ansibold ansiblue">T68</span>/
<span class="ansibold ansiblue">S9</span>/                                     <span class="ansibold ansiblue">SB142</span>/  <span class="ansibold ansiblue">T78</span>/
<span class="ansibold ansiblue">S90</span>/                                    <span class="ansibold ansiblue">SB146</span>/  target_locus.csv
<span class="ansibold ansiblue">S92</span>/                                    <span class="ansibold ansiblue">SB147</span>/
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[4]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>head RefListCurat1.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>









</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[5]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>fetch_from_db.py -t RefListCurat1.txt -m rbcl -o eyorks_flora_curat1 -@ callumjmacgregor@gmail.com
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>
metaBEAT may be querying NCBI&apos;s Entrez databases to fetch/verify taxonomic ids. Entrez User requirements state that you need to identify yourself by providing an email address so that NCBI can contact you in case there is a problem.

You have specified: &apos;callumjmacgregor@gmail.com&apos;


QUERYING GENBANK

check for synonyms for &quot;rbcl&quot; (this is relevant only for Genbank searches)

fetching accessions ..

#	Abies alba	8
#	Abies cephalonica	2
#	Abies cilicica	1
#	Abies concolor	6
#	Abies delavayi	6
#	Abies firma	6
#	Abies grandis	4
#	Abies homolepis	4
#	Abies lasiocarpa	5
#	Abies nordmanniana	4
#	Abies pinsapo	5
#	Abies procera	1
#	Abies veitchii	4
#	Acaena novae-zelandiae	3
#	Acer campestre	18
#	Acer platanoides	6
#	Acer pseudoplatanus	16
#	Achillea alpina	0
#	Achillea distans	0
#	Achillea ligustica	0
#	Achillea millefolium	39
#	Achillea ptarmica	4
#	Aconitum napellus	5
#	Acorus calamus	33
#	Actaea spicata	5
#	Adonis annua	2
#	Adoxa moschatellina	7
#	Aegopodium podagraria	7
#	Aesculus carnea	0
#	Aesculus hippocastanum	11
#	Aethusa cynapium	5
#	Agrimonia eupatoria	5
#	Agrimonia procera	5
#	Agrostemma githago	7
#	Agrostis canina	3
#	Agrostis capillaris	5
#	Agrostis castellana	0
#	Agrostis gigantea	11
#	Agrostis scabra	6
#	Agrostis stolonifera	20
#	Agrostis vinealis	3
#	Aira caryophyllea	6
#	Aira praecox	6
#	Ajuga reptans	10
#	Alcea rosea	2
#	Alchemilla glabra	5
#	Alchemilla mollis	2
#	Alchemilla vulgaris	2
#	Alchemilla xanthochlora	5
#	Alisma lanceolatum	4
#	Alisma plantago-aquatica	17
#	Alliaria petiolata	21
#	Allium carinatum	1
#	Allium moly	3
#	Allium neapolitanum	9
#	Allium oleraceum	3
#	Allium schoenoprasum	3
#	Allium scorodoprasum	2
#	Allium triquetrum	6
#	Allium ursinum	10
#	Allium vineale	3
#	Alnus cordata	1
#	Alnus glutinosa	17
#	Alnus incana	11
#	Alopecurus aequalis	16
#	Alopecurus geniculatus	4
#	Alopecurus myosuroides	4
#	Alopecurus pratensis	9
#	Alyssum alyssoides	0
#	Amaranthus albus	5
#	Amaranthus hybridus	18
#	Amaranthus retroflexus	16
#	Amaranthus thunbergii	0
#	Ambrosia artemisiifolia	19
#	Ammi majus	1
#	Ammi visnaga	1
#	Amsinckia lycopsoides	0
#	Amsinckia micrantha	1
#	Anacamptis morio	1
#	Anacamptis pyramidalis	2
#	Anagallis arvensis	9
#	Anagallis tenella	5
#	Anaphalis margaritacea	7
#	Anchusa arvensis	4
#	Andromeda polifolia	15
#	Anemone apennina	0
#	Anemone nemorosa	6
#	Anemone ranunculoides	1
#	Anethum graveolens	6
#	Angelica sylvestris	4
#	Anisantha diandra	0
#	Anisantha madritensis	0
#	Anisantha rigida	7
#	Anisantha rubens	0
#	Anisantha sterilis	7
#	Anisantha tectorum	0
#	Antennaria dioica	4
#	Anthemis arvensis	2
#	Anthemis cotula	2
#	Anthemis tinctoria	1
#	Anthoxanthum aristatum	0
#	Anthoxanthum odoratum	14
#	Anthriscus caucalis	2
#	Anthriscus sylvestris	9
#	Anthyllis vulneraria	8
#	Antirrhinum majus	7
#	Apera interrupta	1
#	Apera spica-venti	1
#	Aphanes arvensis	4
#	Aphanes australis	2
#	Apium graveolens	10
#	Apium inundatum	2
#	Apium nodiflorum	6
#	Apium repens	0
#	Aquilegia vulgaris	7
#	Arabidopsis thaliana	187
#	Arabis caucasica	1
#	Arabis hirsuta	7
#	Araucaria araucana	5
#	Arbutus unedo	3
#	Arctium lappa	11
#	Arctium minus	15
#	Arctium nemorosum	0
#	Arenaria leptoclados	0
#	Arenaria serpyllifolia	15
#	Armeria maritima	6
#	Armoracia rusticana	5
#	Arnoseris minima	0
#	Arrhenatherum elatius	11
#	Artemisia absinthium	8
#	Artemisia campestris	3
#	Artemisia maritima	2
#	Artemisia verlotiorum	0
#	Artemisia vulgaris	21
#	Arum italicum	9
#	Arum maculatum	7
#	Asparagus officinalis	17
#	Asperugo procumbens	1
#	Asperula arvensis	0
#	Asperula cynanchica	6
#	Asplenium adiantum-nigrum	15
#	Asplenium ceterach	9
#	Asplenium marinum	2
#	Asplenium ruta-muraria	23
#	Asplenium scolopendrium	8
#	Asplenium trichomanes	20
#	Aster lanceolatus	7
#	Aster novi-belgii	1
#	Aster tripolium	3
#	Astragalus danicus	1
#	Astragalus glycyphyllos	3
#	Astrantia major	3
#	Athyrium filix-femina	17
#	Atriplex littoralis	4
#	Atriplex patula	13
#	Atriplex prostrata	5
#	Atropa belladonna	13
#	Aubrieta deltoidea	1
#	Avena fatua	15
#	Avena sativa	10
#	Avenula pratensis	2
#	Avenula pubescens	5
#	Azolla filiculoides	6
#	Baldellia ranunculoides	4
#	Ballota nigra	6
#	Barbarea intermedia	1
#	Barbarea stricta	0
#	Barbarea verna	4
#	Barbarea vulgaris	16
#	Bassia scoparia	4
#	Bellis perennis	12
#	Berberis darwinii	0
#	Berberis vulgaris	3
#	Berteroa incana	3
#	Berula erecta	5
#	Beta trigyna	1
#	Beta vulgaris	121
#	Betonica officinalis	8
#	Betula pendula	18
#	Betula pubescens	8
#	Bidens cernua	12
#	Bidens tripartita	7
#	Blackstonia perfoliata	5
#	Blechnum spicant	4
#	Blysmus compressus	8
#	Bolboschoenus maritimus	7
#	Borago officinalis	9
#	Botrychium lunaria	11
#	Brachypodium pinnatum	7
#	Brachypodium rupestre	3
#	Brachypodium sylvaticum	9
#	Brassica juncea	16
#	Brassica napus	290
#	Brassica nigra	11
#	Brassica oleracea	102
#	Brassica rapa	82
#	Briza maxima	5
#	Briza media	7
#	Bromopsis benekenii	2
#	Bromopsis erecta	9
#	Bromopsis inermis	22
#	Bromopsis ramosa	3
#	Bromus arvensis	1
#	Bromus commutatus	4
#	Bromus hordeaceus	8
#	Bromus lanceolatus	0
#	Bromus lepidus	0
#	Bromus racemosus	3
#	Bromus secalinus	2
#	Bryonia dioica	5
#	Buddleja davidii	6
#	Buddleja globosa	0
#	Buddleja officinalis	1
#	Bunias orientalis	1
#	Bupleurum baldense	0
#	Bupleurum rotundifolium	0
#	Bupleurum subovatum	0
#	Bupleurum tenuissimum	2
#	Butomus umbellatus	7
#	Buxus sempervirens	13
#	Cakile maritima	4
#	Calamagrostis canescens	5
#	Calamagrostis epigejos	13
#	Calamagrostis gracilescens	0
#	Calendula officinalis	5
#	Calla palustris	5
#	Callitriche hermaphroditica	6
#	Callitriche obtusangula	3
#	Callitriche platycarpa	5
#	Callitriche stagnalis	7
#	Calluna vulgaris	11
#	Caltha palustris	32
#	Calystegia pulchra	2
#	Calystegia sepium	18
#	Calystegia silvatica	4
#	Calystegia soldanella	6
#	Camelina sativa	156
#	Campanula glomerata	11
#	Campanula latifolia	6
#	Campanula medium	2
#	Campanula patula	10
#	Campanula persicifolia	2
#	Campanula portenschlagiana	2
#	Campanula poscharskyana	1
#	Campanula rapunculoides	10
#	Campanula rapunculus	5
#	Campanula rotundifolia	22
#	Campanula trachelium	9
#	Cannabis sativa	26
#	Capsella bursa-pastoris	24
#	Cardamine flexuosa	5
#	Cardamine hirsuta	13
#	Cardamine pratensis	24
#	Carduus crispus	3
#	Carduus nutans	6
#	Carduus pycnocephalus	0
#	Carduus tenuiflorus	4
#	Carex acuta	3
#	Carex acutiformis	3
#	Carex appropinquata	0
#	Carex arenaria	5
#	Carex binervis	3
#	Carex canescens	8
#	Carex caryophyllea	4
#	Carex demissa	0
#	Carex diandra	5
#	Carex dioica	4
#	Carex distans	4
#	Carex disticha	3
#	Carex divisa	3
#	Carex divulsa	5
#	Carex echinata	3
#	Carex elata	3
#	Carex elongata	2
#	Carex extensa	5
#	Carex flacca	8
#	Carex hirta	4
#	Carex hostiana	5
#	Carex lasiocarpa	4
#	Carex lepidocarpa	0
#	Carex leporina	0
#	Carex muricata	4
#	Carex nigra	7
#	Carex oederi	0
#	Carex otrubae	5
#	Carex pallescens	5
#	Carex panicea	7
#	Carex paniculata	4
#	Carex pendula	9
#	Carex pilulifera	5
#	Carex pseudocyperus	6
#	Carex pulicaris	5
#	Carex remota	4
#	Carex riparia	3
#	Carex rostrata	4
#	Carex spicata	4
#	Carex sylvatica	5
#	Carex vesicaria	3
#	Carex viridula	4
#	Carlina vulgaris	5
#	Carpinus betulus	8
#	Carum carvi	6
#	Castanea sativa	12
#	Catabrosa aquatica	6
#	Catapodium marinum	3
#	Catapodium rigidum	6
#	Cedrus atlantica	6
#	Cedrus deodara	17
#	Cedrus libani	4
#	Centaurea aspera	0
#	Centaurea calcitrapa	0
#	Centaurea cyanus	5
#	Centaurea diluta	0
#	Centaurea montana	1
#	Centaurea nigra	6
#	Centaurea scabiosa	5
#	Centaurea solstitialis	0
#	Centaurium erythraea	7
#	Centaurium littorale	1
#	Centaurium pulchellum	5
#	Centranthus ruber	7
#	Centunculus minimus	0
#	Cephalaria gigantea	1
#	Cerastium arvense	7
#	Cerastium diffusum	4
#	Cerastium fontanum	15
#	Cerastium glomeratum	6
#	Cerastium semidecandrum	2
#	Cerastium tomentosum	2
#	Ceratocapnos claviculata	4
#	Ceratochloa carinata	2
#	Ceratochloa cathartica	12
#	Ceratophyllum demersum	15
#	Ceratophyllum submersum	4
#	Chaenorhinum minus	7
#	Chaerophyllum temulum	9
#	Chamaecyparis lawsoniana	8
#	Chamaecyparis obtusa	7
#	Chamaecyparis pisifera	7
#	Chamaemelum nobile	5
#	Chamerion angustifolium	29
#	Chara vulgaris	8
#	Chelidonium majus	14
#	Chenopodium album	46
#	Chenopodium bonus-henricus	6
#	Chenopodium ficifolium	5
#	Chenopodium glaucum	7
#	Chenopodium hybridum	0
#	Chenopodium opulifolium	2
#	Chenopodium polyspermum	4
#	Chenopodium pratericola	0
#	Chenopodium quinoa	13
#	Chenopodium rubrum	4
#	Chenopodium urbicum	4
#	Chenopodium vulvaria	4
#	Chrysosplenium alternifolium	4
#	Chrysosplenium oppositifolium	4
#	Cichorium intybus	17
#	Cicuta virosa	7
#	Circaea lutetiana	15
#	Cirsium acaule	4
#	Cirsium arvense	14
#	Cirsium dissectum	3
#	Cirsium eriophorum	3
#	Cirsium palustre	8
#	Cirsium vulgare	18
#	Cladium mariscus	7
#	Claytonia perfoliata	1
#	Claytonia sibirica	2
#	Clematis montana	2
#	Clematis tangutica	1
#	Clematis vitalba	8
#	Clinopodium acinos	5
#	Clinopodium ascendens	1
#	Clinopodium vulgare	17
#	Cochlearia anglica	4
#	Cochlearia danica	4
#	Cochlearia officinalis	3
#	Coeloglossum viride	6
#	Coincya monensis	2
#	Colchicum autumnale	14
#	Comarum palustre	8
#	Conium maculatum	8
#	Conopodium majus	5
#	Conringia orientalis	0
#	Consolida ajacis	3
#	Convolvulus arvensis	20
#	Conyza canadensis	9
#	Conyza floribunda	0
#	Coriandrum sativum	7
#	Cornus alba	1
#	Cornus sanguinea	11
#	Coronilla scorpioides	0
#	Coronilla valentina	0
#	Corrigiola litoralis	1
#	Cortaderia selloana	3
#	Corylus avellana	22
#	Cotoneaster dielsianus	3
#	Cotoneaster horizontalis	5
#	Cotoneaster integrifolius	0
#	Cotoneaster lacteus	2
#	Cotoneaster rehderi	2
#	Cotoneaster simonsii	1
#	Cotoneaster splendens	0
#	Crambe hispanica	0
#	Crambe maritima	3
#	Crassula helmsii	4
#	Crataegus laevigata	9
#	Crataegus monogyna	36
#	Crepis biennis	2
#	Crepis capillaris	6
#	Crepis nicaeensis	0
#	Crepis paludosa	3
#	Crepis vesicaria	2
#	Crocus vernus	2
#	Cruciata laevipes	6
#	Cupressus duclouxiana	8
#	Cupressus goveniana	4
#	Cupressus macrocarpa	8
#	Cupressus sempervirens	10
#	Cuscuta campestris	5
#	Cuscuta epithymum	7
#	Cuscuta europaea	5
#	Cuscuta suaveolens	2
#	Cymbalaria muralis	2
#	Cymbalaria pallida	0
#	Cynoglossum officinale	7
#	Cynosurus cristatus	8
#	Cynosurus echinatus	2
#	Cyperus eragrostis	4
#	Cyperus longus	7
#	Cyrtomium falcatum	13
#	Cyrtomium fortunei	21
#	Cystopteris fragilis	19
#	Cytisus scoparius	8
#	Dactylis glomerata	32
#	Dactylorhiza fuchsii	4
#	Dactylorhiza incarnata	4
#	Dactylorhiza maculata	5
#	Dactylorhiza praetermissa	3
#	Dactylorhiza purpurella	4
#	Damasonium alisma	1
#	Danthonia decumbens	6
#	Daphne laureola	4
#	Daucus carota	42
#	Deschampsia cespitosa	53
#	Deschampsia flexuosa	5
#	Deschampsia setacea	2
#	Descurainia sophia	5
#	Dianthus barbatus	1
#	Dianthus deltoides	8
#	Digitalis purpurea	14
#	Digitaria sanguinalis	9
#	Diplotaxis muralis	1
#	Diplotaxis tenuifolia	9
#	Dipsacus fullonum	13
#	Dipsacus pilosus	4
#	Dipsacus sativus	1
#	Doronicum pardalianches	1
#	Drosera intermedia	1
#	Drosera rotundifolia	10
#	Dryopteris affinis	4
#	Dryopteris borreri	4
#	Dryopteris cambrensis	0
#	Dryopteris carthusiana	22
#	Dryopteris cristata	20
#	Dryopteris dilatata	11
#	Dryopteris filix-mas	15
#	Dysphania pumilio	0
#	Echinochloa colona	9
#	Echinochloa crus-galli	33
#	Echium plantagineum	3
#	Echium vulgare	5
#	Eleocharis acicularis	11
#	Eleocharis multicaulis	3
#	Eleocharis palustris	13
#	Eleocharis quinqueflora	5
#	Eleocharis uniglumis	4
#	Eleogiton fluitans	9
#	Eleusine indica	18
#	Elodea canadensis	7
#	Elodea nuttallii	8
#	Elymus caninus	9
#	Elytrigia atherica	2
#	Elytrigia juncea	3
#	Elytrigia repens	25
#	Epilobium ciliatum	4
#	Epilobium hirsutum	8
#	Epilobium komarovianum	0
#	Epilobium montanum	7
#	Epilobium obscurum	3
#	Epilobium palustre	18
#	Epilobium parviflorum	13
#	Epilobium roseum	3
#	Epilobium tetragonum	6
#	Epipactis helleborine	21
#	Epipactis palustris	5
#	Equisetum arvense	44
#	Equisetum fluviatile	10
#	Equisetum hyemale	23
#	Equisetum palustre	7
#	Equisetum telmateia	3
#	Eragrostis cilianensis	2
#	Eragrostis pectinacea	13
#	Eranthis hyemalis	2
#	Erica cinerea	5
#	Erica tetralix	5
#	Erigeron acris	4
#	Erigeron karvinskianus	2
#	Eriophorum angustifolium	23
#	Eriophorum latifolium	4
#	Eriophorum vaginatum	19
#	Erodium cicutarium	6
#	Erodium moschatum	5
#	Erophila verna	10
#	Eryngium maritimum	6
#	Erysimum cheiranthoides	11
#	Erysimum cheiri	3
#	Escallonia macrantha	1
#	Eschscholzia californica	7
#	Euonymus europaeus	8
#	Eupatorium cannabinum	3
#	Euphorbia cyparissias	13
#	Euphorbia esula	8
#	Euphorbia exigua	6
#	Euphorbia helioscopia	10
#	Euphorbia lathyris	5
#	Euphorbia peplus	13
#	Euphorbia platyphyllos	3
#	Euphrasia micrantha	3
#	Euphrasia nemorosa	3
#	Euphrasia officinalis	2
#	Fagopyrum esculentum	12
#	Fagus sylvatica	14
#	Falcaria vulgaris	1
#	Fallopia baldschuanica	2
#	Fallopia convolvulus	13
#	Fallopia japonica	114
#	Fallopia sachalinensis	191
#	Festuca brevipila	0
#	Festuca filiformis	2
#	Festuca ovina	10
#	Festuca rubra	52
#	Ficaria verna	6
#	Ficus carica	12
#	Filago lutescens	0
#	Filago minima	2
#	Filago pyramidata	1
#	Filago vulgaris	3
#	Filipendula ulmaria	9
#	Foeniculum vulgare	17
#	Fragaria ananassa	5
#	Fragaria moschata	2
#	Fragaria vesca	69
#	Frangula alnus	12
#	Fritillaria meleagris	4
#	Fumaria densiflora	2
#	Fumaria muralis	5
#	Fumaria officinalis	4
#	Fumaria parviflora	0
#	Fumaria purpurea	2
#	Fumaria vaillantii	0
#	Gagea bohemica	3
#	Gagea lutea	7
#	Galanthus nivalis	2
#	Galanthus plicatus	1
#	Galanthus woronowii	0
#	Galega officinalis	2
#	Galeopsis angustifolia	3
#	Galeopsis bifida	5
#	Galeopsis speciosa	3
#	Galeopsis tetrahit	8
#	Galinsoga parviflora	3
#	Galinsoga quadriradiata	11
#	Galium aparine	25
#	Galium odoratum	5
#	Galium palustre	14
#	Galium saxatile	5
#	Galium uliginosum	3
#	Galium verum	14
#	Gentiana pneumonanthe	2
#	Gentianella amarella	7
#	Gentianella campestris	4
#	Geranium columbinum	4
#	Geranium dissectum	7
#	Geranium endressii	3
#	Geranium lucidum	4
#	Geranium molle	9
#	Geranium nodosum	3
#	Geranium phaeum	6
#	Geranium pratense	4
#	Geranium pusillum	3
#	Geranium pyrenaicum	2
#	Geranium robertianum	13
#	Geranium rotundifolium	6
#	Geranium sanguineum	10
#	Geranium sylvaticum	6
#	Geranium versicolor	0
#	Geum rivale	5
#	Geum urbanum	8
#	Ginkgo biloba	30
#	Gladiolus communis	1
#	Glaucium flavum	6
#	Glaux maritima	5
#	Glebionis segetum	6
#	Glechoma hederacea	19
#	Glyceria declinata	5
#	Glyceria fluitans	7
#	Glyceria maxima	3
#	Glyceria notata	5
#	Glycine max	159
#	Gnaphalium luteoalbum	3
#	Gnaphalium sylvaticum	1
#	Gnaphalium uliginosum	6
#	Goodyera repens	9
#	Groenlandia densa	6
#	Gymnadenia conopsea	9
#	Gymnadenia densiflora	0
#	Gymnocarpium dryopteris	17
#	Gymnocarpium robertianum	3
#	Hedera helix	26
#	Hedera hibernica	2
#	Helianthemum nummularium	9
#	Helianthus annuus	14
#	Helianthus decapetalus	7
#	Helianthus tuberosus	11
#	Helleborus foetidus	3
#	Helleborus viridis	4
#	Helminthotheca echioides	6
#	Helosciadium nodiflorum	6
#	Hemizonia pungens	0
#	Hepatica nobilis	8
#	Heracleum mantegazzianum	0
#	Heracleum sphondylium	6
#	Herniaria glabra	2
#	Hesperis matronalis	7
#	Hieracium acuminatum	0
#	Hieracium aggregatum	0
#	Hieracium calcaricola	0
#	Hieracium decolor	0
#	Hieracium diaphanoides	0
#	Hieracium diaphanum	0
#	Hieracium grandidens	0
#	Hieracium lepidulum	1
#	Hieracium sabaudum	0
#	Hieracium umbellatum	6
#	Hieracium vagum	0
#	Hieracium vulgatum	0
#	Hippocrepis comosa	7
#	Hippophae rhamnoides	14
#	Hippuris vulgaris	21
#	Hirschfeldia incana	2
#	Holcus lanatus	9
#	Holcus mollis	3
#	Honckenya peploides	21
#	Hordeum distichon	0
#	Hordeum murinum	19
#	Hordeum secalinum	7
#	Hordeum vulgare	34
#	Hottonia palustris	3
#	Humulus lupulus	20
#	Hyacinthoides hispanica	28
#	Hyacinthoides non-scripta	15
#	Hydrangea anomala	5
#	Hydrangea arborescens	9
#	Hydrangea aspera	2
#	Hydrocharis morsus-ranae	4
#	Hydrocotyle ranunculoides	1
#	Hydrocotyle vulgaris	5
#	Hyoscyamus niger	7
#	Hypericum androsaemum	5
#	Hypericum calycinum	1
#	Hypericum elodes	7
#	Hypericum hirsutum	3
#	Hypericum humifusum	4
#	Hypericum montanum	2
#	Hypericum perforatum	23
#	Hypericum pulchrum	4
#	Hypericum tetrapterum	3
#	Hypochaeris glabra	5
#	Hypochaeris maculata	1
#	Hypochaeris radicata	8
#	Hypopitys monotropa	1
#	Iberis amara	3
#	Ilex aquifolium	19
#	Impatiens capensis	17
#	Impatiens glandulifera	2
#	Impatiens noli-tangere	5
#	Impatiens parviflora	2
#	Inula conyzae	3
#	Inula helenium	11
#	Iris foetidissima	3
#	Iris germanica	6
#	Iris pseudacorus	12
#	Isoetes lacustris	2
#	Isolepis setacea	11
#	Iva xanthiifolia	0
#	Jacobaea vulgaris	7
#	Jasione montana	6
#	Juglans nigra	10
#	Juglans regia	8
#	Juncus acutiflorus	3
#	Juncus articulatus	8
#	Juncus bufonius	12
#	Juncus bulbosus	7
#	Juncus compressus	5
#	Juncus conglomeratus	6
#	Juncus effusus	20
#	Juncus gerardii	6
#	Juncus inflexus	5
#	Juncus maritimus	7
#	Juncus ranarius	1
#	Juncus squarrosus	5
#	Juncus subnodulosus	3
#	Juncus tenuis	7
#	Kickxia elatine	4
#	Knautia arvensis	8
#	Koeleria macrantha	7
#	Lactuca sativa	12
#	Lactuca serriola	14
#	Lactuca virosa	3
#	Lagarosiphon major	14
#	Lamiastrum galeobdolon	7
#	Lamium album	9
#	Lamium amplexicaule	10
#	Lamium confertum	4
#	Lamium hybridum	3
#	Lamium maculatum	5
#	Lamium purpureum	14
#	Lappula squarrosa	2
#	Lapsana communis	17
#	Larix decidua	9
#	Larix kaempferi	11
#	Lathraea clandestina	12
#	Lathraea squamaria	5
#	Lathyrus aphaca	14
#	Lathyrus grandiflorus	0
#	Lathyrus hirsutus	2
#	Lathyrus latifolius	3
#	Lathyrus linifolius	3
#	Lathyrus nissolia	2
#	Lathyrus palustris	6
#	Lathyrus pratensis	12
#	Lathyrus sativus	4
#	Lathyrus sylvestris	10
#	Lathyrus tuberosus	11
#	Legousia hybrida	3
#	Lemna gibba	15
#	Lemna minor	21
#	Lemna minuta	6
#	Lemna trisulca	11
#	Leontodon hispidus	6
#	Leontodon saxatilis	6
#	Leonurus cardiaca	8
#	Lepidium campestre	12
#	Lepidium coronopus	4
#	Lepidium didymum	2
#	Lepidium draba	3
#	Lepidium heterophyllum	4
#	Lepidium latifolium	3
#	Lepidium ruderale	6
#	Lepidium sativum	1
#	Lepidium virginicum	4
#	Leucanthemum vulgare	17
#	Leucojum aestivum	2
#	Leycesteria formosa	4
#	Leymus arenarius	10
#	Ligularia dentata	1
#	Ligustrum ovalifolium	13
#	Ligustrum vulgare	15
#	Lilium martagon	5
#	Lilium pyrenaicum	2
#	Limnanthes douglasii	1
#	Limonium vulgare	5
#	Limosella aquatica	6
#	Linaria purpurea	1
#	Linaria repens	3
#	Linaria vulgaris	17
#	Linum bienne	7
#	Linum catharticum	5
#	Linum perenne	8
#	Linum usitatissimum	2
#	Lithospermum arvense	4
#	Lithospermum officinale	7
#	Lobelia erinus	2
#	Lobularia maritima	4
#	Lolium multiflorum	17
#	Lolium perenne	28
#	Lolium rigidum	9
#	Lolium temulentum	8
#	Lonicera caprifolium	0
#	Lonicera involucrata	0
#	Lonicera periclymenum	6
#	Lonicera xylosteum	1
#	Lotus corniculatus	21
#	Lotus pedunculatus	6
#	Lotus tenuis	3
#	Lunaria annua	1
#	Lupinus arboreus	1
#	Lupinus polyphyllus	2
#	Luzula campestris	9
#	Luzula luzuloides	0
#	Luzula multiflora	9
#	Luzula pilosa	6
#	Luzula sylvatica	6
#	Lycium barbarum	6
#	Lycium chinense	10
#	Lycopodiella inundata	2
#	Lycopodium clavatum	13
#	Lycopus europaeus	3
#	Lysimachia nemorum	3
#	Lysimachia nummularia	6
#	Lysimachia punctata	1
#	Lysimachia thyrsiflora	0
#	Lysimachia vulgaris	5
#	Lythrum hyssopifolia	2
#	Lythrum portula	4
#	Lythrum salicaria	16
#	Mahonia aquifolium	2
#	Malus pumila	16
#	Malus sylvestris	5
#	Malva arborea	3
#	Malva moschata	5
#	Malva neglecta	8
#	Malva parviflora	6
#	Malva pseudolavatera	1
#	Malva pusilla	3
#	Malva setigera	0
#	Malva sylvestris	14
#	Marrubium vulgare	8
#	Matricaria chamomilla	34
#	Matricaria discoidea	6
#	Matthiola sinuata	3
#	Medicago arabica	5
#	Medicago lupulina	20
#	Medicago polymorpha	7
#	Medicago sativa	18
#	Melampyrum pratense	7
#	Melica uniflora	9
#	Melilotus albus	15
#	Melilotus altissimus	6
#	Melilotus indicus	11
#	Melilotus officinalis	15
#	Melissa officinalis	10
#	Mentha aquatica	6
#	Mentha arvensis	7
#	Mentha pulegium	10
#	Mentha spicata	16
#	Menyanthes trifoliata	17
#	Mercurialis annua	9
#	Mercurialis perennis	10
#	Metasequoia glyptostroboides	13
#	Milium effusum	8
#	Minuartia hybrida	3
#	Misopates orontium	4
#	Moehringia trinervia	6
#	Molinia caerulea	9
#	Montia fontana	10
#	Muscari armeniacum	1
#	Myagrum perfoliatum	0
#	Mycelis muralis	6
#	Myosotis arvensis	5
#	Myosotis discolor	5
#	Myosotis laxa	8
#	Myosotis ramosissima	4
#	Myosotis scorpioides	4
#	Myosotis secunda	5
#	Myosotis sylvatica	6
#	Myosoton aquaticum	4
#	Myosurus minimus	3
#	Myrica gale	12
#	Myrrhis odorata	2
#	Narcissus poeticus	0
#	Narcissus pseudonarcissus	1
#	Nardus stricta	7
#	Narthecium ossifragum	9
#	Nasturtium microphyllum	7
#	Nasturtium officinale	13
#	Neotinea ustulata	1
#	Neottia cordata	2
#	Neottia nidus-avis	7
#	Neottia ovata	7
#	Nepeta cataria	12
#	Neslia paniculata	2
#	Nicandra physalodes	4
#	Nigella damascena	3
#	Nuphar lutea	5
#	Nymphaea alba	12
#	Nymphoides peltata	2
#	Odontites vernus	6
#	Oenanthe aquatica	3
#	Oenanthe fistulosa	2
#	Oenothera biennis	17
#	Oenothera glazioviana	7
#	Omphalodes verna	1
#	Onobrychis viciifolia	3
#	Onoclea sensibilis	18
#	Ononis repens	6
#	Ononis spinosa	7
#	Onopordum acanthium	4
#	Ophioglossum vulgatum	8
#	Ophrys apifera	8
#	Ophrys insectifera	3
#	Orchis mascula	4
#	Orchis simia	1
#	Oreopteris limbosperma	3
#	Origanum vulgare	20
#	Ornithogalum nutans	1
#	Ornithopus perpusillus	5
#	Orobanche elatior	1
#	Orobanche hederae	11
#	Orobanche minor	46
#	Orobanche rapum-genistae	8
#	Orobanche reticulata	0
#	Osmunda regalis	41
#	Oxalis acetosella	18
#	Oxalis articulata	3
#	Oxalis corniculata	13
#	Oxalis exilis	1
#	Oxalis incarnata	1
#	Oxalis pes-caprae	3
#	Oxalis stricta	13
#	Panicum capillare	13
#	Panicum miliaceum	8
#	Papaver argemone	2
#	Papaver dubium	5
#	Papaver hybridum	3
#	Papaver lecoqii	0
#	Papaver rhoeas	15
#	Papaver somniferum	10
#	Parapholis incurva	5
#	Parapholis strigosa	4
#	Parentucellia viscosa	7
#	Parietaria judaica	8
#	Paris quadrifolia	8
#	Parnassia palustris	29
#	Pastinaca sativa	7
#	Pedicularis palustris	3
#	Pedicularis sylvatica	5
#	Pentaglottis sempervirens	3
#	Persicaria amphibia	14
#	Persicaria bistorta	6
#	Persicaria capitata	5
#	Persicaria hydropiper	28
#	Persicaria lapathifolia	17
#	Persicaria maculosa	13
#	Persicaria minor	4
#	Persicaria pensylvanica	10
#	Persicaria wallichii	6
#	Petasites albus	2
#	Petasites fragrans	1
#	Petasites hybridus	5
#	Petroselinum crispum	8
#	Petroselinum segetum	2
#	Phacelia ciliata	0
#	Phacelia parviflora	0
#	Phacelia tanacetifolia	1
#	Phalaris aquatica	0
#	Phalaris arundinacea	26
#	Phalaris canariensis	1
#	Phalaris paradoxa	1
#	Phegopteris connectilis	12
#	Philadelphus coronarius	2
#	Phleum arenarium	3
#	Phleum bertolonii	4
#	Phleum pratense	23
#	Phragmites australis	45
#	Physalis alkekengi	3
#	Phytolacca acinosa	0
#	Picea abies	33
#	Picea sitchensis	10
#	Pilosella aurantiaca	8
#	Pilosella officinarum	10
#	Pilularia globulifera	5
#	Pimpinella major	1
#	Pimpinella saxifraga	8
#	Pinguicula vulgaris	21
#	Pinus contorta	9
#	Pinus nigra	15
#	Pinus sylvestris	27
#	Pisum sativum	31
#	Plantago arenaria	0
#	Plantago coronopus	12
#	Plantago lanceolata	18
#	Plantago major	24
#	Plantago maritima	9
#	Plantago media	10
#	Platanthera bifolia	2
#	Platanthera chlorantha	8
#	Poa angustifolia	4
#	Poa annua	27
#	Poa compressa	6
#	Poa humilis	5
#	Poa nemoralis	6
#	Poa pratensis	89
#	Poa trivialis	11
#	Polemonium caeruleum	3
#	Polygala calcarea	1
#	Polygala serpyllifolia	7
#	Polygala vulgaris	7
#	Polygonum arenarium	0
#	Polygonum arenastrum	4
#	Polygonum aviculare	21
#	Polygonum rurivagum	4
#	Polypodium interjectum	3
#	Polypodium vulgare	6
#	Polypogon monspeliensis	4
#	Polystichum aculeatum	2
#	Polystichum setiferum	3
#	Pontederia cordata	4
#	Populus alba	22
#	Populus nigra	164
#	Populus tremula	17
#	Populus trichocarpa	85
#	Portulaca oleracea	17
#	Potamogeton acutifolius	0
#	Potamogeton alpinus	9
#	Potamogeton berchtoldii	4
#	Potamogeton coloratus	3
#	Potamogeton compressus	6
#	Potamogeton crispus	18
#	Potamogeton friesii	2
#	Potamogeton gramineus	11
#	Potamogeton lucens	8
#	Potamogeton natans	16
#	Potamogeton obtusifolius	6
#	Potamogeton pectinatus	13
#	Potamogeton perfoliatus	22
#	Potamogeton polygonifolius	4
#	Potamogeton pusillus	17
#	Potamogeton trichoides	3
#	Potentilla anglica	4
#	Potentilla anserina	8
#	Potentilla argentea	4
#	Potentilla erecta	5
#	Potentilla intermedia	0
#	Potentilla norvegica	4
#	Potentilla recta	4
#	Potentilla reptans	9
#	Potentilla sterilis	5
#	Potentilla tabernaemontani	0
#	Poterium sanguisorba	9
#	Primula auricula	0
#	Primula pulverulenta	3
#	Primula veris	10
#	Primula vulgaris	7
#	Prunella vulgaris	33
#	Prunus avium	11
#	Prunus cerasifera	13
#	Prunus domestica	17
#	Prunus dulcis	8
#	Prunus laurocerasus	4
#	Prunus padus	15
#	Prunus spinosa	22
#	Pseudofumaria lutea	4
#	Pseudotsuga menziesii	9
#	Pteridium aquilinum	20
#	Pteris cretica	21
#	Pteris incompleta	1
#	Pteris tremula	5
#	Pteris vittata	47
#	Puccinellia distans	11
#	Puccinellia maritima	4
#	Puccinellia rupestris	2
#	Pulicaria dysenterica	4
#	Pulmonaria officinalis	2
#	Pyracantha coccinea	10
#	Pyrola minor	4
#	Pyrus communis	17
#	Pyrus pyraster	2
#	Quercus cerris	4
#	Quercus ilex	63
#	Quercus petraea	11
#	Quercus robur	19
#	Quercus rubra	19
#	Radiola linoides	6
#	Ranunculus acris	18
#	Ranunculus aquatilis	9
#	Ranunculus arvensis	1
#	Ranunculus auricomus	4
#	Ranunculus baudotii	2
#	Ranunculus bulbosus	10
#	Ranunculus circinatus	3
#	Ranunculus flammula	7
#	Ranunculus fluitans	1
#	Ranunculus hederaceus	1
#	Ranunculus lingua	5
#	Ranunculus peltatus	3
#	Ranunculus penicillatus	2
#	Ranunculus repens	10
#	Ranunculus sardous	3
#	Ranunculus sceleratus	8
#	Ranunculus trichophyllus	5
#	Ranunculus water-crowfoot	0
#	Raphanus sativus	35
#	Rapistrum rugosum	4
#	Reseda alba	3
#	Reseda lutea	8
#	Reseda luteola	5
#	Rhamnus cathartica	23
#	Rhinanthus angustifolius	0
#	Rhinanthus minor	8
#	Rhododendron ponticum	4
#	Rhus typhina	8
#	Ribes alpinum	5
#	Ribes nigrum	1
#	Ribes rubrum	8
#	Ribes sanguineum	0
#	Ribes uva-crispa	1
#	Ricinus communis	111
#	Robinia pseudoacacia	14
#	Rorippa amphibia	3
#	Rorippa palustris	5
#	Rorippa sylvestris	5
#	Rosa arvensis	5
#	Rosa caesia	5
#	Rosa canina	10
#	Rosa micrantha	4
#	Rosa mollis	2
#	Rosa multiflora	12
#	Rosa obtusifolia	3
#	Rosa rubiginosa	4
#	Rosa rugosa	6
#	Rosa sherardii	4
#	Rosa spinosissima	3
#	Rosmarinus officinalis	12
#	Rubus armeniacus	0
#	Rubus caesius	5
#	Rubus dasyphyllus	0
#	Rubus eboracensis	0
#	Rubus echinatoides	0
#	Rubus fruticosus	4
#	Rubus halsteadensis	0
#	Rubus idaeus	20
#	Rubus nemoralis	0
#	Rubus polyanthemus	0
#	Rubus saxatilis	1
#	Rubus spectabilis	1
#	Rubus sprengelii	0
#	Rubus ulmifolius	4
#	Rubus warrenii	0
#	Rudbeckia laciniata	5
#	Rumex acetosa	20
#	Rumex acetosella	12
#	Rumex conglomeratus	6
#	Rumex crispus	22
#	Rumex hydrolapathum	5
#	Rumex obtusifolius	18
#	Rumex sanguineus	4
#	Ruppia cirrhosa	8
#	Ruppia maritima	48
#	Sagina apetala	5
#	Sagina filicaulis	0
#	Sagina maritima	3
#	Sagina nodosa	3
#	Sagina procumbens	4
#	Sagittaria sagittifolia	2
#	Salicornia dolichostachya	4
#	Salicornia europaea	8
#	Salicornia fragilis	1
#	Salicornia ramosissima	3
#	Salix alba	19
#	Salix aurita	6
#	Salix caprea	13
#	Salix cinerea	10
#	Salix pentandra	10
#	Salix purpurea	12
#	Salix repens	6
#	Salix triandra	11
#	Salix viminalis	8
#	Salsola kali	8
#	Salvia pratensis	8
#	Salvia reflexa	0
#	Salvia verbenaca	5
#	Salvia verticillata	1
#	Sambucus nigra	35
#	Samolus valerandi	6
#	Sanguisorba officinalis	15
#	Sanicula europaea	5
#	Saponaria officinalis	14
#	Saxifraga granulata	2
#	Saxifraga hirsuta	2
#	Saxifraga hypnoides	3
#	Saxifraga tridactylites	3
#	Scabiosa atropurpurea	1
#	Scabiosa columbaria	7
#	Scandix pecten-veneris	3
#	Schedonorus arundinaceus	30
#	Schedonorus giganteus	6
#	Schedonorus pratensis	16
#	Schoenoplectus lacustris	6
#	Schoenoplectus tabernaemontani	12
#	Schoenus nigricans	7
#	Scilla bifolia	1
#	Scilla forbesii	1
#	Scirpus sylvaticus	8
#	Scleranthus annuus	4
#	Scolymus hispanicus	1
#	Scorzoneroides autumnalis	6
#	Scrophularia auriculata	5
#	Scrophularia nodosa	2
#	Scutellaria altissima	1
#	Scutellaria galericulata	6
#	Scutellaria minor	4
#	Securigera varia	5
#	Selaginella selaginoides	3
#	Senecio erucifolius	5
#	Senecio inaequidens	3
#	Senecio squalidus	0
#	Senecio sylvaticus	3
#	Senecio viscosus	0
#	Senecio vulgaris	11
#	Serratula tinctoria	3
#	Sesleria caerulea	3
#	Setaria italica	83
#	Setaria pumila	12
#	Setaria verticillata	9
#	Setaria viridis	16
#	Sherardia arvensis	9
#	Sidalcea candida	0
#	Sidalcea malviflora	0
#	Silaum silaus	3
#	Silene dioica	22
#	Silene flos-cuculi	7
#	Silene gallica	5
#	Silene latifolia	73
#	Silene noctiflora	5
#	Silene vulgaris	17
#	Sinapis arvensis	10
#	Sison amomum	5
#	Sisymbrium altissimum	2
#	Sisymbrium irio	1
#	Sisymbrium loeselii	1
#	Sisymbrium officinale	12
#	Sisymbrium orientale	3
#	Sisymbrium strictissimum	0
#	Sisymbrium volgense	0
#	Sisyrinchium montanum	10
#	Sium latifolium	3
#	Smyrnium olusatrum	2
#	Solanum chenopodioides	3
#	Solanum dulcamara	16
#	Solanum lycopersicum	138
#	Solanum nigrum	31
#	Solanum physalifolium	2
#	Solanum rostratum	1
#	Solanum sarachoides	0
#	Solanum sisymbriifolium	1
#	Solanum tuberosum	89
#	Soleirolia soleirolii	5
#	Solidago canadensis	18
#	Solidago gigantea	7
#	Solidago graminifolia	7
#	Solidago virgaurea	6
#	Sonchus arvensis	11
#	Sonchus asper	13
#	Sonchus oleraceus	15
#	Sorbus aria	8
#	Sorbus aucuparia	16
#	Sorbus intermedia	7
#	Sorbus latifolia	0
#	Sorbus torminalis	7
#	Sparganium emersum	12
#	Sparganium erectum	6
#	Sparganium natans	7
#	Spartina anglica	6
#	Spergula arvensis	5
#	Spergularia marina	5
#	Spergularia media	3
#	Spergularia rubra	4
#	Spilanthes decumbens	0
#	Spiraea alba	2
#	Spiraea douglasii	0
#	Spiraea salicifolia	5
#	Spiranthes spiralis	3
#	Spirodela polyrhiza	21
#	Stachys annua	0
#	Stachys arvensis	5
#	Stachys byzantina	4
#	Stachys palustris	5
#	Stachys sylvatica	11
#	Stellaria alsine	3
#	Stellaria graminea	6
#	Stellaria holostea	5
#	Stellaria media	16
#	Stellaria neglecta	2
#	Stellaria nemorum	3
#	Stellaria pallida	3
#	Stellaria palustris	4
#	Stratiotes aloides	5
#	Suaeda maritima	13
#	Succisa pratensis	4
#	Symphoricarpos albus	1
#	Symphoricarpos orbiculatus	2
#	Symphytum bulbosum	0
#	Symphytum officinale	10
#	Symphytum orientale	1
#	Syringa vulgaris	9
#	Tagetes minuta	4
#	Tamarix gallica	2
#	Tamus communis	15
#	Tanacetum parthenium	4
#	Tanacetum vulgare	7
#	Taraxacum aequilobum	0
#	Taraxacum alatum	0
#	Taraxacum ancistrolobum	0
#	Taraxacum arenastrum	0
#	Taraxacum argutum	0
#	Taraxacum aurosulum	0
#	Taraxacum boekmanii	0
#	Taraxacum brachyglossum	0
#	Taraxacum bracteatum	0
#	Taraxacum cordatum	0
#	Taraxacum croceiflorum	0
#	Taraxacum dilatatum	0
#	Taraxacum duplidentifrons	0
#	Taraxacum euryphyllum	0
#	Taraxacum exacutum	0
#	Taraxacum expallidiforme	0
#	Taraxacum faeroense	0
#	Taraxacum fasciatum	0
#	Taraxacum fulgidum	0
#	Taraxacum fulviforme	0
#	Taraxacum haematicum	0
#	Taraxacum hamatiforme	0
#	Taraxacum hamatum	0
#	Taraxacum huelphersianum	0
#	Taraxacum insigne	0
#	Taraxacum interveniens	0
#	Taraxacum kernianum	0
#	Taraxacum laciniosifrons	0
#	Taraxacum lacistophyllum	0
#	Taraxacum lamprophyllum	0
#	Taraxacum latisectum	0
#	Taraxacum lingulatum	0
#	Taraxacum maculosum	0
#	Taraxacum marklundii	0
#	Taraxacum nordstedtii	0
#	Taraxacum ochrochlorum	0
#	Taraxacum ostenfeldii	0
#	Taraxacum oxoniense	0
#	Taraxacum pachymerum	0
#	Taraxacum pallescens	0
#	Taraxacum palustre	0
#	Taraxacum pannucium	0
#	Taraxacum polyodon	0
#	Taraxacum proximum	0
#	Taraxacum pseudohamatum	0
#	Taraxacum pseudolarssonii	0
#	Taraxacum quadrans	0
#	Taraxacum richardsianum	0
#	Taraxacum rubicundum	0
#	Taraxacum sagittipotens	0
#	Taraxacum sellandii	0
#	Taraxacum sinuatum	0
#	Taraxacum stenacrum	0
#	Taraxacum stictophyllum	0
#	Taraxacum subcyanolepis	0
#	Taraxacum subhamatum	0
#	Taraxacum sublaeticolor	0
#	Taraxacum subundulatum	0
#	Taraxacum trilobatum	0
#	Taraxacum undulatiflorum	0
#	Taraxacum xanthostigma	0
#	Taxus baccata	24
#	Teesdalia nudicaulis	1
#	Tellima grandiflora	2
#	Teucrium scorodonia	7
#	Thalictrum flavum	3
#	Thelypteris palustris	14
#	Thlaspi arvense	13
#	Thuja plicata	4
#	Thymus polytrichus	8
#	Thymus pulegioides	10
#	Thyselium palustre	0
#	Tilia cordata	6
#	Tilia platyphyllos	6
#	Tolmiea menziesii	1
#	Torilis arvensis	7
#	Torilis japonica	8
#	Torilis nodosa	4
#	Tragopogon porrifolius	2
#	Tragopogon pratensis	11
#	Trichophorum germanicum	0
#	Trifolium arvense	8
#	Trifolium campestre	11
#	Trifolium dubium	9
#	Trifolium fragiferum	4
#	Trifolium hybridum	7
#	Trifolium medium	4
#	Trifolium micranthum	3
#	Trifolium pratense	30
#	Trifolium repens	24
#	Triglochin maritima	21
#	Triglochin palustris	14
#	Tripleurospermum inodorum	4
#	Tripleurospermum maritimum	26
#	Trisetum flavescens	7
#	Triticum aestivum	70
#	Triticum spelta	9
#	Tropaeolum majus	6
#	Tsuga heterophylla	1
#	Tulipa gesneriana	1
#	Turgenia latifolia	1
#	Tussilago farfara	15
#	Typha latifolia	14
#	Ulex europaeus	5
#	Ulmus glabra	12
#	Ulmus minor	8
#	Ulmus plotii	0
#	Ulmus procera	6
#	Urtica dioica	16
#	Urtica urens	8
#	Utricularia minor	5
#	Utricularia vulgaris	2
#	Vaccaria hispanica	3
#	Vaccinium myrtillus	11
#	Vaccinium oxycoccos	4
#	Vaccinium vitis-idaea	24
#	Valeriana dioica	7
#	Valeriana officinalis	16
#	Valeriana pyrenaica	3
#	Valerianella carinata	3
#	Valerianella dentata	4
#	Valerianella eriocarpa	3
#	Valerianella locusta	7
#	Valerianella rimosa	1
#	Verbascum blattaria	0
#	Verbascum nigrum	5
#	Verbascum phlomoides	1
#	Verbascum thapsus	14
#	Verbascum virgatum	3
#	Verbena bonariensis	7
#	Verbena hastata	7
#	Verbena officinalis	13
#	Veronica agrestis	5
#	Veronica anagallis-aquatica	6
#	Veronica arvensis	9
#	Veronica beccabunga	4
#	Veronica catenata	4
#	Veronica chamaedrys	8
#	Veronica filiformis	1
#	Veronica hederifolia	3
#	Veronica longifolia	0
#	Veronica montana	4
#	Veronica officinalis	13
#	Veronica persica	9
#	Veronica polita	2
#	Veronica scutellata	4
#	Veronica serpyllifolia	14
#	Veronica spicata	5
#	Viburnum lantana	9
#	Vicia cracca	27
#	Vicia faba	18
#	Vicia hirsuta	8
#	Vicia lathyroides	2
#	Vicia lutea	4
#	Vicia sativa	35
#	Vicia sepium	6
#	Vicia sylvatica	5
#	Vicia tetrasperma	17
#	Vinca major	1
#	Vinca minor	12
#	Viola arvensis	9
#	Viola canina	5
#	Viola hirta	4
#	Viola odorata	6
#	Viola palustris	5
#	Viola reichenbachiana	5
#	Viola riviniana	10
#	Viola tricolor	4
#	Viscum album	8
#	Vitis vinifera	129
#	Vulpia bromoides	7
#	Vulpia myuros	14
#	Xanthium spinosum	1
#	Xanthium strumarium	3
#	Zannichellia palustris	13
#	Zantedeschia aethiopica	13
#	Zea mays	113
#	Zostera marina	33
#	Zostera noltei	5

total number of accessions fetched: 12636


downloading 12636 records .. processing 1000 accessions per batch

[Fri Oct 21 2016 14:22:46]	batch: 0000001	1000 unique records downloaded	-&gt; eyorks_flora_curat1.0000001.gb	(total: 1000 (7.91 %); remaining: 11636 (92.09 %) -&gt; eyorks_flora_curat1.accessions.txt)
[Fri Oct 21 2016 14:23:41]	batch: 0000002	1000 unique records downloaded	-&gt; eyorks_flora_curat1.0000002.gb	(total: 2000 (15.83 %); remaining: 10636 (84.17 %) -&gt; eyorks_flora_curat1.accessions.txt)
[Fri Oct 21 2016 14:25:26]	batch: 0000003	1000 unique records downloaded	-&gt; eyorks_flora_curat1.0000003.gb	(total: 3000 (23.74 %); remaining: 9636 (76.26 %) -&gt; eyorks_flora_curat1.accessions.txt)
[Fri Oct 21 2016 14:26:55]	batch: 0000004	1000 unique records downloaded	-&gt; eyorks_flora_curat1.0000004.gb	(total: 4000 (31.66 %); remaining: 8636 (68.34 %) -&gt; eyorks_flora_curat1.accessions.txt)
[Fri Oct 21 2016 14:27:46]	batch: 0000005	1000 unique records downloaded	-&gt; eyorks_flora_curat1.0000005.gb	(total: 5000 (39.57 %); remaining: 7636 (60.43 %) -&gt; eyorks_flora_curat1.accessions.txt)
[Fri Oct 21 2016 14:29:08]	batch: 0000006	1000 unique records downloaded	-&gt; eyorks_flora_curat1.0000006.gb	(total: 6000 (47.48 %); remaining: 6636 (52.52 %) -&gt; eyorks_flora_curat1.accessions.txt)
[Fri Oct 21 2016 14:29:56]	batch: 0000007	1000 unique records downloaded	-&gt; eyorks_flora_curat1.0000007.gb	(total: 7000 (55.40 %); remaining: 5636 (44.60 %) -&gt; eyorks_flora_curat1.accessions.txt)
[Fri Oct 21 2016 14:30:57]	batch: 0000008	1000 unique records downloaded	-&gt; eyorks_flora_curat1.0000008.gb	(total: 8000 (63.31 %); remaining: 4636 (36.69 %) -&gt; eyorks_flora_curat1.accessions.txt)
[Fri Oct 21 2016 14:34:16]	batch: 0000009	1000 unique records downloaded	-&gt; eyorks_flora_curat1.0000009.gb	(total: 9000 (71.23 %); remaining: 3636 (28.77 %) -&gt; eyorks_flora_curat1.accessions.txt)
[Fri Oct 21 2016 14:35:50]	batch: 0000010	1000 unique records downloaded	-&gt; eyorks_flora_curat1.0000010.gb	(total: 10000 (79.14 %); remaining: 2636 (20.86 %) -&gt; eyorks_flora_curat1.accessions.txt)
[Fri Oct 21 2016 14:36:48]	batch: 0000011	1000 unique records downloaded	-&gt; eyorks_flora_curat1.0000011.gb	(total: 11000 (87.05 %); remaining: 1636 (12.95 %) -&gt; eyorks_flora_curat1.accessions.txt)
[Fri Oct 21 2016 14:38:24]	batch: 0000012	1000 unique records downloaded	-&gt; eyorks_flora_curat1.0000012.gb	(total: 12000 (94.97 %); remaining: 636 (5.03 %) -&gt; eyorks_flora_curat1.accessions.txt)
[Fri Oct 21 2016 15:10:55]	batch: 0000013	636 unique records downloaded	-&gt; eyorks_flora_curat1.0000013.gb	(total: 12636 (100.00 %); remaining: 0 -&gt; DONE!)

Concatenate all files	-&gt; eyorks_flora_curat1.gb

</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="Remove-bad-sequences">Remove bad sequences<a class="anchor-link" href="#Remove-bad-sequences">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>We have a few blacklisted accession numbers to remove from this file:</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[5]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>head AccessionBlacklist.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>









</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>And we also want to run Sativa, to check for any putative mis-labelled sequences in the reference db. This is a lengthy process so let's do it in a separate notebook. However, we can separately just remove the blacklisted accessions from the local copy of the reference db here.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[8]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="kn">from</span> <span class="nn">Bio</span> <span class="kn">import</span> <span class="n">SeqIO</span>

<span class="n">ids</span><span class="o">=</span><span class="p">[]</span>
<span class="n">temp</span><span class="o">=</span><span class="p">[]</span>
<span class="n">blacklist</span><span class="o">=</span><span class="p">[]</span>
<span class="n">accessions</span> <span class="o">=</span> <span class="nb">open</span><span class="p">(</span><span class="s">&#39;AccessionBlacklist.txt&#39;</span><span class="p">,</span><span class="s">&#39;r&#39;</span><span class="p">)</span>
<span class="n">dropped</span><span class="o">=</span><span class="mi">0</span>
<span class="n">numberofrecords</span><span class="o">=</span><span class="mi">0</span>

<span class="k">for</span> <span class="n">line</span> <span class="ow">in</span> <span class="n">accessions</span><span class="p">:</span>
    <span class="n">ids</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">line</span><span class="o">.</span><span class="n">strip</span><span class="p">())</span>
    

<span class="n">recs_to_keep</span> <span class="o">=</span> <span class="p">{</span><span class="s">&#39;rbcL&#39;</span><span class="p">:</span> <span class="p">[]}</span>
<span class="n">recs_to_drop</span> <span class="o">=</span> <span class="p">{</span><span class="s">&#39;rbcL&#39;</span><span class="p">:</span> <span class="n">ids</span><span class="p">}</span>

<span class="k">print</span> <span class="s">&quot;Blacklist recs to drop: </span><span class="si">%s</span><span class="s">&quot;</span> <span class="o">%</span><span class="k">recs_to_drop</span>[&#39;rbcL&#39;]

<span class="n">records</span> <span class="o">=</span> <span class="n">SeqIO</span><span class="o">.</span><span class="n">parse</span><span class="p">(</span><span class="s">&#39;eyorks_flora_curat1.gb&#39;</span><span class="p">,</span><span class="s">&#39;genbank&#39;</span><span class="p">)</span>

<span class="k">for</span> <span class="n">r</span> <span class="ow">in</span> <span class="n">records</span><span class="p">:</span>       
    <span class="n">numberofrecords</span><span class="o">+=</span><span class="mi">1</span>
    <span class="k">if</span> <span class="ow">not</span> <span class="n">r</span><span class="o">.</span><span class="n">id</span> <span class="ow">in</span> <span class="n">recs_to_drop</span><span class="p">[</span><span class="s">&#39;rbcL&#39;</span><span class="p">]:</span> 
        <span class="n">recs_to_keep</span><span class="p">[</span><span class="s">&#39;rbcL&#39;</span><span class="p">]</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">r</span><span class="o">.</span><span class="n">id</span><span class="p">)</span>
        <span class="n">temp</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">r</span><span class="p">)</span>
    
    <span class="k">else</span><span class="p">:</span>
        <span class="k">print</span> <span class="s">&quot;exclude: </span><span class="si">%s</span><span class="s">&quot;</span> <span class="o">%</span><span class="k">r</span>.id
        <span class="n">dropped</span><span class="o">+=</span><span class="mi">1</span>
        <span class="n">blacklist</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">r</span><span class="o">.</span><span class="n">id</span><span class="p">)</span>
        
<span class="n">out</span> <span class="o">=</span> <span class="nb">open</span><span class="p">(</span><span class="s">&#39;eyorks_flora_curated.gb&#39;</span><span class="p">,</span> <span class="s">&#39;w&#39;</span><span class="p">)</span>
<span class="n">SeqIO</span><span class="o">.</span><span class="n">write</span><span class="p">(</span><span class="n">temp</span><span class="p">,</span> <span class="n">out</span><span class="p">,</span> <span class="s">&#39;genbank&#39;</span><span class="p">)</span>
<span class="n">out</span><span class="o">.</span><span class="n">close</span><span class="p">()</span>

<span class="k">print</span> <span class="s">&quot;Read </span><span class="si">%s</span><span class="s"> records&quot;</span> <span class="o">%</span><span class="k">numberofrecords</span>
<span class="k">print</span> <span class="s">&quot;Dropped blacklist records: </span><span class="si">%s</span><span class="s"> (of </span><span class="si">%s</span><span class="s">)&quot;</span> <span class="o">%</span><span class="p">(</span><span class="n">dropped</span><span class="p">,</span><span class="nb">len</span><span class="p">(</span><span class="n">recs_to_drop</span><span class="p">[</span><span class="s">&#39;rbcL&#39;</span><span class="p">]))</span>
<span class="k">print</span> <span class="s">&quot;Kept: </span><span class="si">%s</span><span class="s">&quot;</span> <span class="o">%</span><span class="k">len</span>(temp)

<span class="k">del</span><span class="p">(</span><span class="n">temp</span><span class="p">)</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>Blacklist recs to drop: [&apos;KU569098.1&apos;, &apos;Z37446.1&apos;, &apos;KP643954.1&apos;, &apos;JN965994.1&apos;, &apos;JQ412340.1&apos;, &apos;HQ623997.1&apos;, &apos;HQ590280.1&apos;, &apos;KJ204422.1&apos;, &apos;JN892973.1&apos;, &apos;JN892974.1&apos;, &apos;KF613060.1&apos;, &apos;Y08501.2&apos;, &apos;HG670306.1&apos;]
exclude: Y08501.2
exclude: KJ204422.1
exclude: HQ590280.1
exclude: KF613060.1
exclude: JN892973.1
exclude: JN892974.1
exclude: HG670306.1
Read 12636 records
Dropped blacklist records: 7 (of 13)
Kept: 12629
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>We can see that this has worked. Some records from the blacklist have not been found in the reference database because they have been removed during previous curation steps; those that have been identified now have been removed.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Update the Querymap.txt file to adjust for the additional level down that we're now in. The REFmap.txt file shouldn't need updating as we've placed reference databases with the same names in the new directory.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[9]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

<span class="k">for </span>sample in <span class="k">$(</span>cat ../Querymap.txt | cut -f 1<span class="k">)</span>
<span class="k">do</span>
<span class="k">    </span><span class="nv">fasta</span><span class="o">=</span><span class="k">$(</span>ls -1 ../../<span class="nv">$sample</span>/<span class="nv">$sample</span><span class="se">\_</span>trimmed.fasta<span class="k">)</span>
    <span class="nb">echo</span> -e <span class="s2">&quot;$sample\tfasta\t$fasta&quot;</span>
<span class="k">done</span> &gt; Querymap.txt

head Querymap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>SA177	fasta	../../SA177/SA177_trimmed.fasta
SB140	fasta	../../SB140/SB140_trimmed.fasta
T19	fasta	../../T19/T19_trimmed.fasta
Lg45	fasta	../../Lg45/Lg45_trimmed.fasta
Q69	fasta	../../Q69/Q69_trimmed.fasta
S72	fasta	../../S72/S72_trimmed.fasta
SC66	fasta	../../SC66/SC66_trimmed.fasta
S31	fasta	../../S31/S31_trimmed.fasta
SC74	fasta	../../SC74/SC74_trimmed.fasta
SA68	fasta	../../SA68/SA68_trimmed.fasta
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[10]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>head REFmap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb	gb
positives_curated.gb	gb
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[7]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre><span class="ansibold ansigreen">AccessionBlacklist.txt</span>*                 <span class="ansibold ansiblue">S92</span>/    <span class="ansibold ansiblue">SB147</span>/
<span class="ansibold ansiblue">Breadth2%</span>/                              <span class="ansibold ansiblue">S96</span>/    <span class="ansibold ansiblue">SB15</span>/
<span class="ansibold ansiblue">Breadth3%</span>/                              <span class="ansibold ansiblue">S97</span>/    <span class="ansibold ansiblue">SB150</span>/
<span class="ansibold ansiblue">Breadth5%</span>/                              <span class="ansibold ansiblue">SA1</span>/    <span class="ansibold ansiblue">SB153</span>/
eyorks_flora_curat1.gb                  <span class="ansibold ansiblue">SA104</span>/  <span class="ansibold ansiblue">SB159</span>/
eyorks_flora_curated.gb                 <span class="ansibold ansiblue">SA105</span>/  <span class="ansibold ansiblue">SB163</span>/
<span class="ansibold ansiblue">GLOBAL</span>/                                 <span class="ansibold ansiblue">SA109</span>/  <span class="ansibold ansiblue">SB166</span>/
<span class="ansibold ansiblue">Lg27</span>/                                   <span class="ansibold ansiblue">SA111</span>/  <span class="ansibold ansiblue">SB170</span>/
<span class="ansibold ansiblue">Lg37</span>/                                   <span class="ansibold ansiblue">SA112</span>/  <span class="ansibold ansiblue">SB171</span>/
<span class="ansibold ansiblue">Lg45</span>/                                   <span class="ansibold ansiblue">SA113</span>/  <span class="ansibold ansiblue">SB172</span>/
<span class="ansibold ansiblue">Lg53</span>/                                   <span class="ansibold ansiblue">SA114</span>/  <span class="ansibold ansiblue">SB180</span>/
<span class="ansibold ansiblue">Lg57</span>/                                   <span class="ansibold ansiblue">SA115</span>/  <span class="ansibold ansiblue">SB182</span>/
<span class="ansibold ansiblue">Lg65</span>/                                   <span class="ansibold ansiblue">SA116</span>/  <span class="ansibold ansiblue">SB183</span>/
<span class="ansibold ansiblue">Lg9</span>/                                    <span class="ansibold ansiblue">SA117</span>/  <span class="ansibold ansiblue">SB186</span>/
loci_counts.txt                         <span class="ansibold ansiblue">SA119</span>/  <span class="ansibold ansiblue">SB194</span>/
loci.csv                                <span class="ansibold ansiblue">SA120</span>/  <span class="ansibold ansiblue">SB195</span>/
log_curat1                              <span class="ansibold ansiblue">SA121</span>/  <span class="ansibold ansiblue">SB197</span>/
MothPollenReflistCurat1_read_stats.csv  <span class="ansibold ansiblue">SA122</span>/  <span class="ansibold ansiblue">SB199</span>/
<span class="ansibold ansiblue">NEG</span>/                                    <span class="ansibold ansiblue">SA124</span>/  <span class="ansibold ansiblue">SB200</span>/
<span class="ansibold ansiblue">O84</span>/                                    <span class="ansibold ansiblue">SA125</span>/  <span class="ansibold ansiblue">SB201</span>/
<span class="ansibold ansiblue">O96</span>/                                    <span class="ansibold ansiblue">SA126</span>/  <span class="ansibold ansiblue">SB204</span>/
<span class="ansibold ansiblue">P18</span>/                                    <span class="ansibold ansiblue">SA128</span>/  <span class="ansibold ansiblue">SB21</span>/
<span class="ansibold ansiblue">P54</span>/                                    <span class="ansibold ansiblue">SA13</span>/   <span class="ansibold ansiblue">SB22</span>/
<span class="ansibold ansiblue">P59</span>/                                    <span class="ansibold ansiblue">SA130</span>/  <span class="ansibold ansiblue">SB23</span>/
<span class="ansibold ansiblue">P60</span>/                                    <span class="ansibold ansiblue">SA132</span>/  <span class="ansibold ansiblue">SB24</span>/
<span class="ansibold ansiblue">P73</span>/                                    <span class="ansibold ansiblue">SA138</span>/  <span class="ansibold ansiblue">SB27</span>/
<span class="ansibold ansiblue">P98</span>/                                    <span class="ansibold ansiblue">SA140</span>/  <span class="ansibold ansiblue">SB29</span>/
<span class="ansibold ansiblue">POS1</span>/                                   <span class="ansibold ansiblue">SA144</span>/  <span class="ansibold ansiblue">SB3</span>/
<span class="ansibold ansiblue">POS2</span>/                                   <span class="ansibold ansiblue">SA147</span>/  <span class="ansibold ansiblue">SB33</span>/
<span class="ansibold ansiblue">POS3</span>/                                   <span class="ansibold ansiblue">SA15</span>/   <span class="ansibold ansiblue">SB34</span>/
<span class="ansibold ansiblue">POS4</span>/                                   <span class="ansibold ansiblue">SA153</span>/  <span class="ansibold ansiblue">SB37</span>/
positives_curated.gb                    <span class="ansibold ansiblue">SA155</span>/  <span class="ansibold ansiblue">SB4</span>/
<span class="ansibold ansiblue">PROC</span>/                                   <span class="ansibold ansiblue">SA156</span>/  <span class="ansibold ansiblue">SB45</span>/
<span class="ansibold ansiblue">Q10</span>/                                    <span class="ansibold ansiblue">SA157</span>/  <span class="ansibold ansiblue">SB48</span>/
<span class="ansibold ansiblue">Q30</span>/                                    <span class="ansibold ansiblue">SA160</span>/  <span class="ansibold ansiblue">SB51</span>/
<span class="ansibold ansiblue">Q43</span>/                                    <span class="ansibold ansiblue">SA162</span>/  <span class="ansibold ansiblue">SB53</span>/
<span class="ansibold ansiblue">Q64</span>/                                    <span class="ansibold ansiblue">SA164</span>/  <span class="ansibold ansiblue">SB59</span>/
<span class="ansibold ansiblue">Q69</span>/                                    <span class="ansibold ansiblue">SA165</span>/  <span class="ansibold ansiblue">SB60</span>/
<span class="ansibold ansiblue">Q83</span>/                                    <span class="ansibold ansiblue">SA166</span>/  <span class="ansibold ansiblue">SB63</span>/
Querymap.txt                            <span class="ansibold ansiblue">SA171</span>/  <span class="ansibold ansiblue">SB64</span>/
<span class="ansibold ansiblue">R1</span>/                                     <span class="ansibold ansiblue">SA172</span>/  <span class="ansibold ansiblue">SB67</span>/
<span class="ansibold ansiblue">R10</span>/                                    <span class="ansibold ansiblue">SA174</span>/  <span class="ansibold ansiblue">SB71</span>/
<span class="ansibold ansiblue">R102</span>/                                   <span class="ansibold ansiblue">SA177</span>/  <span class="ansibold ansiblue">SB73</span>/
<span class="ansibold ansiblue">R13</span>/                                    <span class="ansibold ansiblue">SA180</span>/  <span class="ansibold ansiblue">SB76</span>/
<span class="ansibold ansiblue">R15</span>/                                    <span class="ansibold ansiblue">SA182</span>/  <span class="ansibold ansiblue">SB77</span>/
<span class="ansibold ansiblue">R16</span>/                                    <span class="ansibold ansiblue">SA189</span>/  <span class="ansibold ansiblue">SB8</span>/
<span class="ansibold ansiblue">R20</span>/                                    <span class="ansibold ansiblue">SA19</span>/   <span class="ansibold ansiblue">SB80</span>/
<span class="ansibold ansiblue">R22</span>/                                    <span class="ansibold ansiblue">SA190</span>/  <span class="ansibold ansiblue">SB82</span>/
<span class="ansibold ansiblue">R23</span>/                                    <span class="ansibold ansiblue">SA196</span>/  <span class="ansibold ansiblue">SB83</span>/
<span class="ansibold ansiblue">R27</span>/                                    <span class="ansibold ansiblue">SA198</span>/  <span class="ansibold ansiblue">SB85</span>/
<span class="ansibold ansiblue">R29</span>/                                    <span class="ansibold ansiblue">SA199</span>/  <span class="ansibold ansiblue">SB86</span>/
<span class="ansibold ansiblue">R30</span>/                                    <span class="ansibold ansiblue">SA2</span>/    <span class="ansibold ansiblue">SB87</span>/
<span class="ansibold ansiblue">R31</span>/                                    <span class="ansibold ansiblue">SA202</span>/  <span class="ansibold ansiblue">SB91</span>/
<span class="ansibold ansiblue">R32</span>/                                    <span class="ansibold ansiblue">SA21</span>/   <span class="ansibold ansiblue">SB93</span>/
<span class="ansibold ansiblue">R34</span>/                                    <span class="ansibold ansiblue">SA28</span>/   <span class="ansibold ansiblue">SB99</span>/
<span class="ansibold ansiblue">R36</span>/                                    <span class="ansibold ansiblue">SA31</span>/   <span class="ansibold ansiblue">SC10</span>/
<span class="ansibold ansiblue">R39</span>/                                    <span class="ansibold ansiblue">SA32</span>/   <span class="ansibold ansiblue">SC101</span>/
<span class="ansibold ansiblue">R4</span>/                                     <span class="ansibold ansiblue">SA33</span>/   <span class="ansibold ansiblue">SC106</span>/
<span class="ansibold ansiblue">R40</span>/                                    <span class="ansibold ansiblue">SA36</span>/   <span class="ansibold ansiblue">SC107</span>/
<span class="ansibold ansiblue">R43</span>/                                    <span class="ansibold ansiblue">SA43</span>/   <span class="ansibold ansiblue">SC110</span>/
<span class="ansibold ansiblue">R44</span>/                                    <span class="ansibold ansiblue">SA46</span>/   <span class="ansibold ansiblue">SC113</span>/
<span class="ansibold ansiblue">R45</span>/                                    <span class="ansibold ansiblue">SA47</span>/   <span class="ansibold ansiblue">SC117</span>/
<span class="ansibold ansiblue">R48</span>/                                    <span class="ansibold ansiblue">SA48</span>/   <span class="ansibold ansiblue">SC119</span>/
<span class="ansibold ansiblue">R49</span>/                                    <span class="ansibold ansiblue">SA49</span>/   <span class="ansibold ansiblue">SC122</span>/
<span class="ansibold ansiblue">R59</span>/                                    <span class="ansibold ansiblue">SA5</span>/    <span class="ansibold ansiblue">SC124</span>/
<span class="ansibold ansiblue">R60</span>/                                    <span class="ansibold ansiblue">SA50</span>/   <span class="ansibold ansiblue">SC125</span>/
<span class="ansibold ansiblue">R67</span>/                                    <span class="ansibold ansiblue">SA51</span>/   <span class="ansibold ansiblue">SC141</span>/
<span class="ansibold ansiblue">R68</span>/                                    <span class="ansibold ansiblue">SA53</span>/   <span class="ansibold ansiblue">SC142</span>/
<span class="ansibold ansiblue">R71</span>/                                    <span class="ansibold ansiblue">SA55</span>/   <span class="ansibold ansiblue">SC143</span>/
<span class="ansibold ansiblue">R74</span>/                                    <span class="ansibold ansiblue">SA58</span>/   <span class="ansibold ansiblue">SC157</span>/
<span class="ansibold ansiblue">R8</span>/                                     <span class="ansibold ansiblue">SA62</span>/   <span class="ansibold ansiblue">SC16</span>/
<span class="ansibold ansiblue">R85</span>/                                    <span class="ansibold ansiblue">SA64</span>/   <span class="ansibold ansiblue">SC168</span>/
<span class="ansibold ansiblue">R9</span>/                                     <span class="ansibold ansiblue">SA67</span>/   <span class="ansibold ansiblue">SC178</span>/
<span class="ansibold ansiblue">R90</span>/                                    <span class="ansibold ansiblue">SA68</span>/   <span class="ansibold ansiblue">SC187</span>/
<span class="ansibold ansiblue">R95</span>/                                    <span class="ansibold ansiblue">SA7</span>/    <span class="ansibold ansiblue">SC198</span>/
<span class="ansibold ansiblue">R98</span>/                                    <span class="ansibold ansiblue">SA70</span>/   <span class="ansibold ansiblue">SC19A</span>/
<span class="ansibold ansiblue">R99</span>/                                    <span class="ansibold ansiblue">SA73</span>/   <span class="ansibold ansiblue">SC202</span>/
raw.pkl                                 <span class="ansibold ansiblue">SA75</span>/   <span class="ansibold ansiblue">SC26</span>/
rbcl_synonyms.txt                       <span class="ansibold ansiblue">SA76</span>/   <span class="ansibold ansiblue">SC28</span>/
<span class="ansibold ansigreen">RefListCurat1.txt</span>*                      <span class="ansibold ansiblue">SA81</span>/   <span class="ansibold ansiblue">SC34</span>/
REFmap.txt                              <span class="ansibold ansiblue">SA86</span>/   <span class="ansibold ansiblue">SC35</span>/
<span class="ansibold ansiblue">S100</span>/                                   <span class="ansibold ansiblue">SA87</span>/   <span class="ansibold ansiblue">SC43</span>/
<span class="ansibold ansiblue">S101</span>/                                   <span class="ansibold ansiblue">SA88</span>/   <span class="ansibold ansiblue">SC44</span>/
<span class="ansibold ansiblue">S102</span>/                                   <span class="ansibold ansiblue">SA89</span>/   <span class="ansibold ansiblue">SC48</span>/
<span class="ansibold ansiblue">S11</span>/                                    <span class="ansibold ansiblue">SA9</span>/    <span class="ansibold ansiblue">SC49</span>/
<span class="ansibold ansiblue">S13</span>/                                    <span class="ansibold ansiblue">SA91</span>/   <span class="ansibold ansiblue">SC51</span>/
<span class="ansibold ansiblue">S18</span>/                                    <span class="ansibold ansiblue">SA93</span>/   <span class="ansibold ansiblue">SC51A</span>/
<span class="ansibold ansiblue">S19</span>/                                    <span class="ansibold ansiblue">SA94</span>/   <span class="ansibold ansiblue">SC54A</span>/
<span class="ansibold ansiblue">S24</span>/                                    <span class="ansibold ansiblue">SA95</span>/   <span class="ansibold ansiblue">SC55</span>/
<span class="ansibold ansiblue">S30</span>/                                    <span class="ansibold ansiblue">SA96</span>/   <span class="ansibold ansiblue">SC57</span>/
<span class="ansibold ansiblue">S31</span>/                                    <span class="ansibold ansiblue">SA97</span>/   <span class="ansibold ansiblue">SC60</span>/
<span class="ansibold ansiblue">S33</span>/                                    <span class="ansibold ansiblue">SA98</span>/   <span class="ansibold ansiblue">SC61</span>/
<span class="ansibold ansiblue">S35</span>/                                    <span class="ansibold ansiblue">SB102</span>/  <span class="ansibold ansiblue">SC66</span>/
<span class="ansibold ansiblue">S39</span>/                                    <span class="ansibold ansiblue">SB104</span>/  <span class="ansibold ansiblue">SC7</span>/
<span class="ansibold ansiblue">S43</span>/                                    <span class="ansibold ansiblue">SB106</span>/  <span class="ansibold ansiblue">SC72</span>/
<span class="ansibold ansiblue">S50</span>/                                    <span class="ansibold ansiblue">SB111</span>/  <span class="ansibold ansiblue">SC74</span>/
<span class="ansibold ansiblue">S56</span>/                                    <span class="ansibold ansiblue">SB116</span>/  <span class="ansibold ansiblue">SC77</span>/
<span class="ansibold ansiblue">S59</span>/                                    <span class="ansibold ansiblue">SB119</span>/  <span class="ansibold ansiblue">SC78</span>/
<span class="ansibold ansiblue">S62</span>/                                    <span class="ansibold ansiblue">SB120</span>/  <span class="ansibold ansiblue">SC81</span>/
<span class="ansibold ansiblue">S63</span>/                                    <span class="ansibold ansiblue">SB127</span>/  <span class="ansibold ansiblue">SC82</span>/
<span class="ansibold ansiblue">S65</span>/                                    <span class="ansibold ansiblue">SB128</span>/  <span class="ansibold ansiblue">SC86A</span>/
<span class="ansibold ansiblue">S67</span>/                                    <span class="ansibold ansiblue">SB129</span>/  <span class="ansibold ansiblue">SC88</span>/
<span class="ansibold ansiblue">S7</span>/                                     <span class="ansibold ansiblue">SB13</span>/   <span class="ansibold ansiblue">SC9</span>/
<span class="ansibold ansiblue">S72</span>/                                    <span class="ansibold ansiblue">SB131</span>/  <span class="ansibold ansiblue">SC93</span>/
<span class="ansibold ansiblue">S77</span>/                                    <span class="ansibold ansiblue">SB135</span>/  species_vs_locus_raw.csv
<span class="ansibold ansiblue">S79</span>/                                    <span class="ansibold ansiblue">SB136</span>/  <span class="ansibold ansiblue">T19</span>/
<span class="ansibold ansiblue">S8</span>/                                     <span class="ansibold ansiblue">SB138</span>/  <span class="ansibold ansiblue">T21</span>/
<span class="ansibold ansiblue">S80</span>/                                    <span class="ansibold ansiblue">SB139</span>/  <span class="ansibold ansiblue">T32</span>/
<span class="ansibold ansiblue">S81</span>/                                    <span class="ansibold ansiblue">SB140</span>/  <span class="ansibold ansiblue">T49</span>/
<span class="ansibold ansiblue">S83</span>/                                    <span class="ansibold ansiblue">SB141</span>/  <span class="ansibold ansiblue">T68</span>/
<span class="ansibold ansiblue">S9</span>/                                     <span class="ansibold ansiblue">SB142</span>/  <span class="ansibold ansiblue">T78</span>/
<span class="ansibold ansiblue">S90</span>/                                    <span class="ansibold ansiblue">SB146</span>/  target_locus.csv
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>We can see that everything is now set up correctly to run a BLAST against our new, curated reference db</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[19]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-Q Querymap.txt <span class="se">\</span>
--cluster <span class="se">\</span>
--clust_match 1 <span class="se">\</span>
--clust_cov 5 <span class="se">\</span>
--blast <span class="se">\</span>
--REFlist REFmap.txt <span class="se">\</span>
--min_ident 0.95 <span class="se">\</span>
-n 5 <span class="se">\</span>
-v <span class="se">\</span>
-@ callumjmacgregor@gmail.com <span class="se">\</span>
-o MothPollenReflistCurat1 &gt; log_curat1
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[20]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash
blastn -query /home/working/AgainstRefDB/Curat1/GLOBAL/global_centroids.fasta <span class="se">\</span>
-db /home/working/AgainstRefDB/Curat1/GLOBAL/BLAST_0.95/marker_blast_db <span class="se">\</span>
-out /home/working/AgainstRefDB/Curat1/GLOBAL/BLAST_0.95/global_blastn.out.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="Restrict-width-of-BLAST">Restrict width of BLAST<a class="anchor-link" href="#Restrict-width-of-BLAST">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Christoph has now implemented some settings in metaBEAT that allow you to move away from some of the default BLAST settings, which didn't seem to be appropriate for my data. Let's now try using that implementation to see what happens!</p>

</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>First let's just try changing the breadth of the BLAST search (which by default was 10% of the bitscore of the top hit) to lower values: 5%, 3%, 2%.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="5%">5%<a class="anchor-link" href="#5%">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[3]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cd</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">AgainstRefDB</span><span class="o">/</span><span class="n">Curat1</span><span class="o">/</span><span class="n">Breadth5</span><span class="o">%/</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>/home/working/AgainstRefDB/Curat1/Breadth5%
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>The Querymap and the REFmap can be adjusted from the previous BLAST.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[9]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

<span class="k">for </span>sample in <span class="k">$(</span>cat ../Querymap.txt | cut -f 1<span class="k">)</span>
<span class="k">do</span>
<span class="k">    </span><span class="nv">fasta</span><span class="o">=</span><span class="k">$(</span>ls -1 ../../../<span class="nv">$sample</span>/<span class="nv">$sample</span><span class="se">\_</span>trimmed.fasta<span class="k">)</span>
    <span class="nb">echo</span> -e <span class="s2">&quot;$sample\tfasta\t$fasta&quot;</span>
<span class="k">done</span> &gt; Querymap.txt

head Querymap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>SA177	fasta	../../../SA177/SA177_trimmed.fasta
SB140	fasta	../../../SB140/SB140_trimmed.fasta
T19	fasta	../../../T19/T19_trimmed.fasta
Lg45	fasta	../../../Lg45/Lg45_trimmed.fasta
Q69	fasta	../../../Q69/Q69_trimmed.fasta
S72	fasta	../../../S72/S72_trimmed.fasta
SC66	fasta	../../../SC66/SC66_trimmed.fasta
S31	fasta	../../../S31/S31_trimmed.fasta
SC74	fasta	../../../SC74/SC74_trimmed.fasta
SA68	fasta	../../../SA68/SA68_trimmed.fasta
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[17]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>cp ../REFmap.txt .

<span class="o">!</span>head REFmap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb	gb
positives_curated.gb	gb
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[18]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../</span><span class="n">eyorks_flora_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[19]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../</span><span class="n">positives_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[20]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb  positives_curated.gb  REFmap.txt
log_breadth5%            Querymap.txt
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[21]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-Q Querymap.txt <span class="se">\</span>
--cluster <span class="se">\</span>
--clust_match 1 <span class="se">\</span>
--clust_cov 5 <span class="se">\</span>
--blast <span class="se">\</span>
--bitscore_skim_LCA 0.05 <span class="se">\</span>
--REFlist REFmap.txt <span class="se">\</span>
--min_ident 0.95 <span class="se">\</span>
-n 5 <span class="se">\</span>
-v <span class="se">\</span>
-@ callumjmacgregor@gmail.com <span class="se">\</span>
-o MothPollenReflistCurat1Breadth5 &gt; log_breadth5%
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[22]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash
blastn -query /home/working/AgainstRefDB/Curat1/Breadth5%/GLOBAL/global_centroids.fasta <span class="se">\</span>
-db /home/working/AgainstRefDB/Curat1/Breadth5%/GLOBAL/BLAST_0.95/marker_blast_db <span class="se">\</span>
-out /home/working/AgainstRefDB/Curat1/Breadth5%/GLOBAL/BLAST_0.95/global_blastn.out.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="3%">3%<a class="anchor-link" href="#3%">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[42]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cd</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">AgainstRefDB</span><span class="o">/</span><span class="n">Curat1</span><span class="o">/</span><span class="n">Breadth3</span><span class="o">%/</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>/home/working/AgainstRefDB/Curat1/Breadth3%
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[43]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

<span class="k">for </span>sample in <span class="k">$(</span>cat ../Querymap.txt | cut -f 1<span class="k">)</span>
<span class="k">do</span>
<span class="k">    </span><span class="nv">fasta</span><span class="o">=</span><span class="k">$(</span>ls -1 ../../../<span class="nv">$sample</span>/<span class="nv">$sample</span><span class="se">\_</span>trimmed.fasta<span class="k">)</span>
    <span class="nb">echo</span> -e <span class="s2">&quot;$sample\tfasta\t$fasta&quot;</span>
<span class="k">done</span> &gt; Querymap.txt

head Querymap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>SA177	fasta	../../../SA177/SA177_trimmed.fasta
SB140	fasta	../../../SB140/SB140_trimmed.fasta
T19	fasta	../../../T19/T19_trimmed.fasta
Lg45	fasta	../../../Lg45/Lg45_trimmed.fasta
Q69	fasta	../../../Q69/Q69_trimmed.fasta
S72	fasta	../../../S72/S72_trimmed.fasta
SC66	fasta	../../../SC66/SC66_trimmed.fasta
S31	fasta	../../../S31/S31_trimmed.fasta
SC74	fasta	../../../SC74/SC74_trimmed.fasta
SA68	fasta	../../../SA68/SA68_trimmed.fasta
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[44]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>cp ../REFmap.txt .

<span class="o">!</span>head REFmap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb	gb
positives_curated.gb	gb
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[45]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../</span><span class="n">eyorks_flora_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[46]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../</span><span class="n">positives_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[47]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb                         <span class="ansibold ansiblue">S33</span>/    <span class="ansibold ansiblue">SA46</span>/   <span class="ansibold ansiblue">SB34</span>/
<span class="ansibold ansiblue">GLOBAL</span>/                                         <span class="ansibold ansiblue">S35</span>/    <span class="ansibold ansiblue">SA47</span>/   <span class="ansibold ansiblue">SB37</span>/
<span class="ansibold ansiblue">Lg27</span>/                                           <span class="ansibold ansiblue">S39</span>/    <span class="ansibold ansiblue">SA48</span>/   <span class="ansibold ansiblue">SB4</span>/
<span class="ansibold ansiblue">Lg37</span>/                                           <span class="ansibold ansiblue">S43</span>/    <span class="ansibold ansiblue">SA49</span>/   <span class="ansibold ansiblue">SB45</span>/
<span class="ansibold ansiblue">Lg45</span>/                                           <span class="ansibold ansiblue">S50</span>/    <span class="ansibold ansiblue">SA5</span>/    <span class="ansibold ansiblue">SB48</span>/
<span class="ansibold ansiblue">Lg53</span>/                                           <span class="ansibold ansiblue">S56</span>/    <span class="ansibold ansiblue">SA50</span>/   <span class="ansibold ansiblue">SB51</span>/
<span class="ansibold ansiblue">Lg57</span>/                                           <span class="ansibold ansiblue">S59</span>/    <span class="ansibold ansiblue">SA51</span>/   <span class="ansibold ansiblue">SB53</span>/
<span class="ansibold ansiblue">Lg65</span>/                                           <span class="ansibold ansiblue">S62</span>/    <span class="ansibold ansiblue">SA53</span>/   <span class="ansibold ansiblue">SB59</span>/
<span class="ansibold ansiblue">Lg9</span>/                                            <span class="ansibold ansiblue">S63</span>/    <span class="ansibold ansiblue">SA55</span>/   <span class="ansibold ansiblue">SB60</span>/
log_breadth3%                                   <span class="ansibold ansiblue">S65</span>/    <span class="ansibold ansiblue">SA58</span>/   <span class="ansibold ansiblue">SB63</span>/
MothPollenReflistCurat1Breadth5_read_stats.csv  <span class="ansibold ansiblue">S67</span>/    <span class="ansibold ansiblue">SA62</span>/   <span class="ansibold ansiblue">SB64</span>/
<span class="ansibold ansiblue">NEG</span>/                                            <span class="ansibold ansiblue">S7</span>/     <span class="ansibold ansiblue">SA64</span>/   <span class="ansibold ansiblue">SB67</span>/
<span class="ansibold ansiblue">O84</span>/                                            <span class="ansibold ansiblue">S72</span>/    <span class="ansibold ansiblue">SA67</span>/   <span class="ansibold ansiblue">SB71</span>/
<span class="ansibold ansiblue">O96</span>/                                            <span class="ansibold ansiblue">S77</span>/    <span class="ansibold ansiblue">SA68</span>/   <span class="ansibold ansiblue">SB73</span>/
<span class="ansibold ansiblue">P18</span>/                                            <span class="ansibold ansiblue">S79</span>/    <span class="ansibold ansiblue">SA7</span>/    <span class="ansibold ansiblue">SB76</span>/
<span class="ansibold ansiblue">P54</span>/                                            <span class="ansibold ansiblue">S8</span>/     <span class="ansibold ansiblue">SA70</span>/   <span class="ansibold ansiblue">SB77</span>/
<span class="ansibold ansiblue">P59</span>/                                            <span class="ansibold ansiblue">S80</span>/    <span class="ansibold ansiblue">SA73</span>/   <span class="ansibold ansiblue">SB8</span>/
<span class="ansibold ansiblue">P60</span>/                                            <span class="ansibold ansiblue">S81</span>/    <span class="ansibold ansiblue">SA75</span>/   <span class="ansibold ansiblue">SB80</span>/
<span class="ansibold ansiblue">P73</span>/                                            <span class="ansibold ansiblue">S83</span>/    <span class="ansibold ansiblue">SA76</span>/   <span class="ansibold ansiblue">SB82</span>/
<span class="ansibold ansiblue">P98</span>/                                            <span class="ansibold ansiblue">S9</span>/     <span class="ansibold ansiblue">SA81</span>/   <span class="ansibold ansiblue">SB83</span>/
<span class="ansibold ansiblue">POS1</span>/                                           <span class="ansibold ansiblue">S90</span>/    <span class="ansibold ansiblue">SA86</span>/   <span class="ansibold ansiblue">SB85</span>/
<span class="ansibold ansiblue">POS2</span>/                                           <span class="ansibold ansiblue">S92</span>/    <span class="ansibold ansiblue">SA87</span>/   <span class="ansibold ansiblue">SB86</span>/
<span class="ansibold ansiblue">POS3</span>/                                           <span class="ansibold ansiblue">S96</span>/    <span class="ansibold ansiblue">SA88</span>/   <span class="ansibold ansiblue">SB87</span>/
<span class="ansibold ansiblue">POS4</span>/                                           <span class="ansibold ansiblue">S97</span>/    <span class="ansibold ansiblue">SA89</span>/   <span class="ansibold ansiblue">SB91</span>/
positives_curated.gb                            <span class="ansibold ansiblue">SA1</span>/    <span class="ansibold ansiblue">SA9</span>/    <span class="ansibold ansiblue">SB93</span>/
<span class="ansibold ansiblue">PROC</span>/                                           <span class="ansibold ansiblue">SA104</span>/  <span class="ansibold ansiblue">SA91</span>/   <span class="ansibold ansiblue">SB99</span>/
<span class="ansibold ansiblue">Q10</span>/                                            <span class="ansibold ansiblue">SA105</span>/  <span class="ansibold ansiblue">SA93</span>/   <span class="ansibold ansiblue">SC10</span>/
<span class="ansibold ansiblue">Q30</span>/                                            <span class="ansibold ansiblue">SA109</span>/  <span class="ansibold ansiblue">SA94</span>/   <span class="ansibold ansiblue">SC101</span>/
<span class="ansibold ansiblue">Q43</span>/                                            <span class="ansibold ansiblue">SA111</span>/  <span class="ansibold ansiblue">SA95</span>/   <span class="ansibold ansiblue">SC106</span>/
<span class="ansibold ansiblue">Q64</span>/                                            <span class="ansibold ansiblue">SA112</span>/  <span class="ansibold ansiblue">SA96</span>/   <span class="ansibold ansiblue">SC107</span>/
<span class="ansibold ansiblue">Q69</span>/                                            <span class="ansibold ansiblue">SA113</span>/  <span class="ansibold ansiblue">SA97</span>/   <span class="ansibold ansiblue">SC110</span>/
<span class="ansibold ansiblue">Q83</span>/                                            <span class="ansibold ansiblue">SA114</span>/  <span class="ansibold ansiblue">SA98</span>/   <span class="ansibold ansiblue">SC113</span>/
Querymap.txt                                    <span class="ansibold ansiblue">SA115</span>/  <span class="ansibold ansiblue">SB102</span>/  <span class="ansibold ansiblue">SC117</span>/
<span class="ansibold ansiblue">R1</span>/                                             <span class="ansibold ansiblue">SA116</span>/  <span class="ansibold ansiblue">SB104</span>/  <span class="ansibold ansiblue">SC119</span>/
<span class="ansibold ansiblue">R10</span>/                                            <span class="ansibold ansiblue">SA117</span>/  <span class="ansibold ansiblue">SB106</span>/  <span class="ansibold ansiblue">SC122</span>/
<span class="ansibold ansiblue">R102</span>/                                           <span class="ansibold ansiblue">SA119</span>/  <span class="ansibold ansiblue">SB111</span>/  <span class="ansibold ansiblue">SC124</span>/
<span class="ansibold ansiblue">R13</span>/                                            <span class="ansibold ansiblue">SA120</span>/  <span class="ansibold ansiblue">SB116</span>/  <span class="ansibold ansiblue">SC125</span>/
<span class="ansibold ansiblue">R15</span>/                                            <span class="ansibold ansiblue">SA121</span>/  <span class="ansibold ansiblue">SB119</span>/  <span class="ansibold ansiblue">SC141</span>/
<span class="ansibold ansiblue">R16</span>/                                            <span class="ansibold ansiblue">SA122</span>/  <span class="ansibold ansiblue">SB120</span>/  <span class="ansibold ansiblue">SC142</span>/
<span class="ansibold ansiblue">R20</span>/                                            <span class="ansibold ansiblue">SA124</span>/  <span class="ansibold ansiblue">SB127</span>/  <span class="ansibold ansiblue">SC143</span>/
<span class="ansibold ansiblue">R22</span>/                                            <span class="ansibold ansiblue">SA125</span>/  <span class="ansibold ansiblue">SB128</span>/  <span class="ansibold ansiblue">SC157</span>/
<span class="ansibold ansiblue">R23</span>/                                            <span class="ansibold ansiblue">SA126</span>/  <span class="ansibold ansiblue">SB129</span>/  <span class="ansibold ansiblue">SC16</span>/
<span class="ansibold ansiblue">R27</span>/                                            <span class="ansibold ansiblue">SA128</span>/  <span class="ansibold ansiblue">SB13</span>/   <span class="ansibold ansiblue">SC168</span>/
<span class="ansibold ansiblue">R29</span>/                                            <span class="ansibold ansiblue">SA13</span>/   <span class="ansibold ansiblue">SB131</span>/  <span class="ansibold ansiblue">SC178</span>/
<span class="ansibold ansiblue">R30</span>/                                            <span class="ansibold ansiblue">SA130</span>/  <span class="ansibold ansiblue">SB135</span>/  <span class="ansibold ansiblue">SC187</span>/
<span class="ansibold ansiblue">R31</span>/                                            <span class="ansibold ansiblue">SA132</span>/  <span class="ansibold ansiblue">SB136</span>/  <span class="ansibold ansiblue">SC198</span>/
<span class="ansibold ansiblue">R32</span>/                                            <span class="ansibold ansiblue">SA138</span>/  <span class="ansibold ansiblue">SB138</span>/  <span class="ansibold ansiblue">SC19A</span>/
<span class="ansibold ansiblue">R34</span>/                                            <span class="ansibold ansiblue">SA140</span>/  <span class="ansibold ansiblue">SB139</span>/  <span class="ansibold ansiblue">SC202</span>/
<span class="ansibold ansiblue">R36</span>/                                            <span class="ansibold ansiblue">SA144</span>/  <span class="ansibold ansiblue">SB140</span>/  <span class="ansibold ansiblue">SC26</span>/
<span class="ansibold ansiblue">R39</span>/                                            <span class="ansibold ansiblue">SA147</span>/  <span class="ansibold ansiblue">SB141</span>/  <span class="ansibold ansiblue">SC28</span>/
<span class="ansibold ansiblue">R4</span>/                                             <span class="ansibold ansiblue">SA15</span>/   <span class="ansibold ansiblue">SB142</span>/  <span class="ansibold ansiblue">SC34</span>/
<span class="ansibold ansiblue">R40</span>/                                            <span class="ansibold ansiblue">SA153</span>/  <span class="ansibold ansiblue">SB146</span>/  <span class="ansibold ansiblue">SC35</span>/
<span class="ansibold ansiblue">R43</span>/                                            <span class="ansibold ansiblue">SA155</span>/  <span class="ansibold ansiblue">SB147</span>/  <span class="ansibold ansiblue">SC43</span>/
<span class="ansibold ansiblue">R44</span>/                                            <span class="ansibold ansiblue">SA156</span>/  <span class="ansibold ansiblue">SB15</span>/   <span class="ansibold ansiblue">SC44</span>/
<span class="ansibold ansiblue">R45</span>/                                            <span class="ansibold ansiblue">SA157</span>/  <span class="ansibold ansiblue">SB150</span>/  <span class="ansibold ansiblue">SC48</span>/
<span class="ansibold ansiblue">R48</span>/                                            <span class="ansibold ansiblue">SA160</span>/  <span class="ansibold ansiblue">SB153</span>/  <span class="ansibold ansiblue">SC49</span>/
<span class="ansibold ansiblue">R49</span>/                                            <span class="ansibold ansiblue">SA162</span>/  <span class="ansibold ansiblue">SB159</span>/  <span class="ansibold ansiblue">SC51</span>/
<span class="ansibold ansiblue">R59</span>/                                            <span class="ansibold ansiblue">SA164</span>/  <span class="ansibold ansiblue">SB163</span>/  <span class="ansibold ansiblue">SC51A</span>/
<span class="ansibold ansiblue">R60</span>/                                            <span class="ansibold ansiblue">SA165</span>/  <span class="ansibold ansiblue">SB166</span>/  <span class="ansibold ansiblue">SC54A</span>/
<span class="ansibold ansiblue">R67</span>/                                            <span class="ansibold ansiblue">SA166</span>/  <span class="ansibold ansiblue">SB170</span>/  <span class="ansibold ansiblue">SC55</span>/
<span class="ansibold ansiblue">R68</span>/                                            <span class="ansibold ansiblue">SA171</span>/  <span class="ansibold ansiblue">SB171</span>/  <span class="ansibold ansiblue">SC57</span>/
<span class="ansibold ansiblue">R71</span>/                                            <span class="ansibold ansiblue">SA172</span>/  <span class="ansibold ansiblue">SB172</span>/  <span class="ansibold ansiblue">SC60</span>/
<span class="ansibold ansiblue">R74</span>/                                            <span class="ansibold ansiblue">SA174</span>/  <span class="ansibold ansiblue">SB180</span>/  <span class="ansibold ansiblue">SC61</span>/
<span class="ansibold ansiblue">R8</span>/                                             <span class="ansibold ansiblue">SA177</span>/  <span class="ansibold ansiblue">SB182</span>/  <span class="ansibold ansiblue">SC66</span>/
<span class="ansibold ansiblue">R85</span>/                                            <span class="ansibold ansiblue">SA180</span>/  <span class="ansibold ansiblue">SB183</span>/  <span class="ansibold ansiblue">SC7</span>/
<span class="ansibold ansiblue">R9</span>/                                             <span class="ansibold ansiblue">SA182</span>/  <span class="ansibold ansiblue">SB186</span>/  <span class="ansibold ansiblue">SC72</span>/
<span class="ansibold ansiblue">R90</span>/                                            <span class="ansibold ansiblue">SA189</span>/  <span class="ansibold ansiblue">SB194</span>/  <span class="ansibold ansiblue">SC74</span>/
<span class="ansibold ansiblue">R95</span>/                                            <span class="ansibold ansiblue">SA19</span>/   <span class="ansibold ansiblue">SB195</span>/  <span class="ansibold ansiblue">SC77</span>/
<span class="ansibold ansiblue">R98</span>/                                            <span class="ansibold ansiblue">SA190</span>/  <span class="ansibold ansiblue">SB197</span>/  <span class="ansibold ansiblue">SC78</span>/
<span class="ansibold ansiblue">R99</span>/                                            <span class="ansibold ansiblue">SA196</span>/  <span class="ansibold ansiblue">SB199</span>/  <span class="ansibold ansiblue">SC81</span>/
REFmap.txt                                      <span class="ansibold ansiblue">SA198</span>/  <span class="ansibold ansiblue">SB200</span>/  <span class="ansibold ansiblue">SC82</span>/
<span class="ansibold ansiblue">S100</span>/                                           <span class="ansibold ansiblue">SA199</span>/  <span class="ansibold ansiblue">SB201</span>/  <span class="ansibold ansiblue">SC86A</span>/
<span class="ansibold ansiblue">S101</span>/                                           <span class="ansibold ansiblue">SA2</span>/    <span class="ansibold ansiblue">SB204</span>/  <span class="ansibold ansiblue">SC88</span>/
<span class="ansibold ansiblue">S102</span>/                                           <span class="ansibold ansiblue">SA202</span>/  <span class="ansibold ansiblue">SB21</span>/   <span class="ansibold ansiblue">SC9</span>/
<span class="ansibold ansiblue">S11</span>/                                            <span class="ansibold ansiblue">SA21</span>/   <span class="ansibold ansiblue">SB22</span>/   <span class="ansibold ansiblue">SC93</span>/
<span class="ansibold ansiblue">S13</span>/                                            <span class="ansibold ansiblue">SA28</span>/   <span class="ansibold ansiblue">SB23</span>/   <span class="ansibold ansiblue">T19</span>/
<span class="ansibold ansiblue">S18</span>/                                            <span class="ansibold ansiblue">SA31</span>/   <span class="ansibold ansiblue">SB24</span>/   <span class="ansibold ansiblue">T21</span>/
<span class="ansibold ansiblue">S19</span>/                                            <span class="ansibold ansiblue">SA32</span>/   <span class="ansibold ansiblue">SB27</span>/   <span class="ansibold ansiblue">T32</span>/
<span class="ansibold ansiblue">S24</span>/                                            <span class="ansibold ansiblue">SA33</span>/   <span class="ansibold ansiblue">SB29</span>/   <span class="ansibold ansiblue">T49</span>/
<span class="ansibold ansiblue">S30</span>/                                            <span class="ansibold ansiblue">SA36</span>/   <span class="ansibold ansiblue">SB3</span>/    <span class="ansibold ansiblue">T68</span>/
<span class="ansibold ansiblue">S31</span>/                                            <span class="ansibold ansiblue">SA43</span>/   <span class="ansibold ansiblue">SB33</span>/   <span class="ansibold ansiblue">T78</span>/
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[49]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-Q Querymap.txt <span class="se">\</span>
--cluster <span class="se">\</span>
--clust_match 1 <span class="se">\</span>
--clust_cov 5 <span class="se">\</span>
--blast <span class="se">\</span>
--bitscore_skim_LCA 0.03 <span class="se">\</span>
--REFlist REFmap.txt <span class="se">\</span>
--min_ident 0.95 <span class="se">\</span>
-n 5 <span class="se">\</span>
-v <span class="se">\</span>
-@ callumjmacgregor@gmail.com <span class="se">\</span>
-o MothPollenReflistCurat1Breadth3 &gt; log_breadth3%
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[50]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash
blastn -query /home/working/AgainstRefDB/Curat1/Breadth3%/GLOBAL/global_centroids.fasta <span class="se">\</span>
-db /home/working/AgainstRefDB/Curat1/Breadth3%/GLOBAL/BLAST_0.95/marker_blast_db <span class="se">\</span>
-out /home/working/AgainstRefDB/Curat1/Breadth3%/GLOBAL/BLAST_0.95/global_blastn.out.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="2%">2%<a class="anchor-link" href="#2%">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[51]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cd</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">AgainstRefDB</span><span class="o">/</span><span class="n">Curat1</span><span class="o">/</span><span class="n">Breadth2</span><span class="o">%/</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>/home/working/AgainstRefDB/Curat1/Breadth2%
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[52]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

<span class="k">for </span>sample in <span class="k">$(</span>cat ../Querymap.txt | cut -f 1<span class="k">)</span>
<span class="k">do</span>
<span class="k">    </span><span class="nv">fasta</span><span class="o">=</span><span class="k">$(</span>ls -1 ../../../<span class="nv">$sample</span>/<span class="nv">$sample</span><span class="se">\_</span>trimmed.fasta<span class="k">)</span>
    <span class="nb">echo</span> -e <span class="s2">&quot;$sample\tfasta\t$fasta&quot;</span>
<span class="k">done</span> &gt; Querymap.txt

head Querymap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>SA177	fasta	../../../SA177/SA177_trimmed.fasta
SB140	fasta	../../../SB140/SB140_trimmed.fasta
T19	fasta	../../../T19/T19_trimmed.fasta
Lg45	fasta	../../../Lg45/Lg45_trimmed.fasta
Q69	fasta	../../../Q69/Q69_trimmed.fasta
S72	fasta	../../../S72/S72_trimmed.fasta
SC66	fasta	../../../SC66/SC66_trimmed.fasta
S31	fasta	../../../S31/S31_trimmed.fasta
SC74	fasta	../../../SC74/SC74_trimmed.fasta
SA68	fasta	../../../SA68/SA68_trimmed.fasta
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[53]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>cp ../REFmap.txt .

<span class="o">!</span>head REFmap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb	gb
positives_curated.gb	gb
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[54]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../</span><span class="n">eyorks_flora_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[55]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../</span><span class="n">positives_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[56]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb                         <span class="ansibold ansiblue">S33</span>/    <span class="ansibold ansiblue">SA46</span>/   <span class="ansibold ansiblue">SB34</span>/
<span class="ansibold ansiblue">GLOBAL</span>/                                         <span class="ansibold ansiblue">S35</span>/    <span class="ansibold ansiblue">SA47</span>/   <span class="ansibold ansiblue">SB37</span>/
<span class="ansibold ansiblue">Lg27</span>/                                           <span class="ansibold ansiblue">S39</span>/    <span class="ansibold ansiblue">SA48</span>/   <span class="ansibold ansiblue">SB4</span>/
<span class="ansibold ansiblue">Lg37</span>/                                           <span class="ansibold ansiblue">S43</span>/    <span class="ansibold ansiblue">SA49</span>/   <span class="ansibold ansiblue">SB45</span>/
<span class="ansibold ansiblue">Lg45</span>/                                           <span class="ansibold ansiblue">S50</span>/    <span class="ansibold ansiblue">SA5</span>/    <span class="ansibold ansiblue">SB48</span>/
<span class="ansibold ansiblue">Lg53</span>/                                           <span class="ansibold ansiblue">S56</span>/    <span class="ansibold ansiblue">SA50</span>/   <span class="ansibold ansiblue">SB51</span>/
<span class="ansibold ansiblue">Lg57</span>/                                           <span class="ansibold ansiblue">S59</span>/    <span class="ansibold ansiblue">SA51</span>/   <span class="ansibold ansiblue">SB53</span>/
<span class="ansibold ansiblue">Lg65</span>/                                           <span class="ansibold ansiblue">S62</span>/    <span class="ansibold ansiblue">SA53</span>/   <span class="ansibold ansiblue">SB59</span>/
<span class="ansibold ansiblue">Lg9</span>/                                            <span class="ansibold ansiblue">S63</span>/    <span class="ansibold ansiblue">SA55</span>/   <span class="ansibold ansiblue">SB60</span>/
log_breadth2%                                   <span class="ansibold ansiblue">S65</span>/    <span class="ansibold ansiblue">SA58</span>/   <span class="ansibold ansiblue">SB63</span>/
MothPollenReflistCurat1Breadth5_read_stats.csv  <span class="ansibold ansiblue">S67</span>/    <span class="ansibold ansiblue">SA62</span>/   <span class="ansibold ansiblue">SB64</span>/
<span class="ansibold ansiblue">NEG</span>/                                            <span class="ansibold ansiblue">S7</span>/     <span class="ansibold ansiblue">SA64</span>/   <span class="ansibold ansiblue">SB67</span>/
<span class="ansibold ansiblue">O84</span>/                                            <span class="ansibold ansiblue">S72</span>/    <span class="ansibold ansiblue">SA67</span>/   <span class="ansibold ansiblue">SB71</span>/
<span class="ansibold ansiblue">O96</span>/                                            <span class="ansibold ansiblue">S77</span>/    <span class="ansibold ansiblue">SA68</span>/   <span class="ansibold ansiblue">SB73</span>/
<span class="ansibold ansiblue">P18</span>/                                            <span class="ansibold ansiblue">S79</span>/    <span class="ansibold ansiblue">SA7</span>/    <span class="ansibold ansiblue">SB76</span>/
<span class="ansibold ansiblue">P54</span>/                                            <span class="ansibold ansiblue">S8</span>/     <span class="ansibold ansiblue">SA70</span>/   <span class="ansibold ansiblue">SB77</span>/
<span class="ansibold ansiblue">P59</span>/                                            <span class="ansibold ansiblue">S80</span>/    <span class="ansibold ansiblue">SA73</span>/   <span class="ansibold ansiblue">SB8</span>/
<span class="ansibold ansiblue">P60</span>/                                            <span class="ansibold ansiblue">S81</span>/    <span class="ansibold ansiblue">SA75</span>/   <span class="ansibold ansiblue">SB80</span>/
<span class="ansibold ansiblue">P73</span>/                                            <span class="ansibold ansiblue">S83</span>/    <span class="ansibold ansiblue">SA76</span>/   <span class="ansibold ansiblue">SB82</span>/
<span class="ansibold ansiblue">P98</span>/                                            <span class="ansibold ansiblue">S9</span>/     <span class="ansibold ansiblue">SA81</span>/   <span class="ansibold ansiblue">SB83</span>/
<span class="ansibold ansiblue">POS1</span>/                                           <span class="ansibold ansiblue">S90</span>/    <span class="ansibold ansiblue">SA86</span>/   <span class="ansibold ansiblue">SB85</span>/
<span class="ansibold ansiblue">POS2</span>/                                           <span class="ansibold ansiblue">S92</span>/    <span class="ansibold ansiblue">SA87</span>/   <span class="ansibold ansiblue">SB86</span>/
<span class="ansibold ansiblue">POS3</span>/                                           <span class="ansibold ansiblue">S96</span>/    <span class="ansibold ansiblue">SA88</span>/   <span class="ansibold ansiblue">SB87</span>/
<span class="ansibold ansiblue">POS4</span>/                                           <span class="ansibold ansiblue">S97</span>/    <span class="ansibold ansiblue">SA89</span>/   <span class="ansibold ansiblue">SB91</span>/
positives_curated.gb                            <span class="ansibold ansiblue">SA1</span>/    <span class="ansibold ansiblue">SA9</span>/    <span class="ansibold ansiblue">SB93</span>/
<span class="ansibold ansiblue">PROC</span>/                                           <span class="ansibold ansiblue">SA104</span>/  <span class="ansibold ansiblue">SA91</span>/   <span class="ansibold ansiblue">SB99</span>/
<span class="ansibold ansiblue">Q10</span>/                                            <span class="ansibold ansiblue">SA105</span>/  <span class="ansibold ansiblue">SA93</span>/   <span class="ansibold ansiblue">SC10</span>/
<span class="ansibold ansiblue">Q30</span>/                                            <span class="ansibold ansiblue">SA109</span>/  <span class="ansibold ansiblue">SA94</span>/   <span class="ansibold ansiblue">SC101</span>/
<span class="ansibold ansiblue">Q43</span>/                                            <span class="ansibold ansiblue">SA111</span>/  <span class="ansibold ansiblue">SA95</span>/   <span class="ansibold ansiblue">SC106</span>/
<span class="ansibold ansiblue">Q64</span>/                                            <span class="ansibold ansiblue">SA112</span>/  <span class="ansibold ansiblue">SA96</span>/   <span class="ansibold ansiblue">SC107</span>/
<span class="ansibold ansiblue">Q69</span>/                                            <span class="ansibold ansiblue">SA113</span>/  <span class="ansibold ansiblue">SA97</span>/   <span class="ansibold ansiblue">SC110</span>/
<span class="ansibold ansiblue">Q83</span>/                                            <span class="ansibold ansiblue">SA114</span>/  <span class="ansibold ansiblue">SA98</span>/   <span class="ansibold ansiblue">SC113</span>/
Querymap.txt                                    <span class="ansibold ansiblue">SA115</span>/  <span class="ansibold ansiblue">SB102</span>/  <span class="ansibold ansiblue">SC117</span>/
<span class="ansibold ansiblue">R1</span>/                                             <span class="ansibold ansiblue">SA116</span>/  <span class="ansibold ansiblue">SB104</span>/  <span class="ansibold ansiblue">SC119</span>/
<span class="ansibold ansiblue">R10</span>/                                            <span class="ansibold ansiblue">SA117</span>/  <span class="ansibold ansiblue">SB106</span>/  <span class="ansibold ansiblue">SC122</span>/
<span class="ansibold ansiblue">R102</span>/                                           <span class="ansibold ansiblue">SA119</span>/  <span class="ansibold ansiblue">SB111</span>/  <span class="ansibold ansiblue">SC124</span>/
<span class="ansibold ansiblue">R13</span>/                                            <span class="ansibold ansiblue">SA120</span>/  <span class="ansibold ansiblue">SB116</span>/  <span class="ansibold ansiblue">SC125</span>/
<span class="ansibold ansiblue">R15</span>/                                            <span class="ansibold ansiblue">SA121</span>/  <span class="ansibold ansiblue">SB119</span>/  <span class="ansibold ansiblue">SC141</span>/
<span class="ansibold ansiblue">R16</span>/                                            <span class="ansibold ansiblue">SA122</span>/  <span class="ansibold ansiblue">SB120</span>/  <span class="ansibold ansiblue">SC142</span>/
<span class="ansibold ansiblue">R20</span>/                                            <span class="ansibold ansiblue">SA124</span>/  <span class="ansibold ansiblue">SB127</span>/  <span class="ansibold ansiblue">SC143</span>/
<span class="ansibold ansiblue">R22</span>/                                            <span class="ansibold ansiblue">SA125</span>/  <span class="ansibold ansiblue">SB128</span>/  <span class="ansibold ansiblue">SC157</span>/
<span class="ansibold ansiblue">R23</span>/                                            <span class="ansibold ansiblue">SA126</span>/  <span class="ansibold ansiblue">SB129</span>/  <span class="ansibold ansiblue">SC16</span>/
<span class="ansibold ansiblue">R27</span>/                                            <span class="ansibold ansiblue">SA128</span>/  <span class="ansibold ansiblue">SB13</span>/   <span class="ansibold ansiblue">SC168</span>/
<span class="ansibold ansiblue">R29</span>/                                            <span class="ansibold ansiblue">SA13</span>/   <span class="ansibold ansiblue">SB131</span>/  <span class="ansibold ansiblue">SC178</span>/
<span class="ansibold ansiblue">R30</span>/                                            <span class="ansibold ansiblue">SA130</span>/  <span class="ansibold ansiblue">SB135</span>/  <span class="ansibold ansiblue">SC187</span>/
<span class="ansibold ansiblue">R31</span>/                                            <span class="ansibold ansiblue">SA132</span>/  <span class="ansibold ansiblue">SB136</span>/  <span class="ansibold ansiblue">SC198</span>/
<span class="ansibold ansiblue">R32</span>/                                            <span class="ansibold ansiblue">SA138</span>/  <span class="ansibold ansiblue">SB138</span>/  <span class="ansibold ansiblue">SC19A</span>/
<span class="ansibold ansiblue">R34</span>/                                            <span class="ansibold ansiblue">SA140</span>/  <span class="ansibold ansiblue">SB139</span>/  <span class="ansibold ansiblue">SC202</span>/
<span class="ansibold ansiblue">R36</span>/                                            <span class="ansibold ansiblue">SA144</span>/  <span class="ansibold ansiblue">SB140</span>/  <span class="ansibold ansiblue">SC26</span>/
<span class="ansibold ansiblue">R39</span>/                                            <span class="ansibold ansiblue">SA147</span>/  <span class="ansibold ansiblue">SB141</span>/  <span class="ansibold ansiblue">SC28</span>/
<span class="ansibold ansiblue">R4</span>/                                             <span class="ansibold ansiblue">SA15</span>/   <span class="ansibold ansiblue">SB142</span>/  <span class="ansibold ansiblue">SC34</span>/
<span class="ansibold ansiblue">R40</span>/                                            <span class="ansibold ansiblue">SA153</span>/  <span class="ansibold ansiblue">SB146</span>/  <span class="ansibold ansiblue">SC35</span>/
<span class="ansibold ansiblue">R43</span>/                                            <span class="ansibold ansiblue">SA155</span>/  <span class="ansibold ansiblue">SB147</span>/  <span class="ansibold ansiblue">SC43</span>/
<span class="ansibold ansiblue">R44</span>/                                            <span class="ansibold ansiblue">SA156</span>/  <span class="ansibold ansiblue">SB15</span>/   <span class="ansibold ansiblue">SC44</span>/
<span class="ansibold ansiblue">R45</span>/                                            <span class="ansibold ansiblue">SA157</span>/  <span class="ansibold ansiblue">SB150</span>/  <span class="ansibold ansiblue">SC48</span>/
<span class="ansibold ansiblue">R48</span>/                                            <span class="ansibold ansiblue">SA160</span>/  <span class="ansibold ansiblue">SB153</span>/  <span class="ansibold ansiblue">SC49</span>/
<span class="ansibold ansiblue">R49</span>/                                            <span class="ansibold ansiblue">SA162</span>/  <span class="ansibold ansiblue">SB159</span>/  <span class="ansibold ansiblue">SC51</span>/
<span class="ansibold ansiblue">R59</span>/                                            <span class="ansibold ansiblue">SA164</span>/  <span class="ansibold ansiblue">SB163</span>/  <span class="ansibold ansiblue">SC51A</span>/
<span class="ansibold ansiblue">R60</span>/                                            <span class="ansibold ansiblue">SA165</span>/  <span class="ansibold ansiblue">SB166</span>/  <span class="ansibold ansiblue">SC54A</span>/
<span class="ansibold ansiblue">R67</span>/                                            <span class="ansibold ansiblue">SA166</span>/  <span class="ansibold ansiblue">SB170</span>/  <span class="ansibold ansiblue">SC55</span>/
<span class="ansibold ansiblue">R68</span>/                                            <span class="ansibold ansiblue">SA171</span>/  <span class="ansibold ansiblue">SB171</span>/  <span class="ansibold ansiblue">SC57</span>/
<span class="ansibold ansiblue">R71</span>/                                            <span class="ansibold ansiblue">SA172</span>/  <span class="ansibold ansiblue">SB172</span>/  <span class="ansibold ansiblue">SC60</span>/
<span class="ansibold ansiblue">R74</span>/                                            <span class="ansibold ansiblue">SA174</span>/  <span class="ansibold ansiblue">SB180</span>/  <span class="ansibold ansiblue">SC61</span>/
<span class="ansibold ansiblue">R8</span>/                                             <span class="ansibold ansiblue">SA177</span>/  <span class="ansibold ansiblue">SB182</span>/  <span class="ansibold ansiblue">SC66</span>/
<span class="ansibold ansiblue">R85</span>/                                            <span class="ansibold ansiblue">SA180</span>/  <span class="ansibold ansiblue">SB183</span>/  <span class="ansibold ansiblue">SC7</span>/
<span class="ansibold ansiblue">R9</span>/                                             <span class="ansibold ansiblue">SA182</span>/  <span class="ansibold ansiblue">SB186</span>/  <span class="ansibold ansiblue">SC72</span>/
<span class="ansibold ansiblue">R90</span>/                                            <span class="ansibold ansiblue">SA189</span>/  <span class="ansibold ansiblue">SB194</span>/  <span class="ansibold ansiblue">SC74</span>/
<span class="ansibold ansiblue">R95</span>/                                            <span class="ansibold ansiblue">SA19</span>/   <span class="ansibold ansiblue">SB195</span>/  <span class="ansibold ansiblue">SC77</span>/
<span class="ansibold ansiblue">R98</span>/                                            <span class="ansibold ansiblue">SA190</span>/  <span class="ansibold ansiblue">SB197</span>/  <span class="ansibold ansiblue">SC78</span>/
<span class="ansibold ansiblue">R99</span>/                                            <span class="ansibold ansiblue">SA196</span>/  <span class="ansibold ansiblue">SB199</span>/  <span class="ansibold ansiblue">SC81</span>/
REFmap.txt                                      <span class="ansibold ansiblue">SA198</span>/  <span class="ansibold ansiblue">SB200</span>/  <span class="ansibold ansiblue">SC82</span>/
<span class="ansibold ansiblue">S100</span>/                                           <span class="ansibold ansiblue">SA199</span>/  <span class="ansibold ansiblue">SB201</span>/  <span class="ansibold ansiblue">SC86A</span>/
<span class="ansibold ansiblue">S101</span>/                                           <span class="ansibold ansiblue">SA2</span>/    <span class="ansibold ansiblue">SB204</span>/  <span class="ansibold ansiblue">SC88</span>/
<span class="ansibold ansiblue">S102</span>/                                           <span class="ansibold ansiblue">SA202</span>/  <span class="ansibold ansiblue">SB21</span>/   <span class="ansibold ansiblue">SC9</span>/
<span class="ansibold ansiblue">S11</span>/                                            <span class="ansibold ansiblue">SA21</span>/   <span class="ansibold ansiblue">SB22</span>/   <span class="ansibold ansiblue">SC93</span>/
<span class="ansibold ansiblue">S13</span>/                                            <span class="ansibold ansiblue">SA28</span>/   <span class="ansibold ansiblue">SB23</span>/   <span class="ansibold ansiblue">T19</span>/
<span class="ansibold ansiblue">S18</span>/                                            <span class="ansibold ansiblue">SA31</span>/   <span class="ansibold ansiblue">SB24</span>/   <span class="ansibold ansiblue">T21</span>/
<span class="ansibold ansiblue">S19</span>/                                            <span class="ansibold ansiblue">SA32</span>/   <span class="ansibold ansiblue">SB27</span>/   <span class="ansibold ansiblue">T32</span>/
<span class="ansibold ansiblue">S24</span>/                                            <span class="ansibold ansiblue">SA33</span>/   <span class="ansibold ansiblue">SB29</span>/   <span class="ansibold ansiblue">T49</span>/
<span class="ansibold ansiblue">S30</span>/                                            <span class="ansibold ansiblue">SA36</span>/   <span class="ansibold ansiblue">SB3</span>/    <span class="ansibold ansiblue">T68</span>/
<span class="ansibold ansiblue">S31</span>/                                            <span class="ansibold ansiblue">SA43</span>/   <span class="ansibold ansiblue">SB33</span>/   <span class="ansibold ansiblue">T78</span>/
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[57]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-Q Querymap.txt <span class="se">\</span>
--cluster <span class="se">\</span>
--clust_match 1 <span class="se">\</span>
--clust_cov 5 <span class="se">\</span>
--blast <span class="se">\</span>
--bitscore_skim_LCA 0.02 <span class="se">\</span>
--REFlist REFmap.txt <span class="se">\</span>
--min_ident 0.95 <span class="se">\</span>
-n 5 <span class="se">\</span>
-v <span class="se">\</span>
-@ callumjmacgregor@gmail.com <span class="se">\</span>
-o MothPollenReflistCurat1Breadth2 &gt; log_breadth2%
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[58]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash
blastn -query /home/working/AgainstRefDB/Curat1/Breadth2%/GLOBAL/global_centroids.fasta <span class="se">\</span>
-db /home/working/AgainstRefDB/Curat1/Breadth2%/GLOBAL/BLAST_0.95/marker_blast_db <span class="se">\</span>
-out /home/working/AgainstRefDB/Curat1/Breadth2%/GLOBAL/BLAST_0.95/global_blastn.out.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>At this point I have carefully examined the BLAST assignments on a case-by-case basis. I have made the following decisions:</p>
<ul>
<li><p>of the three search breadths, 5% sometimes makes assignments to family or order level that on inspection can clearly be made to genus level (effectively Type II error). 2%, however, sometimes does not consider matches that are quite close to the top hit and therefore seems to risk Type I errors. Therefore I will proceed with 3% search breadth in the final BLAST, which in most cases strikes a suitable balance between the two.</p>
</li>
<li><p>in some cases, assignments have been made to the species level on account of the "if 100% full-length match, do not consider lower bit-scores" that appear to be possible Type I errors. Therefore I will turn this rule off in the final BLAST.</p>
</li>
<li><p>in a number of cases, more accurate taxonomic assignments could be made with the removal of certain species from the reference database which are not plausibly present in the samples, either as a wind-carried pollen species or as a flower visited by moths in East Yorkshire during late summer. Therefore I will re-curate the reference database one final time before proceeding with the final BLAST.</p>
</li>
<li><p>lastly, I have got top-hits to things that are not a very close match. Given how invariable this region appears to be, there seems to be a strong chance that these are not correct identifications; therefore, I will experiment with changing the --min_ident flag to 0.98 or even 0.99 before doing the final BLAST.</p>
</li>
</ul>

</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="Try-increasing-the-minimum-identities-for-the-top-hit">Try increasing the minimum identities for the top hit<a class="anchor-link" href="#Try-increasing-the-minimum-identities-for-the-top-hit">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>I'll try this in conjunction with my planned final conditions: 3% BLAST with no 100% match rule. For comparison I will also run a no-rule BLAST with the original 95% setting, so I can see exactly what my options are.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="95%">95%<a class="anchor-link" href="#95%">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[96]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cd</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">AgainstRefDB</span><span class="o">/</span><span class="n">Curat1</span><span class="o">/</span><span class="n">Breadth3</span><span class="o">%/</span><span class="n">NoRule</span><span class="o">/</span><span class="mi">95</span><span class="n">ident</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>/home/working/AgainstRefDB/Curat1/Breadth3%/NoRule/95ident
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[97]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

<span class="k">for </span>sample in <span class="k">$(</span>cat ../../Querymap.txt | cut -f 1<span class="k">)</span>
<span class="k">do</span>
<span class="k">    </span><span class="nv">fasta</span><span class="o">=</span><span class="k">$(</span>ls -1 ../../../../../<span class="nv">$sample</span>/<span class="nv">$sample</span><span class="se">\_</span>trimmed.fasta<span class="k">)</span>
    <span class="nb">echo</span> -e <span class="s2">&quot;$sample\tfasta\t$fasta&quot;</span>
<span class="k">done</span> &gt; Querymap.txt

head Querymap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>SA177	fasta	../../../../../SA177/SA177_trimmed.fasta
SB140	fasta	../../../../../SB140/SB140_trimmed.fasta
T19	fasta	../../../../../T19/T19_trimmed.fasta
Lg45	fasta	../../../../../Lg45/Lg45_trimmed.fasta
Q69	fasta	../../../../../Q69/Q69_trimmed.fasta
S72	fasta	../../../../../S72/S72_trimmed.fasta
SC66	fasta	../../../../../SC66/SC66_trimmed.fasta
S31	fasta	../../../../../S31/S31_trimmed.fasta
SC74	fasta	../../../../../SC74/SC74_trimmed.fasta
SA68	fasta	../../../../../SA68/SA68_trimmed.fasta
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[98]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>cp ../../REFmap.txt .

<span class="o">!</span>head REFmap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb	gb
positives_curated.gb	gb
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[99]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../../</span><span class="n">eyorks_flora_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[100]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../../</span><span class="n">positives_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[101]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb  positives_curated.gb  Querymap.txt  REFmap.txt
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[102]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-Q Querymap.txt <span class="se">\</span>
--cluster <span class="se">\</span>
--clust_match 1 <span class="se">\</span>
--clust_cov 5 <span class="se">\</span>
--blast <span class="se">\</span>
--bitscore_skim_LCA 0.03 <span class="se">\</span>
--bitscore_skim_adjust_off <span class="se">\</span>
--REFlist REFmap.txt <span class="se">\</span>
--min_ident 0.95 <span class="se">\</span>
-n 5 <span class="se">\</span>
-v <span class="se">\</span>
-@ callumjmacgregor@gmail.com <span class="se">\</span>
-o MothPollenReflistCurat1Breadth3-95Ident &gt; log_95ident
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[103]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash
blastn -query /home/working/AgainstRefDB/Curat1/Breadth3%/NoRule/95ident/GLOBAL/global_centroids.fasta <span class="se">\</span>
-db /home/working/AgainstRefDB/Curat1/Breadth3%/NoRule/95ident/GLOBAL/BLAST_0.95/marker_blast_db <span class="se">\</span>
-out /home/working/AgainstRefDB/Curat1/Breadth3%/NoRule/95ident/GLOBAL/BLAST_0.95/global_blastn.out.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="98%">98%<a class="anchor-link" href="#98%">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[76]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cd</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">AgainstRefDB</span><span class="o">/</span><span class="n">Curat1</span><span class="o">/</span><span class="n">Breadth3</span><span class="o">%/</span><span class="n">NoRule</span><span class="o">/</span><span class="mi">98</span><span class="n">ident</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>/home/working/AgainstRefDB/Curat1/Breadth3%/NoRule/98ident
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[77]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

<span class="k">for </span>sample in <span class="k">$(</span>cat ../../Querymap.txt | cut -f 1<span class="k">)</span>
<span class="k">do</span>
<span class="k">    </span><span class="nv">fasta</span><span class="o">=</span><span class="k">$(</span>ls -1 ../../../../../<span class="nv">$sample</span>/<span class="nv">$sample</span><span class="se">\_</span>trimmed.fasta<span class="k">)</span>
    <span class="nb">echo</span> -e <span class="s2">&quot;$sample\tfasta\t$fasta&quot;</span>
<span class="k">done</span> &gt; Querymap.txt

head Querymap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>SA177	fasta	../../../../../SA177/SA177_trimmed.fasta
SB140	fasta	../../../../../SB140/SB140_trimmed.fasta
T19	fasta	../../../../../T19/T19_trimmed.fasta
Lg45	fasta	../../../../../Lg45/Lg45_trimmed.fasta
Q69	fasta	../../../../../Q69/Q69_trimmed.fasta
S72	fasta	../../../../../S72/S72_trimmed.fasta
SC66	fasta	../../../../../SC66/SC66_trimmed.fasta
S31	fasta	../../../../../S31/S31_trimmed.fasta
SC74	fasta	../../../../../SC74/SC74_trimmed.fasta
SA68	fasta	../../../../../SA68/SA68_trimmed.fasta
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[78]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>cp ../../REFmap.txt .

<span class="o">!</span>head REFmap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb	gb
positives_curated.gb	gb
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[79]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../../</span><span class="n">eyorks_flora_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[80]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../../</span><span class="n">positives_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[81]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb  positives_curated.gb  Querymap.txt  REFmap.txt
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[82]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-Q Querymap.txt <span class="se">\</span>
--cluster <span class="se">\</span>
--clust_match 1 <span class="se">\</span>
--clust_cov 5 <span class="se">\</span>
--blast <span class="se">\</span>
--bitscore_skim_LCA 0.03 <span class="se">\</span>
--bitscore_skim_adjust_off <span class="se">\</span>
--REFlist REFmap.txt <span class="se">\</span>
--min_ident 0.98 <span class="se">\</span>
-n 5 <span class="se">\</span>
-v <span class="se">\</span>
-@ callumjmacgregor@gmail.com <span class="se">\</span>
-o MothPollenReflistCurat1Breadth3-98Ident &gt; log_98ident
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[83]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash
blastn -query /home/working/AgainstRefDB/Curat1/Breadth3%/NoRule/98ident/GLOBAL/global_centroids.fasta <span class="se">\</span>
-db /home/working/AgainstRefDB/Curat1/Breadth3%/NoRule/98ident/GLOBAL/BLAST_0.98/marker_blast_db <span class="se">\</span>
-out /home/working/AgainstRefDB/Curat1/Breadth3%/NoRule/98ident/GLOBAL/BLAST_0.98/global_blastn.out.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="99%">99%<a class="anchor-link" href="#99%">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[84]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cd</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">AgainstRefDB</span><span class="o">/</span><span class="n">Curat1</span><span class="o">/</span><span class="n">Breadth3</span><span class="o">%/</span><span class="n">NoRule</span><span class="o">/</span><span class="mi">99</span><span class="n">ident</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>/home/working/AgainstRefDB/Curat1/Breadth3%/NoRule/99ident
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[85]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

<span class="k">for </span>sample in <span class="k">$(</span>cat ../../Querymap.txt | cut -f 1<span class="k">)</span>
<span class="k">do</span>
<span class="k">    </span><span class="nv">fasta</span><span class="o">=</span><span class="k">$(</span>ls -1 ../../../../../<span class="nv">$sample</span>/<span class="nv">$sample</span><span class="se">\_</span>trimmed.fasta<span class="k">)</span>
    <span class="nb">echo</span> -e <span class="s2">&quot;$sample\tfasta\t$fasta&quot;</span>
<span class="k">done</span> &gt; Querymap.txt

head Querymap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>SA177	fasta	../../../../../SA177/SA177_trimmed.fasta
SB140	fasta	../../../../../SB140/SB140_trimmed.fasta
T19	fasta	../../../../../T19/T19_trimmed.fasta
Lg45	fasta	../../../../../Lg45/Lg45_trimmed.fasta
Q69	fasta	../../../../../Q69/Q69_trimmed.fasta
S72	fasta	../../../../../S72/S72_trimmed.fasta
SC66	fasta	../../../../../SC66/SC66_trimmed.fasta
S31	fasta	../../../../../S31/S31_trimmed.fasta
SC74	fasta	../../../../../SC74/SC74_trimmed.fasta
SA68	fasta	../../../../../SA68/SA68_trimmed.fasta
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[86]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>cp ../../REFmap.txt .

<span class="o">!</span>head REFmap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb	gb
positives_curated.gb	gb
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[87]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../../</span><span class="n">eyorks_flora_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[88]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../../</span><span class="n">positives_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[89]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb  positives_curated.gb  Querymap.txt  REFmap.txt
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[94]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-Q Querymap.txt <span class="se">\</span>
--cluster <span class="se">\</span>
--clust_match 1 <span class="se">\</span>
--clust_cov 5 <span class="se">\</span>
--blast <span class="se">\</span>
--bitscore_skim_LCA 0.03 <span class="se">\</span>
--bitscore_skim_adjust_off <span class="se">\</span>
--REFlist REFmap.txt <span class="se">\</span>
--min_ident 0.99 <span class="se">\</span>
-n 5 <span class="se">\</span>
-v <span class="se">\</span>
-@ callumjmacgregor@gmail.com <span class="se">\</span>
-o MothPollenReflistCurat1Breadth3-99Ident &gt; log_99ident
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[95]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash
blastn -query /home/working/AgainstRefDB/Curat1/Breadth3%/NoRule/99ident/GLOBAL/global_centroids.fasta <span class="se">\</span>
-db /home/working/AgainstRefDB/Curat1/Breadth3%/NoRule/99ident/GLOBAL/BLAST_0.99/marker_blast_db <span class="se">\</span>
-out /home/working/AgainstRefDB/Curat1/Breadth3%/NoRule/99ident/GLOBAL/BLAST_0.99/global_blastn.out.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>There is not very much difference at all between the settings here, so let's just proceed with 95% identities.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="Final-BLAST">Final BLAST<a class="anchor-link" href="#Final-BLAST">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>First I will move the things I need into a new subdirectory. This time I am going to include the list of positive controls in my main species list, so that I can work with a single reference database file.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[106]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cd</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">AgainstRefDB</span><span class="o">/</span><span class="n">Curat1</span><span class="o">/</span><span class="n">FINAL</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>/home/working/AgainstRefDB/Curat1/FINAL
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[107]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../</span><span class="n">AccessionBlacklist</span><span class="o">.</span><span class="n">txt</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[108]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>AccessionBlacklist.txt   Querymap.txt            REFmap.txt
eyorks_flora_curated.gb  <span class="ansibold ansigreen">RefListCuratFinal.txt</span>*  <span class="ansibold ansiblue">Sativa</span>/
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[109]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>head RefListCuratFinal.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>









</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[69]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>fetch_from_db.py -t RefListCuratFinal.txt -m rbcl -o eyorks_flora_curated -@ callumjmacgregor@gmail.com
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>
metaBEAT may be querying NCBI&apos;s Entrez databases to fetch/verify taxonomic ids. Entrez User requirements state that you need to identify yourself by providing an email address so that NCBI can contact you in case there is a problem.

You have specified: &apos;callumjmacgregor@gmail.com&apos;


QUERYING GENBANK

check for synonyms for &quot;rbcl&quot; (this is relevant only for Genbank searches)

fetching accessions ..

#	Abies alba	8
#	Abies cephalonica	2
#	Abies cilicica	1
#	Abies concolor	6
#	Abies delavayi	6
#	Abies firma	6
#	Abies grandis	4
#	Abies homolepis	4
#	Abies lasiocarpa	5
#	Abies nordmanniana	4
#	Abies pinsapo	5
#	Abies procera	1
#	Abies veitchii	4
#	Acaena novae-zelandiae	3
#	Acer campestre	18
#	Acer platanoides	6
#	Acer pseudoplatanus	16
#	Achillea alpina	0
#	Achillea distans	0
#	Achillea ligustica	0
#	Achillea millefolium	39
#	Achillea ptarmica	4
#	Aconitum napellus	5
#	Acorus calamus	33
#	Actaea spicata	5
#	Adonis annua	2
#	Adoxa moschatellina	7
#	Aegopodium podagraria	7
#	Aesculus carnea	0
#	Aesculus hippocastanum	11
#	Aethusa cynapium	5
#	Agrimonia eupatoria	5
#	Agrimonia procera	5
#	Agrostemma githago	7
#	Agrostis canina	3
#	Agrostis capillaris	5
#	Agrostis castellana	0
#	Agrostis gigantea	11
#	Agrostis scabra	6
#	Agrostis stolonifera	20
#	Agrostis vinealis	3
#	Aira caryophyllea	6
#	Aira praecox	6
#	Ajuga reptans	10
#	Alcea rosea	2
#	Alchemilla glabra	5
#	Alchemilla mollis	2
#	Alchemilla vulgaris	2
#	Alchemilla xanthochlora	5
#	Alisma lanceolatum	4
#	Alisma plantago-aquatica	17
#	Alliaria petiolata	21
#	Allium carinatum	1
#	Allium moly	3
#	Allium neapolitanum	9
#	Allium oleraceum	3
#	Allium schoenoprasum	3
#	Allium scorodoprasum	2
#	Allium triquetrum	6
#	Allium ursinum	10
#	Allium vineale	3
#	Alnus cordata	1
#	Alnus glutinosa	17
#	Alnus incana	11
#	Alopecurus aequalis	16
#	Alopecurus geniculatus	4
#	Alopecurus myosuroides	4
#	Alopecurus pratensis	9
#	Alyssum alyssoides	0
#	Amaranthus albus	5
#	Amaranthus hybridus	18
#	Amaranthus retroflexus	16
#	Amaranthus thunbergii	0
#	Ambrosia artemisiifolia	19
#	Ammi majus	1
#	Ammi visnaga	1
#	Amsinckia lycopsoides	0
#	Amsinckia micrantha	1
#	Anacamptis morio	1
#	Anacamptis pyramidalis	2
#	Anagallis arvensis	9
#	Anagallis tenella	5
#	Anaphalis margaritacea	7
#	Anchusa arvensis	4
#	Andromeda polifolia	15
#	Anemone apennina	0
#	Anemone nemorosa	6
#	Anemone ranunculoides	1
#	Anethum graveolens	6
#	Angelica sylvestris	4
#	Anisantha diandra	0
#	Anisantha madritensis	0
#	Anisantha rigida	7
#	Anisantha rubens	0
#	Anisantha sterilis	7
#	Anisantha tectorum	0
#	Antennaria dioica	4
#	Anthemis arvensis	2
#	Anthemis cotula	2
#	Anthemis tinctoria	1
#	Anthoxanthum aristatum	0
#	Anthoxanthum odoratum	14
#	Anthriscus caucalis	2
#	Anthriscus sylvestris	9
#	Anthyllis vulneraria	8
#	Antirrhinum majus	7
#	Apera interrupta	1
#	Apera spica-venti	1
#	Aphanes arvensis	4
#	Aphanes australis	2
#	Apium graveolens	10
#	Apium inundatum	2
#	Apium nodiflorum	6
#	Apium repens	0
#	Aquilegia vulgaris	7
#	Arabidopsis thaliana	189
#	Arabis caucasica	1
#	Arabis hirsuta	7
#	Araucaria araucana	5
#	Arbutus unedo	3
#	Arctium lappa	11
#	Arctium minus	15
#	Arctium nemorosum	0
#	Arenaria leptoclados	0
#	Arenaria serpyllifolia	15
#	Armeria maritima	6
#	Armoracia rusticana	5
#	Arnoseris minima	0
#	Arrhenatherum elatius	11
#	Artemisia absinthium	8
#	Artemisia campestris	3
#	Artemisia maritima	2
#	Artemisia verlotiorum	0
#	Artemisia vulgaris	21
#	Arum italicum	9
#	Arum maculatum	7
#	Asparagus officinalis	17
#	Asperugo procumbens	1
#	Asperula arvensis	0
#	Asperula cynanchica	6
#	Asplenium adiantum-nigrum	15
#	Asplenium ceterach	9
#	Asplenium marinum	2
#	Asplenium ruta-muraria	23
#	Asplenium scolopendrium	8
#	Asplenium trichomanes	20
#	Aster lanceolatus	7
#	Aster novi-belgii	1
#	Aster tripolium	3
#	Astragalus danicus	1
#	Astragalus glycyphyllos	3
#	Astrantia major	3
#	Athyrium filix-femina	17
#	Atriplex littoralis	4
#	Atriplex patula	13
#	Atriplex prostrata	5
#	Atropa belladonna	13
#	Aubrieta deltoidea	1
#	Avena fatua	15
#	Avena sativa	10
#	Avenula pratensis	2
#	Avenula pubescens	5
#	Azolla filiculoides	6
#	Baldellia ranunculoides	4
#	Ballota nigra	6
#	Barbarea intermedia	1
#	Barbarea stricta	0
#	Barbarea verna	4
#	Barbarea vulgaris	16
#	Bassia scoparia	4
#	Bellis perennis	13
#	Berberis darwinii	0
#	Berberis vulgaris	3
#	Berteroa incana	3
#	Berula erecta	5
#	Beta trigyna	1
#	Beta vulgaris	128
#	Betonica officinalis	8
#	Betula pendula	18
#	Betula pubescens	8
#	Bidens cernua	12
#	Bidens tripartita	7
#	Blackstonia perfoliata	5
#	Blechnum spicant	4
#	Blysmus compressus	8
#	Bolboschoenus maritimus	7
#	Borago officinalis	9
#	Botrychium lunaria	11
#	Brachypodium pinnatum	7
#	Brachypodium rupestre	3
#	Brachypodium sylvaticum	9
#	Brassica juncea	17
#	Brassica napus	292
#	Brassica nigra	11
#	Brassica oleracea	103
#	Brassica rapa	86
#	Briza maxima	5
#	Briza media	7
#	Bromopsis benekenii	2
#	Bromopsis erecta	9
#	Bromopsis inermis	22
#	Bromopsis ramosa	3
#	Bromus arvensis	1
#	Bromus commutatus	4
#	Bromus hordeaceus	8
#	Bromus lanceolatus	0
#	Bromus lepidus	0
#	Bromus racemosus	3
#	Bromus secalinus	2
#	Bryonia dioica	5
#	Bryophyllum pinnatum	0
#	Buddleja davidii	6
#	Buddleja globosa	0
#	Buddleja officinalis	1
#	Bunias orientalis	1
#	Bupleurum baldense	0
#	Bupleurum rotundifolium	0
#	Bupleurum subovatum	0
#	Bupleurum tenuissimum	2
#	Butomus umbellatus	7
#	Buxus sempervirens	13
#	Cakile maritima	4
#	Calamagrostis canescens	5
#	Calamagrostis epigejos	13
#	Calamagrostis gracilescens	0
#	Calendula officinalis	5
#	Calla palustris	5
#	Callitriche hermaphroditica	6
#	Callitriche obtusangula	3
#	Callitriche platycarpa	5
#	Callitriche stagnalis	7
#	Calluna vulgaris	11
#	Caltha palustris	32
#	Calystegia pulchra	2
#	Calystegia sepium	18
#	Calystegia silvatica	4
#	Calystegia soldanella	6
#	Camelina sativa	157
#	Campanula glomerata	11
#	Campanula latifolia	6
#	Campanula medium	2
#	Campanula patula	10
#	Campanula persicifolia	2
#	Campanula portenschlagiana	2
#	Campanula poscharskyana	1
#	Campanula rapunculoides	10
#	Campanula rapunculus	5
#	Campanula rotundifolia	22
#	Campanula trachelium	9
#	Cannabis sativa	26
#	Capsella bursa-pastoris	24
#	Cardamine flexuosa	5
#	Cardamine hirsuta	13
#	Cardamine pratensis	24
#	Carduus crispus	3
#	Carduus nutans	6
#	Carduus pycnocephalus	0
#	Carduus tenuiflorus	4
#	Carex acuta	3
#	Carex acutiformis	3
#	Carex appropinquata	0
#	Carex arenaria	5
#	Carex binervis	3
#	Carex canescens	8
#	Carex caryophyllea	4
#	Carex demissa	0
#	Carex diandra	5
#	Carex dioica	4
#	Carex distans	4
#	Carex disticha	3
#	Carex divisa	3
#	Carex divulsa	5
#	Carex echinata	3
#	Carex elata	3
#	Carex elongata	2
#	Carex extensa	5
#	Carex flacca	8
#	Carex hirta	4
#	Carex hostiana	5
#	Carex lasiocarpa	4
#	Carex lepidocarpa	0
#	Carex leporina	0
#	Carex muricata	4
#	Carex nigra	7
#	Carex oederi	0
#	Carex otrubae	5
#	Carex pallescens	5
#	Carex panicea	7
#	Carex paniculata	4
#	Carex pendula	9
#	Carex pilulifera	5
#	Carex pseudocyperus	6
#	Carex pulicaris	5
#	Carex remota	4
#	Carex riparia	3
#	Carex rostrata	4
#	Carex spicata	4
#	Carex sylvatica	5
#	Carex vesicaria	3
#	Carex viridula	4
#	Carlina vulgaris	5
#	Carpinus betulus	8
#	Carum carvi	6
#	Castanea sativa	12
#	Catabrosa aquatica	6
#	Catapodium marinum	3
#	Catapodium rigidum	6
#	Cedrus atlantica	6
#	Cedrus deodara	17
#	Cedrus libani	4
#	Centaurea aspera	0
#	Centaurea calcitrapa	0
#	Centaurea cyanus	5
#	Centaurea diluta	0
#	Centaurea montana	1
#	Centaurea nigra	6
#	Centaurea scabiosa	5
#	Centaurea solstitialis	0
#	Centaurium erythraea	7
#	Centaurium littorale	1
#	Centaurium pulchellum	5
#	Centranthus ruber	7
#	Centunculus minimus	0
#	Cephalaria gigantea	1
#	Cerastium arvense	7
#	Cerastium diffusum	4
#	Cerastium fontanum	15
#	Cerastium glomeratum	6
#	Cerastium semidecandrum	2
#	Cerastium tomentosum	2
#	Ceratocapnos claviculata	4
#	Ceratochloa carinata	2
#	Ceratochloa cathartica	12
#	Ceratophyllum demersum	15
#	Ceratophyllum submersum	4
#	Chaenorhinum minus	7
#	Chaerophyllum temulum	9
#	Chamaecyparis lawsoniana	8
#	Chamaecyparis obtusa	7
#	Chamaecyparis pisifera	7
#	Chamaemelum nobile	5
#	Chamerion angustifolium	29
#	Chara vulgaris	8
#	Chelidonium majus	14
#	Chenopodium album	46
#	Chenopodium bonus-henricus	6
#	Chenopodium ficifolium	5
#	Chenopodium glaucum	7
#	Chenopodium hybridum	0
#	Chenopodium opulifolium	2
#	Chenopodium polyspermum	4
#	Chenopodium pratericola	0
#	Chenopodium quinoa	13
#	Chenopodium rubrum	4
#	Chenopodium urbicum	4
#	Chenopodium vulvaria	4
#	Chrysosplenium alternifolium	4
#	Chrysosplenium oppositifolium	4
#	Cichorium intybus	17
#	Cicuta virosa	7
#	Circaea lutetiana	15
#	Cirsium acaule	4
#	Cirsium arvense	14
#	Cirsium dissectum	3
#	Cirsium eriophorum	3
#	Cirsium palustre	8
#	Cirsium vulgare	18
#	Cladium mariscus	7
#	Claytonia perfoliata	1
#	Claytonia sibirica	2
#	Clematis montana	2
#	Clematis tangutica	1
#	Clematis vitalba	8
#	Clinopodium acinos	5
#	Clinopodium ascendens	1
#	Clinopodium vulgare	17
#	Cochlearia anglica	4
#	Cochlearia danica	4
#	Cochlearia officinalis	3
#	Coeloglossum viride	6
#	Coincya monensis	2
#	Colchicum autumnale	14
#	Comarum palustre	8
#	Conium maculatum	8
#	Conopodium majus	5
#	Conringia orientalis	0
#	Consolida ajacis	3
#	Convolvulus arvensis	20
#	Conyza canadensis	9
#	Conyza floribunda	0
#	Coriandrum sativum	7
#	Cornus alba	1
#	Cornus sanguinea	11
#	Coronilla scorpioides	0
#	Coronilla valentina	0
#	Corrigiola litoralis	1
#	Cortaderia selloana	3
#	Corylus avellana	22
#	Cotoneaster dielsianus	3
#	Cotoneaster horizontalis	5
#	Cotoneaster integrifolius	0
#	Cotoneaster lacteus	2
#	Cotoneaster rehderi	2
#	Cotoneaster simonsii	1
#	Cotoneaster splendens	0
#	Crambe hispanica	0
#	Crambe maritima	3
#	Crassula helmsii	4
#	Crataegus laevigata	9
#	Crataegus monogyna	36
#	Crepis biennis	2
#	Crepis capillaris	6
#	Crepis nicaeensis	0
#	Crepis paludosa	3
#	Crepis vesicaria	2
#	Crocus vernus	2
#	Cruciata laevipes	6
#	Cupressus duclouxiana	8
#	Cupressus goveniana	4
#	Cupressus macrocarpa	8
#	Cupressus sempervirens	11
#	Cuscuta campestris	5
#	Cuscuta epithymum	7
#	Cuscuta europaea	5
#	Cuscuta suaveolens	2
#	Cymbalaria muralis	2
#	Cymbalaria pallida	0
#	Cynoglossum officinale	7
#	Cynosurus cristatus	8
#	Cynosurus echinatus	2
#	Cyperus eragrostis	4
#	Cyperus longus	7
#	Cyrtomium falcatum	13
#	Cyrtomium fortunei	21
#	Cystopteris fragilis	19
#	Cytisus scoparius	8
#	Dactylis glomerata	32
#	Dactylorhiza fuchsii	4
#	Dactylorhiza incarnata	4
#	Dactylorhiza maculata	5
#	Dactylorhiza praetermissa	3
#	Dactylorhiza purpurella	4
#	Damasonium alisma	1
#	Danthonia decumbens	6
#	Daphne laureola	4
#	Daucus carota	43
#	Deschampsia cespitosa	53
#	Deschampsia flexuosa	5
#	Deschampsia setacea	2
#	Descurainia sophia	5
#	Dianthus barbatus	1
#	Dianthus deltoides	8
#	Digitalis purpurea	14
#	Digitaria sanguinalis	9
#	Diplotaxis muralis	1
#	Diplotaxis tenuifolia	9
#	Dipsacus fullonum	13
#	Dipsacus pilosus	4
#	Dipsacus sativus	1
#	Doronicum pardalianches	1
#	Dracaena aletriformis	2
#	Dracaena draco	3
#	Dracaena fragrans	1
#	Dracaena mannii	1
#	Dracaena transvaalensis	1
#	Drosera intermedia	1
#	Drosera rotundifolia	10
#	Dryopteris affinis	4
#	Dryopteris borreri	4
#	Dryopteris cambrensis	0
#	Dryopteris carthusiana	22
#	Dryopteris cristata	20
#	Dryopteris dilatata	11
#	Dryopteris filix-mas	15
#	Dysphania pumilio	0
#	Echinochloa colona	9
#	Echinochloa crus-galli	33
#	Echium plantagineum	3
#	Echium vulgare	5
#	Eleocharis acicularis	11
#	Eleocharis multicaulis	3
#	Eleocharis palustris	13
#	Eleocharis quinqueflora	5
#	Eleocharis uniglumis	4
#	Eleogiton fluitans	9
#	Eleusine indica	18
#	Elodea canadensis	7
#	Elodea nuttallii	8
#	Elymus caninus	9
#	Elytrigia atherica	2
#	Elytrigia juncea	3
#	Elytrigia repens	25
#	Epilobium ciliatum	4
#	Epilobium hirsutum	8
#	Epilobium komarovianum	0
#	Epilobium montanum	7
#	Epilobium obscurum	3
#	Epilobium palustre	18
#	Epilobium parviflorum	13
#	Epilobium roseum	3
#	Epilobium tetragonum	6
#	Epipactis helleborine	21
#	Epipactis palustris	5
#	Equisetum arvense	44
#	Equisetum fluviatile	10
#	Equisetum hyemale	23
#	Equisetum palustre	7
#	Equisetum telmateia	3
#	Eragrostis cilianensis	2
#	Eragrostis pectinacea	13
#	Eranthis hyemalis	2
#	Erica cinerea	5
#	Erica tetralix	5
#	Erigeron acris	4
#	Erigeron karvinskianus	2
#	Eriophorum angustifolium	23
#	Eriophorum latifolium	4
#	Eriophorum vaginatum	19
#	Erodium cicutarium	6
#	Erodium moschatum	5
#	Erophila verna	10
#	Eryngium maritimum	6
#	Erysimum cheiranthoides	11
#	Erysimum cheiri	3
#	Escallonia macrantha	1
#	Eschscholzia californica	7
#	Euonymus europaeus	8
#	Eupatorium cannabinum	3
#	Euphorbia cyparissias	13
#	Euphorbia esula	8
#	Euphorbia exigua	6
#	Euphorbia helioscopia	10
#	Euphorbia lathyris	5
#	Euphorbia peplus	13
#	Euphorbia platyphyllos	3
#	Euphrasia micrantha	3
#	Euphrasia nemorosa	3
#	Euphrasia officinalis	2
#	Fagopyrum esculentum	12
#	Fagus sylvatica	14
#	Falcaria vulgaris	1
#	Fallopia baldschuanica	2
#	Fallopia convolvulus	13
#	Fallopia japonica	114
#	Fallopia sachalinensis	191
#	Festuca brevipila	0
#	Festuca filiformis	2
#	Festuca ovina	10
#	Festuca rubra	52
#	Ficaria verna	6
#	Ficus carica	12
#	Filago lutescens	0
#	Filago minima	2
#	Filago pyramidata	1
#	Filago vulgaris	3
#	Filipendula ulmaria	9
#	Foeniculum vulgare	17
#	Fragaria ananassa	5
#	Fragaria moschata	2
#	Fragaria vesca	69
#	Frangula alnus	12
#	Fritillaria meleagris	4
#	Fumaria densiflora	2
#	Fumaria muralis	5
#	Fumaria officinalis	4
#	Fumaria parviflora	0
#	Fumaria purpurea	2
#	Fumaria vaillantii	0
#	Gagea bohemica	3
#	Gagea lutea	7
#	Galanthus nivalis	2
#	Galanthus plicatus	1
#	Galanthus woronowii	0
#	Galega officinalis	2
#	Galeopsis angustifolia	3
#	Galeopsis bifida	5
#	Galeopsis speciosa	3
#	Galeopsis tetrahit	8
#	Galinsoga parviflora	3
#	Galinsoga quadriradiata	11
#	Galium aparine	25
#	Galium odoratum	5
#	Galium palustre	14
#	Galium saxatile	5
#	Galium uliginosum	3
#	Galium verum	14
#	Gentiana pneumonanthe	2
#	Gentianella amarella	7
#	Gentianella campestris	4
#	Geranium columbinum	4
#	Geranium dissectum	7
#	Geranium endressii	3
#	Geranium lucidum	4
#	Geranium molle	9
#	Geranium nodosum	3
#	Geranium phaeum	6
#	Geranium pratense	4
#	Geranium pusillum	3
#	Geranium pyrenaicum	2
#	Geranium robertianum	13
#	Geranium rotundifolium	6
#	Geranium sanguineum	10
#	Geranium sylvaticum	6
#	Geranium versicolor	0
#	Geum rivale	5
#	Geum urbanum	8
#	Ginkgo biloba	30
#	Gladiolus communis	1
#	Glaucium flavum	6
#	Glaux maritima	5
#	Glebionis segetum	6
#	Glechoma hederacea	19
#	Glyceria declinata	5
#	Glyceria fluitans	7
#	Glyceria maxima	3
#	Glyceria notata	5
#	Glycine max	159
#	Gnaphalium luteoalbum	3
#	Gnaphalium sylvaticum	1
#	Gnaphalium uliginosum	6
#	Goodyera repens	9
#	Groenlandia densa	6
#	Gymnadenia conopsea	9
#	Gymnadenia densiflora	0
#	Gymnocarpium dryopteris	17
#	Gymnocarpium robertianum	3
#	Hedera helix	26
#	Hedera hibernica	2
#	Helianthemum nummularium	9
#	Helianthus annuus	14
#	Helianthus decapetalus	7
#	Helianthus tuberosus	11
#	Helleborus foetidus	3
#	Helleborus viridis	4
#	Helminthotheca echioides	6
#	Helosciadium nodiflorum	6
#	Hemizonia pungens	0
#	Hepatica nobilis	8
#	Heracleum mantegazzianum	0
#	Heracleum sphondylium	6
#	Herniaria glabra	2
#	Hesperis matronalis	7
#	Hieracium acuminatum	0
#	Hieracium aggregatum	0
#	Hieracium calcaricola	0
#	Hieracium decolor	0
#	Hieracium diaphanoides	0
#	Hieracium diaphanum	0
#	Hieracium grandidens	0
#	Hieracium lepidulum	1
#	Hieracium sabaudum	0
#	Hieracium umbellatum	6
#	Hieracium vagum	0
#	Hieracium vulgatum	0
#	Hippocrepis comosa	7
#	Hippophae rhamnoides	14
#	Hippuris vulgaris	21
#	Hirschfeldia incana	2
#	Holcus lanatus	9
#	Holcus mollis	3
#	Honckenya peploides	21
#	Hordeum distichon	0
#	Hordeum murinum	19
#	Hordeum secalinum	7
#	Hordeum vulgare	34
#	Hottonia palustris	3
#	Humulus lupulus	20
#	Hyacinthoides hispanica	28
#	Hyacinthoides non-scripta	15
#	Hydrangea anomala	5
#	Hydrangea arborescens	9
#	Hydrangea aspera	2
#	Hydrocharis morsus-ranae	4
#	Hydrocotyle ranunculoides	1
#	Hydrocotyle vulgaris	5
#	Hyoscyamus niger	7
#	Hypericum androsaemum	5
#	Hypericum calycinum	1
#	Hypericum elodes	7
#	Hypericum hirsutum	3
#	Hypericum humifusum	4
#	Hypericum montanum	2
#	Hypericum perforatum	23
#	Hypericum pulchrum	4
#	Hypericum tetrapterum	3
#	Hypochaeris glabra	5
#	Hypochaeris maculata	1
#	Hypochaeris radicata	8
#	Hypopitys monotropa	1
#	Iberis amara	3
#	Ilex aquifolium	19
#	Impatiens capensis	17
#	Impatiens glandulifera	2
#	Impatiens noli-tangere	5
#	Impatiens parviflora	2
#	Inula conyzae	3
#	Inula helenium	11
#	Iris foetidissima	3
#	Iris germanica	6
#	Iris pseudacorus	12
#	Isoetes lacustris	2
#	Isolepis setacea	11
#	Iva xanthiifolia	0
#	Jacobaea vulgaris	7
#	Jasione montana	6
#	Juglans nigra	10
#	Juglans regia	8
#	Juncus acutiflorus	3
#	Juncus articulatus	8
#	Juncus bufonius	12
#	Juncus bulbosus	7
#	Juncus compressus	5
#	Juncus conglomeratus	6
#	Juncus effusus	20
#	Juncus gerardii	6
#	Juncus inflexus	5
#	Juncus maritimus	7
#	Juncus ranarius	1
#	Juncus squarrosus	5
#	Juncus subnodulosus	3
#	Juncus tenuis	7
#	Kalanchoe pinnata	7
#	Kickxia elatine	4
#	Knautia arvensis	8
#	Koeleria macrantha	7
#	Lactuca sativa	12
#	Lactuca serriola	14
#	Lactuca virosa	3
#	Lagarosiphon major	14
#	Lamiastrum galeobdolon	7
#	Lamium album	9
#	Lamium amplexicaule	10
#	Lamium confertum	4
#	Lamium hybridum	3
#	Lamium maculatum	5
#	Lamium purpureum	14
#	Lappula squarrosa	2
#	Lapsana communis	17
#	Larix decidua	9
#	Larix kaempferi	11
#	Lathraea clandestina	12
#	Lathraea squamaria	5
#	Lathyrus aphaca	14
#	Lathyrus grandiflorus	0
#	Lathyrus hirsutus	2
#	Lathyrus latifolius	3
#	Lathyrus linifolius	3
#	Lathyrus nissolia	2
#	Lathyrus palustris	6
#	Lathyrus pratensis	12
#	Lathyrus sativus	4
#	Lathyrus sylvestris	10
#	Lathyrus tuberosus	11
#	Legousia hybrida	3
#	Lemna gibba	21
#	Lemna minor	21
#	Lemna minuta	6
#	Lemna trisulca	11
#	Leontodon hispidus	6
#	Leontodon saxatilis	6
#	Leonurus cardiaca	8
#	Lepidium campestre	12
#	Lepidium coronopus	4
#	Lepidium didymum	2
#	Lepidium draba	3
#	Lepidium heterophyllum	4
#	Lepidium latifolium	3
#	Lepidium ruderale	6
#	Lepidium sativum	1
#	Lepidium virginicum	4
#	Leucanthemum vulgare	17
#	Leucojum aestivum	2
#	Leycesteria formosa	4
#	Leymus arenarius	10
#	Ligularia dentata	1
#	Ligustrum ovalifolium	13
#	Ligustrum vulgare	15
#	Lilium martagon	5
#	Lilium pyrenaicum	2
#	Limnanthes douglasii	1
#	Limonium vulgare	5
#	Limosella aquatica	6
#	Linaria purpurea	1
#	Linaria repens	3
#	Linaria vulgaris	17
#	Linum bienne	7
#	Linum catharticum	5
#	Linum perenne	8
#	Linum usitatissimum	2
#	Lithospermum arvense	4
#	Lithospermum officinale	7
#	Lobelia erinus	2
#	Lobularia maritima	4
#	Lolium multiflorum	17
#	Lolium perenne	28
#	Lolium rigidum	9
#	Lolium temulentum	8
#	Lonicera caprifolium	0
#	Lonicera involucrata	0
#	Lonicera periclymenum	6
#	Lonicera xylosteum	1
#	Lotus corniculatus	21
#	Lotus pedunculatus	6
#	Lotus tenuis	3
#	Lunaria annua	1
#	Lupinus arboreus	1
#	Lupinus polyphyllus	2
#	Luzula campestris	9
#	Luzula luzuloides	0
#	Luzula multiflora	9
#	Luzula pilosa	6
#	Luzula sylvatica	6
#	Lycium barbarum	6
#	Lycium chinense	10
#	Lycopodiella inundata	2
#	Lycopodium clavatum	13
#	Lycopus europaeus	3
#	Lysimachia nemorum	3
#	Lysimachia nummularia	6
#	Lysimachia punctata	1
#	Lysimachia thyrsiflora	0
#	Lysimachia vulgaris	5
#	Lythrum hyssopifolia	2
#	Lythrum portula	4
#	Lythrum salicaria	16
#	Mahonia aquifolium	2
#	Malus pumila	16
#	Malus sylvestris	5
#	Malva arborea	3
#	Malva moschata	5
#	Malva neglecta	8
#	Malva parviflora	6
#	Malva pseudolavatera	1
#	Malva pusilla	3
#	Malva setigera	0
#	Malva sylvestris	14
#	Marrubium vulgare	8
#	Matricaria chamomilla	34
#	Matricaria discoidea	6
#	Matthiola sinuata	3
#	Medicago arabica	5
#	Medicago lupulina	20
#	Medicago polymorpha	7
#	Medicago sativa	18
#	Melampyrum pratense	7
#	Melica uniflora	9
#	Melilotus albus	15
#	Melilotus altissimus	6
#	Melilotus indicus	11
#	Melilotus officinalis	15
#	Melissa officinalis	10
#	Mentha aquatica	6
#	Mentha arvensis	7
#	Mentha pulegium	10
#	Mentha spicata	16
#	Menyanthes trifoliata	17
#	Mercurialis annua	9
#	Mercurialis perennis	10
#	Metasequoia glyptostroboides	13
#	Milium effusum	8
#	Minuartia hybrida	3
#	Misopates orontium	4
#	Moehringia trinervia	6
#	Molinia caerulea	9
#	Montia fontana	10
#	Muscari armeniacum	1
#	Myagrum perfoliatum	0
#	Mycelis muralis	6
#	Myosotis arvensis	5
#	Myosotis discolor	5
#	Myosotis laxa	8
#	Myosotis ramosissima	4
#	Myosotis scorpioides	4
#	Myosotis secunda	5
#	Myosotis sylvatica	6
#	Myosoton aquaticum	4
#	Myosurus minimus	3
#	Myrica gale	12
#	Myrrhis odorata	2
#	Narcissus poeticus	0
#	Narcissus pseudonarcissus	1
#	Nardus stricta	7
#	Narthecium ossifragum	9
#	Nasturtium microphyllum	7
#	Nasturtium officinale	13
#	Neotinea ustulata	1
#	Neottia cordata	2
#	Neottia nidus-avis	7
#	Neottia ovata	7
#	Nepeta cataria	12
#	Neslia paniculata	2
#	Nicandra physalodes	4
#	Nigella damascena	3
#	Nuphar lutea	5
#	Nymphaea alba	12
#	Nymphoides peltata	2
#	Odontites vernus	6
#	Oenanthe aquatica	3
#	Oenanthe fistulosa	2
#	Oenothera biennis	17
#	Oenothera glazioviana	7
#	Omphalodes verna	1
#	Onobrychis viciifolia	3
#	Onoclea sensibilis	18
#	Ononis repens	6
#	Ononis spinosa	7
#	Onopordum acanthium	4
#	Ophioglossum vulgatum	8
#	Ophrys apifera	8
#	Ophrys insectifera	3
#	Orchis mascula	4
#	Orchis simia	1
#	Oreopteris limbosperma	3
#	Origanum vulgare	20
#	Ornithogalum nutans	1
#	Ornithopus perpusillus	5
#	Orobanche elatior	1
#	Orobanche hederae	11
#	Orobanche minor	46
#	Orobanche rapum-genistae	8
#	Orobanche reticulata	0
#	Osmunda regalis	41
#	Oxalis acetosella	18
#	Oxalis articulata	3
#	Oxalis corniculata	13
#	Oxalis exilis	1
#	Oxalis incarnata	1
#	Oxalis pes-caprae	3
#	Oxalis stricta	13
#	Panicum capillare	13
#	Panicum miliaceum	8
#	Papaver argemone	2
#	Papaver dubium	5
#	Papaver hybridum	3
#	Papaver lecoqii	0
#	Papaver rhoeas	15
#	Papaver somniferum	10
#	Parapholis incurva	5
#	Parapholis strigosa	4
#	Parentucellia viscosa	7
#	Parietaria judaica	8
#	Paris quadrifolia	8
#	Parnassia palustris	29
#	Pastinaca sativa	7
#	Pedicularis palustris	3
#	Pedicularis sylvatica	5
#	Pentaglottis sempervirens	3
#	Persicaria amphibia	14
#	Persicaria bistorta	6
#	Persicaria capitata	5
#	Persicaria hydropiper	28
#	Persicaria lapathifolia	17
#	Persicaria maculosa	13
#	Persicaria minor	4
#	Persicaria pensylvanica	10
#	Persicaria wallichii	6
#	Petasites albus	2
#	Petasites fragrans	1
#	Petasites hybridus	5
#	Petroselinum crispum	8
#	Petroselinum segetum	2
#	Phacelia ciliata	0
#	Phacelia parviflora	0
#	Phacelia tanacetifolia	1
#	Phalaris aquatica	0
#	Phalaris arundinacea	26
#	Phalaris canariensis	1
#	Phalaris paradoxa	1
#	Phegopteris connectilis	12
#	Philadelphus coronarius	2
#	Phleum arenarium	3
#	Phleum bertolonii	4
#	Phleum pratense	23
#	Phragmites australis	45
#	Physalis alkekengi	3
#	Phytolacca acinosa	0
#	Picea abies	34
#	Picea sitchensis	10
#	Pilosella aurantiaca	8
#	Pilosella officinarum	10
#	Pilularia globulifera	5
#	Pimpinella major	1
#	Pimpinella saxifraga	8
#	Pinguicula vulgaris	21
#	Pinus contorta	9
#	Pinus nigra	15
#	Pinus sylvestris	27
#	Pisum sativum	32
#	Plantago arenaria	0
#	Plantago coronopus	12
#	Plantago lanceolata	18
#	Plantago major	24
#	Plantago maritima	9
#	Plantago media	10
#	Platanthera bifolia	2
#	Platanthera chlorantha	8
#	Poa angustifolia	4
#	Poa annua	27
#	Poa compressa	6
#	Poa humilis	5
#	Poa nemoralis	6
#	Poa pratensis	89
#	Poa trivialis	11
#	Polemonium caeruleum	3
#	Polygala calcarea	1
#	Polygala serpyllifolia	7
#	Polygala vulgaris	7
#	Polygonum arenarium	0
#	Polygonum arenastrum	4
#	Polygonum aviculare	21
#	Polygonum rurivagum	4
#	Polypodium interjectum	3
#	Polypodium vulgare	6
#	Polypogon monspeliensis	4
#	Polystichum aculeatum	2
#	Polystichum setiferum	3
#	Pontederia cordata	4
#	Populus alba	22
#	Populus nigra	164
#	Populus tremula	17
#	Populus trichocarpa	85
#	Portulaca oleracea	17
#	Potamogeton acutifolius	0
#	Potamogeton alpinus	9
#	Potamogeton berchtoldii	4
#	Potamogeton coloratus	3
#	Potamogeton compressus	6
#	Potamogeton crispus	18
#	Potamogeton friesii	2
#	Potamogeton gramineus	11
#	Potamogeton lucens	8
#	Potamogeton natans	16
#	Potamogeton obtusifolius	6
#	Potamogeton pectinatus	13
#	Potamogeton perfoliatus	22
#	Potamogeton polygonifolius	4
#	Potamogeton pusillus	17
#	Potamogeton trichoides	3
#	Potentilla anglica	4
#	Potentilla anserina	8
#	Potentilla argentea	4
#	Potentilla erecta	5
#	Potentilla intermedia	0
#	Potentilla norvegica	4
#	Potentilla recta	4
#	Potentilla reptans	9
#	Potentilla sterilis	5
#	Potentilla tabernaemontani	0
#	Poterium sanguisorba	9
#	Primula auricula	0
#	Primula pulverulenta	3
#	Primula veris	10
#	Primula vulgaris	7
#	Prunella vulgaris	33
#	Prunus avium	11
#	Prunus cerasifera	13
#	Prunus domestica	17
#	Prunus dulcis	8
#	Prunus laurocerasus	4
#	Prunus padus	15
#	Prunus spinosa	22
#	Pseudofumaria lutea	4
#	Pseudotsuga menziesii	9
#	Pteridium aquilinum	20
#	Pteris cretica	21
#	Pteris incompleta	1
#	Pteris tremula	5
#	Pteris vittata	47
#	Puccinellia distans	11
#	Puccinellia maritima	4
#	Puccinellia rupestris	2
#	Pulicaria dysenterica	4
#	Pulmonaria officinalis	2
#	Pyracantha coccinea	10
#	Pyrola minor	4
#	Pyrus communis	17
#	Pyrus pyraster	2
#	Quercus cerris	4
#	Quercus ilex	63
#	Quercus petraea	11
#	Quercus robur	19
#	Quercus rubra	19
#	Radiola linoides	6
#	Ranunculus acris	18
#	Ranunculus aquatilis	9
#	Ranunculus arvensis	1
#	Ranunculus auricomus	4
#	Ranunculus baudotii	2
#	Ranunculus bulbosus	10
#	Ranunculus circinatus	3
#	Ranunculus flammula	7
#	Ranunculus fluitans	1
#	Ranunculus hederaceus	1
#	Ranunculus lingua	5
#	Ranunculus peltatus	3
#	Ranunculus penicillatus	2
#	Ranunculus repens	10
#	Ranunculus sardous	3
#	Ranunculus sceleratus	8
#	Ranunculus trichophyllus	5
#	Ranunculus water-crowfoot	0
#	Raphanus sativus	37
#	Rapistrum rugosum	4
#	Reseda alba	3
#	Reseda lutea	8
#	Reseda luteola	5
#	Rhamnus cathartica	23
#	Rhinanthus angustifolius	0
#	Rhinanthus minor	8
#	Rhododendron ponticum	4
#	Rhus typhina	8
#	Ribes alpinum	5
#	Ribes nigrum	1
#	Ribes rubrum	8
#	Ribes sanguineum	0
#	Ribes uva-crispa	1
#	Ricinus communis	111
#	Robinia pseudoacacia	14
#	Rorippa amphibia	3
#	Rorippa palustris	5
#	Rorippa sylvestris	5
#	Rosa arvensis	5
#	Rosa caesia	5
#	Rosa canina	10
#	Rosa micrantha	4
#	Rosa mollis	2
#	Rosa multiflora	12
#	Rosa obtusifolia	3
#	Rosa rubiginosa	4
#	Rosa rugosa	6
#	Rosa sherardii	4
#	Rosa spinosissima	3
#	Rosmarinus officinalis	12
#	Rubus armeniacus	0
#	Rubus caesius	5
#	Rubus dasyphyllus	0
#	Rubus eboracensis	0
#	Rubus echinatoides	0
#	Rubus fruticosus	4
#	Rubus halsteadensis	0
#	Rubus idaeus	20
#	Rubus nemoralis	0
#	Rubus polyanthemus	0
#	Rubus saxatilis	1
#	Rubus spectabilis	1
#	Rubus sprengelii	0
#	Rubus ulmifolius	4
#	Rubus warrenii	0
#	Rudbeckia laciniata	5
#	Rumex acetosa	20
#	Rumex acetosella	12
#	Rumex conglomeratus	6
#	Rumex crispus	22
#	Rumex hydrolapathum	5
#	Rumex obtusifolius	18
#	Rumex sanguineus	4
#	Ruppia cirrhosa	8
#	Ruppia maritima	48
#	Sagina apetala	5
#	Sagina filicaulis	0
#	Sagina maritima	3
#	Sagina nodosa	3
#	Sagina procumbens	4
#	Sagittaria sagittifolia	2
#	Salicornia dolichostachya	4
#	Salicornia europaea	8
#	Salicornia fragilis	1
#	Salicornia ramosissima	3
#	Salix alba	19
#	Salix aurita	6
#	Salix caprea	13
#	Salix cinerea	10
#	Salix pentandra	10
#	Salix purpurea	12
#	Salix repens	6
#	Salix triandra	11
#	Salix viminalis	8
#	Salsola kali	8
#	Salvia pratensis	8
#	Salvia reflexa	0
#	Salvia verbenaca	5
#	Salvia verticillata	1
#	Sambucus nigra	35
#	Samolus valerandi	6
#	Sanguisorba officinalis	15
#	Sanicula europaea	5
#	Saponaria officinalis	14
#	Saxifraga granulata	2
#	Saxifraga hirsuta	2
#	Saxifraga hypnoides	3
#	Saxifraga tridactylites	3
#	Scabiosa atropurpurea	1
#	Scabiosa columbaria	7
#	Scandix pecten-veneris	3
#	Schedonorus arundinaceus	30
#	Schedonorus giganteus	6
#	Schedonorus pratensis	16
#	Schoenoplectus lacustris	6
#	Schoenoplectus tabernaemontani	12
#	Schoenus nigricans	7
#	Scilla bifolia	1
#	Scilla forbesii	1
#	Scirpus sylvaticus	8
#	Scleranthus annuus	4
#	Scolymus hispanicus	1
#	Scorzoneroides autumnalis	6
#	Scrophularia auriculata	5
#	Scrophularia nodosa	2
#	Scutellaria altissima	1
#	Scutellaria galericulata	6
#	Scutellaria minor	4
#	Securigera varia	5
#	Selaginella selaginoides	3
#	Senecio erucifolius	5
#	Senecio inaequidens	3
#	Senecio squalidus	0
#	Senecio sylvaticus	3
#	Senecio viscosus	0
#	Senecio vulgaris	11
#	Serratula tinctoria	3
#	Sesleria caerulea	3
#	Setaria italica	83
#	Setaria pumila	12
#	Setaria verticillata	9
#	Setaria viridis	16
#	Sherardia arvensis	9
#	Sidalcea candida	0
#	Sidalcea malviflora	0
#	Silaum silaus	3
#	Silene dioica	22
#	Silene flos-cuculi	7
#	Silene gallica	5
#	Silene latifolia	73
#	Silene noctiflora	5
#	Silene vulgaris	17
#	Sinapis arvensis	10
#	Sison amomum	5
#	Sisymbrium altissimum	2
#	Sisymbrium irio	1
#	Sisymbrium loeselii	1
#	Sisymbrium officinale	12
#	Sisymbrium orientale	3
#	Sisymbrium strictissimum	0
#	Sisymbrium volgense	0
#	Sisyrinchium montanum	10
#	Sium latifolium	3
#	Smyrnium olusatrum	2
#	Solanum chenopodioides	3
#	Solanum dulcamara	16
#	Solanum lycopersicum	139
#	Solanum nigrum	31
#	Solanum physalifolium	2
#	Solanum rostratum	1
#	Solanum sarachoides	0
#	Solanum sisymbriifolium	1
#	Solanum tuberosum	89
#	Soleirolia soleirolii	5
#	Solidago canadensis	18
#	Solidago gigantea	7
#	Solidago graminifolia	7
#	Solidago virgaurea	6
#	Sonchus arvensis	11
#	Sonchus asper	13
#	Sonchus oleraceus	15
#	Sorbus aria	8
#	Sorbus aucuparia	16
#	Sorbus intermedia	7
#	Sorbus latifolia	0
#	Sorbus torminalis	7
#	Sparganium emersum	12
#	Sparganium erectum	6
#	Sparganium natans	7
#	Spartina anglica	6
#	Spergula arvensis	5
#	Spergularia marina	5
#	Spergularia media	3
#	Spergularia rubra	4
#	Spilanthes decumbens	0
#	Spiraea alba	2
#	Spiraea douglasii	0
#	Spiraea salicifolia	5
#	Spiranthes spiralis	3
#	Spirodela polyrhiza	21
#	Stachys annua	0
#	Stachys arvensis	5
#	Stachys byzantina	4
#	Stachys palustris	5
#	Stachys sylvatica	11
#	Stellaria alsine	3
#	Stellaria graminea	6
#	Stellaria holostea	5
#	Stellaria media	16
#	Stellaria neglecta	2
#	Stellaria nemorum	3
#	Stellaria pallida	3
#	Stellaria palustris	4
#	Stratiotes aloides	5
#	Suaeda maritima	13
#	Succisa pratensis	4
#	Symphoricarpos albus	1
#	Symphoricarpos orbiculatus	2
#	Symphytum bulbosum	0
#	Symphytum officinale	10
#	Symphytum orientale	1
#	Syringa vulgaris	9
#	Tagetes minuta	4
#	Tamarix gallica	2
#	Tamus communis	15
#	Tanacetum parthenium	4
#	Tanacetum vulgare	7
#	Taraxacum aequilobum	0
#	Taraxacum alatum	0
#	Taraxacum ancistrolobum	0
#	Taraxacum arenastrum	0
#	Taraxacum argutum	0
#	Taraxacum aurosulum	0
#	Taraxacum boekmanii	0
#	Taraxacum brachyglossum	0
#	Taraxacum bracteatum	0
#	Taraxacum cordatum	0
#	Taraxacum croceiflorum	0
#	Taraxacum dilatatum	0
#	Taraxacum duplidentifrons	0
#	Taraxacum euryphyllum	0
#	Taraxacum exacutum	0
#	Taraxacum expallidiforme	0
#	Taraxacum faeroense	0
#	Taraxacum fasciatum	0
#	Taraxacum fulgidum	0
#	Taraxacum fulviforme	0
#	Taraxacum haematicum	0
#	Taraxacum hamatiforme	0
#	Taraxacum hamatum	0
#	Taraxacum huelphersianum	0
#	Taraxacum insigne	0
#	Taraxacum interveniens	0
#	Taraxacum kernianum	0
#	Taraxacum laciniosifrons	0
#	Taraxacum lacistophyllum	0
#	Taraxacum lamprophyllum	0
#	Taraxacum latisectum	0
#	Taraxacum lingulatum	0
#	Taraxacum maculosum	0
#	Taraxacum marklundii	0
#	Taraxacum nordstedtii	0
#	Taraxacum ochrochlorum	0
#	Taraxacum ostenfeldii	0
#	Taraxacum oxoniense	0
#	Taraxacum pachymerum	0
#	Taraxacum pallescens	0
#	Taraxacum palustre	0
#	Taraxacum pannucium	0
#	Taraxacum polyodon	0
#	Taraxacum proximum	0
#	Taraxacum pseudohamatum	0
#	Taraxacum pseudolarssonii	0
#	Taraxacum quadrans	0
#	Taraxacum richardsianum	0
#	Taraxacum rubicundum	0
#	Taraxacum sagittipotens	0
#	Taraxacum sellandii	0
#	Taraxacum sinuatum	0
#	Taraxacum stenacrum	0
#	Taraxacum stictophyllum	0
#	Taraxacum subcyanolepis	0
#	Taraxacum subhamatum	0
#	Taraxacum sublaeticolor	0
#	Taraxacum subundulatum	0
#	Taraxacum trilobatum	0
#	Taraxacum undulatiflorum	0
#	Taraxacum xanthostigma	0
#	Taxus baccata	24
#	Teesdalia nudicaulis	1
#	Tellima grandiflora	2
#	Teucrium scorodonia	7
#	Thalictrum flavum	3
#	Thelypteris palustris	14
#	Thlaspi arvense	13
#	Thuja plicata	4
#	Thymus polytrichus	8
#	Thymus pulegioides	10
#	Thyselium palustre	0
#	Tilia cordata	6
#	Tilia platyphyllos	6
#	Tolmiea menziesii	1
#	Torilis arvensis	7
#	Torilis japonica	8
#	Torilis nodosa	4
#	Tragopogon porrifolius	2
#	Tragopogon pratensis	11
#	Trichophorum germanicum	0
#	Trifolium arvense	8
#	Trifolium campestre	11
#	Trifolium dubium	9
#	Trifolium fragiferum	4
#	Trifolium hybridum	7
#	Trifolium medium	4
#	Trifolium micranthum	3
#	Trifolium pratense	30
#	Trifolium repens	24
#	Triglochin maritima	21
#	Triglochin palustris	14
#	Tripleurospermum inodorum	4
#	Tripleurospermum maritimum	26
#	Trisetum flavescens	7
#	Triticum aestivum	70
#	Triticum spelta	9
#	Tropaeolum majus	6
#	Tsuga heterophylla	1
#	Tulipa gesneriana	1
#	Turgenia latifolia	1
#	Tussilago farfara	15
#	Typha latifolia	14
#	Ulex europaeus	5
#	Ulmus glabra	12
#	Ulmus minor	8
#	Ulmus plotii	0
#	Ulmus procera	6
#	Urtica dioica	16
#	Urtica urens	8
#	Utricularia minor	5
#	Utricularia vulgaris	2
#	Vaccaria hispanica	3
#	Vaccinium myrtillus	11
#	Vaccinium oxycoccos	4
#	Vaccinium vitis-idaea	24
#	Valeriana dioica	7
#	Valeriana officinalis	16
#	Valeriana pyrenaica	3
#	Valerianella carinata	3
#	Valerianella dentata	4
#	Valerianella eriocarpa	3
#	Valerianella locusta	7
#	Valerianella rimosa	1
#	Verbascum blattaria	0
#	Verbascum nigrum	5
#	Verbascum phlomoides	1
#	Verbascum thapsus	14
#	Verbascum virgatum	3
#	Verbena bonariensis	7
#	Verbena hastata	7
#	Verbena officinalis	13
#	Veronica agrestis	5
#	Veronica anagallis-aquatica	6
#	Veronica arvensis	9
#	Veronica beccabunga	4
#	Veronica catenata	4
#	Veronica chamaedrys	8
#	Veronica filiformis	1
#	Veronica hederifolia	3
#	Veronica longifolia	0
#	Veronica montana	4
#	Veronica officinalis	13
#	Veronica persica	9
#	Veronica polita	2
#	Veronica scutellata	4
#	Veronica serpyllifolia	14
#	Veronica spicata	5
#	Viburnum lantana	9
#	Vicia cracca	27
#	Vicia faba	18
#	Vicia hirsuta	8
#	Vicia lathyroides	2
#	Vicia lutea	4
#	Vicia sativa	35
#	Vicia sepium	6
#	Vicia sylvatica	5
#	Vicia tetrasperma	17
#	Vinca major	1
#	Vinca minor	12
#	Viola arvensis	9
#	Viola canina	5
#	Viola hirta	4
#	Viola odorata	6
#	Viola palustris	5
#	Viola reichenbachiana	5
#	Viola riviniana	10
#	Viola tricolor	4
#	Viscum album	8
#	Vitis vinifera	129
#	Vulpia bromoides	7
#	Vulpia myuros	14
#	Xanthium spinosum	1
#	Xanthium strumarium	3
#	Zannichellia palustris	13
#	Zantedeschia aethiopica	13
#	Zea mays	117
#	Zostera marina	33
#	Zostera noltei	5

total number of accessions fetched: 12687


downloading 12687 records .. processing 1000 accessions per batch

[Thu Oct 27 2016 16:04:19]	batch: 0000001	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000001.gb	(total: 1000 (7.88 %); remaining: 11687 (92.12 %) -&gt; eyorks_flora_curated.accessions.txt)
[Thu Oct 27 2016 16:05:09]	batch: 0000002	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000002.gb	(total: 2000 (15.76 %); remaining: 10687 (84.24 %) -&gt; eyorks_flora_curated.accessions.txt)
[Thu Oct 27 2016 16:06:54]	batch: 0000003	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000003.gb	(total: 3000 (23.65 %); remaining: 9687 (76.35 %) -&gt; eyorks_flora_curated.accessions.txt)
[Thu Oct 27 2016 16:08:06]	batch: 0000004	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000004.gb	(total: 4000 (31.53 %); remaining: 8687 (68.47 %) -&gt; eyorks_flora_curated.accessions.txt)
[Thu Oct 27 2016 16:08:56]	batch: 0000005	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000005.gb	(total: 5000 (39.41 %); remaining: 7687 (60.59 %) -&gt; eyorks_flora_curated.accessions.txt)
[Thu Oct 27 2016 16:10:09]	batch: 0000006	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000006.gb	(total: 6000 (47.29 %); remaining: 6687 (52.71 %) -&gt; eyorks_flora_curated.accessions.txt)
[Thu Oct 27 2016 16:11:01]	batch: 0000007	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000007.gb	(total: 7000 (55.17 %); remaining: 5687 (44.83 %) -&gt; eyorks_flora_curated.accessions.txt)
[Thu Oct 27 2016 16:12:05]	batch: 0000008	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000008.gb	(total: 8000 (63.06 %); remaining: 4687 (36.94 %) -&gt; eyorks_flora_curated.accessions.txt)
[Thu Oct 27 2016 16:14:05]	batch: 0000009	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000009.gb	(total: 9000 (70.94 %); remaining: 3687 (29.06 %) -&gt; eyorks_flora_curated.accessions.txt)
[Thu Oct 27 2016 16:15:37]	batch: 0000010	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000010.gb	(total: 10000 (78.82 %); remaining: 2687 (21.18 %) -&gt; eyorks_flora_curated.accessions.txt)
[Thu Oct 27 2016 16:16:30]	batch: 0000011	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000011.gb	(total: 11000 (86.70 %); remaining: 1687 (13.30 %) -&gt; eyorks_flora_curated.accessions.txt)
[Thu Oct 27 2016 16:17:37]	batch: 0000012	1000 unique records downloaded	-&gt; eyorks_flora_curated.0000012.gb	(total: 12000 (94.59 %); remaining: 687 (5.41 %) -&gt; eyorks_flora_curated.accessions.txt)
[Thu Oct 27 2016 16:52:46]	batch: 0000013	687 unique records downloaded	-&gt; eyorks_flora_curated.0000013.gb	(total: 12687 (100.00 %); remaining: 0 -&gt; DONE!)

Concatenate all files	-&gt; eyorks_flora_curated.gb

</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>At this point I will put this reference database back through SATIVA (which will also remove the records in the AccessionBlacklist) and place the final database in this directory for blasting.</p>
<p>Just to keep things separate, I'm going to copy the SATIVA notebook and files into a new subdirectory within this one and run it separately from the previous version of SATIVA.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>The above is now complete and ready for BLASTing. We need an updated Querymap and REFmap:</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[110]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

<span class="k">for </span>sample in <span class="k">$(</span>cat ../Querymap.txt | cut -f 1<span class="k">)</span>
<span class="k">do</span>
<span class="k">    </span><span class="nv">fasta</span><span class="o">=</span><span class="k">$(</span>ls -1 ../../../<span class="nv">$sample</span>/<span class="nv">$sample</span><span class="se">\_</span>trimmed.fasta<span class="k">)</span>
    <span class="nb">echo</span> -e <span class="s2">&quot;$sample\tfasta\t$fasta&quot;</span>
<span class="k">done</span> &gt; Querymap.txt

head Querymap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>SA177	fasta	../../../SA177/SA177_trimmed.fasta
SB140	fasta	../../../SB140/SB140_trimmed.fasta
T19	fasta	../../../T19/T19_trimmed.fasta
Lg45	fasta	../../../Lg45/Lg45_trimmed.fasta
Q69	fasta	../../../Q69/Q69_trimmed.fasta
S72	fasta	../../../S72/S72_trimmed.fasta
SC66	fasta	../../../SC66/SC66_trimmed.fasta
S31	fasta	../../../S31/S31_trimmed.fasta
SC74	fasta	../../../SC74/SC74_trimmed.fasta
SA68	fasta	../../../SA68/SA68_trimmed.fasta
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[111]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span><span class="nb">echo</span> <span class="s2">&quot;eyorks_flora_final.gb\tgb&quot;</span> &gt; REFmap.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>And the new curated reference db, which has been run through SATIVA:</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[117]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="n">Sativa</span><span class="o">/</span><span class="n">eyorks_flora_final</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[118]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>AccessionBlacklist.txt          <span class="ansibold ansiblue">R9</span>/                     <span class="ansibold ansiblue">SA147</span>/   <span class="ansibold ansiblue">SB104</span>/  <span class="ansibold ansiblue">SB82</span>/
eyorks_flora_curated.gb         <span class="ansibold ansiblue">R90</span>/                    <span class="ansibold ansiblue">SA15</span>/    <span class="ansibold ansiblue">SB106</span>/  <span class="ansibold ansiblue">SB83</span>/
eyorks_flora_final.gb           <span class="ansibold ansiblue">R95</span>/                    <span class="ansibold ansiblue">SA153</span>/   <span class="ansibold ansiblue">SB111</span>/  <span class="ansibold ansiblue">SB85</span>/
<span class="ansibold ansiblue">GLOBAL</span>/                         <span class="ansibold ansiblue">R98</span>/                    <span class="ansibold ansiblue">SA155</span>/   <span class="ansibold ansiblue">SB116</span>/  <span class="ansibold ansiblue">SB86</span>/
<span class="ansibold ansiblue">Lg27</span>/                           <span class="ansibold ansiblue">R99</span>/                    <span class="ansibold ansiblue">SA156</span>/   <span class="ansibold ansiblue">SB119</span>/  <span class="ansibold ansiblue">SB87</span>/
<span class="ansibold ansiblue">Lg37</span>/                           <span class="ansibold ansigreen">RefListCuratFinal.txt</span>*  <span class="ansibold ansiblue">SA157</span>/   <span class="ansibold ansiblue">SB120</span>/  <span class="ansibold ansiblue">SB91</span>/
<span class="ansibold ansiblue">Lg45</span>/                           REFmap.txt              <span class="ansibold ansiblue">SA160</span>/   <span class="ansibold ansiblue">SB127</span>/  <span class="ansibold ansiblue">SB93</span>/
<span class="ansibold ansiblue">Lg53</span>/                           <span class="ansibold ansiblue">S100</span>/                   <span class="ansibold ansiblue">SA162</span>/   <span class="ansibold ansiblue">SB128</span>/  <span class="ansibold ansiblue">SB99</span>/
<span class="ansibold ansiblue">Lg57</span>/                           <span class="ansibold ansiblue">S101</span>/                   <span class="ansibold ansiblue">SA164</span>/   <span class="ansibold ansiblue">SB129</span>/  <span class="ansibold ansiblue">SC10</span>/
<span class="ansibold ansiblue">Lg65</span>/                           <span class="ansibold ansiblue">S102</span>/                   <span class="ansibold ansiblue">SA165</span>/   <span class="ansibold ansiblue">SB13</span>/   <span class="ansibold ansiblue">SC101</span>/
<span class="ansibold ansiblue">Lg9</span>/                            <span class="ansibold ansiblue">S11</span>/                    <span class="ansibold ansiblue">SA166</span>/   <span class="ansibold ansiblue">SB131</span>/  <span class="ansibold ansiblue">SC106</span>/
log_final                       <span class="ansibold ansiblue">S13</span>/                    <span class="ansibold ansiblue">SA171</span>/   <span class="ansibold ansiblue">SB135</span>/  <span class="ansibold ansiblue">SC107</span>/
MothPollenFinal_read_stats.csv  <span class="ansibold ansiblue">S18</span>/                    <span class="ansibold ansiblue">SA172</span>/   <span class="ansibold ansiblue">SB136</span>/  <span class="ansibold ansiblue">SC110</span>/
<span class="ansibold ansiblue">NEG</span>/                            <span class="ansibold ansiblue">S19</span>/                    <span class="ansibold ansiblue">SA174</span>/   <span class="ansibold ansiblue">SB138</span>/  <span class="ansibold ansiblue">SC113</span>/
<span class="ansibold ansiblue">O84</span>/                            <span class="ansibold ansiblue">S24</span>/                    <span class="ansibold ansiblue">SA177</span>/   <span class="ansibold ansiblue">SB139</span>/  <span class="ansibold ansiblue">SC117</span>/
<span class="ansibold ansiblue">O96</span>/                            <span class="ansibold ansiblue">S30</span>/                    <span class="ansibold ansiblue">SA180</span>/   <span class="ansibold ansiblue">SB140</span>/  <span class="ansibold ansiblue">SC119</span>/
<span class="ansibold ansiblue">P18</span>/                            <span class="ansibold ansiblue">S31</span>/                    <span class="ansibold ansiblue">SA182</span>/   <span class="ansibold ansiblue">SB141</span>/  <span class="ansibold ansiblue">SC122</span>/
<span class="ansibold ansiblue">P54</span>/                            <span class="ansibold ansiblue">S33</span>/                    <span class="ansibold ansiblue">SA189</span>/   <span class="ansibold ansiblue">SB142</span>/  <span class="ansibold ansiblue">SC124</span>/
<span class="ansibold ansiblue">P59</span>/                            <span class="ansibold ansiblue">S35</span>/                    <span class="ansibold ansiblue">SA19</span>/    <span class="ansibold ansiblue">SB146</span>/  <span class="ansibold ansiblue">SC125</span>/
<span class="ansibold ansiblue">P60</span>/                            <span class="ansibold ansiblue">S39</span>/                    <span class="ansibold ansiblue">SA190</span>/   <span class="ansibold ansiblue">SB147</span>/  <span class="ansibold ansiblue">SC141</span>/
<span class="ansibold ansiblue">P73</span>/                            <span class="ansibold ansiblue">S43</span>/                    <span class="ansibold ansiblue">SA196</span>/   <span class="ansibold ansiblue">SB15</span>/   <span class="ansibold ansiblue">SC142</span>/
<span class="ansibold ansiblue">P98</span>/                            <span class="ansibold ansiblue">S50</span>/                    <span class="ansibold ansiblue">SA198</span>/   <span class="ansibold ansiblue">SB150</span>/  <span class="ansibold ansiblue">SC143</span>/
<span class="ansibold ansiblue">POS1</span>/                           <span class="ansibold ansiblue">S56</span>/                    <span class="ansibold ansiblue">SA199</span>/   <span class="ansibold ansiblue">SB153</span>/  <span class="ansibold ansiblue">SC157</span>/
<span class="ansibold ansiblue">POS2</span>/                           <span class="ansibold ansiblue">S59</span>/                    <span class="ansibold ansiblue">SA2</span>/     <span class="ansibold ansiblue">SB159</span>/  <span class="ansibold ansiblue">SC16</span>/
<span class="ansibold ansiblue">POS3</span>/                           <span class="ansibold ansiblue">S62</span>/                    <span class="ansibold ansiblue">SA202</span>/   <span class="ansibold ansiblue">SB163</span>/  <span class="ansibold ansiblue">SC168</span>/
<span class="ansibold ansiblue">POS4</span>/                           <span class="ansibold ansiblue">S63</span>/                    <span class="ansibold ansiblue">SA21</span>/    <span class="ansibold ansiblue">SB166</span>/  <span class="ansibold ansiblue">SC178</span>/
<span class="ansibold ansiblue">PROC</span>/                           <span class="ansibold ansiblue">S65</span>/                    <span class="ansibold ansiblue">SA28</span>/    <span class="ansibold ansiblue">SB170</span>/  <span class="ansibold ansiblue">SC187</span>/
<span class="ansibold ansiblue">Q10</span>/                            <span class="ansibold ansiblue">S67</span>/                    <span class="ansibold ansiblue">SA31</span>/    <span class="ansibold ansiblue">SB171</span>/  <span class="ansibold ansiblue">SC198</span>/
<span class="ansibold ansiblue">Q30</span>/                            <span class="ansibold ansiblue">S7</span>/                     <span class="ansibold ansiblue">SA32</span>/    <span class="ansibold ansiblue">SB172</span>/  <span class="ansibold ansiblue">SC19A</span>/
<span class="ansibold ansiblue">Q43</span>/                            <span class="ansibold ansiblue">S72</span>/                    <span class="ansibold ansiblue">SA33</span>/    <span class="ansibold ansiblue">SB180</span>/  <span class="ansibold ansiblue">SC202</span>/
<span class="ansibold ansiblue">Q64</span>/                            <span class="ansibold ansiblue">S77</span>/                    <span class="ansibold ansiblue">SA36</span>/    <span class="ansibold ansiblue">SB182</span>/  <span class="ansibold ansiblue">SC26</span>/
<span class="ansibold ansiblue">Q69</span>/                            <span class="ansibold ansiblue">S79</span>/                    <span class="ansibold ansiblue">SA43</span>/    <span class="ansibold ansiblue">SB183</span>/  <span class="ansibold ansiblue">SC28</span>/
<span class="ansibold ansiblue">Q83</span>/                            <span class="ansibold ansiblue">S8</span>/                     <span class="ansibold ansiblue">SA46</span>/    <span class="ansibold ansiblue">SB186</span>/  <span class="ansibold ansiblue">SC34</span>/
Querymap.txt                    <span class="ansibold ansiblue">S80</span>/                    <span class="ansibold ansiblue">SA47</span>/    <span class="ansibold ansiblue">SB194</span>/  <span class="ansibold ansiblue">SC35</span>/
<span class="ansibold ansiblue">R1</span>/                             <span class="ansibold ansiblue">S81</span>/                    <span class="ansibold ansiblue">SA48</span>/    <span class="ansibold ansiblue">SB195</span>/  <span class="ansibold ansiblue">SC43</span>/
<span class="ansibold ansiblue">R10</span>/                            <span class="ansibold ansiblue">S83</span>/                    <span class="ansibold ansiblue">SA49</span>/    <span class="ansibold ansiblue">SB197</span>/  <span class="ansibold ansiblue">SC44</span>/
<span class="ansibold ansiblue">R102</span>/                           <span class="ansibold ansiblue">S9</span>/                     <span class="ansibold ansiblue">SA5</span>/     <span class="ansibold ansiblue">SB199</span>/  <span class="ansibold ansiblue">SC48</span>/
<span class="ansibold ansiblue">R13</span>/                            <span class="ansibold ansiblue">S90</span>/                    <span class="ansibold ansiblue">SA50</span>/    <span class="ansibold ansiblue">SB200</span>/  <span class="ansibold ansiblue">SC49</span>/
<span class="ansibold ansiblue">R15</span>/                            <span class="ansibold ansiblue">S92</span>/                    <span class="ansibold ansiblue">SA51</span>/    <span class="ansibold ansiblue">SB201</span>/  <span class="ansibold ansiblue">SC51</span>/
<span class="ansibold ansiblue">R16</span>/                            <span class="ansibold ansiblue">S96</span>/                    <span class="ansibold ansiblue">SA53</span>/    <span class="ansibold ansiblue">SB204</span>/  <span class="ansibold ansiblue">SC51A</span>/
<span class="ansibold ansiblue">R20</span>/                            <span class="ansibold ansiblue">S97</span>/                    <span class="ansibold ansiblue">SA55</span>/    <span class="ansibold ansiblue">SB21</span>/   <span class="ansibold ansiblue">SC54A</span>/
<span class="ansibold ansiblue">R22</span>/                            <span class="ansibold ansiblue">SA1</span>/                    <span class="ansibold ansiblue">SA58</span>/    <span class="ansibold ansiblue">SB22</span>/   <span class="ansibold ansiblue">SC55</span>/
<span class="ansibold ansiblue">R23</span>/                            <span class="ansibold ansiblue">SA104</span>/                  <span class="ansibold ansiblue">SA62</span>/    <span class="ansibold ansiblue">SB23</span>/   <span class="ansibold ansiblue">SC57</span>/
<span class="ansibold ansiblue">R27</span>/                            <span class="ansibold ansiblue">SA105</span>/                  <span class="ansibold ansiblue">SA64</span>/    <span class="ansibold ansiblue">SB24</span>/   <span class="ansibold ansiblue">SC60</span>/
<span class="ansibold ansiblue">R29</span>/                            <span class="ansibold ansiblue">SA109</span>/                  <span class="ansibold ansiblue">SA67</span>/    <span class="ansibold ansiblue">SB27</span>/   <span class="ansibold ansiblue">SC61</span>/
<span class="ansibold ansiblue">R30</span>/                            <span class="ansibold ansiblue">SA111</span>/                  <span class="ansibold ansiblue">SA68</span>/    <span class="ansibold ansiblue">SB29</span>/   <span class="ansibold ansiblue">SC66</span>/
<span class="ansibold ansiblue">R31</span>/                            <span class="ansibold ansiblue">SA112</span>/                  <span class="ansibold ansiblue">SA7</span>/     <span class="ansibold ansiblue">SB3</span>/    <span class="ansibold ansiblue">SC7</span>/
<span class="ansibold ansiblue">R32</span>/                            <span class="ansibold ansiblue">SA113</span>/                  <span class="ansibold ansiblue">SA70</span>/    <span class="ansibold ansiblue">SB33</span>/   <span class="ansibold ansiblue">SC72</span>/
<span class="ansibold ansiblue">R34</span>/                            <span class="ansibold ansiblue">SA114</span>/                  <span class="ansibold ansiblue">SA73</span>/    <span class="ansibold ansiblue">SB34</span>/   <span class="ansibold ansiblue">SC74</span>/
<span class="ansibold ansiblue">R36</span>/                            <span class="ansibold ansiblue">SA115</span>/                  <span class="ansibold ansiblue">SA75</span>/    <span class="ansibold ansiblue">SB37</span>/   <span class="ansibold ansiblue">SC77</span>/
<span class="ansibold ansiblue">R39</span>/                            <span class="ansibold ansiblue">SA116</span>/                  <span class="ansibold ansiblue">SA76</span>/    <span class="ansibold ansiblue">SB4</span>/    <span class="ansibold ansiblue">SC78</span>/
<span class="ansibold ansiblue">R4</span>/                             <span class="ansibold ansiblue">SA117</span>/                  <span class="ansibold ansiblue">SA81</span>/    <span class="ansibold ansiblue">SB45</span>/   <span class="ansibold ansiblue">SC81</span>/
<span class="ansibold ansiblue">R40</span>/                            <span class="ansibold ansiblue">SA119</span>/                  <span class="ansibold ansiblue">SA86</span>/    <span class="ansibold ansiblue">SB48</span>/   <span class="ansibold ansiblue">SC82</span>/
<span class="ansibold ansiblue">R43</span>/                            <span class="ansibold ansiblue">SA120</span>/                  <span class="ansibold ansiblue">SA87</span>/    <span class="ansibold ansiblue">SB51</span>/   <span class="ansibold ansiblue">SC86A</span>/
<span class="ansibold ansiblue">R44</span>/                            <span class="ansibold ansiblue">SA121</span>/                  <span class="ansibold ansiblue">SA88</span>/    <span class="ansibold ansiblue">SB53</span>/   <span class="ansibold ansiblue">SC88</span>/
<span class="ansibold ansiblue">R45</span>/                            <span class="ansibold ansiblue">SA122</span>/                  <span class="ansibold ansiblue">SA89</span>/    <span class="ansibold ansiblue">SB59</span>/   <span class="ansibold ansiblue">SC9</span>/
<span class="ansibold ansiblue">R48</span>/                            <span class="ansibold ansiblue">SA124</span>/                  <span class="ansibold ansiblue">SA9</span>/     <span class="ansibold ansiblue">SB60</span>/   <span class="ansibold ansiblue">SC93</span>/
<span class="ansibold ansiblue">R49</span>/                            <span class="ansibold ansiblue">SA125</span>/                  <span class="ansibold ansiblue">SA91</span>/    <span class="ansibold ansiblue">SB63</span>/   <span class="ansibold ansiblue">T19</span>/
<span class="ansibold ansiblue">R59</span>/                            <span class="ansibold ansiblue">SA126</span>/                  <span class="ansibold ansiblue">SA93</span>/    <span class="ansibold ansiblue">SB64</span>/   <span class="ansibold ansiblue">T21</span>/
<span class="ansibold ansiblue">R60</span>/                            <span class="ansibold ansiblue">SA128</span>/                  <span class="ansibold ansiblue">SA94</span>/    <span class="ansibold ansiblue">SB67</span>/   <span class="ansibold ansiblue">T32</span>/
<span class="ansibold ansiblue">R67</span>/                            <span class="ansibold ansiblue">SA13</span>/                   <span class="ansibold ansiblue">SA95</span>/    <span class="ansibold ansiblue">SB71</span>/   <span class="ansibold ansiblue">T49</span>/
<span class="ansibold ansiblue">R68</span>/                            <span class="ansibold ansiblue">SA130</span>/                  <span class="ansibold ansiblue">SA96</span>/    <span class="ansibold ansiblue">SB73</span>/   <span class="ansibold ansiblue">T68</span>/
<span class="ansibold ansiblue">R71</span>/                            <span class="ansibold ansiblue">SA132</span>/                  <span class="ansibold ansiblue">SA97</span>/    <span class="ansibold ansiblue">SB76</span>/   <span class="ansibold ansiblue">T78</span>/
<span class="ansibold ansiblue">R74</span>/                            <span class="ansibold ansiblue">SA138</span>/                  <span class="ansibold ansiblue">SA98</span>/    <span class="ansibold ansiblue">SB77</span>/
<span class="ansibold ansiblue">R8</span>/                             <span class="ansibold ansiblue">SA140</span>/                  <span class="ansibold ansiblue">Sativa</span>/  <span class="ansibold ansiblue">SB8</span>/
<span class="ansibold ansiblue">R85</span>/                            <span class="ansibold ansiblue">SA144</span>/                  <span class="ansibold ansiblue">SB102</span>/   <span class="ansibold ansiblue">SB80</span>/
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[119]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-Q Querymap.txt <span class="se">\</span>
--cluster <span class="se">\</span>
--clust_match 1 <span class="se">\</span>
--clust_cov 5 <span class="se">\</span>
--blast <span class="se">\</span>
--bitscore_skim_LCA 0.03 <span class="se">\</span>
--bitscore_skim_adjust_off <span class="se">\</span>
--REFlist REFmap.txt <span class="se">\</span>
--min_ident 0.95 <span class="se">\</span>
-n 5 <span class="se">\</span>
-v <span class="se">\</span>
-@ callumjmacgregor@gmail.com <span class="se">\</span>
-o MothPollenFinal &gt; log_final
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[120]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash
blastn -query /home/working/AgainstRefDB/Curat1/FINAL/GLOBAL/global_centroids.fasta <span class="se">\</span>
-db /home/working/AgainstRefDB/Curat1/FINAL/GLOBAL/BLAST_0.95/marker_blast_db <span class="se">\</span>
-out /home/working/AgainstRefDB/Curat1/FINAL/GLOBAL/BLAST_0.95/global_blastn.out.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="Kraken">Kraken<a class="anchor-link" href="#Kraken">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Now this is all done, we want to try a different method, called Kraken, which is k-mer based. We will need our reference databases from here in the new folder.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[25]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="n">positives_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">Kraken</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[26]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="n">eyorks_flora_curated</span><span class="o">.</span><span class="n">gb</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">Kraken</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[27]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="n">Querymap</span><span class="o">.</span><span class="n">txt</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">Kraken</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[39]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cd</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">Kraken</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>/home/working/Kraken
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[40]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_curated.gb            log                             Querymap.txt
eyorks_flora_curated.kraken.fasta  positives_curated.gb            REFmap.txt
<span class="ansibold ansiblue">GLOBAL</span>/                            positives_curated.kraken.fasta
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Because of an error caused by the very large file size, we have manually constructed the Kraken database outside of the pipeline, using these steps:</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[&nbsp;]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="kn">from</span> <span class="nn">Bio</span> <span class="kn">import</span> <span class="n">SeqIO</span>

<span class="n">db_in_gb</span> <span class="o">=</span> <span class="s">&#39;eyorks_flora_curated.gb&#39;</span>
<span class="n">db_for_kraken</span> <span class="o">=</span> <span class="s">&#39;eyorks_flora_curated.kraken.fasta&#39;</span>

<span class="n">Seqs</span> <span class="o">=</span> <span class="n">SeqIO</span><span class="o">.</span><span class="n">parse</span><span class="p">(</span><span class="nb">open</span><span class="p">(</span><span class="n">db_in_gb</span><span class="p">,</span><span class="s">&#39;r&#39;</span><span class="p">),</span> <span class="s">&#39;genbank&#39;</span><span class="p">)</span>
<span class="n">Seqs_new</span> <span class="o">=</span> <span class="p">[]</span>

<span class="k">for</span> <span class="n">r</span> <span class="ow">in</span> <span class="n">Seqs</span><span class="p">:</span>
<span class="c">#    print r.id</span>
    <span class="n">source</span><span class="o">=</span><span class="n">r</span><span class="o">.</span><span class="n">features</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span>
    <span class="k">for</span> <span class="n">t</span> <span class="ow">in</span> <span class="n">source</span><span class="o">.</span><span class="n">qualifiers</span><span class="p">[</span><span class="s">&#39;db_xref&#39;</span><span class="p">]:</span>
        <span class="k">if</span> <span class="s">&#39;taxon&#39;</span> <span class="ow">in</span> <span class="n">t</span><span class="p">:</span>
            <span class="n">taxid</span> <span class="o">=</span> <span class="n">t</span><span class="o">.</span><span class="n">split</span><span class="p">(</span><span class="s">&quot;:&quot;</span><span class="p">)[</span><span class="mi">1</span><span class="p">]</span>
<span class="c">#            print taxid</span>
<span class="c">#    print &quot;%s|kraken:taxid|%s&quot; %(r.id,taxid)</span>
    <span class="n">r</span><span class="o">.</span><span class="n">id</span> <span class="o">=</span> <span class="s">&quot;</span><span class="si">%s</span><span class="s">|kraken:taxid|</span><span class="si">%s</span><span class="s">&quot;</span> <span class="o">%</span><span class="p">(</span><span class="n">r</span><span class="o">.</span><span class="n">id</span><span class="p">,</span><span class="n">taxid</span><span class="p">)</span>
    <span class="n">r</span><span class="o">.</span><span class="n">description</span> <span class="o">=</span> <span class="n">r</span><span class="o">.</span><span class="n">id</span>
    <span class="n">Seqs_new</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">r</span><span class="p">)</span>
    
<span class="n">out</span> <span class="o">=</span> <span class="nb">open</span><span class="p">(</span><span class="n">db_for_kraken</span><span class="p">,</span><span class="s">&#39;w&#39;</span><span class="p">)</span>
<span class="n">SeqIO</span><span class="o">.</span><span class="n">write</span><span class="p">(</span><span class="n">Seqs_new</span><span class="p">,</span> <span class="n">out</span><span class="p">,</span> <span class="s">&#39;fasta&#39;</span><span class="p">)</span>
<span class="n">out</span><span class="o">.</span><span class="n">close</span><span class="p">()</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[&nbsp;]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>kraken-build --download-taxonomy --db ./KRAKEN_DB/
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[&nbsp;]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>kraken-build --add-to-library eyorks_flora_curated.kraken.fasta --db ./KRAKEN_DB/
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[&nbsp;]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>kraken-build --build --threads 5 --db ./KRAKEN_DB/ --jellyfish-hash-size 2900M
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>And now we can run Kraken, using the same metaBEAT syntax</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[41]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-B ../AgainstRefDB/Curat1/GLOBAL/MothPollenReflistCurat1-OTU-denovo.biom <span class="se">\</span>
--kraken <span class="se">\</span>
--kraken_db GLOBAL/KRAKEN/KRAKEN_DB <span class="se">\</span>
-@ callumjmacgregor@gmail.com <span class="se">\</span>
-n 5 -o Kraken &gt; log
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Visual inspection of the Kraken output has revealed some very weird assignments, which are much less plausible than the equivalent BLAST assignments (e.g. species-level assignment to Silene gallica, as opposed to genus-level assignment to Silene due to confusion between S. latifolia and S. dioica. Therefore I will not pursue Kraken any further at this stage.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="Phylogenetic-placement-(pplacer)">Phylogenetic placement (pplacer)<a class="anchor-link" href="#Phylogenetic-placement-(pplacer)">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Now we have completed all the steps in the SATIVA process, we also have a cleaned, trimmed reference alignment that can be used to attempt assignments by phylogenetic placement!</p>
<p>We first need to build a reference tree, using the cleaned reference database and the final alignment.</p>
<p>Navigate to a new directory, copy over the relevant files. Then read in the Genbank file and create taxids.txt and seq_info.csv files.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[121]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cd</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">pplacer</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>/home/working/pplacer
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[122]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">AgainstRefDB</span><span class="o">/</span><span class="n">Curat1</span><span class="o">/</span><span class="n">FINAL</span><span class="o">/</span><span class="n">Sativa</span><span class="o">/</span><span class="n">postSATIVA_cleaned</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[123]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">AgainstRefDB</span><span class="o">/</span><span class="n">Curat1</span><span class="o">/</span><span class="n">FINAL</span><span class="o">/</span><span class="n">rbcL_eyorks_clean_alignment</span><span class="o">.</span><span class="n">phylip</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[127]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">AgainstRefDB</span><span class="o">/</span><span class="n">Curat1</span><span class="o">/</span><span class="n">FINAL</span><span class="o">/</span><span class="n">rbcL_eyorks_clean_alignment</span><span class="o">.</span><span class="n">fasta</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[124]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>postSATIVA_cleaned.gb  rbcL_eyorks_clean_alignment.phylip
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[125]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="kn">from</span> <span class="nn">Bio</span> <span class="kn">import</span> <span class="n">SeqIO</span>

<span class="n">gb</span> <span class="o">=</span> <span class="s">&#39;postSATIVA_cleaned.gb&#39;</span>
<span class="n">seqinfo_file</span> <span class="o">=</span> <span class="s">&#39;seq_info.csv&#39;</span>
<span class="n">taxid_file</span> <span class="o">=</span> <span class="s">&#39;taxids.txt&#39;</span>

<span class="n">seq_info</span> <span class="o">=</span> <span class="p">[</span><span class="s">&#39;&quot;seqname&quot;,&quot;accession&quot;,&quot;tax_id&quot;,&quot;species_name&quot;,&quot;is.type&quot;&#39;</span><span class="p">]</span>
<span class="n">taxids</span> <span class="o">=</span> <span class="p">[]</span>
<span class="n">Seqs</span> <span class="o">=</span> <span class="n">SeqIO</span><span class="o">.</span><span class="n">parse</span><span class="p">(</span><span class="nb">open</span><span class="p">(</span><span class="n">gb</span><span class="p">,</span><span class="s">&#39;r&#39;</span><span class="p">),</span> <span class="s">&#39;genbank&#39;</span><span class="p">)</span>

<span class="k">for</span> <span class="n">r</span> <span class="ow">in</span> <span class="n">Seqs</span><span class="p">:</span>
    <span class="n">sp</span> <span class="o">=</span> <span class="n">r</span><span class="o">.</span><span class="n">features</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span><span class="o">.</span><span class="n">qualifiers</span><span class="p">[</span><span class="s">&#39;organism&#39;</span><span class="p">][</span><span class="mi">0</span><span class="p">]</span>
    <span class="k">for</span> <span class="n">db_xref</span> <span class="ow">in</span> <span class="n">r</span><span class="o">.</span><span class="n">features</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span><span class="o">.</span><span class="n">qualifiers</span><span class="p">[</span><span class="s">&#39;db_xref&#39;</span><span class="p">]:</span>
        <span class="k">if</span> <span class="s">&#39;taxon&#39;</span> <span class="ow">in</span> <span class="n">db_xref</span><span class="p">:</span>
            <span class="n">taxid</span> <span class="o">=</span> <span class="n">db_xref</span><span class="o">.</span><span class="n">split</span><span class="p">(</span><span class="s">&quot;:&quot;</span><span class="p">)[</span><span class="mi">1</span><span class="p">]</span>
    <span class="n">r_seqinfo</span> <span class="o">=</span> <span class="s">&#39;&quot;</span><span class="si">%s</span><span class="s">&quot;,&quot;</span><span class="si">%s</span><span class="s">&quot;,&quot;</span><span class="si">%s</span><span class="s">&quot;,&quot;</span><span class="si">%s</span><span class="s">&quot;,&quot;0&quot;&#39;</span> <span class="o">%</span><span class="p">(</span><span class="n">r</span><span class="o">.</span><span class="n">id</span><span class="p">,</span><span class="n">r</span><span class="o">.</span><span class="n">id</span><span class="p">,</span> <span class="n">taxid</span><span class="p">,</span> <span class="n">sp</span><span class="p">)</span>
    <span class="n">seq_info</span><span class="o">.</span><span class="n">extend</span><span class="p">([</span><span class="n">r_seqinfo</span><span class="p">])</span>
    <span class="n">taxids</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">taxid</span><span class="p">)</span>
    
<span class="n">seq_info_out</span> <span class="o">=</span> <span class="nb">open</span><span class="p">(</span><span class="n">seqinfo_file</span><span class="p">,</span> <span class="s">&#39;w&#39;</span><span class="p">)</span>
<span class="k">for</span> <span class="n">l</span> <span class="ow">in</span> <span class="n">seq_info</span><span class="p">:</span>
    <span class="n">seq_info_out</span><span class="o">.</span><span class="n">write</span><span class="p">(</span><span class="n">l</span><span class="o">+</span><span class="s">&quot;</span><span class="se">\n</span><span class="s">&quot;</span><span class="p">)</span>
<span class="n">seq_info_out</span><span class="o">.</span><span class="n">close</span><span class="p">()</span>

<span class="n">taxids_out</span> <span class="o">=</span> <span class="nb">open</span><span class="p">(</span><span class="n">taxid_file</span><span class="p">,</span> <span class="s">&#39;w&#39;</span><span class="p">)</span>
<span class="k">for</span> <span class="n">t</span> <span class="ow">in</span> <span class="nb">list</span><span class="p">(</span><span class="nb">set</span><span class="p">(</span><span class="n">taxids</span><span class="p">)):</span>
    <span class="n">taxids_out</span><span class="o">.</span><span class="n">write</span><span class="p">(</span><span class="n">t</span><span class="o">+</span><span class="s">&quot;</span><span class="se">\n</span><span class="s">&quot;</span><span class="p">)</span>
<span class="n">taxids_out</span><span class="o">.</span><span class="n">close</span><span class="p">()</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Build a HMM for the reference alignment, using the program hmmbuild</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[126]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>hmmbuild -h
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre># hmmbuild :: profile HMM construction from multiple sequence alignments
# HMMER 3.1b1 (May 2013); http://hmmer.org/
# Copyright (C) 2013 Howard Hughes Medical Institute.
# Freely distributed under the GNU General Public License (GPLv3).
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Usage: hmmbuild [-options] &lt;hmmfile_out&gt; &lt;msafile&gt;

Basic options:
  -h     : show brief help on version and usage
  -n &lt;s&gt; : name the HMM &lt;s&gt;
  -o &lt;f&gt; : direct summary output to file &lt;f&gt;, not stdout
  -O &lt;f&gt; : resave annotated, possibly modified MSA to file &lt;f&gt;

Options for selecting alphabet rather than guessing it:
  --amino : input alignment is protein sequence data
  --dna   : input alignment is DNA sequence data
  --rna   : input alignment is RNA sequence data

Alternative model construction strategies:
  --fast           : assign cols w/ &gt;= symfrac residues as consensus  [default]
  --hand           : manual construction (requires reference annotation)
  --symfrac &lt;x&gt;    : sets sym fraction controlling --fast construction  [0.5]
  --fragthresh &lt;x&gt; : if L &lt;= x*alen, tag sequence as a fragment  [0.5]

Alternative relative sequence weighting strategies:
  --wpb     : Henikoff position-based weights  [default]
  --wgsc    : Gerstein/Sonnhammer/Chothia tree weights
  --wblosum : Henikoff simple filter weights
  --wnone   : don&apos;t do any relative weighting; set all to 1
  --wgiven  : use weights as given in MSA file
  --wid &lt;x&gt; : for --wblosum: set identity cutoff  [0.62]  (0&lt;=x&lt;=1)

Alternative effective sequence weighting strategies:
  --eent       : adjust eff seq # to achieve relative entropy target  [default]
  --eclust     : eff seq # is # of single linkage clusters
  --enone      : no effective seq # weighting: just use nseq
  --eset &lt;x&gt;   : set eff seq # for all models to &lt;x&gt;
  --ere &lt;x&gt;    : for --eent: set minimum rel entropy/position to &lt;x&gt;
  --esigma &lt;x&gt; : for --eent: set sigma param to &lt;x&gt;  [45.0]
  --eid &lt;x&gt;    : for --eclust: set fractional identity cutoff to &lt;x&gt;  [0.62]

Alternative prior strategies:
  --pnone    : don&apos;t use any prior; parameters are frequencies
  --plaplace : use a Laplace +1 prior

Handling single sequence inputs:
  --singlemx    : use substitution score matrix for single-sequence inputs
  --popen &lt;x&gt;   : gap open probability (with --singlemx)
  --pextend &lt;x&gt; : gap extend probability (with --singlemx)
  --mx &lt;s&gt;      : substitution score matrix (built-in matrices, with --singlemx)
  --mxfile &lt;f&gt;  : read substitution score matrix from file &lt;f&gt; (with --singlemx)

Control of E-value calibration:
  --EmL &lt;n&gt; : length of sequences for MSV Gumbel mu fit  [200]  (n&gt;0)
  --EmN &lt;n&gt; : number of sequences for MSV Gumbel mu fit  [200]  (n&gt;0)
  --EvL &lt;n&gt; : length of sequences for Viterbi Gumbel mu fit  [200]  (n&gt;0)
  --EvN &lt;n&gt; : number of sequences for Viterbi Gumbel mu fit  [200]  (n&gt;0)
  --EfL &lt;n&gt; : length of sequences for Forward exp tail tau fit  [100]  (n&gt;0)
  --EfN &lt;n&gt; : number of sequences for Forward exp tail tau fit  [200]  (n&gt;0)
  --Eft &lt;x&gt; : tail mass for Forward exponential tail tau fit  [0.04]  (0&lt;x&lt;1)

Other options:
  --cpu &lt;n&gt;          : number of parallel CPU workers for multithreads
  --stall            : arrest after start: for attaching debugger to process
  --informat &lt;s&gt;     : assert input alifile is in format &lt;s&gt; (no autodetect)
  --seed &lt;n&gt;         : set RNG seed to &lt;n&gt; (if 0: one-time arbitrary seed)  [42]
  --w_beta &lt;x&gt;       : tail mass at which window length is determined
  --w_length &lt;n&gt;     : window length 
  --maxinsertlen &lt;n&gt; : pretend all inserts are length &lt;= &lt;n&gt;
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[128]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>hmmbuild rbcL_ref.hmm rbcL_eyorks_clean_alignment.fasta
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre># hmmbuild :: profile HMM construction from multiple sequence alignments
# HMMER 3.1b1 (May 2013); http://hmmer.org/
# Copyright (C) 2013 Howard Hughes Medical Institute.
# Freely distributed under the GNU General Public License (GPLv3).
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# input alignment file:             rbcL_eyorks_clean_alignment.fasta
# output HMM file:                  rbcL_ref.hmm
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

# idx name                  nseq  alen  mlen     W eff_nseq re/pos description
#---- -------------------- ----- ----- ----- ----- -------- ------ -----------
1     rbcL_eyorks_clean_alignment  2309   174   174   293     1.87  0.450 

# CPU time: 0.12u 0.00s 00:00:00.12 Elapsed: 00:00:00.14
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Create a reference package for pplacer:</p>
<p>First we will need to compile some information about the taxonomy of the reference sequences.</p>
<p>We start by producing a taxonomy table for the set of taxa that is used as reference. The file taxids.txt is a simple text file that contains the taxonomic ids for all taxa.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[129]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>head taxids.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>271541
500232
204222
61450
59351
202448
57918
241213
313220
267568
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>We will use a tool from the taxtastic package to fetch the taxonomic information for these taxa from NCBI taxonomy (in the 'taxonomy dump').</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[130]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>taxit taxtable -d /usr/bin/taxonomy.db -t taxids.txt -o taxa.csv
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>The resulting taxa.csv file contains just the taxonomic information relevant for the reference sequences to be used for the phylogenetic placement.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[131]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>head taxa.csv
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>









</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>We also need to provide information linking the taxids to the sequence ids. This file is called the 'seqinfo' file by taxtastic. We provide this as seq_info.csv.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[132]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>head seq_info.csv
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>&quot;seqname&quot;,&quot;accession&quot;,&quot;tax_id&quot;,&quot;species_name&quot;,&quot;is.type&quot;
&quot;JN893788.1&quot;,&quot;JN893788.1&quot;,&quot;12990&quot;,&quot;Carpinus betulus&quot;,&quot;0&quot;
&quot;KJ841577.1&quot;,&quot;KJ841577.1&quot;,&quot;45834&quot;,&quot;Solanum dulcamara&quot;,&quot;0&quot;
&quot;HM559497.1&quot;,&quot;HM559497.1&quot;,&quot;13821&quot;,&quot;Pteris vittata&quot;,&quot;0&quot;
&quot;AY034021.1&quot;,&quot;AY034021.1&quot;,&quot;74693&quot;,&quot;Veronica anagallis-aquatica&quot;,&quot;0&quot;
&quot;KF724370.1&quot;,&quot;KF724370.1&quot;,&quot;28472&quot;,&quot;Epipactis helleborine&quot;,&quot;0&quot;
&quot;HE963482.1&quot;,&quot;HE963482.1&quot;,&quot;76027&quot;,&quot;Fallopia baldschuanica&quot;,&quot;0&quot;
&quot;KT695421.1&quot;,&quot;KT695421.1&quot;,&quot;65561&quot;,&quot;Hypericum perforatum&quot;,&quot;0&quot;
&quot;JN005477.1&quot;,&quot;JN005477.1&quot;,&quot;164261&quot;,&quot;Goodyera repens&quot;,&quot;0&quot;
&quot;JN893409.1&quot;,&quot;JN893409.1&quot;,&quot;35899&quot;,&quot;Galium odoratum&quot;,&quot;0&quot;
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>The reference package also needs to contain a reference tree, the log from the tree inference, the underlying alignment in fasta format as well as the HMM profile that has just been produced to align query sequences to. The tree has been built in the Sativa_final notebook, and we need to generate a raxml info file containing the model parameters:</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[133]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">AgainstRefDB</span><span class="o">/</span><span class="n">Curat1</span><span class="o">/</span><span class="n">FINAL</span><span class="o">/</span><span class="n">Sativa</span><span class="o">/</span><span class="n">RAxML_bestTree</span><span class="o">.</span><span class="n">rbcL</span><span class="nd">@mafftLinsi</span><span class="o">-</span><span class="n">SATIVA</span><span class="nd">@gappyout0</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[134]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

raxmlHPC-PTHREADS -f e -T 5 <span class="se">\</span>
-t RAxML_bestTree.rbcL@mafftLinsi-SATIVA@gappyout0 <span class="se">\</span>
-m GTRGAMMA <span class="se">\</span>
-s rbcL_eyorks_clean_alignment.phylip <span class="se">\</span>
-n <span class="nb">test</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>

IMPORTANT WARNING: Sequences AY034021.1 and JN890743.1 are exactly identical


IMPORTANT WARNING: Sequences KF724370.1 and KF419090.1 are exactly identical


IMPORTANT WARNING: Sequences KF724370.1 and FJ454882.1 are exactly identical


IMPORTANT WARNING: Sequences KF724370.1 and JX094817.1 are exactly identical


IMPORTANT WARNING: Sequences HE963482.1 and FM883609.1 are exactly identical


IMPORTANT WARNING: Sequences HE963482.1 and KJ841323.1 are exactly identical


IMPORTANT WARNING: Sequences JN005477.1 and FJ571330.1 are exactly identical


IMPORTANT WARNING: Sequences JN005477.1 and KF296608.1 are exactly identical


IMPORTANT WARNING: Sequences JN005477.1 and KF296609.1 are exactly identical


IMPORTANT WARNING: Sequences JN005477.1 and JN005476.1 are exactly identical


IMPORTANT WARNING: Sequences JN005477.1 and JN005475.1 are exactly identical


IMPORTANT WARNING: Sequences JN893409.1 and JN893085.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841573.1 and KT878383.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841573.1 and KP030753.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841573.1 and JX572919.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841573.1 and KU739567.1 are exactly identical


IMPORTANT WARNING: Sequences KT740813.1 and KM360673.1 are exactly identical


IMPORTANT WARNING: Sequences KT740813.1 and KF997359.1 are exactly identical


IMPORTANT WARNING: Sequences KT740813.1 and KJ841381.1 are exactly identical


IMPORTANT WARNING: Sequences KT740813.1 and JN891191.1 are exactly identical


IMPORTANT WARNING: Sequences KJ204369.1 and JN890985.1 are exactly identical


IMPORTANT WARNING: Sequences HQ644069.1 and X81106.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841341.1 and X81101.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841341.1 and JN891586.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841341.1 and KP098266.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841341.1 and HM850020.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841341.1 and JN893414.1 are exactly identical


IMPORTANT WARNING: Sequences DQ673303.1 and KX162971.1 are exactly identical


IMPORTANT WARNING: Sequences DQ673303.1 and HQ590293.1 are exactly identical


IMPORTANT WARNING: Sequences DQ673303.1 and KM361002.1 are exactly identical


IMPORTANT WARNING: Sequences HF572814.1 and DQ231562.1 are exactly identical


IMPORTANT WARNING: Sequences KP963378.1 and KJ204350.1 are exactly identical


IMPORTANT WARNING: Sequences KP963378.1 and KJ841346.1 are exactly identical


IMPORTANT WARNING: Sequences KF997298.1 and KM361018.1 are exactly identical


IMPORTANT WARNING: Sequences KF997298.1 and GQ436477.1 are exactly identical


IMPORTANT WARNING: Sequences KF997298.1 and KT897491.1 are exactly identical


IMPORTANT WARNING: Sequences KF163549.1 and HQ600366.1 are exactly identical


IMPORTANT WARNING: Sequences KF163549.1 and HQ600389.1 are exactly identical


IMPORTANT WARNING: Sequences KJ863243.1 and KJ863234.1 are exactly identical


IMPORTANT WARNING: Sequences KJ863243.1 and FM883615.1 are exactly identical


IMPORTANT WARNING: Sequences KJ863243.1 and KJ863271.1 are exactly identical


IMPORTANT WARNING: Sequences KJ863243.1 and AF297125.1 are exactly identical


IMPORTANT WARNING: Sequences KJ863243.1 and KJ863216.1 are exactly identical


IMPORTANT WARNING: Sequences KJ863243.1 and AF297131.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and JN892846.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and FN689363.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and AB125025.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and AY548965.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and FN675722.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and AB125024.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and KP088816.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and M94936.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and KM527353.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and HM849869.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and AB125020.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and NC_020152.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and LM653081.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and AB125026.1 are exactly identical


IMPORTANT WARNING: Sequences KX163018.1 and KF683156.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and HQ600438.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and KJ841479.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and JN891007.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and JN965734.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and HM849758.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and HQ600471.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and KX513087.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and EF125147.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and JN965218.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and KF712932.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and KJ841092.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and KX513078.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and AY691635.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and KF713050.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and JN891035.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and HQ600423.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and JN892394.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and EF125152.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and HQ600469.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and KJ599150.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and JN965503.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and KJ841324.1 are exactly identical


IMPORTANT WARNING: Sequences KX513086.1 and EF125141.1 are exactly identical


IMPORTANT WARNING: Sequences AM849347.1 and KF724330.1 are exactly identical


IMPORTANT WARNING: Sequences AM849347.1 and HE963356.1 are exactly identical


IMPORTANT WARNING: Sequences AM849347.1 and AJ746258.1 are exactly identical


IMPORTANT WARNING: Sequences KJ593671.1 and KM360961.1 are exactly identical


IMPORTANT WARNING: Sequences KJ593671.1 and EU676990.1 are exactly identical


IMPORTANT WARNING: Sequences KJ593671.1 and HM850315.1 are exactly identical


IMPORTANT WARNING: Sequences KJ593671.1 and FR865141.1 are exactly identical


IMPORTANT WARNING: Sequences KJ593671.1 and U06825.1 are exactly identical


IMPORTANT WARNING: Sequences KJ593671.1 and FN689382.1 are exactly identical


IMPORTANT WARNING: Sequences KJ593671.1 and JN890623.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and AY395535.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and JN890930.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and KP711232.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and KX421110.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and HE963335.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and KM360939.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and HQ600473.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and KJ204293.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and KJ599166.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and HM849748.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and HQ600437.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and HM849945.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and EF125154.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and JN890970.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and JN965216.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and HE963444.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and JN893779.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and JX848486.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and JN893042.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and KF712975.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and HM849870.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and EF115543.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and KM974736.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and JX848494.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and NC_027473.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and JN892362.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and HQ600462.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and HM850275.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and FR865119.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and JN890889.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and EF125151.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and KF713031.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and KJ599171.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and AJ784823.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and KF713057.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and LT576792.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and EF125150.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and KM360652.1 are exactly identical


IMPORTANT WARNING: Sequences KP711233.1 and KJ593195.1 are exactly identical


IMPORTANT WARNING: Sequences JF941958.1 and KP120319.1 are exactly identical


IMPORTANT WARNING: Sequences JF941958.1 and DQ006098.1 are exactly identical


IMPORTANT WARNING: Sequences JF941958.1 and AF323202.1 are exactly identical


IMPORTANT WARNING: Sequences JF941958.1 and KP643835.1 are exactly identical


IMPORTANT WARNING: Sequences AB196943.1 and AB196845.1 are exactly identical


IMPORTANT WARNING: Sequences AB196943.1 and AB196946.1 are exactly identical


IMPORTANT WARNING: Sequences AB196943.1 and HM850283.1 are exactly identical


IMPORTANT WARNING: Sequences AB196943.1 and JF943537.1 are exactly identical


IMPORTANT WARNING: Sequences AB196943.1 and HM850282.1 are exactly identical


IMPORTANT WARNING: Sequences AB196943.1 and AB196846.1 are exactly identical


IMPORTANT WARNING: Sequences AB196943.1 and AB196950.1 are exactly identical


IMPORTANT WARNING: Sequences AB196943.1 and AB196847.1 are exactly identical


IMPORTANT WARNING: Sequences L08759.1 and KC584878.1 are exactly identical


IMPORTANT WARNING: Sequences L08759.1 and KX527487.1 are exactly identical


IMPORTANT WARNING: Sequences L08759.1 and JN893068.1 are exactly identical


IMPORTANT WARNING: Sequences L08759.1 and HM849753.1 are exactly identical


IMPORTANT WARNING: Sequences KJ204359.1 and JN891378.1 are exactly identical


IMPORTANT WARNING: Sequences KJ204359.1 and HQ590146.1 are exactly identical


IMPORTANT WARNING: Sequences JN890693.1 and JN890925.1 are exactly identical


IMPORTANT WARNING: Sequences JN890693.1 and KJ841647.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841580.1 and KM360730.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841580.1 and JQ933521.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841580.1 and KF997334.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841580.1 and EU677022.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841580.1 and HE574593.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841580.1 and KM218337.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841580.1 and HM849980.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841580.1 and HQ590045.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841580.1 and KJ841607.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841580.1 and JX848424.1 are exactly identical


IMPORTANT WARNING: Sequences KF997498.1 and JX913454.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and HQ590151.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and AY395534.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and KJ841116.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and JQ933362.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and JN890909.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and HM849788.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and KM360888.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and KF997466.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and HM850342.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and KM360904.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and KM360842.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and JN891399.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and AB530978.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and AB530982.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and KC589831.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and GQ120446.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and KM360702.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and HM850099.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and KC589792.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and HM849897.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and HE574613.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and JQ230997.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and EU676928.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and HM850098.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and JN890832.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and KT178106.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and JQ933247.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and KM360645.1 are exactly identical


IMPORTANT WARNING: Sequences KF196019.1 and JX848418.1 are exactly identical


IMPORTANT WARNING: Sequences JN893387.1 and HM849756.1 are exactly identical


IMPORTANT WARNING: Sequences JN893387.1 and HE963312.1 are exactly identical


IMPORTANT WARNING: Sequences HE858481.1 and KF296624.1 are exactly identical


IMPORTANT WARNING: Sequences HE858481.1 and JQ933294.1 are exactly identical


IMPORTANT WARNING: Sequences HE858481.1 and KJ451495.1 are exactly identical


IMPORTANT WARNING: Sequences HE858481.1 and KF296614.1 are exactly identical


IMPORTANT WARNING: Sequences HE858481.1 and HE858476.1 are exactly identical


IMPORTANT WARNING: Sequences HE858481.1 and JN891189.1 are exactly identical


IMPORTANT WARNING: Sequences HE858481.1 and KF997335.1 are exactly identical


IMPORTANT WARNING: Sequences HE858481.1 and KJ451493.1 are exactly identical


IMPORTANT WARNING: Sequences KX016341.1 and JN890628.1 are exactly identical


IMPORTANT WARNING: Sequences KX016341.1 and KX016369.1 are exactly identical


IMPORTANT WARNING: Sequences KX016341.1 and FN689365.1 are exactly identical


IMPORTANT WARNING: Sequences KX016341.1 and AJ849577.1 are exactly identical


IMPORTANT WARNING: Sequences KX016341.1 and KJ204400.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841413.1 and KF997369.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841413.1 and KJ773679.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841413.1 and JN376005.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841413.1 and JX848463.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841413.1 and HM850168.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841413.1 and KP126850.1 are exactly identical


IMPORTANT WARNING: Sequences KF196023.1 and EU385018.1 are exactly identical


IMPORTANT WARNING: Sequences KF196023.1 and KJ841588.1 are exactly identical


IMPORTANT WARNING: Sequences KF196023.1 and KJ593690.1 are exactly identical


IMPORTANT WARNING: Sequences KF196023.1 and JX848427.1 are exactly identical


IMPORTANT WARNING: Sequences AF248022.1 and JN893579.1 are exactly identical


IMPORTANT WARNING: Sequences AF248022.1 and JN893580.1 are exactly identical


IMPORTANT WARNING: Sequences HF572817.1 and KP125337.1 are exactly identical


IMPORTANT WARNING: Sequences EU269705.1 and AM177358.1 are exactly identical


IMPORTANT WARNING: Sequences KT724052.1 and L36453.1 are exactly identical


IMPORTANT WARNING: Sequences KP899647.1 and AF240645.1 are exactly identical


IMPORTANT WARNING: Sequences KP899647.1 and HQ676496.1 are exactly identical


IMPORTANT WARNING: Sequences KP899647.1 and U30607.1 are exactly identical


IMPORTANT WARNING: Sequences L13929.1 and KM360812.1 are exactly identical


IMPORTANT WARNING: Sequences L13929.1 and AB530968.1 are exactly identical


IMPORTANT WARNING: Sequences L13929.1 and AY215124.1 are exactly identical


IMPORTANT WARNING: Sequences L13929.1 and DQ383815.1 are exactly identical


IMPORTANT WARNING: Sequences L13929.1 and KF746357.1 are exactly identical


IMPORTANT WARNING: Sequences L13929.1 and KC473895.1 are exactly identical


IMPORTANT WARNING: Sequences L13929.1 and KF746363.1 are exactly identical


IMPORTANT WARNING: Sequences KC699617.1 and JN892057.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600432.1 and LN626641.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600432.1 and AY137453.1 are exactly identical


IMPORTANT WARNING: Sequences NC_009270.1 and KM218338.1 are exactly identical


IMPORTANT WARNING: Sequences NC_009270.1 and JN847826.1 are exactly identical


IMPORTANT WARNING: Sequences JN890922.1 and HM850413.1 are exactly identical


IMPORTANT WARNING: Sequences JN890922.1 and JN892960.1 are exactly identical


IMPORTANT WARNING: Sequences JN890922.1 and KC894706.1 are exactly identical


IMPORTANT WARNING: Sequences JN890922.1 and KJ841632.1 are exactly identical


IMPORTANT WARNING: Sequences JN890922.1 and HM850420.1 are exactly identical


IMPORTANT WARNING: Sequences JN890922.1 and KM361020.1 are exactly identical


IMPORTANT WARNING: Sequences JN893657.1 and HM850171.1 are exactly identical


IMPORTANT WARNING: Sequences JN893657.1 and HQ590041.1 are exactly identical


IMPORTANT WARNING: Sequences JN893657.1 and HE963342.1 are exactly identical


IMPORTANT WARNING: Sequences JN893657.1 and JN892956.1 are exactly identical


IMPORTANT WARNING: Sequences JN893657.1 and KU499887.1 are exactly identical


IMPORTANT WARNING: Sequences JN893657.1 and GU344680.1 are exactly identical


IMPORTANT WARNING: Sequences JN893657.1 and KC571787.1 are exactly identical


IMPORTANT WARNING: Sequences JN893657.1 and KJ204411.1 are exactly identical


IMPORTANT WARNING: Sequences JN893657.1 and KM360725.1 are exactly identical


IMPORTANT WARNING: Sequences JN893657.1 and KX527273.1 are exactly identical


IMPORTANT WARNING: Sequences JN893657.1 and KJ746270.1 are exactly identical


IMPORTANT WARNING: Sequences HF572813.1 and AM087200.3 are exactly identical


IMPORTANT WARNING: Sequences HF572813.1 and KC436099.1 are exactly identical


IMPORTANT WARNING: Sequences HF572813.1 and KC436090.1 are exactly identical


IMPORTANT WARNING: Sequences HF572813.1 and KC436088.1 are exactly identical


IMPORTANT WARNING: Sequences HF572813.1 and KC436089.1 are exactly identical


IMPORTANT WARNING: Sequences HF572813.1 and KC436087.1 are exactly identical


IMPORTANT WARNING: Sequences HF572813.1 and KC436092.1 are exactly identical


IMPORTANT WARNING: Sequences KC415372.1 and FJ788587.1 are exactly identical


IMPORTANT WARNING: Sequences FM883642.1 and KJ204390.1 are exactly identical


IMPORTANT WARNING: Sequences FM883642.1 and EU676943.1 are exactly identical


IMPORTANT WARNING: Sequences FM883642.1 and EU676944.1 are exactly identical


IMPORTANT WARNING: Sequences FM883642.1 and AF297127.1 are exactly identical


IMPORTANT WARNING: Sequences FM883642.1 and KJ841485.1 are exactly identical


IMPORTANT WARNING: Sequences FM883642.1 and KJ841487.1 are exactly identical


IMPORTANT WARNING: Sequences AB639168.1 and KF992506.1 are exactly identical


IMPORTANT WARNING: Sequences AB639168.1 and EF588706.1 are exactly identical


IMPORTANT WARNING: Sequences AB639168.1 and AB639165.1 are exactly identical


IMPORTANT WARNING: Sequences AB639168.1 and AB639167.1 are exactly identical


IMPORTANT WARNING: Sequences JQ933501.1 and JN892477.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and KF997277.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and FN689370.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and KX162920.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and HG765093.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and HM850304.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and JQ391382.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and JN891605.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and HG765052.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and KJ506867.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and GU363807.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and JQ391311.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and JQ391393.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and JQ391279.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and KM360951.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and KM360737.1 are exactly identical


IMPORTANT WARNING: Sequences KM360949.1 and KX163045.1 are exactly identical


IMPORTANT WARNING: Sequences AB906846.1 and JN225350.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and KJ204292.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and JN892406.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and AM234813.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and L01885.2 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and JN892970.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and HM850469.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and KP974255.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and KM360919.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and KP974248.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and KM360814.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and KM360787.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and JN893705.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and JN891604.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and KF997429.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and JN890659.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and JN892715.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and KM360984.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and KM360982.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and HM850050.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and JN890586.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and HM849916.1 are exactly identical


IMPORTANT WARNING: Sequences KF613096.1 and HM850008.1 are exactly identical


IMPORTANT WARNING: Sequences L11680.1 and HM849818.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600436.1 and HQ600453.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600436.1 and GU140017.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600436.1 and KF713002.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600436.1 and KF713005.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600436.1 and FN870385.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600436.1 and AB042240.3 are exactly identical


IMPORTANT WARNING: Sequences HQ600436.1 and JN891931.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600436.1 and HE574609.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600436.1 and LT576864.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600436.1 and JN893062.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600436.1 and KR092101.1 are exactly identical


IMPORTANT WARNING: Sequences HM849757.1 and FN689372.1 are exactly identical


IMPORTANT WARNING: Sequences KC482191.1 and JN892765.1 are exactly identical


IMPORTANT WARNING: Sequences KC482191.1 and JN891766.1 are exactly identical


IMPORTANT WARNING: Sequences KC482191.1 and KJ512640.1 are exactly identical


IMPORTANT WARNING: Sequences KC482191.1 and HE574583.1 are exactly identical


IMPORTANT WARNING: Sequences KC482191.1 and JN890714.1 are exactly identical


IMPORTANT WARNING: Sequences KC482191.1 and JX848444.1 are exactly identical


IMPORTANT WARNING: Sequences KC482191.1 and JN571978.1 are exactly identical


IMPORTANT WARNING: Sequences KC482191.1 and JN571977.1 are exactly identical


IMPORTANT WARNING: Sequences KC482191.1 and JN571979.1 are exactly identical


IMPORTANT WARNING: Sequences KC482191.1 and EU713443.1 are exactly identical


IMPORTANT WARNING: Sequences KM360880.1 and FJ395570.1 are exactly identical


IMPORTANT WARNING: Sequences KM360880.1 and AY794944.1 are exactly identical


IMPORTANT WARNING: Sequences KM360880.1 and JN890833.1 are exactly identical


IMPORTANT WARNING: Sequences U80727.1 and KT695594.1 are exactly identical


IMPORTANT WARNING: Sequences U80727.1 and AB196953.1 are exactly identical


IMPORTANT WARNING: Sequences JN587331.1 and KC481638.1 are exactly identical


IMPORTANT WARNING: Sequences JN587331.1 and KJ593578.1 are exactly identical


IMPORTANT WARNING: Sequences JN587331.1 and DQ006110.1 are exactly identical


IMPORTANT WARNING: Sequences JN587331.1 and AB233943.1 are exactly identical


IMPORTANT WARNING: Sequences JN587331.1 and KC481640.1 are exactly identical


IMPORTANT WARNING: Sequences JN587331.1 and KJ841439.1 are exactly identical


IMPORTANT WARNING: Sequences KF055131.1 and HE963530.1 are exactly identical


IMPORTANT WARNING: Sequences KF055131.1 and KP402762.1 are exactly identical


IMPORTANT WARNING: Sequences KF055131.1 and KF055128.1 are exactly identical


IMPORTANT WARNING: Sequences KF055131.1 and AB266224.1 are exactly identical


IMPORTANT WARNING: Sequences KF055131.1 and JN891872.1 are exactly identical


IMPORTANT WARNING: Sequences KF055131.1 and HM850103.1 are exactly identical


IMPORTANT WARNING: Sequences HM849847.1 and HM849848.1 are exactly identical


IMPORTANT WARNING: Sequences HM849847.1 and JN891764.1 are exactly identical


IMPORTANT WARNING: Sequences HM849847.1 and AP009376.1 are exactly identical


IMPORTANT WARNING: Sequences HM849847.1 and KJ773335.1 are exactly identical


IMPORTANT WARNING: Sequences HM849847.1 and HQ619739.1 are exactly identical


IMPORTANT WARNING: Sequences HM849847.1 and HM850197.1 are exactly identical


IMPORTANT WARNING: Sequences HM849847.1 and JN965330.1 are exactly identical


IMPORTANT WARNING: Sequences KJ916496.1 and JN892291.1 are exactly identical


IMPORTANT WARNING: Sequences KJ916496.1 and JQ412370.1 are exactly identical


IMPORTANT WARNING: Sequences KJ916496.1 and HM850028.1 are exactly identical


IMPORTANT WARNING: Sequences KJ916496.1 and KM360803.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600426.1 and KM974751.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600426.1 and JX871941.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600426.1 and KX421145.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600426.1 and AM777385.2 are exactly identical


IMPORTANT WARNING: Sequences HQ600426.1 and KF713028.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600426.1 and KJ599212.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600426.1 and NC_019651.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600426.1 and HQ590164.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600426.1 and HM850001.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600426.1 and AY395547.1 are exactly identical


IMPORTANT WARNING: Sequences AM999826.1 and EU832952.1 are exactly identical


IMPORTANT WARNING: Sequences HM850308.1 and KM360959.1 are exactly identical


IMPORTANT WARNING: Sequences HM850308.1 and KJ841520.1 are exactly identical


IMPORTANT WARNING: Sequences FR667683.1 and KU343177.1 are exactly identical


IMPORTANT WARNING: Sequences FR667683.1 and KC164307.1 are exactly identical


IMPORTANT WARNING: Sequences FR667683.1 and AM849333.1 are exactly identical


IMPORTANT WARNING: Sequences KU724141.1 and AF502006.1 are exactly identical


IMPORTANT WARNING: Sequences HM850316.1 and JN893123.1 are exactly identical


IMPORTANT WARNING: Sequences HM850316.1 and KT960570.1 are exactly identical


IMPORTANT WARNING: Sequences JN893846.1 and HQ590141.1 are exactly identical


IMPORTANT WARNING: Sequences JN892762.1 and KJ841639.1 are exactly identical


IMPORTANT WARNING: Sequences JN892762.1 and JQ933515.1 are exactly identical


IMPORTANT WARNING: Sequences JN892762.1 and HQ590313.1 are exactly identical


IMPORTANT WARNING: Sequences JN892389.1 and Y12951.1 are exactly identical


IMPORTANT WARNING: Sequences JN892389.1 and JN892798.1 are exactly identical


IMPORTANT WARNING: Sequences JN892389.1 and JX644681.1 are exactly identical


IMPORTANT WARNING: Sequences JN892389.1 and AM999830.1 are exactly identical


IMPORTANT WARNING: Sequences AB626644.1 and AB574681.1 are exactly identical


IMPORTANT WARNING: Sequences AB626644.1 and HQ676505.1 are exactly identical


IMPORTANT WARNING: Sequences AB626644.1 and JQ349076.1 are exactly identical


IMPORTANT WARNING: Sequences KC164308.1 and KF163504.1 are exactly identical


IMPORTANT WARNING: Sequences KC164308.1 and KF997405.1 are exactly identical


IMPORTANT WARNING: Sequences KC164308.1 and KC164306.1 are exactly identical


IMPORTANT WARNING: Sequences KC164308.1 and KF163505.1 are exactly identical


IMPORTANT WARNING: Sequences KC164308.1 and KC164309.1 are exactly identical


IMPORTANT WARNING: Sequences KC164308.1 and HQ600375.1 are exactly identical


IMPORTANT WARNING: Sequences KC164308.1 and KT365292.1 are exactly identical


IMPORTANT WARNING: Sequences KC164308.1 and KJ000047.1 are exactly identical


IMPORTANT WARNING: Sequences HE963371.1 and JN891388.1 are exactly identical


IMPORTANT WARNING: Sequences KX527402.1 and JN891980.1 are exactly identical


IMPORTANT WARNING: Sequences KX527402.1 and HQ590098.1 are exactly identical


IMPORTANT WARNING: Sequences KX527402.1 and KJ593405.1 are exactly identical


IMPORTANT WARNING: Sequences KX527402.1 and KJ773504.1 are exactly identical


IMPORTANT WARNING: Sequences KC415360.1 and HQ590254.1 are exactly identical


IMPORTANT WARNING: Sequences KC415360.1 and KM002826.1 are exactly identical


IMPORTANT WARNING: Sequences KC415360.1 and JN890626.1 are exactly identical


IMPORTANT WARNING: Sequences KM361016.1 and AM234827.1 are exactly identical


IMPORTANT WARNING: Sequences KM361016.1 and FJ395562.1 are exactly identical


IMPORTANT WARNING: Sequences KM361016.1 and JN892516.1 are exactly identical


IMPORTANT WARNING: Sequences KM361016.1 and JN890678.1 are exactly identical


IMPORTANT WARNING: Sequences GU195652.1 and M91634.1 are exactly identical


IMPORTANT WARNING: Sequences GU195652.1 and KC584886.1 are exactly identical


IMPORTANT WARNING: Sequences KT899951.1 and EU741051.1 are exactly identical


IMPORTANT WARNING: Sequences JN890813.1 and KM360851.1 are exactly identical


IMPORTANT WARNING: Sequences JN890813.1 and KF997319.1 are exactly identical


IMPORTANT WARNING: Sequences JN890813.1 and Z69218.1 are exactly identical


IMPORTANT WARNING: Sequences AM712908.1 and EF614270.1 are exactly identical


IMPORTANT WARNING: Sequences AM712908.1 and D89473.1 are exactly identical


IMPORTANT WARNING: Sequences AM712908.1 and AF197599.1 are exactly identical


IMPORTANT WARNING: Sequences AM712908.1 and KC584879.1 are exactly identical


IMPORTANT WARNING: Sequences HM850306.1 and AY794915.1 are exactly identical


IMPORTANT WARNING: Sequences HM850306.1 and KF022500.1 are exactly identical


IMPORTANT WARNING: Sequences HM850306.1 and KF381098.1 are exactly identical


IMPORTANT WARNING: Sequences HM850306.1 and NC_016736.1 are exactly identical


IMPORTANT WARNING: Sequences HM849804.1 and KT351784.1 are exactly identical


IMPORTANT WARNING: Sequences HM849804.1 and HQ600434.1 are exactly identical


IMPORTANT WARNING: Sequences HM849804.1 and KM974733.1 are exactly identical


IMPORTANT WARNING: Sequences JN890795.1 and KJ841333.1 are exactly identical


IMPORTANT WARNING: Sequences JN892532.1 and KJ841511.1 are exactly identical


IMPORTANT WARNING: Sequences JN892775.1 and Z37468.1 are exactly identical


IMPORTANT WARNING: Sequences JN892775.1 and Z37469.1 are exactly identical


IMPORTANT WARNING: Sequences JN892775.1 and HM850397.1 are exactly identical


IMPORTANT WARNING: Sequences Y12983.1 and KM360974.1 are exactly identical


IMPORTANT WARNING: Sequences AJ746255.1 and HQ600460.1 are exactly identical


IMPORTANT WARNING: Sequences AJ746255.1 and AJ746280.1 are exactly identical


IMPORTANT WARNING: Sequences AJ746255.1 and JN891527.1 are exactly identical


IMPORTANT WARNING: Sequences AJ746255.1 and HM849752.1 are exactly identical


IMPORTANT WARNING: Sequences AJ746255.1 and HM849751.1 are exactly identical


IMPORTANT WARNING: Sequences KF319005.1 and KF319000.1 are exactly identical


IMPORTANT WARNING: Sequences KF319005.1 and HM849888.1 are exactly identical


IMPORTANT WARNING: Sequences KF319005.1 and JX848451.1 are exactly identical


IMPORTANT WARNING: Sequences KF319005.1 and KF319008.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590160.1 and JN891755.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590160.1 and KJ841379.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590160.1 and KJ204366.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590160.1 and KJ746274.1 are exactly identical


IMPORTANT WARNING: Sequences AJ320208.1 and AY101060.1 are exactly identical


IMPORTANT WARNING: Sequences HM850165.1 and KJ773677.1 are exactly identical


IMPORTANT WARNING: Sequences KF997280.1 and KF158095.1 are exactly identical


IMPORTANT WARNING: Sequences KJ863221.1 and EU554015.1 are exactly identical


IMPORTANT WARNING: Sequences KJ863221.1 and KJ863232.1 are exactly identical


IMPORTANT WARNING: Sequences KF418943.1 and KX162896.1 are exactly identical


IMPORTANT WARNING: Sequences AB872567.1 and KX162887.1 are exactly identical


IMPORTANT WARNING: Sequences AB872567.1 and HG765032.1 are exactly identical


IMPORTANT WARNING: Sequences JN965519.1 and JN892931.1 are exactly identical


IMPORTANT WARNING: Sequences HM559486.1 and HM559511.1 are exactly identical


IMPORTANT WARNING: Sequences HM559486.1 and HM559505.1 are exactly identical


IMPORTANT WARNING: Sequences HM559486.1 and HM559510.1 are exactly identical


IMPORTANT WARNING: Sequences HM559486.1 and KR816700.1 are exactly identical


IMPORTANT WARNING: Sequences HM559486.1 and KM008232.1 are exactly identical


IMPORTANT WARNING: Sequences HM559486.1 and HM559504.1 are exactly identical


IMPORTANT WARNING: Sequences KC164340.1 and KJ001642.1 are exactly identical


IMPORTANT WARNING: Sequences KC164340.1 and KF646538.1 are exactly identical


IMPORTANT WARNING: Sequences JX913469.1 and KF997367.1 are exactly identical


IMPORTANT WARNING: Sequences JX913469.1 and KP963389.1 are exactly identical


IMPORTANT WARNING: Sequences JX913469.1 and JN893651.1 are exactly identical


IMPORTANT WARNING: Sequences JX913469.1 and KP963377.1 are exactly identical


IMPORTANT WARNING: Sequences JX913469.1 and HM850026.1 are exactly identical


IMPORTANT WARNING: Sequences JX913469.1 and KP963385.1 are exactly identical


IMPORTANT WARNING: Sequences JX913469.1 and KF997286.1 are exactly identical


IMPORTANT WARNING: Sequences KR184827.1 and JN040400.1 are exactly identical


IMPORTANT WARNING: Sequences KR184827.1 and HF565316.1 are exactly identical


IMPORTANT WARNING: Sequences FN689371.1 and FJ844569.1 are exactly identical


IMPORTANT WARNING: Sequences FN689371.1 and KX162893.1 are exactly identical


IMPORTANT WARNING: Sequences FN689371.1 and X56618.1 are exactly identical


IMPORTANT WARNING: Sequences FN689371.1 and KT695533.1 are exactly identical


IMPORTANT WARNING: Sequences FN689371.1 and FJ844583.1 are exactly identical


IMPORTANT WARNING: Sequences FN689371.1 and KF418927.1 are exactly identical


IMPORTANT WARNING: Sequences HM849770.1 and M88343.1 are exactly identical


IMPORTANT WARNING: Sequences AJ419768.1 and JN892854.1 are exactly identical


IMPORTANT WARNING: Sequences HM850078.1 and Y12962.1 are exactly identical


IMPORTANT WARNING: Sequences HM850078.1 and AB983296.1 are exactly identical


IMPORTANT WARNING: Sequences HM850078.1 and JN892383.1 are exactly identical


IMPORTANT WARNING: Sequences HM850078.1 and JX644698.1 are exactly identical


IMPORTANT WARNING: Sequences HM850078.1 and AM999789.1 are exactly identical


IMPORTANT WARNING: Sequences EU840292.1 and EF653762.1 are exactly identical


IMPORTANT WARNING: Sequences EU840292.1 and AB000309.1 are exactly identical


IMPORTANT WARNING: Sequences EU840292.1 and EU254477.1 are exactly identical


IMPORTANT WARNING: Sequences EU713431.1 and FJ587264.1 are exactly identical


IMPORTANT WARNING: Sequences KC482499.1 and KC482497.1 are exactly identical


IMPORTANT WARNING: Sequences KC482499.1 and JN892155.1 are exactly identical


IMPORTANT WARNING: Sequences KC482499.1 and JN892158.1 are exactly identical


IMPORTANT WARNING: Sequences KC482499.1 and KT280091.1 are exactly identical


IMPORTANT WARNING: Sequences GQ248547.1 and HE963316.1 are exactly identical


IMPORTANT WARNING: Sequences GQ248547.1 and KJ841093.1 are exactly identical


IMPORTANT WARNING: Sequences KP189363.1 and EF394238.1 are exactly identical


IMPORTANT WARNING: Sequences HQ235499.1 and GU363791.1 are exactly identical


IMPORTANT WARNING: Sequences HQ235499.1 and HQ235465.1 are exactly identical


IMPORTANT WARNING: Sequences KJ204372.1 and JN893191.1 are exactly identical


IMPORTANT WARNING: Sequences KJ204372.1 and KF997347.1 are exactly identical


IMPORTANT WARNING: Sequences KJ204372.1 and KM360873.1 are exactly identical


IMPORTANT WARNING: Sequences KF154863.1 and L01947.2 are exactly identical


IMPORTANT WARNING: Sequences KF154863.1 and KX163002.1 are exactly identical


IMPORTANT WARNING: Sequences KF154863.1 and AF227901.1 are exactly identical


IMPORTANT WARNING: Sequences KF154863.1 and AF227900.1 are exactly identical


IMPORTANT WARNING: Sequences KF154863.1 and HQ235431.1 are exactly identical


IMPORTANT WARNING: Sequences KF154863.1 and GU363826.1 are exactly identical


IMPORTANT WARNING: Sequences KF154863.1 and AF227904.1 are exactly identical


IMPORTANT WARNING: Sequences JF701589.1 and JN854179.1 are exactly identical


IMPORTANT WARNING: Sequences JF701589.1 and KX162985.1 are exactly identical


IMPORTANT WARNING: Sequences JF701589.1 and KP088742.1 are exactly identical


IMPORTANT WARNING: Sequences JF701589.1 and HQ619767.1 are exactly identical


IMPORTANT WARNING: Sequences JF701589.1 and JN854158.1 are exactly identical


IMPORTANT WARNING: Sequences JF701589.1 and JQ512593.1 are exactly identical


IMPORTANT WARNING: Sequences JF701589.1 and AB097775.1 are exactly identical


IMPORTANT WARNING: Sequences JF701589.1 and FR831925.1 are exactly identical


IMPORTANT WARNING: Sequences HQ902802.1 and Z37399.1 are exactly identical


IMPORTANT WARNING: Sequences HQ902802.1 and JN891017.1 are exactly identical


IMPORTANT WARNING: Sequences HQ902802.1 and HM850101.1 are exactly identical


IMPORTANT WARNING: Sequences KM360731.1 and FJ395589.1 are exactly identical


IMPORTANT WARNING: Sequences KM360731.1 and KX162907.1 are exactly identical


IMPORTANT WARNING: Sequences HM850096.1 and JN892932.1 are exactly identical


IMPORTANT WARNING: Sequences JX913459.1 and HM850025.1 are exactly identical


IMPORTANT WARNING: Sequences JX913459.1 and JN890948.1 are exactly identical


IMPORTANT WARNING: Sequences HM850155.1 and JN893431.1 are exactly identical


IMPORTANT WARNING: Sequences KT280096.1 and KT695507.1 are exactly identical


IMPORTANT WARNING: Sequences KT280096.1 and FJ395563.1 are exactly identical


IMPORTANT WARNING: Sequences KT280096.1 and HM849977.1 are exactly identical


IMPORTANT WARNING: Sequences KT280096.1 and FJ395561.1 are exactly identical


IMPORTANT WARNING: Sequences KT280096.1 and HM849976.1 are exactly identical


IMPORTANT WARNING: Sequences KT280096.1 and KF997326.1 are exactly identical


IMPORTANT WARNING: Sequences KT280096.1 and KF997508.1 are exactly identical


IMPORTANT WARNING: Sequences KT280096.1 and FJ395568.1 are exactly identical


IMPORTANT WARNING: Sequences JQ773661.1 and HM850004.1 are exactly identical


IMPORTANT WARNING: Sequences JQ773661.1 and KP088598.1 are exactly identical


IMPORTANT WARNING: Sequences U80685.1 and JN890979.1 are exactly identical


IMPORTANT WARNING: Sequences U80685.1 and AY149345.1 are exactly identical


IMPORTANT WARNING: Sequences U80685.1 and KM360683.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841416.1 and KM192096.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841416.1 and EF173093.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841416.1 and L14006.2 are exactly identical


IMPORTANT WARNING: Sequences KF997304.1 and KF158086.1 are exactly identical


IMPORTANT WARNING: Sequences KF997304.1 and KM360745.1 are exactly identical


IMPORTANT WARNING: Sequences KF997304.1 and JQ933378.1 are exactly identical


IMPORTANT WARNING: Sequences JN225354.1 and JN225352.1 are exactly identical


IMPORTANT WARNING: Sequences JN225354.1 and U80734.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN891455.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN891457.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and GQ469851.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and HM849860.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN891911.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN892341.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and GQ469843.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN891516.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and HM849856.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN892570.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and KM360694.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and HE963382.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and KJ841178.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN891904.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and KJ204312.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and KJ204311.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN892446.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN892135.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and KC482257.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN892137.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN890574.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN892136.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and AY395533.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN965344.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JX644625.1 are exactly identical


IMPORTANT WARNING: Sequences JN891690.1 and JN891463.1 are exactly identical


IMPORTANT WARNING: Sequences KX551970.1 and U91966.1 are exactly identical


IMPORTANT WARNING: Sequences KX551970.1 and KM360643.1 are exactly identical


IMPORTANT WARNING: Sequences KX551970.1 and AY174632.1 are exactly identical


IMPORTANT WARNING: Sequences JF802268.1 and JN892056.1 are exactly identical


IMPORTANT WARNING: Sequences AB530957.1 and JN891654.1 are exactly identical


IMPORTANT WARNING: Sequences JN891119.1 and KJ667660.1 are exactly identical


IMPORTANT WARNING: Sequences JN891119.1 and JN892100.1 are exactly identical


IMPORTANT WARNING: Sequences JN891119.1 and JN965799.1 are exactly identical


IMPORTANT WARNING: Sequences JN891119.1 and JN965798.1 are exactly identical


IMPORTANT WARNING: Sequences JN891119.1 and JN892284.1 are exactly identical


IMPORTANT WARNING: Sequences JN891119.1 and JN893277.1 are exactly identical


IMPORTANT WARNING: Sequences JN891119.1 and L08766.1 are exactly identical


IMPORTANT WARNING: Sequences JN891119.1 and JN892536.1 are exactly identical


IMPORTANT WARNING: Sequences JN890903.1 and KF855326.1 are exactly identical


IMPORTANT WARNING: Sequences EF647059.1 and JF715055.1 are exactly identical


IMPORTANT WARNING: Sequences EF647059.1 and KJ841571.1 are exactly identical


IMPORTANT WARNING: Sequences EF647059.1 and EF646915.1 are exactly identical


IMPORTANT WARNING: Sequences EF647059.1 and KT962039.1 are exactly identical


IMPORTANT WARNING: Sequences EF647059.1 and EF646920.1 are exactly identical


IMPORTANT WARNING: Sequences KU722855.1 and KU722897.1 are exactly identical


IMPORTANT WARNING: Sequences DQ646001.1 and KC117177.1 are exactly identical


IMPORTANT WARNING: Sequences KT596044.1 and KT178135.1 are exactly identical


IMPORTANT WARNING: Sequences KT596044.1 and HM849909.1 are exactly identical


IMPORTANT WARNING: Sequences HM850319.1 and KJ841532.1 are exactly identical


IMPORTANT WARNING: Sequences HM850319.1 and JN893115.1 are exactly identical


IMPORTANT WARNING: Sequences HM850319.1 and HQ644066.1 are exactly identical


IMPORTANT WARNING: Sequences HM850319.1 and KX095196.1 are exactly identical


IMPORTANT WARNING: Sequences HM850319.1 and AF297126.1 are exactly identical


IMPORTANT WARNING: Sequences HM850319.1 and HQ590252.1 are exactly identical


IMPORTANT WARNING: Sequences KJ593592.1 and AB575038.1 are exactly identical


IMPORTANT WARNING: Sequences KR048286.1 and U50220.1 are exactly identical


IMPORTANT WARNING: Sequences KR048286.1 and KM360614.1 are exactly identical


IMPORTANT WARNING: Sequences KR048286.1 and JN893470.1 are exactly identical


IMPORTANT WARNING: Sequences HQ676509.1 and HQ676510.1 are exactly identical


IMPORTANT WARNING: Sequences HQ676509.1 and AF537254.1 are exactly identical


IMPORTANT WARNING: Sequences AF318596.1 and KP899653.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600435.1 and JN890775.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600435.1 and AY836154.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600435.1 and AJ746286.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600435.1 and HE575814.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600435.1 and AJ746287.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600435.1 and KF712964.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600435.1 and HQ600445.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600435.1 and FR865129.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600435.1 and HQ652722.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600435.1 and HM849828.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600435.1 and AY395531.1 are exactly identical


IMPORTANT WARNING: Sequences JN891775.1 and KF997266.1 are exactly identical


IMPORTANT WARNING: Sequences JN891775.1 and KC529315.1 are exactly identical


IMPORTANT WARNING: Sequences JN891775.1 and HM849841.1 are exactly identical


IMPORTANT WARNING: Sequences JN891775.1 and HQ589980.1 are exactly identical


IMPORTANT WARNING: Sequences HM850015.1 and KM364757.1 are exactly identical


IMPORTANT WARNING: Sequences HM850015.1 and JN890816.1 are exactly identical


IMPORTANT WARNING: Sequences KT695485.1 and HQ619775.1 are exactly identical


IMPORTANT WARNING: Sequences KT695485.1 and AY395557.1 are exactly identical


IMPORTANT WARNING: Sequences KT695485.1 and HQ590234.1 are exactly identical


IMPORTANT WARNING: Sequences KT695485.1 and HM850293.1 are exactly identical


IMPORTANT WARNING: Sequences KT695485.1 and JN892097.1 are exactly identical


IMPORTANT WARNING: Sequences JN890923.1 and KF613052.1 are exactly identical


IMPORTANT WARNING: Sequences JN890923.1 and KX910828.1 are exactly identical


IMPORTANT WARNING: Sequences KM360958.1 and JN890694.1 are exactly identical


IMPORTANT WARNING: Sequences KM360958.1 and KF997397.1 are exactly identical


IMPORTANT WARNING: Sequences KM361039.1 and HQ901572.1 are exactly identical


IMPORTANT WARNING: Sequences KM361039.1 and U03725.1 are exactly identical


IMPORTANT WARNING: Sequences AF145457.1 and JQ512519.1 are exactly identical


IMPORTANT WARNING: Sequences AF145457.1 and HG765043.1 are exactly identical


IMPORTANT WARNING: Sequences AF145457.1 and KM360701.1 are exactly identical


IMPORTANT WARNING: Sequences AF145457.1 and AB480043.1 are exactly identical


IMPORTANT WARNING: Sequences AF145457.1 and X63662.1 are exactly identical


IMPORTANT WARNING: Sequences L01883.2 and KM360613.1 are exactly identical


IMPORTANT WARNING: Sequences JN890921.1 and KP126856.1 are exactly identical


IMPORTANT WARNING: Sequences JN890921.1 and HM850412.1 are exactly identical


IMPORTANT WARNING: Sequences JN890921.1 and HE574596.1 are exactly identical


IMPORTANT WARNING: Sequences JN890921.1 and EU677056.1 are exactly identical


IMPORTANT WARNING: Sequences JN890921.1 and KJ773958.1 are exactly identical


IMPORTANT WARNING: Sequences JN890921.1 and FJ395579.1 are exactly identical


IMPORTANT WARNING: Sequences JN890921.1 and KJ841631.1 are exactly identical


IMPORTANT WARNING: Sequences JN890921.1 and KM360875.1 are exactly identical


IMPORTANT WARNING: Sequences JN890921.1 and HM850419.1 are exactly identical


IMPORTANT WARNING: Sequences JN890921.1 and JQ858238.1 are exactly identical


IMPORTANT WARNING: Sequences JN890921.1 and JN891065.1 are exactly identical


IMPORTANT WARNING: Sequences AY215120.1 and GU817762.1 are exactly identical


IMPORTANT WARNING: Sequences AY215120.1 and KM360797.1 are exactly identical


IMPORTANT WARNING: Sequences AY215120.1 and HM850016.1 are exactly identical


IMPORTANT WARNING: Sequences AY215120.1 and HM850017.1 are exactly identical


IMPORTANT WARNING: Sequences HM535629.1 and FR865150.1 are exactly identical


IMPORTANT WARNING: Sequences HM535629.1 and KJ663801.1 are exactly identical


IMPORTANT WARNING: Sequences HM535629.1 and KR816697.1 are exactly identical


IMPORTANT WARNING: Sequences HM535629.1 and KJ841502.1 are exactly identical


IMPORTANT WARNING: Sequences JN891156.1 and HE963634.1 are exactly identical


IMPORTANT WARNING: Sequences JN891156.1 and U06805.1 are exactly identical


IMPORTANT WARNING: Sequences JN891156.1 and JN890849.1 are exactly identical


IMPORTANT WARNING: Sequences JN891156.1 and KU881195.1 are exactly identical


IMPORTANT WARNING: Sequences JN891156.1 and FN689381.1 are exactly identical


IMPORTANT WARNING: Sequences JN891156.1 and JF345175.1 are exactly identical


IMPORTANT WARNING: Sequences JN891156.1 and JX118090.1 are exactly identical


IMPORTANT WARNING: Sequences JN891156.1 and HM850009.1 are exactly identical


IMPORTANT WARNING: Sequences Z70111.1 and KM361025.1 are exactly identical


IMPORTANT WARNING: Sequences JX848446.1 and KF997331.1 are exactly identical


IMPORTANT WARNING: Sequences JX848446.1 and KJ841226.1 are exactly identical


IMPORTANT WARNING: Sequences JX848446.1 and FJ395574.1 are exactly identical


IMPORTANT WARNING: Sequences JX848446.1 and KF997321.1 are exactly identical


IMPORTANT WARNING: Sequences EF660721.1 and HM591065.1 are exactly identical


IMPORTANT WARNING: Sequences EF660721.1 and HQ619788.1 are exactly identical


IMPORTANT WARNING: Sequences EF660721.1 and KP088875.1 are exactly identical


IMPORTANT WARNING: Sequences EF660721.1 and HE574621.1 are exactly identical


IMPORTANT WARNING: Sequences EF660721.1 and AF456388.1 are exactly identical


IMPORTANT WARNING: Sequences EF660721.1 and FR831952.1 are exactly identical


IMPORTANT WARNING: Sequences JN893388.1 and JN893124.1 are exactly identical


IMPORTANT WARNING: Sequences KM877906.1 and AF307474.1 are exactly identical


IMPORTANT WARNING: Sequences KM877906.1 and FJ395602.1 are exactly identical


IMPORTANT WARNING: Sequences HM850361.1 and KM489055.2 are exactly identical


IMPORTANT WARNING: Sequences HM850361.1 and HQ235621.1 are exactly identical


IMPORTANT WARNING: Sequences HM850361.1 and HF572811.1 are exactly identical


IMPORTANT WARNING: Sequences HM850361.1 and KR737187.1 are exactly identical


IMPORTANT WARNING: Sequences HM850361.1 and KR736921.1 are exactly identical


IMPORTANT WARNING: Sequences HM850361.1 and HM850367.1 are exactly identical


IMPORTANT WARNING: Sequences HM850361.1 and JF772170.2 are exactly identical


IMPORTANT WARNING: Sequences HM850361.1 and GQ436617.1 are exactly identical


IMPORTANT WARNING: Sequences FN689369.1 and AB033894.1 are exactly identical


IMPORTANT WARNING: Sequences FN689369.1 and NC_028032.1 are exactly identical


IMPORTANT WARNING: Sequences FN689369.1 and AF061992.1 are exactly identical


IMPORTANT WARNING: Sequences FN689369.1 and AB033761.1 are exactly identical


IMPORTANT WARNING: Sequences AY395542.1 and HE963414.1 are exactly identical


IMPORTANT WARNING: Sequences AY395542.1 and JN892797.1 are exactly identical


IMPORTANT WARNING: Sequences AY395542.1 and HQ590035.1 are exactly identical


IMPORTANT WARNING: Sequences JN892761.1 and AY935732.1 are exactly identical


IMPORTANT WARNING: Sequences JN892761.1 and JN965707.1 are exactly identical


IMPORTANT WARNING: Sequences JN891951.1 and HQ590068.1 are exactly identical


IMPORTANT WARNING: Sequences DQ026595.1 and Y07936.1 are exactly identical


IMPORTANT WARNING: Sequences DQ026595.1 and KF977478.1 are exactly identical


IMPORTANT WARNING: Sequences DQ026595.1 and KU295046.1 are exactly identical


IMPORTANT WARNING: Sequences KX162930.1 and HM024277.1 are exactly identical


IMPORTANT WARNING: Sequences KX162930.1 and KP099643.1 are exactly identical


IMPORTANT WARNING: Sequences KX162930.1 and KP088534.1 are exactly identical


IMPORTANT WARNING: Sequences KX162930.1 and AY380891.1 are exactly identical


IMPORTANT WARNING: Sequences KX162930.1 and JF941305.1 are exactly identical


IMPORTANT WARNING: Sequences KX162930.1 and FN689378.1 are exactly identical


IMPORTANT WARNING: Sequences KX162930.1 and AY380888.1 are exactly identical


IMPORTANT WARNING: Sequences KP643714.1 and KF724374.1 are exactly identical


IMPORTANT WARNING: Sequences KP643714.1 and AB024948.1 are exactly identical


IMPORTANT WARNING: Sequences KP643714.1 and KJ593567.1 are exactly identical


IMPORTANT WARNING: Sequences JN891844.1 and KP643935.1 are exactly identical


IMPORTANT WARNING: Sequences JN891844.1 and KM361012.1 are exactly identical


IMPORTANT WARNING: Sequences JN891844.1 and KM360623.1 are exactly identical


IMPORTANT WARNING: Sequences KJ204371.1 and KM360806.1 are exactly identical


IMPORTANT WARNING: Sequences KJ204371.1 and JN893452.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841648.1 and HM850442.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841648.1 and L36452.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841648.1 and KM361030.1 are exactly identical


IMPORTANT WARNING: Sequences KF632846.1 and AM905819.1 are exactly identical


IMPORTANT WARNING: Sequences KF632846.1 and EU193178.1 are exactly identical


IMPORTANT WARNING: Sequences AY270079.1 and KF613023.1 are exactly identical


IMPORTANT WARNING: Sequences AY270079.1 and JN891253.1 are exactly identical


IMPORTANT WARNING: Sequences KC146521.1 and FJ587260.1 are exactly identical


IMPORTANT WARNING: Sequences KC146521.1 and HE963375.1 are exactly identical


IMPORTANT WARNING: Sequences KC146521.1 and FJ587261.1 are exactly identical


IMPORTANT WARNING: Sequences KC146521.1 and KM360690.1 are exactly identical


IMPORTANT WARNING: Sequences KC146521.1 and DQ356118.1 are exactly identical


IMPORTANT WARNING: Sequences KC146521.1 and FJ587285.1 are exactly identical


IMPORTANT WARNING: Sequences KC146521.1 and HQ589981.1 are exactly identical


IMPORTANT WARNING: Sequences KM360808.1 and KJ841354.1 are exactly identical


IMPORTANT WARNING: Sequences HE574582.1 and KM360671.1 are exactly identical


IMPORTANT WARNING: Sequences HE574582.1 and HQ589973.1 are exactly identical


IMPORTANT WARNING: Sequences KC342451.1 and EF653777.1 are exactly identical


IMPORTANT WARNING: Sequences KC342451.1 and EF653776.1 are exactly identical


IMPORTANT WARNING: Sequences KU878847.1 and GQ184365.1 are exactly identical


IMPORTANT WARNING: Sequences KU878847.1 and GQ184366.1 are exactly identical


IMPORTANT WARNING: Sequences KU878847.1 and GQ184375.1 are exactly identical


IMPORTANT WARNING: Sequences KU878847.1 and KU324629.1 are exactly identical


IMPORTANT WARNING: Sequences KU878847.1 and HM849824.1 are exactly identical


IMPORTANT WARNING: Sequences KU878847.1 and KJ716483.1 are exactly identical


IMPORTANT WARNING: Sequences KU878847.1 and AF267640.1 are exactly identical


IMPORTANT WARNING: Sequences JN891226.1 and KC483202.1 are exactly identical


IMPORTANT WARNING: Sequences JN891226.1 and JQ933413.1 are exactly identical


IMPORTANT WARNING: Sequences HM849943.1 and KM360746.1 are exactly identical


IMPORTANT WARNING: Sequences AB697623.1 and HM582635.1 are exactly identical


IMPORTANT WARNING: Sequences AB697623.1 and AB697624.1 are exactly identical


IMPORTANT WARNING: Sequences AB697623.1 and AB697626.1 are exactly identical


IMPORTANT WARNING: Sequences AB697623.1 and KM008163.1 are exactly identical


IMPORTANT WARNING: Sequences AB697623.1 and EF452170.1 are exactly identical


IMPORTANT WARNING: Sequences AB697623.1 and HM582645.1 are exactly identical


IMPORTANT WARNING: Sequences AB697623.1 and KF289654.1 are exactly identical


IMPORTANT WARNING: Sequences AY270137.1 and KR057197.1 are exactly identical


IMPORTANT WARNING: Sequences KP172046.1 and Z37413.1 are exactly identical


IMPORTANT WARNING: Sequences KP172046.1 and Z37414.1 are exactly identical


IMPORTANT WARNING: Sequences KF296634.1 and JN891411.1 are exactly identical


IMPORTANT WARNING: Sequences KF296634.1 and KJ451539.1 are exactly identical


IMPORTANT WARNING: Sequences KJ593665.1 and HQ590236.1 are exactly identical


IMPORTANT WARNING: Sequences KJ593665.1 and GU935441.1 are exactly identical


IMPORTANT WARNING: Sequences HE963531.1 and JN890635.1 are exactly identical


IMPORTANT WARNING: Sequences KX162952.1 and FJ395559.1 are exactly identical


IMPORTANT WARNING: Sequences KX162952.1 and AB817810.1 are exactly identical


IMPORTANT WARNING: Sequences KX162952.1 and KC737405.1 are exactly identical


IMPORTANT WARNING: Sequences KX162952.1 and FN689376.1 are exactly identical


IMPORTANT WARNING: Sequences FR865147.1 and AY395545.1 are exactly identical


IMPORTANT WARNING: Sequences JQ512564.1 and JQ512566.1 are exactly identical


IMPORTANT WARNING: Sequences JQ512564.1 and KU215903.2 are exactly identical


IMPORTANT WARNING: Sequences JQ512564.1 and FR831933.1 are exactly identical


IMPORTANT WARNING: Sequences JQ512564.1 and EF440570.1 are exactly identical


IMPORTANT WARNING: Sequences HE963343.1 and FR775291.1 are exactly identical


IMPORTANT WARNING: Sequences HE963343.1 and HM849801.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590028.1 and JX087694.1 are exactly identical


IMPORTANT WARNING: Sequences KX162958.1 and KM360831.1 are exactly identical


IMPORTANT WARNING: Sequences HM849779.1 and JN893133.1 are exactly identical


IMPORTANT WARNING: Sequences L12672.2 and GQ469841.1 are exactly identical


IMPORTANT WARNING: Sequences KX163064.1 and HQ591736.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and LT576797.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and JX848399.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and JX848403.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and KF613019.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and KX582038.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and HM849740.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and JX848431.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and KC870884.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and EU384938.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and KX527157.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and KM360610.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and KM361004.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and JN966050.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and JQ230995.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and HE574579.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and KM360653.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and JN891958.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746261.1 and L13641.1 are exactly identical


IMPORTANT WARNING: Sequences JX068719.1 and AB574872.1 are exactly identical


IMPORTANT WARNING: Sequences JX068719.1 and JX068711.1 are exactly identical


IMPORTANT WARNING: Sequences JX068719.1 and KP899645.1 are exactly identical


IMPORTANT WARNING: Sequences HE574586.1 and KJ746249.1 are exactly identical


IMPORTANT WARNING: Sequences HE574586.1 and JN892432.1 are exactly identical


IMPORTANT WARNING: Sequences HE574586.1 and KM360704.1 are exactly identical


IMPORTANT WARNING: Sequences JN893742.1 and JF943501.1 are exactly identical


IMPORTANT WARNING: Sequences JN893742.1 and HM357898.1 are exactly identical


IMPORTANT WARNING: Sequences JN893742.1 and EU676951.1 are exactly identical


IMPORTANT WARNING: Sequences KM360655.1 and HM849798.1 are exactly identical


IMPORTANT WARNING: Sequences KM360655.1 and KJ841121.1 are exactly identical


IMPORTANT WARNING: Sequences KM360655.1 and AY149374.1 are exactly identical


IMPORTANT WARNING: Sequences DQ099441.1 and JN892442.1 are exactly identical


IMPORTANT WARNING: Sequences KM538823.1 and JN891197.1 are exactly identical


IMPORTANT WARNING: Sequences KM538823.1 and AJ746276.1 are exactly identical


IMPORTANT WARNING: Sequences KM538823.1 and JN892561.1 are exactly identical


IMPORTANT WARNING: Sequences JN892868.1 and JN965557.1 are exactly identical


IMPORTANT WARNING: Sequences JN893462.1 and HM850459.1 are exactly identical


IMPORTANT WARNING: Sequences EF125156.1 and LT576857.1 are exactly identical


IMPORTANT WARNING: Sequences EF125156.1 and FR865131.1 are exactly identical


IMPORTANT WARNING: Sequences EF125156.1 and LT576855.1 are exactly identical


IMPORTANT WARNING: Sequences EF125156.1 and HE577877.1 are exactly identical


IMPORTANT WARNING: Sequences JN892037.1 and KJ841123.1 are exactly identical


IMPORTANT WARNING: Sequences KM361028.1 and KF163410.1 are exactly identical


IMPORTANT WARNING: Sequences KM361028.1 and HF572812.1 are exactly identical


IMPORTANT WARNING: Sequences AB598689.1 and EF394237.1 are exactly identical


IMPORTANT WARNING: Sequences AB598689.1 and KC795814.1 are exactly identical


IMPORTANT WARNING: Sequences AB598689.1 and KC795822.1 are exactly identical


IMPORTANT WARNING: Sequences AB598689.1 and AB575106.1 are exactly identical


IMPORTANT WARNING: Sequences JN891758.1 and HQ590129.1 are exactly identical


IMPORTANT WARNING: Sequences JN891758.1 and JN847834.1 are exactly identical


IMPORTANT WARNING: Sequences JN891758.1 and KM360682.1 are exactly identical


IMPORTANT WARNING: Sequences JN891758.1 and NC_009274.1 are exactly identical


IMPORTANT WARNING: Sequences JN891758.1 and HE963547.1 are exactly identical


IMPORTANT WARNING: Sequences JN891758.1 and HM850131.1 are exactly identical


IMPORTANT WARNING: Sequences JN891758.1 and JN892935.1 are exactly identical


IMPORTANT WARNING: Sequences JN891758.1 and KM360983.1 are exactly identical


IMPORTANT WARNING: Sequences JN891758.1 and AY167982.1 are exactly identical


IMPORTANT WARNING: Sequences AY664855.1 and FR831929.1 are exactly identical


IMPORTANT WARNING: Sequences AY664855.1 and JQ512507.1 are exactly identical


IMPORTANT WARNING: Sequences AY664855.1 and AB015648.1 are exactly identical


IMPORTANT WARNING: Sequences AY664855.1 and AB015649.1 are exactly identical


IMPORTANT WARNING: Sequences AY664855.1 and AB029654.1 are exactly identical


IMPORTANT WARNING: Sequences AY664855.1 and AB029651.1 are exactly identical


IMPORTANT WARNING: Sequences AY664855.1 and AB029646.1 are exactly identical


IMPORTANT WARNING: Sequences KP643990.1 and KJ773848.1 are exactly identical


IMPORTANT WARNING: Sequences U03729.1 and HM850324.1 are exactly identical


IMPORTANT WARNING: Sequences U03729.1 and JN113277.1 are exactly identical


IMPORTANT WARNING: Sequences U03729.1 and JN113275.1 are exactly identical


IMPORTANT WARNING: Sequences U03729.1 and JN113276.1 are exactly identical


IMPORTANT WARNING: Sequences U03729.1 and JN113278.1 are exactly identical


IMPORTANT WARNING: Sequences U03729.1 and JN113279.1 are exactly identical


IMPORTANT WARNING: Sequences FJ395569.1 and LNRQ01000011.1 are exactly identical


IMPORTANT WARNING: Sequences FJ395569.1 and HM849948.1 are exactly identical


IMPORTANT WARNING: Sequences U06223.1 and U06222.1 are exactly identical


IMPORTANT WARNING: Sequences KJ204289.1 and JN890567.1 are exactly identical


IMPORTANT WARNING: Sequences KX015731.1 and KX015747.1 are exactly identical


IMPORTANT WARNING: Sequences HM850117.1 and JN847819.1 are exactly identical


IMPORTANT WARNING: Sequences JN891624.1 and KJ746229.1 are exactly identical


IMPORTANT WARNING: Sequences KX015735.1 and HM849765.1 are exactly identical


IMPORTANT WARNING: Sequences KX015735.1 and KX090211.1 are exactly identical


IMPORTANT WARNING: Sequences KX015735.1 and KX090212.1 are exactly identical


IMPORTANT WARNING: Sequences KX015735.1 and KM360629.1 are exactly identical


IMPORTANT WARNING: Sequences JF832067.1 and HE574628.1 are exactly identical


IMPORTANT WARNING: Sequences JF832067.1 and KM114199.1 are exactly identical


IMPORTANT WARNING: Sequences KM974746.1 and HM850251.1 are exactly identical


IMPORTANT WARNING: Sequences JQ412434.1 and HM850437.1 are exactly identical


IMPORTANT WARNING: Sequences JQ412434.1 and JN893503.1 are exactly identical


IMPORTANT WARNING: Sequences HQ589951.1 and JN891313.1 are exactly identical


IMPORTANT WARNING: Sequences HQ589951.1 and KJ841100.1 are exactly identical


IMPORTANT WARNING: Sequences HQ589951.1 and HE574612.1 are exactly identical


IMPORTANT WARNING: Sequences HQ589951.1 and KT626769.1 are exactly identical


IMPORTANT WARNING: Sequences EU599847.1 and HQ590188.1 are exactly identical


IMPORTANT WARNING: Sequences EU599847.1 and JN890916.1 are exactly identical


IMPORTANT WARNING: Sequences JF832076.1 and U62034.1 are exactly identical


IMPORTANT WARNING: Sequences JX090450.1 and KM360826.1 are exactly identical


IMPORTANT WARNING: Sequences JX090450.1 and JX903179.1 are exactly identical


IMPORTANT WARNING: Sequences FN689364.1 and HE963717.1 are exactly identical


IMPORTANT WARNING: Sequences KX527462.1 and JN890583.1 are exactly identical


IMPORTANT WARNING: Sequences JQ512558.1 and AB501189.1 are exactly identical


IMPORTANT WARNING: Sequences JN892290.1 and JX913527.1 are exactly identical


IMPORTANT WARNING: Sequences HG765079.1 and KX163037.1 are exactly identical


IMPORTANT WARNING: Sequences HG765079.1 and KJ773864.1 are exactly identical


IMPORTANT WARNING: Sequences HG765079.1 and HQ644067.1 are exactly identical


IMPORTANT WARNING: Sequences HG765079.1 and HM850334.1 are exactly identical


IMPORTANT WARNING: Sequences HG765079.1 and KC584884.1 are exactly identical


IMPORTANT WARNING: Sequences KM372984.1 and JN890605.1 are exactly identical


IMPORTANT WARNING: Sequences KX162914.1 and FR865138.1 are exactly identical


IMPORTANT WARNING: Sequences KX162914.1 and HM849918.1 are exactly identical


IMPORTANT WARNING: Sequences JN935607.1 and JN935609.1 are exactly identical


IMPORTANT WARNING: Sequences JN935607.1 and JN935615.1 are exactly identical


IMPORTANT WARNING: Sequences JN935607.1 and JN935650.1 are exactly identical


IMPORTANT WARNING: Sequences JN935607.1 and JN935653.1 are exactly identical


IMPORTANT WARNING: Sequences KM356099.1 and JQ933250.1 are exactly identical


IMPORTANT WARNING: Sequences JN891217.1 and AJ746290.1 are exactly identical


IMPORTANT WARNING: Sequences JN891217.1 and JN891301.1 are exactly identical


IMPORTANT WARNING: Sequences JN891217.1 and KJ599188.1 are exactly identical


IMPORTANT WARNING: Sequences AF419837.1 and KC484316.1 are exactly identical


IMPORTANT WARNING: Sequences AF419837.1 and JN892593.1 are exactly identical


IMPORTANT WARNING: Sequences AF419837.1 and KF163412.1 are exactly identical


IMPORTANT WARNING: Sequences HM849778.1 and JN891846.1 are exactly identical


IMPORTANT WARNING: Sequences HM849778.1 and JN893478.1 are exactly identical


IMPORTANT WARNING: Sequences KF997368.1 and JN892562.1 are exactly identical


IMPORTANT WARNING: Sequences KM360656.1 and JN891746.1 are exactly identical


IMPORTANT WARNING: Sequences AJ389596.1 and JN407338.2 are exactly identical


IMPORTANT WARNING: Sequences AJ389596.1 and EU676933.1 are exactly identical


IMPORTANT WARNING: Sequences AJ389596.1 and JN407339.1 are exactly identical


IMPORTANT WARNING: Sequences AJ389596.1 and JN407340.1 are exactly identical


IMPORTANT WARNING: Sequences AJ389596.1 and JN892249.1 are exactly identical


IMPORTANT WARNING: Sequences AJ389596.1 and KJ204387.1 are exactly identical


IMPORTANT WARNING: Sequences LT576798.1 and JN891162.1 are exactly identical


IMPORTANT WARNING: Sequences LT576798.1 and LK054370.1 are exactly identical


IMPORTANT WARNING: Sequences LT576798.1 and HM849814.1 are exactly identical


IMPORTANT WARNING: Sequences KU291475.1 and AM849355.1 are exactly identical


IMPORTANT WARNING: Sequences KU291475.1 and KX198597.1 are exactly identical


IMPORTANT WARNING: Sequences JN893241.1 and KJ204392.1 are exactly identical


IMPORTANT WARNING: Sequences JN893241.1 and HM850285.1 are exactly identical


IMPORTANT WARNING: Sequences JN893241.1 and HM850284.1 are exactly identical


IMPORTANT WARNING: Sequences JQ512561.1 and KR061358.1 are exactly identical


IMPORTANT WARNING: Sequences JQ512561.1 and KM360881.1 are exactly identical


IMPORTANT WARNING: Sequences KF997479.1 and JN892522.1 are exactly identical


IMPORTANT WARNING: Sequences KF997479.1 and KP896769.1 are exactly identical


IMPORTANT WARNING: Sequences KF997479.1 and JN661202.1 are exactly identical


IMPORTANT WARNING: Sequences KF997479.1 and JN891169.1 are exactly identical


IMPORTANT WARNING: Sequences KF997479.1 and JN891171.1 are exactly identical


IMPORTANT WARNING: Sequences KF997479.1 and KX421139.1 are exactly identical


IMPORTANT WARNING: Sequences KF997479.1 and KX421137.1 are exactly identical


IMPORTANT WARNING: Sequences KF997479.1 and KJ806199.1 are exactly identical


IMPORTANT WARNING: Sequences KF997479.1 and KF997296.1 are exactly identical


IMPORTANT WARNING: Sequences KJ008942.1 and AY570423.1 are exactly identical


IMPORTANT WARNING: Sequences KJ008942.1 and Z37435.1 are exactly identical


IMPORTANT WARNING: Sequences KJ008942.1 and Z37447.1 are exactly identical


IMPORTANT WARNING: Sequences KJ008942.1 and HM850333.1 are exactly identical


IMPORTANT WARNING: Sequences KJ008942.1 and KM360960.1 are exactly identical


IMPORTANT WARNING: Sequences KJ008942.1 and JN893721.1 are exactly identical


IMPORTANT WARNING: Sequences KJ008942.1 and FJ513157.1 are exactly identical


IMPORTANT WARNING: Sequences FJ626614.1 and GQ248663.1 are exactly identical


IMPORTANT WARNING: Sequences KM360946.1 and JN891494.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841424.1 and HQ590187.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841424.1 and HM850187.1 are exactly identical


IMPORTANT WARNING: Sequences JN892314.1 and AY216653.1 are exactly identical


IMPORTANT WARNING: Sequences JF942334.1 and FJ914171.1 are exactly identical


IMPORTANT WARNING: Sequences JF942334.1 and JX511990.1 are exactly identical


IMPORTANT WARNING: Sequences KM360950.1 and JN890802.1 are exactly identical


IMPORTANT WARNING: Sequences KC584885.1 and JN160603.2 are exactly identical


IMPORTANT WARNING: Sequences KC584885.1 and KM360996.1 are exactly identical


IMPORTANT WARNING: Sequences JN965301.1 and DQ849146.1 are exactly identical


IMPORTANT WARNING: Sequences JN965301.1 and AB574664.1 are exactly identical


IMPORTANT WARNING: Sequences KF997366.1 and KX163046.1 are exactly identical


IMPORTANT WARNING: Sequences JN890967.1 and LC004545.1 are exactly identical


IMPORTANT WARNING: Sequences JN890967.1 and HE574620.1 are exactly identical


IMPORTANT WARNING: Sequences JN890967.1 and JN966010.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600472.1 and AJ746298.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600472.1 and HM850252.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600472.1 and AJ784832.1 are exactly identical


IMPORTANT WARNING: Sequences HE963488.1 and KM360785.1 are exactly identical


IMPORTANT WARNING: Sequences HE963488.1 and JN892443.1 are exactly identical


IMPORTANT WARNING: Sequences HE963488.1 and JN892444.1 are exactly identical


IMPORTANT WARNING: Sequences JN892707.1 and HM849830.1 are exactly identical


IMPORTANT WARNING: Sequences KM361029.1 and Y10707.1 are exactly identical


IMPORTANT WARNING: Sequences KM361029.1 and JN891409.1 are exactly identical


IMPORTANT WARNING: Sequences FJ395572.1 and KM360998.1 are exactly identical


IMPORTANT WARNING: Sequences KC483119.1 and AY395548.1 are exactly identical


IMPORTANT WARNING: Sequences KC483119.1 and AJ419945.1 are exactly identical


IMPORTANT WARNING: Sequences HM850120.1 and D88906.1 are exactly identical


IMPORTANT WARNING: Sequences JN891132.1 and KC584889.1 are exactly identical


IMPORTANT WARNING: Sequences HM850118.1 and JN847821.1 are exactly identical


IMPORTANT WARNING: Sequences U06792.1 and JN892118.1 are exactly identical


IMPORTANT WARNING: Sequences KM538834.1 and FN870386.1 are exactly identical


IMPORTANT WARNING: Sequences KM538834.1 and DQ005609.1 are exactly identical


IMPORTANT WARNING: Sequences KM538834.1 and KM360749.1 are exactly identical


IMPORTANT WARNING: Sequences KM538834.1 and HM849917.1 are exactly identical


IMPORTANT WARNING: Sequences JF830457.1 and KP644045.1 are exactly identical


IMPORTANT WARNING: Sequences JF830457.1 and KX162969.1 are exactly identical


IMPORTANT WARNING: Sequences JF830457.1 and HQ384901.1 are exactly identical


IMPORTANT WARNING: Sequences JN891160.1 and JN891802.1 are exactly identical


IMPORTANT WARNING: Sequences KX015736.1 and GQ436528.1 are exactly identical


IMPORTANT WARNING: Sequences KX015736.1 and KX015738.1 are exactly identical


IMPORTANT WARNING: Sequences KP402620.1 and JN890565.1 are exactly identical


IMPORTANT WARNING: Sequences KP402620.1 and KM360890.1 are exactly identical


IMPORTANT WARNING: Sequences KP402620.1 and AF206823.1 are exactly identical


IMPORTANT WARNING: Sequences KP402620.1 and M62570.1 are exactly identical


IMPORTANT WARNING: Sequences EU676931.1 and HM850265.1 are exactly identical


IMPORTANT WARNING: Sequences KP638676.1 and KP638675.1 are exactly identical


IMPORTANT WARNING: Sequences AB787656.1 and DQ400350.1 are exactly identical


IMPORTANT WARNING: Sequences AB787656.1 and JN892886.1 are exactly identical


IMPORTANT WARNING: Sequences AB787656.1 and AM905730.1 are exactly identical


IMPORTANT WARNING: Sequences AB787656.1 and JN892481.1 are exactly identical


IMPORTANT WARNING: Sequences AB787656.1 and JN965625.1 are exactly identical


IMPORTANT WARNING: Sequences FN689367.1 and KC485205.1 are exactly identical


IMPORTANT WARNING: Sequences FN689367.1 and JF429908.2 are exactly identical


IMPORTANT WARNING: Sequences FN689367.1 and AP008956.1 are exactly identical


IMPORTANT WARNING: Sequences FR865151.1 and KX162903.1 are exactly identical


IMPORTANT WARNING: Sequences JN892527.1 and HM850084.1 are exactly identical


IMPORTANT WARNING: Sequences AB088811.1 and U80714.1 are exactly identical


IMPORTANT WARNING: Sequences AB088811.1 and JN890676.1 are exactly identical


IMPORTANT WARNING: Sequences AB088811.1 and JX848473.1 are exactly identical


IMPORTANT WARNING: Sequences AB088811.1 and JN966049.1 are exactly identical


IMPORTANT WARNING: Sequences AB088811.1 and GQ452334.1 are exactly identical


IMPORTANT WARNING: Sequences AB088811.1 and KX527477.1 are exactly identical


IMPORTANT WARNING: Sequences LM993032.1 and LM993030.1 are exactly identical


IMPORTANT WARNING: Sequences LM993032.1 and KF997467.1 are exactly identical


IMPORTANT WARNING: Sequences JN891258.1 and U80701.1 are exactly identical


IMPORTANT WARNING: Sequences Z37398.1 and Z37396.1 are exactly identical


IMPORTANT WARNING: Sequences Z37398.1 and Z37397.1 are exactly identical


IMPORTANT WARNING: Sequences KJ204351.1 and JN890881.1 are exactly identical


IMPORTANT WARNING: Sequences JQ512528.1 and JQ512530.1 are exactly identical


IMPORTANT WARNING: Sequences JQ512538.1 and JN867578.1 are exactly identical


IMPORTANT WARNING: Sequences JQ512538.1 and GQ436729.1 are exactly identical


IMPORTANT WARNING: Sequences JN892818.1 and JN892550.1 are exactly identical


IMPORTANT WARNING: Sequences JN892818.1 and GU344675.1 are exactly identical


IMPORTANT WARNING: Sequences JN892818.1 and HM849970.1 are exactly identical


IMPORTANT WARNING: Sequences JN892818.1 and AB369953.1 are exactly identical


IMPORTANT WARNING: Sequences JN892818.1 and JN965505.1 are exactly identical


IMPORTANT WARNING: Sequences JN891195.1 and AY836172.1 are exactly identical


IMPORTANT WARNING: Sequences JN891195.1 and KM360823.1 are exactly identical


IMPORTANT WARNING: Sequences JN891195.1 and LT576827.1 are exactly identical


IMPORTANT WARNING: Sequences JN891195.1 and AY601672.1 are exactly identical


IMPORTANT WARNING: Sequences JN891363.1 and KJ746288.1 are exactly identical


IMPORTANT WARNING: Sequences JN891363.1 and KJ746287.1 are exactly identical


IMPORTANT WARNING: Sequences JN891363.1 and HQ590096.1 are exactly identical


IMPORTANT WARNING: Sequences JN891363.1 and HM849987.1 are exactly identical


IMPORTANT WARNING: Sequences JN891363.1 and HM849989.1 are exactly identical


IMPORTANT WARNING: Sequences KR233931.1 and KR233930.1 are exactly identical


IMPORTANT WARNING: Sequences KR233931.1 and JX068689.1 are exactly identical


IMPORTANT WARNING: Sequences KR233931.1 and JX068691.1 are exactly identical


IMPORTANT WARNING: Sequences KR233931.1 and KP899628.1 are exactly identical


IMPORTANT WARNING: Sequences KR233931.1 and JX068688.1 are exactly identical


IMPORTANT WARNING: Sequences KR233931.1 and JX068690.1 are exactly identical


IMPORTANT WARNING: Sequences JN893746.1 and FM883607.1 are exactly identical


IMPORTANT WARNING: Sequences EU053919.1 and KP402634.1 are exactly identical


IMPORTANT WARNING: Sequences HQ593829.1 and JN893559.1 are exactly identical


IMPORTANT WARNING: Sequences HQ593829.1 and KC483455.1 are exactly identical


IMPORTANT WARNING: Sequences HQ593829.1 and KR297244.1 are exactly identical


IMPORTANT WARNING: Sequences D28865.1 and KM360612.1 are exactly identical


IMPORTANT WARNING: Sequences D28865.1 and AJ879453.1 are exactly identical


IMPORTANT WARNING: Sequences KT626832.1 and KM360736.1 are exactly identical


IMPORTANT WARNING: Sequences JN892323.1 and JN893495.1 are exactly identical


IMPORTANT WARNING: Sequences JN892323.1 and HM850263.1 are exactly identical


IMPORTANT WARNING: Sequences KC164338.1 and KF163544.1 are exactly identical


IMPORTANT WARNING: Sequences KC164338.1 and FR821349.1 are exactly identical


IMPORTANT WARNING: Sequences JN890602.1 and KP126862.1 are exactly identical


IMPORTANT WARNING: Sequences JN890602.1 and KU321683.1 are exactly identical


IMPORTANT WARNING: Sequences JN890602.1 and Z70173.1 are exactly identical


IMPORTANT WARNING: Sequences JN890602.1 and HM850166.1 are exactly identical


IMPORTANT WARNING: Sequences KP172074.1 and HQ619754.1 are exactly identical


IMPORTANT WARNING: Sequences JN965586.1 and JN892528.1 are exactly identical


IMPORTANT WARNING: Sequences JN965586.1 and KX527474.1 are exactly identical


IMPORTANT WARNING: Sequences KX162865.1 and FN689361.1 are exactly identical


IMPORTANT WARNING: Sequences KX162865.1 and DQ978425.1 are exactly identical


IMPORTANT WARNING: Sequences KX162865.1 and FJ395591.1 are exactly identical


IMPORTANT WARNING: Sequences KX162865.1 and KX162875.1 are exactly identical


IMPORTANT WARNING: Sequences KX162865.1 and KX162877.1 are exactly identical


IMPORTANT WARNING: Sequences KX162865.1 and DQ978424.1 are exactly identical


IMPORTANT WARNING: Sequences KX162865.1 and KU902617.1 are exactly identical


IMPORTANT WARNING: Sequences KX162865.1 and DQ978399.1 are exactly identical


IMPORTANT WARNING: Sequences NC_012119.1 and AB856289.1 are exactly identical


IMPORTANT WARNING: Sequences Y07940.1 and AB574651.1 are exactly identical


IMPORTANT WARNING: Sequences EF469949.1 and KM360662.1 are exactly identical


IMPORTANT WARNING: Sequences EF469949.1 and JX993044.1 are exactly identical


IMPORTANT WARNING: Sequences HM850139.1 and U74213.1 are exactly identical


IMPORTANT WARNING: Sequences HM850139.1 and KM360864.1 are exactly identical


IMPORTANT WARNING: Sequences HM850139.1 and KJ841388.1 are exactly identical


IMPORTANT WARNING: Sequences LN908031.1 and KJ599158.1 are exactly identical


IMPORTANT WARNING: Sequences LN908031.1 and KT280101.1 are exactly identical


IMPORTANT WARNING: Sequences LN908031.1 and AJ784822.1 are exactly identical


IMPORTANT WARNING: Sequences AY570436.1 and KM360964.1 are exactly identical


IMPORTANT WARNING: Sequences FN689362.1 and KX162935.1 are exactly identical


IMPORTANT WARNING: Sequences L13189.2 and KM360955.1 are exactly identical


IMPORTANT WARNING: Sequences FJ395560.1 and JN890766.1 are exactly identical


IMPORTANT WARNING: Sequences FJ395560.1 and HM849876.1 are exactly identical


IMPORTANT WARNING: Sequences X81091.1 and HQ590107.1 are exactly identical


IMPORTANT WARNING: Sequences X81091.1 and HM850018.1 are exactly identical


IMPORTANT WARNING: Sequences X81091.1 and KJ841334.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841228.1 and HM849929.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841228.1 and JN893327.1 are exactly identical


IMPORTANT WARNING: Sequences JN892601.1 and HQ332073.1 are exactly identical


IMPORTANT WARNING: Sequences KM360811.1 and FR865140.1 are exactly identical


IMPORTANT WARNING: Sequences KM360811.1 and JN893033.1 are exactly identical


IMPORTANT WARNING: Sequences JN892526.1 and KM360822.1 are exactly identical


IMPORTANT WARNING: Sequences JN892526.1 and JN965588.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841629.1 and HQ644077.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841629.1 and HM850409.1 are exactly identical


IMPORTANT WARNING: Sequences GQ248601.1 and AY226135.1 are exactly identical


IMPORTANT WARNING: Sequences KM360819.1 and HM849822.1 are exactly identical


IMPORTANT WARNING: Sequences JN891111.1 and HE963479.1 are exactly identical


IMPORTANT WARNING: Sequences HM850467.1 and HM850468.1 are exactly identical


IMPORTANT WARNING: Sequences HM850467.1 and JN891131.1 are exactly identical


IMPORTANT WARNING: Sequences HM850467.1 and JN891418.1 are exactly identical


IMPORTANT WARNING: Sequences KM360839.1 and KF055121.1 are exactly identical


IMPORTANT WARNING: Sequences KM360839.1 and HM850102.1 are exactly identical


IMPORTANT WARNING: Sequences NC_024261.1 and JN891545.1 are exactly identical


IMPORTANT WARNING: Sequences AF394976.1 and JN891810.1 are exactly identical


IMPORTANT WARNING: Sequences JN234970.1 and EU554022.1 are exactly identical


IMPORTANT WARNING: Sequences JN234970.1 and KJ841450.1 are exactly identical


IMPORTANT WARNING: Sequences AJ746279.1 and HM850053.1 are exactly identical


IMPORTANT WARNING: Sequences AJ746279.1 and HQ600455.1 are exactly identical


IMPORTANT WARNING: Sequences AJ746279.1 and AM235062.1 are exactly identical


IMPORTANT WARNING: Sequences EU677010.1 and JN891547.1 are exactly identical


IMPORTANT WARNING: Sequences EU677010.1 and HM850363.1 are exactly identical


IMPORTANT WARNING: Sequences EU677010.1 and HQ590279.1 are exactly identical


IMPORTANT WARNING: Sequences EU193236.1 and GU067578.1 are exactly identical


IMPORTANT WARNING: Sequences EU193236.1 and DQ005607.1 are exactly identical


IMPORTANT WARNING: Sequences EU193236.1 and EU193250.1 are exactly identical


IMPORTANT WARNING: Sequences EU193236.1 and KM360654.1 are exactly identical


IMPORTANT WARNING: Sequences GQ436684.1 and JN892623.1 are exactly identical


IMPORTANT WARNING: Sequences GQ436684.1 and JN892185.1 are exactly identical


IMPORTANT WARNING: Sequences GQ436684.1 and JN891708.1 are exactly identical


IMPORTANT WARNING: Sequences KJ773275.1 and HQ589962.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590070.1 and JN189511.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590070.1 and AY268883.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590070.1 and JQ935266.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590070.1 and EU677095.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590070.1 and JQ935269.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590070.1 and JQ935245.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590070.1 and JQ935276.1 are exactly identical


IMPORTANT WARNING: Sequences FR865160.1 and JN892252.1 are exactly identical


IMPORTANT WARNING: Sequences NC_027467.1 and KF522678.1 are exactly identical


IMPORTANT WARNING: Sequences NC_027467.1 and HE963320.1 are exactly identical


IMPORTANT WARNING: Sequences HQ619740.1 and KM360644.1 are exactly identical


IMPORTANT WARNING: Sequences KT626775.1 and AM999865.1 are exactly identical


IMPORTANT WARNING: Sequences AP009370.1 and KU739561.1 are exactly identical


IMPORTANT WARNING: Sequences AP009370.1 and HQ589972.1 are exactly identical


IMPORTANT WARNING: Sequences AP009370.1 and KM360664.1 are exactly identical


IMPORTANT WARNING: Sequences EU053906.1 and FJ626581.1 are exactly identical


IMPORTANT WARNING: Sequences EU053906.1 and HQ589979.1 are exactly identical


IMPORTANT WARNING: Sequences HQ901565.1 and AB004896.1 are exactly identical


IMPORTANT WARNING: Sequences HQ901565.1 and U80709.1 are exactly identical


IMPORTANT WARNING: Sequences FJ395581.1 and JN891289.1 are exactly identical


IMPORTANT WARNING: Sequences FJ395581.1 and KJ746275.1 are exactly identical


IMPORTANT WARNING: Sequences AF525276.1 and EF463157.1 are exactly identical


IMPORTANT WARNING: Sequences AF525276.1 and KP774920.1 are exactly identical


IMPORTANT WARNING: Sequences KC482467.1 and KP899658.1 are exactly identical


IMPORTANT WARNING: Sequences KC482467.1 and KT960524.1 are exactly identical


IMPORTANT WARNING: Sequences KF997453.1 and KJ746214.1 are exactly identical


IMPORTANT WARNING: Sequences KF997453.1 and JX848530.1 are exactly identical


IMPORTANT WARNING: Sequences KX162979.1 and HQ590174.1 are exactly identical


IMPORTANT WARNING: Sequences KX162979.1 and JQ391355.1 are exactly identical


IMPORTANT WARNING: Sequences Z11973.1 and KP966114.1 are exactly identical


IMPORTANT WARNING: Sequences FJ395580.1 and HE963663.1 are exactly identical


IMPORTANT WARNING: Sequences FJ395580.1 and HQ619732.1 are exactly identical


IMPORTANT WARNING: Sequences FJ395580.1 and JN890944.1 are exactly identical


IMPORTANT WARNING: Sequences HM850234.1 and KM360915.1 are exactly identical


IMPORTANT WARNING: Sequences U32163.1 and NC_023102.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590324.1 and JN891024.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590324.1 and JN891026.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590324.1 and JN891062.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590324.1 and KR080512.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590324.1 and JN891027.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590324.1 and KJ746276.1 are exactly identical


IMPORTANT WARNING: Sequences KJ204402.1 and JN891310.1 are exactly identical


IMPORTANT WARNING: Sequences KJ204402.1 and KM360972.1 are exactly identical


IMPORTANT WARNING: Sequences JN892089.1 and HM850063.1 are exactly identical


IMPORTANT WARNING: Sequences KM360973.1 and KP974259.1 are exactly identical


IMPORTANT WARNING: Sequences FJ670181.1 and JN892165.1 are exactly identical


IMPORTANT WARNING: Sequences DQ859163.1 and HM849805.1 are exactly identical


IMPORTANT WARNING: Sequences HM850470.1 and KF713070.1 are exactly identical


IMPORTANT WARNING: Sequences HM850470.1 and JN891423.1 are exactly identical


IMPORTANT WARNING: Sequences HM850470.1 and KJ599162.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600395.1 and AY632367.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600395.1 and KP056891.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600395.1 and AJ746295.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600395.1 and KF537347.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600395.1 and HE577856.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600395.1 and KF697233.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600395.1 and KF537348.1 are exactly identical


IMPORTANT WARNING: Sequences HQ600395.1 and KM360930.1 are exactly identical


IMPORTANT WARNING: Sequences JN892322.1 and HM850085.1 are exactly identical


IMPORTANT WARNING: Sequences JN892322.1 and HM850091.1 are exactly identical


IMPORTANT WARNING: Sequences AB044899.1 and EF551065.1 are exactly identical


IMPORTANT WARNING: Sequences AB044899.1 and HE963606.1 are exactly identical


IMPORTANT WARNING: Sequences L05037.2 and HM850074.1 are exactly identical


IMPORTANT WARNING: Sequences L05037.2 and HM850075.1 are exactly identical


IMPORTANT WARNING: Sequences L05037.2 and KP089569.1 are exactly identical


IMPORTANT WARNING: Sequences HM850243.1 and EF653767.1 are exactly identical


IMPORTANT WARNING: Sequences JN892509.1 and KC482839.1 are exactly identical


IMPORTANT WARNING: Sequences KP088884.1 and JN890677.1 are exactly identical


IMPORTANT WARNING: Sequences AM999810.1 and HM849940.1 are exactly identical


IMPORTANT WARNING: Sequences AM999810.1 and Y13015.1 are exactly identical


IMPORTANT WARNING: Sequences JN890687.1 and HM849744.1 are exactly identical


IMPORTANT WARNING: Sequences HM850229.1 and JN890645.1 are exactly identical


IMPORTANT WARNING: Sequences KR073292.1 and LT576825.1 are exactly identical


IMPORTANT WARNING: Sequences KC481634.1 and JN587320.1 are exactly identical


IMPORTANT WARNING: Sequences KC481634.1 and KJ593575.1 are exactly identical


IMPORTANT WARNING: Sequences HM849857.1 and JN891136.1 are exactly identical


IMPORTANT WARNING: Sequences JN891311.1 and HM850345.1 are exactly identical


IMPORTANT WARNING: Sequences JN891311.1 and Z37459.1 are exactly identical


IMPORTANT WARNING: Sequences FJ212219.1 and GQ891228.1 are exactly identical


IMPORTANT WARNING: Sequences FJ212219.1 and HM850302.1 are exactly identical


IMPORTANT WARNING: Sequences KP896759.1 and HM850456.1 are exactly identical


IMPORTANT WARNING: Sequences KP896759.1 and KJ850242.1 are exactly identical


IMPORTANT WARNING: Sequences KP896759.1 and KU053796.1 are exactly identical


IMPORTANT WARNING: Sequences KP896759.1 and AB517630.1 are exactly identical


IMPORTANT WARNING: Sequences KP896759.1 and KP896802.1 are exactly identical


IMPORTANT WARNING: Sequences KP896759.1 and HQ590328.1 are exactly identical


IMPORTANT WARNING: Sequences KP896759.1 and KJ773978.1 are exactly identical


IMPORTANT WARNING: Sequences KM360786.1 and JN890655.1 are exactly identical


IMPORTANT WARNING: Sequences M83542.1 and JN892006.1 are exactly identical


IMPORTANT WARNING: Sequences JN891837.1 and JN893082.1 are exactly identical


IMPORTANT WARNING: Sequences KF293392.1 and KC817303.1 are exactly identical


IMPORTANT WARNING: Sequences KF293392.1 and EU554017.1 are exactly identical


IMPORTANT WARNING: Sequences FJ395600.1 and KF138259.1 are exactly identical


IMPORTANT WARNING: Sequences AB574992.1 and KJ593452.1 are exactly identical


IMPORTANT WARNING: Sequences AB574992.1 and JX874069.1 are exactly identical


IMPORTANT WARNING: Sequences AB574992.1 and HQ590127.1 are exactly identical


IMPORTANT WARNING: Sequences JN890853.1 and GU344672.1 are exactly identical


IMPORTANT WARNING: Sequences KM360843.1 and JN890871.1 are exactly identical


IMPORTANT WARNING: Sequences JN891161.1 and HM850381.1 are exactly identical


IMPORTANT WARNING: Sequences JN891161.1 and JN891830.1 are exactly identical


IMPORTANT WARNING: Sequences FJ626582.1 and JF331667.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841619.1 and KF186519.1 are exactly identical


IMPORTANT WARNING: Sequences KF158087.1 and HE963457.1 are exactly identical


IMPORTANT WARNING: Sequences KF158087.1 and JN892864.1 are exactly identical


IMPORTANT WARNING: Sequences JN892381.1 and U96659.1 are exactly identical


IMPORTANT WARNING: Sequences KF186509.1 and HQ590297.1 are exactly identical


IMPORTANT WARNING: Sequences KM360833.1 and KF997297.1 are exactly identical


IMPORTANT WARNING: Sequences FJ956852.1 and FJ956829.1 are exactly identical


IMPORTANT WARNING: Sequences HE963613.1 and Z37433.1 are exactly identical


IMPORTANT WARNING: Sequences HE963613.1 and KP644119.1 are exactly identical


IMPORTANT WARNING: Sequences HE963613.1 and KM886813.1 are exactly identical


IMPORTANT WARNING: Sequences HE963613.1 and KM360941.1 are exactly identical


IMPORTANT WARNING: Sequences JX100698.1 and KF632851.1 are exactly identical


IMPORTANT WARNING: Sequences KM360777.1 and KX162933.1 are exactly identical


IMPORTANT WARNING: Sequences KM360877.1 and JN892933.1 are exactly identical


IMPORTANT WARNING: Sequences KM360877.1 and JN893344.1 are exactly identical


IMPORTANT WARNING: Sequences KM360877.1 and KJ204376.1 are exactly identical


IMPORTANT WARNING: Sequences JX090479.1 and KM360715.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841377.1 and HE574588.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841377.1 and JN891348.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841377.1 and HE574589.1 are exactly identical


IMPORTANT WARNING: Sequences KJ841377.1 and AY395546.1 are exactly identical


IMPORTANT WARNING: Sequences KM360660.1 and HF572808.1 are exactly identical


IMPORTANT WARNING: Sequences KM360660.1 and AJ316582.1 are exactly identical


IMPORTANT WARNING: Sequences JN891514.1 and HM850354.1 are exactly identical


IMPORTANT WARNING: Sequences JN891514.1 and JN892746.1 are exactly identical


IMPORTANT WARNING: Sequences AJ542434.1 and JN891782.1 are exactly identical


IMPORTANT WARNING: Sequences GU191334.1 and KJ841301.1 are exactly identical


IMPORTANT WARNING: Sequences GU191334.1 and DQ463101.1 are exactly identical


IMPORTANT WARNING: Sequences JN891672.1 and JN893002.1 are exactly identical


IMPORTANT WARNING: Sequences JN893775.1 and KM360632.1 are exactly identical


IMPORTANT WARNING: Sequences JQ512525.1 and HM024272.1 are exactly identical


IMPORTANT WARNING: Sequences JQ512525.1 and L12570.2 are exactly identical


IMPORTANT WARNING: Sequences JN893231.1 and JN965450.1 are exactly identical


IMPORTANT WARNING: Sequences JN891515.1 and HM850343.1 are exactly identical


IMPORTANT WARNING: Sequences JN891515.1 and JN892046.1 are exactly identical


IMPORTANT WARNING: Sequences HM596073.1 and HF565315.1 are exactly identical


IMPORTANT WARNING: Sequences FJ395603.1 and KU724140.1 are exactly identical


IMPORTANT WARNING: Sequences KM360901.1 and EU262889.2 are exactly identical


IMPORTANT WARNING: Sequences JN892733.1 and JN892850.1 are exactly identical


IMPORTANT WARNING: Sequences JN890902.1 and JN890644.1 are exactly identical


IMPORTANT WARNING: Sequences JN891486.1 and HE963374.1 are exactly identical


IMPORTANT WARNING: Sequences JN891486.1 and KJ512643.1 are exactly identical


IMPORTANT WARNING: Sequences EF646925.1 and JN890613.1 are exactly identical


IMPORTANT WARNING: Sequences KP769034.1 and HQ235393.1 are exactly identical


IMPORTANT WARNING: Sequences KP769034.1 and HE963614.1 are exactly identical


IMPORTANT WARNING: Sequences HQ235559.1 and KM360942.1 are exactly identical


IMPORTANT WARNING: Sequences HQ235559.1 and KX163006.1 are exactly identical


IMPORTANT WARNING: Sequences KF724321.1 and KJ468102.1 are exactly identical


IMPORTANT WARNING: Sequences KF724321.1 and U74220.1 are exactly identical


IMPORTANT WARNING: Sequences Z83142.1 and KT178138.1 are exactly identical


IMPORTANT WARNING: Sequences AJ746296.1 and JN893337.1 are exactly identical


IMPORTANT WARNING: Sequences AJ746296.1 and HM850196.1 are exactly identical


IMPORTANT WARNING: Sequences HM850384.1 and JN891833.1 are exactly identical


IMPORTANT WARNING: Sequences KM360963.1 and AY270125.1 are exactly identical


IMPORTANT WARNING: Sequences KM360963.1 and KJ629116.1 are exactly identical


IMPORTANT WARNING: Sequences EU998740.4 and AY497230.1 are exactly identical


IMPORTANT WARNING: Sequences JN891584.1 and AY034235.1 are exactly identical


IMPORTANT WARNING: Sequences JN891431.1 and HQ590036.1 are exactly identical


IMPORTANT WARNING: Sequences JN891431.1 and AM235666.1 are exactly identical


IMPORTANT WARNING: Sequences AF500344.1 and KM586477.1 are exactly identical


IMPORTANT WARNING: Sequences HM849813.1 and KJ204304.1 are exactly identical


IMPORTANT WARNING: Sequences HM849813.1 and AY395530.1 are exactly identical


IMPORTANT WARNING: Sequences AF020327.1 and KJ841521.1 are exactly identical


IMPORTANT WARNING: Sequences JN893371.1 and HQ590147.1 are exactly identical


IMPORTANT WARNING: Sequences HM850272.1 and JN893631.1 are exactly identical


IMPORTANT WARNING: Sequences KT178127.1 and Z37421.1 are exactly identical


IMPORTANT WARNING: Sequences AB008781.1 and FM883633.1 are exactly identical


IMPORTANT WARNING: Sequences AY216619.1 and JN893823.1 are exactly identical


IMPORTANT WARNING: Sequences KX015761.1 and EU883476.1 are exactly identical


IMPORTANT WARNING: Sequences KM360703.1 and HM849879.1 are exactly identical


IMPORTANT WARNING: Sequences KM360727.1 and KT178116.1 are exactly identical


IMPORTANT WARNING: Sequences KM360727.1 and JN893157.1 are exactly identical


IMPORTANT WARNING: Sequences JQ349075.1 and AY138414.1 are exactly identical


IMPORTANT WARNING: Sequences FR865128.1 and U50229.1 are exactly identical


IMPORTANT WARNING: Sequences FR865128.1 and KM360934.1 are exactly identical


IMPORTANT WARNING: Sequences AB004888.1 and DQ859167.1 are exactly identical


IMPORTANT WARNING: Sequences AB004888.1 and JQ310743.1 are exactly identical


IMPORTANT WARNING: Sequences HQ902786.1 and JN891280.1 are exactly identical


IMPORTANT WARNING: Sequences DQ006077.1 and KM586489.1 are exactly identical


IMPORTANT WARNING: Sequences JN891340.1 and KJ746211.1 are exactly identical


IMPORTANT WARNING: Sequences AF206785.1 and KF418920.1 are exactly identical


IMPORTANT WARNING: Sequences HM998606.1 and HM029370.1 are exactly identical


IMPORTANT WARNING: Sequences HM998606.1 and KJ806203.1 are exactly identical


IMPORTANT WARNING: Sequences AB029848.1 and HM849958.1 are exactly identical


IMPORTANT WARNING: Sequences KF724303.1 and KX421115.1 are exactly identical


IMPORTANT WARNING: Sequences JN890840.1 and HM850180.1 are exactly identical


IMPORTANT WARNING: Sequences JN890840.1 and HE963321.1 are exactly identical


IMPORTANT WARNING: Sequences JN890840.1 and HM849781.1 are exactly identical


IMPORTANT WARNING: Sequences EU676939.1 and FN870404.1 are exactly identical


IMPORTANT WARNING: Sequences JN854170.1 and KM360944.1 are exactly identical


IMPORTANT WARNING: Sequences AJ286560.1 and JX032757.1 are exactly identical


IMPORTANT WARNING: Sequences AJ286560.1 and JN890810.1 are exactly identical


IMPORTANT WARNING: Sequences JF317488.1 and JF941931.1 are exactly identical


IMPORTANT WARNING: Sequences JF317488.1 and JF941930.1 are exactly identical


IMPORTANT WARNING: Sequences DQ006070.1 and KJ841670.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746221.1 and JN891849.1 are exactly identical


IMPORTANT WARNING: Sequences HM849936.1 and AM999805.1 are exactly identical


IMPORTANT WARNING: Sequences L14706.2 and HM850429.1 are exactly identical


IMPORTANT WARNING: Sequences KM360633.1 and HM849908.1 are exactly identical


IMPORTANT WARNING: Sequences KM360642.1 and HM849784.1 are exactly identical


IMPORTANT WARNING: Sequences KM360642.1 and JN892873.1 are exactly identical


IMPORTANT WARNING: Sequences JN891074.1 and KM360892.1 are exactly identical


IMPORTANT WARNING: Sequences JN891074.1 and HE574581.1 are exactly identical


IMPORTANT WARNING: Sequences JN891074.1 and KJ841110.1 are exactly identical


IMPORTANT WARNING: Sequences JF941275.1 and JN891249.1 are exactly identical


IMPORTANT WARNING: Sequences JF943517.1 and FM883640.1 are exactly identical


IMPORTANT WARNING: Sequences JN891170.1 and JX505519.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590177.1 and HM850160.1 are exactly identical


IMPORTANT WARNING: Sequences KC584902.1 and KC484305.1 are exactly identical


IMPORTANT WARNING: Sequences KC584902.1 and JN890555.1 are exactly identical


IMPORTANT WARNING: Sequences JX068710.1 and JX068709.1 are exactly identical


IMPORTANT WARNING: Sequences JX068710.1 and JX068712.1 are exactly identical


IMPORTANT WARNING: Sequences JX068710.1 and JX068708.1 are exactly identical


IMPORTANT WARNING: Sequences JN893824.1 and JN891941.1 are exactly identical


IMPORTANT WARNING: Sequences KU551271.1 and KM360858.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590327.1 and KM361032.1 are exactly identical


IMPORTANT WARNING: Sequences HQ590327.1 and HE574597.1 are exactly identical


IMPORTANT WARNING: Sequences JX664057.1 and KM360855.1 are exactly identical


IMPORTANT WARNING: Sequences KJ746322.1 and JN891769.1 are exactly identical


IMPORTANT WARNING: Sequences KF022476.1 and KJ667664.1 are exactly identical


IMPORTANT WARNING: Sequences JN965451.1 and JN893736.1 are exactly identical


IMPORTANT WARNING: Sequences KP056841.1 and KU833246.1 are exactly identical


IMPORTANT WARNING: Sequences KP056841.1 and KP056842.1 are exactly identical


IMPORTANT WARNING: Sequences KP056841.1 and GQ436380.1 are exactly identical


IMPORTANT WARNING: Sequences HE963499.1 and JN892286.1 are exactly identical


IMPORTANT WARNING: Sequences KJ082090.1 and LN908012.1 are exactly identical


IMPORTANT WARNING: Sequences JN890821.1 and JN890820.1 are exactly identical

IMPORTANT WARNING
Found 1336 sequences that are exactly identical to other sequences in the alignment.
Normally they should be excluded from the analysis.

Just in case you might need it, an alignment file with 
sequence duplicates removed is printed to file rbcL_eyorks_clean_alignment.phylip.reduced

This is the RAxML Master Pthread

This is RAxML Worker Pthread Number: 1

This is RAxML Worker Pthread Number: 2

This is RAxML Worker Pthread Number: 3

This is RAxML Worker Pthread Number: 4


This is RAxML version 7.2.6 released by Alexandros Stamatakis in February 2010.

With greatly appreciated code contributions by:
Andre Aberer (TUM)
Simon Berger (TUM)
John Cazes (TACC)
Michael Ott (TUM)
Nick Pattengale (UNM)
Wayne Pfeiffer (SDSC)


Alignment has 166 distinct alignment patterns

Proportion of gaps and completely undetermined characters in this alignment: 0.44%

RAxML Model Optimization up to an accuracy of 0.100000 log likelihood units

Using 1 distinct models/data partitions with joint branch length optimization


All free model parameters will be estimated by RAxML
GAMMA model of rate heteorgeneity, ML estimate of alpha-parameter

GAMMA Model parameters will be estimated up to an accuracy of 0.1000000000 Log Likelihood units

Partition: 0
Alignment Patterns: 166
Name: No Name Provided
DataType: DNA
Substitution Matrix: GTR




RAxML was called as follows:

raxmlHPC-PTHREADS -f e -T 5 -t RAxML_bestTree.rbcL@mafftLinsi-SATIVA@gappyout0 -m GTRGAMMA -s rbcL_eyorks_clean_alignment.phylip -n test 


12.858189 -12995.772619


Overall Time for Tree Evaluation 12.861440
Final GAMMA  likelihood: -12995.772619
Number of free parameters for AIC-TEST(BR-LEN): 4624
Number of free parameters for AIC-TEST(NO-BR-LEN): 9


Model Parameters of Partition 0, Name: No Name Provided, Type of Data: DNA
alpha: 0.531976
Tree-Length: 28.606605
rate A &lt;-&gt; C: 0.813701
rate A &lt;-&gt; G: 3.394765
rate A &lt;-&gt; T: 0.438605
rate C &lt;-&gt; G: 0.481482
rate C &lt;-&gt; T: 3.780905
rate G &lt;-&gt; T: 1.000000

freq pi(A): 0.267957
freq pi(C): 0.218970
freq pi(G): 0.218295
freq pi(T): 0.294778

Final tree written to:                 /home/working/pplacer/RAxML_result.test
Execution Log File written to:         /home/working/pplacer/RAxML_log.test
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Now build the reference package</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[136]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash
taxit create <span class="se">\</span>
-l rbcL <span class="se">\</span>
-P rbcL.refpkg <span class="se">\</span>
--aln-fasta rbcL_eyorks_clean_alignment.fasta <span class="se">\</span>
--tree-stats RAxML_info.test <span class="se">\</span>
--tree-file RAxML_bestTree.rbcL@mafftLinsi-SATIVA@gappyout0 <span class="se">\</span>
--profile rbcL_ref.hmm <span class="se">\</span>
--seq-info seq_info.csv <span class="se">\</span>
--taxonomy taxa.csv
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>rerooting at below_below_below_below_phylum
root found at node 2055
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Finally, pplacer needs to run an initial BLAST search so we need to copy the relevant files over from the final BLAST.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[144]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../</span><span class="n">AgainstRefDB</span><span class="o">/</span><span class="n">Curat1</span><span class="o">/</span><span class="n">FINAL</span><span class="o">/</span><span class="n">Sativa</span><span class="o">/</span><span class="n">eyorks_flora_final</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[145]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span><span class="nb">echo</span> <span class="s2">&quot;eyorks_flora_final.gb\tgb&quot;</span> &gt; REFmap.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>We are now ready to run assignment with phylogenetic placement!</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[139]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>postSATIVA_cleaned.gb
RAxML_bestTree.rbcL@mafftLinsi-SATIVA@gappyout0
RAxML_info.test
RAxML_log.test
RAxML_result.test
rbcL_eyorks_clean_alignment.fasta
rbcL_eyorks_clean_alignment.phylip
rbcL_eyorks_clean_alignment.phylip.reduced
rbcL_ref.hmm
<span class="ansibold ansiblue">rbcL.refpkg</span>/
seq_info.csv
taxa.csv
taxids.txt
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[146]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-B ../AgainstRefDB/Curat1/GLOBAL/MothPollenReflistCurat1-OTU-denovo.biom <span class="se">\</span>
-R REFmap.txt <span class="se">\</span>
--blast --min_ident 0.95 <span class="se">\</span>
--pplace <span class="se">\</span>
--refpkg rbcL.refpkg/ <span class="se">\</span>
-@ callumjmacgregor@gmail.com <span class="se">\</span>
-n 5 -o pplacer &gt; log
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stderr output_text">
<pre>Traceback (most recent call last):
  File &quot;/usr/bin/metaBEAT_global.py&quot;, line 2628, in &lt;module&gt;
    taxonomy_count = assign_taxonomy_pplacer(pplacer_out=pplacer_out_dict, tax_dict=tax_dict, v=args.verbose)
  File &quot;/usr/bin/metaBEAT_global.py&quot;, line 1516, in assign_taxonomy_pplacer
    index = tax_dict[&quot;tax_id&quot;].index(tax_dict[pplacer_out[&apos;hit&apos;][query][0]][1])
KeyError: u&apos;126436&apos;
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<h1 id="Re-run-final,-corrected-BLAST">Re-run final, corrected BLAST<a class="anchor-link" href="#Re-run-final,-corrected-BLAST">&#182;</a></h1>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>At this point I have discovered an error in the Querymap for the original trimming and clustering which led to positives and negatives being overwritten with each successive plate and retained for plate 4 only, so I am going to just rerun this stage and the final BLAST to correct this.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[1]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cd</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">working</span><span class="o">/</span><span class="n">FinalCorrected</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>/home/working/FinalCorrected
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>I have updated the Querymap_global file to mark which plate each control comes from so that their names are different for each instance.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[11]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>grep <span class="s2">&quot;POS&quot;</span> Querymap_global_corrected.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>















</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>I have also copied the raw data across.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[10]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre><span class="ansibold ansigreen">Querymap_global_corrected.txt</span>*  <span class="ansibold ansiblue">raw_data</span>/
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[12]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-Q Querymap_global_corrected.txt <span class="se">\</span>
--trim_qual 30 <span class="se">\</span>
--trim_minlength 90 <span class="se">\</span>
--merge <span class="se">\</span>
--product_length 350 <span class="se">\</span>
--merged_only <span class="se">\</span>
--cluster <span class="se">\</span>
--clust_match 1 <span class="se">\</span>
--clust_cov 5 <span class="se">\</span>
-o MothPollenFinalCorrected <span class="se">\</span>
-@ callumjmacgregor@gmail.com
-n 5 -v &amp;&gt; log
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>
metaBEAT - metaBarcoding and Environmental DNA Analyses tool
version: v.0.97.7-global


Sat Nov  5 10:36:33 2016

/usr/bin/metaBEAT_global.py -Q Querymap_global_corrected.txt --trim_qual 30 --trim_minlength 90 --merge --product_length 350 --merged_only --cluster --clust_match 1 --clust_cov 5 -o MothPollenFinalCorrected -@ callumjmacgregor@gmail.com


metaBEAT may be querying NCBI&apos;s Entrez databases to fetch/verify taxonomic ids. Entrez User requirements state that you need to identify yourself by providing an email address so that NCBI can contact you in case there is a problem.

You have specified: &apos;callumjmacgregor@gmail.com&apos;

taxonomy.db found at /usr/bin/taxonomy.db

Parsing querylist file

Number of samples to process: 335
Sequence input format: defaultdict(&lt;type &apos;int&apos;&gt;, {&apos;fastq&apos;: 336})
Barcodes for demultiplexing provided for 336 samples
Cropping instructions provided for 336 samples


Sat Nov  5 10:36:33 2016


### DEMULTIPLEXING ###

assessing basic characteristics
data comes as paired end - ok
process_shortreads -1 /home/working/FinalCorrected/raw_data/Moth4_S4_L001_R1_001.fastq.gz -2 /home/working/FinalCorrected/raw_data/Moth4_S4_L001_R2_001.fastq.gz -i gzfastq -o . -b barcodes --inline_inline -y fastq
Using Phred+33 encoding for quality scores.
Reads trimmed shorter than 31 nucleotides will be discarded.
Found 1 paired input file(s).
Searching for single and paired-end, inlined barcodes.
Loaded 48 barcodes (8bp / 8bp).
Processing file 1 of 1 [Moth4_S4_L001_R1_001.fastq.gz]
  Reading data from
    /home/working/FinalCorrected/raw_data/Moth4_S4_L001_R1_001.fastq.gz and
    /home/working/FinalCorrected/raw_data/Moth4_S4_L001_R2_001.fastq.gz
  8464282 total reads; -1099580 ambiguous barcodes; +0 recovered; -0 low quality reads; 7364702 retained reads.
    0 trimmed reads; 0 orphaned paired-ends.
Closing files, flushing buffers...
Outputing details to log: &apos;./process_shortreads.log&apos;

8464282 total sequences;
  1099580 ambiguous barcode drops;
  0 low quality read drops;
  0 trimmed reads;
  0 orphaned paired-end reads;
7364702 retained reads.

assessing basic characteristics
data comes as paired end - ok
process_shortreads -1 /home/working/FinalCorrected/raw_data/Moth1_S1_L001_R1_001.fastq.gz -2 /home/working/FinalCorrected/raw_data/Moth1_S1_L001_R2_001.fastq.gz -i gzfastq -o . -b barcodes --inline_inline -y fastq
Using Phred+33 encoding for quality scores.
Reads trimmed shorter than 31 nucleotides will be discarded.
Found 1 paired input file(s).
Searching for single and paired-end, inlined barcodes.
Loaded 96 barcodes (8bp / 8bp).
Processing file 1 of 1 [Moth1_S1_L001_R1_001.fastq.gz]
  Reading data from
    /home/working/FinalCorrected/raw_data/Moth1_S1_L001_R1_001.fastq.gz and
    /home/working/FinalCorrected/raw_data/Moth1_S1_L001_R2_001.fastq.gz
  5976182 total reads; -1259396 ambiguous barcodes; +0 recovered; -0 low quality reads; 4716786 retained reads.
    0 trimmed reads; 0 orphaned paired-ends.
Closing files, flushing buffers...
Outputing details to log: &apos;./process_shortreads.log&apos;

5976182 total sequences;
  1259396 ambiguous barcode drops;
  0 low quality read drops;
  0 trimmed reads;
  0 orphaned paired-end reads;
4716786 retained reads.

assessing basic characteristics
data comes as paired end - ok
process_shortreads -1 /home/working/FinalCorrected/raw_data/Moth2_S2_L001_R1_001.fastq.gz -2 /home/working/FinalCorrected/raw_data/Moth2_S2_L001_R2_001.fastq.gz -i gzfastq -o . -b barcodes --inline_inline -y fastq
Using Phred+33 encoding for quality scores.
Reads trimmed shorter than 31 nucleotides will be discarded.
Found 1 paired input file(s).
Searching for single and paired-end, inlined barcodes.
Loaded 95 barcodes (8bp / 8bp).
Processing file 1 of 1 [Moth2_S2_L001_R1_001.fastq.gz]
  Reading data from
    /home/working/FinalCorrected/raw_data/Moth2_S2_L001_R1_001.fastq.gz and
    /home/working/FinalCorrected/raw_data/Moth2_S2_L001_R2_001.fastq.gz
  4193542 total reads; -826944 ambiguous barcodes; +0 recovered; -0 low quality reads; 3366598 retained reads.
    0 trimmed reads; 0 orphaned paired-ends.
Closing files, flushing buffers...
Outputing details to log: &apos;./process_shortreads.log&apos;

4193542 total sequences;
  826944 ambiguous barcode drops;
  0 low quality read drops;
  0 trimmed reads;
  0 orphaned paired-end reads;
3366598 retained reads.

assessing basic characteristics
data comes as paired end - ok
process_shortreads -1 /home/working/FinalCorrected/raw_data/Moth3_S3_L001_R1_001.fastq.gz -2 /home/working/FinalCorrected/raw_data/Moth3_S3_L001_R2_001.fastq.gz -i gzfastq -o . -b barcodes --inline_inline -y fastq
Using Phred+33 encoding for quality scores.
Reads trimmed shorter than 31 nucleotides will be discarded.
Found 1 paired input file(s).
Searching for single and paired-end, inlined barcodes.
Loaded 96 barcodes (8bp / 8bp).
Processing file 1 of 1 [Moth3_S3_L001_R1_001.fastq.gz]
  Reading data from
    /home/working/FinalCorrected/raw_data/Moth3_S3_L001_R1_001.fastq.gz and
    /home/working/FinalCorrected/raw_data/Moth3_S3_L001_R2_001.fastq.gz
  6757328 total reads; -1226048 ambiguous barcodes; +0 recovered; -0 low quality reads; 5531280 retained reads.
    0 trimmed reads; 0 orphaned paired-ends.
Closing files, flushing buffers...
Outputing details to log: &apos;./process_shortreads.log&apos;

6757328 total sequences;
  1226048 ambiguous barcode drops;
  0 low quality read drops;
  0 trimmed reads;
  0 orphaned paired-end reads;
5531280 retained reads.


##### processing query ID: Lg27 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg27_1.fastq /home/working/FinalCorrected/demultiplexed/Lg27_2.fastq Lg27_forward.paired.fastq.gz Lg27_forward.singletons.fastq.gz Lg27_reverse.paired.fastq.gz Lg27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg27_1.fastq /home/working/FinalCorrected/demultiplexed/Lg27_2.fastq Lg27_forward.paired.fastq.gz Lg27_forward.singletons.fastq.gz Lg27_reverse.paired.fastq.gz Lg27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:38:19 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/Lg27/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26514 Both Surviving: 6740 (25.42%) Forward Only Surviving: 3298 (12.44%) Reverse Only Surviving: 209 (0.79%) Dropped: 16267 (61.35%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: Lg27_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: Lg27_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: Lg27_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: Lg27_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg27_forward.paired.headclipped.fastq.gz Lg27_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg27 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg27_forward.paired.headclipped.fastq.gz
[FLASH]     Lg27_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg27.extendedFrags.fastq.gz
[FLASH]     ./Lg27.notCombined_1.fastq.gz
[FLASH]     ./Lg27.notCombined_2.fastq.gz
[FLASH]     ./Lg27.hist
[FLASH]     ./Lg27.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6740 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6740
[FLASH]     Combined pairs:   5348
[FLASH]     Uncombined pairs: 1392
[FLASH]     Percent combined: 79.35%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.152 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5348 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/Lg27/Lg27_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg27_centroids.fasta --uc Lg27.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5348 sequences and identified 2955 clusters (clustering threshold 1.00) - 72 clusters (representing 2021 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Lg37 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg37_1.fastq /home/working/FinalCorrected/demultiplexed/Lg37_2.fastq Lg37_forward.paired.fastq.gz Lg37_forward.singletons.fastq.gz Lg37_reverse.paired.fastq.gz Lg37_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg37_1.fastq /home/working/FinalCorrected/demultiplexed/Lg37_2.fastq Lg37_forward.paired.fastq.gz Lg37_forward.singletons.fastq.gz Lg37_reverse.paired.fastq.gz Lg37_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:38:32 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/Lg37/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31577 Both Surviving: 247 (0.78%) Forward Only Surviving: 489 (1.55%) Reverse Only Surviving: 197 (0.62%) Dropped: 30644 (97.05%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: Lg37_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: Lg37_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: Lg37_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: Lg37_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg37_forward.paired.headclipped.fastq.gz Lg37_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg37 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg37_forward.paired.headclipped.fastq.gz
[FLASH]     Lg37_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg37.extendedFrags.fastq.gz
[FLASH]     ./Lg37.notCombined_1.fastq.gz
[FLASH]     ./Lg37.notCombined_2.fastq.gz
[FLASH]     ./Lg37.hist
[FLASH]     ./Lg37.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 247 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      247
[FLASH]     Combined pairs:   28
[FLASH]     Uncombined pairs: 219
[FLASH]     Percent combined: 11.34%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

28 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/Lg37/Lg37_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg37_centroids.fasta --uc Lg37.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 28 sequences and identified 26 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Lg45 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg45_1.fastq /home/working/FinalCorrected/demultiplexed/Lg45_2.fastq Lg45_forward.paired.fastq.gz Lg45_forward.singletons.fastq.gz Lg45_reverse.paired.fastq.gz Lg45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg45_1.fastq /home/working/FinalCorrected/demultiplexed/Lg45_2.fastq Lg45_forward.paired.fastq.gz Lg45_forward.singletons.fastq.gz Lg45_reverse.paired.fastq.gz Lg45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:38:34 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/Lg45/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9421 Both Surviving: 2219 (23.55%) Forward Only Surviving: 590 (6.26%) Reverse Only Surviving: 112 (1.19%) Dropped: 6500 (68.99%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: Lg45_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: Lg45_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: Lg45_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: Lg45_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg45_forward.paired.headclipped.fastq.gz Lg45_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg45 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg45_forward.paired.headclipped.fastq.gz
[FLASH]     Lg45_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg45.extendedFrags.fastq.gz
[FLASH]     ./Lg45.notCombined_1.fastq.gz
[FLASH]     ./Lg45.notCombined_2.fastq.gz
[FLASH]     ./Lg45.hist
[FLASH]     ./Lg45.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2219 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2219
[FLASH]     Combined pairs:   2093
[FLASH]     Uncombined pairs: 126
[FLASH]     Percent combined: 94.32%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.075 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2093 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/Lg45/Lg45_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg45_centroids.fasta --uc Lg45.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2093 sequences and identified 1056 clusters (clustering threshold 1.00) - 21 clusters (representing 859 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Lg53 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg53_1.fastq /home/working/FinalCorrected/demultiplexed/Lg53_2.fastq Lg53_forward.paired.fastq.gz Lg53_forward.singletons.fastq.gz Lg53_reverse.paired.fastq.gz Lg53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg53_1.fastq /home/working/FinalCorrected/demultiplexed/Lg53_2.fastq Lg53_forward.paired.fastq.gz Lg53_forward.singletons.fastq.gz Lg53_reverse.paired.fastq.gz Lg53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:38:39 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/Lg53/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22402 Both Surviving: 5675 (25.33%) Forward Only Surviving: 1586 (7.08%) Reverse Only Surviving: 283 (1.26%) Dropped: 14858 (66.32%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: Lg53_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: Lg53_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: Lg53_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: Lg53_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg53_forward.paired.headclipped.fastq.gz Lg53_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg53 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg53_forward.paired.headclipped.fastq.gz
[FLASH]     Lg53_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg53.extendedFrags.fastq.gz
[FLASH]     ./Lg53.notCombined_1.fastq.gz
[FLASH]     ./Lg53.notCombined_2.fastq.gz
[FLASH]     ./Lg53.hist
[FLASH]     ./Lg53.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5675 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5675
[FLASH]     Combined pairs:   877
[FLASH]     Uncombined pairs: 4798
[FLASH]     Percent combined: 15.45%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.031 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

877 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/Lg53/Lg53_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg53_centroids.fasta --uc Lg53.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 877 sequences and identified 582 clusters (clustering threshold 1.00) - 15 clusters (representing 245 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Lg57 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg57_1.fastq /home/working/FinalCorrected/demultiplexed/Lg57_2.fastq Lg57_forward.paired.fastq.gz Lg57_forward.singletons.fastq.gz Lg57_reverse.paired.fastq.gz Lg57_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg57_1.fastq /home/working/FinalCorrected/demultiplexed/Lg57_2.fastq Lg57_forward.paired.fastq.gz Lg57_forward.singletons.fastq.gz Lg57_reverse.paired.fastq.gz Lg57_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:38:44 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/Lg57/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15465 Both Surviving: 4506 (29.14%) Forward Only Surviving: 1245 (8.05%) Reverse Only Surviving: 171 (1.11%) Dropped: 9543 (61.71%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: Lg57_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: Lg57_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: Lg57_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: Lg57_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg57_forward.paired.headclipped.fastq.gz Lg57_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg57 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg57_forward.paired.headclipped.fastq.gz
[FLASH]     Lg57_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg57.extendedFrags.fastq.gz
[FLASH]     ./Lg57.notCombined_1.fastq.gz
[FLASH]     ./Lg57.notCombined_2.fastq.gz
[FLASH]     ./Lg57.hist
[FLASH]     ./Lg57.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4506 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4506
[FLASH]     Combined pairs:   399
[FLASH]     Uncombined pairs: 4107
[FLASH]     Percent combined: 8.85%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.028 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

399 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/Lg57/Lg57_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg57_centroids.fasta --uc Lg57.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 399 sequences and identified 318 clusters (clustering threshold 1.00) - 7 clusters (representing 56 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Lg65 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg65_1.fastq /home/working/FinalCorrected/demultiplexed/Lg65_2.fastq Lg65_forward.paired.fastq.gz Lg65_forward.singletons.fastq.gz Lg65_reverse.paired.fastq.gz Lg65_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg65_1.fastq /home/working/FinalCorrected/demultiplexed/Lg65_2.fastq Lg65_forward.paired.fastq.gz Lg65_forward.singletons.fastq.gz Lg65_reverse.paired.fastq.gz Lg65_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:38:48 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/Lg65/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37917 Both Surviving: 804 (2.12%) Forward Only Surviving: 550 (1.45%) Reverse Only Surviving: 304 (0.80%) Dropped: 36259 (95.63%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: Lg65_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: Lg65_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: Lg65_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: Lg65_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg65_forward.paired.headclipped.fastq.gz Lg65_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg65 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg65_forward.paired.headclipped.fastq.gz
[FLASH]     Lg65_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg65.extendedFrags.fastq.gz
[FLASH]     ./Lg65.notCombined_1.fastq.gz
[FLASH]     ./Lg65.notCombined_2.fastq.gz
[FLASH]     ./Lg65.hist
[FLASH]     ./Lg65.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 804 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      804
[FLASH]     Combined pairs:   84
[FLASH]     Uncombined pairs: 720
[FLASH]     Percent combined: 10.45%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.007 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

84 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/Lg65/Lg65_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg65_centroids.fasta --uc Lg65.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 84 sequences and identified 61 clusters (clustering threshold 1.00) - 1 clusters (representing 22 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Lg9 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg9_1.fastq /home/working/FinalCorrected/demultiplexed/Lg9_2.fastq Lg9_forward.paired.fastq.gz Lg9_forward.singletons.fastq.gz Lg9_reverse.paired.fastq.gz Lg9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Lg9_1.fastq /home/working/FinalCorrected/demultiplexed/Lg9_2.fastq Lg9_forward.paired.fastq.gz Lg9_forward.singletons.fastq.gz Lg9_reverse.paired.fastq.gz Lg9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:38:52 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/Lg9/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 19281 Both Surviving: 90 (0.47%) Forward Only Surviving: 335 (1.74%) Reverse Only Surviving: 199 (1.03%) Dropped: 18657 (96.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: Lg9_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: Lg9_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: Lg9_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: Lg9_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Lg9_forward.paired.headclipped.fastq.gz Lg9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg9 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Lg9_forward.paired.headclipped.fastq.gz
[FLASH]     Lg9_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Lg9.extendedFrags.fastq.gz
[FLASH]     ./Lg9.notCombined_1.fastq.gz
[FLASH]     ./Lg9.notCombined_2.fastq.gz
[FLASH]     ./Lg9.hist
[FLASH]     ./Lg9.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 90 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      90
[FLASH]     Combined pairs:   67
[FLASH]     Uncombined pairs: 23
[FLASH]     Percent combined: 74.44%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

67 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/Lg9/Lg9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg9_centroids.fasta --uc Lg9.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 67 sequences and identified 52 clusters (clustering threshold 1.00) - 2 clusters (representing 13 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: NEG_1 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/NEG_1_1.fastq /home/working/FinalCorrected/demultiplexed/NEG_1_2.fastq NEG_1_forward.paired.fastq.gz NEG_1_forward.singletons.fastq.gz NEG_1_reverse.paired.fastq.gz NEG_1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/NEG_1_1.fastq /home/working/FinalCorrected/demultiplexed/NEG_1_2.fastq NEG_1_forward.paired.fastq.gz NEG_1_forward.singletons.fastq.gz NEG_1_reverse.paired.fastq.gz NEG_1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:38:54 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/NEG_1/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 2923 Both Surviving: 303 (10.37%) Forward Only Surviving: 120 (4.11%) Reverse Only Surviving: 63 (2.16%) Dropped: 2437 (83.37%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: NEG_1_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: NEG_1_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: NEG_1_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: NEG_1_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash NEG_1_forward.paired.headclipped.fastq.gz NEG_1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o NEG_1 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     NEG_1_forward.paired.headclipped.fastq.gz
[FLASH]     NEG_1_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./NEG_1.extendedFrags.fastq.gz
[FLASH]     ./NEG_1.notCombined_1.fastq.gz
[FLASH]     ./NEG_1.notCombined_2.fastq.gz
[FLASH]     ./NEG_1.hist
[FLASH]     ./NEG_1.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 303 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      303
[FLASH]     Combined pairs:   30
[FLASH]     Uncombined pairs: 273
[FLASH]     Percent combined: 9.90%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

30 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/NEG_1/NEG_1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids NEG_1_centroids.fasta --uc NEG_1.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 30 sequences and identified 29 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: NEG_2 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/NEG_2_1.fastq /home/working/FinalCorrected/demultiplexed/NEG_2_2.fastq NEG_2_forward.paired.fastq.gz NEG_2_forward.singletons.fastq.gz NEG_2_reverse.paired.fastq.gz NEG_2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/NEG_2_1.fastq /home/working/FinalCorrected/demultiplexed/NEG_2_2.fastq NEG_2_forward.paired.fastq.gz NEG_2_forward.singletons.fastq.gz NEG_2_reverse.paired.fastq.gz NEG_2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:38:54 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/NEG_2/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 3505 Both Surviving: 402 (11.47%) Forward Only Surviving: 167 (4.76%) Reverse Only Surviving: 54 (1.54%) Dropped: 2882 (82.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: NEG_2_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: NEG_2_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: NEG_2_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: NEG_2_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash NEG_2_forward.paired.headclipped.fastq.gz NEG_2_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o NEG_2 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     NEG_2_forward.paired.headclipped.fastq.gz
[FLASH]     NEG_2_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./NEG_2.extendedFrags.fastq.gz
[FLASH]     ./NEG_2.notCombined_1.fastq.gz
[FLASH]     ./NEG_2.notCombined_2.fastq.gz
[FLASH]     ./NEG_2.hist
[FLASH]     ./NEG_2.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 402 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      402
[FLASH]     Combined pairs:   139
[FLASH]     Uncombined pairs: 263
[FLASH]     Percent combined: 34.58%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.006 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

139 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/NEG_2/NEG_2_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids NEG_2_centroids.fasta --uc NEG_2.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 139 sequences and identified 115 clusters (clustering threshold 1.00) - 2 clusters (representing 15 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: NEG_3 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/NEG_3_1.fastq /home/working/FinalCorrected/demultiplexed/NEG_3_2.fastq NEG_3_forward.paired.fastq.gz NEG_3_forward.singletons.fastq.gz NEG_3_reverse.paired.fastq.gz NEG_3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/NEG_3_1.fastq /home/working/FinalCorrected/demultiplexed/NEG_3_2.fastq NEG_3_forward.paired.fastq.gz NEG_3_forward.singletons.fastq.gz NEG_3_reverse.paired.fastq.gz NEG_3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:38:56 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/NEG_3/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 1900 Both Surviving: 318 (16.74%) Forward Only Surviving: 104 (5.47%) Reverse Only Surviving: 52 (2.74%) Dropped: 1426 (75.05%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: NEG_3_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: NEG_3_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: NEG_3_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: NEG_3_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash NEG_3_forward.paired.headclipped.fastq.gz NEG_3_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o NEG_3 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     NEG_3_forward.paired.headclipped.fastq.gz
[FLASH]     NEG_3_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./NEG_3.extendedFrags.fastq.gz
[FLASH]     ./NEG_3.notCombined_1.fastq.gz
[FLASH]     ./NEG_3.notCombined_2.fastq.gz
[FLASH]     ./NEG_3.hist
[FLASH]     ./NEG_3.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 318 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      318
[FLASH]     Combined pairs:   63
[FLASH]     Uncombined pairs: 255
[FLASH]     Percent combined: 19.81%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

63 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/NEG_3/NEG_3_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids NEG_3_centroids.fasta --uc NEG_3.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 63 sequences and identified 59 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: NEG_4 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/NEG_4_1.fastq /home/working/FinalCorrected/demultiplexed/NEG_4_2.fastq NEG_4_forward.paired.fastq.gz NEG_4_forward.singletons.fastq.gz NEG_4_reverse.paired.fastq.gz NEG_4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/NEG_4_1.fastq /home/working/FinalCorrected/demultiplexed/NEG_4_2.fastq NEG_4_forward.paired.fastq.gz NEG_4_forward.singletons.fastq.gz NEG_4_reverse.paired.fastq.gz NEG_4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:38:58 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/NEG_4/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23603 Both Surviving: 6446 (27.31%) Forward Only Surviving: 1967 (8.33%) Reverse Only Surviving: 669 (2.83%) Dropped: 14521 (61.52%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: NEG_4_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: NEG_4_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: NEG_4_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: NEG_4_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash NEG_4_forward.paired.headclipped.fastq.gz NEG_4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o NEG_4 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     NEG_4_forward.paired.headclipped.fastq.gz
[FLASH]     NEG_4_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./NEG_4.extendedFrags.fastq.gz
[FLASH]     ./NEG_4.notCombined_1.fastq.gz
[FLASH]     ./NEG_4.notCombined_2.fastq.gz
[FLASH]     ./NEG_4.hist
[FLASH]     ./NEG_4.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6446 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6446
[FLASH]     Combined pairs:   864
[FLASH]     Uncombined pairs: 5582
[FLASH]     Percent combined: 13.40%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.039 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

864 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/NEG_4/NEG_4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids NEG_4_centroids.fasta --uc NEG_4.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 864 sequences and identified 623 clusters (clustering threshold 1.00) - 11 clusters (representing 179 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: O84 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/O84_1.fastq /home/working/FinalCorrected/demultiplexed/O84_2.fastq O84_forward.paired.fastq.gz O84_forward.singletons.fastq.gz O84_reverse.paired.fastq.gz O84_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/O84_1.fastq /home/working/FinalCorrected/demultiplexed/O84_2.fastq O84_forward.paired.fastq.gz O84_forward.singletons.fastq.gz O84_reverse.paired.fastq.gz O84_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:39:05 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/O84/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 36398 Both Surviving: 15690 (43.11%) Forward Only Surviving: 5948 (16.34%) Reverse Only Surviving: 326 (0.90%) Dropped: 14434 (39.66%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: O84_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: O84_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: O84_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: O84_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash O84_forward.paired.headclipped.fastq.gz O84_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o O84 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     O84_forward.paired.headclipped.fastq.gz
[FLASH]     O84_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./O84.extendedFrags.fastq.gz
[FLASH]     ./O84.notCombined_1.fastq.gz
[FLASH]     ./O84.notCombined_2.fastq.gz
[FLASH]     ./O84.hist
[FLASH]     ./O84.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 15690 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      15690
[FLASH]     Combined pairs:   13492
[FLASH]     Uncombined pairs: 2198
[FLASH]     Percent combined: 85.99%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.347 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

13492 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/O84/O84_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids O84_centroids.fasta --uc O84.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 13492 sequences and identified 4050 clusters (clustering threshold 1.00) - 312 clusters (representing 9371 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: O96 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/O96_1.fastq /home/working/FinalCorrected/demultiplexed/O96_2.fastq O96_forward.paired.fastq.gz O96_forward.singletons.fastq.gz O96_reverse.paired.fastq.gz O96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/O96_1.fastq /home/working/FinalCorrected/demultiplexed/O96_2.fastq O96_forward.paired.fastq.gz O96_forward.singletons.fastq.gz O96_reverse.paired.fastq.gz O96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:39:29 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/O96/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14802 Both Surviving: 5346 (36.12%) Forward Only Surviving: 1579 (10.67%) Reverse Only Surviving: 210 (1.42%) Dropped: 7667 (51.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: O96_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: O96_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: O96_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: O96_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash O96_forward.paired.headclipped.fastq.gz O96_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o O96 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     O96_forward.paired.headclipped.fastq.gz
[FLASH]     O96_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./O96.extendedFrags.fastq.gz
[FLASH]     ./O96.notCombined_1.fastq.gz
[FLASH]     ./O96.notCombined_2.fastq.gz
[FLASH]     ./O96.hist
[FLASH]     ./O96.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5346 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5346
[FLASH]     Combined pairs:   1914
[FLASH]     Uncombined pairs: 3432
[FLASH]     Percent combined: 35.80%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.058 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1914 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/O96/O96_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids O96_centroids.fasta --uc O96.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1914 sequences and identified 1228 clusters (clustering threshold 1.00) - 28 clusters (representing 583 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: P18 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/P18_1.fastq /home/working/FinalCorrected/demultiplexed/P18_2.fastq P18_forward.paired.fastq.gz P18_forward.singletons.fastq.gz P18_reverse.paired.fastq.gz P18_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/P18_1.fastq /home/working/FinalCorrected/demultiplexed/P18_2.fastq P18_forward.paired.fastq.gz P18_forward.singletons.fastq.gz P18_reverse.paired.fastq.gz P18_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:39:36 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/P18/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25495 Both Surviving: 5531 (21.69%) Forward Only Surviving: 1954 (7.66%) Reverse Only Surviving: 326 (1.28%) Dropped: 17684 (69.36%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: P18_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: P18_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: P18_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: P18_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash P18_forward.paired.headclipped.fastq.gz P18_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P18 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     P18_forward.paired.headclipped.fastq.gz
[FLASH]     P18_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./P18.extendedFrags.fastq.gz
[FLASH]     ./P18.notCombined_1.fastq.gz
[FLASH]     ./P18.notCombined_2.fastq.gz
[FLASH]     ./P18.hist
[FLASH]     ./P18.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5531 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5531
[FLASH]     Combined pairs:   3010
[FLASH]     Uncombined pairs: 2521
[FLASH]     Percent combined: 54.42%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.084 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3010 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/P18/P18_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P18_centroids.fasta --uc P18.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3010 sequences and identified 1116 clusters (clustering threshold 1.00) - 67 clusters (representing 1723 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: P54 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/P54_1.fastq /home/working/FinalCorrected/demultiplexed/P54_2.fastq P54_forward.paired.fastq.gz P54_forward.singletons.fastq.gz P54_reverse.paired.fastq.gz P54_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/P54_1.fastq /home/working/FinalCorrected/demultiplexed/P54_2.fastq P54_forward.paired.fastq.gz P54_forward.singletons.fastq.gz P54_reverse.paired.fastq.gz P54_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:39:46 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/P54/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 50485 Both Surviving: 15012 (29.74%) Forward Only Surviving: 3807 (7.54%) Reverse Only Surviving: 1539 (3.05%) Dropped: 30127 (59.68%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: P54_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: P54_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: P54_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: P54_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash P54_forward.paired.headclipped.fastq.gz P54_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P54 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     P54_forward.paired.headclipped.fastq.gz
[FLASH]     P54_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./P54.extendedFrags.fastq.gz
[FLASH]     ./P54.notCombined_1.fastq.gz
[FLASH]     ./P54.notCombined_2.fastq.gz
[FLASH]     ./P54.hist
[FLASH]     ./P54.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 15012 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      15012
[FLASH]     Combined pairs:   5359
[FLASH]     Uncombined pairs: 9653
[FLASH]     Percent combined: 35.70%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.167 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5359 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/P54/P54_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P54_centroids.fasta --uc P54.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5359 sequences and identified 2153 clusters (clustering threshold 1.00) - 86 clusters (representing 2811 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: P59 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/P59_1.fastq /home/working/FinalCorrected/demultiplexed/P59_2.fastq P59_forward.paired.fastq.gz P59_forward.singletons.fastq.gz P59_reverse.paired.fastq.gz P59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/P59_1.fastq /home/working/FinalCorrected/demultiplexed/P59_2.fastq P59_forward.paired.fastq.gz P59_forward.singletons.fastq.gz P59_reverse.paired.fastq.gz P59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:40:01 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/P59/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 10337 Both Surviving: 1710 (16.54%) Forward Only Surviving: 420 (4.06%) Reverse Only Surviving: 272 (2.63%) Dropped: 7935 (76.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: P59_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: P59_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: P59_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: P59_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash P59_forward.paired.headclipped.fastq.gz P59_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P59 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     P59_forward.paired.headclipped.fastq.gz
[FLASH]     P59_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./P59.extendedFrags.fastq.gz
[FLASH]     ./P59.notCombined_1.fastq.gz
[FLASH]     ./P59.notCombined_2.fastq.gz
[FLASH]     ./P59.hist
[FLASH]     ./P59.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1710 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1710
[FLASH]     Combined pairs:   29
[FLASH]     Uncombined pairs: 1681
[FLASH]     Percent combined: 1.70%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.013 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

29 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/P59/P59_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P59_centroids.fasta --uc P59.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 29 sequences and identified 28 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: P60 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/P60_1.fastq /home/working/FinalCorrected/demultiplexed/P60_2.fastq P60_forward.paired.fastq.gz P60_forward.singletons.fastq.gz P60_reverse.paired.fastq.gz P60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/P60_1.fastq /home/working/FinalCorrected/demultiplexed/P60_2.fastq P60_forward.paired.fastq.gz P60_forward.singletons.fastq.gz P60_reverse.paired.fastq.gz P60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:40:05 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/P60/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26955 Both Surviving: 10127 (37.57%) Forward Only Surviving: 3468 (12.87%) Reverse Only Surviving: 309 (1.15%) Dropped: 13051 (48.42%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: P60_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: P60_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: P60_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: P60_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash P60_forward.paired.headclipped.fastq.gz P60_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P60 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     P60_forward.paired.headclipped.fastq.gz
[FLASH]     P60_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./P60.extendedFrags.fastq.gz
[FLASH]     ./P60.notCombined_1.fastq.gz
[FLASH]     ./P60.notCombined_2.fastq.gz
[FLASH]     ./P60.hist
[FLASH]     ./P60.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10127 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10127
[FLASH]     Combined pairs:   5608
[FLASH]     Uncombined pairs: 4519
[FLASH]     Percent combined: 55.38%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.156 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5608 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/P60/P60_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P60_centroids.fasta --uc P60.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5608 sequences and identified 3054 clusters (clustering threshold 1.00) - 92 clusters (representing 2206 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: P73 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/P73_1.fastq /home/working/FinalCorrected/demultiplexed/P73_2.fastq P73_forward.paired.fastq.gz P73_forward.singletons.fastq.gz P73_reverse.paired.fastq.gz P73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/P73_1.fastq /home/working/FinalCorrected/demultiplexed/P73_2.fastq P73_forward.paired.fastq.gz P73_forward.singletons.fastq.gz P73_reverse.paired.fastq.gz P73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:40:18 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/P73/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: P73_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: P73_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: P73_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: P73_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash P73_forward.paired.headclipped.fastq.gz P73_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P73 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     P73_forward.paired.headclipped.fastq.gz
[FLASH]     P73_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./P73.extendedFrags.fastq.gz
[FLASH]     ./P73.notCombined_1.fastq.gz
[FLASH]     ./P73.notCombined_2.fastq.gz
[FLASH]     ./P73.hist
[FLASH]     ./P73.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/P73/P73_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P73_centroids.fasta --uc P73.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: P98 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/P98_1.fastq /home/working/FinalCorrected/demultiplexed/P98_2.fastq P98_forward.paired.fastq.gz P98_forward.singletons.fastq.gz P98_reverse.paired.fastq.gz P98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/P98_1.fastq /home/working/FinalCorrected/demultiplexed/P98_2.fastq P98_forward.paired.fastq.gz P98_forward.singletons.fastq.gz P98_reverse.paired.fastq.gz P98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:40:18 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/P98/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 4 Both Surviving: 1 (25.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 3 (75.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: P98_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: P98_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: P98_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: P98_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash P98_forward.paired.headclipped.fastq.gz P98_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P98 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     P98_forward.paired.headclipped.fastq.gz
[FLASH]     P98_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./P98.extendedFrags.fastq.gz
[FLASH]     ./P98.notCombined_1.fastq.gz
[FLASH]     ./P98.notCombined_2.fastq.gz
[FLASH]     ./P98.hist
[FLASH]     ./P98.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1
[FLASH]     Combined pairs:   1
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 100.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/P98/P98_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P98_centroids.fasta --uc P98.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1 sequences and identified 1 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS1_1 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS1_1_1.fastq /home/working/FinalCorrected/demultiplexed/POS1_1_2.fastq POS1_1_forward.paired.fastq.gz POS1_1_forward.singletons.fastq.gz POS1_1_reverse.paired.fastq.gz POS1_1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS1_1_1.fastq /home/working/FinalCorrected/demultiplexed/POS1_1_2.fastq POS1_1_forward.paired.fastq.gz POS1_1_forward.singletons.fastq.gz POS1_1_reverse.paired.fastq.gz POS1_1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:40:22 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS1_1/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 57653 Both Surviving: 13947 (24.19%) Forward Only Surviving: 4619 (8.01%) Reverse Only Surviving: 745 (1.29%) Dropped: 38342 (66.50%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: POS1_1_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS1_1_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: POS1_1_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: POS1_1_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS1_1_forward.paired.headclipped.fastq.gz POS1_1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS1_1 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS1_1_forward.paired.headclipped.fastq.gz
[FLASH]     POS1_1_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS1_1.extendedFrags.fastq.gz
[FLASH]     ./POS1_1.notCombined_1.fastq.gz
[FLASH]     ./POS1_1.notCombined_2.fastq.gz
[FLASH]     ./POS1_1.hist
[FLASH]     ./POS1_1.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 13947 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      13947
[FLASH]     Combined pairs:   6759
[FLASH]     Uncombined pairs: 7188
[FLASH]     Percent combined: 48.46%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.195 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6759 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS1_1/POS1_1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS1_1_centroids.fasta --uc POS1_1.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6759 sequences and identified 3918 clusters (clustering threshold 1.00) - 97 clusters (representing 2354 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS1_2 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS1_2_1.fastq /home/working/FinalCorrected/demultiplexed/POS1_2_2.fastq POS1_2_forward.paired.fastq.gz POS1_2_forward.singletons.fastq.gz POS1_2_reverse.paired.fastq.gz POS1_2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS1_2_1.fastq /home/working/FinalCorrected/demultiplexed/POS1_2_2.fastq POS1_2_forward.paired.fastq.gz POS1_2_forward.singletons.fastq.gz POS1_2_reverse.paired.fastq.gz POS1_2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:40:40 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS1_2/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28492 Both Surviving: 10615 (37.26%) Forward Only Surviving: 3913 (13.73%) Reverse Only Surviving: 296 (1.04%) Dropped: 13668 (47.97%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: POS1_2_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS1_2_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: POS1_2_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: POS1_2_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS1_2_forward.paired.headclipped.fastq.gz POS1_2_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS1_2 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS1_2_forward.paired.headclipped.fastq.gz
[FLASH]     POS1_2_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS1_2.extendedFrags.fastq.gz
[FLASH]     ./POS1_2.notCombined_1.fastq.gz
[FLASH]     ./POS1_2.notCombined_2.fastq.gz
[FLASH]     ./POS1_2.hist
[FLASH]     ./POS1_2.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10615 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10615
[FLASH]     Combined pairs:   6159
[FLASH]     Uncombined pairs: 4456
[FLASH]     Percent combined: 58.02%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.168 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6159 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS1_2/POS1_2_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS1_2_centroids.fasta --uc POS1_2.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6159 sequences and identified 4116 clusters (clustering threshold 1.00) - 88 clusters (representing 1672 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS1_3 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS1_3_1.fastq /home/working/FinalCorrected/demultiplexed/POS1_3_2.fastq POS1_3_forward.paired.fastq.gz POS1_3_forward.singletons.fastq.gz POS1_3_reverse.paired.fastq.gz POS1_3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS1_3_1.fastq /home/working/FinalCorrected/demultiplexed/POS1_3_2.fastq POS1_3_forward.paired.fastq.gz POS1_3_forward.singletons.fastq.gz POS1_3_reverse.paired.fastq.gz POS1_3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:40:54 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS1_3/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14714 Both Surviving: 1994 (13.55%) Forward Only Surviving: 957 (6.50%) Reverse Only Surviving: 102 (0.69%) Dropped: 11661 (79.25%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: POS1_3_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS1_3_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: POS1_3_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: POS1_3_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS1_3_forward.paired.headclipped.fastq.gz POS1_3_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS1_3 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS1_3_forward.paired.headclipped.fastq.gz
[FLASH]     POS1_3_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS1_3.extendedFrags.fastq.gz
[FLASH]     ./POS1_3.notCombined_1.fastq.gz
[FLASH]     ./POS1_3.notCombined_2.fastq.gz
[FLASH]     ./POS1_3.hist
[FLASH]     ./POS1_3.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1994 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1994
[FLASH]     Combined pairs:   924
[FLASH]     Uncombined pairs: 1070
[FLASH]     Percent combined: 46.34%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.027 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

924 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS1_3/POS1_3_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS1_3_centroids.fasta --uc POS1_3.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 924 sequences and identified 744 clusters (clustering threshold 1.00) - 11 clusters (representing 154 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS1_4 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS1_4_1.fastq /home/working/FinalCorrected/demultiplexed/POS1_4_2.fastq POS1_4_forward.paired.fastq.gz POS1_4_forward.singletons.fastq.gz POS1_4_reverse.paired.fastq.gz POS1_4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS1_4_1.fastq /home/working/FinalCorrected/demultiplexed/POS1_4_2.fastq POS1_4_forward.paired.fastq.gz POS1_4_forward.singletons.fastq.gz POS1_4_reverse.paired.fastq.gz POS1_4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:40:59 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS1_4/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41811 Both Surviving: 4754 (11.37%) Forward Only Surviving: 1957 (4.68%) Reverse Only Surviving: 417 (1.00%) Dropped: 34683 (82.95%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: POS1_4_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS1_4_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: POS1_4_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: POS1_4_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS1_4_forward.paired.headclipped.fastq.gz POS1_4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS1_4 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS1_4_forward.paired.headclipped.fastq.gz
[FLASH]     POS1_4_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS1_4.extendedFrags.fastq.gz
[FLASH]     ./POS1_4.notCombined_1.fastq.gz
[FLASH]     ./POS1_4.notCombined_2.fastq.gz
[FLASH]     ./POS1_4.hist
[FLASH]     ./POS1_4.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4754 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4754
[FLASH]     Combined pairs:   1213
[FLASH]     Uncombined pairs: 3541
[FLASH]     Percent combined: 25.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.038 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1213 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS1_4/POS1_4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS1_4_centroids.fasta --uc POS1_4.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1213 sequences and identified 780 clusters (clustering threshold 1.00) - 17 clusters (representing 359 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS2_1 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS2_1_1.fastq /home/working/FinalCorrected/demultiplexed/POS2_1_2.fastq POS2_1_forward.paired.fastq.gz POS2_1_forward.singletons.fastq.gz POS2_1_reverse.paired.fastq.gz POS2_1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS2_1_1.fastq /home/working/FinalCorrected/demultiplexed/POS2_1_2.fastq POS2_1_forward.paired.fastq.gz POS2_1_forward.singletons.fastq.gz POS2_1_reverse.paired.fastq.gz POS2_1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:41:06 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS2_1/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16222 Both Surviving: 1224 (7.55%) Forward Only Surviving: 363 (2.24%) Reverse Only Surviving: 113 (0.70%) Dropped: 14522 (89.52%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: POS2_1_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: POS2_1_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: POS2_1_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: POS2_1_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS2_1_forward.paired.headclipped.fastq.gz POS2_1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS2_1 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS2_1_forward.paired.headclipped.fastq.gz
[FLASH]     POS2_1_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS2_1.extendedFrags.fastq.gz
[FLASH]     ./POS2_1.notCombined_1.fastq.gz
[FLASH]     ./POS2_1.notCombined_2.fastq.gz
[FLASH]     ./POS2_1.hist
[FLASH]     ./POS2_1.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1224 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1224
[FLASH]     Combined pairs:   587
[FLASH]     Uncombined pairs: 637
[FLASH]     Percent combined: 47.96%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.018 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

587 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS2_1/POS2_1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS2_1_centroids.fasta --uc POS2_1.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 587 sequences and identified 354 clusters (clustering threshold 1.00) - 4 clusters (representing 189 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS2_2 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS2_2_1.fastq /home/working/FinalCorrected/demultiplexed/POS2_2_2.fastq POS2_2_forward.paired.fastq.gz POS2_2_forward.singletons.fastq.gz POS2_2_reverse.paired.fastq.gz POS2_2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS2_2_1.fastq /home/working/FinalCorrected/demultiplexed/POS2_2_2.fastq POS2_2_forward.paired.fastq.gz POS2_2_forward.singletons.fastq.gz POS2_2_reverse.paired.fastq.gz POS2_2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:41:09 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS2_2/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11207 Both Surviving: 1086 (9.69%) Forward Only Surviving: 317 (2.83%) Reverse Only Surviving: 109 (0.97%) Dropped: 9695 (86.51%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: POS2_2_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: POS2_2_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: POS2_2_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: POS2_2_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS2_2_forward.paired.headclipped.fastq.gz POS2_2_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS2_2 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS2_2_forward.paired.headclipped.fastq.gz
[FLASH]     POS2_2_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS2_2.extendedFrags.fastq.gz
[FLASH]     ./POS2_2.notCombined_1.fastq.gz
[FLASH]     ./POS2_2.notCombined_2.fastq.gz
[FLASH]     ./POS2_2.hist
[FLASH]     ./POS2_2.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1086 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1086
[FLASH]     Combined pairs:   356
[FLASH]     Uncombined pairs: 730
[FLASH]     Percent combined: 32.78%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.011 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

356 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS2_2/POS2_2_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS2_2_centroids.fasta --uc POS2_2.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 356 sequences and identified 221 clusters (clustering threshold 1.00) - 3 clusters (representing 121 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS2_3 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS2_3_1.fastq /home/working/FinalCorrected/demultiplexed/POS2_3_2.fastq POS2_3_forward.paired.fastq.gz POS2_3_forward.singletons.fastq.gz POS2_3_reverse.paired.fastq.gz POS2_3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS2_3_1.fastq /home/working/FinalCorrected/demultiplexed/POS2_3_2.fastq POS2_3_forward.paired.fastq.gz POS2_3_forward.singletons.fastq.gz POS2_3_reverse.paired.fastq.gz POS2_3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:41:11 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS2_3/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 4462 Both Surviving: 804 (18.02%) Forward Only Surviving: 223 (5.00%) Reverse Only Surviving: 49 (1.10%) Dropped: 3386 (75.89%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: POS2_3_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: POS2_3_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: POS2_3_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: POS2_3_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS2_3_forward.paired.headclipped.fastq.gz POS2_3_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS2_3 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS2_3_forward.paired.headclipped.fastq.gz
[FLASH]     POS2_3_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS2_3.extendedFrags.fastq.gz
[FLASH]     ./POS2_3.notCombined_1.fastq.gz
[FLASH]     ./POS2_3.notCombined_2.fastq.gz
[FLASH]     ./POS2_3.hist
[FLASH]     ./POS2_3.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 804 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      804
[FLASH]     Combined pairs:   124
[FLASH]     Uncombined pairs: 680
[FLASH]     Percent combined: 15.42%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.006 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

124 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS2_3/POS2_3_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS2_3_centroids.fasta --uc POS2_3.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 124 sequences and identified 107 clusters (clustering threshold 1.00) - 1 clusters (representing 15 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS2_4 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS2_4_1.fastq /home/working/FinalCorrected/demultiplexed/POS2_4_2.fastq POS2_4_forward.paired.fastq.gz POS2_4_forward.singletons.fastq.gz POS2_4_reverse.paired.fastq.gz POS2_4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS2_4_1.fastq /home/working/FinalCorrected/demultiplexed/POS2_4_2.fastq POS2_4_forward.paired.fastq.gz POS2_4_forward.singletons.fastq.gz POS2_4_reverse.paired.fastq.gz POS2_4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:41:14 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS2_4/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22290 Both Surviving: 5759 (25.84%) Forward Only Surviving: 1666 (7.47%) Reverse Only Surviving: 337 (1.51%) Dropped: 14528 (65.18%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: POS2_4_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: POS2_4_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: POS2_4_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: POS2_4_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS2_4_forward.paired.headclipped.fastq.gz POS2_4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS2_4 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS2_4_forward.paired.headclipped.fastq.gz
[FLASH]     POS2_4_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS2_4.extendedFrags.fastq.gz
[FLASH]     ./POS2_4.notCombined_1.fastq.gz
[FLASH]     ./POS2_4.notCombined_2.fastq.gz
[FLASH]     ./POS2_4.hist
[FLASH]     ./POS2_4.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5759 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5759
[FLASH]     Combined pairs:   1693
[FLASH]     Uncombined pairs: 4066
[FLASH]     Percent combined: 29.40%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.051 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1693 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS2_4/POS2_4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS2_4_centroids.fasta --uc POS2_4.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1693 sequences and identified 928 clusters (clustering threshold 1.00) - 26 clusters (representing 639 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS3_1 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS3_1_1.fastq /home/working/FinalCorrected/demultiplexed/POS3_1_2.fastq POS3_1_forward.paired.fastq.gz POS3_1_forward.singletons.fastq.gz POS3_1_reverse.paired.fastq.gz POS3_1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS3_1_1.fastq /home/working/FinalCorrected/demultiplexed/POS3_1_2.fastq POS3_1_forward.paired.fastq.gz POS3_1_forward.singletons.fastq.gz POS3_1_reverse.paired.fastq.gz POS3_1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:41:21 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS3_1/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 21956 Both Surviving: 7403 (33.72%) Forward Only Surviving: 1314 (5.98%) Reverse Only Surviving: 438 (1.99%) Dropped: 12801 (58.30%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: POS3_1_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS3_1_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: POS3_1_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS3_1_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS3_1_forward.paired.headclipped.fastq.gz POS3_1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS3_1 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS3_1_forward.paired.headclipped.fastq.gz
[FLASH]     POS3_1_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS3_1.extendedFrags.fastq.gz
[FLASH]     ./POS3_1.notCombined_1.fastq.gz
[FLASH]     ./POS3_1.notCombined_2.fastq.gz
[FLASH]     ./POS3_1.hist
[FLASH]     ./POS3_1.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7403 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7403
[FLASH]     Combined pairs:   6470
[FLASH]     Uncombined pairs: 933
[FLASH]     Percent combined: 87.40%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.167 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6470 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS3_1/POS3_1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS3_1_centroids.fasta --uc POS3_1.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6470 sequences and identified 3518 clusters (clustering threshold 1.00) - 144 clusters (representing 2565 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS3_2 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS3_2_1.fastq /home/working/FinalCorrected/demultiplexed/POS3_2_2.fastq POS3_2_forward.paired.fastq.gz POS3_2_forward.singletons.fastq.gz POS3_2_reverse.paired.fastq.gz POS3_2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS3_2_1.fastq /home/working/FinalCorrected/demultiplexed/POS3_2_2.fastq POS3_2_forward.paired.fastq.gz POS3_2_forward.singletons.fastq.gz POS3_2_reverse.paired.fastq.gz POS3_2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:41:33 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS3_2/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 10258 Both Surviving: 3469 (33.82%) Forward Only Surviving: 838 (8.17%) Reverse Only Surviving: 138 (1.35%) Dropped: 5813 (56.67%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: POS3_2_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS3_2_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: POS3_2_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS3_2_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS3_2_forward.paired.headclipped.fastq.gz POS3_2_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS3_2 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS3_2_forward.paired.headclipped.fastq.gz
[FLASH]     POS3_2_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS3_2.extendedFrags.fastq.gz
[FLASH]     ./POS3_2.notCombined_1.fastq.gz
[FLASH]     ./POS3_2.notCombined_2.fastq.gz
[FLASH]     ./POS3_2.hist
[FLASH]     ./POS3_2.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3469 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3469
[FLASH]     Combined pairs:   2854
[FLASH]     Uncombined pairs: 615
[FLASH]     Percent combined: 82.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.075 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2854 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS3_2/POS3_2_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS3_2_centroids.fasta --uc POS3_2.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2854 sequences and identified 1775 clusters (clustering threshold 1.00) - 27 clusters (representing 794 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS3_3 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS3_3_1.fastq /home/working/FinalCorrected/demultiplexed/POS3_3_2.fastq POS3_3_forward.paired.fastq.gz POS3_3_forward.singletons.fastq.gz POS3_3_reverse.paired.fastq.gz POS3_3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS3_3_1.fastq /home/working/FinalCorrected/demultiplexed/POS3_3_2.fastq POS3_3_forward.paired.fastq.gz POS3_3_forward.singletons.fastq.gz POS3_3_reverse.paired.fastq.gz POS3_3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:41:40 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS3_3/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18806 Both Surviving: 7250 (38.55%) Forward Only Surviving: 1482 (7.88%) Reverse Only Surviving: 296 (1.57%) Dropped: 9778 (51.99%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: POS3_3_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS3_3_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: POS3_3_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS3_3_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS3_3_forward.paired.headclipped.fastq.gz POS3_3_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS3_3 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS3_3_forward.paired.headclipped.fastq.gz
[FLASH]     POS3_3_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS3_3.extendedFrags.fastq.gz
[FLASH]     ./POS3_3.notCombined_1.fastq.gz
[FLASH]     ./POS3_3.notCombined_2.fastq.gz
[FLASH]     ./POS3_3.hist
[FLASH]     ./POS3_3.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7250 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7250
[FLASH]     Combined pairs:   6193
[FLASH]     Uncombined pairs: 1057
[FLASH]     Percent combined: 85.42%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.163 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6193 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS3_3/POS3_3_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS3_3_centroids.fasta --uc POS3_3.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6193 sequences and identified 3588 clusters (clustering threshold 1.00) - 130 clusters (representing 2228 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS3_4 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS3_4_1.fastq /home/working/FinalCorrected/demultiplexed/POS3_4_2.fastq POS3_4_forward.paired.fastq.gz POS3_4_forward.singletons.fastq.gz POS3_4_reverse.paired.fastq.gz POS3_4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS3_4_1.fastq /home/working/FinalCorrected/demultiplexed/POS3_4_2.fastq POS3_4_forward.paired.fastq.gz POS3_4_forward.singletons.fastq.gz POS3_4_reverse.paired.fastq.gz POS3_4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:41:57 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS3_4/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 81440 Both Surviving: 25807 (31.69%) Forward Only Surviving: 6843 (8.40%) Reverse Only Surviving: 1355 (1.66%) Dropped: 47435 (58.25%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: POS3_4_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS3_4_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: POS3_4_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: POS3_4_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS3_4_forward.paired.headclipped.fastq.gz POS3_4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS3_4 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS3_4_forward.paired.headclipped.fastq.gz
[FLASH]     POS3_4_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS3_4.extendedFrags.fastq.gz
[FLASH]     ./POS3_4.notCombined_1.fastq.gz
[FLASH]     ./POS3_4.notCombined_2.fastq.gz
[FLASH]     ./POS3_4.hist
[FLASH]     ./POS3_4.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 25807 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      25807
[FLASH]     Combined pairs:   17223
[FLASH]     Uncombined pairs: 8584
[FLASH]     Percent combined: 66.74%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.467 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

17223 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS3_4/POS3_4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS3_4_centroids.fasta --uc POS3_4.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 17223 sequences and identified 8732 clusters (clustering threshold 1.00) - 384 clusters (representing 8089 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS4_1 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS4_1_1.fastq /home/working/FinalCorrected/demultiplexed/POS4_1_2.fastq POS4_1_forward.paired.fastq.gz POS4_1_forward.singletons.fastq.gz POS4_1_reverse.paired.fastq.gz POS4_1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS4_1_1.fastq /home/working/FinalCorrected/demultiplexed/POS4_1_2.fastq POS4_1_forward.paired.fastq.gz POS4_1_forward.singletons.fastq.gz POS4_1_reverse.paired.fastq.gz POS4_1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:42:34 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS4_1/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17420 Both Surviving: 2359 (13.54%) Forward Only Surviving: 478 (2.74%) Reverse Only Surviving: 256 (1.47%) Dropped: 14327 (82.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: POS4_1_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: POS4_1_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: POS4_1_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: POS4_1_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS4_1_forward.paired.headclipped.fastq.gz POS4_1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS4_1 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS4_1_forward.paired.headclipped.fastq.gz
[FLASH]     POS4_1_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS4_1.extendedFrags.fastq.gz
[FLASH]     ./POS4_1.notCombined_1.fastq.gz
[FLASH]     ./POS4_1.notCombined_2.fastq.gz
[FLASH]     ./POS4_1.hist
[FLASH]     ./POS4_1.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2359 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2359
[FLASH]     Combined pairs:   1684
[FLASH]     Uncombined pairs: 675
[FLASH]     Percent combined: 71.39%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.043 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1684 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS4_1/POS4_1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS4_1_centroids.fasta --uc POS4_1.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1684 sequences and identified 1002 clusters (clustering threshold 1.00) - 32 clusters (representing 516 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS4_2 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS4_2_1.fastq /home/working/FinalCorrected/demultiplexed/POS4_2_2.fastq POS4_2_forward.paired.fastq.gz POS4_2_forward.singletons.fastq.gz POS4_2_reverse.paired.fastq.gz POS4_2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS4_2_1.fastq /home/working/FinalCorrected/demultiplexed/POS4_2_2.fastq POS4_2_forward.paired.fastq.gz POS4_2_forward.singletons.fastq.gz POS4_2_reverse.paired.fastq.gz POS4_2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:42:39 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS4_2/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11301 Both Surviving: 1999 (17.69%) Forward Only Surviving: 548 (4.85%) Reverse Only Surviving: 109 (0.96%) Dropped: 8645 (76.50%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: POS4_2_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: POS4_2_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: POS4_2_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: POS4_2_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS4_2_forward.paired.headclipped.fastq.gz POS4_2_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS4_2 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS4_2_forward.paired.headclipped.fastq.gz
[FLASH]     POS4_2_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS4_2.extendedFrags.fastq.gz
[FLASH]     ./POS4_2.notCombined_1.fastq.gz
[FLASH]     ./POS4_2.notCombined_2.fastq.gz
[FLASH]     ./POS4_2.hist
[FLASH]     ./POS4_2.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1999 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1999
[FLASH]     Combined pairs:   1192
[FLASH]     Uncombined pairs: 807
[FLASH]     Percent combined: 59.63%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.033 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1192 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS4_2/POS4_2_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS4_2_centroids.fasta --uc POS4_2.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1192 sequences and identified 786 clusters (clustering threshold 1.00) - 13 clusters (representing 296 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS4_3 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS4_3_1.fastq /home/working/FinalCorrected/demultiplexed/POS4_3_2.fastq POS4_3_forward.paired.fastq.gz POS4_3_forward.singletons.fastq.gz POS4_3_reverse.paired.fastq.gz POS4_3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS4_3_1.fastq /home/working/FinalCorrected/demultiplexed/POS4_3_2.fastq POS4_3_forward.paired.fastq.gz POS4_3_forward.singletons.fastq.gz POS4_3_reverse.paired.fastq.gz POS4_3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:42:44 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS4_3/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 19703 Both Surviving: 3445 (17.48%) Forward Only Surviving: 733 (3.72%) Reverse Only Surviving: 274 (1.39%) Dropped: 15251 (77.40%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: POS4_3_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: POS4_3_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: POS4_3_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: POS4_3_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS4_3_forward.paired.headclipped.fastq.gz POS4_3_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS4_3 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS4_3_forward.paired.headclipped.fastq.gz
[FLASH]     POS4_3_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS4_3.extendedFrags.fastq.gz
[FLASH]     ./POS4_3.notCombined_1.fastq.gz
[FLASH]     ./POS4_3.notCombined_2.fastq.gz
[FLASH]     ./POS4_3.hist
[FLASH]     ./POS4_3.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3445 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3445
[FLASH]     Combined pairs:   2556
[FLASH]     Uncombined pairs: 889
[FLASH]     Percent combined: 74.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.066 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2556 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS4_3/POS4_3_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS4_3_centroids.fasta --uc POS4_3.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2556 sequences and identified 1540 clusters (clustering threshold 1.00) - 54 clusters (representing 833 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: POS4_4 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS4_4_1.fastq /home/working/FinalCorrected/demultiplexed/POS4_4_2.fastq POS4_4_forward.paired.fastq.gz POS4_4_forward.singletons.fastq.gz POS4_4_reverse.paired.fastq.gz POS4_4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/POS4_4_1.fastq /home/working/FinalCorrected/demultiplexed/POS4_4_2.fastq POS4_4_forward.paired.fastq.gz POS4_4_forward.singletons.fastq.gz POS4_4_reverse.paired.fastq.gz POS4_4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:42:53 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/POS4_4/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41036 Both Surviving: 6364 (15.51%) Forward Only Surviving: 1879 (4.58%) Reverse Only Surviving: 561 (1.37%) Dropped: 32232 (78.55%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: POS4_4_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: POS4_4_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: POS4_4_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: POS4_4_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash POS4_4_forward.paired.headclipped.fastq.gz POS4_4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS4_4 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     POS4_4_forward.paired.headclipped.fastq.gz
[FLASH]     POS4_4_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./POS4_4.extendedFrags.fastq.gz
[FLASH]     ./POS4_4.notCombined_1.fastq.gz
[FLASH]     ./POS4_4.notCombined_2.fastq.gz
[FLASH]     ./POS4_4.hist
[FLASH]     ./POS4_4.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6364 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6364
[FLASH]     Combined pairs:   2674
[FLASH]     Uncombined pairs: 3690
[FLASH]     Percent combined: 42.02%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.078 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2674 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/POS4_4/POS4_4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS4_4_centroids.fasta --uc POS4_4.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2674 sequences and identified 1561 clusters (clustering threshold 1.00) - 59 clusters (representing 920 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: PROC_1 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/PROC_1_1.fastq /home/working/FinalCorrected/demultiplexed/PROC_1_2.fastq PROC_1_forward.paired.fastq.gz PROC_1_forward.singletons.fastq.gz PROC_1_reverse.paired.fastq.gz PROC_1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/PROC_1_1.fastq /home/working/FinalCorrected/demultiplexed/PROC_1_2.fastq PROC_1_forward.paired.fastq.gz PROC_1_forward.singletons.fastq.gz PROC_1_reverse.paired.fastq.gz PROC_1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:43:03 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/PROC_1/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 33456 Both Surviving: 6311 (18.86%) Forward Only Surviving: 1442 (4.31%) Reverse Only Surviving: 423 (1.26%) Dropped: 25280 (75.56%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: PROC_1_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: PROC_1_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: PROC_1_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: PROC_1_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash PROC_1_forward.paired.headclipped.fastq.gz PROC_1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o PROC_1 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     PROC_1_forward.paired.headclipped.fastq.gz
[FLASH]     PROC_1_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./PROC_1.extendedFrags.fastq.gz
[FLASH]     ./PROC_1.notCombined_1.fastq.gz
[FLASH]     ./PROC_1.notCombined_2.fastq.gz
[FLASH]     ./PROC_1.hist
[FLASH]     ./PROC_1.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6311 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6311
[FLASH]     Combined pairs:   1393
[FLASH]     Uncombined pairs: 4918
[FLASH]     Percent combined: 22.07%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.065 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1393 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/PROC_1/PROC_1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids PROC_1_centroids.fasta --uc PROC_1.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1393 sequences and identified 250 clusters (clustering threshold 1.00) - 20 clusters (representing 1050 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: PROC_2 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/PROC_2_1.fastq /home/working/FinalCorrected/demultiplexed/PROC_2_2.fastq PROC_2_forward.paired.fastq.gz PROC_2_forward.singletons.fastq.gz PROC_2_reverse.paired.fastq.gz PROC_2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/PROC_2_1.fastq /home/working/FinalCorrected/demultiplexed/PROC_2_2.fastq PROC_2_forward.paired.fastq.gz PROC_2_forward.singletons.fastq.gz PROC_2_reverse.paired.fastq.gz PROC_2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:43:10 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/PROC_2/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16777 Both Surviving: 3210 (19.13%) Forward Only Surviving: 860 (5.13%) Reverse Only Surviving: 167 (1.00%) Dropped: 12540 (74.75%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: PROC_2_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: PROC_2_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: PROC_2_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: PROC_2_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash PROC_2_forward.paired.headclipped.fastq.gz PROC_2_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o PROC_2 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     PROC_2_forward.paired.headclipped.fastq.gz
[FLASH]     PROC_2_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./PROC_2.extendedFrags.fastq.gz
[FLASH]     ./PROC_2.notCombined_1.fastq.gz
[FLASH]     ./PROC_2.notCombined_2.fastq.gz
[FLASH]     ./PROC_2.hist
[FLASH]     ./PROC_2.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3210 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3210
[FLASH]     Combined pairs:   155
[FLASH]     Uncombined pairs: 3055
[FLASH]     Percent combined: 4.83%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.020 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

155 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/PROC_2/PROC_2_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids PROC_2_centroids.fasta --uc PROC_2.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 155 sequences and identified 116 clusters (clustering threshold 1.00) - 4 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: PROC_3 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/PROC_3_1.fastq /home/working/FinalCorrected/demultiplexed/PROC_3_2.fastq PROC_3_forward.paired.fastq.gz PROC_3_forward.singletons.fastq.gz PROC_3_reverse.paired.fastq.gz PROC_3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/PROC_3_1.fastq /home/working/FinalCorrected/demultiplexed/PROC_3_2.fastq PROC_3_forward.paired.fastq.gz PROC_3_forward.singletons.fastq.gz PROC_3_reverse.paired.fastq.gz PROC_3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:43:14 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/PROC_3/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 27475 Both Surviving: 8699 (31.66%) Forward Only Surviving: 1666 (6.06%) Reverse Only Surviving: 454 (1.65%) Dropped: 16656 (60.62%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: PROC_3_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: PROC_3_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: PROC_3_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: PROC_3_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash PROC_3_forward.paired.headclipped.fastq.gz PROC_3_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o PROC_3 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     PROC_3_forward.paired.headclipped.fastq.gz
[FLASH]     PROC_3_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./PROC_3.extendedFrags.fastq.gz
[FLASH]     ./PROC_3.notCombined_1.fastq.gz
[FLASH]     ./PROC_3.notCombined_2.fastq.gz
[FLASH]     ./PROC_3.hist
[FLASH]     ./PROC_3.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8699 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8699
[FLASH]     Combined pairs:   1377
[FLASH]     Uncombined pairs: 7322
[FLASH]     Percent combined: 15.83%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.057 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1377 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/PROC_3/PROC_3_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids PROC_3_centroids.fasta --uc PROC_3.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1377 sequences and identified 861 clusters (clustering threshold 1.00) - 18 clusters (representing 376 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: PROC_4 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/PROC_4_1.fastq /home/working/FinalCorrected/demultiplexed/PROC_4_2.fastq PROC_4_forward.paired.fastq.gz PROC_4_forward.singletons.fastq.gz PROC_4_reverse.paired.fastq.gz PROC_4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/PROC_4_1.fastq /home/working/FinalCorrected/demultiplexed/PROC_4_2.fastq PROC_4_forward.paired.fastq.gz PROC_4_forward.singletons.fastq.gz PROC_4_reverse.paired.fastq.gz PROC_4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:43:23 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/PROC_4/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 49705 Both Surviving: 15941 (32.07%) Forward Only Surviving: 3992 (8.03%) Reverse Only Surviving: 863 (1.74%) Dropped: 28909 (58.16%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: PROC_4_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: PROC_4_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: PROC_4_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: PROC_4_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash PROC_4_forward.paired.headclipped.fastq.gz PROC_4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o PROC_4 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     PROC_4_forward.paired.headclipped.fastq.gz
[FLASH]     PROC_4_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./PROC_4.extendedFrags.fastq.gz
[FLASH]     ./PROC_4.notCombined_1.fastq.gz
[FLASH]     ./PROC_4.notCombined_2.fastq.gz
[FLASH]     ./PROC_4.hist
[FLASH]     ./PROC_4.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 15941 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      15941
[FLASH]     Combined pairs:   3516
[FLASH]     Uncombined pairs: 12425
[FLASH]     Percent combined: 22.06%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.131 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3516 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/PROC_4/PROC_4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids PROC_4_centroids.fasta --uc PROC_4.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3516 sequences and identified 1481 clusters (clustering threshold 1.00) - 45 clusters (representing 1736 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Q10 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Q10_1.fastq /home/working/FinalCorrected/demultiplexed/Q10_2.fastq Q10_forward.paired.fastq.gz Q10_forward.singletons.fastq.gz Q10_reverse.paired.fastq.gz Q10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Q10_1.fastq /home/working/FinalCorrected/demultiplexed/Q10_2.fastq Q10_forward.paired.fastq.gz Q10_forward.singletons.fastq.gz Q10_reverse.paired.fastq.gz Q10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:43:36 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/Q10/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23215 Both Surviving: 1090 (4.70%) Forward Only Surviving: 694 (2.99%) Reverse Only Surviving: 300 (1.29%) Dropped: 21131 (91.02%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: Q10_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: Q10_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: Q10_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: Q10_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Q10_forward.paired.headclipped.fastq.gz Q10_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q10 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Q10_forward.paired.headclipped.fastq.gz
[FLASH]     Q10_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Q10.extendedFrags.fastq.gz
[FLASH]     ./Q10.notCombined_1.fastq.gz
[FLASH]     ./Q10.notCombined_2.fastq.gz
[FLASH]     ./Q10.hist
[FLASH]     ./Q10.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1090 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1090
[FLASH]     Combined pairs:   251
[FLASH]     Uncombined pairs: 839
[FLASH]     Percent combined: 23.03%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.010 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

251 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/Q10/Q10_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q10_centroids.fasta --uc Q10.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 251 sequences and identified 203 clusters (clustering threshold 1.00) - 4 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Q30 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Q30_1.fastq /home/working/FinalCorrected/demultiplexed/Q30_2.fastq Q30_forward.paired.fastq.gz Q30_forward.singletons.fastq.gz Q30_reverse.paired.fastq.gz Q30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Q30_1.fastq /home/working/FinalCorrected/demultiplexed/Q30_2.fastq Q30_forward.paired.fastq.gz Q30_forward.singletons.fastq.gz Q30_reverse.paired.fastq.gz Q30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:43:39 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/Q30/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 2 Both Surviving: 2 (100.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 0 (0.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: Q30_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: Q30_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: Q30_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: Q30_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Q30_forward.paired.headclipped.fastq.gz Q30_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q30 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Q30_forward.paired.headclipped.fastq.gz
[FLASH]     Q30_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Q30.extendedFrags.fastq.gz
[FLASH]     ./Q30.notCombined_1.fastq.gz
[FLASH]     ./Q30.notCombined_2.fastq.gz
[FLASH]     ./Q30.hist
[FLASH]     ./Q30.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2
[FLASH]     Combined pairs:   2
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 100.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/Q30/Q30_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q30_centroids.fasta --uc Q30.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2 sequences and identified 2 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Q43 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Q43_1.fastq /home/working/FinalCorrected/demultiplexed/Q43_2.fastq Q43_forward.paired.fastq.gz Q43_forward.singletons.fastq.gz Q43_reverse.paired.fastq.gz Q43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Q43_1.fastq /home/working/FinalCorrected/demultiplexed/Q43_2.fastq Q43_forward.paired.fastq.gz Q43_forward.singletons.fastq.gz Q43_reverse.paired.fastq.gz Q43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:43:39 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/Q43/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 1 Both Surviving: 0 (0.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 1 (100.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: Q43_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: Q43_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: Q43_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: Q43_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Q43_forward.paired.headclipped.fastq.gz Q43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q43 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Q43_forward.paired.headclipped.fastq.gz
[FLASH]     Q43_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Q43.extendedFrags.fastq.gz
[FLASH]     ./Q43.notCombined_1.fastq.gz
[FLASH]     ./Q43.notCombined_2.fastq.gz
[FLASH]     ./Q43.hist
[FLASH]     ./Q43.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/Q43/Q43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q43_centroids.fasta --uc Q43.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Q64 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Q64_1.fastq /home/working/FinalCorrected/demultiplexed/Q64_2.fastq Q64_forward.paired.fastq.gz Q64_forward.singletons.fastq.gz Q64_reverse.paired.fastq.gz Q64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Q64_1.fastq /home/working/FinalCorrected/demultiplexed/Q64_2.fastq Q64_forward.paired.fastq.gz Q64_forward.singletons.fastq.gz Q64_reverse.paired.fastq.gz Q64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:43:39 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/Q64/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 2 Both Surviving: 1 (50.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 1 (50.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: Q64_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: Q64_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: Q64_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: Q64_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Q64_forward.paired.headclipped.fastq.gz Q64_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q64 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Q64_forward.paired.headclipped.fastq.gz
[FLASH]     Q64_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Q64.extendedFrags.fastq.gz
[FLASH]     ./Q64.notCombined_1.fastq.gz
[FLASH]     ./Q64.notCombined_2.fastq.gz
[FLASH]     ./Q64.hist
[FLASH]     ./Q64.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1
[FLASH]     Combined pairs:   1
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 100.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/Q64/Q64_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q64_centroids.fasta --uc Q64.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1 sequences and identified 1 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Q69 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Q69_1.fastq /home/working/FinalCorrected/demultiplexed/Q69_2.fastq Q69_forward.paired.fastq.gz Q69_forward.singletons.fastq.gz Q69_reverse.paired.fastq.gz Q69_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Q69_1.fastq /home/working/FinalCorrected/demultiplexed/Q69_2.fastq Q69_forward.paired.fastq.gz Q69_forward.singletons.fastq.gz Q69_reverse.paired.fastq.gz Q69_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:43:40 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/Q69/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 4825 Both Surviving: 51 (1.06%) Forward Only Surviving: 97 (2.01%) Reverse Only Surviving: 34 (0.70%) Dropped: 4643 (96.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: Q69_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: Q69_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: Q69_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: Q69_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Q69_forward.paired.headclipped.fastq.gz Q69_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q69 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Q69_forward.paired.headclipped.fastq.gz
[FLASH]     Q69_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Q69.extendedFrags.fastq.gz
[FLASH]     ./Q69.notCombined_1.fastq.gz
[FLASH]     ./Q69.notCombined_2.fastq.gz
[FLASH]     ./Q69.hist
[FLASH]     ./Q69.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 51 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      51
[FLASH]     Combined pairs:   21
[FLASH]     Uncombined pairs: 30
[FLASH]     Percent combined: 41.18%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

21 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/Q69/Q69_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q69_centroids.fasta --uc Q69.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 21 sequences and identified 20 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: Q83 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Q83_1.fastq /home/working/FinalCorrected/demultiplexed/Q83_2.fastq Q83_forward.paired.fastq.gz Q83_forward.singletons.fastq.gz Q83_reverse.paired.fastq.gz Q83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/Q83_1.fastq /home/working/FinalCorrected/demultiplexed/Q83_2.fastq Q83_forward.paired.fastq.gz Q83_forward.singletons.fastq.gz Q83_reverse.paired.fastq.gz Q83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:43:42 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/Q83/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29958 Both Surviving: 5419 (18.09%) Forward Only Surviving: 1242 (4.15%) Reverse Only Surviving: 466 (1.56%) Dropped: 22831 (76.21%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: Q83_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: Q83_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: Q83_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: Q83_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash Q83_forward.paired.headclipped.fastq.gz Q83_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q83 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     Q83_forward.paired.headclipped.fastq.gz
[FLASH]     Q83_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./Q83.extendedFrags.fastq.gz
[FLASH]     ./Q83.notCombined_1.fastq.gz
[FLASH]     ./Q83.notCombined_2.fastq.gz
[FLASH]     ./Q83.hist
[FLASH]     ./Q83.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5419 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5419
[FLASH]     Combined pairs:   1324
[FLASH]     Uncombined pairs: 4095
[FLASH]     Percent combined: 24.43%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.052 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1324 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/Q83/Q83_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q83_centroids.fasta --uc Q83.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1324 sequences and identified 822 clusters (clustering threshold 1.00) - 17 clusters (representing 405 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R1 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R1_1.fastq /home/working/FinalCorrected/demultiplexed/R1_2.fastq R1_forward.paired.fastq.gz R1_forward.singletons.fastq.gz R1_reverse.paired.fastq.gz R1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R1_1.fastq /home/working/FinalCorrected/demultiplexed/R1_2.fastq R1_forward.paired.fastq.gz R1_forward.singletons.fastq.gz R1_reverse.paired.fastq.gz R1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:43:48 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R1/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16656 Both Surviving: 5127 (30.78%) Forward Only Surviving: 1237 (7.43%) Reverse Only Surviving: 238 (1.43%) Dropped: 10054 (60.36%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: R1_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: R1_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: R1_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: R1_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R1_forward.paired.headclipped.fastq.gz R1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R1 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R1_forward.paired.headclipped.fastq.gz
[FLASH]     R1_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R1.extendedFrags.fastq.gz
[FLASH]     ./R1.notCombined_1.fastq.gz
[FLASH]     ./R1.notCombined_2.fastq.gz
[FLASH]     ./R1.hist
[FLASH]     ./R1.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5127 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5127
[FLASH]     Combined pairs:   2511
[FLASH]     Uncombined pairs: 2616
[FLASH]     Percent combined: 48.98%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.074 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2511 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R1/R1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R1_centroids.fasta --uc R1.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2511 sequences and identified 872 clusters (clustering threshold 1.00) - 44 clusters (representing 1438 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R10 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R10_1.fastq /home/working/FinalCorrected/demultiplexed/R10_2.fastq R10_forward.paired.fastq.gz R10_forward.singletons.fastq.gz R10_reverse.paired.fastq.gz R10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R10_1.fastq /home/working/FinalCorrected/demultiplexed/R10_2.fastq R10_forward.paired.fastq.gz R10_forward.singletons.fastq.gz R10_reverse.paired.fastq.gz R10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:43:55 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R10/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: R10_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: R10_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R10_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R10_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R10_forward.paired.headclipped.fastq.gz R10_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R10 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R10_forward.paired.headclipped.fastq.gz
[FLASH]     R10_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R10.extendedFrags.fastq.gz
[FLASH]     ./R10.notCombined_1.fastq.gz
[FLASH]     ./R10.notCombined_2.fastq.gz
[FLASH]     ./R10.hist
[FLASH]     ./R10.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R10/R10_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R10_centroids.fasta --uc R10.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R102 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R102_1.fastq /home/working/FinalCorrected/demultiplexed/R102_2.fastq R102_forward.paired.fastq.gz R102_forward.singletons.fastq.gz R102_reverse.paired.fastq.gz R102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R102_1.fastq /home/working/FinalCorrected/demultiplexed/R102_2.fastq R102_forward.paired.fastq.gz R102_forward.singletons.fastq.gz R102_reverse.paired.fastq.gz R102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:43:59 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R102/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 62773 Both Surviving: 13515 (21.53%) Forward Only Surviving: 3949 (6.29%) Reverse Only Surviving: 1852 (2.95%) Dropped: 43457 (69.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: R102_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: R102_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R102_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R102_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R102_forward.paired.headclipped.fastq.gz R102_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R102 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R102_forward.paired.headclipped.fastq.gz
[FLASH]     R102_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R102.extendedFrags.fastq.gz
[FLASH]     ./R102.notCombined_1.fastq.gz
[FLASH]     ./R102.notCombined_2.fastq.gz
[FLASH]     ./R102.hist
[FLASH]     ./R102.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 13515 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      13515
[FLASH]     Combined pairs:   765
[FLASH]     Uncombined pairs: 12750
[FLASH]     Percent combined: 5.66%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.086 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

765 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R102/R102_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R102_centroids.fasta --uc R102.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 765 sequences and identified 582 clusters (clustering threshold 1.00) - 14 clusters (representing 145 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R13 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R13_1.fastq /home/working/FinalCorrected/demultiplexed/R13_2.fastq R13_forward.paired.fastq.gz R13_forward.singletons.fastq.gz R13_reverse.paired.fastq.gz R13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R13_1.fastq /home/working/FinalCorrected/demultiplexed/R13_2.fastq R13_forward.paired.fastq.gz R13_forward.singletons.fastq.gz R13_reverse.paired.fastq.gz R13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:44:08 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R13/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 19033 Both Surviving: 4660 (24.48%) Forward Only Surviving: 1222 (6.42%) Reverse Only Surviving: 224 (1.18%) Dropped: 12927 (67.92%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: R13_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: R13_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R13_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R13_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R13_forward.paired.headclipped.fastq.gz R13_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R13 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R13_forward.paired.headclipped.fastq.gz
[FLASH]     R13_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R13.extendedFrags.fastq.gz
[FLASH]     ./R13.notCombined_1.fastq.gz
[FLASH]     ./R13.notCombined_2.fastq.gz
[FLASH]     ./R13.hist
[FLASH]     ./R13.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4660 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4660
[FLASH]     Combined pairs:   629
[FLASH]     Uncombined pairs: 4031
[FLASH]     Percent combined: 13.50%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.031 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

629 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R13/R13_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R13_centroids.fasta --uc R13.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 629 sequences and identified 406 clusters (clustering threshold 1.00) - 12 clusters (representing 203 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R15 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R15_1.fastq /home/working/FinalCorrected/demultiplexed/R15_2.fastq R15_forward.paired.fastq.gz R15_forward.singletons.fastq.gz R15_reverse.paired.fastq.gz R15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R15_1.fastq /home/working/FinalCorrected/demultiplexed/R15_2.fastq R15_forward.paired.fastq.gz R15_forward.singletons.fastq.gz R15_reverse.paired.fastq.gz R15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:44:15 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R15/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 65631 Both Surviving: 3298 (5.03%) Forward Only Surviving: 2188 (3.33%) Reverse Only Surviving: 804 (1.23%) Dropped: 59341 (90.42%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: R15_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: R15_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: R15_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: R15_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R15_forward.paired.headclipped.fastq.gz R15_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R15 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R15_forward.paired.headclipped.fastq.gz
[FLASH]     R15_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R15.extendedFrags.fastq.gz
[FLASH]     ./R15.notCombined_1.fastq.gz
[FLASH]     ./R15.notCombined_2.fastq.gz
[FLASH]     ./R15.hist
[FLASH]     ./R15.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3298 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3298
[FLASH]     Combined pairs:   2086
[FLASH]     Uncombined pairs: 1212
[FLASH]     Percent combined: 63.25%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.062 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2086 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R15/R15_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R15_centroids.fasta --uc R15.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2086 sequences and identified 1204 clusters (clustering threshold 1.00) - 32 clusters (representing 807 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R16 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R16_1.fastq /home/working/FinalCorrected/demultiplexed/R16_2.fastq R16_forward.paired.fastq.gz R16_forward.singletons.fastq.gz R16_reverse.paired.fastq.gz R16_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R16_1.fastq /home/working/FinalCorrected/demultiplexed/R16_2.fastq R16_forward.paired.fastq.gz R16_forward.singletons.fastq.gz R16_reverse.paired.fastq.gz R16_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:44:24 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R16/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25112 Both Surviving: 8135 (32.39%) Forward Only Surviving: 3246 (12.93%) Reverse Only Surviving: 283 (1.13%) Dropped: 13448 (53.55%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R16_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R16_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: R16_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: R16_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R16_forward.paired.headclipped.fastq.gz R16_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R16 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R16_forward.paired.headclipped.fastq.gz
[FLASH]     R16_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R16.extendedFrags.fastq.gz
[FLASH]     ./R16.notCombined_1.fastq.gz
[FLASH]     ./R16.notCombined_2.fastq.gz
[FLASH]     ./R16.hist
[FLASH]     ./R16.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8135 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8135
[FLASH]     Combined pairs:   4002
[FLASH]     Uncombined pairs: 4133
[FLASH]     Percent combined: 49.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.115 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4002 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R16/R16_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R16_centroids.fasta --uc R16.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4002 sequences and identified 2239 clusters (clustering threshold 1.00) - 59 clusters (representing 1497 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R20 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R20_1.fastq /home/working/FinalCorrected/demultiplexed/R20_2.fastq R20_forward.paired.fastq.gz R20_forward.singletons.fastq.gz R20_reverse.paired.fastq.gz R20_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R20_1.fastq /home/working/FinalCorrected/demultiplexed/R20_2.fastq R20_forward.paired.fastq.gz R20_forward.singletons.fastq.gz R20_reverse.paired.fastq.gz R20_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:44:37 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R20/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41196 Both Surviving: 10311 (25.03%) Forward Only Surviving: 6801 (16.51%) Reverse Only Surviving: 389 (0.94%) Dropped: 23695 (57.52%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R20_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R20_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: R20_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: R20_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R20_forward.paired.headclipped.fastq.gz R20_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R20 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R20_forward.paired.headclipped.fastq.gz
[FLASH]     R20_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R20.extendedFrags.fastq.gz
[FLASH]     ./R20.notCombined_1.fastq.gz
[FLASH]     ./R20.notCombined_2.fastq.gz
[FLASH]     ./R20.hist
[FLASH]     ./R20.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10311 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10311
[FLASH]     Combined pairs:   8629
[FLASH]     Uncombined pairs: 1682
[FLASH]     Percent combined: 83.69%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.229 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

8629 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R20/R20_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R20_centroids.fasta --uc R20.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 8629 sequences and identified 3449 clusters (clustering threshold 1.00) - 186 clusters (representing 4843 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R22 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R22_1.fastq /home/working/FinalCorrected/demultiplexed/R22_2.fastq R22_forward.paired.fastq.gz R22_forward.singletons.fastq.gz R22_reverse.paired.fastq.gz R22_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R22_1.fastq /home/working/FinalCorrected/demultiplexed/R22_2.fastq R22_forward.paired.fastq.gz R22_forward.singletons.fastq.gz R22_reverse.paired.fastq.gz R22_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:44:56 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R22/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 40727 Both Surviving: 5266 (12.93%) Forward Only Surviving: 1531 (3.76%) Reverse Only Surviving: 618 (1.52%) Dropped: 33312 (81.79%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R22_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R22_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: R22_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: R22_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R22_forward.paired.headclipped.fastq.gz R22_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R22 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R22_forward.paired.headclipped.fastq.gz
[FLASH]     R22_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R22.extendedFrags.fastq.gz
[FLASH]     ./R22.notCombined_1.fastq.gz
[FLASH]     ./R22.notCombined_2.fastq.gz
[FLASH]     ./R22.hist
[FLASH]     ./R22.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5266 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5266
[FLASH]     Combined pairs:   1400
[FLASH]     Uncombined pairs: 3866
[FLASH]     Percent combined: 26.59%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.046 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1400 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R22/R22_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R22_centroids.fasta --uc R22.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1400 sequences and identified 870 clusters (clustering threshold 1.00) - 21 clusters (representing 458 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R23 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R23_1.fastq /home/working/FinalCorrected/demultiplexed/R23_2.fastq R23_forward.paired.fastq.gz R23_forward.singletons.fastq.gz R23_reverse.paired.fastq.gz R23_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R23_1.fastq /home/working/FinalCorrected/demultiplexed/R23_2.fastq R23_forward.paired.fastq.gz R23_forward.singletons.fastq.gz R23_reverse.paired.fastq.gz R23_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:45:04 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R23/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 27238 Both Surviving: 5046 (18.53%) Forward Only Surviving: 994 (3.65%) Reverse Only Surviving: 810 (2.97%) Dropped: 20388 (74.85%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: R23_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: R23_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R23_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R23_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R23_forward.paired.headclipped.fastq.gz R23_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R23 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R23_forward.paired.headclipped.fastq.gz
[FLASH]     R23_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R23.extendedFrags.fastq.gz
[FLASH]     ./R23.notCombined_1.fastq.gz
[FLASH]     ./R23.notCombined_2.fastq.gz
[FLASH]     ./R23.hist
[FLASH]     ./R23.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5046 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5046
[FLASH]     Combined pairs:   642
[FLASH]     Uncombined pairs: 4404
[FLASH]     Percent combined: 12.72%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.030 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

642 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R23/R23_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R23_centroids.fasta --uc R23.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 642 sequences and identified 364 clusters (clustering threshold 1.00) - 13 clusters (representing 256 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R27 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R27_1.fastq /home/working/FinalCorrected/demultiplexed/R27_2.fastq R27_forward.paired.fastq.gz R27_forward.singletons.fastq.gz R27_reverse.paired.fastq.gz R27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R27_1.fastq /home/working/FinalCorrected/demultiplexed/R27_2.fastq R27_forward.paired.fastq.gz R27_forward.singletons.fastq.gz R27_reverse.paired.fastq.gz R27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:45:11 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R27/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41488 Both Surviving: 9286 (22.38%) Forward Only Surviving: 3270 (7.88%) Reverse Only Surviving: 522 (1.26%) Dropped: 28410 (68.48%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: R27_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: R27_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R27_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R27_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R27_forward.paired.headclipped.fastq.gz R27_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R27 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R27_forward.paired.headclipped.fastq.gz
[FLASH]     R27_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R27.extendedFrags.fastq.gz
[FLASH]     ./R27.notCombined_1.fastq.gz
[FLASH]     ./R27.notCombined_2.fastq.gz
[FLASH]     ./R27.hist
[FLASH]     ./R27.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9286 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9286
[FLASH]     Combined pairs:   7926
[FLASH]     Uncombined pairs: 1360
[FLASH]     Percent combined: 85.35%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.205 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

7926 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R27/R27_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R27_centroids.fasta --uc R27.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 7926 sequences and identified 2335 clusters (clustering threshold 1.00) - 172 clusters (representing 5330 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R29 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R29_1.fastq /home/working/FinalCorrected/demultiplexed/R29_2.fastq R29_forward.paired.fastq.gz R29_forward.singletons.fastq.gz R29_reverse.paired.fastq.gz R29_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R29_1.fastq /home/working/FinalCorrected/demultiplexed/R29_2.fastq R29_forward.paired.fastq.gz R29_forward.singletons.fastq.gz R29_reverse.paired.fastq.gz R29_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:45:26 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R29/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 33797 Both Surviving: 8652 (25.60%) Forward Only Surviving: 2104 (6.23%) Reverse Only Surviving: 517 (1.53%) Dropped: 22524 (66.64%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: R29_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: R29_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: R29_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: R29_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R29_forward.paired.headclipped.fastq.gz R29_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R29 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R29_forward.paired.headclipped.fastq.gz
[FLASH]     R29_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R29.extendedFrags.fastq.gz
[FLASH]     ./R29.notCombined_1.fastq.gz
[FLASH]     ./R29.notCombined_2.fastq.gz
[FLASH]     ./R29.hist
[FLASH]     ./R29.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8652 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8652
[FLASH]     Combined pairs:   2616
[FLASH]     Uncombined pairs: 6036
[FLASH]     Percent combined: 30.24%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.082 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2616 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R29/R29_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R29_centroids.fasta --uc R29.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2616 sequences and identified 1564 clusters (clustering threshold 1.00) - 49 clusters (representing 816 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R30 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R30_1.fastq /home/working/FinalCorrected/demultiplexed/R30_2.fastq R30_forward.paired.fastq.gz R30_forward.singletons.fastq.gz R30_reverse.paired.fastq.gz R30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R30_1.fastq /home/working/FinalCorrected/demultiplexed/R30_2.fastq R30_forward.paired.fastq.gz R30_forward.singletons.fastq.gz R30_reverse.paired.fastq.gz R30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:45:34 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R30/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16649 Both Surviving: 2332 (14.01%) Forward Only Surviving: 703 (4.22%) Reverse Only Surviving: 184 (1.11%) Dropped: 13430 (80.67%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: R30_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: R30_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: R30_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: R30_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R30_forward.paired.headclipped.fastq.gz R30_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R30 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R30_forward.paired.headclipped.fastq.gz
[FLASH]     R30_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R30.extendedFrags.fastq.gz
[FLASH]     ./R30.notCombined_1.fastq.gz
[FLASH]     ./R30.notCombined_2.fastq.gz
[FLASH]     ./R30.hist
[FLASH]     ./R30.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2332 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2332
[FLASH]     Combined pairs:   531
[FLASH]     Uncombined pairs: 1801
[FLASH]     Percent combined: 22.77%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.017 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

531 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R30/R30_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R30_centroids.fasta --uc R30.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 531 sequences and identified 277 clusters (clustering threshold 1.00) - 5 clusters (representing 225 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R31 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R31_1.fastq /home/working/FinalCorrected/demultiplexed/R31_2.fastq R31_forward.paired.fastq.gz R31_forward.singletons.fastq.gz R31_reverse.paired.fastq.gz R31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R31_1.fastq /home/working/FinalCorrected/demultiplexed/R31_2.fastq R31_forward.paired.fastq.gz R31_forward.singletons.fastq.gz R31_reverse.paired.fastq.gz R31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:45:38 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R31/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26755 Both Surviving: 5140 (19.21%) Forward Only Surviving: 1230 (4.60%) Reverse Only Surviving: 349 (1.30%) Dropped: 20036 (74.89%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: R31_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: R31_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R31_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R31_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R31_forward.paired.headclipped.fastq.gz R31_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R31 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R31_forward.paired.headclipped.fastq.gz
[FLASH]     R31_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R31.extendedFrags.fastq.gz
[FLASH]     ./R31.notCombined_1.fastq.gz
[FLASH]     ./R31.notCombined_2.fastq.gz
[FLASH]     ./R31.hist
[FLASH]     ./R31.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5140 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5140
[FLASH]     Combined pairs:   592
[FLASH]     Uncombined pairs: 4548
[FLASH]     Percent combined: 11.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.032 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

592 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R31/R31_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R31_centroids.fasta --uc R31.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 592 sequences and identified 414 clusters (clustering threshold 1.00) - 10 clusters (representing 132 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R32 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R32_1.fastq /home/working/FinalCorrected/demultiplexed/R32_2.fastq R32_forward.paired.fastq.gz R32_forward.singletons.fastq.gz R32_reverse.paired.fastq.gz R32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R32_1.fastq /home/working/FinalCorrected/demultiplexed/R32_2.fastq R32_forward.paired.fastq.gz R32_forward.singletons.fastq.gz R32_reverse.paired.fastq.gz R32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:45:44 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R32/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 32624 Both Surviving: 1639 (5.02%) Forward Only Surviving: 1353 (4.15%) Reverse Only Surviving: 415 (1.27%) Dropped: 29217 (89.56%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: R32_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: R32_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R32_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R32_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R32_forward.paired.headclipped.fastq.gz R32_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R32 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R32_forward.paired.headclipped.fastq.gz
[FLASH]     R32_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R32.extendedFrags.fastq.gz
[FLASH]     ./R32.notCombined_1.fastq.gz
[FLASH]     ./R32.notCombined_2.fastq.gz
[FLASH]     ./R32.hist
[FLASH]     ./R32.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1639 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1639
[FLASH]     Combined pairs:   328
[FLASH]     Uncombined pairs: 1311
[FLASH]     Percent combined: 20.01%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.016 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

328 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R32/R32_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R32_centroids.fasta --uc R32.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 328 sequences and identified 263 clusters (clustering threshold 1.00) - 5 clusters (representing 44 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R34 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R34_1.fastq /home/working/FinalCorrected/demultiplexed/R34_2.fastq R34_forward.paired.fastq.gz R34_forward.singletons.fastq.gz R34_reverse.paired.fastq.gz R34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R34_1.fastq /home/working/FinalCorrected/demultiplexed/R34_2.fastq R34_forward.paired.fastq.gz R34_forward.singletons.fastq.gz R34_reverse.paired.fastq.gz R34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:45:47 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R34/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9121 Both Surviving: 2265 (24.83%) Forward Only Surviving: 636 (6.97%) Reverse Only Surviving: 93 (1.02%) Dropped: 6127 (67.17%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: R34_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: R34_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: R34_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: R34_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R34_forward.paired.headclipped.fastq.gz R34_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R34 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R34_forward.paired.headclipped.fastq.gz
[FLASH]     R34_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R34.extendedFrags.fastq.gz
[FLASH]     ./R34.notCombined_1.fastq.gz
[FLASH]     ./R34.notCombined_2.fastq.gz
[FLASH]     ./R34.hist
[FLASH]     ./R34.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2265 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2265
[FLASH]     Combined pairs:   74
[FLASH]     Uncombined pairs: 2191
[FLASH]     Percent combined: 3.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.015 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

74 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R34/R34_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R34_centroids.fasta --uc R34.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 74 sequences and identified 66 clusters (clustering threshold 1.00) - 1 clusters (representing 7 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R36 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R36_1.fastq /home/working/FinalCorrected/demultiplexed/R36_2.fastq R36_forward.paired.fastq.gz R36_forward.singletons.fastq.gz R36_reverse.paired.fastq.gz R36_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R36_1.fastq /home/working/FinalCorrected/demultiplexed/R36_2.fastq R36_forward.paired.fastq.gz R36_forward.singletons.fastq.gz R36_reverse.paired.fastq.gz R36_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:45:49 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R36/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 12398 Both Surviving: 2888 (23.29%) Forward Only Surviving: 1053 (8.49%) Reverse Only Surviving: 127 (1.02%) Dropped: 8330 (67.19%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: R36_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: R36_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R36_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R36_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R36_forward.paired.headclipped.fastq.gz R36_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R36 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R36_forward.paired.headclipped.fastq.gz
[FLASH]     R36_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R36.extendedFrags.fastq.gz
[FLASH]     ./R36.notCombined_1.fastq.gz
[FLASH]     ./R36.notCombined_2.fastq.gz
[FLASH]     ./R36.hist
[FLASH]     ./R36.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2888 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2888
[FLASH]     Combined pairs:   2562
[FLASH]     Uncombined pairs: 326
[FLASH]     Percent combined: 88.71%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.069 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2562 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R36/R36_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R36_centroids.fasta --uc R36.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2562 sequences and identified 1528 clusters (clustering threshold 1.00) - 16 clusters (representing 849 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R39 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R39_1.fastq /home/working/FinalCorrected/demultiplexed/R39_2.fastq R39_forward.paired.fastq.gz R39_forward.singletons.fastq.gz R39_reverse.paired.fastq.gz R39_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R39_1.fastq /home/working/FinalCorrected/demultiplexed/R39_2.fastq R39_forward.paired.fastq.gz R39_forward.singletons.fastq.gz R39_reverse.paired.fastq.gz R39_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:45:56 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R39/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 30931 Both Surviving: 8970 (29.00%) Forward Only Surviving: 3918 (12.67%) Reverse Only Surviving: 373 (1.21%) Dropped: 17670 (57.13%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: R39_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: R39_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: R39_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: R39_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R39_forward.paired.headclipped.fastq.gz R39_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R39 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R39_forward.paired.headclipped.fastq.gz
[FLASH]     R39_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R39.extendedFrags.fastq.gz
[FLASH]     ./R39.notCombined_1.fastq.gz
[FLASH]     ./R39.notCombined_2.fastq.gz
[FLASH]     ./R39.hist
[FLASH]     ./R39.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8970 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8970
[FLASH]     Combined pairs:   3486
[FLASH]     Uncombined pairs: 5484
[FLASH]     Percent combined: 38.86%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.100 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3486 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R39/R39_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R39_centroids.fasta --uc R39.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3486 sequences and identified 1707 clusters (clustering threshold 1.00) - 62 clusters (representing 1535 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R4 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R4_1.fastq /home/working/FinalCorrected/demultiplexed/R4_2.fastq R4_forward.paired.fastq.gz R4_forward.singletons.fastq.gz R4_reverse.paired.fastq.gz R4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R4_1.fastq /home/working/FinalCorrected/demultiplexed/R4_2.fastq R4_forward.paired.fastq.gz R4_forward.singletons.fastq.gz R4_reverse.paired.fastq.gz R4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:46:06 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R4/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 12106 Both Surviving: 3362 (27.77%) Forward Only Surviving: 786 (6.49%) Reverse Only Surviving: 225 (1.86%) Dropped: 7733 (63.88%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: R4_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: R4_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R4_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R4_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R4_forward.paired.headclipped.fastq.gz R4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R4 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R4_forward.paired.headclipped.fastq.gz
[FLASH]     R4_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R4.extendedFrags.fastq.gz
[FLASH]     ./R4.notCombined_1.fastq.gz
[FLASH]     ./R4.notCombined_2.fastq.gz
[FLASH]     ./R4.hist
[FLASH]     ./R4.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3362 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3362
[FLASH]     Combined pairs:   1701
[FLASH]     Uncombined pairs: 1661
[FLASH]     Percent combined: 50.59%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.053 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1701 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R4/R4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R4_centroids.fasta --uc R4.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1701 sequences and identified 1031 clusters (clustering threshold 1.00) - 31 clusters (representing 573 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R40 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R40_1.fastq /home/working/FinalCorrected/demultiplexed/R40_2.fastq R40_forward.paired.fastq.gz R40_forward.singletons.fastq.gz R40_reverse.paired.fastq.gz R40_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R40_1.fastq /home/working/FinalCorrected/demultiplexed/R40_2.fastq R40_forward.paired.fastq.gz R40_forward.singletons.fastq.gz R40_reverse.paired.fastq.gz R40_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:46:10 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R40/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 10491 Both Surviving: 1666 (15.88%) Forward Only Surviving: 412 (3.93%) Reverse Only Surviving: 153 (1.46%) Dropped: 8260 (78.73%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: R40_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: R40_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R40_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R40_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R40_forward.paired.headclipped.fastq.gz R40_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R40 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R40_forward.paired.headclipped.fastq.gz
[FLASH]     R40_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R40.extendedFrags.fastq.gz
[FLASH]     ./R40.notCombined_1.fastq.gz
[FLASH]     ./R40.notCombined_2.fastq.gz
[FLASH]     ./R40.hist
[FLASH]     ./R40.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1666 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1666
[FLASH]     Combined pairs:   349
[FLASH]     Uncombined pairs: 1317
[FLASH]     Percent combined: 20.95%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.014 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

349 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R40/R40_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R40_centroids.fasta --uc R40.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 349 sequences and identified 246 clusters (clustering threshold 1.00) - 3 clusters (representing 91 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R43 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R43_1.fastq /home/working/FinalCorrected/demultiplexed/R43_2.fastq R43_forward.paired.fastq.gz R43_forward.singletons.fastq.gz R43_reverse.paired.fastq.gz R43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R43_1.fastq /home/working/FinalCorrected/demultiplexed/R43_2.fastq R43_forward.paired.fastq.gz R43_forward.singletons.fastq.gz R43_reverse.paired.fastq.gz R43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:46:17 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R43/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 102059 Both Surviving: 26955 (26.41%) Forward Only Surviving: 7384 (7.24%) Reverse Only Surviving: 1624 (1.59%) Dropped: 66096 (64.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: R43_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: R43_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: R43_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: R43_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R43_forward.paired.headclipped.fastq.gz R43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R43 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R43_forward.paired.headclipped.fastq.gz
[FLASH]     R43_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R43.extendedFrags.fastq.gz
[FLASH]     ./R43.notCombined_1.fastq.gz
[FLASH]     ./R43.notCombined_2.fastq.gz
[FLASH]     ./R43.hist
[FLASH]     ./R43.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 26955 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      26955
[FLASH]     Combined pairs:   5230
[FLASH]     Uncombined pairs: 21725
[FLASH]     Percent combined: 19.40%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.201 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5230 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R43/R43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R43_centroids.fasta --uc R43.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5230 sequences and identified 2691 clusters (clustering threshold 1.00) - 84 clusters (representing 2203 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R44 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R44_1.fastq /home/working/FinalCorrected/demultiplexed/R44_2.fastq R44_forward.paired.fastq.gz R44_forward.singletons.fastq.gz R44_reverse.paired.fastq.gz R44_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R44_1.fastq /home/working/FinalCorrected/demultiplexed/R44_2.fastq R44_forward.paired.fastq.gz R44_forward.singletons.fastq.gz R44_reverse.paired.fastq.gz R44_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:46:40 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R44/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 59807 Both Surviving: 3915 (6.55%) Forward Only Surviving: 1319 (2.21%) Reverse Only Surviving: 818 (1.37%) Dropped: 53755 (89.88%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R44_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R44_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: R44_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: R44_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R44_forward.paired.headclipped.fastq.gz R44_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R44 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R44_forward.paired.headclipped.fastq.gz
[FLASH]     R44_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R44.extendedFrags.fastq.gz
[FLASH]     ./R44.notCombined_1.fastq.gz
[FLASH]     ./R44.notCombined_2.fastq.gz
[FLASH]     ./R44.hist
[FLASH]     ./R44.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3915 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3915
[FLASH]     Combined pairs:   1308
[FLASH]     Uncombined pairs: 2607
[FLASH]     Percent combined: 33.41%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.037 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1308 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R44/R44_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R44_centroids.fasta --uc R44.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1308 sequences and identified 566 clusters (clustering threshold 1.00) - 13 clusters (representing 617 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R45 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R45_1.fastq /home/working/FinalCorrected/demultiplexed/R45_2.fastq R45_forward.paired.fastq.gz R45_forward.singletons.fastq.gz R45_reverse.paired.fastq.gz R45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R45_1.fastq /home/working/FinalCorrected/demultiplexed/R45_2.fastq R45_forward.paired.fastq.gz R45_forward.singletons.fastq.gz R45_reverse.paired.fastq.gz R45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:46:49 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R45/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 52759 Both Surviving: 17663 (33.48%) Forward Only Surviving: 10402 (19.72%) Reverse Only Surviving: 418 (0.79%) Dropped: 24276 (46.01%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R45_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R45_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: R45_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: R45_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R45_forward.paired.headclipped.fastq.gz R45_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R45 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R45_forward.paired.headclipped.fastq.gz
[FLASH]     R45_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R45.extendedFrags.fastq.gz
[FLASH]     ./R45.notCombined_1.fastq.gz
[FLASH]     ./R45.notCombined_2.fastq.gz
[FLASH]     ./R45.hist
[FLASH]     ./R45.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 17663 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      17663
[FLASH]     Combined pairs:   14829
[FLASH]     Uncombined pairs: 2834
[FLASH]     Percent combined: 83.96%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.397 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

14829 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R45/R45_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R45_centroids.fasta --uc R45.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 14829 sequences and identified 6262 clusters (clustering threshold 1.00) - 316 clusters (representing 8149 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R48 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R48_1.fastq /home/working/FinalCorrected/demultiplexed/R48_2.fastq R48_forward.paired.fastq.gz R48_forward.singletons.fastq.gz R48_reverse.paired.fastq.gz R48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R48_1.fastq /home/working/FinalCorrected/demultiplexed/R48_2.fastq R48_forward.paired.fastq.gz R48_forward.singletons.fastq.gz R48_reverse.paired.fastq.gz R48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:47:18 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R48/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20979 Both Surviving: 5392 (25.70%) Forward Only Surviving: 1126 (5.37%) Reverse Only Surviving: 316 (1.51%) Dropped: 14145 (67.42%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: R48_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: R48_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: R48_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: R48_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R48_forward.paired.headclipped.fastq.gz R48_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R48 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R48_forward.paired.headclipped.fastq.gz
[FLASH]     R48_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R48.extendedFrags.fastq.gz
[FLASH]     ./R48.notCombined_1.fastq.gz
[FLASH]     ./R48.notCombined_2.fastq.gz
[FLASH]     ./R48.hist
[FLASH]     ./R48.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5392 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5392
[FLASH]     Combined pairs:   2042
[FLASH]     Uncombined pairs: 3350
[FLASH]     Percent combined: 37.87%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.063 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2042 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R48/R48_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R48_centroids.fasta --uc R48.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2042 sequences and identified 1303 clusters (clustering threshold 1.00) - 26 clusters (representing 655 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R49 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R49_1.fastq /home/working/FinalCorrected/demultiplexed/R49_2.fastq R49_forward.paired.fastq.gz R49_forward.singletons.fastq.gz R49_reverse.paired.fastq.gz R49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R49_1.fastq /home/working/FinalCorrected/demultiplexed/R49_2.fastq R49_forward.paired.fastq.gz R49_forward.singletons.fastq.gz R49_reverse.paired.fastq.gz R49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:47:25 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R49/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9114 Both Surviving: 1505 (16.51%) Forward Only Surviving: 341 (3.74%) Reverse Only Surviving: 295 (3.24%) Dropped: 6973 (76.51%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: R49_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: R49_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R49_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R49_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R49_forward.paired.headclipped.fastq.gz R49_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R49 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R49_forward.paired.headclipped.fastq.gz
[FLASH]     R49_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R49.extendedFrags.fastq.gz
[FLASH]     ./R49.notCombined_1.fastq.gz
[FLASH]     ./R49.notCombined_2.fastq.gz
[FLASH]     ./R49.hist
[FLASH]     ./R49.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1505 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1505
[FLASH]     Combined pairs:   358
[FLASH]     Uncombined pairs: 1147
[FLASH]     Percent combined: 23.79%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.013 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

358 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R49/R49_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R49_centroids.fasta --uc R49.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 358 sequences and identified 195 clusters (clustering threshold 1.00) - 6 clusters (representing 156 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R59 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R59_1.fastq /home/working/FinalCorrected/demultiplexed/R59_2.fastq R59_forward.paired.fastq.gz R59_forward.singletons.fastq.gz R59_reverse.paired.fastq.gz R59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R59_1.fastq /home/working/FinalCorrected/demultiplexed/R59_2.fastq R59_forward.paired.fastq.gz R59_forward.singletons.fastq.gz R59_reverse.paired.fastq.gz R59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:47:28 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R59/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 35926 Both Surviving: 3298 (9.18%) Forward Only Surviving: 970 (2.70%) Reverse Only Surviving: 529 (1.47%) Dropped: 31129 (86.65%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R59_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R59_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: R59_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: R59_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R59_forward.paired.headclipped.fastq.gz R59_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R59 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R59_forward.paired.headclipped.fastq.gz
[FLASH]     R59_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R59.extendedFrags.fastq.gz
[FLASH]     ./R59.notCombined_1.fastq.gz
[FLASH]     ./R59.notCombined_2.fastq.gz
[FLASH]     ./R59.hist
[FLASH]     ./R59.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3298 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3298
[FLASH]     Combined pairs:   129
[FLASH]     Uncombined pairs: 3169
[FLASH]     Percent combined: 3.91%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.020 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

129 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R59/R59_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R59_centroids.fasta --uc R59.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 129 sequences and identified 87 clusters (clustering threshold 1.00) - 3 clusters (representing 34 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R60 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R60_1.fastq /home/working/FinalCorrected/demultiplexed/R60_2.fastq R60_forward.paired.fastq.gz R60_forward.singletons.fastq.gz R60_reverse.paired.fastq.gz R60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R60_1.fastq /home/working/FinalCorrected/demultiplexed/R60_2.fastq R60_forward.paired.fastq.gz R60_forward.singletons.fastq.gz R60_reverse.paired.fastq.gz R60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:47:33 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R60/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 39486 Both Surviving: 5392 (13.66%) Forward Only Surviving: 1158 (2.93%) Reverse Only Surviving: 443 (1.12%) Dropped: 32493 (82.29%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: R60_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: R60_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R60_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R60_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R60_forward.paired.headclipped.fastq.gz R60_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R60 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R60_forward.paired.headclipped.fastq.gz
[FLASH]     R60_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R60.extendedFrags.fastq.gz
[FLASH]     ./R60.notCombined_1.fastq.gz
[FLASH]     ./R60.notCombined_2.fastq.gz
[FLASH]     ./R60.hist
[FLASH]     ./R60.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5392 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5392
[FLASH]     Combined pairs:   47
[FLASH]     Uncombined pairs: 5345
[FLASH]     Percent combined: 0.87%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.036 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

47 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R60/R60_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R60_centroids.fasta --uc R60.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 47 sequences and identified 42 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R67 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R67_1.fastq /home/working/FinalCorrected/demultiplexed/R67_2.fastq R67_forward.paired.fastq.gz R67_forward.singletons.fastq.gz R67_reverse.paired.fastq.gz R67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R67_1.fastq /home/working/FinalCorrected/demultiplexed/R67_2.fastq R67_forward.paired.fastq.gz R67_forward.singletons.fastq.gz R67_reverse.paired.fastq.gz R67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:47:39 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R67/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 27457 Both Surviving: 6301 (22.95%) Forward Only Surviving: 4040 (14.71%) Reverse Only Surviving: 307 (1.12%) Dropped: 16809 (61.22%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: R67_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: R67_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: R67_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: R67_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R67_forward.paired.headclipped.fastq.gz R67_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R67 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R67_forward.paired.headclipped.fastq.gz
[FLASH]     R67_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R67.extendedFrags.fastq.gz
[FLASH]     ./R67.notCombined_1.fastq.gz
[FLASH]     ./R67.notCombined_2.fastq.gz
[FLASH]     ./R67.hist
[FLASH]     ./R67.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6301 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6301
[FLASH]     Combined pairs:   4584
[FLASH]     Uncombined pairs: 1717
[FLASH]     Percent combined: 72.75%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.124 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4584 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R67/R67_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R67_centroids.fasta --uc R67.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4584 sequences and identified 2358 clusters (clustering threshold 1.00) - 60 clusters (representing 1875 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R68 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R68_1.fastq /home/working/FinalCorrected/demultiplexed/R68_2.fastq R68_forward.paired.fastq.gz R68_forward.singletons.fastq.gz R68_reverse.paired.fastq.gz R68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R68_1.fastq /home/working/FinalCorrected/demultiplexed/R68_2.fastq R68_forward.paired.fastq.gz R68_forward.singletons.fastq.gz R68_reverse.paired.fastq.gz R68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:47:49 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R68/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 12701 Both Surviving: 2577 (20.29%) Forward Only Surviving: 1071 (8.43%) Reverse Only Surviving: 201 (1.58%) Dropped: 8852 (69.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: R68_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: R68_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R68_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R68_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R68_forward.paired.headclipped.fastq.gz R68_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R68 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R68_forward.paired.headclipped.fastq.gz
[FLASH]     R68_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R68.extendedFrags.fastq.gz
[FLASH]     ./R68.notCombined_1.fastq.gz
[FLASH]     ./R68.notCombined_2.fastq.gz
[FLASH]     ./R68.hist
[FLASH]     ./R68.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2577 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2577
[FLASH]     Combined pairs:   2078
[FLASH]     Uncombined pairs: 499
[FLASH]     Percent combined: 80.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.056 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2078 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R68/R68_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R68_centroids.fasta --uc R68.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2078 sequences and identified 912 clusters (clustering threshold 1.00) - 34 clusters (representing 1014 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R71 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R71_1.fastq /home/working/FinalCorrected/demultiplexed/R71_2.fastq R71_forward.paired.fastq.gz R71_forward.singletons.fastq.gz R71_reverse.paired.fastq.gz R71_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R71_1.fastq /home/working/FinalCorrected/demultiplexed/R71_2.fastq R71_forward.paired.fastq.gz R71_forward.singletons.fastq.gz R71_reverse.paired.fastq.gz R71_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:47:55 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R71/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 35268 Both Surviving: 4575 (12.97%) Forward Only Surviving: 1404 (3.98%) Reverse Only Surviving: 472 (1.34%) Dropped: 28817 (81.71%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: R71_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: R71_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R71_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R71_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R71_forward.paired.headclipped.fastq.gz R71_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R71 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R71_forward.paired.headclipped.fastq.gz
[FLASH]     R71_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R71.extendedFrags.fastq.gz
[FLASH]     ./R71.notCombined_1.fastq.gz
[FLASH]     ./R71.notCombined_2.fastq.gz
[FLASH]     ./R71.hist
[FLASH]     ./R71.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4575 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4575
[FLASH]     Combined pairs:   262
[FLASH]     Uncombined pairs: 4313
[FLASH]     Percent combined: 5.73%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.028 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

262 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R71/R71_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R71_centroids.fasta --uc R71.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 262 sequences and identified 189 clusters (clustering threshold 1.00) - 7 clusters (representing 55 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R74 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R74_1.fastq /home/working/FinalCorrected/demultiplexed/R74_2.fastq R74_forward.paired.fastq.gz R74_forward.singletons.fastq.gz R74_reverse.paired.fastq.gz R74_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R74_1.fastq /home/working/FinalCorrected/demultiplexed/R74_2.fastq R74_forward.paired.fastq.gz R74_forward.singletons.fastq.gz R74_reverse.paired.fastq.gz R74_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:48:01 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R74/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37447 Both Surviving: 185 (0.49%) Forward Only Surviving: 449 (1.20%) Reverse Only Surviving: 251 (0.67%) Dropped: 36562 (97.64%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: R74_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: R74_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R74_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R74_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R74_forward.paired.headclipped.fastq.gz R74_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R74 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R74_forward.paired.headclipped.fastq.gz
[FLASH]     R74_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R74.extendedFrags.fastq.gz
[FLASH]     ./R74.notCombined_1.fastq.gz
[FLASH]     ./R74.notCombined_2.fastq.gz
[FLASH]     ./R74.hist
[FLASH]     ./R74.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 185 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      185
[FLASH]     Combined pairs:   128
[FLASH]     Uncombined pairs: 57
[FLASH]     Percent combined: 69.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.005 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

128 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R74/R74_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R74_centroids.fasta --uc R74.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 128 sequences and identified 73 clusters (clustering threshold 1.00) - 3 clusters (representing 52 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R8 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R8_1.fastq /home/working/FinalCorrected/demultiplexed/R8_2.fastq R8_forward.paired.fastq.gz R8_forward.singletons.fastq.gz R8_reverse.paired.fastq.gz R8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R8_1.fastq /home/working/FinalCorrected/demultiplexed/R8_2.fastq R8_forward.paired.fastq.gz R8_forward.singletons.fastq.gz R8_reverse.paired.fastq.gz R8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:48:05 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R8/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29705 Both Surviving: 2428 (8.17%) Forward Only Surviving: 762 (2.57%) Reverse Only Surviving: 299 (1.01%) Dropped: 26216 (88.25%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R8_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R8_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: R8_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: R8_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R8_forward.paired.headclipped.fastq.gz R8_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R8 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R8_forward.paired.headclipped.fastq.gz
[FLASH]     R8_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R8.extendedFrags.fastq.gz
[FLASH]     ./R8.notCombined_1.fastq.gz
[FLASH]     ./R8.notCombined_2.fastq.gz
[FLASH]     ./R8.hist
[FLASH]     ./R8.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2428 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2428
[FLASH]     Combined pairs:   2143
[FLASH]     Uncombined pairs: 285
[FLASH]     Percent combined: 88.26%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.059 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2143 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R8/R8_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R8_centroids.fasta --uc R8.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2143 sequences and identified 1180 clusters (clustering threshold 1.00) - 27 clusters (representing 850 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R85 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R85_1.fastq /home/working/FinalCorrected/demultiplexed/R85_2.fastq R85_forward.paired.fastq.gz R85_forward.singletons.fastq.gz R85_reverse.paired.fastq.gz R85_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R85_1.fastq /home/working/FinalCorrected/demultiplexed/R85_2.fastq R85_forward.paired.fastq.gz R85_forward.singletons.fastq.gz R85_reverse.paired.fastq.gz R85_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:48:11 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R85/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18834 Both Surviving: 3948 (20.96%) Forward Only Surviving: 1269 (6.74%) Reverse Only Surviving: 242 (1.28%) Dropped: 13375 (71.02%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: R85_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: R85_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: R85_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: R85_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R85_forward.paired.headclipped.fastq.gz R85_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R85 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R85_forward.paired.headclipped.fastq.gz
[FLASH]     R85_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R85.extendedFrags.fastq.gz
[FLASH]     ./R85.notCombined_1.fastq.gz
[FLASH]     ./R85.notCombined_2.fastq.gz
[FLASH]     ./R85.hist
[FLASH]     ./R85.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3948 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3948
[FLASH]     Combined pairs:   1219
[FLASH]     Uncombined pairs: 2729
[FLASH]     Percent combined: 30.88%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.040 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1219 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R85/R85_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R85_centroids.fasta --uc R85.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1219 sequences and identified 733 clusters (clustering threshold 1.00) - 16 clusters (representing 439 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R9 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R9_1.fastq /home/working/FinalCorrected/demultiplexed/R9_2.fastq R9_forward.paired.fastq.gz R9_forward.singletons.fastq.gz R9_reverse.paired.fastq.gz R9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R9_1.fastq /home/working/FinalCorrected/demultiplexed/R9_2.fastq R9_forward.paired.fastq.gz R9_forward.singletons.fastq.gz R9_reverse.paired.fastq.gz R9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:48:16 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R9/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22136 Both Surviving: 5623 (25.40%) Forward Only Surviving: 1911 (8.63%) Reverse Only Surviving: 301 (1.36%) Dropped: 14301 (64.61%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: R9_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: R9_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: R9_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: R9_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R9_forward.paired.headclipped.fastq.gz R9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R9 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R9_forward.paired.headclipped.fastq.gz
[FLASH]     R9_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R9.extendedFrags.fastq.gz
[FLASH]     ./R9.notCombined_1.fastq.gz
[FLASH]     ./R9.notCombined_2.fastq.gz
[FLASH]     ./R9.hist
[FLASH]     ./R9.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5623 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5623
[FLASH]     Combined pairs:   251
[FLASH]     Uncombined pairs: 5372
[FLASH]     Percent combined: 4.46%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.036 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

251 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R9/R9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R9_centroids.fasta --uc R9.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 251 sequences and identified 219 clusters (clustering threshold 1.00) - 2 clusters (representing 16 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R90 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R90_1.fastq /home/working/FinalCorrected/demultiplexed/R90_2.fastq R90_forward.paired.fastq.gz R90_forward.singletons.fastq.gz R90_reverse.paired.fastq.gz R90_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R90_1.fastq /home/working/FinalCorrected/demultiplexed/R90_2.fastq R90_forward.paired.fastq.gz R90_forward.singletons.fastq.gz R90_reverse.paired.fastq.gz R90_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:48:20 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R90/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15190 Both Surviving: 328 (2.16%) Forward Only Surviving: 380 (2.50%) Reverse Only Surviving: 160 (1.05%) Dropped: 14322 (94.29%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: R90_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: R90_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: R90_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: R90_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R90_forward.paired.headclipped.fastq.gz R90_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R90 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R90_forward.paired.headclipped.fastq.gz
[FLASH]     R90_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R90.extendedFrags.fastq.gz
[FLASH]     ./R90.notCombined_1.fastq.gz
[FLASH]     ./R90.notCombined_2.fastq.gz
[FLASH]     ./R90.hist
[FLASH]     ./R90.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 328 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      328
[FLASH]     Combined pairs:   231
[FLASH]     Uncombined pairs: 97
[FLASH]     Percent combined: 70.43%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.007 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

231 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R90/R90_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R90_centroids.fasta --uc R90.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 231 sequences and identified 178 clusters (clustering threshold 1.00) - 3 clusters (representing 45 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R95 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R95_1.fastq /home/working/FinalCorrected/demultiplexed/R95_2.fastq R95_forward.paired.fastq.gz R95_forward.singletons.fastq.gz R95_reverse.paired.fastq.gz R95_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R95_1.fastq /home/working/FinalCorrected/demultiplexed/R95_2.fastq R95_forward.paired.fastq.gz R95_forward.singletons.fastq.gz R95_reverse.paired.fastq.gz R95_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:48:22 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R95/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15768 Both Surviving: 300 (1.90%) Forward Only Surviving: 279 (1.77%) Reverse Only Surviving: 149 (0.94%) Dropped: 15040 (95.38%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: R95_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: R95_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: R95_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: R95_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R95_forward.paired.headclipped.fastq.gz R95_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R95 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R95_forward.paired.headclipped.fastq.gz
[FLASH]     R95_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R95.extendedFrags.fastq.gz
[FLASH]     ./R95.notCombined_1.fastq.gz
[FLASH]     ./R95.notCombined_2.fastq.gz
[FLASH]     ./R95.hist
[FLASH]     ./R95.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 300 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      300
[FLASH]     Combined pairs:   60
[FLASH]     Uncombined pairs: 240
[FLASH]     Percent combined: 20.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

60 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R95/R95_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R95_centroids.fasta --uc R95.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 60 sequences and identified 53 clusters (clustering threshold 1.00) - 1 clusters (representing 8 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R98 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R98_1.fastq /home/working/FinalCorrected/demultiplexed/R98_2.fastq R98_forward.paired.fastq.gz R98_forward.singletons.fastq.gz R98_reverse.paired.fastq.gz R98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R98_1.fastq /home/working/FinalCorrected/demultiplexed/R98_2.fastq R98_forward.paired.fastq.gz R98_forward.singletons.fastq.gz R98_reverse.paired.fastq.gz R98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:48:27 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R98/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 47670 Both Surviving: 8088 (16.97%) Forward Only Surviving: 2258 (4.74%) Reverse Only Surviving: 560 (1.17%) Dropped: 36764 (77.12%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: R98_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: R98_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: R98_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: R98_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R98_forward.paired.headclipped.fastq.gz R98_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R98 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R98_forward.paired.headclipped.fastq.gz
[FLASH]     R98_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R98.extendedFrags.fastq.gz
[FLASH]     ./R98.notCombined_1.fastq.gz
[FLASH]     ./R98.notCombined_2.fastq.gz
[FLASH]     ./R98.hist
[FLASH]     ./R98.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8088 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8088
[FLASH]     Combined pairs:   3682
[FLASH]     Uncombined pairs: 4406
[FLASH]     Percent combined: 45.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.108 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3682 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R98/R98_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R98_centroids.fasta --uc R98.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3682 sequences and identified 1948 clusters (clustering threshold 1.00) - 49 clusters (representing 1523 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: R99 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R99_1.fastq /home/working/FinalCorrected/demultiplexed/R99_2.fastq R99_forward.paired.fastq.gz R99_forward.singletons.fastq.gz R99_reverse.paired.fastq.gz R99_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/R99_1.fastq /home/working/FinalCorrected/demultiplexed/R99_2.fastq R99_forward.paired.fastq.gz R99_forward.singletons.fastq.gz R99_reverse.paired.fastq.gz R99_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:48:41 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/R99/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 82372 Both Surviving: 1529 (1.86%) Forward Only Surviving: 2684 (3.26%) Reverse Only Surviving: 882 (1.07%) Dropped: 77277 (93.81%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: R99_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: R99_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: R99_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: R99_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash R99_forward.paired.headclipped.fastq.gz R99_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R99 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     R99_forward.paired.headclipped.fastq.gz
[FLASH]     R99_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./R99.extendedFrags.fastq.gz
[FLASH]     ./R99.notCombined_1.fastq.gz
[FLASH]     ./R99.notCombined_2.fastq.gz
[FLASH]     ./R99.hist
[FLASH]     ./R99.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1529 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1529
[FLASH]     Combined pairs:   736
[FLASH]     Uncombined pairs: 793
[FLASH]     Percent combined: 48.14%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.018 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

736 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/R99/R99_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R99_centroids.fasta --uc R99.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 736 sequences and identified 559 clusters (clustering threshold 1.00) - 10 clusters (representing 117 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S100 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S100_1.fastq /home/working/FinalCorrected/demultiplexed/S100_2.fastq S100_forward.paired.fastq.gz S100_forward.singletons.fastq.gz S100_reverse.paired.fastq.gz S100_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S100_1.fastq /home/working/FinalCorrected/demultiplexed/S100_2.fastq S100_forward.paired.fastq.gz S100_forward.singletons.fastq.gz S100_reverse.paired.fastq.gz S100_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:48:47 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S100/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15016 Both Surviving: 1579 (10.52%) Forward Only Surviving: 676 (4.50%) Reverse Only Surviving: 115 (0.77%) Dropped: 12646 (84.22%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S100_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S100_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: S100_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: S100_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S100_forward.paired.headclipped.fastq.gz S100_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S100 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S100_forward.paired.headclipped.fastq.gz
[FLASH]     S100_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S100.extendedFrags.fastq.gz
[FLASH]     ./S100.notCombined_1.fastq.gz
[FLASH]     ./S100.notCombined_2.fastq.gz
[FLASH]     ./S100.hist
[FLASH]     ./S100.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1579 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1579
[FLASH]     Combined pairs:   1345
[FLASH]     Uncombined pairs: 234
[FLASH]     Percent combined: 85.18%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.038 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1345 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S100/S100_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S100_centroids.fasta --uc S100.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1345 sequences and identified 747 clusters (clustering threshold 1.00) - 22 clusters (representing 525 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S101 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S101_1.fastq /home/working/FinalCorrected/demultiplexed/S101_2.fastq S101_forward.paired.fastq.gz S101_forward.singletons.fastq.gz S101_reverse.paired.fastq.gz S101_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S101_1.fastq /home/working/FinalCorrected/demultiplexed/S101_2.fastq S101_forward.paired.fastq.gz S101_forward.singletons.fastq.gz S101_reverse.paired.fastq.gz S101_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:48:50 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S101/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9613 Both Surviving: 328 (3.41%) Forward Only Surviving: 199 (2.07%) Reverse Only Surviving: 243 (2.53%) Dropped: 8843 (91.99%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: S101_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: S101_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S101_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S101_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S101_forward.paired.headclipped.fastq.gz S101_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S101 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S101_forward.paired.headclipped.fastq.gz
[FLASH]     S101_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S101.extendedFrags.fastq.gz
[FLASH]     ./S101.notCombined_1.fastq.gz
[FLASH]     ./S101.notCombined_2.fastq.gz
[FLASH]     ./S101.hist
[FLASH]     ./S101.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 328 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      328
[FLASH]     Combined pairs:   59
[FLASH]     Uncombined pairs: 269
[FLASH]     Percent combined: 17.99%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

59 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S101/S101_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S101_centroids.fasta --uc S101.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 59 sequences and identified 50 clusters (clustering threshold 1.00) - 1 clusters (representing 10 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S102 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S102_1.fastq /home/working/FinalCorrected/demultiplexed/S102_2.fastq S102_forward.paired.fastq.gz S102_forward.singletons.fastq.gz S102_reverse.paired.fastq.gz S102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S102_1.fastq /home/working/FinalCorrected/demultiplexed/S102_2.fastq S102_forward.paired.fastq.gz S102_forward.singletons.fastq.gz S102_reverse.paired.fastq.gz S102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:48:52 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S102/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17129 Both Surviving: 3440 (20.08%) Forward Only Surviving: 1158 (6.76%) Reverse Only Surviving: 149 (0.87%) Dropped: 12382 (72.29%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: S102_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: S102_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: S102_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: S102_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S102_forward.paired.headclipped.fastq.gz S102_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S102 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S102_forward.paired.headclipped.fastq.gz
[FLASH]     S102_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S102.extendedFrags.fastq.gz
[FLASH]     ./S102.notCombined_1.fastq.gz
[FLASH]     ./S102.notCombined_2.fastq.gz
[FLASH]     ./S102.hist
[FLASH]     ./S102.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3440 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3440
[FLASH]     Combined pairs:   1551
[FLASH]     Uncombined pairs: 1889
[FLASH]     Percent combined: 45.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.048 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1551 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S102/S102_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S102_centroids.fasta --uc S102.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1551 sequences and identified 816 clusters (clustering threshold 1.00) - 14 clusters (representing 650 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S11 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S11_1.fastq /home/working/FinalCorrected/demultiplexed/S11_2.fastq S11_forward.paired.fastq.gz S11_forward.singletons.fastq.gz S11_reverse.paired.fastq.gz S11_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S11_1.fastq /home/working/FinalCorrected/demultiplexed/S11_2.fastq S11_forward.paired.fastq.gz S11_forward.singletons.fastq.gz S11_reverse.paired.fastq.gz S11_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:48:59 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S11/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41543 Both Surviving: 671 (1.62%) Forward Only Surviving: 1052 (2.53%) Reverse Only Surviving: 567 (1.36%) Dropped: 39253 (94.49%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: S11_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: S11_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S11_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S11_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S11_forward.paired.headclipped.fastq.gz S11_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S11 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S11_forward.paired.headclipped.fastq.gz
[FLASH]     S11_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S11.extendedFrags.fastq.gz
[FLASH]     ./S11.notCombined_1.fastq.gz
[FLASH]     ./S11.notCombined_2.fastq.gz
[FLASH]     ./S11.hist
[FLASH]     ./S11.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 671 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      671
[FLASH]     Combined pairs:   345
[FLASH]     Uncombined pairs: 326
[FLASH]     Percent combined: 51.42%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.010 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

345 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S11/S11_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S11_centroids.fasta --uc S11.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 345 sequences and identified 185 clusters (clustering threshold 1.00) - 5 clusters (representing 141 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S13 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S13_1.fastq /home/working/FinalCorrected/demultiplexed/S13_2.fastq S13_forward.paired.fastq.gz S13_forward.singletons.fastq.gz S13_reverse.paired.fastq.gz S13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S13_1.fastq /home/working/FinalCorrected/demultiplexed/S13_2.fastq S13_forward.paired.fastq.gz S13_forward.singletons.fastq.gz S13_reverse.paired.fastq.gz S13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:49:03 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S13/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 21977 Both Surviving: 3585 (16.31%) Forward Only Surviving: 869 (3.95%) Reverse Only Surviving: 356 (1.62%) Dropped: 17167 (78.11%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: S13_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: S13_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: S13_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: S13_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S13_forward.paired.headclipped.fastq.gz S13_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S13 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S13_forward.paired.headclipped.fastq.gz
[FLASH]     S13_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S13.extendedFrags.fastq.gz
[FLASH]     ./S13.notCombined_1.fastq.gz
[FLASH]     ./S13.notCombined_2.fastq.gz
[FLASH]     ./S13.hist
[FLASH]     ./S13.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3585 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3585
[FLASH]     Combined pairs:   904
[FLASH]     Uncombined pairs: 2681
[FLASH]     Percent combined: 25.22%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.029 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

904 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S13/S13_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S13_centroids.fasta --uc S13.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 904 sequences and identified 511 clusters (clustering threshold 1.00) - 9 clusters (representing 347 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S18 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S18_1.fastq /home/working/FinalCorrected/demultiplexed/S18_2.fastq S18_forward.paired.fastq.gz S18_forward.singletons.fastq.gz S18_reverse.paired.fastq.gz S18_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S18_1.fastq /home/working/FinalCorrected/demultiplexed/S18_2.fastq S18_forward.paired.fastq.gz S18_forward.singletons.fastq.gz S18_reverse.paired.fastq.gz S18_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:49:09 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S18/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 45130 Both Surviving: 6734 (14.92%) Forward Only Surviving: 1728 (3.83%) Reverse Only Surviving: 829 (1.84%) Dropped: 35839 (79.41%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: S18_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: S18_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: S18_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: S18_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S18_forward.paired.headclipped.fastq.gz S18_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S18 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S18_forward.paired.headclipped.fastq.gz
[FLASH]     S18_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S18.extendedFrags.fastq.gz
[FLASH]     ./S18.notCombined_1.fastq.gz
[FLASH]     ./S18.notCombined_2.fastq.gz
[FLASH]     ./S18.hist
[FLASH]     ./S18.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6734 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6734
[FLASH]     Combined pairs:   71
[FLASH]     Uncombined pairs: 6663
[FLASH]     Percent combined: 1.05%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.045 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

71 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S18/S18_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S18_centroids.fasta --uc S18.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 71 sequences and identified 61 clusters (clustering threshold 1.00) - 1 clusters (representing 9 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S19 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S19_1.fastq /home/working/FinalCorrected/demultiplexed/S19_2.fastq S19_forward.paired.fastq.gz S19_forward.singletons.fastq.gz S19_reverse.paired.fastq.gz S19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S19_1.fastq /home/working/FinalCorrected/demultiplexed/S19_2.fastq S19_forward.paired.fastq.gz S19_forward.singletons.fastq.gz S19_reverse.paired.fastq.gz S19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:49:14 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S19/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14524 Both Surviving: 3946 (27.17%) Forward Only Surviving: 1547 (10.65%) Reverse Only Surviving: 301 (2.07%) Dropped: 8730 (60.11%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: S19_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: S19_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S19_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S19_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S19_forward.paired.headclipped.fastq.gz S19_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S19 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S19_forward.paired.headclipped.fastq.gz
[FLASH]     S19_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S19.extendedFrags.fastq.gz
[FLASH]     ./S19.notCombined_1.fastq.gz
[FLASH]     ./S19.notCombined_2.fastq.gz
[FLASH]     ./S19.hist
[FLASH]     ./S19.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3946 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3946
[FLASH]     Combined pairs:   2350
[FLASH]     Uncombined pairs: 1596
[FLASH]     Percent combined: 59.55%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.067 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2350 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S19/S19_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S19_centroids.fasta --uc S19.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2350 sequences and identified 1203 clusters (clustering threshold 1.00) - 28 clusters (representing 980 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S24 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S24_1.fastq /home/working/FinalCorrected/demultiplexed/S24_2.fastq S24_forward.paired.fastq.gz S24_forward.singletons.fastq.gz S24_reverse.paired.fastq.gz S24_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S24_1.fastq /home/working/FinalCorrected/demultiplexed/S24_2.fastq S24_forward.paired.fastq.gz S24_forward.singletons.fastq.gz S24_reverse.paired.fastq.gz S24_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:49:20 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S24/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9162 Both Surviving: 875 (9.55%) Forward Only Surviving: 690 (7.53%) Reverse Only Surviving: 70 (0.76%) Dropped: 7527 (82.15%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S24_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S24_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S24_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S24_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S24_forward.paired.headclipped.fastq.gz S24_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S24 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S24_forward.paired.headclipped.fastq.gz
[FLASH]     S24_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S24.extendedFrags.fastq.gz
[FLASH]     ./S24.notCombined_1.fastq.gz
[FLASH]     ./S24.notCombined_2.fastq.gz
[FLASH]     ./S24.hist
[FLASH]     ./S24.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 875 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      875
[FLASH]     Combined pairs:   288
[FLASH]     Uncombined pairs: 587
[FLASH]     Percent combined: 32.91%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.008 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

288 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S24/S24_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S24_centroids.fasta --uc S24.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 288 sequences and identified 240 clusters (clustering threshold 1.00) - 2 clusters (representing 24 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S30 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S30_1.fastq /home/working/FinalCorrected/demultiplexed/S30_2.fastq S30_forward.paired.fastq.gz S30_forward.singletons.fastq.gz S30_reverse.paired.fastq.gz S30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S30_1.fastq /home/working/FinalCorrected/demultiplexed/S30_2.fastq S30_forward.paired.fastq.gz S30_forward.singletons.fastq.gz S30_reverse.paired.fastq.gz S30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:49:23 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S30/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11465 Both Surviving: 2208 (19.26%) Forward Only Surviving: 583 (5.09%) Reverse Only Surviving: 164 (1.43%) Dropped: 8510 (74.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: S30_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: S30_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S30_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S30_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S30_forward.paired.headclipped.fastq.gz S30_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S30 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S30_forward.paired.headclipped.fastq.gz
[FLASH]     S30_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S30.extendedFrags.fastq.gz
[FLASH]     ./S30.notCombined_1.fastq.gz
[FLASH]     ./S30.notCombined_2.fastq.gz
[FLASH]     ./S30.hist
[FLASH]     ./S30.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2208 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2208
[FLASH]     Combined pairs:   32
[FLASH]     Uncombined pairs: 2176
[FLASH]     Percent combined: 1.45%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.014 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

32 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S30/S30_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S30_centroids.fasta --uc S30.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 32 sequences and identified 29 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S31 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S31_1.fastq /home/working/FinalCorrected/demultiplexed/S31_2.fastq S31_forward.paired.fastq.gz S31_forward.singletons.fastq.gz S31_reverse.paired.fastq.gz S31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S31_1.fastq /home/working/FinalCorrected/demultiplexed/S31_2.fastq S31_forward.paired.fastq.gz S31_forward.singletons.fastq.gz S31_reverse.paired.fastq.gz S31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:49:25 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S31/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 8667 Both Surviving: 111 (1.28%) Forward Only Surviving: 129 (1.49%) Reverse Only Surviving: 100 (1.15%) Dropped: 8327 (96.08%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S31_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S31_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: S31_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: S31_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S31_forward.paired.headclipped.fastq.gz S31_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S31 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S31_forward.paired.headclipped.fastq.gz
[FLASH]     S31_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S31.extendedFrags.fastq.gz
[FLASH]     ./S31.notCombined_1.fastq.gz
[FLASH]     ./S31.notCombined_2.fastq.gz
[FLASH]     ./S31.hist
[FLASH]     ./S31.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 111 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      111
[FLASH]     Combined pairs:   34
[FLASH]     Uncombined pairs: 77
[FLASH]     Percent combined: 30.63%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

34 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S31/S31_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S31_centroids.fasta --uc S31.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 34 sequences and identified 33 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S33 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S33_1.fastq /home/working/FinalCorrected/demultiplexed/S33_2.fastq S33_forward.paired.fastq.gz S33_forward.singletons.fastq.gz S33_reverse.paired.fastq.gz S33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S33_1.fastq /home/working/FinalCorrected/demultiplexed/S33_2.fastq S33_forward.paired.fastq.gz S33_forward.singletons.fastq.gz S33_reverse.paired.fastq.gz S33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:49:27 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S33/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22355 Both Surviving: 761 (3.40%) Forward Only Surviving: 773 (3.46%) Reverse Only Surviving: 235 (1.05%) Dropped: 20586 (92.09%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: S33_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: S33_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: S33_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: S33_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S33_forward.paired.headclipped.fastq.gz S33_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S33 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S33_forward.paired.headclipped.fastq.gz
[FLASH]     S33_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S33.extendedFrags.fastq.gz
[FLASH]     ./S33.notCombined_1.fastq.gz
[FLASH]     ./S33.notCombined_2.fastq.gz
[FLASH]     ./S33.hist
[FLASH]     ./S33.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 761 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      761
[FLASH]     Combined pairs:   432
[FLASH]     Uncombined pairs: 329
[FLASH]     Percent combined: 56.77%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.014 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

432 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S33/S33_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S33_centroids.fasta --uc S33.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 432 sequences and identified 320 clusters (clustering threshold 1.00) - 3 clusters (representing 89 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S35 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S35_1.fastq /home/working/FinalCorrected/demultiplexed/S35_2.fastq S35_forward.paired.fastq.gz S35_forward.singletons.fastq.gz S35_reverse.paired.fastq.gz S35_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S35_1.fastq /home/working/FinalCorrected/demultiplexed/S35_2.fastq S35_forward.paired.fastq.gz S35_forward.singletons.fastq.gz S35_reverse.paired.fastq.gz S35_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:49:34 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S35/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 77091 Both Surviving: 5196 (6.74%) Forward Only Surviving: 3552 (4.61%) Reverse Only Surviving: 988 (1.28%) Dropped: 67355 (87.37%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S35_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S35_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: S35_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: S35_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S35_forward.paired.headclipped.fastq.gz S35_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S35 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S35_forward.paired.headclipped.fastq.gz
[FLASH]     S35_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S35.extendedFrags.fastq.gz
[FLASH]     ./S35.notCombined_1.fastq.gz
[FLASH]     ./S35.notCombined_2.fastq.gz
[FLASH]     ./S35.hist
[FLASH]     ./S35.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5196 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5196
[FLASH]     Combined pairs:   2485
[FLASH]     Uncombined pairs: 2711
[FLASH]     Percent combined: 47.83%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.074 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2485 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S35/S35_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S35_centroids.fasta --uc S35.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2485 sequences and identified 1438 clusters (clustering threshold 1.00) - 40 clusters (representing 905 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S39 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S39_1.fastq /home/working/FinalCorrected/demultiplexed/S39_2.fastq S39_forward.paired.fastq.gz S39_forward.singletons.fastq.gz S39_reverse.paired.fastq.gz S39_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S39_1.fastq /home/working/FinalCorrected/demultiplexed/S39_2.fastq S39_forward.paired.fastq.gz S39_forward.singletons.fastq.gz S39_reverse.paired.fastq.gz S39_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:49:44 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S39/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16769 Both Surviving: 3591 (21.41%) Forward Only Surviving: 1075 (6.41%) Reverse Only Surviving: 215 (1.28%) Dropped: 11888 (70.89%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: S39_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: S39_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S39_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S39_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S39_forward.paired.headclipped.fastq.gz S39_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S39 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S39_forward.paired.headclipped.fastq.gz
[FLASH]     S39_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S39.extendedFrags.fastq.gz
[FLASH]     ./S39.notCombined_1.fastq.gz
[FLASH]     ./S39.notCombined_2.fastq.gz
[FLASH]     ./S39.hist
[FLASH]     ./S39.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3591 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3591
[FLASH]     Combined pairs:   482
[FLASH]     Uncombined pairs: 3109
[FLASH]     Percent combined: 13.42%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.025 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

482 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S39/S39_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S39_centroids.fasta --uc S39.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 482 sequences and identified 345 clusters (clustering threshold 1.00) - 12 clusters (representing 128 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S43 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S43_1.fastq /home/working/FinalCorrected/demultiplexed/S43_2.fastq S43_forward.paired.fastq.gz S43_forward.singletons.fastq.gz S43_reverse.paired.fastq.gz S43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S43_1.fastq /home/working/FinalCorrected/demultiplexed/S43_2.fastq S43_forward.paired.fastq.gz S43_forward.singletons.fastq.gz S43_reverse.paired.fastq.gz S43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:49:48 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S43/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9106 Both Surviving: 1630 (17.90%) Forward Only Surviving: 825 (9.06%) Reverse Only Surviving: 113 (1.24%) Dropped: 6538 (71.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: S43_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: S43_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S43_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S43_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S43_forward.paired.headclipped.fastq.gz S43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S43 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S43_forward.paired.headclipped.fastq.gz
[FLASH]     S43_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S43.extendedFrags.fastq.gz
[FLASH]     ./S43.notCombined_1.fastq.gz
[FLASH]     ./S43.notCombined_2.fastq.gz
[FLASH]     ./S43.hist
[FLASH]     ./S43.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1630 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1630
[FLASH]     Combined pairs:   1138
[FLASH]     Uncombined pairs: 492
[FLASH]     Percent combined: 69.82%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.032 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1138 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S43/S43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S43_centroids.fasta --uc S43.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1138 sequences and identified 567 clusters (clustering threshold 1.00) - 8 clusters (representing 479 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S50 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S50_1.fastq /home/working/FinalCorrected/demultiplexed/S50_2.fastq S50_forward.paired.fastq.gz S50_forward.singletons.fastq.gz S50_reverse.paired.fastq.gz S50_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S50_1.fastq /home/working/FinalCorrected/demultiplexed/S50_2.fastq S50_forward.paired.fastq.gz S50_forward.singletons.fastq.gz S50_reverse.paired.fastq.gz S50_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:49:52 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S50/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23841 Both Surviving: 5537 (23.22%) Forward Only Surviving: 1563 (6.56%) Reverse Only Surviving: 325 (1.36%) Dropped: 16416 (68.86%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: S50_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: S50_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: S50_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: S50_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S50_forward.paired.headclipped.fastq.gz S50_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S50 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S50_forward.paired.headclipped.fastq.gz
[FLASH]     S50_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S50.extendedFrags.fastq.gz
[FLASH]     ./S50.notCombined_1.fastq.gz
[FLASH]     ./S50.notCombined_2.fastq.gz
[FLASH]     ./S50.hist
[FLASH]     ./S50.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5537 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5537
[FLASH]     Combined pairs:   2823
[FLASH]     Uncombined pairs: 2714
[FLASH]     Percent combined: 50.98%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.080 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2823 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S50/S50_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S50_centroids.fasta --uc S50.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2823 sequences and identified 1523 clusters (clustering threshold 1.00) - 37 clusters (representing 1105 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S56 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S56_1.fastq /home/working/FinalCorrected/demultiplexed/S56_2.fastq S56_forward.paired.fastq.gz S56_forward.singletons.fastq.gz S56_reverse.paired.fastq.gz S56_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S56_1.fastq /home/working/FinalCorrected/demultiplexed/S56_2.fastq S56_forward.paired.fastq.gz S56_forward.singletons.fastq.gz S56_reverse.paired.fastq.gz S56_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:49:58 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S56/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: S56_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: S56_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S56_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S56_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S56_forward.paired.headclipped.fastq.gz S56_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S56 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S56_forward.paired.headclipped.fastq.gz
[FLASH]     S56_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S56.extendedFrags.fastq.gz
[FLASH]     ./S56.notCombined_1.fastq.gz
[FLASH]     ./S56.notCombined_2.fastq.gz
[FLASH]     ./S56.hist
[FLASH]     ./S56.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S56/S56_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S56_centroids.fasta --uc S56.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S59 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S59_1.fastq /home/working/FinalCorrected/demultiplexed/S59_2.fastq S59_forward.paired.fastq.gz S59_forward.singletons.fastq.gz S59_reverse.paired.fastq.gz S59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S59_1.fastq /home/working/FinalCorrected/demultiplexed/S59_2.fastq S59_forward.paired.fastq.gz S59_forward.singletons.fastq.gz S59_reverse.paired.fastq.gz S59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:49:59 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S59/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18121 Both Surviving: 3365 (18.57%) Forward Only Surviving: 1089 (6.01%) Reverse Only Surviving: 238 (1.31%) Dropped: 13429 (74.11%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: S59_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: S59_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: S59_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: S59_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S59_forward.paired.headclipped.fastq.gz S59_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S59 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S59_forward.paired.headclipped.fastq.gz
[FLASH]     S59_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S59.extendedFrags.fastq.gz
[FLASH]     ./S59.notCombined_1.fastq.gz
[FLASH]     ./S59.notCombined_2.fastq.gz
[FLASH]     ./S59.hist
[FLASH]     ./S59.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3365 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3365
[FLASH]     Combined pairs:   312
[FLASH]     Uncombined pairs: 3053
[FLASH]     Percent combined: 9.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.022 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

312 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S59/S59_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S59_centroids.fasta --uc S59.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 312 sequences and identified 237 clusters (clustering threshold 1.00) - 4 clusters (representing 65 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S62 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S62_1.fastq /home/working/FinalCorrected/demultiplexed/S62_2.fastq S62_forward.paired.fastq.gz S62_forward.singletons.fastq.gz S62_reverse.paired.fastq.gz S62_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S62_1.fastq /home/working/FinalCorrected/demultiplexed/S62_2.fastq S62_forward.paired.fastq.gz S62_forward.singletons.fastq.gz S62_reverse.paired.fastq.gz S62_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:50:04 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S62/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 39931 Both Surviving: 4343 (10.88%) Forward Only Surviving: 1632 (4.09%) Reverse Only Surviving: 405 (1.01%) Dropped: 33551 (84.02%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: S62_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: S62_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S62_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S62_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S62_forward.paired.headclipped.fastq.gz S62_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S62 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S62_forward.paired.headclipped.fastq.gz
[FLASH]     S62_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S62.extendedFrags.fastq.gz
[FLASH]     ./S62.notCombined_1.fastq.gz
[FLASH]     ./S62.notCombined_2.fastq.gz
[FLASH]     ./S62.hist
[FLASH]     ./S62.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4343 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4343
[FLASH]     Combined pairs:   1611
[FLASH]     Uncombined pairs: 2732
[FLASH]     Percent combined: 37.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.050 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1611 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S62/S62_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S62_centroids.fasta --uc S62.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1611 sequences and identified 697 clusters (clustering threshold 1.00) - 26 clusters (representing 796 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S63 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S63_1.fastq /home/working/FinalCorrected/demultiplexed/S63_2.fastq S63_forward.paired.fastq.gz S63_forward.singletons.fastq.gz S63_reverse.paired.fastq.gz S63_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S63_1.fastq /home/working/FinalCorrected/demultiplexed/S63_2.fastq S63_forward.paired.fastq.gz S63_forward.singletons.fastq.gz S63_reverse.paired.fastq.gz S63_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:50:10 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S63/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 6702 Both Surviving: 610 (9.10%) Forward Only Surviving: 253 (3.77%) Reverse Only Surviving: 80 (1.19%) Dropped: 5759 (85.93%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S63_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S63_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S63_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S63_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S63_forward.paired.headclipped.fastq.gz S63_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S63 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S63_forward.paired.headclipped.fastq.gz
[FLASH]     S63_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S63.extendedFrags.fastq.gz
[FLASH]     ./S63.notCombined_1.fastq.gz
[FLASH]     ./S63.notCombined_2.fastq.gz
[FLASH]     ./S63.hist
[FLASH]     ./S63.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 610 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      610
[FLASH]     Combined pairs:   540
[FLASH]     Uncombined pairs: 70
[FLASH]     Percent combined: 88.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.017 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

540 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S63/S63_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S63_centroids.fasta --uc S63.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 540 sequences and identified 341 clusters (clustering threshold 1.00) - 6 clusters (representing 176 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S65 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S65_1.fastq /home/working/FinalCorrected/demultiplexed/S65_2.fastq S65_forward.paired.fastq.gz S65_forward.singletons.fastq.gz S65_reverse.paired.fastq.gz S65_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S65_1.fastq /home/working/FinalCorrected/demultiplexed/S65_2.fastq S65_forward.paired.fastq.gz S65_forward.singletons.fastq.gz S65_reverse.paired.fastq.gz S65_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:50:12 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S65/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 7804 Both Surviving: 2569 (32.92%) Forward Only Surviving: 1071 (13.72%) Reverse Only Surviving: 88 (1.13%) Dropped: 4076 (52.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S65_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S65_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: S65_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: S65_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S65_forward.paired.headclipped.fastq.gz S65_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S65 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S65_forward.paired.headclipped.fastq.gz
[FLASH]     S65_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S65.extendedFrags.fastq.gz
[FLASH]     ./S65.notCombined_1.fastq.gz
[FLASH]     ./S65.notCombined_2.fastq.gz
[FLASH]     ./S65.hist
[FLASH]     ./S65.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2569 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2569
[FLASH]     Combined pairs:   2216
[FLASH]     Uncombined pairs: 353
[FLASH]     Percent combined: 86.26%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.059 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2216 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S65/S65_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S65_centroids.fasta --uc S65.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2216 sequences and identified 1018 clusters (clustering threshold 1.00) - 46 clusters (representing 1026 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S67 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S67_1.fastq /home/working/FinalCorrected/demultiplexed/S67_2.fastq S67_forward.paired.fastq.gz S67_forward.singletons.fastq.gz S67_reverse.paired.fastq.gz S67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S67_1.fastq /home/working/FinalCorrected/demultiplexed/S67_2.fastq S67_forward.paired.fastq.gz S67_forward.singletons.fastq.gz S67_reverse.paired.fastq.gz S67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:50:17 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S67/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18116 Both Surviving: 4019 (22.18%) Forward Only Surviving: 945 (5.22%) Reverse Only Surviving: 327 (1.81%) Dropped: 12825 (70.79%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: S67_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: S67_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S67_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S67_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S67_forward.paired.headclipped.fastq.gz S67_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S67 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S67_forward.paired.headclipped.fastq.gz
[FLASH]     S67_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S67.extendedFrags.fastq.gz
[FLASH]     ./S67.notCombined_1.fastq.gz
[FLASH]     ./S67.notCombined_2.fastq.gz
[FLASH]     ./S67.hist
[FLASH]     ./S67.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4019 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4019
[FLASH]     Combined pairs:   224
[FLASH]     Uncombined pairs: 3795
[FLASH]     Percent combined: 5.57%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.024 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

224 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S67/S67_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S67_centroids.fasta --uc S67.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 224 sequences and identified 174 clusters (clustering threshold 1.00) - 3 clusters (representing 37 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S7 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S7_1.fastq /home/working/FinalCorrected/demultiplexed/S7_2.fastq S7_forward.paired.fastq.gz S7_forward.singletons.fastq.gz S7_reverse.paired.fastq.gz S7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S7_1.fastq /home/working/FinalCorrected/demultiplexed/S7_2.fastq S7_forward.paired.fastq.gz S7_forward.singletons.fastq.gz S7_reverse.paired.fastq.gz S7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:50:22 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S7/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 32963 Both Surviving: 4645 (14.09%) Forward Only Surviving: 1505 (4.57%) Reverse Only Surviving: 245 (0.74%) Dropped: 26568 (80.60%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: S7_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: S7_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: S7_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: S7_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S7_forward.paired.headclipped.fastq.gz S7_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S7 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S7_forward.paired.headclipped.fastq.gz
[FLASH]     S7_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S7.extendedFrags.fastq.gz
[FLASH]     ./S7.notCombined_1.fastq.gz
[FLASH]     ./S7.notCombined_2.fastq.gz
[FLASH]     ./S7.hist
[FLASH]     ./S7.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4645 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4645
[FLASH]     Combined pairs:   4316
[FLASH]     Uncombined pairs: 329
[FLASH]     Percent combined: 92.92%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.113 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4316 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S7/S7_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S7_centroids.fasta --uc S7.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4316 sequences and identified 2427 clusters (clustering threshold 1.00) - 46 clusters (representing 1620 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S72 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S72_1.fastq /home/working/FinalCorrected/demultiplexed/S72_2.fastq S72_forward.paired.fastq.gz S72_forward.singletons.fastq.gz S72_reverse.paired.fastq.gz S72_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S72_1.fastq /home/working/FinalCorrected/demultiplexed/S72_2.fastq S72_forward.paired.fastq.gz S72_forward.singletons.fastq.gz S72_reverse.paired.fastq.gz S72_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:50:34 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S72/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 38851 Both Surviving: 115 (0.30%) Forward Only Surviving: 569 (1.46%) Reverse Only Surviving: 398 (1.02%) Dropped: 37769 (97.22%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S72_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S72_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: S72_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: S72_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S72_forward.paired.headclipped.fastq.gz S72_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S72 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S72_forward.paired.headclipped.fastq.gz
[FLASH]     S72_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S72.extendedFrags.fastq.gz
[FLASH]     ./S72.notCombined_1.fastq.gz
[FLASH]     ./S72.notCombined_2.fastq.gz
[FLASH]     ./S72.hist
[FLASH]     ./S72.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 115 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      115
[FLASH]     Combined pairs:   80
[FLASH]     Uncombined pairs: 35
[FLASH]     Percent combined: 69.57%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

80 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S72/S72_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S72_centroids.fasta --uc S72.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 80 sequences and identified 71 clusters (clustering threshold 1.00) - 1 clusters (representing 6 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S77 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S77_1.fastq /home/working/FinalCorrected/demultiplexed/S77_2.fastq S77_forward.paired.fastq.gz S77_forward.singletons.fastq.gz S77_reverse.paired.fastq.gz S77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S77_1.fastq /home/working/FinalCorrected/demultiplexed/S77_2.fastq S77_forward.paired.fastq.gz S77_forward.singletons.fastq.gz S77_reverse.paired.fastq.gz S77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:50:37 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S77/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14430 Both Surviving: 1819 (12.61%) Forward Only Surviving: 613 (4.25%) Reverse Only Surviving: 134 (0.93%) Dropped: 11864 (82.22%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: S77_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: S77_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S77_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S77_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S77_forward.paired.headclipped.fastq.gz S77_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S77 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S77_forward.paired.headclipped.fastq.gz
[FLASH]     S77_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S77.extendedFrags.fastq.gz
[FLASH]     ./S77.notCombined_1.fastq.gz
[FLASH]     ./S77.notCombined_2.fastq.gz
[FLASH]     ./S77.hist
[FLASH]     ./S77.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1819 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1819
[FLASH]     Combined pairs:   115
[FLASH]     Uncombined pairs: 1704
[FLASH]     Percent combined: 6.32%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.012 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

115 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S77/S77_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S77_centroids.fasta --uc S77.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 115 sequences and identified 101 clusters (clustering threshold 1.00) - 1 clusters (representing 10 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S79 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S79_1.fastq /home/working/FinalCorrected/demultiplexed/S79_2.fastq S79_forward.paired.fastq.gz S79_forward.singletons.fastq.gz S79_reverse.paired.fastq.gz S79_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S79_1.fastq /home/working/FinalCorrected/demultiplexed/S79_2.fastq S79_forward.paired.fastq.gz S79_forward.singletons.fastq.gz S79_reverse.paired.fastq.gz S79_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:50:49 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S79/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 143319 Both Surviving: 50529 (35.26%) Forward Only Surviving: 11833 (8.26%) Reverse Only Surviving: 2377 (1.66%) Dropped: 78580 (54.83%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: S79_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: S79_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: S79_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: S79_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S79_forward.paired.headclipped.fastq.gz S79_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S79 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S79_forward.paired.headclipped.fastq.gz
[FLASH]     S79_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S79.extendedFrags.fastq.gz
[FLASH]     ./S79.notCombined_1.fastq.gz
[FLASH]     ./S79.notCombined_2.fastq.gz
[FLASH]     ./S79.hist
[FLASH]     ./S79.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 50000 read pairs
[FLASH] Processed 50529 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      50529
[FLASH]     Combined pairs:   32082
[FLASH]     Uncombined pairs: 18447
[FLASH]     Percent combined: 63.49%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.858 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

32082 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S79/S79_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S79_centroids.fasta --uc S79.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 32082 sequences and identified 9396 clusters (clustering threshold 1.00) - 592 clusters (representing 21271 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S8 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S8_1.fastq /home/working/FinalCorrected/demultiplexed/S8_2.fastq S8_forward.paired.fastq.gz S8_forward.singletons.fastq.gz S8_reverse.paired.fastq.gz S8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S8_1.fastq /home/working/FinalCorrected/demultiplexed/S8_2.fastq S8_forward.paired.fastq.gz S8_forward.singletons.fastq.gz S8_reverse.paired.fastq.gz S8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:51:57 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S8/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26392 Both Surviving: 9081 (34.41%) Forward Only Surviving: 3915 (14.83%) Reverse Only Surviving: 236 (0.89%) Dropped: 13160 (49.86%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: S8_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: S8_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S8_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S8_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S8_forward.paired.headclipped.fastq.gz S8_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S8 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S8_forward.paired.headclipped.fastq.gz
[FLASH]     S8_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S8.extendedFrags.fastq.gz
[FLASH]     ./S8.notCombined_1.fastq.gz
[FLASH]     ./S8.notCombined_2.fastq.gz
[FLASH]     ./S8.hist
[FLASH]     ./S8.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9081 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9081
[FLASH]     Combined pairs:   5726
[FLASH]     Uncombined pairs: 3355
[FLASH]     Percent combined: 63.05%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.159 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5726 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S8/S8_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S8_centroids.fasta --uc S8.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5726 sequences and identified 2705 clusters (clustering threshold 1.00) - 81 clusters (representing 2588 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S80 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S80_1.fastq /home/working/FinalCorrected/demultiplexed/S80_2.fastq S80_forward.paired.fastq.gz S80_forward.singletons.fastq.gz S80_reverse.paired.fastq.gz S80_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S80_1.fastq /home/working/FinalCorrected/demultiplexed/S80_2.fastq S80_forward.paired.fastq.gz S80_forward.singletons.fastq.gz S80_reverse.paired.fastq.gz S80_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:52:09 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S80/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 10756 Both Surviving: 1785 (16.60%) Forward Only Surviving: 535 (4.97%) Reverse Only Surviving: 148 (1.38%) Dropped: 8288 (77.05%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: S80_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: S80_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S80_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S80_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S80_forward.paired.headclipped.fastq.gz S80_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S80 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S80_forward.paired.headclipped.fastq.gz
[FLASH]     S80_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S80.extendedFrags.fastq.gz
[FLASH]     ./S80.notCombined_1.fastq.gz
[FLASH]     ./S80.notCombined_2.fastq.gz
[FLASH]     ./S80.hist
[FLASH]     ./S80.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1785 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1785
[FLASH]     Combined pairs:   362
[FLASH]     Uncombined pairs: 1423
[FLASH]     Percent combined: 20.28%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.015 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

362 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S80/S80_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S80_centroids.fasta --uc S80.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 362 sequences and identified 234 clusters (clustering threshold 1.00) - 6 clusters (representing 121 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S81 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S81_1.fastq /home/working/FinalCorrected/demultiplexed/S81_2.fastq S81_forward.paired.fastq.gz S81_forward.singletons.fastq.gz S81_reverse.paired.fastq.gz S81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S81_1.fastq /home/working/FinalCorrected/demultiplexed/S81_2.fastq S81_forward.paired.fastq.gz S81_forward.singletons.fastq.gz S81_reverse.paired.fastq.gz S81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:52:12 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S81/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20291 Both Surviving: 373 (1.84%) Forward Only Surviving: 524 (2.58%) Reverse Only Surviving: 177 (0.87%) Dropped: 19217 (94.71%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S81_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S81_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S81_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S81_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S81_forward.paired.headclipped.fastq.gz S81_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S81 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S81_forward.paired.headclipped.fastq.gz
[FLASH]     S81_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S81.extendedFrags.fastq.gz
[FLASH]     ./S81.notCombined_1.fastq.gz
[FLASH]     ./S81.notCombined_2.fastq.gz
[FLASH]     ./S81.hist
[FLASH]     ./S81.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 373 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      373
[FLASH]     Combined pairs:   240
[FLASH]     Uncombined pairs: 133
[FLASH]     Percent combined: 64.34%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.007 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

240 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S81/S81_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S81_centroids.fasta --uc S81.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 240 sequences and identified 190 clusters (clustering threshold 1.00) - 3 clusters (representing 43 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S83 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S83_1.fastq /home/working/FinalCorrected/demultiplexed/S83_2.fastq S83_forward.paired.fastq.gz S83_forward.singletons.fastq.gz S83_reverse.paired.fastq.gz S83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S83_1.fastq /home/working/FinalCorrected/demultiplexed/S83_2.fastq S83_forward.paired.fastq.gz S83_forward.singletons.fastq.gz S83_reverse.paired.fastq.gz S83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:52:18 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S83/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 40119 Both Surviving: 17744 (44.23%) Forward Only Surviving: 8944 (22.29%) Reverse Only Surviving: 329 (0.82%) Dropped: 13102 (32.66%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: S83_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: S83_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S83_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S83_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S83_forward.paired.headclipped.fastq.gz S83_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S83 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S83_forward.paired.headclipped.fastq.gz
[FLASH]     S83_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S83.extendedFrags.fastq.gz
[FLASH]     ./S83.notCombined_1.fastq.gz
[FLASH]     ./S83.notCombined_2.fastq.gz
[FLASH]     ./S83.hist
[FLASH]     ./S83.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 17744 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      17744
[FLASH]     Combined pairs:   13157
[FLASH]     Uncombined pairs: 4587
[FLASH]     Percent combined: 74.15%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.352 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

13157 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S83/S83_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S83_centroids.fasta --uc S83.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 13157 sequences and identified 3794 clusters (clustering threshold 1.00) - 314 clusters (representing 9363 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S9 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S9_1.fastq /home/working/FinalCorrected/demultiplexed/S9_2.fastq S9_forward.paired.fastq.gz S9_forward.singletons.fastq.gz S9_reverse.paired.fastq.gz S9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S9_1.fastq /home/working/FinalCorrected/demultiplexed/S9_2.fastq S9_forward.paired.fastq.gz S9_forward.singletons.fastq.gz S9_reverse.paired.fastq.gz S9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:52:44 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S9/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41954 Both Surviving: 13198 (31.46%) Forward Only Surviving: 4239 (10.10%) Reverse Only Surviving: 607 (1.45%) Dropped: 23910 (56.99%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: S9_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: S9_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: S9_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: S9_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S9_forward.paired.headclipped.fastq.gz S9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S9 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S9_forward.paired.headclipped.fastq.gz
[FLASH]     S9_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S9.extendedFrags.fastq.gz
[FLASH]     ./S9.notCombined_1.fastq.gz
[FLASH]     ./S9.notCombined_2.fastq.gz
[FLASH]     ./S9.hist
[FLASH]     ./S9.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 13198 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      13198
[FLASH]     Combined pairs:   8987
[FLASH]     Uncombined pairs: 4211
[FLASH]     Percent combined: 68.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.247 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

8987 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S9/S9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S9_centroids.fasta --uc S9.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 8987 sequences and identified 2196 clusters (clustering threshold 1.00) - 235 clusters (representing 6715 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S90 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S90_1.fastq /home/working/FinalCorrected/demultiplexed/S90_2.fastq S90_forward.paired.fastq.gz S90_forward.singletons.fastq.gz S90_reverse.paired.fastq.gz S90_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S90_1.fastq /home/working/FinalCorrected/demultiplexed/S90_2.fastq S90_forward.paired.fastq.gz S90_forward.singletons.fastq.gz S90_reverse.paired.fastq.gz S90_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:53:02 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S90/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18970 Both Surviving: 5970 (31.47%) Forward Only Surviving: 1659 (8.75%) Reverse Only Surviving: 249 (1.31%) Dropped: 11092 (58.47%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: S90_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: S90_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S90_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S90_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S90_forward.paired.headclipped.fastq.gz S90_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S90 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S90_forward.paired.headclipped.fastq.gz
[FLASH]     S90_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S90.extendedFrags.fastq.gz
[FLASH]     ./S90.notCombined_1.fastq.gz
[FLASH]     ./S90.notCombined_2.fastq.gz
[FLASH]     ./S90.hist
[FLASH]     ./S90.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5970 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5970
[FLASH]     Combined pairs:   3361
[FLASH]     Uncombined pairs: 2609
[FLASH]     Percent combined: 56.30%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.093 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3361 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S90/S90_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S90_centroids.fasta --uc S90.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3361 sequences and identified 2119 clusters (clustering threshold 1.00) - 41 clusters (representing 1066 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S92 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S92_1.fastq /home/working/FinalCorrected/demultiplexed/S92_2.fastq S92_forward.paired.fastq.gz S92_forward.singletons.fastq.gz S92_reverse.paired.fastq.gz S92_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S92_1.fastq /home/working/FinalCorrected/demultiplexed/S92_2.fastq S92_forward.paired.fastq.gz S92_forward.singletons.fastq.gz S92_reverse.paired.fastq.gz S92_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:53:13 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S92/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 58656 Both Surviving: 3312 (5.65%) Forward Only Surviving: 2833 (4.83%) Reverse Only Surviving: 755 (1.29%) Dropped: 51756 (88.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S92_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S92_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: S92_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: S92_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S92_forward.paired.headclipped.fastq.gz S92_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S92 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S92_forward.paired.headclipped.fastq.gz
[FLASH]     S92_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S92.extendedFrags.fastq.gz
[FLASH]     ./S92.notCombined_1.fastq.gz
[FLASH]     ./S92.notCombined_2.fastq.gz
[FLASH]     ./S92.hist
[FLASH]     ./S92.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3312 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3312
[FLASH]     Combined pairs:   805
[FLASH]     Uncombined pairs: 2507
[FLASH]     Percent combined: 24.31%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.037 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

805 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S92/S92_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S92_centroids.fasta --uc S92.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 805 sequences and identified 575 clusters (clustering threshold 1.00) - 13 clusters (representing 167 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S96 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S96_1.fastq /home/working/FinalCorrected/demultiplexed/S96_2.fastq S96_forward.paired.fastq.gz S96_forward.singletons.fastq.gz S96_reverse.paired.fastq.gz S96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S96_1.fastq /home/working/FinalCorrected/demultiplexed/S96_2.fastq S96_forward.paired.fastq.gz S96_forward.singletons.fastq.gz S96_reverse.paired.fastq.gz S96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:53:19 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S96/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17331 Both Surviving: 2217 (12.79%) Forward Only Surviving: 807 (4.66%) Reverse Only Surviving: 207 (1.19%) Dropped: 14100 (81.36%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: S96_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: S96_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: S96_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: S96_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S96_forward.paired.headclipped.fastq.gz S96_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S96 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S96_forward.paired.headclipped.fastq.gz
[FLASH]     S96_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S96.extendedFrags.fastq.gz
[FLASH]     ./S96.notCombined_1.fastq.gz
[FLASH]     ./S96.notCombined_2.fastq.gz
[FLASH]     ./S96.hist
[FLASH]     ./S96.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2217 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2217
[FLASH]     Combined pairs:   2021
[FLASH]     Uncombined pairs: 196
[FLASH]     Percent combined: 91.16%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.053 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2021 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S96/S96_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S96_centroids.fasta --uc S96.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2021 sequences and identified 1156 clusters (clustering threshold 1.00) - 26 clusters (representing 799 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: S97 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S97_1.fastq /home/working/FinalCorrected/demultiplexed/S97_2.fastq S97_forward.paired.fastq.gz S97_forward.singletons.fastq.gz S97_reverse.paired.fastq.gz S97_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/S97_1.fastq /home/working/FinalCorrected/demultiplexed/S97_2.fastq S97_forward.paired.fastq.gz S97_forward.singletons.fastq.gz S97_reverse.paired.fastq.gz S97_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:53:25 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/S97/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 19611 Both Surviving: 5305 (27.05%) Forward Only Surviving: 1691 (8.62%) Reverse Only Surviving: 253 (1.29%) Dropped: 12362 (63.04%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: S97_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: S97_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: S97_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: S97_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash S97_forward.paired.headclipped.fastq.gz S97_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S97 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     S97_forward.paired.headclipped.fastq.gz
[FLASH]     S97_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./S97.extendedFrags.fastq.gz
[FLASH]     ./S97.notCombined_1.fastq.gz
[FLASH]     ./S97.notCombined_2.fastq.gz
[FLASH]     ./S97.hist
[FLASH]     ./S97.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5305 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5305
[FLASH]     Combined pairs:   276
[FLASH]     Uncombined pairs: 5029
[FLASH]     Percent combined: 5.20%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.033 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

276 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/S97/S97_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S97_centroids.fasta --uc S97.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 276 sequences and identified 211 clusters (clustering threshold 1.00) - 5 clusters (representing 37 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA1 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA1_1.fastq /home/working/FinalCorrected/demultiplexed/SA1_2.fastq SA1_forward.paired.fastq.gz SA1_forward.singletons.fastq.gz SA1_reverse.paired.fastq.gz SA1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA1_1.fastq /home/working/FinalCorrected/demultiplexed/SA1_2.fastq SA1_forward.paired.fastq.gz SA1_forward.singletons.fastq.gz SA1_reverse.paired.fastq.gz SA1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:53:28 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA1/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 8479 Both Surviving: 396 (4.67%) Forward Only Surviving: 277 (3.27%) Reverse Only Surviving: 151 (1.78%) Dropped: 7655 (90.28%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA1_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA1_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA1_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA1_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA1_forward.paired.headclipped.fastq.gz SA1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA1 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA1_forward.paired.headclipped.fastq.gz
[FLASH]     SA1_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA1.extendedFrags.fastq.gz
[FLASH]     ./SA1.notCombined_1.fastq.gz
[FLASH]     ./SA1.notCombined_2.fastq.gz
[FLASH]     ./SA1.hist
[FLASH]     ./SA1.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 396 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      396
[FLASH]     Combined pairs:   329
[FLASH]     Uncombined pairs: 67
[FLASH]     Percent combined: 83.08%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.009 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

329 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA1/SA1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA1_centroids.fasta --uc SA1.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 329 sequences and identified 217 clusters (clustering threshold 1.00) - 10 clusters (representing 103 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA104 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA104_1.fastq /home/working/FinalCorrected/demultiplexed/SA104_2.fastq SA104_forward.paired.fastq.gz SA104_forward.singletons.fastq.gz SA104_reverse.paired.fastq.gz SA104_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA104_1.fastq /home/working/FinalCorrected/demultiplexed/SA104_2.fastq SA104_forward.paired.fastq.gz SA104_forward.singletons.fastq.gz SA104_reverse.paired.fastq.gz SA104_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:53:31 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA104/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 24043 Both Surviving: 3279 (13.64%) Forward Only Surviving: 915 (3.81%) Reverse Only Surviving: 431 (1.79%) Dropped: 19418 (80.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA104_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA104_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA104_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA104_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA104_forward.paired.headclipped.fastq.gz SA104_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA104 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA104_forward.paired.headclipped.fastq.gz
[FLASH]     SA104_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA104.extendedFrags.fastq.gz
[FLASH]     ./SA104.notCombined_1.fastq.gz
[FLASH]     ./SA104.notCombined_2.fastq.gz
[FLASH]     ./SA104.hist
[FLASH]     ./SA104.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3279 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3279
[FLASH]     Combined pairs:   82
[FLASH]     Uncombined pairs: 3197
[FLASH]     Percent combined: 2.50%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.021 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

82 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA104/SA104_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA104_centroids.fasta --uc SA104.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 82 sequences and identified 62 clusters (clustering threshold 1.00) - 2 clusters (representing 10 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA105 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA105_1.fastq /home/working/FinalCorrected/demultiplexed/SA105_2.fastq SA105_forward.paired.fastq.gz SA105_forward.singletons.fastq.gz SA105_reverse.paired.fastq.gz SA105_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA105_1.fastq /home/working/FinalCorrected/demultiplexed/SA105_2.fastq SA105_forward.paired.fastq.gz SA105_forward.singletons.fastq.gz SA105_reverse.paired.fastq.gz SA105_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:53:35 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA105/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23190 Both Surviving: 114 (0.49%) Forward Only Surviving: 242 (1.04%) Reverse Only Surviving: 165 (0.71%) Dropped: 22669 (97.75%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA105_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA105_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA105_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA105_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA105_forward.paired.headclipped.fastq.gz SA105_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA105 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA105_forward.paired.headclipped.fastq.gz
[FLASH]     SA105_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA105.extendedFrags.fastq.gz
[FLASH]     ./SA105.notCombined_1.fastq.gz
[FLASH]     ./SA105.notCombined_2.fastq.gz
[FLASH]     ./SA105.hist
[FLASH]     ./SA105.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 114 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      114
[FLASH]     Combined pairs:   85
[FLASH]     Uncombined pairs: 29
[FLASH]     Percent combined: 74.56%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

85 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA105/SA105_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA105_centroids.fasta --uc SA105.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 85 sequences and identified 59 clusters (clustering threshold 1.00) - 2 clusters (representing 24 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA109 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA109_1.fastq /home/working/FinalCorrected/demultiplexed/SA109_2.fastq SA109_forward.paired.fastq.gz SA109_forward.singletons.fastq.gz SA109_reverse.paired.fastq.gz SA109_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA109_1.fastq /home/working/FinalCorrected/demultiplexed/SA109_2.fastq SA109_forward.paired.fastq.gz SA109_forward.singletons.fastq.gz SA109_reverse.paired.fastq.gz SA109_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:53:37 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA109/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 6 Both Surviving: 1 (16.67%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 5 (83.33%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SA109_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA109_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA109_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA109_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA109_forward.paired.headclipped.fastq.gz SA109_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA109 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA109_forward.paired.headclipped.fastq.gz
[FLASH]     SA109_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA109.extendedFrags.fastq.gz
[FLASH]     ./SA109.notCombined_1.fastq.gz
[FLASH]     ./SA109.notCombined_2.fastq.gz
[FLASH]     ./SA109.hist
[FLASH]     ./SA109.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1
[FLASH]     Combined pairs:   1
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 100.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA109/SA109_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA109_centroids.fasta --uc SA109.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1 sequences and identified 1 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA111 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA111_1.fastq /home/working/FinalCorrected/demultiplexed/SA111_2.fastq SA111_forward.paired.fastq.gz SA111_forward.singletons.fastq.gz SA111_reverse.paired.fastq.gz SA111_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA111_1.fastq /home/working/FinalCorrected/demultiplexed/SA111_2.fastq SA111_forward.paired.fastq.gz SA111_forward.singletons.fastq.gz SA111_reverse.paired.fastq.gz SA111_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:53:38 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA111/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16872 Both Surviving: 3412 (20.22%) Forward Only Surviving: 1618 (9.59%) Reverse Only Surviving: 186 (1.10%) Dropped: 11656 (69.08%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA111_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA111_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA111_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA111_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA111_forward.paired.headclipped.fastq.gz SA111_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA111 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA111_forward.paired.headclipped.fastq.gz
[FLASH]     SA111_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA111.extendedFrags.fastq.gz
[FLASH]     ./SA111.notCombined_1.fastq.gz
[FLASH]     ./SA111.notCombined_2.fastq.gz
[FLASH]     ./SA111.hist
[FLASH]     ./SA111.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3412 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3412
[FLASH]     Combined pairs:   412
[FLASH]     Uncombined pairs: 3000
[FLASH]     Percent combined: 12.08%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.024 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

412 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA111/SA111_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA111_centroids.fasta --uc SA111.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 412 sequences and identified 319 clusters (clustering threshold 1.00) - 8 clusters (representing 56 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA112 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA112_1.fastq /home/working/FinalCorrected/demultiplexed/SA112_2.fastq SA112_forward.paired.fastq.gz SA112_forward.singletons.fastq.gz SA112_reverse.paired.fastq.gz SA112_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA112_1.fastq /home/working/FinalCorrected/demultiplexed/SA112_2.fastq SA112_forward.paired.fastq.gz SA112_forward.singletons.fastq.gz SA112_reverse.paired.fastq.gz SA112_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:53:46 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA112/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 86230 Both Surviving: 26880 (31.17%) Forward Only Surviving: 5576 (6.47%) Reverse Only Surviving: 3113 (3.61%) Dropped: 50661 (58.75%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SA112_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA112_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA112_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA112_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA112_forward.paired.headclipped.fastq.gz SA112_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA112 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA112_forward.paired.headclipped.fastq.gz
[FLASH]     SA112_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA112.extendedFrags.fastq.gz
[FLASH]     ./SA112.notCombined_1.fastq.gz
[FLASH]     ./SA112.notCombined_2.fastq.gz
[FLASH]     ./SA112.hist
[FLASH]     ./SA112.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 26880 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      26880
[FLASH]     Combined pairs:   5185
[FLASH]     Uncombined pairs: 21695
[FLASH]     Percent combined: 19.29%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.207 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5185 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA112/SA112_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA112_centroids.fasta --uc SA112.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5185 sequences and identified 2926 clusters (clustering threshold 1.00) - 76 clusters (representing 1962 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA113 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA113_1.fastq /home/working/FinalCorrected/demultiplexed/SA113_2.fastq SA113_forward.paired.fastq.gz SA113_forward.singletons.fastq.gz SA113_reverse.paired.fastq.gz SA113_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA113_1.fastq /home/working/FinalCorrected/demultiplexed/SA113_2.fastq SA113_forward.paired.fastq.gz SA113_forward.singletons.fastq.gz SA113_reverse.paired.fastq.gz SA113_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:54:07 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA113/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31186 Both Surviving: 6006 (19.26%) Forward Only Surviving: 1650 (5.29%) Reverse Only Surviving: 427 (1.37%) Dropped: 23103 (74.08%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA113_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA113_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA113_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA113_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA113_forward.paired.headclipped.fastq.gz SA113_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA113 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA113_forward.paired.headclipped.fastq.gz
[FLASH]     SA113_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA113.extendedFrags.fastq.gz
[FLASH]     ./SA113.notCombined_1.fastq.gz
[FLASH]     ./SA113.notCombined_2.fastq.gz
[FLASH]     ./SA113.hist
[FLASH]     ./SA113.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6006 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6006
[FLASH]     Combined pairs:   500
[FLASH]     Uncombined pairs: 5506
[FLASH]     Percent combined: 8.33%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.036 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

500 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA113/SA113_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA113_centroids.fasta --uc SA113.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 500 sequences and identified 338 clusters (clustering threshold 1.00) - 13 clusters (representing 149 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA114 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA114_1.fastq /home/working/FinalCorrected/demultiplexed/SA114_2.fastq SA114_forward.paired.fastq.gz SA114_forward.singletons.fastq.gz SA114_reverse.paired.fastq.gz SA114_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA114_1.fastq /home/working/FinalCorrected/demultiplexed/SA114_2.fastq SA114_forward.paired.fastq.gz SA114_forward.singletons.fastq.gz SA114_reverse.paired.fastq.gz SA114_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:54:20 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA114/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 133965 Both Surviving: 33203 (24.78%) Forward Only Surviving: 12322 (9.20%) Reverse Only Surviving: 1774 (1.32%) Dropped: 86666 (64.69%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SA114_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SA114_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SA114_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SA114_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA114_forward.paired.headclipped.fastq.gz SA114_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA114 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA114_forward.paired.headclipped.fastq.gz
[FLASH]     SA114_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA114.extendedFrags.fastq.gz
[FLASH]     ./SA114.notCombined_1.fastq.gz
[FLASH]     ./SA114.notCombined_2.fastq.gz
[FLASH]     ./SA114.hist
[FLASH]     ./SA114.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 33203 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      33203
[FLASH]     Combined pairs:   28702
[FLASH]     Uncombined pairs: 4501
[FLASH]     Percent combined: 86.44%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.733 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

28702 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA114/SA114_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA114_centroids.fasta --uc SA114.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 28702 sequences and identified 11395 clusters (clustering threshold 1.00) - 476 clusters (representing 16481 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA115 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA115_1.fastq /home/working/FinalCorrected/demultiplexed/SA115_2.fastq SA115_forward.paired.fastq.gz SA115_forward.singletons.fastq.gz SA115_reverse.paired.fastq.gz SA115_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA115_1.fastq /home/working/FinalCorrected/demultiplexed/SA115_2.fastq SA115_forward.paired.fastq.gz SA115_forward.singletons.fastq.gz SA115_reverse.paired.fastq.gz SA115_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:55:18 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA115/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 21323 Both Surviving: 323 (1.51%) Forward Only Surviving: 345 (1.62%) Reverse Only Surviving: 232 (1.09%) Dropped: 20423 (95.78%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA115_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA115_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA115_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA115_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA115_forward.paired.headclipped.fastq.gz SA115_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA115 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA115_forward.paired.headclipped.fastq.gz
[FLASH]     SA115_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA115.extendedFrags.fastq.gz
[FLASH]     ./SA115.notCombined_1.fastq.gz
[FLASH]     ./SA115.notCombined_2.fastq.gz
[FLASH]     ./SA115.hist
[FLASH]     ./SA115.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 323 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      323
[FLASH]     Combined pairs:   79
[FLASH]     Uncombined pairs: 244
[FLASH]     Percent combined: 24.46%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

79 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA115/SA115_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA115_centroids.fasta --uc SA115.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 79 sequences and identified 61 clusters (clustering threshold 1.00) - 2 clusters (representing 15 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA116 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA116_1.fastq /home/working/FinalCorrected/demultiplexed/SA116_2.fastq SA116_forward.paired.fastq.gz SA116_forward.singletons.fastq.gz SA116_reverse.paired.fastq.gz SA116_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA116_1.fastq /home/working/FinalCorrected/demultiplexed/SA116_2.fastq SA116_forward.paired.fastq.gz SA116_forward.singletons.fastq.gz SA116_reverse.paired.fastq.gz SA116_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:55:21 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA116/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31818 Both Surviving: 2035 (6.40%) Forward Only Surviving: 881 (2.77%) Reverse Only Surviving: 589 (1.85%) Dropped: 28313 (88.98%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA116_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA116_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA116_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA116_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA116_forward.paired.headclipped.fastq.gz SA116_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA116 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA116_forward.paired.headclipped.fastq.gz
[FLASH]     SA116_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA116.extendedFrags.fastq.gz
[FLASH]     ./SA116.notCombined_1.fastq.gz
[FLASH]     ./SA116.notCombined_2.fastq.gz
[FLASH]     ./SA116.hist
[FLASH]     ./SA116.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2035 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2035
[FLASH]     Combined pairs:   13
[FLASH]     Uncombined pairs: 2022
[FLASH]     Percent combined: 0.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.016 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

13 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA116/SA116_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA116_centroids.fasta --uc SA116.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 13 sequences and identified 13 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA117 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA117_1.fastq /home/working/FinalCorrected/demultiplexed/SA117_2.fastq SA117_forward.paired.fastq.gz SA117_forward.singletons.fastq.gz SA117_reverse.paired.fastq.gz SA117_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA117_1.fastq /home/working/FinalCorrected/demultiplexed/SA117_2.fastq SA117_forward.paired.fastq.gz SA117_forward.singletons.fastq.gz SA117_reverse.paired.fastq.gz SA117_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:55:26 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA117/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37714 Both Surviving: 7464 (19.79%) Forward Only Surviving: 1783 (4.73%) Reverse Only Surviving: 467 (1.24%) Dropped: 28000 (74.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA117_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA117_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA117_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA117_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA117_forward.paired.headclipped.fastq.gz SA117_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA117 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA117_forward.paired.headclipped.fastq.gz
[FLASH]     SA117_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA117.extendedFrags.fastq.gz
[FLASH]     ./SA117.notCombined_1.fastq.gz
[FLASH]     ./SA117.notCombined_2.fastq.gz
[FLASH]     ./SA117.hist
[FLASH]     ./SA117.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7464 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7464
[FLASH]     Combined pairs:   1516
[FLASH]     Uncombined pairs: 5948
[FLASH]     Percent combined: 20.31%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.054 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1516 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA117/SA117_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA117_centroids.fasta --uc SA117.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1516 sequences and identified 932 clusters (clustering threshold 1.00) - 19 clusters (representing 523 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA119 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA119_1.fastq /home/working/FinalCorrected/demultiplexed/SA119_2.fastq SA119_forward.paired.fastq.gz SA119_forward.singletons.fastq.gz SA119_reverse.paired.fastq.gz SA119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA119_1.fastq /home/working/FinalCorrected/demultiplexed/SA119_2.fastq SA119_forward.paired.fastq.gz SA119_forward.singletons.fastq.gz SA119_reverse.paired.fastq.gz SA119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:55:36 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA119/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 76452 Both Surviving: 12333 (16.13%) Forward Only Surviving: 3713 (4.86%) Reverse Only Surviving: 2374 (3.11%) Dropped: 58032 (75.91%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SA119_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA119_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SA119_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA119_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA119_forward.paired.headclipped.fastq.gz SA119_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA119 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA119_forward.paired.headclipped.fastq.gz
[FLASH]     SA119_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA119.extendedFrags.fastq.gz
[FLASH]     ./SA119.notCombined_1.fastq.gz
[FLASH]     ./SA119.notCombined_2.fastq.gz
[FLASH]     ./SA119.hist
[FLASH]     ./SA119.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 12333 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      12333
[FLASH]     Combined pairs:   548
[FLASH]     Uncombined pairs: 11785
[FLASH]     Percent combined: 4.44%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.082 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

548 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA119/SA119_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA119_centroids.fasta --uc SA119.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 548 sequences and identified 429 clusters (clustering threshold 1.00) - 3 clusters (representing 86 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA120 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA120_1.fastq /home/working/FinalCorrected/demultiplexed/SA120_2.fastq SA120_forward.paired.fastq.gz SA120_forward.singletons.fastq.gz SA120_reverse.paired.fastq.gz SA120_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA120_1.fastq /home/working/FinalCorrected/demultiplexed/SA120_2.fastq SA120_forward.paired.fastq.gz SA120_forward.singletons.fastq.gz SA120_reverse.paired.fastq.gz SA120_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:55:46 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA120/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26355 Both Surviving: 872 (3.31%) Forward Only Surviving: 734 (2.79%) Reverse Only Surviving: 223 (0.85%) Dropped: 24526 (93.06%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA120_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA120_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA120_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA120_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA120_forward.paired.headclipped.fastq.gz SA120_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA120 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA120_forward.paired.headclipped.fastq.gz
[FLASH]     SA120_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA120.extendedFrags.fastq.gz
[FLASH]     ./SA120.notCombined_1.fastq.gz
[FLASH]     ./SA120.notCombined_2.fastq.gz
[FLASH]     ./SA120.hist
[FLASH]     ./SA120.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 872 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      872
[FLASH]     Combined pairs:   669
[FLASH]     Uncombined pairs: 203
[FLASH]     Percent combined: 76.72%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.020 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

669 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA120/SA120_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA120_centroids.fasta --uc SA120.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 669 sequences and identified 419 clusters (clustering threshold 1.00) - 15 clusters (representing 234 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA121 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA121_1.fastq /home/working/FinalCorrected/demultiplexed/SA121_2.fastq SA121_forward.paired.fastq.gz SA121_forward.singletons.fastq.gz SA121_reverse.paired.fastq.gz SA121_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA121_1.fastq /home/working/FinalCorrected/demultiplexed/SA121_2.fastq SA121_forward.paired.fastq.gz SA121_forward.singletons.fastq.gz SA121_reverse.paired.fastq.gz SA121_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:55:52 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA121/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 50866 Both Surviving: 10952 (21.53%) Forward Only Surviving: 3441 (6.76%) Reverse Only Surviving: 729 (1.43%) Dropped: 35744 (70.27%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SA121_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SA121_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA121_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA121_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA121_forward.paired.headclipped.fastq.gz SA121_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA121 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA121_forward.paired.headclipped.fastq.gz
[FLASH]     SA121_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA121.extendedFrags.fastq.gz
[FLASH]     ./SA121.notCombined_1.fastq.gz
[FLASH]     ./SA121.notCombined_2.fastq.gz
[FLASH]     ./SA121.hist
[FLASH]     ./SA121.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10952 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10952
[FLASH]     Combined pairs:   1060
[FLASH]     Uncombined pairs: 9892
[FLASH]     Percent combined: 9.68%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.068 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1060 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA121/SA121_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA121_centroids.fasta --uc SA121.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1060 sequences and identified 715 clusters (clustering threshold 1.00) - 17 clusters (representing 283 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA122 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA122_1.fastq /home/working/FinalCorrected/demultiplexed/SA122_2.fastq SA122_forward.paired.fastq.gz SA122_forward.singletons.fastq.gz SA122_reverse.paired.fastq.gz SA122_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA122_1.fastq /home/working/FinalCorrected/demultiplexed/SA122_2.fastq SA122_forward.paired.fastq.gz SA122_forward.singletons.fastq.gz SA122_reverse.paired.fastq.gz SA122_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:56:00 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA122/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 13391 Both Surviving: 4648 (34.71%) Forward Only Surviving: 940 (7.02%) Reverse Only Surviving: 201 (1.50%) Dropped: 7602 (56.77%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SA122_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SA122_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA122_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA122_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA122_forward.paired.headclipped.fastq.gz SA122_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA122 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA122_forward.paired.headclipped.fastq.gz
[FLASH]     SA122_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA122.extendedFrags.fastq.gz
[FLASH]     ./SA122.notCombined_1.fastq.gz
[FLASH]     ./SA122.notCombined_2.fastq.gz
[FLASH]     ./SA122.hist
[FLASH]     ./SA122.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4648 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4648
[FLASH]     Combined pairs:   2464
[FLASH]     Uncombined pairs: 2184
[FLASH]     Percent combined: 53.01%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.073 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2464 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA122/SA122_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA122_centroids.fasta --uc SA122.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2464 sequences and identified 1373 clusters (clustering threshold 1.00) - 56 clusters (representing 941 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA124 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA124_1.fastq /home/working/FinalCorrected/demultiplexed/SA124_2.fastq SA124_forward.paired.fastq.gz SA124_forward.singletons.fastq.gz SA124_reverse.paired.fastq.gz SA124_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA124_1.fastq /home/working/FinalCorrected/demultiplexed/SA124_2.fastq SA124_forward.paired.fastq.gz SA124_forward.singletons.fastq.gz SA124_reverse.paired.fastq.gz SA124_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:56:05 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA124/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SA124_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA124_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA124_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA124_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA124_forward.paired.headclipped.fastq.gz SA124_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA124 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA124_forward.paired.headclipped.fastq.gz
[FLASH]     SA124_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA124.extendedFrags.fastq.gz
[FLASH]     ./SA124.notCombined_1.fastq.gz
[FLASH]     ./SA124.notCombined_2.fastq.gz
[FLASH]     ./SA124.hist
[FLASH]     ./SA124.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA124/SA124_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA124_centroids.fasta --uc SA124.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA125 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA125_1.fastq /home/working/FinalCorrected/demultiplexed/SA125_2.fastq SA125_forward.paired.fastq.gz SA125_forward.singletons.fastq.gz SA125_reverse.paired.fastq.gz SA125_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA125_1.fastq /home/working/FinalCorrected/demultiplexed/SA125_2.fastq SA125_forward.paired.fastq.gz SA125_forward.singletons.fastq.gz SA125_reverse.paired.fastq.gz SA125_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:56:10 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA125/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 84537 Both Surviving: 26226 (31.02%) Forward Only Surviving: 8061 (9.54%) Reverse Only Surviving: 1377 (1.63%) Dropped: 48873 (57.81%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA125_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA125_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA125_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA125_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA125_forward.paired.headclipped.fastq.gz SA125_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA125 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA125_forward.paired.headclipped.fastq.gz
[FLASH]     SA125_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA125.extendedFrags.fastq.gz
[FLASH]     ./SA125.notCombined_1.fastq.gz
[FLASH]     ./SA125.notCombined_2.fastq.gz
[FLASH]     ./SA125.hist
[FLASH]     ./SA125.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 26226 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      26226
[FLASH]     Combined pairs:   6724
[FLASH]     Uncombined pairs: 19502
[FLASH]     Percent combined: 25.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.241 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6724 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA125/SA125_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA125_centroids.fasta --uc SA125.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6724 sequences and identified 2696 clusters (clustering threshold 1.00) - 109 clusters (representing 3473 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA126 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA126_1.fastq /home/working/FinalCorrected/demultiplexed/SA126_2.fastq SA126_forward.paired.fastq.gz SA126_forward.singletons.fastq.gz SA126_reverse.paired.fastq.gz SA126_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA126_1.fastq /home/working/FinalCorrected/demultiplexed/SA126_2.fastq SA126_forward.paired.fastq.gz SA126_forward.singletons.fastq.gz SA126_reverse.paired.fastq.gz SA126_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:56:33 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA126/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 43041 Both Surviving: 12938 (30.06%) Forward Only Surviving: 3091 (7.18%) Reverse Only Surviving: 878 (2.04%) Dropped: 26134 (60.72%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SA126_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SA126_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA126_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA126_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA126_forward.paired.headclipped.fastq.gz SA126_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA126 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA126_forward.paired.headclipped.fastq.gz
[FLASH]     SA126_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA126.extendedFrags.fastq.gz
[FLASH]     ./SA126.notCombined_1.fastq.gz
[FLASH]     ./SA126.notCombined_2.fastq.gz
[FLASH]     ./SA126.hist
[FLASH]     ./SA126.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 12938 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      12938
[FLASH]     Combined pairs:   568
[FLASH]     Uncombined pairs: 12370
[FLASH]     Percent combined: 4.39%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.072 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

568 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA126/SA126_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA126_centroids.fasta --uc SA126.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 568 sequences and identified 417 clusters (clustering threshold 1.00) - 7 clusters (representing 105 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA128 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA128_1.fastq /home/working/FinalCorrected/demultiplexed/SA128_2.fastq SA128_forward.paired.fastq.gz SA128_forward.singletons.fastq.gz SA128_reverse.paired.fastq.gz SA128_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA128_1.fastq /home/working/FinalCorrected/demultiplexed/SA128_2.fastq SA128_forward.paired.fastq.gz SA128_forward.singletons.fastq.gz SA128_reverse.paired.fastq.gz SA128_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:56:41 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA128/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 10561 Both Surviving: 3860 (36.55%) Forward Only Surviving: 809 (7.66%) Reverse Only Surviving: 188 (1.78%) Dropped: 5704 (54.01%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA128_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA128_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA128_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA128_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA128_forward.paired.headclipped.fastq.gz SA128_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA128 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA128_forward.paired.headclipped.fastq.gz
[FLASH]     SA128_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA128.extendedFrags.fastq.gz
[FLASH]     ./SA128.notCombined_1.fastq.gz
[FLASH]     ./SA128.notCombined_2.fastq.gz
[FLASH]     ./SA128.hist
[FLASH]     ./SA128.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3860 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3860
[FLASH]     Combined pairs:   2075
[FLASH]     Uncombined pairs: 1785
[FLASH]     Percent combined: 53.76%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.067 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2075 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA128/SA128_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA128_centroids.fasta --uc SA128.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2075 sequences and identified 1191 clusters (clustering threshold 1.00) - 23 clusters (representing 769 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA13 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA13_1.fastq /home/working/FinalCorrected/demultiplexed/SA13_2.fastq SA13_forward.paired.fastq.gz SA13_forward.singletons.fastq.gz SA13_reverse.paired.fastq.gz SA13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA13_1.fastq /home/working/FinalCorrected/demultiplexed/SA13_2.fastq SA13_forward.paired.fastq.gz SA13_forward.singletons.fastq.gz SA13_reverse.paired.fastq.gz SA13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:56:48 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA13/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31253 Both Surviving: 10666 (34.13%) Forward Only Surviving: 2268 (7.26%) Reverse Only Surviving: 484 (1.55%) Dropped: 17835 (57.07%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA13_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA13_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA13_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA13_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA13_forward.paired.headclipped.fastq.gz SA13_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA13 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA13_forward.paired.headclipped.fastq.gz
[FLASH]     SA13_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA13.extendedFrags.fastq.gz
[FLASH]     ./SA13.notCombined_1.fastq.gz
[FLASH]     ./SA13.notCombined_2.fastq.gz
[FLASH]     ./SA13.hist
[FLASH]     ./SA13.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10666 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10666
[FLASH]     Combined pairs:   6139
[FLASH]     Uncombined pairs: 4527
[FLASH]     Percent combined: 57.56%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.163 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6139 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA13/SA13_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA13_centroids.fasta --uc SA13.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6139 sequences and identified 3472 clusters (clustering threshold 1.00) - 98 clusters (representing 2365 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA130 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA130_1.fastq /home/working/FinalCorrected/demultiplexed/SA130_2.fastq SA130_forward.paired.fastq.gz SA130_forward.singletons.fastq.gz SA130_reverse.paired.fastq.gz SA130_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA130_1.fastq /home/working/FinalCorrected/demultiplexed/SA130_2.fastq SA130_forward.paired.fastq.gz SA130_forward.singletons.fastq.gz SA130_reverse.paired.fastq.gz SA130_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:57:01 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA130/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20540 Both Surviving: 3652 (17.78%) Forward Only Surviving: 800 (3.89%) Reverse Only Surviving: 646 (3.15%) Dropped: 15442 (75.18%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SA130_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA130_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA130_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA130_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA130_forward.paired.headclipped.fastq.gz SA130_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA130 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA130_forward.paired.headclipped.fastq.gz
[FLASH]     SA130_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA130.extendedFrags.fastq.gz
[FLASH]     ./SA130.notCombined_1.fastq.gz
[FLASH]     ./SA130.notCombined_2.fastq.gz
[FLASH]     ./SA130.hist
[FLASH]     ./SA130.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3652 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3652
[FLASH]     Combined pairs:   514
[FLASH]     Uncombined pairs: 3138
[FLASH]     Percent combined: 14.07%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.029 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

514 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA130/SA130_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA130_centroids.fasta --uc SA130.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 514 sequences and identified 284 clusters (clustering threshold 1.00) - 7 clusters (representing 218 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA132 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA132_1.fastq /home/working/FinalCorrected/demultiplexed/SA132_2.fastq SA132_forward.paired.fastq.gz SA132_forward.singletons.fastq.gz SA132_reverse.paired.fastq.gz SA132_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA132_1.fastq /home/working/FinalCorrected/demultiplexed/SA132_2.fastq SA132_forward.paired.fastq.gz SA132_forward.singletons.fastq.gz SA132_reverse.paired.fastq.gz SA132_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:57:04 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA132/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 5677 Both Surviving: 1186 (20.89%) Forward Only Surviving: 263 (4.63%) Reverse Only Surviving: 195 (3.43%) Dropped: 4033 (71.04%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SA132_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA132_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA132_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA132_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA132_forward.paired.headclipped.fastq.gz SA132_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA132 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA132_forward.paired.headclipped.fastq.gz
[FLASH]     SA132_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA132.extendedFrags.fastq.gz
[FLASH]     ./SA132.notCombined_1.fastq.gz
[FLASH]     ./SA132.notCombined_2.fastq.gz
[FLASH]     ./SA132.hist
[FLASH]     ./SA132.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1186 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1186
[FLASH]     Combined pairs:   455
[FLASH]     Uncombined pairs: 731
[FLASH]     Percent combined: 38.36%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.014 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

455 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA132/SA132_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA132_centroids.fasta --uc SA132.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 455 sequences and identified 274 clusters (clustering threshold 1.00) - 7 clusters (representing 159 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA138 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA138_1.fastq /home/working/FinalCorrected/demultiplexed/SA138_2.fastq SA138_forward.paired.fastq.gz SA138_forward.singletons.fastq.gz SA138_reverse.paired.fastq.gz SA138_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA138_1.fastq /home/working/FinalCorrected/demultiplexed/SA138_2.fastq SA138_forward.paired.fastq.gz SA138_forward.singletons.fastq.gz SA138_reverse.paired.fastq.gz SA138_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:57:07 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA138/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16117 Both Surviving: 648 (4.02%) Forward Only Surviving: 393 (2.44%) Reverse Only Surviving: 272 (1.69%) Dropped: 14804 (91.85%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA138_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA138_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA138_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA138_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA138_forward.paired.headclipped.fastq.gz SA138_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA138 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA138_forward.paired.headclipped.fastq.gz
[FLASH]     SA138_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA138.extendedFrags.fastq.gz
[FLASH]     ./SA138.notCombined_1.fastq.gz
[FLASH]     ./SA138.notCombined_2.fastq.gz
[FLASH]     ./SA138.hist
[FLASH]     ./SA138.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 648 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      648
[FLASH]     Combined pairs:   332
[FLASH]     Uncombined pairs: 316
[FLASH]     Percent combined: 51.23%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.012 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

332 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA138/SA138_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA138_centroids.fasta --uc SA138.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 332 sequences and identified 196 clusters (clustering threshold 1.00) - 7 clusters (representing 135 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA140 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA140_1.fastq /home/working/FinalCorrected/demultiplexed/SA140_2.fastq SA140_forward.paired.fastq.gz SA140_forward.singletons.fastq.gz SA140_reverse.paired.fastq.gz SA140_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA140_1.fastq /home/working/FinalCorrected/demultiplexed/SA140_2.fastq SA140_forward.paired.fastq.gz SA140_forward.singletons.fastq.gz SA140_reverse.paired.fastq.gz SA140_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:57:13 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA140/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 81551 Both Surviving: 8759 (10.74%) Forward Only Surviving: 2778 (3.41%) Reverse Only Surviving: 1121 (1.37%) Dropped: 68893 (84.48%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA140_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA140_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA140_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA140_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA140_forward.paired.headclipped.fastq.gz SA140_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA140 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA140_forward.paired.headclipped.fastq.gz
[FLASH]     SA140_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA140.extendedFrags.fastq.gz
[FLASH]     ./SA140.notCombined_1.fastq.gz
[FLASH]     ./SA140.notCombined_2.fastq.gz
[FLASH]     ./SA140.hist
[FLASH]     ./SA140.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8759 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8759
[FLASH]     Combined pairs:   242
[FLASH]     Uncombined pairs: 8517
[FLASH]     Percent combined: 2.76%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.049 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

242 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA140/SA140_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA140_centroids.fasta --uc SA140.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 242 sequences and identified 192 clusters (clustering threshold 1.00) - 3 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA144 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA144_1.fastq /home/working/FinalCorrected/demultiplexed/SA144_2.fastq SA144_forward.paired.fastq.gz SA144_forward.singletons.fastq.gz SA144_reverse.paired.fastq.gz SA144_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA144_1.fastq /home/working/FinalCorrected/demultiplexed/SA144_2.fastq SA144_forward.paired.fastq.gz SA144_forward.singletons.fastq.gz SA144_reverse.paired.fastq.gz SA144_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:57:22 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA144/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28135 Both Surviving: 7735 (27.49%) Forward Only Surviving: 2906 (10.33%) Reverse Only Surviving: 328 (1.17%) Dropped: 17166 (61.01%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA144_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA144_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA144_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA144_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA144_forward.paired.headclipped.fastq.gz SA144_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA144 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA144_forward.paired.headclipped.fastq.gz
[FLASH]     SA144_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA144.extendedFrags.fastq.gz
[FLASH]     ./SA144.notCombined_1.fastq.gz
[FLASH]     ./SA144.notCombined_2.fastq.gz
[FLASH]     ./SA144.hist
[FLASH]     ./SA144.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7735 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7735
[FLASH]     Combined pairs:   6523
[FLASH]     Uncombined pairs: 1212
[FLASH]     Percent combined: 84.33%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.183 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6523 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA144/SA144_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA144_centroids.fasta --uc SA144.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6523 sequences and identified 3140 clusters (clustering threshold 1.00) - 131 clusters (representing 3022 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA147 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA147_1.fastq /home/working/FinalCorrected/demultiplexed/SA147_2.fastq SA147_forward.paired.fastq.gz SA147_forward.singletons.fastq.gz SA147_reverse.paired.fastq.gz SA147_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA147_1.fastq /home/working/FinalCorrected/demultiplexed/SA147_2.fastq SA147_forward.paired.fastq.gz SA147_forward.singletons.fastq.gz SA147_reverse.paired.fastq.gz SA147_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:57:37 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA147/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 52843 Both Surviving: 9208 (17.43%) Forward Only Surviving: 2868 (5.43%) Reverse Only Surviving: 630 (1.19%) Dropped: 40137 (75.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA147_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA147_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA147_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA147_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA147_forward.paired.headclipped.fastq.gz SA147_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA147 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA147_forward.paired.headclipped.fastq.gz
[FLASH]     SA147_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA147.extendedFrags.fastq.gz
[FLASH]     ./SA147.notCombined_1.fastq.gz
[FLASH]     ./SA147.notCombined_2.fastq.gz
[FLASH]     ./SA147.hist
[FLASH]     ./SA147.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9208 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9208
[FLASH]     Combined pairs:   3170
[FLASH]     Uncombined pairs: 6038
[FLASH]     Percent combined: 34.43%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.103 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3170 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA147/SA147_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA147_centroids.fasta --uc SA147.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3170 sequences and identified 1383 clusters (clustering threshold 1.00) - 67 clusters (representing 1555 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA15 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA15_1.fastq /home/working/FinalCorrected/demultiplexed/SA15_2.fastq SA15_forward.paired.fastq.gz SA15_forward.singletons.fastq.gz SA15_reverse.paired.fastq.gz SA15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA15_1.fastq /home/working/FinalCorrected/demultiplexed/SA15_2.fastq SA15_forward.paired.fastq.gz SA15_forward.singletons.fastq.gz SA15_reverse.paired.fastq.gz SA15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:57:50 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA15/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 86082 Both Surviving: 26672 (30.98%) Forward Only Surviving: 6078 (7.06%) Reverse Only Surviving: 1725 (2.00%) Dropped: 51607 (59.95%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA15_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA15_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA15_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA15_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA15_forward.paired.headclipped.fastq.gz SA15_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA15 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA15_forward.paired.headclipped.fastq.gz
[FLASH]     SA15_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA15.extendedFrags.fastq.gz
[FLASH]     ./SA15.notCombined_1.fastq.gz
[FLASH]     ./SA15.notCombined_2.fastq.gz
[FLASH]     ./SA15.hist
[FLASH]     ./SA15.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 26672 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      26672
[FLASH]     Combined pairs:   889
[FLASH]     Uncombined pairs: 25783
[FLASH]     Percent combined: 3.33%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.154 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

889 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA15/SA15_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA15_centroids.fasta --uc SA15.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 889 sequences and identified 586 clusters (clustering threshold 1.00) - 20 clusters (representing 252 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA153 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA153_1.fastq /home/working/FinalCorrected/demultiplexed/SA153_2.fastq SA153_forward.paired.fastq.gz SA153_forward.singletons.fastq.gz SA153_reverse.paired.fastq.gz SA153_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA153_1.fastq /home/working/FinalCorrected/demultiplexed/SA153_2.fastq SA153_forward.paired.fastq.gz SA153_forward.singletons.fastq.gz SA153_reverse.paired.fastq.gz SA153_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:58:05 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA153/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 54014 Both Surviving: 11939 (22.10%) Forward Only Surviving: 2234 (4.14%) Reverse Only Surviving: 992 (1.84%) Dropped: 38849 (71.92%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA153_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA153_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA153_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA153_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA153_forward.paired.headclipped.fastq.gz SA153_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA153 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA153_forward.paired.headclipped.fastq.gz
[FLASH]     SA153_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA153.extendedFrags.fastq.gz
[FLASH]     ./SA153.notCombined_1.fastq.gz
[FLASH]     ./SA153.notCombined_2.fastq.gz
[FLASH]     ./SA153.hist
[FLASH]     ./SA153.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11939 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11939
[FLASH]     Combined pairs:   1679
[FLASH]     Uncombined pairs: 10260
[FLASH]     Percent combined: 14.06%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.068 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1679 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA153/SA153_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA153_centroids.fasta --uc SA153.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1679 sequences and identified 1046 clusters (clustering threshold 1.00) - 29 clusters (representing 564 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA155 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA155_1.fastq /home/working/FinalCorrected/demultiplexed/SA155_2.fastq SA155_forward.paired.fastq.gz SA155_forward.singletons.fastq.gz SA155_reverse.paired.fastq.gz SA155_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA155_1.fastq /home/working/FinalCorrected/demultiplexed/SA155_2.fastq SA155_forward.paired.fastq.gz SA155_forward.singletons.fastq.gz SA155_reverse.paired.fastq.gz SA155_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:58:15 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA155/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31128 Both Surviving: 752 (2.42%) Forward Only Surviving: 717 (2.30%) Reverse Only Surviving: 480 (1.54%) Dropped: 29179 (93.74%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA155_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA155_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA155_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA155_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA155_forward.paired.headclipped.fastq.gz SA155_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA155 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA155_forward.paired.headclipped.fastq.gz
[FLASH]     SA155_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA155.extendedFrags.fastq.gz
[FLASH]     ./SA155.notCombined_1.fastq.gz
[FLASH]     ./SA155.notCombined_2.fastq.gz
[FLASH]     ./SA155.hist
[FLASH]     ./SA155.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 752 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      752
[FLASH]     Combined pairs:   561
[FLASH]     Uncombined pairs: 191
[FLASH]     Percent combined: 74.60%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.020 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

561 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA155/SA155_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA155_centroids.fasta --uc SA155.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 561 sequences and identified 321 clusters (clustering threshold 1.00) - 8 clusters (representing 201 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA156 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA156_1.fastq /home/working/FinalCorrected/demultiplexed/SA156_2.fastq SA156_forward.paired.fastq.gz SA156_forward.singletons.fastq.gz SA156_reverse.paired.fastq.gz SA156_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA156_1.fastq /home/working/FinalCorrected/demultiplexed/SA156_2.fastq SA156_forward.paired.fastq.gz SA156_forward.singletons.fastq.gz SA156_reverse.paired.fastq.gz SA156_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:58:21 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA156/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 53417 Both Surviving: 2365 (4.43%) Forward Only Surviving: 1836 (3.44%) Reverse Only Surviving: 713 (1.33%) Dropped: 48503 (90.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA156_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA156_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA156_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA156_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA156_forward.paired.headclipped.fastq.gz SA156_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA156 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA156_forward.paired.headclipped.fastq.gz
[FLASH]     SA156_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA156.extendedFrags.fastq.gz
[FLASH]     ./SA156.notCombined_1.fastq.gz
[FLASH]     ./SA156.notCombined_2.fastq.gz
[FLASH]     ./SA156.hist
[FLASH]     ./SA156.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2365 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2365
[FLASH]     Combined pairs:   312
[FLASH]     Uncombined pairs: 2053
[FLASH]     Percent combined: 13.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.021 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

312 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA156/SA156_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA156_centroids.fasta --uc SA156.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 312 sequences and identified 258 clusters (clustering threshold 1.00) - 3 clusters (representing 21 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA157 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA157_1.fastq /home/working/FinalCorrected/demultiplexed/SA157_2.fastq SA157_forward.paired.fastq.gz SA157_forward.singletons.fastq.gz SA157_reverse.paired.fastq.gz SA157_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA157_1.fastq /home/working/FinalCorrected/demultiplexed/SA157_2.fastq SA157_forward.paired.fastq.gz SA157_forward.singletons.fastq.gz SA157_reverse.paired.fastq.gz SA157_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:58:27 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA157/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37950 Both Surviving: 6388 (16.83%) Forward Only Surviving: 1776 (4.68%) Reverse Only Surviving: 666 (1.75%) Dropped: 29120 (76.73%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA157_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA157_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA157_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA157_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA157_forward.paired.headclipped.fastq.gz SA157_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA157 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA157_forward.paired.headclipped.fastq.gz
[FLASH]     SA157_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA157.extendedFrags.fastq.gz
[FLASH]     ./SA157.notCombined_1.fastq.gz
[FLASH]     ./SA157.notCombined_2.fastq.gz
[FLASH]     ./SA157.hist
[FLASH]     ./SA157.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6388 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6388
[FLASH]     Combined pairs:   41
[FLASH]     Uncombined pairs: 6347
[FLASH]     Percent combined: 0.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.041 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

41 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA157/SA157_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA157_centroids.fasta --uc SA157.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 41 sequences and identified 35 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA160 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA160_1.fastq /home/working/FinalCorrected/demultiplexed/SA160_2.fastq SA160_forward.paired.fastq.gz SA160_forward.singletons.fastq.gz SA160_reverse.paired.fastq.gz SA160_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA160_1.fastq /home/working/FinalCorrected/demultiplexed/SA160_2.fastq SA160_forward.paired.fastq.gz SA160_forward.singletons.fastq.gz SA160_reverse.paired.fastq.gz SA160_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:58:33 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA160/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 39762 Both Surviving: 6875 (17.29%) Forward Only Surviving: 1725 (4.34%) Reverse Only Surviving: 618 (1.55%) Dropped: 30544 (76.82%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA160_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA160_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA160_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA160_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA160_forward.paired.headclipped.fastq.gz SA160_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA160 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA160_forward.paired.headclipped.fastq.gz
[FLASH]     SA160_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA160.extendedFrags.fastq.gz
[FLASH]     ./SA160.notCombined_1.fastq.gz
[FLASH]     ./SA160.notCombined_2.fastq.gz
[FLASH]     ./SA160.hist
[FLASH]     ./SA160.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6875 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6875
[FLASH]     Combined pairs:   758
[FLASH]     Uncombined pairs: 6117
[FLASH]     Percent combined: 11.03%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.039 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

758 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA160/SA160_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA160_centroids.fasta --uc SA160.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 758 sequences and identified 367 clusters (clustering threshold 1.00) - 14 clusters (representing 367 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA162 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA162_1.fastq /home/working/FinalCorrected/demultiplexed/SA162_2.fastq SA162_forward.paired.fastq.gz SA162_forward.singletons.fastq.gz SA162_reverse.paired.fastq.gz SA162_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA162_1.fastq /home/working/FinalCorrected/demultiplexed/SA162_2.fastq SA162_forward.paired.fastq.gz SA162_forward.singletons.fastq.gz SA162_reverse.paired.fastq.gz SA162_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:58:40 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA162/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37273 Both Surviving: 11447 (30.71%) Forward Only Surviving: 3157 (8.47%) Reverse Only Surviving: 439 (1.18%) Dropped: 22230 (59.64%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA162_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA162_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SA162_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA162_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA162_forward.paired.headclipped.fastq.gz SA162_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA162 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA162_forward.paired.headclipped.fastq.gz
[FLASH]     SA162_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA162.extendedFrags.fastq.gz
[FLASH]     ./SA162.notCombined_1.fastq.gz
[FLASH]     ./SA162.notCombined_2.fastq.gz
[FLASH]     ./SA162.hist
[FLASH]     ./SA162.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11447 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11447
[FLASH]     Combined pairs:   5798
[FLASH]     Uncombined pairs: 5649
[FLASH]     Percent combined: 50.65%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.163 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5798 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA162/SA162_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA162_centroids.fasta --uc SA162.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5798 sequences and identified 2921 clusters (clustering threshold 1.00) - 79 clusters (representing 2543 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA164 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA164_1.fastq /home/working/FinalCorrected/demultiplexed/SA164_2.fastq SA164_forward.paired.fastq.gz SA164_forward.singletons.fastq.gz SA164_reverse.paired.fastq.gz SA164_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA164_1.fastq /home/working/FinalCorrected/demultiplexed/SA164_2.fastq SA164_forward.paired.fastq.gz SA164_forward.singletons.fastq.gz SA164_reverse.paired.fastq.gz SA164_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:58:52 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA164/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SA164_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA164_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA164_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA164_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA164_forward.paired.headclipped.fastq.gz SA164_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA164 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA164_forward.paired.headclipped.fastq.gz
[FLASH]     SA164_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA164.extendedFrags.fastq.gz
[FLASH]     ./SA164.notCombined_1.fastq.gz
[FLASH]     ./SA164.notCombined_2.fastq.gz
[FLASH]     ./SA164.hist
[FLASH]     ./SA164.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA164/SA164_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA164_centroids.fasta --uc SA164.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA165 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA165_1.fastq /home/working/FinalCorrected/demultiplexed/SA165_2.fastq SA165_forward.paired.fastq.gz SA165_forward.singletons.fastq.gz SA165_reverse.paired.fastq.gz SA165_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA165_1.fastq /home/working/FinalCorrected/demultiplexed/SA165_2.fastq SA165_forward.paired.fastq.gz SA165_forward.singletons.fastq.gz SA165_reverse.paired.fastq.gz SA165_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:58:54 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA165/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31845 Both Surviving: 9829 (30.87%) Forward Only Surviving: 2198 (6.90%) Reverse Only Surviving: 609 (1.91%) Dropped: 19209 (60.32%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA165_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA165_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA165_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA165_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA165_forward.paired.headclipped.fastq.gz SA165_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA165 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA165_forward.paired.headclipped.fastq.gz
[FLASH]     SA165_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA165.extendedFrags.fastq.gz
[FLASH]     ./SA165.notCombined_1.fastq.gz
[FLASH]     ./SA165.notCombined_2.fastq.gz
[FLASH]     ./SA165.hist
[FLASH]     ./SA165.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9829 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9829
[FLASH]     Combined pairs:   4629
[FLASH]     Uncombined pairs: 5200
[FLASH]     Percent combined: 47.10%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.145 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4629 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA165/SA165_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA165_centroids.fasta --uc SA165.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4629 sequences and identified 2550 clusters (clustering threshold 1.00) - 59 clusters (representing 1820 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA166 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA166_1.fastq /home/working/FinalCorrected/demultiplexed/SA166_2.fastq SA166_forward.paired.fastq.gz SA166_forward.singletons.fastq.gz SA166_reverse.paired.fastq.gz SA166_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA166_1.fastq /home/working/FinalCorrected/demultiplexed/SA166_2.fastq SA166_forward.paired.fastq.gz SA166_forward.singletons.fastq.gz SA166_reverse.paired.fastq.gz SA166_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:59:06 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA166/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22760 Both Surviving: 9143 (40.17%) Forward Only Surviving: 1993 (8.76%) Reverse Only Surviving: 322 (1.41%) Dropped: 11302 (49.66%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA166_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA166_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA166_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA166_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA166_forward.paired.headclipped.fastq.gz SA166_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA166 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA166_forward.paired.headclipped.fastq.gz
[FLASH]     SA166_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA166.extendedFrags.fastq.gz
[FLASH]     ./SA166.notCombined_1.fastq.gz
[FLASH]     ./SA166.notCombined_2.fastq.gz
[FLASH]     ./SA166.hist
[FLASH]     ./SA166.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9143 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9143
[FLASH]     Combined pairs:   4628
[FLASH]     Uncombined pairs: 4515
[FLASH]     Percent combined: 50.62%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.131 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4628 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA166/SA166_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA166_centroids.fasta --uc SA166.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4628 sequences and identified 2605 clusters (clustering threshold 1.00) - 110 clusters (representing 1757 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA171 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA171_1.fastq /home/working/FinalCorrected/demultiplexed/SA171_2.fastq SA171_forward.paired.fastq.gz SA171_forward.singletons.fastq.gz SA171_reverse.paired.fastq.gz SA171_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA171_1.fastq /home/working/FinalCorrected/demultiplexed/SA171_2.fastq SA171_forward.paired.fastq.gz SA171_forward.singletons.fastq.gz SA171_reverse.paired.fastq.gz SA171_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:59:16 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA171/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 7014 Both Surviving: 2316 (33.02%) Forward Only Surviving: 562 (8.01%) Reverse Only Surviving: 193 (2.75%) Dropped: 3943 (56.22%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SA171_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA171_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA171_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA171_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA171_forward.paired.headclipped.fastq.gz SA171_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA171 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA171_forward.paired.headclipped.fastq.gz
[FLASH]     SA171_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA171.extendedFrags.fastq.gz
[FLASH]     ./SA171.notCombined_1.fastq.gz
[FLASH]     ./SA171.notCombined_2.fastq.gz
[FLASH]     ./SA171.hist
[FLASH]     ./SA171.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2316 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2316
[FLASH]     Combined pairs:   456
[FLASH]     Uncombined pairs: 1860
[FLASH]     Percent combined: 19.69%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.023 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

456 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA171/SA171_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA171_centroids.fasta --uc SA171.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 456 sequences and identified 336 clusters (clustering threshold 1.00) - 8 clusters (representing 110 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA172 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA172_1.fastq /home/working/FinalCorrected/demultiplexed/SA172_2.fastq SA172_forward.paired.fastq.gz SA172_forward.singletons.fastq.gz SA172_reverse.paired.fastq.gz SA172_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA172_1.fastq /home/working/FinalCorrected/demultiplexed/SA172_2.fastq SA172_forward.paired.fastq.gz SA172_forward.singletons.fastq.gz SA172_reverse.paired.fastq.gz SA172_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:59:20 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA172/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29978 Both Surviving: 9181 (30.63%) Forward Only Surviving: 3718 (12.40%) Reverse Only Surviving: 329 (1.10%) Dropped: 16750 (55.87%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA172_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA172_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA172_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA172_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA172_forward.paired.headclipped.fastq.gz SA172_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA172 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA172_forward.paired.headclipped.fastq.gz
[FLASH]     SA172_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA172.extendedFrags.fastq.gz
[FLASH]     ./SA172.notCombined_1.fastq.gz
[FLASH]     ./SA172.notCombined_2.fastq.gz
[FLASH]     ./SA172.hist
[FLASH]     ./SA172.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9181 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9181
[FLASH]     Combined pairs:   4464
[FLASH]     Uncombined pairs: 4717
[FLASH]     Percent combined: 48.62%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.128 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4464 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA172/SA172_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA172_centroids.fasta --uc SA172.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4464 sequences and identified 1339 clusters (clustering threshold 1.00) - 127 clusters (representing 2965 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA174 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA174_1.fastq /home/working/FinalCorrected/demultiplexed/SA174_2.fastq SA174_forward.paired.fastq.gz SA174_forward.singletons.fastq.gz SA174_reverse.paired.fastq.gz SA174_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA174_1.fastq /home/working/FinalCorrected/demultiplexed/SA174_2.fastq SA174_forward.paired.fastq.gz SA174_forward.singletons.fastq.gz SA174_reverse.paired.fastq.gz SA174_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:59:30 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA174/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 24531 Both Surviving: 7385 (30.10%) Forward Only Surviving: 1759 (7.17%) Reverse Only Surviving: 279 (1.14%) Dropped: 15108 (61.59%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA174_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA174_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA174_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA174_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA174_forward.paired.headclipped.fastq.gz SA174_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA174 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA174_forward.paired.headclipped.fastq.gz
[FLASH]     SA174_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA174.extendedFrags.fastq.gz
[FLASH]     ./SA174.notCombined_1.fastq.gz
[FLASH]     ./SA174.notCombined_2.fastq.gz
[FLASH]     ./SA174.hist
[FLASH]     ./SA174.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7385 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7385
[FLASH]     Combined pairs:   2705
[FLASH]     Uncombined pairs: 4680
[FLASH]     Percent combined: 36.63%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.085 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2705 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA174/SA174_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA174_centroids.fasta --uc SA174.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2705 sequences and identified 1744 clusters (clustering threshold 1.00) - 31 clusters (representing 808 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA177 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA177_1.fastq /home/working/FinalCorrected/demultiplexed/SA177_2.fastq SA177_forward.paired.fastq.gz SA177_forward.singletons.fastq.gz SA177_reverse.paired.fastq.gz SA177_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA177_1.fastq /home/working/FinalCorrected/demultiplexed/SA177_2.fastq SA177_forward.paired.fastq.gz SA177_forward.singletons.fastq.gz SA177_reverse.paired.fastq.gz SA177_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:59:38 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA177/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18782 Both Surviving: 385 (2.05%) Forward Only Surviving: 385 (2.05%) Reverse Only Surviving: 127 (0.68%) Dropped: 17885 (95.22%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA177_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA177_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA177_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA177_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA177_forward.paired.headclipped.fastq.gz SA177_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA177 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA177_forward.paired.headclipped.fastq.gz
[FLASH]     SA177_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA177.extendedFrags.fastq.gz
[FLASH]     ./SA177.notCombined_1.fastq.gz
[FLASH]     ./SA177.notCombined_2.fastq.gz
[FLASH]     ./SA177.hist
[FLASH]     ./SA177.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 385 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      385
[FLASH]     Combined pairs:   258
[FLASH]     Uncombined pairs: 127
[FLASH]     Percent combined: 67.01%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.008 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

258 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA177/SA177_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA177_centroids.fasta --uc SA177.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 258 sequences and identified 176 clusters (clustering threshold 1.00) - 3 clusters (representing 63 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA180 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA180_1.fastq /home/working/FinalCorrected/demultiplexed/SA180_2.fastq SA180_forward.paired.fastq.gz SA180_forward.singletons.fastq.gz SA180_reverse.paired.fastq.gz SA180_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA180_1.fastq /home/working/FinalCorrected/demultiplexed/SA180_2.fastq SA180_forward.paired.fastq.gz SA180_forward.singletons.fastq.gz SA180_reverse.paired.fastq.gz SA180_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:59:40 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA180/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15643 Both Surviving: 717 (4.58%) Forward Only Surviving: 310 (1.98%) Reverse Only Surviving: 175 (1.12%) Dropped: 14441 (92.32%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA180_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA180_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA180_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA180_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA180_forward.paired.headclipped.fastq.gz SA180_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA180 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA180_forward.paired.headclipped.fastq.gz
[FLASH]     SA180_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA180.extendedFrags.fastq.gz
[FLASH]     ./SA180.notCombined_1.fastq.gz
[FLASH]     ./SA180.notCombined_2.fastq.gz
[FLASH]     ./SA180.hist
[FLASH]     ./SA180.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 717 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      717
[FLASH]     Combined pairs:   223
[FLASH]     Uncombined pairs: 494
[FLASH]     Percent combined: 31.10%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.010 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

223 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA180/SA180_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA180_centroids.fasta --uc SA180.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 223 sequences and identified 142 clusters (clustering threshold 1.00) - 2 clusters (representing 70 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA182 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA182_1.fastq /home/working/FinalCorrected/demultiplexed/SA182_2.fastq SA182_forward.paired.fastq.gz SA182_forward.singletons.fastq.gz SA182_reverse.paired.fastq.gz SA182_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA182_1.fastq /home/working/FinalCorrected/demultiplexed/SA182_2.fastq SA182_forward.paired.fastq.gz SA182_forward.singletons.fastq.gz SA182_reverse.paired.fastq.gz SA182_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:59:43 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA182/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14801 Both Surviving: 2276 (15.38%) Forward Only Surviving: 487 (3.29%) Reverse Only Surviving: 441 (2.98%) Dropped: 11597 (78.35%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SA182_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA182_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA182_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA182_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA182_forward.paired.headclipped.fastq.gz SA182_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA182 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA182_forward.paired.headclipped.fastq.gz
[FLASH]     SA182_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA182.extendedFrags.fastq.gz
[FLASH]     ./SA182.notCombined_1.fastq.gz
[FLASH]     ./SA182.notCombined_2.fastq.gz
[FLASH]     ./SA182.hist
[FLASH]     ./SA182.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2276 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2276
[FLASH]     Combined pairs:   1194
[FLASH]     Uncombined pairs: 1082
[FLASH]     Percent combined: 52.46%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.039 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1194 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA182/SA182_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA182_centroids.fasta --uc SA182.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1194 sequences and identified 669 clusters (clustering threshold 1.00) - 16 clusters (representing 465 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA189 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA189_1.fastq /home/working/FinalCorrected/demultiplexed/SA189_2.fastq SA189_forward.paired.fastq.gz SA189_forward.singletons.fastq.gz SA189_reverse.paired.fastq.gz SA189_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA189_1.fastq /home/working/FinalCorrected/demultiplexed/SA189_2.fastq SA189_forward.paired.fastq.gz SA189_forward.singletons.fastq.gz SA189_reverse.paired.fastq.gz SA189_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:59:48 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA189/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 40281 Both Surviving: 6626 (16.45%) Forward Only Surviving: 1736 (4.31%) Reverse Only Surviving: 589 (1.46%) Dropped: 31330 (77.78%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA189_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA189_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA189_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA189_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA189_forward.paired.headclipped.fastq.gz SA189_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA189 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA189_forward.paired.headclipped.fastq.gz
[FLASH]     SA189_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA189.extendedFrags.fastq.gz
[FLASH]     ./SA189.notCombined_1.fastq.gz
[FLASH]     ./SA189.notCombined_2.fastq.gz
[FLASH]     ./SA189.hist
[FLASH]     ./SA189.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6626 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6626
[FLASH]     Combined pairs:   1042
[FLASH]     Uncombined pairs: 5584
[FLASH]     Percent combined: 15.73%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.042 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1042 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA189/SA189_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA189_centroids.fasta --uc SA189.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1042 sequences and identified 562 clusters (clustering threshold 1.00) - 14 clusters (representing 431 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA19 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA19_1.fastq /home/working/FinalCorrected/demultiplexed/SA19_2.fastq SA19_forward.paired.fastq.gz SA19_forward.singletons.fastq.gz SA19_reverse.paired.fastq.gz SA19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA19_1.fastq /home/working/FinalCorrected/demultiplexed/SA19_2.fastq SA19_forward.paired.fastq.gz SA19_forward.singletons.fastq.gz SA19_reverse.paired.fastq.gz SA19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 10:59:58 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA19/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 97463 Both Surviving: 21686 (22.25%) Forward Only Surviving: 6419 (6.59%) Reverse Only Surviving: 1547 (1.59%) Dropped: 67811 (69.58%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA19_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA19_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA19_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA19_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA19_forward.paired.headclipped.fastq.gz SA19_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA19 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA19_forward.paired.headclipped.fastq.gz
[FLASH]     SA19_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA19.extendedFrags.fastq.gz
[FLASH]     ./SA19.notCombined_1.fastq.gz
[FLASH]     ./SA19.notCombined_2.fastq.gz
[FLASH]     ./SA19.hist
[FLASH]     ./SA19.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 21686 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      21686
[FLASH]     Combined pairs:   1454
[FLASH]     Uncombined pairs: 20232
[FLASH]     Percent combined: 6.70%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.127 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1454 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA19/SA19_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA19_centroids.fasta --uc SA19.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1454 sequences and identified 925 clusters (clustering threshold 1.00) - 33 clusters (representing 447 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA190 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA190_1.fastq /home/working/FinalCorrected/demultiplexed/SA190_2.fastq SA190_forward.paired.fastq.gz SA190_forward.singletons.fastq.gz SA190_reverse.paired.fastq.gz SA190_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA190_1.fastq /home/working/FinalCorrected/demultiplexed/SA190_2.fastq SA190_forward.paired.fastq.gz SA190_forward.singletons.fastq.gz SA190_reverse.paired.fastq.gz SA190_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:00:12 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA190/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 57953 Both Surviving: 708 (1.22%) Forward Only Surviving: 1425 (2.46%) Reverse Only Surviving: 656 (1.13%) Dropped: 55164 (95.19%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA190_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA190_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA190_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA190_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA190_forward.paired.headclipped.fastq.gz SA190_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA190 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA190_forward.paired.headclipped.fastq.gz
[FLASH]     SA190_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA190.extendedFrags.fastq.gz
[FLASH]     ./SA190.notCombined_1.fastq.gz
[FLASH]     ./SA190.notCombined_2.fastq.gz
[FLASH]     ./SA190.hist
[FLASH]     ./SA190.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 708 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      708
[FLASH]     Combined pairs:   189
[FLASH]     Uncombined pairs: 519
[FLASH]     Percent combined: 26.69%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.011 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

189 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA190/SA190_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA190_centroids.fasta --uc SA190.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 189 sequences and identified 153 clusters (clustering threshold 1.00) - 1 clusters (representing 8 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA196 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA196_1.fastq /home/working/FinalCorrected/demultiplexed/SA196_2.fastq SA196_forward.paired.fastq.gz SA196_forward.singletons.fastq.gz SA196_reverse.paired.fastq.gz SA196_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA196_1.fastq /home/working/FinalCorrected/demultiplexed/SA196_2.fastq SA196_forward.paired.fastq.gz SA196_forward.singletons.fastq.gz SA196_reverse.paired.fastq.gz SA196_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:00:17 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA196/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 30293 Both Surviving: 431 (1.42%) Forward Only Surviving: 604 (1.99%) Reverse Only Surviving: 436 (1.44%) Dropped: 28822 (95.14%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA196_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA196_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA196_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA196_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA196_forward.paired.headclipped.fastq.gz SA196_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA196 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA196_forward.paired.headclipped.fastq.gz
[FLASH]     SA196_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA196.extendedFrags.fastq.gz
[FLASH]     ./SA196.notCombined_1.fastq.gz
[FLASH]     ./SA196.notCombined_2.fastq.gz
[FLASH]     ./SA196.hist
[FLASH]     ./SA196.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 431 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      431
[FLASH]     Combined pairs:   304
[FLASH]     Uncombined pairs: 127
[FLASH]     Percent combined: 70.53%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.012 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

304 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA196/SA196_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA196_centroids.fasta --uc SA196.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 304 sequences and identified 156 clusters (clustering threshold 1.00) - 6 clusters (representing 143 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA198 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA198_1.fastq /home/working/FinalCorrected/demultiplexed/SA198_2.fastq SA198_forward.paired.fastq.gz SA198_forward.singletons.fastq.gz SA198_reverse.paired.fastq.gz SA198_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA198_1.fastq /home/working/FinalCorrected/demultiplexed/SA198_2.fastq SA198_forward.paired.fastq.gz SA198_forward.singletons.fastq.gz SA198_reverse.paired.fastq.gz SA198_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:00:20 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA198/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14709 Both Surviving: 3771 (25.64%) Forward Only Surviving: 1102 (7.49%) Reverse Only Surviving: 201 (1.37%) Dropped: 9635 (65.50%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA198_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA198_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA198_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA198_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA198_forward.paired.headclipped.fastq.gz SA198_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA198 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA198_forward.paired.headclipped.fastq.gz
[FLASH]     SA198_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA198.extendedFrags.fastq.gz
[FLASH]     ./SA198.notCombined_1.fastq.gz
[FLASH]     ./SA198.notCombined_2.fastq.gz
[FLASH]     ./SA198.hist
[FLASH]     ./SA198.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3771 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3771
[FLASH]     Combined pairs:   660
[FLASH]     Uncombined pairs: 3111
[FLASH]     Percent combined: 17.50%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.028 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

660 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA198/SA198_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA198_centroids.fasta --uc SA198.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 660 sequences and identified 452 clusters (clustering threshold 1.00) - 12 clusters (representing 189 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA199 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA199_1.fastq /home/working/FinalCorrected/demultiplexed/SA199_2.fastq SA199_forward.paired.fastq.gz SA199_forward.singletons.fastq.gz SA199_reverse.paired.fastq.gz SA199_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA199_1.fastq /home/working/FinalCorrected/demultiplexed/SA199_2.fastq SA199_forward.paired.fastq.gz SA199_forward.singletons.fastq.gz SA199_reverse.paired.fastq.gz SA199_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:00:23 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA199/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 1 Both Surviving: 0 (0.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 1 (100.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SA199_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA199_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA199_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA199_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA199_forward.paired.headclipped.fastq.gz SA199_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA199 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA199_forward.paired.headclipped.fastq.gz
[FLASH]     SA199_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA199.extendedFrags.fastq.gz
[FLASH]     ./SA199.notCombined_1.fastq.gz
[FLASH]     ./SA199.notCombined_2.fastq.gz
[FLASH]     ./SA199.hist
[FLASH]     ./SA199.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA199/SA199_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA199_centroids.fasta --uc SA199.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA2 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA2_1.fastq /home/working/FinalCorrected/demultiplexed/SA2_2.fastq SA2_forward.paired.fastq.gz SA2_forward.singletons.fastq.gz SA2_reverse.paired.fastq.gz SA2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA2_1.fastq /home/working/FinalCorrected/demultiplexed/SA2_2.fastq SA2_forward.paired.fastq.gz SA2_forward.singletons.fastq.gz SA2_reverse.paired.fastq.gz SA2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:00:29 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA2/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 96310 Both Surviving: 30413 (31.58%) Forward Only Surviving: 7502 (7.79%) Reverse Only Surviving: 1434 (1.49%) Dropped: 56961 (59.14%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA2_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA2_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA2_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA2_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA2_forward.paired.headclipped.fastq.gz SA2_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA2 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA2_forward.paired.headclipped.fastq.gz
[FLASH]     SA2_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA2.extendedFrags.fastq.gz
[FLASH]     ./SA2.notCombined_1.fastq.gz
[FLASH]     ./SA2.notCombined_2.fastq.gz
[FLASH]     ./SA2.hist
[FLASH]     ./SA2.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 30413 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      30413
[FLASH]     Combined pairs:   18029
[FLASH]     Uncombined pairs: 12384
[FLASH]     Percent combined: 59.28%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.484 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

18029 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA2/SA2_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA2_centroids.fasta --uc SA2.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 18029 sequences and identified 5296 clusters (clustering threshold 1.00) - 434 clusters (representing 12156 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA202 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA202_1.fastq /home/working/FinalCorrected/demultiplexed/SA202_2.fastq SA202_forward.paired.fastq.gz SA202_forward.singletons.fastq.gz SA202_reverse.paired.fastq.gz SA202_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA202_1.fastq /home/working/FinalCorrected/demultiplexed/SA202_2.fastq SA202_forward.paired.fastq.gz SA202_forward.singletons.fastq.gz SA202_reverse.paired.fastq.gz SA202_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:01:10 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA202/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 102107 Both Surviving: 26482 (25.94%) Forward Only Surviving: 7417 (7.26%) Reverse Only Surviving: 1638 (1.60%) Dropped: 66570 (65.20%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA202_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA202_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SA202_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA202_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA202_forward.paired.headclipped.fastq.gz SA202_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA202 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA202_forward.paired.headclipped.fastq.gz
[FLASH]     SA202_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA202.extendedFrags.fastq.gz
[FLASH]     ./SA202.notCombined_1.fastq.gz
[FLASH]     ./SA202.notCombined_2.fastq.gz
[FLASH]     ./SA202.hist
[FLASH]     ./SA202.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 26482 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      26482
[FLASH]     Combined pairs:   2971
[FLASH]     Uncombined pairs: 23511
[FLASH]     Percent combined: 11.22%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.159 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2971 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA202/SA202_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA202_centroids.fasta --uc SA202.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2971 sequences and identified 1577 clusters (clustering threshold 1.00) - 41 clusters (representing 1228 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA21 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA21_1.fastq /home/working/FinalCorrected/demultiplexed/SA21_2.fastq SA21_forward.paired.fastq.gz SA21_forward.singletons.fastq.gz SA21_reverse.paired.fastq.gz SA21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA21_1.fastq /home/working/FinalCorrected/demultiplexed/SA21_2.fastq SA21_forward.paired.fastq.gz SA21_forward.singletons.fastq.gz SA21_reverse.paired.fastq.gz SA21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:01:28 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA21/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37070 Both Surviving: 5135 (13.85%) Forward Only Surviving: 1558 (4.20%) Reverse Only Surviving: 623 (1.68%) Dropped: 29754 (80.26%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA21_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA21_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA21_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA21_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA21_forward.paired.headclipped.fastq.gz SA21_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA21 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA21_forward.paired.headclipped.fastq.gz
[FLASH]     SA21_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA21.extendedFrags.fastq.gz
[FLASH]     ./SA21.notCombined_1.fastq.gz
[FLASH]     ./SA21.notCombined_2.fastq.gz
[FLASH]     ./SA21.hist
[FLASH]     ./SA21.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5135 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5135
[FLASH]     Combined pairs:   56
[FLASH]     Uncombined pairs: 5079
[FLASH]     Percent combined: 1.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.035 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

56 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA21/SA21_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA21_centroids.fasta --uc SA21.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 56 sequences and identified 49 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA28 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA28_1.fastq /home/working/FinalCorrected/demultiplexed/SA28_2.fastq SA28_forward.paired.fastq.gz SA28_forward.singletons.fastq.gz SA28_reverse.paired.fastq.gz SA28_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA28_1.fastq /home/working/FinalCorrected/demultiplexed/SA28_2.fastq SA28_forward.paired.fastq.gz SA28_forward.singletons.fastq.gz SA28_reverse.paired.fastq.gz SA28_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:01:34 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA28/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 55630 Both Surviving: 13372 (24.04%) Forward Only Surviving: 4577 (8.23%) Reverse Only Surviving: 769 (1.38%) Dropped: 36912 (66.35%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA28_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA28_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA28_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA28_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA28_forward.paired.headclipped.fastq.gz SA28_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA28 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA28_forward.paired.headclipped.fastq.gz
[FLASH]     SA28_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA28.extendedFrags.fastq.gz
[FLASH]     ./SA28.notCombined_1.fastq.gz
[FLASH]     ./SA28.notCombined_2.fastq.gz
[FLASH]     ./SA28.hist
[FLASH]     ./SA28.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 13372 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      13372
[FLASH]     Combined pairs:   2894
[FLASH]     Uncombined pairs: 10478
[FLASH]     Percent combined: 21.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.102 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2894 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA28/SA28_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA28_centroids.fasta --uc SA28.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2894 sequences and identified 1162 clusters (clustering threshold 1.00) - 33 clusters (representing 1432 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA31 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA31_1.fastq /home/working/FinalCorrected/demultiplexed/SA31_2.fastq SA31_forward.paired.fastq.gz SA31_forward.singletons.fastq.gz SA31_reverse.paired.fastq.gz SA31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA31_1.fastq /home/working/FinalCorrected/demultiplexed/SA31_2.fastq SA31_forward.paired.fastq.gz SA31_forward.singletons.fastq.gz SA31_reverse.paired.fastq.gz SA31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:01:46 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA31/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 33983 Both Surviving: 11512 (33.88%) Forward Only Surviving: 2671 (7.86%) Reverse Only Surviving: 509 (1.50%) Dropped: 19291 (56.77%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA31_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA31_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA31_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA31_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA31_forward.paired.headclipped.fastq.gz SA31_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA31 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA31_forward.paired.headclipped.fastq.gz
[FLASH]     SA31_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA31.extendedFrags.fastq.gz
[FLASH]     ./SA31.notCombined_1.fastq.gz
[FLASH]     ./SA31.notCombined_2.fastq.gz
[FLASH]     ./SA31.hist
[FLASH]     ./SA31.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11512 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11512
[FLASH]     Combined pairs:   3713
[FLASH]     Uncombined pairs: 7799
[FLASH]     Percent combined: 32.25%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.111 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3713 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA31/SA31_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA31_centroids.fasta --uc SA31.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3713 sequences and identified 1957 clusters (clustering threshold 1.00) - 100 clusters (representing 1543 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA32 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA32_1.fastq /home/working/FinalCorrected/demultiplexed/SA32_2.fastq SA32_forward.paired.fastq.gz SA32_forward.singletons.fastq.gz SA32_reverse.paired.fastq.gz SA32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA32_1.fastq /home/working/FinalCorrected/demultiplexed/SA32_2.fastq SA32_forward.paired.fastq.gz SA32_forward.singletons.fastq.gz SA32_reverse.paired.fastq.gz SA32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:01:56 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA32/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 21785 Both Surviving: 2978 (13.67%) Forward Only Surviving: 887 (4.07%) Reverse Only Surviving: 276 (1.27%) Dropped: 17644 (80.99%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA32_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA32_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA32_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA32_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA32_forward.paired.headclipped.fastq.gz SA32_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA32 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA32_forward.paired.headclipped.fastq.gz
[FLASH]     SA32_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA32.extendedFrags.fastq.gz
[FLASH]     ./SA32.notCombined_1.fastq.gz
[FLASH]     ./SA32.notCombined_2.fastq.gz
[FLASH]     ./SA32.hist
[FLASH]     ./SA32.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2978 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2978
[FLASH]     Combined pairs:   1249
[FLASH]     Uncombined pairs: 1729
[FLASH]     Percent combined: 41.94%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.040 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1249 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA32/SA32_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA32_centroids.fasta --uc SA32.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1249 sequences and identified 682 clusters (clustering threshold 1.00) - 21 clusters (representing 527 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA33 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA33_1.fastq /home/working/FinalCorrected/demultiplexed/SA33_2.fastq SA33_forward.paired.fastq.gz SA33_forward.singletons.fastq.gz SA33_reverse.paired.fastq.gz SA33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA33_1.fastq /home/working/FinalCorrected/demultiplexed/SA33_2.fastq SA33_forward.paired.fastq.gz SA33_forward.singletons.fastq.gz SA33_reverse.paired.fastq.gz SA33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:02:04 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA33/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 98419 Both Surviving: 3584 (3.64%) Forward Only Surviving: 3007 (3.06%) Reverse Only Surviving: 995 (1.01%) Dropped: 90833 (92.29%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA33_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA33_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SA33_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA33_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA33_forward.paired.headclipped.fastq.gz SA33_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA33 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA33_forward.paired.headclipped.fastq.gz
[FLASH]     SA33_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA33.extendedFrags.fastq.gz
[FLASH]     ./SA33.notCombined_1.fastq.gz
[FLASH]     ./SA33.notCombined_2.fastq.gz
[FLASH]     ./SA33.hist
[FLASH]     ./SA33.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3584 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3584
[FLASH]     Combined pairs:   2545
[FLASH]     Uncombined pairs: 1039
[FLASH]     Percent combined: 71.01%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.073 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2545 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA33/SA33_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA33_centroids.fasta --uc SA33.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2545 sequences and identified 1255 clusters (clustering threshold 1.00) - 41 clusters (representing 1163 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA36 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA36_1.fastq /home/working/FinalCorrected/demultiplexed/SA36_2.fastq SA36_forward.paired.fastq.gz SA36_forward.singletons.fastq.gz SA36_reverse.paired.fastq.gz SA36_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA36_1.fastq /home/working/FinalCorrected/demultiplexed/SA36_2.fastq SA36_forward.paired.fastq.gz SA36_forward.singletons.fastq.gz SA36_reverse.paired.fastq.gz SA36_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:02:15 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA36/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 51138 Both Surviving: 7411 (14.49%) Forward Only Surviving: 1797 (3.51%) Reverse Only Surviving: 663 (1.30%) Dropped: 41267 (80.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA36_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA36_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SA36_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SA36_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA36_forward.paired.headclipped.fastq.gz SA36_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA36 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA36_forward.paired.headclipped.fastq.gz
[FLASH]     SA36_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA36.extendedFrags.fastq.gz
[FLASH]     ./SA36.notCombined_1.fastq.gz
[FLASH]     ./SA36.notCombined_2.fastq.gz
[FLASH]     ./SA36.hist
[FLASH]     ./SA36.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7411 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7411
[FLASH]     Combined pairs:   200
[FLASH]     Uncombined pairs: 7211
[FLASH]     Percent combined: 2.70%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.043 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

200 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA36/SA36_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA36_centroids.fasta --uc SA36.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 200 sequences and identified 135 clusters (clustering threshold 1.00) - 6 clusters (representing 43 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA43 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA43_1.fastq /home/working/FinalCorrected/demultiplexed/SA43_2.fastq SA43_forward.paired.fastq.gz SA43_forward.singletons.fastq.gz SA43_reverse.paired.fastq.gz SA43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA43_1.fastq /home/working/FinalCorrected/demultiplexed/SA43_2.fastq SA43_forward.paired.fastq.gz SA43_forward.singletons.fastq.gz SA43_reverse.paired.fastq.gz SA43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:02:21 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA43/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22712 Both Surviving: 5221 (22.99%) Forward Only Surviving: 1302 (5.73%) Reverse Only Surviving: 488 (2.15%) Dropped: 15701 (69.13%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA43_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA43_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA43_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA43_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA43_forward.paired.headclipped.fastq.gz SA43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA43 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA43_forward.paired.headclipped.fastq.gz
[FLASH]     SA43_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA43.extendedFrags.fastq.gz
[FLASH]     ./SA43.notCombined_1.fastq.gz
[FLASH]     ./SA43.notCombined_2.fastq.gz
[FLASH]     ./SA43.hist
[FLASH]     ./SA43.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5221 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5221
[FLASH]     Combined pairs:   57
[FLASH]     Uncombined pairs: 5164
[FLASH]     Percent combined: 1.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.035 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

57 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA43/SA43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA43_centroids.fasta --uc SA43.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 57 sequences and identified 48 clusters (clustering threshold 1.00) - 1 clusters (representing 6 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA46 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA46_1.fastq /home/working/FinalCorrected/demultiplexed/SA46_2.fastq SA46_forward.paired.fastq.gz SA46_forward.singletons.fastq.gz SA46_reverse.paired.fastq.gz SA46_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA46_1.fastq /home/working/FinalCorrected/demultiplexed/SA46_2.fastq SA46_forward.paired.fastq.gz SA46_forward.singletons.fastq.gz SA46_reverse.paired.fastq.gz SA46_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:02:27 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA46/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 62158 Both Surviving: 14590 (23.47%) Forward Only Surviving: 4249 (6.84%) Reverse Only Surviving: 926 (1.49%) Dropped: 42393 (68.20%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA46_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA46_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA46_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA46_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA46_forward.paired.headclipped.fastq.gz SA46_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA46 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA46_forward.paired.headclipped.fastq.gz
[FLASH]     SA46_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA46.extendedFrags.fastq.gz
[FLASH]     ./SA46.notCombined_1.fastq.gz
[FLASH]     ./SA46.notCombined_2.fastq.gz
[FLASH]     ./SA46.hist
[FLASH]     ./SA46.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 14590 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      14590
[FLASH]     Combined pairs:   1007
[FLASH]     Uncombined pairs: 13583
[FLASH]     Percent combined: 6.90%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.087 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1007 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA46/SA46_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA46_centroids.fasta --uc SA46.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1007 sequences and identified 562 clusters (clustering threshold 1.00) - 13 clusters (representing 376 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA47 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA47_1.fastq /home/working/FinalCorrected/demultiplexed/SA47_2.fastq SA47_forward.paired.fastq.gz SA47_forward.singletons.fastq.gz SA47_reverse.paired.fastq.gz SA47_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA47_1.fastq /home/working/FinalCorrected/demultiplexed/SA47_2.fastq SA47_forward.paired.fastq.gz SA47_forward.singletons.fastq.gz SA47_reverse.paired.fastq.gz SA47_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:02:36 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA47/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17517 Both Surviving: 7452 (42.54%) Forward Only Surviving: 3387 (19.34%) Reverse Only Surviving: 204 (1.16%) Dropped: 6474 (36.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA47_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA47_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA47_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA47_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA47_forward.paired.headclipped.fastq.gz SA47_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA47 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA47_forward.paired.headclipped.fastq.gz
[FLASH]     SA47_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA47.extendedFrags.fastq.gz
[FLASH]     ./SA47.notCombined_1.fastq.gz
[FLASH]     ./SA47.notCombined_2.fastq.gz
[FLASH]     ./SA47.hist
[FLASH]     ./SA47.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7452 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7452
[FLASH]     Combined pairs:   5420
[FLASH]     Uncombined pairs: 2032
[FLASH]     Percent combined: 72.73%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.153 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5420 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA47/SA47_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA47_centroids.fasta --uc SA47.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5420 sequences and identified 1636 clusters (clustering threshold 1.00) - 162 clusters (representing 3625 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA48 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA48_1.fastq /home/working/FinalCorrected/demultiplexed/SA48_2.fastq SA48_forward.paired.fastq.gz SA48_forward.singletons.fastq.gz SA48_reverse.paired.fastq.gz SA48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA48_1.fastq /home/working/FinalCorrected/demultiplexed/SA48_2.fastq SA48_forward.paired.fastq.gz SA48_forward.singletons.fastq.gz SA48_reverse.paired.fastq.gz SA48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:02:46 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA48/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11761 Both Surviving: 145 (1.23%) Forward Only Surviving: 144 (1.22%) Reverse Only Surviving: 68 (0.58%) Dropped: 11404 (96.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA48_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA48_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA48_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA48_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA48_forward.paired.headclipped.fastq.gz SA48_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA48 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA48_forward.paired.headclipped.fastq.gz
[FLASH]     SA48_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA48.extendedFrags.fastq.gz
[FLASH]     ./SA48.notCombined_1.fastq.gz
[FLASH]     ./SA48.notCombined_2.fastq.gz
[FLASH]     ./SA48.hist
[FLASH]     ./SA48.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 145 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      145
[FLASH]     Combined pairs:   12
[FLASH]     Uncombined pairs: 133
[FLASH]     Percent combined: 8.28%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

12 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA48/SA48_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA48_centroids.fasta --uc SA48.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 12 sequences and identified 11 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA49 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA49_1.fastq /home/working/FinalCorrected/demultiplexed/SA49_2.fastq SA49_forward.paired.fastq.gz SA49_forward.singletons.fastq.gz SA49_reverse.paired.fastq.gz SA49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA49_1.fastq /home/working/FinalCorrected/demultiplexed/SA49_2.fastq SA49_forward.paired.fastq.gz SA49_forward.singletons.fastq.gz SA49_reverse.paired.fastq.gz SA49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:02:48 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA49/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23373 Both Surviving: 3250 (13.90%) Forward Only Surviving: 989 (4.23%) Reverse Only Surviving: 238 (1.02%) Dropped: 18896 (80.85%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA49_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA49_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA49_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA49_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA49_forward.paired.headclipped.fastq.gz SA49_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA49 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA49_forward.paired.headclipped.fastq.gz
[FLASH]     SA49_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA49.extendedFrags.fastq.gz
[FLASH]     ./SA49.notCombined_1.fastq.gz
[FLASH]     ./SA49.notCombined_2.fastq.gz
[FLASH]     ./SA49.hist
[FLASH]     ./SA49.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3250 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3250
[FLASH]     Combined pairs:   2976
[FLASH]     Uncombined pairs: 274
[FLASH]     Percent combined: 91.57%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.082 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2976 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA49/SA49_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA49_centroids.fasta --uc SA49.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2976 sequences and identified 1540 clusters (clustering threshold 1.00) - 77 clusters (representing 1271 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA5 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA5_1.fastq /home/working/FinalCorrected/demultiplexed/SA5_2.fastq SA5_forward.paired.fastq.gz SA5_forward.singletons.fastq.gz SA5_reverse.paired.fastq.gz SA5_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA5_1.fastq /home/working/FinalCorrected/demultiplexed/SA5_2.fastq SA5_forward.paired.fastq.gz SA5_forward.singletons.fastq.gz SA5_reverse.paired.fastq.gz SA5_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:02:55 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA5/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 39236 Both Surviving: 902 (2.30%) Forward Only Surviving: 560 (1.43%) Reverse Only Surviving: 619 (1.58%) Dropped: 37155 (94.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA5_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA5_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA5_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA5_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA5_forward.paired.headclipped.fastq.gz SA5_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA5 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA5_forward.paired.headclipped.fastq.gz
[FLASH]     SA5_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA5.extendedFrags.fastq.gz
[FLASH]     ./SA5.notCombined_1.fastq.gz
[FLASH]     ./SA5.notCombined_2.fastq.gz
[FLASH]     ./SA5.hist
[FLASH]     ./SA5.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 902 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      902
[FLASH]     Combined pairs:   137
[FLASH]     Uncombined pairs: 765
[FLASH]     Percent combined: 15.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.008 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

137 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA5/SA5_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA5_centroids.fasta --uc SA5.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 137 sequences and identified 80 clusters (clustering threshold 1.00) - 3 clusters (representing 53 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA50 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA50_1.fastq /home/working/FinalCorrected/demultiplexed/SA50_2.fastq SA50_forward.paired.fastq.gz SA50_forward.singletons.fastq.gz SA50_reverse.paired.fastq.gz SA50_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA50_1.fastq /home/working/FinalCorrected/demultiplexed/SA50_2.fastq SA50_forward.paired.fastq.gz SA50_forward.singletons.fastq.gz SA50_reverse.paired.fastq.gz SA50_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:02:58 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA50/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 8879 Both Surviving: 3139 (35.35%) Forward Only Surviving: 843 (9.49%) Reverse Only Surviving: 139 (1.57%) Dropped: 4758 (53.59%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA50_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA50_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA50_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA50_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA50_forward.paired.headclipped.fastq.gz SA50_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA50 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA50_forward.paired.headclipped.fastq.gz
[FLASH]     SA50_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA50.extendedFrags.fastq.gz
[FLASH]     ./SA50.notCombined_1.fastq.gz
[FLASH]     ./SA50.notCombined_2.fastq.gz
[FLASH]     ./SA50.hist
[FLASH]     ./SA50.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3139 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3139
[FLASH]     Combined pairs:   918
[FLASH]     Uncombined pairs: 2221
[FLASH]     Percent combined: 29.24%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.034 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

918 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA50/SA50_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA50_centroids.fasta --uc SA50.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 918 sequences and identified 566 clusters (clustering threshold 1.00) - 11 clusters (representing 313 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA51 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA51_1.fastq /home/working/FinalCorrected/demultiplexed/SA51_2.fastq SA51_forward.paired.fastq.gz SA51_forward.singletons.fastq.gz SA51_reverse.paired.fastq.gz SA51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA51_1.fastq /home/working/FinalCorrected/demultiplexed/SA51_2.fastq SA51_forward.paired.fastq.gz SA51_forward.singletons.fastq.gz SA51_reverse.paired.fastq.gz SA51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:03:02 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA51/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 24102 Both Surviving: 5856 (24.30%) Forward Only Surviving: 1209 (5.02%) Reverse Only Surviving: 454 (1.88%) Dropped: 16583 (68.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA51_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA51_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA51_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA51_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA51_forward.paired.headclipped.fastq.gz SA51_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA51 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA51_forward.paired.headclipped.fastq.gz
[FLASH]     SA51_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA51.extendedFrags.fastq.gz
[FLASH]     ./SA51.notCombined_1.fastq.gz
[FLASH]     ./SA51.notCombined_2.fastq.gz
[FLASH]     ./SA51.hist
[FLASH]     ./SA51.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5856 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5856
[FLASH]     Combined pairs:   1967
[FLASH]     Uncombined pairs: 3889
[FLASH]     Percent combined: 33.59%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.063 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1967 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA51/SA51_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA51_centroids.fasta --uc SA51.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1967 sequences and identified 858 clusters (clustering threshold 1.00) - 27 clusters (representing 976 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA53 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA53_1.fastq /home/working/FinalCorrected/demultiplexed/SA53_2.fastq SA53_forward.paired.fastq.gz SA53_forward.singletons.fastq.gz SA53_reverse.paired.fastq.gz SA53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA53_1.fastq /home/working/FinalCorrected/demultiplexed/SA53_2.fastq SA53_forward.paired.fastq.gz SA53_forward.singletons.fastq.gz SA53_reverse.paired.fastq.gz SA53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:03:11 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA53/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 73318 Both Surviving: 20776 (28.34%) Forward Only Surviving: 5766 (7.86%) Reverse Only Surviving: 1405 (1.92%) Dropped: 45371 (61.88%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA53_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA53_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA53_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA53_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA53_forward.paired.headclipped.fastq.gz SA53_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA53 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA53_forward.paired.headclipped.fastq.gz
[FLASH]     SA53_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA53.extendedFrags.fastq.gz
[FLASH]     ./SA53.notCombined_1.fastq.gz
[FLASH]     ./SA53.notCombined_2.fastq.gz
[FLASH]     ./SA53.hist
[FLASH]     ./SA53.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 20776 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      20776
[FLASH]     Combined pairs:   9111
[FLASH]     Uncombined pairs: 11665
[FLASH]     Percent combined: 43.85%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.275 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

9111 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA53/SA53_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA53_centroids.fasta --uc SA53.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 9111 sequences and identified 3677 clusters (clustering threshold 1.00) - 97 clusters (representing 4680 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA55 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA55_1.fastq /home/working/FinalCorrected/demultiplexed/SA55_2.fastq SA55_forward.paired.fastq.gz SA55_forward.singletons.fastq.gz SA55_reverse.paired.fastq.gz SA55_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA55_1.fastq /home/working/FinalCorrected/demultiplexed/SA55_2.fastq SA55_forward.paired.fastq.gz SA55_forward.singletons.fastq.gz SA55_reverse.paired.fastq.gz SA55_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:03:37 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA55/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 49553 Both Surviving: 11984 (24.18%) Forward Only Surviving: 2637 (5.32%) Reverse Only Surviving: 826 (1.67%) Dropped: 34106 (68.83%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA55_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA55_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SA55_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA55_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA55_forward.paired.headclipped.fastq.gz SA55_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA55 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA55_forward.paired.headclipped.fastq.gz
[FLASH]     SA55_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA55.extendedFrags.fastq.gz
[FLASH]     ./SA55.notCombined_1.fastq.gz
[FLASH]     ./SA55.notCombined_2.fastq.gz
[FLASH]     ./SA55.hist
[FLASH]     ./SA55.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11984 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11984
[FLASH]     Combined pairs:   3330
[FLASH]     Uncombined pairs: 8654
[FLASH]     Percent combined: 27.79%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.105 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3330 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA55/SA55_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA55_centroids.fasta --uc SA55.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3330 sequences and identified 1674 clusters (clustering threshold 1.00) - 47 clusters (representing 1432 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA58 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA58_1.fastq /home/working/FinalCorrected/demultiplexed/SA58_2.fastq SA58_forward.paired.fastq.gz SA58_forward.singletons.fastq.gz SA58_reverse.paired.fastq.gz SA58_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA58_1.fastq /home/working/FinalCorrected/demultiplexed/SA58_2.fastq SA58_forward.paired.fastq.gz SA58_forward.singletons.fastq.gz SA58_reverse.paired.fastq.gz SA58_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:03:49 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA58/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 43499 Both Surviving: 10937 (25.14%) Forward Only Surviving: 2426 (5.58%) Reverse Only Surviving: 671 (1.54%) Dropped: 29465 (67.74%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA58_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA58_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA58_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA58_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA58_forward.paired.headclipped.fastq.gz SA58_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA58 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA58_forward.paired.headclipped.fastq.gz
[FLASH]     SA58_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA58.extendedFrags.fastq.gz
[FLASH]     ./SA58.notCombined_1.fastq.gz
[FLASH]     ./SA58.notCombined_2.fastq.gz
[FLASH]     ./SA58.hist
[FLASH]     ./SA58.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10937 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10937
[FLASH]     Combined pairs:   2439
[FLASH]     Uncombined pairs: 8498
[FLASH]     Percent combined: 22.30%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.081 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2439 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA58/SA58_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA58_centroids.fasta --uc SA58.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2439 sequences and identified 1500 clusters (clustering threshold 1.00) - 29 clusters (representing 816 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA62 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA62_1.fastq /home/working/FinalCorrected/demultiplexed/SA62_2.fastq SA62_forward.paired.fastq.gz SA62_forward.singletons.fastq.gz SA62_reverse.paired.fastq.gz SA62_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA62_1.fastq /home/working/FinalCorrected/demultiplexed/SA62_2.fastq SA62_forward.paired.fastq.gz SA62_forward.singletons.fastq.gz SA62_reverse.paired.fastq.gz SA62_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:03:59 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA62/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 39926 Both Surviving: 104 (0.26%) Forward Only Surviving: 753 (1.89%) Reverse Only Surviving: 484 (1.21%) Dropped: 38585 (96.64%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA62_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA62_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA62_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA62_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA62_forward.paired.headclipped.fastq.gz SA62_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA62 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA62_forward.paired.headclipped.fastq.gz
[FLASH]     SA62_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA62.extendedFrags.fastq.gz
[FLASH]     ./SA62.notCombined_1.fastq.gz
[FLASH]     ./SA62.notCombined_2.fastq.gz
[FLASH]     ./SA62.hist
[FLASH]     ./SA62.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 104 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      104
[FLASH]     Combined pairs:   45
[FLASH]     Uncombined pairs: 59
[FLASH]     Percent combined: 43.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

45 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA62/SA62_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA62_centroids.fasta --uc SA62.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 45 sequences and identified 37 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA64 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA64_1.fastq /home/working/FinalCorrected/demultiplexed/SA64_2.fastq SA64_forward.paired.fastq.gz SA64_forward.singletons.fastq.gz SA64_reverse.paired.fastq.gz SA64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA64_1.fastq /home/working/FinalCorrected/demultiplexed/SA64_2.fastq SA64_forward.paired.fastq.gz SA64_forward.singletons.fastq.gz SA64_reverse.paired.fastq.gz SA64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:04:03 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA64/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28581 Both Surviving: 5231 (18.30%) Forward Only Surviving: 1867 (6.53%) Reverse Only Surviving: 380 (1.33%) Dropped: 21103 (73.84%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA64_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA64_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA64_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA64_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA64_forward.paired.headclipped.fastq.gz SA64_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA64 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA64_forward.paired.headclipped.fastq.gz
[FLASH]     SA64_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA64.extendedFrags.fastq.gz
[FLASH]     ./SA64.notCombined_1.fastq.gz
[FLASH]     ./SA64.notCombined_2.fastq.gz
[FLASH]     ./SA64.hist
[FLASH]     ./SA64.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5231 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5231
[FLASH]     Combined pairs:   366
[FLASH]     Uncombined pairs: 4865
[FLASH]     Percent combined: 7.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.032 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

366 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA64/SA64_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA64_centroids.fasta --uc SA64.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 366 sequences and identified 262 clusters (clustering threshold 1.00) - 8 clusters (representing 78 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA67 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA67_1.fastq /home/working/FinalCorrected/demultiplexed/SA67_2.fastq SA67_forward.paired.fastq.gz SA67_forward.singletons.fastq.gz SA67_reverse.paired.fastq.gz SA67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA67_1.fastq /home/working/FinalCorrected/demultiplexed/SA67_2.fastq SA67_forward.paired.fastq.gz SA67_forward.singletons.fastq.gz SA67_reverse.paired.fastq.gz SA67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:04:08 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA67/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29766 Both Surviving: 11022 (37.03%) Forward Only Surviving: 3974 (13.35%) Reverse Only Surviving: 376 (1.26%) Dropped: 14394 (48.36%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA67_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA67_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA67_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA67_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA67_forward.paired.headclipped.fastq.gz SA67_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA67 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA67_forward.paired.headclipped.fastq.gz
[FLASH]     SA67_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA67.extendedFrags.fastq.gz
[FLASH]     ./SA67.notCombined_1.fastq.gz
[FLASH]     ./SA67.notCombined_2.fastq.gz
[FLASH]     ./SA67.hist
[FLASH]     ./SA67.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11022 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11022
[FLASH]     Combined pairs:   4029
[FLASH]     Uncombined pairs: 6993
[FLASH]     Percent combined: 36.55%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.114 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4029 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA67/SA67_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA67_centroids.fasta --uc SA67.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4029 sequences and identified 2678 clusters (clustering threshold 1.00) - 77 clusters (representing 1169 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA68 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA68_1.fastq /home/working/FinalCorrected/demultiplexed/SA68_2.fastq SA68_forward.paired.fastq.gz SA68_forward.singletons.fastq.gz SA68_reverse.paired.fastq.gz SA68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA68_1.fastq /home/working/FinalCorrected/demultiplexed/SA68_2.fastq SA68_forward.paired.fastq.gz SA68_forward.singletons.fastq.gz SA68_reverse.paired.fastq.gz SA68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:04:20 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA68/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 36383 Both Surviving: 2264 (6.22%) Forward Only Surviving: 770 (2.12%) Reverse Only Surviving: 425 (1.17%) Dropped: 32924 (90.49%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA68_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA68_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA68_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA68_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA68_forward.paired.headclipped.fastq.gz SA68_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA68 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA68_forward.paired.headclipped.fastq.gz
[FLASH]     SA68_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA68.extendedFrags.fastq.gz
[FLASH]     ./SA68.notCombined_1.fastq.gz
[FLASH]     ./SA68.notCombined_2.fastq.gz
[FLASH]     ./SA68.hist
[FLASH]     ./SA68.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2264 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2264
[FLASH]     Combined pairs:   1962
[FLASH]     Uncombined pairs: 302
[FLASH]     Percent combined: 86.66%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.070 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1962 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA68/SA68_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA68_centroids.fasta --uc SA68.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1962 sequences and identified 891 clusters (clustering threshold 1.00) - 32 clusters (representing 959 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA7 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA7_1.fastq /home/working/FinalCorrected/demultiplexed/SA7_2.fastq SA7_forward.paired.fastq.gz SA7_forward.singletons.fastq.gz SA7_reverse.paired.fastq.gz SA7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA7_1.fastq /home/working/FinalCorrected/demultiplexed/SA7_2.fastq SA7_forward.paired.fastq.gz SA7_forward.singletons.fastq.gz SA7_reverse.paired.fastq.gz SA7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:04:26 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA7/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 30674 Both Surviving: 2927 (9.54%) Forward Only Surviving: 1329 (4.33%) Reverse Only Surviving: 307 (1.00%) Dropped: 26111 (85.12%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA7_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA7_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SA7_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA7_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA7_forward.paired.headclipped.fastq.gz SA7_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA7 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA7_forward.paired.headclipped.fastq.gz
[FLASH]     SA7_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA7.extendedFrags.fastq.gz
[FLASH]     ./SA7.notCombined_1.fastq.gz
[FLASH]     ./SA7.notCombined_2.fastq.gz
[FLASH]     ./SA7.hist
[FLASH]     ./SA7.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2927 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2927
[FLASH]     Combined pairs:   2420
[FLASH]     Uncombined pairs: 507
[FLASH]     Percent combined: 82.68%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.072 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2420 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA7/SA7_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA7_centroids.fasta --uc SA7.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2420 sequences and identified 1319 clusters (clustering threshold 1.00) - 49 clusters (representing 954 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA70 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA70_1.fastq /home/working/FinalCorrected/demultiplexed/SA70_2.fastq SA70_forward.paired.fastq.gz SA70_forward.singletons.fastq.gz SA70_reverse.paired.fastq.gz SA70_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA70_1.fastq /home/working/FinalCorrected/demultiplexed/SA70_2.fastq SA70_forward.paired.fastq.gz SA70_forward.singletons.fastq.gz SA70_reverse.paired.fastq.gz SA70_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:04:33 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA70/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 48118 Both Surviving: 417 (0.87%) Forward Only Surviving: 633 (1.32%) Reverse Only Surviving: 620 (1.29%) Dropped: 46448 (96.53%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA70_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA70_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA70_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA70_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA70_forward.paired.headclipped.fastq.gz SA70_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA70 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA70_forward.paired.headclipped.fastq.gz
[FLASH]     SA70_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA70.extendedFrags.fastq.gz
[FLASH]     ./SA70.notCombined_1.fastq.gz
[FLASH]     ./SA70.notCombined_2.fastq.gz
[FLASH]     ./SA70.hist
[FLASH]     ./SA70.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 417 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      417
[FLASH]     Combined pairs:   267
[FLASH]     Uncombined pairs: 150
[FLASH]     Percent combined: 64.03%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.013 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

267 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA70/SA70_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA70_centroids.fasta --uc SA70.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 267 sequences and identified 163 clusters (clustering threshold 1.00) - 7 clusters (representing 97 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA73 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA73_1.fastq /home/working/FinalCorrected/demultiplexed/SA73_2.fastq SA73_forward.paired.fastq.gz SA73_forward.singletons.fastq.gz SA73_reverse.paired.fastq.gz SA73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA73_1.fastq /home/working/FinalCorrected/demultiplexed/SA73_2.fastq SA73_forward.paired.fastq.gz SA73_forward.singletons.fastq.gz SA73_reverse.paired.fastq.gz SA73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:04:39 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA73/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 45415 Both Surviving: 9585 (21.11%) Forward Only Surviving: 2169 (4.78%) Reverse Only Surviving: 756 (1.66%) Dropped: 32905 (72.45%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA73_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA73_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA73_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA73_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA73_forward.paired.headclipped.fastq.gz SA73_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA73 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA73_forward.paired.headclipped.fastq.gz
[FLASH]     SA73_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA73.extendedFrags.fastq.gz
[FLASH]     ./SA73.notCombined_1.fastq.gz
[FLASH]     ./SA73.notCombined_2.fastq.gz
[FLASH]     ./SA73.hist
[FLASH]     ./SA73.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9585 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9585
[FLASH]     Combined pairs:   2029
[FLASH]     Uncombined pairs: 7556
[FLASH]     Percent combined: 21.17%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.063 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2029 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA73/SA73_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA73_centroids.fasta --uc SA73.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2029 sequences and identified 1038 clusters (clustering threshold 1.00) - 37 clusters (representing 840 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA75 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA75_1.fastq /home/working/FinalCorrected/demultiplexed/SA75_2.fastq SA75_forward.paired.fastq.gz SA75_forward.singletons.fastq.gz SA75_reverse.paired.fastq.gz SA75_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA75_1.fastq /home/working/FinalCorrected/demultiplexed/SA75_2.fastq SA75_forward.paired.fastq.gz SA75_forward.singletons.fastq.gz SA75_reverse.paired.fastq.gz SA75_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:04:47 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA75/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17374 Both Surviving: 1201 (6.91%) Forward Only Surviving: 573 (3.30%) Reverse Only Surviving: 227 (1.31%) Dropped: 15373 (88.48%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA75_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA75_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA75_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA75_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA75_forward.paired.headclipped.fastq.gz SA75_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA75 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA75_forward.paired.headclipped.fastq.gz
[FLASH]     SA75_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA75.extendedFrags.fastq.gz
[FLASH]     ./SA75.notCombined_1.fastq.gz
[FLASH]     ./SA75.notCombined_2.fastq.gz
[FLASH]     ./SA75.hist
[FLASH]     ./SA75.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1201 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1201
[FLASH]     Combined pairs:   421
[FLASH]     Uncombined pairs: 780
[FLASH]     Percent combined: 35.05%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.017 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

421 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA75/SA75_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA75_centroids.fasta --uc SA75.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 421 sequences and identified 300 clusters (clustering threshold 1.00) - 6 clusters (representing 110 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA76 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA76_1.fastq /home/working/FinalCorrected/demultiplexed/SA76_2.fastq SA76_forward.paired.fastq.gz SA76_forward.singletons.fastq.gz SA76_reverse.paired.fastq.gz SA76_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA76_1.fastq /home/working/FinalCorrected/demultiplexed/SA76_2.fastq SA76_forward.paired.fastq.gz SA76_forward.singletons.fastq.gz SA76_reverse.paired.fastq.gz SA76_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:04:50 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA76/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20250 Both Surviving: 7598 (37.52%) Forward Only Surviving: 3215 (15.88%) Reverse Only Surviving: 243 (1.20%) Dropped: 9194 (45.40%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA76_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA76_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA76_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA76_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA76_forward.paired.headclipped.fastq.gz SA76_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA76 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA76_forward.paired.headclipped.fastq.gz
[FLASH]     SA76_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA76.extendedFrags.fastq.gz
[FLASH]     ./SA76.notCombined_1.fastq.gz
[FLASH]     ./SA76.notCombined_2.fastq.gz
[FLASH]     ./SA76.hist
[FLASH]     ./SA76.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7598 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7598
[FLASH]     Combined pairs:   6303
[FLASH]     Uncombined pairs: 1295
[FLASH]     Percent combined: 82.96%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.157 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6303 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA76/SA76_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA76_centroids.fasta --uc SA76.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6303 sequences and identified 2397 clusters (clustering threshold 1.00) - 142 clusters (representing 3603 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA81 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA81_1.fastq /home/working/FinalCorrected/demultiplexed/SA81_2.fastq SA81_forward.paired.fastq.gz SA81_forward.singletons.fastq.gz SA81_reverse.paired.fastq.gz SA81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA81_1.fastq /home/working/FinalCorrected/demultiplexed/SA81_2.fastq SA81_forward.paired.fastq.gz SA81_forward.singletons.fastq.gz SA81_reverse.paired.fastq.gz SA81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:05:01 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA81/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 30631 Both Surviving: 585 (1.91%) Forward Only Surviving: 387 (1.26%) Reverse Only Surviving: 304 (0.99%) Dropped: 29355 (95.83%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA81_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA81_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SA81_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA81_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA81_forward.paired.headclipped.fastq.gz SA81_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA81 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA81_forward.paired.headclipped.fastq.gz
[FLASH]     SA81_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA81.extendedFrags.fastq.gz
[FLASH]     ./SA81.notCombined_1.fastq.gz
[FLASH]     ./SA81.notCombined_2.fastq.gz
[FLASH]     ./SA81.hist
[FLASH]     ./SA81.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 585 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      585
[FLASH]     Combined pairs:   47
[FLASH]     Uncombined pairs: 538
[FLASH]     Percent combined: 8.03%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.006 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

47 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA81/SA81_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA81_centroids.fasta --uc SA81.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 47 sequences and identified 42 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA86 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA86_1.fastq /home/working/FinalCorrected/demultiplexed/SA86_2.fastq SA86_forward.paired.fastq.gz SA86_forward.singletons.fastq.gz SA86_reverse.paired.fastq.gz SA86_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA86_1.fastq /home/working/FinalCorrected/demultiplexed/SA86_2.fastq SA86_forward.paired.fastq.gz SA86_forward.singletons.fastq.gz SA86_reverse.paired.fastq.gz SA86_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:05:06 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA86/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 60032 Both Surviving: 8569 (14.27%) Forward Only Surviving: 2738 (4.56%) Reverse Only Surviving: 656 (1.09%) Dropped: 48069 (80.07%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA86_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA86_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA86_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA86_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA86_forward.paired.headclipped.fastq.gz SA86_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA86 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA86_forward.paired.headclipped.fastq.gz
[FLASH]     SA86_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA86.extendedFrags.fastq.gz
[FLASH]     ./SA86.notCombined_1.fastq.gz
[FLASH]     ./SA86.notCombined_2.fastq.gz
[FLASH]     ./SA86.hist
[FLASH]     ./SA86.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8569 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8569
[FLASH]     Combined pairs:   216
[FLASH]     Uncombined pairs: 8353
[FLASH]     Percent combined: 2.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.053 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

216 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA86/SA86_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA86_centroids.fasta --uc SA86.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 216 sequences and identified 181 clusters (clustering threshold 1.00) - 2 clusters (representing 28 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA87 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA87_1.fastq /home/working/FinalCorrected/demultiplexed/SA87_2.fastq SA87_forward.paired.fastq.gz SA87_forward.singletons.fastq.gz SA87_reverse.paired.fastq.gz SA87_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA87_1.fastq /home/working/FinalCorrected/demultiplexed/SA87_2.fastq SA87_forward.paired.fastq.gz SA87_forward.singletons.fastq.gz SA87_reverse.paired.fastq.gz SA87_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:05:13 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA87/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31996 Both Surviving: 1905 (5.95%) Forward Only Surviving: 1110 (3.47%) Reverse Only Surviving: 484 (1.51%) Dropped: 28497 (89.06%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA87_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA87_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA87_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA87_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA87_forward.paired.headclipped.fastq.gz SA87_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA87 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA87_forward.paired.headclipped.fastq.gz
[FLASH]     SA87_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA87.extendedFrags.fastq.gz
[FLASH]     ./SA87.notCombined_1.fastq.gz
[FLASH]     ./SA87.notCombined_2.fastq.gz
[FLASH]     ./SA87.hist
[FLASH]     ./SA87.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1905 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1905
[FLASH]     Combined pairs:   643
[FLASH]     Uncombined pairs: 1262
[FLASH]     Percent combined: 33.75%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.024 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

643 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA87/SA87_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA87_centroids.fasta --uc SA87.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 643 sequences and identified 404 clusters (clustering threshold 1.00) - 9 clusters (representing 198 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA88 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA88_1.fastq /home/working/FinalCorrected/demultiplexed/SA88_2.fastq SA88_forward.paired.fastq.gz SA88_forward.singletons.fastq.gz SA88_reverse.paired.fastq.gz SA88_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA88_1.fastq /home/working/FinalCorrected/demultiplexed/SA88_2.fastq SA88_forward.paired.fastq.gz SA88_forward.singletons.fastq.gz SA88_reverse.paired.fastq.gz SA88_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:05:21 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA88/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 95056 Both Surviving: 32779 (34.48%) Forward Only Surviving: 7861 (8.27%) Reverse Only Surviving: 2531 (2.66%) Dropped: 51885 (54.58%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA88_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA88_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA88_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA88_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA88_forward.paired.headclipped.fastq.gz SA88_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA88 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA88_forward.paired.headclipped.fastq.gz
[FLASH]     SA88_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA88.extendedFrags.fastq.gz
[FLASH]     ./SA88.notCombined_1.fastq.gz
[FLASH]     ./SA88.notCombined_2.fastq.gz
[FLASH]     ./SA88.hist
[FLASH]     ./SA88.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 32779 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      32779
[FLASH]     Combined pairs:   1124
[FLASH]     Uncombined pairs: 31655
[FLASH]     Percent combined: 3.43%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.262 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1124 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA88/SA88_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA88_centroids.fasta --uc SA88.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1124 sequences and identified 882 clusters (clustering threshold 1.00) - 17 clusters (representing 150 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA89 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA89_1.fastq /home/working/FinalCorrected/demultiplexed/SA89_2.fastq SA89_forward.paired.fastq.gz SA89_forward.singletons.fastq.gz SA89_reverse.paired.fastq.gz SA89_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA89_1.fastq /home/working/FinalCorrected/demultiplexed/SA89_2.fastq SA89_forward.paired.fastq.gz SA89_forward.singletons.fastq.gz SA89_reverse.paired.fastq.gz SA89_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:05:38 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA89/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20721 Both Surviving: 4774 (23.04%) Forward Only Surviving: 992 (4.79%) Reverse Only Surviving: 412 (1.99%) Dropped: 14543 (70.18%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA89_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA89_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA89_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA89_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA89_forward.paired.headclipped.fastq.gz SA89_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA89 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA89_forward.paired.headclipped.fastq.gz
[FLASH]     SA89_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA89.extendedFrags.fastq.gz
[FLASH]     ./SA89.notCombined_1.fastq.gz
[FLASH]     ./SA89.notCombined_2.fastq.gz
[FLASH]     ./SA89.hist
[FLASH]     ./SA89.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4774 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4774
[FLASH]     Combined pairs:   177
[FLASH]     Uncombined pairs: 4597
[FLASH]     Percent combined: 3.71%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.029 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

177 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA89/SA89_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA89_centroids.fasta --uc SA89.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 177 sequences and identified 130 clusters (clustering threshold 1.00) - 2 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA9 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA9_1.fastq /home/working/FinalCorrected/demultiplexed/SA9_2.fastq SA9_forward.paired.fastq.gz SA9_forward.singletons.fastq.gz SA9_reverse.paired.fastq.gz SA9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA9_1.fastq /home/working/FinalCorrected/demultiplexed/SA9_2.fastq SA9_forward.paired.fastq.gz SA9_forward.singletons.fastq.gz SA9_reverse.paired.fastq.gz SA9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:05:43 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA9/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 27387 Both Surviving: 3248 (11.86%) Forward Only Surviving: 1038 (3.79%) Reverse Only Surviving: 271 (0.99%) Dropped: 22830 (83.36%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA9_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA9_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SA9_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SA9_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA9_forward.paired.headclipped.fastq.gz SA9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA9 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA9_forward.paired.headclipped.fastq.gz
[FLASH]     SA9_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA9.extendedFrags.fastq.gz
[FLASH]     ./SA9.notCombined_1.fastq.gz
[FLASH]     ./SA9.notCombined_2.fastq.gz
[FLASH]     ./SA9.hist
[FLASH]     ./SA9.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3248 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3248
[FLASH]     Combined pairs:   2832
[FLASH]     Uncombined pairs: 416
[FLASH]     Percent combined: 87.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.088 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2832 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA9/SA9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA9_centroids.fasta --uc SA9.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2832 sequences and identified 1263 clusters (clustering threshold 1.00) - 32 clusters (representing 1368 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA91 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA91_1.fastq /home/working/FinalCorrected/demultiplexed/SA91_2.fastq SA91_forward.paired.fastq.gz SA91_forward.singletons.fastq.gz SA91_reverse.paired.fastq.gz SA91_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA91_1.fastq /home/working/FinalCorrected/demultiplexed/SA91_2.fastq SA91_forward.paired.fastq.gz SA91_forward.singletons.fastq.gz SA91_reverse.paired.fastq.gz SA91_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:05:52 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA91/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 58616 Both Surviving: 9782 (16.69%) Forward Only Surviving: 3409 (5.82%) Reverse Only Surviving: 846 (1.44%) Dropped: 44579 (76.05%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA91_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA91_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA91_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA91_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA91_forward.paired.headclipped.fastq.gz SA91_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA91 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA91_forward.paired.headclipped.fastq.gz
[FLASH]     SA91_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA91.extendedFrags.fastq.gz
[FLASH]     ./SA91.notCombined_1.fastq.gz
[FLASH]     ./SA91.notCombined_2.fastq.gz
[FLASH]     ./SA91.hist
[FLASH]     ./SA91.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9782 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9782
[FLASH]     Combined pairs:   1474
[FLASH]     Uncombined pairs: 8308
[FLASH]     Percent combined: 15.07%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.060 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1474 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA91/SA91_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA91_centroids.fasta --uc SA91.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1474 sequences and identified 745 clusters (clustering threshold 1.00) - 16 clusters (representing 648 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA93 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA93_1.fastq /home/working/FinalCorrected/demultiplexed/SA93_2.fastq SA93_forward.paired.fastq.gz SA93_forward.singletons.fastq.gz SA93_reverse.paired.fastq.gz SA93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA93_1.fastq /home/working/FinalCorrected/demultiplexed/SA93_2.fastq SA93_forward.paired.fastq.gz SA93_forward.singletons.fastq.gz SA93_reverse.paired.fastq.gz SA93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:06:00 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA93/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20338 Both Surviving: 3625 (17.82%) Forward Only Surviving: 818 (4.02%) Reverse Only Surviving: 288 (1.42%) Dropped: 15607 (76.74%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA93_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA93_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA93_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA93_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA93_forward.paired.headclipped.fastq.gz SA93_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA93 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA93_forward.paired.headclipped.fastq.gz
[FLASH]     SA93_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA93.extendedFrags.fastq.gz
[FLASH]     ./SA93.notCombined_1.fastq.gz
[FLASH]     ./SA93.notCombined_2.fastq.gz
[FLASH]     ./SA93.hist
[FLASH]     ./SA93.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3625 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3625
[FLASH]     Combined pairs:   1692
[FLASH]     Uncombined pairs: 1933
[FLASH]     Percent combined: 46.68%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.053 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1692 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA93/SA93_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA93_centroids.fasta --uc SA93.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1692 sequences and identified 1004 clusters (clustering threshold 1.00) - 20 clusters (representing 607 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA94 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA94_1.fastq /home/working/FinalCorrected/demultiplexed/SA94_2.fastq SA94_forward.paired.fastq.gz SA94_forward.singletons.fastq.gz SA94_reverse.paired.fastq.gz SA94_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA94_1.fastq /home/working/FinalCorrected/demultiplexed/SA94_2.fastq SA94_forward.paired.fastq.gz SA94_forward.singletons.fastq.gz SA94_reverse.paired.fastq.gz SA94_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:06:06 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA94/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25533 Both Surviving: 139 (0.54%) Forward Only Surviving: 402 (1.57%) Reverse Only Surviving: 269 (1.05%) Dropped: 24723 (96.83%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SA94_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SA94_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SA94_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA94_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA94_forward.paired.headclipped.fastq.gz SA94_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA94 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA94_forward.paired.headclipped.fastq.gz
[FLASH]     SA94_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA94.extendedFrags.fastq.gz
[FLASH]     ./SA94.notCombined_1.fastq.gz
[FLASH]     ./SA94.notCombined_2.fastq.gz
[FLASH]     ./SA94.hist
[FLASH]     ./SA94.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 139 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      139
[FLASH]     Combined pairs:   72
[FLASH]     Uncombined pairs: 67
[FLASH]     Percent combined: 51.80%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.006 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

72 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA94/SA94_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA94_centroids.fasta --uc SA94.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 72 sequences and identified 44 clusters (clustering threshold 1.00) - 1 clusters (representing 26 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA95 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA95_1.fastq /home/working/FinalCorrected/demultiplexed/SA95_2.fastq SA95_forward.paired.fastq.gz SA95_forward.singletons.fastq.gz SA95_reverse.paired.fastq.gz SA95_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA95_1.fastq /home/working/FinalCorrected/demultiplexed/SA95_2.fastq SA95_forward.paired.fastq.gz SA95_forward.singletons.fastq.gz SA95_reverse.paired.fastq.gz SA95_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:06:08 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA95/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11287 Both Surviving: 698 (6.18%) Forward Only Surviving: 258 (2.29%) Reverse Only Surviving: 124 (1.10%) Dropped: 10207 (90.43%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SA95_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SA95_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA95_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA95_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA95_forward.paired.headclipped.fastq.gz SA95_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA95 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA95_forward.paired.headclipped.fastq.gz
[FLASH]     SA95_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA95.extendedFrags.fastq.gz
[FLASH]     ./SA95.notCombined_1.fastq.gz
[FLASH]     ./SA95.notCombined_2.fastq.gz
[FLASH]     ./SA95.hist
[FLASH]     ./SA95.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 698 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      698
[FLASH]     Combined pairs:   526
[FLASH]     Uncombined pairs: 172
[FLASH]     Percent combined: 75.36%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.020 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

526 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA95/SA95_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA95_centroids.fasta --uc SA95.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 526 sequences and identified 290 clusters (clustering threshold 1.00) - 8 clusters (representing 216 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA96 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA96_1.fastq /home/working/FinalCorrected/demultiplexed/SA96_2.fastq SA96_forward.paired.fastq.gz SA96_forward.singletons.fastq.gz SA96_reverse.paired.fastq.gz SA96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA96_1.fastq /home/working/FinalCorrected/demultiplexed/SA96_2.fastq SA96_forward.paired.fastq.gz SA96_forward.singletons.fastq.gz SA96_reverse.paired.fastq.gz SA96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:06:10 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA96/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15602 Both Surviving: 4680 (30.00%) Forward Only Surviving: 1124 (7.20%) Reverse Only Surviving: 244 (1.56%) Dropped: 9554 (61.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SA96_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SA96_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SA96_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SA96_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA96_forward.paired.headclipped.fastq.gz SA96_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA96 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA96_forward.paired.headclipped.fastq.gz
[FLASH]     SA96_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA96.extendedFrags.fastq.gz
[FLASH]     ./SA96.notCombined_1.fastq.gz
[FLASH]     ./SA96.notCombined_2.fastq.gz
[FLASH]     ./SA96.hist
[FLASH]     ./SA96.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4680 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4680
[FLASH]     Combined pairs:   1353
[FLASH]     Uncombined pairs: 3327
[FLASH]     Percent combined: 28.91%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.050 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1353 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA96/SA96_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA96_centroids.fasta --uc SA96.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1353 sequences and identified 828 clusters (clustering threshold 1.00) - 22 clusters (representing 483 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA97 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA97_1.fastq /home/working/FinalCorrected/demultiplexed/SA97_2.fastq SA97_forward.paired.fastq.gz SA97_forward.singletons.fastq.gz SA97_reverse.paired.fastq.gz SA97_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA97_1.fastq /home/working/FinalCorrected/demultiplexed/SA97_2.fastq SA97_forward.paired.fastq.gz SA97_forward.singletons.fastq.gz SA97_reverse.paired.fastq.gz SA97_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:06:15 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA97/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11026 Both Surviving: 2186 (19.83%) Forward Only Surviving: 409 (3.71%) Reverse Only Surviving: 184 (1.67%) Dropped: 8247 (74.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SA97_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SA97_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SA97_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SA97_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA97_forward.paired.headclipped.fastq.gz SA97_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA97 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA97_forward.paired.headclipped.fastq.gz
[FLASH]     SA97_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA97.extendedFrags.fastq.gz
[FLASH]     ./SA97.notCombined_1.fastq.gz
[FLASH]     ./SA97.notCombined_2.fastq.gz
[FLASH]     ./SA97.hist
[FLASH]     ./SA97.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2186 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2186
[FLASH]     Combined pairs:   816
[FLASH]     Uncombined pairs: 1370
[FLASH]     Percent combined: 37.33%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.030 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

816 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA97/SA97_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA97_centroids.fasta --uc SA97.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 816 sequences and identified 476 clusters (clustering threshold 1.00) - 9 clusters (representing 299 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SA98 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA98_1.fastq /home/working/FinalCorrected/demultiplexed/SA98_2.fastq SA98_forward.paired.fastq.gz SA98_forward.singletons.fastq.gz SA98_reverse.paired.fastq.gz SA98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SA98_1.fastq /home/working/FinalCorrected/demultiplexed/SA98_2.fastq SA98_forward.paired.fastq.gz SA98_forward.singletons.fastq.gz SA98_reverse.paired.fastq.gz SA98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:06:20 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SA98/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 42857 Both Surviving: 5781 (13.49%) Forward Only Surviving: 2218 (5.18%) Reverse Only Surviving: 333 (0.78%) Dropped: 34525 (80.56%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SA98_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SA98_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SA98_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SA98_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SA98_forward.paired.headclipped.fastq.gz SA98_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA98 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SA98_forward.paired.headclipped.fastq.gz
[FLASH]     SA98_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SA98.extendedFrags.fastq.gz
[FLASH]     ./SA98.notCombined_1.fastq.gz
[FLASH]     ./SA98.notCombined_2.fastq.gz
[FLASH]     ./SA98.hist
[FLASH]     ./SA98.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5781 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5781
[FLASH]     Combined pairs:   5329
[FLASH]     Uncombined pairs: 452
[FLASH]     Percent combined: 92.18%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.136 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5329 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SA98/SA98_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA98_centroids.fasta --uc SA98.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5329 sequences and identified 1260 clusters (clustering threshold 1.00) - 143 clusters (representing 3925 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB102 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB102_1.fastq /home/working/FinalCorrected/demultiplexed/SB102_2.fastq SB102_forward.paired.fastq.gz SB102_forward.singletons.fastq.gz SB102_reverse.paired.fastq.gz SB102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB102_1.fastq /home/working/FinalCorrected/demultiplexed/SB102_2.fastq SB102_forward.paired.fastq.gz SB102_forward.singletons.fastq.gz SB102_reverse.paired.fastq.gz SB102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:06:35 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB102/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 95589 Both Surviving: 18233 (19.07%) Forward Only Surviving: 5549 (5.81%) Reverse Only Surviving: 1427 (1.49%) Dropped: 70380 (73.63%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB102_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB102_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB102_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB102_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB102_forward.paired.headclipped.fastq.gz SB102_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB102 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB102_forward.paired.headclipped.fastq.gz
[FLASH]     SB102_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB102.extendedFrags.fastq.gz
[FLASH]     ./SB102.notCombined_1.fastq.gz
[FLASH]     ./SB102.notCombined_2.fastq.gz
[FLASH]     ./SB102.hist
[FLASH]     ./SB102.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 18233 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      18233
[FLASH]     Combined pairs:   980
[FLASH]     Uncombined pairs: 17253
[FLASH]     Percent combined: 5.37%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.105 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

980 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB102/SB102_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB102_centroids.fasta --uc SB102.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 980 sequences and identified 623 clusters (clustering threshold 1.00) - 18 clusters (representing 291 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB104 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB104_1.fastq /home/working/FinalCorrected/demultiplexed/SB104_2.fastq SB104_forward.paired.fastq.gz SB104_forward.singletons.fastq.gz SB104_reverse.paired.fastq.gz SB104_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB104_1.fastq /home/working/FinalCorrected/demultiplexed/SB104_2.fastq SB104_forward.paired.fastq.gz SB104_forward.singletons.fastq.gz SB104_reverse.paired.fastq.gz SB104_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:06:47 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB104/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 21182 Both Surviving: 9019 (42.58%) Forward Only Surviving: 1940 (9.16%) Reverse Only Surviving: 659 (3.11%) Dropped: 9564 (45.15%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SB104_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB104_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB104_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB104_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB104_forward.paired.headclipped.fastq.gz SB104_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB104 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB104_forward.paired.headclipped.fastq.gz
[FLASH]     SB104_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB104.extendedFrags.fastq.gz
[FLASH]     ./SB104.notCombined_1.fastq.gz
[FLASH]     ./SB104.notCombined_2.fastq.gz
[FLASH]     ./SB104.hist
[FLASH]     ./SB104.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9019 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9019
[FLASH]     Combined pairs:   5159
[FLASH]     Uncombined pairs: 3860
[FLASH]     Percent combined: 57.20%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.151 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5159 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB104/SB104_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB104_centroids.fasta --uc SB104.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5159 sequences and identified 3829 clusters (clustering threshold 1.00) - 65 clusters (representing 924 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB106 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB106_1.fastq /home/working/FinalCorrected/demultiplexed/SB106_2.fastq SB106_forward.paired.fastq.gz SB106_forward.singletons.fastq.gz SB106_reverse.paired.fastq.gz SB106_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB106_1.fastq /home/working/FinalCorrected/demultiplexed/SB106_2.fastq SB106_forward.paired.fastq.gz SB106_forward.singletons.fastq.gz SB106_reverse.paired.fastq.gz SB106_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:07:00 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB106/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 49076 Both Surviving: 8570 (17.46%) Forward Only Surviving: 2504 (5.10%) Reverse Only Surviving: 465 (0.95%) Dropped: 37537 (76.49%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB106_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB106_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB106_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB106_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB106_forward.paired.headclipped.fastq.gz SB106_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB106 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB106_forward.paired.headclipped.fastq.gz
[FLASH]     SB106_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB106.extendedFrags.fastq.gz
[FLASH]     ./SB106.notCombined_1.fastq.gz
[FLASH]     ./SB106.notCombined_2.fastq.gz
[FLASH]     ./SB106.hist
[FLASH]     ./SB106.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8570 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8570
[FLASH]     Combined pairs:   7840
[FLASH]     Uncombined pairs: 730
[FLASH]     Percent combined: 91.48%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.214 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

7840 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB106/SB106_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB106_centroids.fasta --uc SB106.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 7840 sequences and identified 3631 clusters (clustering threshold 1.00) - 111 clusters (representing 3692 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB111 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB111_1.fastq /home/working/FinalCorrected/demultiplexed/SB111_2.fastq SB111_forward.paired.fastq.gz SB111_forward.singletons.fastq.gz SB111_reverse.paired.fastq.gz SB111_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB111_1.fastq /home/working/FinalCorrected/demultiplexed/SB111_2.fastq SB111_forward.paired.fastq.gz SB111_forward.singletons.fastq.gz SB111_reverse.paired.fastq.gz SB111_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:07:16 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB111/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31510 Both Surviving: 103 (0.33%) Forward Only Surviving: 529 (1.68%) Reverse Only Surviving: 306 (0.97%) Dropped: 30572 (97.02%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB111_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB111_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB111_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB111_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB111_forward.paired.headclipped.fastq.gz SB111_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB111 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB111_forward.paired.headclipped.fastq.gz
[FLASH]     SB111_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB111.extendedFrags.fastq.gz
[FLASH]     ./SB111.notCombined_1.fastq.gz
[FLASH]     ./SB111.notCombined_2.fastq.gz
[FLASH]     ./SB111.hist
[FLASH]     ./SB111.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 103 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      103
[FLASH]     Combined pairs:   27
[FLASH]     Uncombined pairs: 76
[FLASH]     Percent combined: 26.21%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

27 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB111/SB111_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB111_centroids.fasta --uc SB111.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 27 sequences and identified 26 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB116 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB116_1.fastq /home/working/FinalCorrected/demultiplexed/SB116_2.fastq SB116_forward.paired.fastq.gz SB116_forward.singletons.fastq.gz SB116_reverse.paired.fastq.gz SB116_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB116_1.fastq /home/working/FinalCorrected/demultiplexed/SB116_2.fastq SB116_forward.paired.fastq.gz SB116_forward.singletons.fastq.gz SB116_reverse.paired.fastq.gz SB116_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:07:18 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB116/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 12335 Both Surviving: 2168 (17.58%) Forward Only Surviving: 604 (4.90%) Reverse Only Surviving: 215 (1.74%) Dropped: 9348 (75.78%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB116_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB116_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB116_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB116_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB116_forward.paired.headclipped.fastq.gz SB116_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB116 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB116_forward.paired.headclipped.fastq.gz
[FLASH]     SB116_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB116.extendedFrags.fastq.gz
[FLASH]     ./SB116.notCombined_1.fastq.gz
[FLASH]     ./SB116.notCombined_2.fastq.gz
[FLASH]     ./SB116.hist
[FLASH]     ./SB116.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2168 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2168
[FLASH]     Combined pairs:   333
[FLASH]     Uncombined pairs: 1835
[FLASH]     Percent combined: 15.36%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.015 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

333 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB116/SB116_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB116_centroids.fasta --uc SB116.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 333 sequences and identified 189 clusters (clustering threshold 1.00) - 3 clusters (representing 124 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB119 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB119_1.fastq /home/working/FinalCorrected/demultiplexed/SB119_2.fastq SB119_forward.paired.fastq.gz SB119_forward.singletons.fastq.gz SB119_reverse.paired.fastq.gz SB119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB119_1.fastq /home/working/FinalCorrected/demultiplexed/SB119_2.fastq SB119_forward.paired.fastq.gz SB119_forward.singletons.fastq.gz SB119_reverse.paired.fastq.gz SB119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:07:22 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB119/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 27754 Both Surviving: 5093 (18.35%) Forward Only Surviving: 1118 (4.03%) Reverse Only Surviving: 486 (1.75%) Dropped: 21057 (75.87%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB119_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB119_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB119_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB119_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB119_forward.paired.headclipped.fastq.gz SB119_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB119 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB119_forward.paired.headclipped.fastq.gz
[FLASH]     SB119_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB119.extendedFrags.fastq.gz
[FLASH]     ./SB119.notCombined_1.fastq.gz
[FLASH]     ./SB119.notCombined_2.fastq.gz
[FLASH]     ./SB119.hist
[FLASH]     ./SB119.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5093 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5093
[FLASH]     Combined pairs:   106
[FLASH]     Uncombined pairs: 4987
[FLASH]     Percent combined: 2.08%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.033 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

106 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB119/SB119_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB119_centroids.fasta --uc SB119.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 106 sequences and identified 77 clusters (clustering threshold 1.00) - 1 clusters (representing 27 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB120 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB120_1.fastq /home/working/FinalCorrected/demultiplexed/SB120_2.fastq SB120_forward.paired.fastq.gz SB120_forward.singletons.fastq.gz SB120_reverse.paired.fastq.gz SB120_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB120_1.fastq /home/working/FinalCorrected/demultiplexed/SB120_2.fastq SB120_forward.paired.fastq.gz SB120_forward.singletons.fastq.gz SB120_reverse.paired.fastq.gz SB120_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:07:28 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB120/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 60339 Both Surviving: 12705 (21.06%) Forward Only Surviving: 3855 (6.39%) Reverse Only Surviving: 963 (1.60%) Dropped: 42816 (70.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB120_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB120_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB120_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB120_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB120_forward.paired.headclipped.fastq.gz SB120_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB120 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB120_forward.paired.headclipped.fastq.gz
[FLASH]     SB120_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB120.extendedFrags.fastq.gz
[FLASH]     ./SB120.notCombined_1.fastq.gz
[FLASH]     ./SB120.notCombined_2.fastq.gz
[FLASH]     ./SB120.hist
[FLASH]     ./SB120.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 12705 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      12705
[FLASH]     Combined pairs:   2445
[FLASH]     Uncombined pairs: 10260
[FLASH]     Percent combined: 19.24%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.096 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2445 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB120/SB120_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB120_centroids.fasta --uc SB120.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2445 sequences and identified 1432 clusters (clustering threshold 1.00) - 38 clusters (representing 850 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB127 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB127_1.fastq /home/working/FinalCorrected/demultiplexed/SB127_2.fastq SB127_forward.paired.fastq.gz SB127_forward.singletons.fastq.gz SB127_reverse.paired.fastq.gz SB127_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB127_1.fastq /home/working/FinalCorrected/demultiplexed/SB127_2.fastq SB127_forward.paired.fastq.gz SB127_forward.singletons.fastq.gz SB127_reverse.paired.fastq.gz SB127_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:07:39 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB127/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 38679 Both Surviving: 1764 (4.56%) Forward Only Surviving: 696 (1.80%) Reverse Only Surviving: 470 (1.22%) Dropped: 35749 (92.42%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB127_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB127_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB127_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB127_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB127_forward.paired.headclipped.fastq.gz SB127_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB127 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB127_forward.paired.headclipped.fastq.gz
[FLASH]     SB127_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB127.extendedFrags.fastq.gz
[FLASH]     ./SB127.notCombined_1.fastq.gz
[FLASH]     ./SB127.notCombined_2.fastq.gz
[FLASH]     ./SB127.hist
[FLASH]     ./SB127.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1764 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1764
[FLASH]     Combined pairs:   1660
[FLASH]     Uncombined pairs: 104
[FLASH]     Percent combined: 94.10%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.073 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1660 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB127/SB127_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB127_centroids.fasta --uc SB127.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1660 sequences and identified 765 clusters (clustering threshold 1.00) - 24 clusters (representing 749 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB128 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB128_1.fastq /home/working/FinalCorrected/demultiplexed/SB128_2.fastq SB128_forward.paired.fastq.gz SB128_forward.singletons.fastq.gz SB128_reverse.paired.fastq.gz SB128_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB128_1.fastq /home/working/FinalCorrected/demultiplexed/SB128_2.fastq SB128_forward.paired.fastq.gz SB128_forward.singletons.fastq.gz SB128_reverse.paired.fastq.gz SB128_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:07:47 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB128/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 54931 Both Surviving: 9843 (17.92%) Forward Only Surviving: 2614 (4.76%) Reverse Only Surviving: 903 (1.64%) Dropped: 41571 (75.68%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB128_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB128_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB128_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB128_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB128_forward.paired.headclipped.fastq.gz SB128_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB128 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB128_forward.paired.headclipped.fastq.gz
[FLASH]     SB128_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB128.extendedFrags.fastq.gz
[FLASH]     ./SB128.notCombined_1.fastq.gz
[FLASH]     ./SB128.notCombined_2.fastq.gz
[FLASH]     ./SB128.hist
[FLASH]     ./SB128.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9843 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9843
[FLASH]     Combined pairs:   1131
[FLASH]     Uncombined pairs: 8712
[FLASH]     Percent combined: 11.49%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.057 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1131 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB128/SB128_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB128_centroids.fasta --uc SB128.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1131 sequences and identified 596 clusters (clustering threshold 1.00) - 23 clusters (representing 488 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB129 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB129_1.fastq /home/working/FinalCorrected/demultiplexed/SB129_2.fastq SB129_forward.paired.fastq.gz SB129_forward.singletons.fastq.gz SB129_reverse.paired.fastq.gz SB129_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB129_1.fastq /home/working/FinalCorrected/demultiplexed/SB129_2.fastq SB129_forward.paired.fastq.gz SB129_forward.singletons.fastq.gz SB129_reverse.paired.fastq.gz SB129_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:07:55 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB129/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41461 Both Surviving: 12224 (29.48%) Forward Only Surviving: 2762 (6.66%) Reverse Only Surviving: 712 (1.72%) Dropped: 25763 (62.14%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB129_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB129_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SB129_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB129_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB129_forward.paired.headclipped.fastq.gz SB129_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB129 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB129_forward.paired.headclipped.fastq.gz
[FLASH]     SB129_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB129.extendedFrags.fastq.gz
[FLASH]     ./SB129.notCombined_1.fastq.gz
[FLASH]     ./SB129.notCombined_2.fastq.gz
[FLASH]     ./SB129.hist
[FLASH]     ./SB129.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 12224 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      12224
[FLASH]     Combined pairs:   4737
[FLASH]     Uncombined pairs: 7487
[FLASH]     Percent combined: 38.75%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.144 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4737 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB129/SB129_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB129_centroids.fasta --uc SB129.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4737 sequences and identified 2274 clusters (clustering threshold 1.00) - 71 clusters (representing 2123 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB13 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB13_1.fastq /home/working/FinalCorrected/demultiplexed/SB13_2.fastq SB13_forward.paired.fastq.gz SB13_forward.singletons.fastq.gz SB13_reverse.paired.fastq.gz SB13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB13_1.fastq /home/working/FinalCorrected/demultiplexed/SB13_2.fastq SB13_forward.paired.fastq.gz SB13_forward.singletons.fastq.gz SB13_reverse.paired.fastq.gz SB13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:08:08 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB13/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 34681 Both Surviving: 6218 (17.93%) Forward Only Surviving: 1604 (4.63%) Reverse Only Surviving: 580 (1.67%) Dropped: 26279 (75.77%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB13_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB13_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB13_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB13_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB13_forward.paired.headclipped.fastq.gz SB13_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB13 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB13_forward.paired.headclipped.fastq.gz
[FLASH]     SB13_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB13.extendedFrags.fastq.gz
[FLASH]     ./SB13.notCombined_1.fastq.gz
[FLASH]     ./SB13.notCombined_2.fastq.gz
[FLASH]     ./SB13.hist
[FLASH]     ./SB13.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6218 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6218
[FLASH]     Combined pairs:   40
[FLASH]     Uncombined pairs: 6178
[FLASH]     Percent combined: 0.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.042 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

40 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB13/SB13_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB13_centroids.fasta --uc SB13.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 40 sequences and identified 38 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB131 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB131_1.fastq /home/working/FinalCorrected/demultiplexed/SB131_2.fastq SB131_forward.paired.fastq.gz SB131_forward.singletons.fastq.gz SB131_reverse.paired.fastq.gz SB131_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB131_1.fastq /home/working/FinalCorrected/demultiplexed/SB131_2.fastq SB131_forward.paired.fastq.gz SB131_forward.singletons.fastq.gz SB131_reverse.paired.fastq.gz SB131_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:08:13 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB131/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 40610 Both Surviving: 3564 (8.78%) Forward Only Surviving: 1088 (2.68%) Reverse Only Surviving: 579 (1.43%) Dropped: 35379 (87.12%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB131_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB131_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB131_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB131_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB131_forward.paired.headclipped.fastq.gz SB131_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB131 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB131_forward.paired.headclipped.fastq.gz
[FLASH]     SB131_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB131.extendedFrags.fastq.gz
[FLASH]     ./SB131.notCombined_1.fastq.gz
[FLASH]     ./SB131.notCombined_2.fastq.gz
[FLASH]     ./SB131.hist
[FLASH]     ./SB131.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3564 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3564
[FLASH]     Combined pairs:   95
[FLASH]     Uncombined pairs: 3469
[FLASH]     Percent combined: 2.67%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.023 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

95 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB131/SB131_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB131_centroids.fasta --uc SB131.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 95 sequences and identified 82 clusters (clustering threshold 1.00) - 1 clusters (representing 11 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB135 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB135_1.fastq /home/working/FinalCorrected/demultiplexed/SB135_2.fastq SB135_forward.paired.fastq.gz SB135_forward.singletons.fastq.gz SB135_reverse.paired.fastq.gz SB135_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB135_1.fastq /home/working/FinalCorrected/demultiplexed/SB135_2.fastq SB135_forward.paired.fastq.gz SB135_forward.singletons.fastq.gz SB135_reverse.paired.fastq.gz SB135_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:08:17 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB135/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29244 Both Surviving: 2587 (8.85%) Forward Only Surviving: 787 (2.69%) Reverse Only Surviving: 356 (1.22%) Dropped: 25514 (87.25%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SB135_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SB135_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB135_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB135_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB135_forward.paired.headclipped.fastq.gz SB135_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB135 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB135_forward.paired.headclipped.fastq.gz
[FLASH]     SB135_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB135.extendedFrags.fastq.gz
[FLASH]     ./SB135.notCombined_1.fastq.gz
[FLASH]     ./SB135.notCombined_2.fastq.gz
[FLASH]     ./SB135.hist
[FLASH]     ./SB135.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2587 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2587
[FLASH]     Combined pairs:   277
[FLASH]     Uncombined pairs: 2310
[FLASH]     Percent combined: 10.71%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.015 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

277 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB135/SB135_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB135_centroids.fasta --uc SB135.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 277 sequences and identified 185 clusters (clustering threshold 1.00) - 3 clusters (representing 80 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB136 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB136_1.fastq /home/working/FinalCorrected/demultiplexed/SB136_2.fastq SB136_forward.paired.fastq.gz SB136_forward.singletons.fastq.gz SB136_reverse.paired.fastq.gz SB136_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB136_1.fastq /home/working/FinalCorrected/demultiplexed/SB136_2.fastq SB136_forward.paired.fastq.gz SB136_forward.singletons.fastq.gz SB136_reverse.paired.fastq.gz SB136_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:08:22 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB136/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 43996 Both Surviving: 15162 (34.46%) Forward Only Surviving: 3126 (7.11%) Reverse Only Surviving: 704 (1.60%) Dropped: 25004 (56.83%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB136_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB136_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB136_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB136_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB136_forward.paired.headclipped.fastq.gz SB136_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB136 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB136_forward.paired.headclipped.fastq.gz
[FLASH]     SB136_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB136.extendedFrags.fastq.gz
[FLASH]     ./SB136.notCombined_1.fastq.gz
[FLASH]     ./SB136.notCombined_2.fastq.gz
[FLASH]     ./SB136.hist
[FLASH]     ./SB136.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 15162 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      15162
[FLASH]     Combined pairs:   8678
[FLASH]     Uncombined pairs: 6484
[FLASH]     Percent combined: 57.24%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.243 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

8678 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB136/SB136_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB136_centroids.fasta --uc SB136.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 8678 sequences and identified 3769 clusters (clustering threshold 1.00) - 160 clusters (representing 4524 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB138 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB138_1.fastq /home/working/FinalCorrected/demultiplexed/SB138_2.fastq SB138_forward.paired.fastq.gz SB138_forward.singletons.fastq.gz SB138_reverse.paired.fastq.gz SB138_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB138_1.fastq /home/working/FinalCorrected/demultiplexed/SB138_2.fastq SB138_forward.paired.fastq.gz SB138_forward.singletons.fastq.gz SB138_reverse.paired.fastq.gz SB138_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:08:41 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB138/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 35038 Both Surviving: 11925 (34.03%) Forward Only Surviving: 4745 (13.54%) Reverse Only Surviving: 416 (1.19%) Dropped: 17952 (51.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB138_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB138_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB138_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB138_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB138_forward.paired.headclipped.fastq.gz SB138_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB138 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB138_forward.paired.headclipped.fastq.gz
[FLASH]     SB138_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB138.extendedFrags.fastq.gz
[FLASH]     ./SB138.notCombined_1.fastq.gz
[FLASH]     ./SB138.notCombined_2.fastq.gz
[FLASH]     ./SB138.hist
[FLASH]     ./SB138.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11925 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11925
[FLASH]     Combined pairs:   7192
[FLASH]     Uncombined pairs: 4733
[FLASH]     Percent combined: 60.31%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.201 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

7192 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB138/SB138_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB138_centroids.fasta --uc SB138.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 7192 sequences and identified 2542 clusters (clustering threshold 1.00) - 182 clusters (representing 4448 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB139 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB139_1.fastq /home/working/FinalCorrected/demultiplexed/SB139_2.fastq SB139_forward.paired.fastq.gz SB139_forward.singletons.fastq.gz SB139_reverse.paired.fastq.gz SB139_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB139_1.fastq /home/working/FinalCorrected/demultiplexed/SB139_2.fastq SB139_forward.paired.fastq.gz SB139_forward.singletons.fastq.gz SB139_reverse.paired.fastq.gz SB139_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:08:57 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB139/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 32952 Both Surviving: 6139 (18.63%) Forward Only Surviving: 1682 (5.10%) Reverse Only Surviving: 480 (1.46%) Dropped: 24651 (74.81%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB139_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB139_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB139_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB139_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB139_forward.paired.headclipped.fastq.gz SB139_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB139 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB139_forward.paired.headclipped.fastq.gz
[FLASH]     SB139_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB139.extendedFrags.fastq.gz
[FLASH]     ./SB139.notCombined_1.fastq.gz
[FLASH]     ./SB139.notCombined_2.fastq.gz
[FLASH]     ./SB139.hist
[FLASH]     ./SB139.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6139 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6139
[FLASH]     Combined pairs:   3251
[FLASH]     Uncombined pairs: 2888
[FLASH]     Percent combined: 52.96%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.102 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3251 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB139/SB139_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB139_centroids.fasta --uc SB139.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3251 sequences and identified 1707 clusters (clustering threshold 1.00) - 31 clusters (representing 1401 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB140 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB140_1.fastq /home/working/FinalCorrected/demultiplexed/SB140_2.fastq SB140_forward.paired.fastq.gz SB140_forward.singletons.fastq.gz SB140_reverse.paired.fastq.gz SB140_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB140_1.fastq /home/working/FinalCorrected/demultiplexed/SB140_2.fastq SB140_forward.paired.fastq.gz SB140_forward.singletons.fastq.gz SB140_reverse.paired.fastq.gz SB140_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:09:05 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB140/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11730 Both Surviving: 4648 (39.62%) Forward Only Surviving: 718 (6.12%) Reverse Only Surviving: 205 (1.75%) Dropped: 6159 (52.51%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB140_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB140_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB140_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB140_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB140_forward.paired.headclipped.fastq.gz SB140_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB140 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB140_forward.paired.headclipped.fastq.gz
[FLASH]     SB140_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB140.extendedFrags.fastq.gz
[FLASH]     ./SB140.notCombined_1.fastq.gz
[FLASH]     ./SB140.notCombined_2.fastq.gz
[FLASH]     ./SB140.hist
[FLASH]     ./SB140.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4648 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4648
[FLASH]     Combined pairs:   4230
[FLASH]     Uncombined pairs: 418
[FLASH]     Percent combined: 91.01%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.111 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4230 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB140/SB140_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB140_centroids.fasta --uc SB140.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4230 sequences and identified 1791 clusters (clustering threshold 1.00) - 140 clusters (representing 2275 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB141 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB141_1.fastq /home/working/FinalCorrected/demultiplexed/SB141_2.fastq SB141_forward.paired.fastq.gz SB141_forward.singletons.fastq.gz SB141_reverse.paired.fastq.gz SB141_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB141_1.fastq /home/working/FinalCorrected/demultiplexed/SB141_2.fastq SB141_forward.paired.fastq.gz SB141_forward.singletons.fastq.gz SB141_reverse.paired.fastq.gz SB141_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:09:14 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB141/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9355 Both Surviving: 173 (1.85%) Forward Only Surviving: 141 (1.51%) Reverse Only Surviving: 88 (0.94%) Dropped: 8953 (95.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB141_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB141_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SB141_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SB141_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB141_forward.paired.headclipped.fastq.gz SB141_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB141 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB141_forward.paired.headclipped.fastq.gz
[FLASH]     SB141_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB141.extendedFrags.fastq.gz
[FLASH]     ./SB141.notCombined_1.fastq.gz
[FLASH]     ./SB141.notCombined_2.fastq.gz
[FLASH]     ./SB141.hist
[FLASH]     ./SB141.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 173 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      173
[FLASH]     Combined pairs:   44
[FLASH]     Uncombined pairs: 129
[FLASH]     Percent combined: 25.43%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

44 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB141/SB141_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB141_centroids.fasta --uc SB141.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 44 sequences and identified 39 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB142 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB142_1.fastq /home/working/FinalCorrected/demultiplexed/SB142_2.fastq SB142_forward.paired.fastq.gz SB142_forward.singletons.fastq.gz SB142_reverse.paired.fastq.gz SB142_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB142_1.fastq /home/working/FinalCorrected/demultiplexed/SB142_2.fastq SB142_forward.paired.fastq.gz SB142_forward.singletons.fastq.gz SB142_reverse.paired.fastq.gz SB142_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:09:17 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB142/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22361 Both Surviving: 5354 (23.94%) Forward Only Surviving: 1256 (5.62%) Reverse Only Surviving: 779 (3.48%) Dropped: 14972 (66.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SB142_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB142_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB142_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB142_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB142_forward.paired.headclipped.fastq.gz SB142_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB142 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB142_forward.paired.headclipped.fastq.gz
[FLASH]     SB142_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB142.extendedFrags.fastq.gz
[FLASH]     ./SB142.notCombined_1.fastq.gz
[FLASH]     ./SB142.notCombined_2.fastq.gz
[FLASH]     ./SB142.hist
[FLASH]     ./SB142.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5354 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5354
[FLASH]     Combined pairs:   293
[FLASH]     Uncombined pairs: 5061
[FLASH]     Percent combined: 5.47%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.036 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

293 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB142/SB142_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB142_centroids.fasta --uc SB142.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 293 sequences and identified 239 clusters (clustering threshold 1.00) - 5 clusters (representing 36 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB146 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB146_1.fastq /home/working/FinalCorrected/demultiplexed/SB146_2.fastq SB146_forward.paired.fastq.gz SB146_forward.singletons.fastq.gz SB146_reverse.paired.fastq.gz SB146_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB146_1.fastq /home/working/FinalCorrected/demultiplexed/SB146_2.fastq SB146_forward.paired.fastq.gz SB146_forward.singletons.fastq.gz SB146_reverse.paired.fastq.gz SB146_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:09:21 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB146/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31692 Both Surviving: 5584 (17.62%) Forward Only Surviving: 1183 (3.73%) Reverse Only Surviving: 530 (1.67%) Dropped: 24395 (76.98%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB146_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB146_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB146_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB146_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB146_forward.paired.headclipped.fastq.gz SB146_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB146 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB146_forward.paired.headclipped.fastq.gz
[FLASH]     SB146_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB146.extendedFrags.fastq.gz
[FLASH]     ./SB146.notCombined_1.fastq.gz
[FLASH]     ./SB146.notCombined_2.fastq.gz
[FLASH]     ./SB146.hist
[FLASH]     ./SB146.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5584 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5584
[FLASH]     Combined pairs:   208
[FLASH]     Uncombined pairs: 5376
[FLASH]     Percent combined: 3.72%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.031 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

208 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB146/SB146_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB146_centroids.fasta --uc SB146.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 208 sequences and identified 163 clusters (clustering threshold 1.00) - 1 clusters (representing 32 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB147 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB147_1.fastq /home/working/FinalCorrected/demultiplexed/SB147_2.fastq SB147_forward.paired.fastq.gz SB147_forward.singletons.fastq.gz SB147_reverse.paired.fastq.gz SB147_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB147_1.fastq /home/working/FinalCorrected/demultiplexed/SB147_2.fastq SB147_forward.paired.fastq.gz SB147_forward.singletons.fastq.gz SB147_reverse.paired.fastq.gz SB147_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:09:26 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB147/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 32021 Both Surviving: 8758 (27.35%) Forward Only Surviving: 2026 (6.33%) Reverse Only Surviving: 496 (1.55%) Dropped: 20741 (64.77%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB147_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB147_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB147_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB147_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB147_forward.paired.headclipped.fastq.gz SB147_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB147 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB147_forward.paired.headclipped.fastq.gz
[FLASH]     SB147_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB147.extendedFrags.fastq.gz
[FLASH]     ./SB147.notCombined_1.fastq.gz
[FLASH]     ./SB147.notCombined_2.fastq.gz
[FLASH]     ./SB147.hist
[FLASH]     ./SB147.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8758 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8758
[FLASH]     Combined pairs:   3053
[FLASH]     Uncombined pairs: 5705
[FLASH]     Percent combined: 34.86%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.096 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3053 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB147/SB147_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB147_centroids.fasta --uc SB147.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3053 sequences and identified 1676 clusters (clustering threshold 1.00) - 32 clusters (representing 1242 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB15 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB15_1.fastq /home/working/FinalCorrected/demultiplexed/SB15_2.fastq SB15_forward.paired.fastq.gz SB15_forward.singletons.fastq.gz SB15_reverse.paired.fastq.gz SB15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB15_1.fastq /home/working/FinalCorrected/demultiplexed/SB15_2.fastq SB15_forward.paired.fastq.gz SB15_forward.singletons.fastq.gz SB15_reverse.paired.fastq.gz SB15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:09:38 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB15/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 65483 Both Surviving: 3843 (5.87%) Forward Only Surviving: 1745 (2.66%) Reverse Only Surviving: 903 (1.38%) Dropped: 58992 (90.09%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SB15_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SB15_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB15_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB15_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB15_forward.paired.headclipped.fastq.gz SB15_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB15 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB15_forward.paired.headclipped.fastq.gz
[FLASH]     SB15_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB15.extendedFrags.fastq.gz
[FLASH]     ./SB15.notCombined_1.fastq.gz
[FLASH]     ./SB15.notCombined_2.fastq.gz
[FLASH]     ./SB15.hist
[FLASH]     ./SB15.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3843 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3843
[FLASH]     Combined pairs:   500
[FLASH]     Uncombined pairs: 3343
[FLASH]     Percent combined: 13.01%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.025 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

500 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB15/SB15_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB15_centroids.fasta --uc SB15.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 500 sequences and identified 264 clusters (clustering threshold 1.00) - 8 clusters (representing 214 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB150 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB150_1.fastq /home/working/FinalCorrected/demultiplexed/SB150_2.fastq SB150_forward.paired.fastq.gz SB150_forward.singletons.fastq.gz SB150_reverse.paired.fastq.gz SB150_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB150_1.fastq /home/working/FinalCorrected/demultiplexed/SB150_2.fastq SB150_forward.paired.fastq.gz SB150_forward.singletons.fastq.gz SB150_reverse.paired.fastq.gz SB150_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:09:45 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB150/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 36268 Both Surviving: 8292 (22.86%) Forward Only Surviving: 2052 (5.66%) Reverse Only Surviving: 552 (1.52%) Dropped: 25372 (69.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB150_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB150_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB150_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB150_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB150_forward.paired.headclipped.fastq.gz SB150_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB150 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB150_forward.paired.headclipped.fastq.gz
[FLASH]     SB150_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB150.extendedFrags.fastq.gz
[FLASH]     ./SB150.notCombined_1.fastq.gz
[FLASH]     ./SB150.notCombined_2.fastq.gz
[FLASH]     ./SB150.hist
[FLASH]     ./SB150.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8292 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8292
[FLASH]     Combined pairs:   3195
[FLASH]     Uncombined pairs: 5097
[FLASH]     Percent combined: 38.53%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.096 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3195 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB150/SB150_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB150_centroids.fasta --uc SB150.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3195 sequences and identified 1886 clusters (clustering threshold 1.00) - 37 clusters (representing 1107 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB153 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB153_1.fastq /home/working/FinalCorrected/demultiplexed/SB153_2.fastq SB153_forward.paired.fastq.gz SB153_forward.singletons.fastq.gz SB153_reverse.paired.fastq.gz SB153_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB153_1.fastq /home/working/FinalCorrected/demultiplexed/SB153_2.fastq SB153_forward.paired.fastq.gz SB153_forward.singletons.fastq.gz SB153_reverse.paired.fastq.gz SB153_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:09:55 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB153/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41820 Both Surviving: 11773 (28.15%) Forward Only Surviving: 3169 (7.58%) Reverse Only Surviving: 853 (2.04%) Dropped: 26025 (62.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB153_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB153_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB153_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB153_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB153_forward.paired.headclipped.fastq.gz SB153_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB153 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB153_forward.paired.headclipped.fastq.gz
[FLASH]     SB153_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB153.extendedFrags.fastq.gz
[FLASH]     ./SB153.notCombined_1.fastq.gz
[FLASH]     ./SB153.notCombined_2.fastq.gz
[FLASH]     ./SB153.hist
[FLASH]     ./SB153.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11773 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11773
[FLASH]     Combined pairs:   1450
[FLASH]     Uncombined pairs: 10323
[FLASH]     Percent combined: 12.32%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.087 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1450 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB153/SB153_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB153_centroids.fasta --uc SB153.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1450 sequences and identified 839 clusters (clustering threshold 1.00) - 18 clusters (representing 558 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB159 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB159_1.fastq /home/working/FinalCorrected/demultiplexed/SB159_2.fastq SB159_forward.paired.fastq.gz SB159_forward.singletons.fastq.gz SB159_reverse.paired.fastq.gz SB159_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB159_1.fastq /home/working/FinalCorrected/demultiplexed/SB159_2.fastq SB159_forward.paired.fastq.gz SB159_forward.singletons.fastq.gz SB159_reverse.paired.fastq.gz SB159_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:10:05 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB159/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37399 Both Surviving: 7205 (19.27%) Forward Only Surviving: 1797 (4.80%) Reverse Only Surviving: 683 (1.83%) Dropped: 27714 (74.10%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB159_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB159_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB159_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB159_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB159_forward.paired.headclipped.fastq.gz SB159_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB159 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB159_forward.paired.headclipped.fastq.gz
[FLASH]     SB159_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB159.extendedFrags.fastq.gz
[FLASH]     ./SB159.notCombined_1.fastq.gz
[FLASH]     ./SB159.notCombined_2.fastq.gz
[FLASH]     ./SB159.hist
[FLASH]     ./SB159.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7205 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7205
[FLASH]     Combined pairs:   60
[FLASH]     Uncombined pairs: 7145
[FLASH]     Percent combined: 0.83%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.046 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

60 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB159/SB159_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB159_centroids.fasta --uc SB159.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 60 sequences and identified 51 clusters (clustering threshold 1.00) - 1 clusters (representing 9 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB163 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB163_1.fastq /home/working/FinalCorrected/demultiplexed/SB163_2.fastq SB163_forward.paired.fastq.gz SB163_forward.singletons.fastq.gz SB163_reverse.paired.fastq.gz SB163_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB163_1.fastq /home/working/FinalCorrected/demultiplexed/SB163_2.fastq SB163_forward.paired.fastq.gz SB163_forward.singletons.fastq.gz SB163_reverse.paired.fastq.gz SB163_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:10:13 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB163/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 80933 Both Surviving: 14908 (18.42%) Forward Only Surviving: 5632 (6.96%) Reverse Only Surviving: 1070 (1.32%) Dropped: 59323 (73.30%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SB163_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SB163_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB163_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB163_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB163_forward.paired.headclipped.fastq.gz SB163_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB163 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB163_forward.paired.headclipped.fastq.gz
[FLASH]     SB163_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB163.extendedFrags.fastq.gz
[FLASH]     ./SB163.notCombined_1.fastq.gz
[FLASH]     ./SB163.notCombined_2.fastq.gz
[FLASH]     ./SB163.hist
[FLASH]     ./SB163.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 14908 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      14908
[FLASH]     Combined pairs:   8755
[FLASH]     Uncombined pairs: 6153
[FLASH]     Percent combined: 58.73%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.247 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

8755 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB163/SB163_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB163_centroids.fasta --uc SB163.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 8755 sequences and identified 2594 clusters (clustering threshold 1.00) - 208 clusters (representing 5810 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB166 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB166_1.fastq /home/working/FinalCorrected/demultiplexed/SB166_2.fastq SB166_forward.paired.fastq.gz SB166_forward.singletons.fastq.gz SB166_reverse.paired.fastq.gz SB166_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB166_1.fastq /home/working/FinalCorrected/demultiplexed/SB166_2.fastq SB166_forward.paired.fastq.gz SB166_forward.singletons.fastq.gz SB166_reverse.paired.fastq.gz SB166_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:10:33 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB166/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 24390 Both Surviving: 5441 (22.31%) Forward Only Surviving: 1502 (6.16%) Reverse Only Surviving: 319 (1.31%) Dropped: 17128 (70.23%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB166_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB166_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB166_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB166_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB166_forward.paired.headclipped.fastq.gz SB166_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB166 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB166_forward.paired.headclipped.fastq.gz
[FLASH]     SB166_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB166.extendedFrags.fastq.gz
[FLASH]     ./SB166.notCombined_1.fastq.gz
[FLASH]     ./SB166.notCombined_2.fastq.gz
[FLASH]     ./SB166.hist
[FLASH]     ./SB166.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5441 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5441
[FLASH]     Combined pairs:   2584
[FLASH]     Uncombined pairs: 2857
[FLASH]     Percent combined: 47.49%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.077 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2584 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB166/SB166_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB166_centroids.fasta --uc SB166.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2584 sequences and identified 1272 clusters (clustering threshold 1.00) - 24 clusters (representing 1096 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB170 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB170_1.fastq /home/working/FinalCorrected/demultiplexed/SB170_2.fastq SB170_forward.paired.fastq.gz SB170_forward.singletons.fastq.gz SB170_reverse.paired.fastq.gz SB170_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB170_1.fastq /home/working/FinalCorrected/demultiplexed/SB170_2.fastq SB170_forward.paired.fastq.gz SB170_forward.singletons.fastq.gz SB170_reverse.paired.fastq.gz SB170_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:10:41 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB170/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 40667 Both Surviving: 9713 (23.88%) Forward Only Surviving: 2448 (6.02%) Reverse Only Surviving: 653 (1.61%) Dropped: 27853 (68.49%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB170_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB170_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB170_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB170_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB170_forward.paired.headclipped.fastq.gz SB170_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB170 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB170_forward.paired.headclipped.fastq.gz
[FLASH]     SB170_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB170.extendedFrags.fastq.gz
[FLASH]     ./SB170.notCombined_1.fastq.gz
[FLASH]     ./SB170.notCombined_2.fastq.gz
[FLASH]     ./SB170.hist
[FLASH]     ./SB170.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9713 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9713
[FLASH]     Combined pairs:   2788
[FLASH]     Uncombined pairs: 6925
[FLASH]     Percent combined: 28.70%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.090 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2788 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB170/SB170_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB170_centroids.fasta --uc SB170.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2788 sequences and identified 1482 clusters (clustering threshold 1.00) - 26 clusters (representing 1114 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB171 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB171_1.fastq /home/working/FinalCorrected/demultiplexed/SB171_2.fastq SB171_forward.paired.fastq.gz SB171_forward.singletons.fastq.gz SB171_reverse.paired.fastq.gz SB171_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB171_1.fastq /home/working/FinalCorrected/demultiplexed/SB171_2.fastq SB171_forward.paired.fastq.gz SB171_forward.singletons.fastq.gz SB171_reverse.paired.fastq.gz SB171_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:10:50 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB171/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14032 Both Surviving: 3915 (27.90%) Forward Only Surviving: 1302 (9.28%) Reverse Only Surviving: 152 (1.08%) Dropped: 8663 (61.74%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB171_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB171_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SB171_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SB171_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB171_forward.paired.headclipped.fastq.gz SB171_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB171 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB171_forward.paired.headclipped.fastq.gz
[FLASH]     SB171_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB171.extendedFrags.fastq.gz
[FLASH]     ./SB171.notCombined_1.fastq.gz
[FLASH]     ./SB171.notCombined_2.fastq.gz
[FLASH]     ./SB171.hist
[FLASH]     ./SB171.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3915 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3915
[FLASH]     Combined pairs:   265
[FLASH]     Uncombined pairs: 3650
[FLASH]     Percent combined: 6.77%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.025 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

265 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB171/SB171_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB171_centroids.fasta --uc SB171.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 265 sequences and identified 233 clusters (clustering threshold 1.00) - 2 clusters (representing 17 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB172 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB172_1.fastq /home/working/FinalCorrected/demultiplexed/SB172_2.fastq SB172_forward.paired.fastq.gz SB172_forward.singletons.fastq.gz SB172_reverse.paired.fastq.gz SB172_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB172_1.fastq /home/working/FinalCorrected/demultiplexed/SB172_2.fastq SB172_forward.paired.fastq.gz SB172_forward.singletons.fastq.gz SB172_reverse.paired.fastq.gz SB172_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:10:54 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB172/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 43254 Both Surviving: 10559 (24.41%) Forward Only Surviving: 2599 (6.01%) Reverse Only Surviving: 1370 (3.17%) Dropped: 28726 (66.41%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SB172_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB172_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB172_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB172_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB172_forward.paired.headclipped.fastq.gz SB172_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB172 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB172_forward.paired.headclipped.fastq.gz
[FLASH]     SB172_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB172.extendedFrags.fastq.gz
[FLASH]     ./SB172.notCombined_1.fastq.gz
[FLASH]     ./SB172.notCombined_2.fastq.gz
[FLASH]     ./SB172.hist
[FLASH]     ./SB172.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 10559 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      10559
[FLASH]     Combined pairs:   737
[FLASH]     Uncombined pairs: 9822
[FLASH]     Percent combined: 6.98%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.068 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

737 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB172/SB172_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB172_centroids.fasta --uc SB172.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 737 sequences and identified 480 clusters (clustering threshold 1.00) - 11 clusters (representing 227 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB180 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB180_1.fastq /home/working/FinalCorrected/demultiplexed/SB180_2.fastq SB180_forward.paired.fastq.gz SB180_forward.singletons.fastq.gz SB180_reverse.paired.fastq.gz SB180_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB180_1.fastq /home/working/FinalCorrected/demultiplexed/SB180_2.fastq SB180_forward.paired.fastq.gz SB180_forward.singletons.fastq.gz SB180_reverse.paired.fastq.gz SB180_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:11:02 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB180/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 34105 Both Surviving: 6549 (19.20%) Forward Only Surviving: 1906 (5.59%) Reverse Only Surviving: 465 (1.36%) Dropped: 25185 (73.85%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB180_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB180_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB180_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB180_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB180_forward.paired.headclipped.fastq.gz SB180_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB180 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB180_forward.paired.headclipped.fastq.gz
[FLASH]     SB180_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB180.extendedFrags.fastq.gz
[FLASH]     ./SB180.notCombined_1.fastq.gz
[FLASH]     ./SB180.notCombined_2.fastq.gz
[FLASH]     ./SB180.hist
[FLASH]     ./SB180.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6549 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6549
[FLASH]     Combined pairs:   2670
[FLASH]     Uncombined pairs: 3879
[FLASH]     Percent combined: 40.77%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.080 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2670 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB180/SB180_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB180_centroids.fasta --uc SB180.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2670 sequences and identified 1420 clusters (clustering threshold 1.00) - 24 clusters (representing 1121 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB182 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB182_1.fastq /home/working/FinalCorrected/demultiplexed/SB182_2.fastq SB182_forward.paired.fastq.gz SB182_forward.singletons.fastq.gz SB182_reverse.paired.fastq.gz SB182_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB182_1.fastq /home/working/FinalCorrected/demultiplexed/SB182_2.fastq SB182_forward.paired.fastq.gz SB182_forward.singletons.fastq.gz SB182_reverse.paired.fastq.gz SB182_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:11:12 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB182/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 49576 Both Surviving: 154 (0.31%) Forward Only Surviving: 517 (1.04%) Reverse Only Surviving: 627 (1.26%) Dropped: 48278 (97.38%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB182_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB182_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB182_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB182_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB182_forward.paired.headclipped.fastq.gz SB182_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB182 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB182_forward.paired.headclipped.fastq.gz
[FLASH]     SB182_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB182.extendedFrags.fastq.gz
[FLASH]     ./SB182.notCombined_1.fastq.gz
[FLASH]     ./SB182.notCombined_2.fastq.gz
[FLASH]     ./SB182.hist
[FLASH]     ./SB182.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 154 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      154
[FLASH]     Combined pairs:   54
[FLASH]     Uncombined pairs: 100
[FLASH]     Percent combined: 35.06%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

54 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB182/SB182_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB182_centroids.fasta --uc SB182.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 54 sequences and identified 45 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB183 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB183_1.fastq /home/working/FinalCorrected/demultiplexed/SB183_2.fastq SB183_forward.paired.fastq.gz SB183_forward.singletons.fastq.gz SB183_reverse.paired.fastq.gz SB183_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB183_1.fastq /home/working/FinalCorrected/demultiplexed/SB183_2.fastq SB183_forward.paired.fastq.gz SB183_forward.singletons.fastq.gz SB183_reverse.paired.fastq.gz SB183_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:11:15 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB183/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14364 Both Surviving: 316 (2.20%) Forward Only Surviving: 343 (2.39%) Reverse Only Surviving: 130 (0.91%) Dropped: 13575 (94.51%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB183_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB183_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB183_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB183_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB183_forward.paired.headclipped.fastq.gz SB183_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB183 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB183_forward.paired.headclipped.fastq.gz
[FLASH]     SB183_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB183.extendedFrags.fastq.gz
[FLASH]     ./SB183.notCombined_1.fastq.gz
[FLASH]     ./SB183.notCombined_2.fastq.gz
[FLASH]     ./SB183.hist
[FLASH]     ./SB183.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 316 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      316
[FLASH]     Combined pairs:   237
[FLASH]     Uncombined pairs: 79
[FLASH]     Percent combined: 75.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.012 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

237 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB183/SB183_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB183_centroids.fasta --uc SB183.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 237 sequences and identified 140 clusters (clustering threshold 1.00) - 7 clusters (representing 91 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB186 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB186_1.fastq /home/working/FinalCorrected/demultiplexed/SB186_2.fastq SB186_forward.paired.fastq.gz SB186_forward.singletons.fastq.gz SB186_reverse.paired.fastq.gz SB186_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB186_1.fastq /home/working/FinalCorrected/demultiplexed/SB186_2.fastq SB186_forward.paired.fastq.gz SB186_forward.singletons.fastq.gz SB186_reverse.paired.fastq.gz SB186_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:11:20 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB186/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 72012 Both Surviving: 3107 (4.31%) Forward Only Surviving: 2444 (3.39%) Reverse Only Surviving: 755 (1.05%) Dropped: 65706 (91.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB186_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB186_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB186_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB186_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB186_forward.paired.headclipped.fastq.gz SB186_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB186 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB186_forward.paired.headclipped.fastq.gz
[FLASH]     SB186_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB186.extendedFrags.fastq.gz
[FLASH]     ./SB186.notCombined_1.fastq.gz
[FLASH]     ./SB186.notCombined_2.fastq.gz
[FLASH]     ./SB186.hist
[FLASH]     ./SB186.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3107 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3107
[FLASH]     Combined pairs:   2217
[FLASH]     Uncombined pairs: 890
[FLASH]     Percent combined: 71.36%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.055 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2217 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB186/SB186_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB186_centroids.fasta --uc SB186.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2217 sequences and identified 829 clusters (clustering threshold 1.00) - 35 clusters (representing 1212 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB194 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB194_1.fastq /home/working/FinalCorrected/demultiplexed/SB194_2.fastq SB194_forward.paired.fastq.gz SB194_forward.singletons.fastq.gz SB194_reverse.paired.fastq.gz SB194_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB194_1.fastq /home/working/FinalCorrected/demultiplexed/SB194_2.fastq SB194_forward.paired.fastq.gz SB194_forward.singletons.fastq.gz SB194_reverse.paired.fastq.gz SB194_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:11:32 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB194/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 97194 Both Surviving: 24529 (25.24%) Forward Only Surviving: 8080 (8.31%) Reverse Only Surviving: 1643 (1.69%) Dropped: 62942 (64.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB194_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB194_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB194_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB194_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB194_forward.paired.headclipped.fastq.gz SB194_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB194 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB194_forward.paired.headclipped.fastq.gz
[FLASH]     SB194_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB194.extendedFrags.fastq.gz
[FLASH]     ./SB194.notCombined_1.fastq.gz
[FLASH]     ./SB194.notCombined_2.fastq.gz
[FLASH]     ./SB194.hist
[FLASH]     ./SB194.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 24529 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      24529
[FLASH]     Combined pairs:   2260
[FLASH]     Uncombined pairs: 22269
[FLASH]     Percent combined: 9.21%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.139 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2260 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB194/SB194_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB194_centroids.fasta --uc SB194.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2260 sequences and identified 1427 clusters (clustering threshold 1.00) - 32 clusters (representing 629 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB195 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB195_1.fastq /home/working/FinalCorrected/demultiplexed/SB195_2.fastq SB195_forward.paired.fastq.gz SB195_forward.singletons.fastq.gz SB195_reverse.paired.fastq.gz SB195_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB195_1.fastq /home/working/FinalCorrected/demultiplexed/SB195_2.fastq SB195_forward.paired.fastq.gz SB195_forward.singletons.fastq.gz SB195_reverse.paired.fastq.gz SB195_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:11:47 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB195/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25672 Both Surviving: 6139 (23.91%) Forward Only Surviving: 920 (3.58%) Reverse Only Surviving: 954 (3.72%) Dropped: 17659 (68.79%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SB195_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB195_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB195_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB195_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB195_forward.paired.headclipped.fastq.gz SB195_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB195 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB195_forward.paired.headclipped.fastq.gz
[FLASH]     SB195_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB195.extendedFrags.fastq.gz
[FLASH]     ./SB195.notCombined_1.fastq.gz
[FLASH]     ./SB195.notCombined_2.fastq.gz
[FLASH]     ./SB195.hist
[FLASH]     ./SB195.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6139 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6139
[FLASH]     Combined pairs:   198
[FLASH]     Uncombined pairs: 5941
[FLASH]     Percent combined: 3.23%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.039 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

198 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB195/SB195_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB195_centroids.fasta --uc SB195.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 198 sequences and identified 155 clusters (clustering threshold 1.00) - 3 clusters (representing 27 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB197 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB197_1.fastq /home/working/FinalCorrected/demultiplexed/SB197_2.fastq SB197_forward.paired.fastq.gz SB197_forward.singletons.fastq.gz SB197_reverse.paired.fastq.gz SB197_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB197_1.fastq /home/working/FinalCorrected/demultiplexed/SB197_2.fastq SB197_forward.paired.fastq.gz SB197_forward.singletons.fastq.gz SB197_reverse.paired.fastq.gz SB197_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:11:53 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB197/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 53032 Both Surviving: 13391 (25.25%) Forward Only Surviving: 3997 (7.54%) Reverse Only Surviving: 831 (1.57%) Dropped: 34813 (65.65%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB197_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB197_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB197_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB197_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB197_forward.paired.headclipped.fastq.gz SB197_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB197 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB197_forward.paired.headclipped.fastq.gz
[FLASH]     SB197_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB197.extendedFrags.fastq.gz
[FLASH]     ./SB197.notCombined_1.fastq.gz
[FLASH]     ./SB197.notCombined_2.fastq.gz
[FLASH]     ./SB197.hist
[FLASH]     ./SB197.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 13391 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      13391
[FLASH]     Combined pairs:   461
[FLASH]     Uncombined pairs: 12930
[FLASH]     Percent combined: 3.44%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.081 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

461 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB197/SB197_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB197_centroids.fasta --uc SB197.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 461 sequences and identified 319 clusters (clustering threshold 1.00) - 9 clusters (representing 105 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB199 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB199_1.fastq /home/working/FinalCorrected/demultiplexed/SB199_2.fastq SB199_forward.paired.fastq.gz SB199_forward.singletons.fastq.gz SB199_reverse.paired.fastq.gz SB199_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB199_1.fastq /home/working/FinalCorrected/demultiplexed/SB199_2.fastq SB199_forward.paired.fastq.gz SB199_forward.singletons.fastq.gz SB199_reverse.paired.fastq.gz SB199_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:12:01 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB199/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20753 Both Surviving: 2795 (13.47%) Forward Only Surviving: 931 (4.49%) Reverse Only Surviving: 313 (1.51%) Dropped: 16714 (80.54%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB199_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB199_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB199_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB199_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB199_forward.paired.headclipped.fastq.gz SB199_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB199 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB199_forward.paired.headclipped.fastq.gz
[FLASH]     SB199_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB199.extendedFrags.fastq.gz
[FLASH]     ./SB199.notCombined_1.fastq.gz
[FLASH]     ./SB199.notCombined_2.fastq.gz
[FLASH]     ./SB199.hist
[FLASH]     ./SB199.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2795 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2795
[FLASH]     Combined pairs:   56
[FLASH]     Uncombined pairs: 2739
[FLASH]     Percent combined: 2.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.018 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

56 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB199/SB199_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB199_centroids.fasta --uc SB199.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 56 sequences and identified 49 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB200 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB200_1.fastq /home/working/FinalCorrected/demultiplexed/SB200_2.fastq SB200_forward.paired.fastq.gz SB200_forward.singletons.fastq.gz SB200_reverse.paired.fastq.gz SB200_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB200_1.fastq /home/working/FinalCorrected/demultiplexed/SB200_2.fastq SB200_forward.paired.fastq.gz SB200_forward.singletons.fastq.gz SB200_reverse.paired.fastq.gz SB200_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:12:03 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB200/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SB200_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB200_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB200_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB200_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB200_forward.paired.headclipped.fastq.gz SB200_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB200 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB200_forward.paired.headclipped.fastq.gz
[FLASH]     SB200_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB200.extendedFrags.fastq.gz
[FLASH]     ./SB200.notCombined_1.fastq.gz
[FLASH]     ./SB200.notCombined_2.fastq.gz
[FLASH]     ./SB200.hist
[FLASH]     ./SB200.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB200/SB200_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB200_centroids.fasta --uc SB200.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB201 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB201_1.fastq /home/working/FinalCorrected/demultiplexed/SB201_2.fastq SB201_forward.paired.fastq.gz SB201_forward.singletons.fastq.gz SB201_reverse.paired.fastq.gz SB201_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB201_1.fastq /home/working/FinalCorrected/demultiplexed/SB201_2.fastq SB201_forward.paired.fastq.gz SB201_forward.singletons.fastq.gz SB201_reverse.paired.fastq.gz SB201_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:12:04 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB201/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 33178 Both Surviving: 1958 (5.90%) Forward Only Surviving: 1408 (4.24%) Reverse Only Surviving: 279 (0.84%) Dropped: 29533 (89.01%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB201_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB201_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB201_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB201_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB201_forward.paired.headclipped.fastq.gz SB201_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB201 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB201_forward.paired.headclipped.fastq.gz
[FLASH]     SB201_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB201.extendedFrags.fastq.gz
[FLASH]     ./SB201.notCombined_1.fastq.gz
[FLASH]     ./SB201.notCombined_2.fastq.gz
[FLASH]     ./SB201.hist
[FLASH]     ./SB201.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1958 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1958
[FLASH]     Combined pairs:   826
[FLASH]     Uncombined pairs: 1132
[FLASH]     Percent combined: 42.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.025 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

826 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB201/SB201_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB201_centroids.fasta --uc SB201.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 826 sequences and identified 569 clusters (clustering threshold 1.00) - 19 clusters (representing 228 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB204 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB204_1.fastq /home/working/FinalCorrected/demultiplexed/SB204_2.fastq SB204_forward.paired.fastq.gz SB204_forward.singletons.fastq.gz SB204_reverse.paired.fastq.gz SB204_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB204_1.fastq /home/working/FinalCorrected/demultiplexed/SB204_2.fastq SB204_forward.paired.fastq.gz SB204_forward.singletons.fastq.gz SB204_reverse.paired.fastq.gz SB204_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:12:12 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB204/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 99793 Both Surviving: 5432 (5.44%) Forward Only Surviving: 2232 (2.24%) Reverse Only Surviving: 1231 (1.23%) Dropped: 90898 (91.09%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB204_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB204_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB204_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB204_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB204_forward.paired.headclipped.fastq.gz SB204_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB204 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB204_forward.paired.headclipped.fastq.gz
[FLASH]     SB204_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB204.extendedFrags.fastq.gz
[FLASH]     ./SB204.notCombined_1.fastq.gz
[FLASH]     ./SB204.notCombined_2.fastq.gz
[FLASH]     ./SB204.hist
[FLASH]     ./SB204.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5432 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5432
[FLASH]     Combined pairs:   4767
[FLASH]     Uncombined pairs: 665
[FLASH]     Percent combined: 87.76%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.122 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4767 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB204/SB204_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB204_centroids.fasta --uc SB204.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4767 sequences and identified 1938 clusters (clustering threshold 1.00) - 70 clusters (representing 2414 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB21 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB21_1.fastq /home/working/FinalCorrected/demultiplexed/SB21_2.fastq SB21_forward.paired.fastq.gz SB21_forward.singletons.fastq.gz SB21_reverse.paired.fastq.gz SB21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB21_1.fastq /home/working/FinalCorrected/demultiplexed/SB21_2.fastq SB21_forward.paired.fastq.gz SB21_forward.singletons.fastq.gz SB21_reverse.paired.fastq.gz SB21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:12:26 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB21/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25058 Both Surviving: 5378 (21.46%) Forward Only Surviving: 1236 (4.93%) Reverse Only Surviving: 413 (1.65%) Dropped: 18031 (71.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB21_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB21_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB21_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB21_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB21_forward.paired.headclipped.fastq.gz SB21_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB21 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB21_forward.paired.headclipped.fastq.gz
[FLASH]     SB21_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB21.extendedFrags.fastq.gz
[FLASH]     ./SB21.notCombined_1.fastq.gz
[FLASH]     ./SB21.notCombined_2.fastq.gz
[FLASH]     ./SB21.hist
[FLASH]     ./SB21.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5378 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5378
[FLASH]     Combined pairs:   3058
[FLASH]     Uncombined pairs: 2320
[FLASH]     Percent combined: 56.86%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.093 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3058 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB21/SB21_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB21_centroids.fasta --uc SB21.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3058 sequences and identified 1907 clusters (clustering threshold 1.00) - 45 clusters (representing 1006 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB22 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB22_1.fastq /home/working/FinalCorrected/demultiplexed/SB22_2.fastq SB22_forward.paired.fastq.gz SB22_forward.singletons.fastq.gz SB22_reverse.paired.fastq.gz SB22_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB22_1.fastq /home/working/FinalCorrected/demultiplexed/SB22_2.fastq SB22_forward.paired.fastq.gz SB22_forward.singletons.fastq.gz SB22_reverse.paired.fastq.gz SB22_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:12:36 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB22/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37803 Both Surviving: 9518 (25.18%) Forward Only Surviving: 2446 (6.47%) Reverse Only Surviving: 592 (1.57%) Dropped: 25247 (66.79%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB22_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB22_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB22_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB22_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB22_forward.paired.headclipped.fastq.gz SB22_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB22 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB22_forward.paired.headclipped.fastq.gz
[FLASH]     SB22_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB22.extendedFrags.fastq.gz
[FLASH]     ./SB22.notCombined_1.fastq.gz
[FLASH]     ./SB22.notCombined_2.fastq.gz
[FLASH]     ./SB22.hist
[FLASH]     ./SB22.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9518 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9518
[FLASH]     Combined pairs:   4105
[FLASH]     Uncombined pairs: 5413
[FLASH]     Percent combined: 43.13%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.121 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4105 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB22/SB22_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB22_centroids.fasta --uc SB22.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4105 sequences and identified 1997 clusters (clustering threshold 1.00) - 40 clusters (representing 1786 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB23 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB23_1.fastq /home/working/FinalCorrected/demultiplexed/SB23_2.fastq SB23_forward.paired.fastq.gz SB23_forward.singletons.fastq.gz SB23_reverse.paired.fastq.gz SB23_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB23_1.fastq /home/working/FinalCorrected/demultiplexed/SB23_2.fastq SB23_forward.paired.fastq.gz SB23_forward.singletons.fastq.gz SB23_reverse.paired.fastq.gz SB23_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:12:46 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB23/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SB23_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB23_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB23_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB23_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB23_forward.paired.headclipped.fastq.gz SB23_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB23 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB23_forward.paired.headclipped.fastq.gz
[FLASH]     SB23_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB23.extendedFrags.fastq.gz
[FLASH]     ./SB23.notCombined_1.fastq.gz
[FLASH]     ./SB23.notCombined_2.fastq.gz
[FLASH]     ./SB23.hist
[FLASH]     ./SB23.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB23/SB23_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB23_centroids.fasta --uc SB23.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB24 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB24_1.fastq /home/working/FinalCorrected/demultiplexed/SB24_2.fastq SB24_forward.paired.fastq.gz SB24_forward.singletons.fastq.gz SB24_reverse.paired.fastq.gz SB24_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB24_1.fastq /home/working/FinalCorrected/demultiplexed/SB24_2.fastq SB24_forward.paired.fastq.gz SB24_forward.singletons.fastq.gz SB24_reverse.paired.fastq.gz SB24_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:12:48 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB24/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28519 Both Surviving: 11971 (41.98%) Forward Only Surviving: 2534 (8.89%) Reverse Only Surviving: 494 (1.73%) Dropped: 13520 (47.41%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB24_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB24_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB24_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB24_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB24_forward.paired.headclipped.fastq.gz SB24_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB24 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB24_forward.paired.headclipped.fastq.gz
[FLASH]     SB24_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB24.extendedFrags.fastq.gz
[FLASH]     ./SB24.notCombined_1.fastq.gz
[FLASH]     ./SB24.notCombined_2.fastq.gz
[FLASH]     ./SB24.hist
[FLASH]     ./SB24.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 11971 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      11971
[FLASH]     Combined pairs:   6050
[FLASH]     Uncombined pairs: 5921
[FLASH]     Percent combined: 50.54%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.170 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

6050 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB24/SB24_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB24_centroids.fasta --uc SB24.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 6050 sequences and identified 3589 clusters (clustering threshold 1.00) - 138 clusters (representing 2114 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB27 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB27_1.fastq /home/working/FinalCorrected/demultiplexed/SB27_2.fastq SB27_forward.paired.fastq.gz SB27_forward.singletons.fastq.gz SB27_reverse.paired.fastq.gz SB27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB27_1.fastq /home/working/FinalCorrected/demultiplexed/SB27_2.fastq SB27_forward.paired.fastq.gz SB27_forward.singletons.fastq.gz SB27_reverse.paired.fastq.gz SB27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:13:00 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB27/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15596 Both Surviving: 2718 (17.43%) Forward Only Surviving: 745 (4.78%) Reverse Only Surviving: 204 (1.31%) Dropped: 11929 (76.49%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB27_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB27_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB27_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB27_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB27_forward.paired.headclipped.fastq.gz SB27_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB27 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB27_forward.paired.headclipped.fastq.gz
[FLASH]     SB27_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB27.extendedFrags.fastq.gz
[FLASH]     ./SB27.notCombined_1.fastq.gz
[FLASH]     ./SB27.notCombined_2.fastq.gz
[FLASH]     ./SB27.hist
[FLASH]     ./SB27.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2718 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2718
[FLASH]     Combined pairs:   55
[FLASH]     Uncombined pairs: 2663
[FLASH]     Percent combined: 2.02%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.021 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

55 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB27/SB27_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB27_centroids.fasta --uc SB27.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 55 sequences and identified 46 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB29 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB29_1.fastq /home/working/FinalCorrected/demultiplexed/SB29_2.fastq SB29_forward.paired.fastq.gz SB29_forward.singletons.fastq.gz SB29_reverse.paired.fastq.gz SB29_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB29_1.fastq /home/working/FinalCorrected/demultiplexed/SB29_2.fastq SB29_forward.paired.fastq.gz SB29_forward.singletons.fastq.gz SB29_reverse.paired.fastq.gz SB29_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:13:04 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB29/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37392 Both Surviving: 1224 (3.27%) Forward Only Surviving: 976 (2.61%) Reverse Only Surviving: 315 (0.84%) Dropped: 34877 (93.27%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB29_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB29_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB29_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB29_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB29_forward.paired.headclipped.fastq.gz SB29_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB29 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB29_forward.paired.headclipped.fastq.gz
[FLASH]     SB29_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB29.extendedFrags.fastq.gz
[FLASH]     ./SB29.notCombined_1.fastq.gz
[FLASH]     ./SB29.notCombined_2.fastq.gz
[FLASH]     ./SB29.hist
[FLASH]     ./SB29.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1224 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1224
[FLASH]     Combined pairs:   867
[FLASH]     Uncombined pairs: 357
[FLASH]     Percent combined: 70.83%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.043 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

867 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB29/SB29_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB29_centroids.fasta --uc SB29.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 867 sequences and identified 475 clusters (clustering threshold 1.00) - 12 clusters (representing 341 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB3 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB3_1.fastq /home/working/FinalCorrected/demultiplexed/SB3_2.fastq SB3_forward.paired.fastq.gz SB3_forward.singletons.fastq.gz SB3_reverse.paired.fastq.gz SB3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB3_1.fastq /home/working/FinalCorrected/demultiplexed/SB3_2.fastq SB3_forward.paired.fastq.gz SB3_forward.singletons.fastq.gz SB3_reverse.paired.fastq.gz SB3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:13:13 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB3/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 107968 Both Surviving: 4015 (3.72%) Forward Only Surviving: 5331 (4.94%) Reverse Only Surviving: 1056 (0.98%) Dropped: 97566 (90.37%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB3_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB3_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB3_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB3_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB3_forward.paired.headclipped.fastq.gz SB3_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB3 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB3_forward.paired.headclipped.fastq.gz
[FLASH]     SB3_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB3.extendedFrags.fastq.gz
[FLASH]     ./SB3.notCombined_1.fastq.gz
[FLASH]     ./SB3.notCombined_2.fastq.gz
[FLASH]     ./SB3.hist
[FLASH]     ./SB3.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4015 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4015
[FLASH]     Combined pairs:   1099
[FLASH]     Uncombined pairs: 2916
[FLASH]     Percent combined: 27.37%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.035 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1099 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB3/SB3_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB3_centroids.fasta --uc SB3.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1099 sequences and identified 670 clusters (clustering threshold 1.00) - 31 clusters (representing 390 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB33 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB33_1.fastq /home/working/FinalCorrected/demultiplexed/SB33_2.fastq SB33_forward.paired.fastq.gz SB33_forward.singletons.fastq.gz SB33_reverse.paired.fastq.gz SB33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB33_1.fastq /home/working/FinalCorrected/demultiplexed/SB33_2.fastq SB33_forward.paired.fastq.gz SB33_forward.singletons.fastq.gz SB33_reverse.paired.fastq.gz SB33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:13:34 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB33/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 250333 Both Surviving: 102563 (40.97%) Forward Only Surviving: 23564 (9.41%) Reverse Only Surviving: 4893 (1.95%) Dropped: 119313 (47.66%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB33_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB33_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB33_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB33_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB33_forward.paired.headclipped.fastq.gz SB33_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB33 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB33_forward.paired.headclipped.fastq.gz
[FLASH]     SB33_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB33.extendedFrags.fastq.gz
[FLASH]     ./SB33.notCombined_1.fastq.gz
[FLASH]     ./SB33.notCombined_2.fastq.gz
[FLASH]     ./SB33.hist
[FLASH]     ./SB33.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 50000 read pairs
[FLASH] Processed 75000 read pairs
[FLASH] Processed 100000 read pairs
[FLASH] Processed 102563 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      102563
[FLASH]     Combined pairs:   60103
[FLASH]     Uncombined pairs: 42460
[FLASH]     Percent combined: 58.60%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 1.575 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

60103 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB33/SB33_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB33_centroids.fasta --uc SB33.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 60103 sequences and identified 22427 clusters (clustering threshold 1.00) - 747 clusters (representing 35660 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB34 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB34_1.fastq /home/working/FinalCorrected/demultiplexed/SB34_2.fastq SB34_forward.paired.fastq.gz SB34_forward.singletons.fastq.gz SB34_reverse.paired.fastq.gz SB34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB34_1.fastq /home/working/FinalCorrected/demultiplexed/SB34_2.fastq SB34_forward.paired.fastq.gz SB34_forward.singletons.fastq.gz SB34_reverse.paired.fastq.gz SB34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:15:57 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB34/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 5099 Both Surviving: 1755 (34.42%) Forward Only Surviving: 415 (8.14%) Reverse Only Surviving: 67 (1.31%) Dropped: 2862 (56.13%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB34_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB34_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB34_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB34_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB34_forward.paired.headclipped.fastq.gz SB34_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB34 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB34_forward.paired.headclipped.fastq.gz
[FLASH]     SB34_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB34.extendedFrags.fastq.gz
[FLASH]     ./SB34.notCombined_1.fastq.gz
[FLASH]     ./SB34.notCombined_2.fastq.gz
[FLASH]     ./SB34.hist
[FLASH]     ./SB34.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1755 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1755
[FLASH]     Combined pairs:   721
[FLASH]     Uncombined pairs: 1034
[FLASH]     Percent combined: 41.08%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.027 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

721 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB34/SB34_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB34_centroids.fasta --uc SB34.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 721 sequences and identified 509 clusters (clustering threshold 1.00) - 13 clusters (representing 166 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB37 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB37_1.fastq /home/working/FinalCorrected/demultiplexed/SB37_2.fastq SB37_forward.paired.fastq.gz SB37_forward.singletons.fastq.gz SB37_reverse.paired.fastq.gz SB37_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB37_1.fastq /home/working/FinalCorrected/demultiplexed/SB37_2.fastq SB37_forward.paired.fastq.gz SB37_forward.singletons.fastq.gz SB37_reverse.paired.fastq.gz SB37_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:16:00 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB37/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15290 Both Surviving: 2458 (16.08%) Forward Only Surviving: 686 (4.49%) Reverse Only Surviving: 189 (1.24%) Dropped: 11957 (78.20%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB37_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB37_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB37_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB37_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB37_forward.paired.headclipped.fastq.gz SB37_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB37 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB37_forward.paired.headclipped.fastq.gz
[FLASH]     SB37_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB37.extendedFrags.fastq.gz
[FLASH]     ./SB37.notCombined_1.fastq.gz
[FLASH]     ./SB37.notCombined_2.fastq.gz
[FLASH]     ./SB37.hist
[FLASH]     ./SB37.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2458 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2458
[FLASH]     Combined pairs:   62
[FLASH]     Uncombined pairs: 2396
[FLASH]     Percent combined: 2.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.017 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

62 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB37/SB37_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB37_centroids.fasta --uc SB37.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 62 sequences and identified 44 clusters (clustering threshold 1.00) - 1 clusters (representing 7 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB4 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB4_1.fastq /home/working/FinalCorrected/demultiplexed/SB4_2.fastq SB4_forward.paired.fastq.gz SB4_forward.singletons.fastq.gz SB4_reverse.paired.fastq.gz SB4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB4_1.fastq /home/working/FinalCorrected/demultiplexed/SB4_2.fastq SB4_forward.paired.fastq.gz SB4_forward.singletons.fastq.gz SB4_reverse.paired.fastq.gz SB4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:16:03 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB4/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17746 Both Surviving: 2361 (13.30%) Forward Only Surviving: 616 (3.47%) Reverse Only Surviving: 545 (3.07%) Dropped: 14224 (80.15%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SB4_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB4_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB4_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB4_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB4_forward.paired.headclipped.fastq.gz SB4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB4 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB4_forward.paired.headclipped.fastq.gz
[FLASH]     SB4_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB4.extendedFrags.fastq.gz
[FLASH]     ./SB4.notCombined_1.fastq.gz
[FLASH]     ./SB4.notCombined_2.fastq.gz
[FLASH]     ./SB4.hist
[FLASH]     ./SB4.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2361 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2361
[FLASH]     Combined pairs:   110
[FLASH]     Uncombined pairs: 2251
[FLASH]     Percent combined: 4.66%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.017 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

110 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB4/SB4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB4_centroids.fasta --uc SB4.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 110 sequences and identified 104 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB45 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB45_1.fastq /home/working/FinalCorrected/demultiplexed/SB45_2.fastq SB45_forward.paired.fastq.gz SB45_forward.singletons.fastq.gz SB45_reverse.paired.fastq.gz SB45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB45_1.fastq /home/working/FinalCorrected/demultiplexed/SB45_2.fastq SB45_forward.paired.fastq.gz SB45_forward.singletons.fastq.gz SB45_reverse.paired.fastq.gz SB45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:16:11 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB45/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 114934 Both Surviving: 33820 (29.43%) Forward Only Surviving: 8649 (7.53%) Reverse Only Surviving: 1860 (1.62%) Dropped: 70605 (61.43%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB45_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB45_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB45_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB45_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB45_forward.paired.headclipped.fastq.gz SB45_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB45 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB45_forward.paired.headclipped.fastq.gz
[FLASH]     SB45_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB45.extendedFrags.fastq.gz
[FLASH]     ./SB45.notCombined_1.fastq.gz
[FLASH]     ./SB45.notCombined_2.fastq.gz
[FLASH]     ./SB45.hist
[FLASH]     ./SB45.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 33820 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      33820
[FLASH]     Combined pairs:   16731
[FLASH]     Uncombined pairs: 17089
[FLASH]     Percent combined: 49.47%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.462 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

16731 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB45/SB45_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB45_centroids.fasta --uc SB45.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 16731 sequences and identified 5203 clusters (clustering threshold 1.00) - 315 clusters (representing 10908 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB48 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB48_1.fastq /home/working/FinalCorrected/demultiplexed/SB48_2.fastq SB48_forward.paired.fastq.gz SB48_forward.singletons.fastq.gz SB48_reverse.paired.fastq.gz SB48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB48_1.fastq /home/working/FinalCorrected/demultiplexed/SB48_2.fastq SB48_forward.paired.fastq.gz SB48_forward.singletons.fastq.gz SB48_reverse.paired.fastq.gz SB48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:16:48 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB48/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 38763 Both Surviving: 7572 (19.53%) Forward Only Surviving: 1864 (4.81%) Reverse Only Surviving: 658 (1.70%) Dropped: 28669 (73.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB48_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB48_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB48_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB48_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB48_forward.paired.headclipped.fastq.gz SB48_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB48 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB48_forward.paired.headclipped.fastq.gz
[FLASH]     SB48_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB48.extendedFrags.fastq.gz
[FLASH]     ./SB48.notCombined_1.fastq.gz
[FLASH]     ./SB48.notCombined_2.fastq.gz
[FLASH]     ./SB48.hist
[FLASH]     ./SB48.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7572 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7572
[FLASH]     Combined pairs:   837
[FLASH]     Uncombined pairs: 6735
[FLASH]     Percent combined: 11.05%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.048 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

837 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB48/SB48_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB48_centroids.fasta --uc SB48.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 837 sequences and identified 474 clusters (clustering threshold 1.00) - 20 clusters (representing 329 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB51 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB51_1.fastq /home/working/FinalCorrected/demultiplexed/SB51_2.fastq SB51_forward.paired.fastq.gz SB51_forward.singletons.fastq.gz SB51_reverse.paired.fastq.gz SB51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB51_1.fastq /home/working/FinalCorrected/demultiplexed/SB51_2.fastq SB51_forward.paired.fastq.gz SB51_forward.singletons.fastq.gz SB51_reverse.paired.fastq.gz SB51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:16:53 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB51/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 2 Both Surviving: 2 (100.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 0 (0.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SB51_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB51_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB51_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB51_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB51_forward.paired.headclipped.fastq.gz SB51_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB51 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB51_forward.paired.headclipped.fastq.gz
[FLASH]     SB51_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB51.extendedFrags.fastq.gz
[FLASH]     ./SB51.notCombined_1.fastq.gz
[FLASH]     ./SB51.notCombined_2.fastq.gz
[FLASH]     ./SB51.hist
[FLASH]     ./SB51.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2
[FLASH]     Combined pairs:   1
[FLASH]     Uncombined pairs: 1
[FLASH]     Percent combined: 50.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB51/SB51_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB51_centroids.fasta --uc SB51.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1 sequences and identified 1 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB53 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB53_1.fastq /home/working/FinalCorrected/demultiplexed/SB53_2.fastq SB53_forward.paired.fastq.gz SB53_forward.singletons.fastq.gz SB53_reverse.paired.fastq.gz SB53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB53_1.fastq /home/working/FinalCorrected/demultiplexed/SB53_2.fastq SB53_forward.paired.fastq.gz SB53_forward.singletons.fastq.gz SB53_reverse.paired.fastq.gz SB53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:16:55 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB53/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 34274 Both Surviving: 8781 (25.62%) Forward Only Surviving: 2596 (7.57%) Reverse Only Surviving: 500 (1.46%) Dropped: 22397 (65.35%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB53_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB53_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB53_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB53_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB53_forward.paired.headclipped.fastq.gz SB53_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB53 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB53_forward.paired.headclipped.fastq.gz
[FLASH]     SB53_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB53.extendedFrags.fastq.gz
[FLASH]     ./SB53.notCombined_1.fastq.gz
[FLASH]     ./SB53.notCombined_2.fastq.gz
[FLASH]     ./SB53.hist
[FLASH]     ./SB53.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 8781 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      8781
[FLASH]     Combined pairs:   2543
[FLASH]     Uncombined pairs: 6238
[FLASH]     Percent combined: 28.96%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.084 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2543 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB53/SB53_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB53_centroids.fasta --uc SB53.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2543 sequences and identified 1393 clusters (clustering threshold 1.00) - 20 clusters (representing 1010 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB59 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB59_1.fastq /home/working/FinalCorrected/demultiplexed/SB59_2.fastq SB59_forward.paired.fastq.gz SB59_forward.singletons.fastq.gz SB59_reverse.paired.fastq.gz SB59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB59_1.fastq /home/working/FinalCorrected/demultiplexed/SB59_2.fastq SB59_forward.paired.fastq.gz SB59_forward.singletons.fastq.gz SB59_reverse.paired.fastq.gz SB59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:17:05 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB59/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37151 Both Surviving: 7895 (21.25%) Forward Only Surviving: 2714 (7.31%) Reverse Only Surviving: 494 (1.33%) Dropped: 26048 (70.11%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB59_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB59_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB59_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB59_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB59_forward.paired.headclipped.fastq.gz SB59_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB59 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB59_forward.paired.headclipped.fastq.gz
[FLASH]     SB59_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB59.extendedFrags.fastq.gz
[FLASH]     ./SB59.notCombined_1.fastq.gz
[FLASH]     ./SB59.notCombined_2.fastq.gz
[FLASH]     ./SB59.hist
[FLASH]     ./SB59.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7895 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7895
[FLASH]     Combined pairs:   2360
[FLASH]     Uncombined pairs: 5535
[FLASH]     Percent combined: 29.89%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.069 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2360 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB59/SB59_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB59_centroids.fasta --uc SB59.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2360 sequences and identified 1147 clusters (clustering threshold 1.00) - 32 clusters (representing 1081 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB60 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB60_1.fastq /home/working/FinalCorrected/demultiplexed/SB60_2.fastq SB60_forward.paired.fastq.gz SB60_forward.singletons.fastq.gz SB60_reverse.paired.fastq.gz SB60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB60_1.fastq /home/working/FinalCorrected/demultiplexed/SB60_2.fastq SB60_forward.paired.fastq.gz SB60_forward.singletons.fastq.gz SB60_reverse.paired.fastq.gz SB60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:17:13 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB60/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26851 Both Surviving: 9687 (36.08%) Forward Only Surviving: 2603 (9.69%) Reverse Only Surviving: 322 (1.20%) Dropped: 14239 (53.03%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB60_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB60_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SB60_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SB60_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB60_forward.paired.headclipped.fastq.gz SB60_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB60 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB60_forward.paired.headclipped.fastq.gz
[FLASH]     SB60_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB60.extendedFrags.fastq.gz
[FLASH]     ./SB60.notCombined_1.fastq.gz
[FLASH]     ./SB60.notCombined_2.fastq.gz
[FLASH]     ./SB60.hist
[FLASH]     ./SB60.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9687 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9687
[FLASH]     Combined pairs:   4979
[FLASH]     Uncombined pairs: 4708
[FLASH]     Percent combined: 51.40%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.139 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4979 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB60/SB60_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB60_centroids.fasta --uc SB60.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4979 sequences and identified 2895 clusters (clustering threshold 1.00) - 123 clusters (representing 1906 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB63 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB63_1.fastq /home/working/FinalCorrected/demultiplexed/SB63_2.fastq SB63_forward.paired.fastq.gz SB63_forward.singletons.fastq.gz SB63_reverse.paired.fastq.gz SB63_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB63_1.fastq /home/working/FinalCorrected/demultiplexed/SB63_2.fastq SB63_forward.paired.fastq.gz SB63_forward.singletons.fastq.gz SB63_reverse.paired.fastq.gz SB63_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:17:23 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB63/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 15119 Both Surviving: 3736 (24.71%) Forward Only Surviving: 845 (5.59%) Reverse Only Surviving: 227 (1.50%) Dropped: 10311 (68.20%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB63_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB63_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SB63_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SB63_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB63_forward.paired.headclipped.fastq.gz SB63_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB63 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB63_forward.paired.headclipped.fastq.gz
[FLASH]     SB63_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB63.extendedFrags.fastq.gz
[FLASH]     ./SB63.notCombined_1.fastq.gz
[FLASH]     ./SB63.notCombined_2.fastq.gz
[FLASH]     ./SB63.hist
[FLASH]     ./SB63.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3736 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3736
[FLASH]     Combined pairs:   1303
[FLASH]     Uncombined pairs: 2433
[FLASH]     Percent combined: 34.88%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.039 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1303 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB63/SB63_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB63_centroids.fasta --uc SB63.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1303 sequences and identified 675 clusters (clustering threshold 1.00) - 25 clusters (representing 552 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB64 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB64_1.fastq /home/working/FinalCorrected/demultiplexed/SB64_2.fastq SB64_forward.paired.fastq.gz SB64_forward.singletons.fastq.gz SB64_reverse.paired.fastq.gz SB64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB64_1.fastq /home/working/FinalCorrected/demultiplexed/SB64_2.fastq SB64_forward.paired.fastq.gz SB64_forward.singletons.fastq.gz SB64_reverse.paired.fastq.gz SB64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:17:28 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB64/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20694 Both Surviving: 1095 (5.29%) Forward Only Surviving: 979 (4.73%) Reverse Only Surviving: 253 (1.22%) Dropped: 18367 (88.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB64_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB64_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB64_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB64_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB64_forward.paired.headclipped.fastq.gz SB64_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB64 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB64_forward.paired.headclipped.fastq.gz
[FLASH]     SB64_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB64.extendedFrags.fastq.gz
[FLASH]     ./SB64.notCombined_1.fastq.gz
[FLASH]     ./SB64.notCombined_2.fastq.gz
[FLASH]     ./SB64.hist
[FLASH]     ./SB64.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1095 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1095
[FLASH]     Combined pairs:   180
[FLASH]     Uncombined pairs: 915
[FLASH]     Percent combined: 16.44%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.009 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

180 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB64/SB64_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB64_centroids.fasta --uc SB64.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 180 sequences and identified 149 clusters (clustering threshold 1.00) - 2 clusters (representing 12 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB67 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB67_1.fastq /home/working/FinalCorrected/demultiplexed/SB67_2.fastq SB67_forward.paired.fastq.gz SB67_forward.singletons.fastq.gz SB67_reverse.paired.fastq.gz SB67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB67_1.fastq /home/working/FinalCorrected/demultiplexed/SB67_2.fastq SB67_forward.paired.fastq.gz SB67_forward.singletons.fastq.gz SB67_reverse.paired.fastq.gz SB67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:17:31 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB67/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25560 Both Surviving: 302 (1.18%) Forward Only Surviving: 566 (2.21%) Reverse Only Surviving: 267 (1.04%) Dropped: 24425 (95.56%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SB67_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SB67_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB67_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB67_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB67_forward.paired.headclipped.fastq.gz SB67_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB67 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB67_forward.paired.headclipped.fastq.gz
[FLASH]     SB67_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB67.extendedFrags.fastq.gz
[FLASH]     ./SB67.notCombined_1.fastq.gz
[FLASH]     ./SB67.notCombined_2.fastq.gz
[FLASH]     ./SB67.hist
[FLASH]     ./SB67.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 302 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      302
[FLASH]     Combined pairs:   211
[FLASH]     Uncombined pairs: 91
[FLASH]     Percent combined: 69.87%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.014 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

211 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB67/SB67_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB67_centroids.fasta --uc SB67.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 211 sequences and identified 136 clusters (clustering threshold 1.00) - 4 clusters (representing 77 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB71 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB71_1.fastq /home/working/FinalCorrected/demultiplexed/SB71_2.fastq SB71_forward.paired.fastq.gz SB71_forward.singletons.fastq.gz SB71_reverse.paired.fastq.gz SB71_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB71_1.fastq /home/working/FinalCorrected/demultiplexed/SB71_2.fastq SB71_forward.paired.fastq.gz SB71_forward.singletons.fastq.gz SB71_reverse.paired.fastq.gz SB71_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:17:34 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB71/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29730 Both Surviving: 5425 (18.25%) Forward Only Surviving: 1311 (4.41%) Reverse Only Surviving: 389 (1.31%) Dropped: 22605 (76.03%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB71_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB71_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB71_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB71_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB71_forward.paired.headclipped.fastq.gz SB71_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB71 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB71_forward.paired.headclipped.fastq.gz
[FLASH]     SB71_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB71.extendedFrags.fastq.gz
[FLASH]     ./SB71.notCombined_1.fastq.gz
[FLASH]     ./SB71.notCombined_2.fastq.gz
[FLASH]     ./SB71.hist
[FLASH]     ./SB71.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5425 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5425
[FLASH]     Combined pairs:   5019
[FLASH]     Uncombined pairs: 406
[FLASH]     Percent combined: 92.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.128 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

5019 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB71/SB71_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB71_centroids.fasta --uc SB71.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 5019 sequences and identified 2229 clusters (clustering threshold 1.00) - 89 clusters (representing 2479 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB73 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB73_1.fastq /home/working/FinalCorrected/demultiplexed/SB73_2.fastq SB73_forward.paired.fastq.gz SB73_forward.singletons.fastq.gz SB73_reverse.paired.fastq.gz SB73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB73_1.fastq /home/working/FinalCorrected/demultiplexed/SB73_2.fastq SB73_forward.paired.fastq.gz SB73_forward.singletons.fastq.gz SB73_reverse.paired.fastq.gz SB73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:17:44 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB73/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 30141 Both Surviving: 1194 (3.96%) Forward Only Surviving: 905 (3.00%) Reverse Only Surviving: 309 (1.03%) Dropped: 27733 (92.01%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB73_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB73_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB73_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB73_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB73_forward.paired.headclipped.fastq.gz SB73_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB73 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB73_forward.paired.headclipped.fastq.gz
[FLASH]     SB73_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB73.extendedFrags.fastq.gz
[FLASH]     ./SB73.notCombined_1.fastq.gz
[FLASH]     ./SB73.notCombined_2.fastq.gz
[FLASH]     ./SB73.hist
[FLASH]     ./SB73.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1194 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1194
[FLASH]     Combined pairs:   944
[FLASH]     Uncombined pairs: 250
[FLASH]     Percent combined: 79.06%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.031 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

944 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB73/SB73_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB73_centroids.fasta --uc SB73.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 944 sequences and identified 464 clusters (clustering threshold 1.00) - 8 clusters (representing 438 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB76 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB76_1.fastq /home/working/FinalCorrected/demultiplexed/SB76_2.fastq SB76_forward.paired.fastq.gz SB76_forward.singletons.fastq.gz SB76_reverse.paired.fastq.gz SB76_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB76_1.fastq /home/working/FinalCorrected/demultiplexed/SB76_2.fastq SB76_forward.paired.fastq.gz SB76_forward.singletons.fastq.gz SB76_reverse.paired.fastq.gz SB76_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:17:55 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB76/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 118828 Both Surviving: 47500 (39.97%) Forward Only Surviving: 12180 (10.25%) Reverse Only Surviving: 2009 (1.69%) Dropped: 57139 (48.09%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB76_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB76_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB76_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB76_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB76_forward.paired.headclipped.fastq.gz SB76_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB76 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB76_forward.paired.headclipped.fastq.gz
[FLASH]     SB76_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB76.extendedFrags.fastq.gz
[FLASH]     ./SB76.notCombined_1.fastq.gz
[FLASH]     ./SB76.notCombined_2.fastq.gz
[FLASH]     ./SB76.hist
[FLASH]     ./SB76.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 25000 read pairs
[FLASH] Processed 47500 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      47500
[FLASH]     Combined pairs:   24052
[FLASH]     Uncombined pairs: 23448
[FLASH]     Percent combined: 50.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.662 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

24052 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB76/SB76_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB76_centroids.fasta --uc SB76.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 24052 sequences and identified 9072 clusters (clustering threshold 1.00) - 456 clusters (representing 14503 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB77 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB77_1.fastq /home/working/FinalCorrected/demultiplexed/SB77_2.fastq SB77_forward.paired.fastq.gz SB77_forward.singletons.fastq.gz SB77_reverse.paired.fastq.gz SB77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB77_1.fastq /home/working/FinalCorrected/demultiplexed/SB77_2.fastq SB77_forward.paired.fastq.gz SB77_forward.singletons.fastq.gz SB77_reverse.paired.fastq.gz SB77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:18:45 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB77/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 24892 Both Surviving: 4667 (18.75%) Forward Only Surviving: 1168 (4.69%) Reverse Only Surviving: 427 (1.72%) Dropped: 18630 (74.84%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB77_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB77_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB77_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB77_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB77_forward.paired.headclipped.fastq.gz SB77_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB77 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB77_forward.paired.headclipped.fastq.gz
[FLASH]     SB77_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB77.extendedFrags.fastq.gz
[FLASH]     ./SB77.notCombined_1.fastq.gz
[FLASH]     ./SB77.notCombined_2.fastq.gz
[FLASH]     ./SB77.hist
[FLASH]     ./SB77.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4667 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4667
[FLASH]     Combined pairs:   53
[FLASH]     Uncombined pairs: 4614
[FLASH]     Percent combined: 1.14%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.030 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

53 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB77/SB77_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB77_centroids.fasta --uc SB77.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 53 sequences and identified 48 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB8 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB8_1.fastq /home/working/FinalCorrected/demultiplexed/SB8_2.fastq SB8_forward.paired.fastq.gz SB8_forward.singletons.fastq.gz SB8_reverse.paired.fastq.gz SB8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB8_1.fastq /home/working/FinalCorrected/demultiplexed/SB8_2.fastq SB8_forward.paired.fastq.gz SB8_forward.singletons.fastq.gz SB8_reverse.paired.fastq.gz SB8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:18:49 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB8/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 26459 Both Surviving: 2673 (10.10%) Forward Only Surviving: 944 (3.57%) Reverse Only Surviving: 335 (1.27%) Dropped: 22507 (85.06%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB8_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB8_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SB8_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB8_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB8_forward.paired.headclipped.fastq.gz SB8_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB8 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB8_forward.paired.headclipped.fastq.gz
[FLASH]     SB8_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB8.extendedFrags.fastq.gz
[FLASH]     ./SB8.notCombined_1.fastq.gz
[FLASH]     ./SB8.notCombined_2.fastq.gz
[FLASH]     ./SB8.hist
[FLASH]     ./SB8.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2673 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2673
[FLASH]     Combined pairs:   2308
[FLASH]     Uncombined pairs: 365
[FLASH]     Percent combined: 86.34%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.069 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2308 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB8/SB8_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB8_centroids.fasta --uc SB8.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2308 sequences and identified 1361 clusters (clustering threshold 1.00) - 24 clusters (representing 806 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB80 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB80_1.fastq /home/working/FinalCorrected/demultiplexed/SB80_2.fastq SB80_forward.paired.fastq.gz SB80_forward.singletons.fastq.gz SB80_reverse.paired.fastq.gz SB80_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB80_1.fastq /home/working/FinalCorrected/demultiplexed/SB80_2.fastq SB80_forward.paired.fastq.gz SB80_forward.singletons.fastq.gz SB80_reverse.paired.fastq.gz SB80_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:18:56 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB80/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 34212 Both Surviving: 398 (1.16%) Forward Only Surviving: 465 (1.36%) Reverse Only Surviving: 237 (0.69%) Dropped: 33112 (96.78%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB80_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB80_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB80_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB80_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB80_forward.paired.headclipped.fastq.gz SB80_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB80 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB80_forward.paired.headclipped.fastq.gz
[FLASH]     SB80_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB80.extendedFrags.fastq.gz
[FLASH]     ./SB80.notCombined_1.fastq.gz
[FLASH]     ./SB80.notCombined_2.fastq.gz
[FLASH]     ./SB80.hist
[FLASH]     ./SB80.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 398 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      398
[FLASH]     Combined pairs:   15
[FLASH]     Uncombined pairs: 383
[FLASH]     Percent combined: 3.77%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.005 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

15 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB80/SB80_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB80_centroids.fasta --uc SB80.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 15 sequences and identified 14 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB82 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB82_1.fastq /home/working/FinalCorrected/demultiplexed/SB82_2.fastq SB82_forward.paired.fastq.gz SB82_forward.singletons.fastq.gz SB82_reverse.paired.fastq.gz SB82_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB82_1.fastq /home/working/FinalCorrected/demultiplexed/SB82_2.fastq SB82_forward.paired.fastq.gz SB82_forward.singletons.fastq.gz SB82_reverse.paired.fastq.gz SB82_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:19:01 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB82/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 56602 Both Surviving: 19825 (35.03%) Forward Only Surviving: 3783 (6.68%) Reverse Only Surviving: 937 (1.66%) Dropped: 32057 (56.64%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB82_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB82_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SB82_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SB82_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB82_forward.paired.headclipped.fastq.gz SB82_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB82 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB82_forward.paired.headclipped.fastq.gz
[FLASH]     SB82_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB82.extendedFrags.fastq.gz
[FLASH]     ./SB82.notCombined_1.fastq.gz
[FLASH]     ./SB82.notCombined_2.fastq.gz
[FLASH]     ./SB82.hist
[FLASH]     ./SB82.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 19825 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      19825
[FLASH]     Combined pairs:   10734
[FLASH]     Uncombined pairs: 9091
[FLASH]     Percent combined: 54.14%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.295 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

10734 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB82/SB82_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB82_centroids.fasta --uc SB82.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 10734 sequences and identified 5324 clusters (clustering threshold 1.00) - 215 clusters (representing 4986 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB83 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB83_1.fastq /home/working/FinalCorrected/demultiplexed/SB83_2.fastq SB83_forward.paired.fastq.gz SB83_forward.singletons.fastq.gz SB83_reverse.paired.fastq.gz SB83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB83_1.fastq /home/working/FinalCorrected/demultiplexed/SB83_2.fastq SB83_forward.paired.fastq.gz SB83_forward.singletons.fastq.gz SB83_reverse.paired.fastq.gz SB83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:19:23 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB83/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 13296 Both Surviving: 2639 (19.85%) Forward Only Surviving: 502 (3.78%) Reverse Only Surviving: 223 (1.68%) Dropped: 9932 (74.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SB83_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SB83_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB83_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB83_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB83_forward.paired.headclipped.fastq.gz SB83_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB83 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB83_forward.paired.headclipped.fastq.gz
[FLASH]     SB83_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB83.extendedFrags.fastq.gz
[FLASH]     ./SB83.notCombined_1.fastq.gz
[FLASH]     ./SB83.notCombined_2.fastq.gz
[FLASH]     ./SB83.hist
[FLASH]     ./SB83.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2639 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2639
[FLASH]     Combined pairs:   195
[FLASH]     Uncombined pairs: 2444
[FLASH]     Percent combined: 7.39%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.017 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

195 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB83/SB83_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB83_centroids.fasta --uc SB83.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 195 sequences and identified 88 clusters (clustering threshold 1.00) - 2 clusters (representing 98 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB85 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB85_1.fastq /home/working/FinalCorrected/demultiplexed/SB85_2.fastq SB85_forward.paired.fastq.gz SB85_forward.singletons.fastq.gz SB85_reverse.paired.fastq.gz SB85_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB85_1.fastq /home/working/FinalCorrected/demultiplexed/SB85_2.fastq SB85_forward.paired.fastq.gz SB85_forward.singletons.fastq.gz SB85_reverse.paired.fastq.gz SB85_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:19:26 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB85/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28986 Both Surviving: 5672 (19.57%) Forward Only Surviving: 1435 (4.95%) Reverse Only Surviving: 464 (1.60%) Dropped: 21415 (73.88%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SB85_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB85_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB85_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB85_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB85_forward.paired.headclipped.fastq.gz SB85_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB85 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB85_forward.paired.headclipped.fastq.gz
[FLASH]     SB85_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB85.extendedFrags.fastq.gz
[FLASH]     ./SB85.notCombined_1.fastq.gz
[FLASH]     ./SB85.notCombined_2.fastq.gz
[FLASH]     ./SB85.hist
[FLASH]     ./SB85.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5672 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5672
[FLASH]     Combined pairs:   702
[FLASH]     Uncombined pairs: 4970
[FLASH]     Percent combined: 12.38%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.035 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

702 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB85/SB85_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB85_centroids.fasta --uc SB85.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 702 sequences and identified 426 clusters (clustering threshold 1.00) - 13 clusters (representing 266 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB86 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB86_1.fastq /home/working/FinalCorrected/demultiplexed/SB86_2.fastq SB86_forward.paired.fastq.gz SB86_forward.singletons.fastq.gz SB86_reverse.paired.fastq.gz SB86_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB86_1.fastq /home/working/FinalCorrected/demultiplexed/SB86_2.fastq SB86_forward.paired.fastq.gz SB86_forward.singletons.fastq.gz SB86_reverse.paired.fastq.gz SB86_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:19:31 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB86/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 10822 Both Surviving: 3987 (36.84%) Forward Only Surviving: 1134 (10.48%) Reverse Only Surviving: 161 (1.49%) Dropped: 5540 (51.19%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB86_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB86_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB86_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB86_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB86_forward.paired.headclipped.fastq.gz SB86_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB86 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB86_forward.paired.headclipped.fastq.gz
[FLASH]     SB86_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB86.extendedFrags.fastq.gz
[FLASH]     ./SB86.notCombined_1.fastq.gz
[FLASH]     ./SB86.notCombined_2.fastq.gz
[FLASH]     ./SB86.hist
[FLASH]     ./SB86.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3987 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3987
[FLASH]     Combined pairs:   1868
[FLASH]     Uncombined pairs: 2119
[FLASH]     Percent combined: 46.85%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.057 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1868 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB86/SB86_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB86_centroids.fasta --uc SB86.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1868 sequences and identified 1174 clusters (clustering threshold 1.00) - 27 clusters (representing 600 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB87 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB87_1.fastq /home/working/FinalCorrected/demultiplexed/SB87_2.fastq SB87_forward.paired.fastq.gz SB87_forward.singletons.fastq.gz SB87_reverse.paired.fastq.gz SB87_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB87_1.fastq /home/working/FinalCorrected/demultiplexed/SB87_2.fastq SB87_forward.paired.fastq.gz SB87_forward.singletons.fastq.gz SB87_reverse.paired.fastq.gz SB87_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:19:35 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB87/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SB87_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB87_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SB87_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB87_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB87_forward.paired.headclipped.fastq.gz SB87_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB87 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB87_forward.paired.headclipped.fastq.gz
[FLASH]     SB87_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB87.extendedFrags.fastq.gz
[FLASH]     ./SB87.notCombined_1.fastq.gz
[FLASH]     ./SB87.notCombined_2.fastq.gz
[FLASH]     ./SB87.hist
[FLASH]     ./SB87.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB87/SB87_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB87_centroids.fasta --uc SB87.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB91 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB91_1.fastq /home/working/FinalCorrected/demultiplexed/SB91_2.fastq SB91_forward.paired.fastq.gz SB91_forward.singletons.fastq.gz SB91_reverse.paired.fastq.gz SB91_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB91_1.fastq /home/working/FinalCorrected/demultiplexed/SB91_2.fastq SB91_forward.paired.fastq.gz SB91_forward.singletons.fastq.gz SB91_reverse.paired.fastq.gz SB91_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:19:37 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB91/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 35326 Both Surviving: 2322 (6.57%) Forward Only Surviving: 1198 (3.39%) Reverse Only Surviving: 322 (0.91%) Dropped: 31484 (89.12%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SB91_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SB91_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SB91_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SB91_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB91_forward.paired.headclipped.fastq.gz SB91_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB91 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB91_forward.paired.headclipped.fastq.gz
[FLASH]     SB91_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB91.extendedFrags.fastq.gz
[FLASH]     ./SB91.notCombined_1.fastq.gz
[FLASH]     ./SB91.notCombined_2.fastq.gz
[FLASH]     ./SB91.hist
[FLASH]     ./SB91.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2322 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2322
[FLASH]     Combined pairs:   2028
[FLASH]     Uncombined pairs: 294
[FLASH]     Percent combined: 87.34%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.055 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2028 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB91/SB91_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB91_centroids.fasta --uc SB91.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2028 sequences and identified 1108 clusters (clustering threshold 1.00) - 24 clusters (representing 820 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB93 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB93_1.fastq /home/working/FinalCorrected/demultiplexed/SB93_2.fastq SB93_forward.paired.fastq.gz SB93_forward.singletons.fastq.gz SB93_reverse.paired.fastq.gz SB93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB93_1.fastq /home/working/FinalCorrected/demultiplexed/SB93_2.fastq SB93_forward.paired.fastq.gz SB93_forward.singletons.fastq.gz SB93_reverse.paired.fastq.gz SB93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:19:43 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB93/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20814 Both Surviving: 2455 (11.79%) Forward Only Surviving: 977 (4.69%) Reverse Only Surviving: 388 (1.86%) Dropped: 16994 (81.65%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SB93_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SB93_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SB93_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SB93_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB93_forward.paired.headclipped.fastq.gz SB93_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB93 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB93_forward.paired.headclipped.fastq.gz
[FLASH]     SB93_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB93.extendedFrags.fastq.gz
[FLASH]     ./SB93.notCombined_1.fastq.gz
[FLASH]     ./SB93.notCombined_2.fastq.gz
[FLASH]     ./SB93.hist
[FLASH]     ./SB93.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2455 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2455
[FLASH]     Combined pairs:   365
[FLASH]     Uncombined pairs: 2090
[FLASH]     Percent combined: 14.87%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.016 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

365 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB93/SB93_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB93_centroids.fasta --uc SB93.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 365 sequences and identified 188 clusters (clustering threshold 1.00) - 4 clusters (representing 165 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SB99 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB99_1.fastq /home/working/FinalCorrected/demultiplexed/SB99_2.fastq SB99_forward.paired.fastq.gz SB99_forward.singletons.fastq.gz SB99_reverse.paired.fastq.gz SB99_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SB99_1.fastq /home/working/FinalCorrected/demultiplexed/SB99_2.fastq SB99_forward.paired.fastq.gz SB99_forward.singletons.fastq.gz SB99_reverse.paired.fastq.gz SB99_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:19:47 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SB99/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22690 Both Surviving: 2824 (12.45%) Forward Only Surviving: 846 (3.73%) Reverse Only Surviving: 417 (1.84%) Dropped: 18603 (81.99%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SB99_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SB99_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SB99_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SB99_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SB99_forward.paired.headclipped.fastq.gz SB99_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB99 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SB99_forward.paired.headclipped.fastq.gz
[FLASH]     SB99_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SB99.extendedFrags.fastq.gz
[FLASH]     ./SB99.notCombined_1.fastq.gz
[FLASH]     ./SB99.notCombined_2.fastq.gz
[FLASH]     ./SB99.hist
[FLASH]     ./SB99.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2824 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2824
[FLASH]     Combined pairs:   746
[FLASH]     Uncombined pairs: 2078
[FLASH]     Percent combined: 26.42%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.032 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

746 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SB99/SB99_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB99_centroids.fasta --uc SB99.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 746 sequences and identified 394 clusters (clustering threshold 1.00) - 8 clusters (representing 317 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC10 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC10_1.fastq /home/working/FinalCorrected/demultiplexed/SC10_2.fastq SC10_forward.paired.fastq.gz SC10_forward.singletons.fastq.gz SC10_reverse.paired.fastq.gz SC10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC10_1.fastq /home/working/FinalCorrected/demultiplexed/SC10_2.fastq SC10_forward.paired.fastq.gz SC10_forward.singletons.fastq.gz SC10_reverse.paired.fastq.gz SC10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:19:52 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC10/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 39166 Both Surviving: 16252 (41.50%) Forward Only Surviving: 3651 (9.32%) Reverse Only Surviving: 596 (1.52%) Dropped: 18667 (47.66%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC10_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC10_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SC10_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC10_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC10_forward.paired.headclipped.fastq.gz SC10_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC10 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC10_forward.paired.headclipped.fastq.gz
[FLASH]     SC10_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC10.extendedFrags.fastq.gz
[FLASH]     ./SC10.notCombined_1.fastq.gz
[FLASH]     ./SC10.notCombined_2.fastq.gz
[FLASH]     ./SC10.hist
[FLASH]     ./SC10.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 16252 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      16252
[FLASH]     Combined pairs:   9440
[FLASH]     Uncombined pairs: 6812
[FLASH]     Percent combined: 58.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.256 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

9440 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC10/SC10_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC10_centroids.fasta --uc SC10.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 9440 sequences and identified 4841 clusters (clustering threshold 1.00) - 261 clusters (representing 4381 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC101 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC101_1.fastq /home/working/FinalCorrected/demultiplexed/SC101_2.fastq SC101_forward.paired.fastq.gz SC101_forward.singletons.fastq.gz SC101_reverse.paired.fastq.gz SC101_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC101_1.fastq /home/working/FinalCorrected/demultiplexed/SC101_2.fastq SC101_forward.paired.fastq.gz SC101_forward.singletons.fastq.gz SC101_reverse.paired.fastq.gz SC101_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:11 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC101/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29995 Both Surviving: 6087 (20.29%) Forward Only Surviving: 1131 (3.77%) Reverse Only Surviving: 494 (1.65%) Dropped: 22283 (74.29%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC101_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC101_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC101_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC101_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC101_forward.paired.headclipped.fastq.gz SC101_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC101 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC101_forward.paired.headclipped.fastq.gz
[FLASH]     SC101_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC101.extendedFrags.fastq.gz
[FLASH]     ./SC101.notCombined_1.fastq.gz
[FLASH]     ./SC101.notCombined_2.fastq.gz
[FLASH]     ./SC101.hist
[FLASH]     ./SC101.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6087 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6087
[FLASH]     Combined pairs:   11
[FLASH]     Uncombined pairs: 6076
[FLASH]     Percent combined: 0.18%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.039 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

11 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC101/SC101_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC101_centroids.fasta --uc SC101.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 11 sequences and identified 11 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC106 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC106_1.fastq /home/working/FinalCorrected/demultiplexed/SC106_2.fastq SC106_forward.paired.fastq.gz SC106_forward.singletons.fastq.gz SC106_reverse.paired.fastq.gz SC106_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC106_1.fastq /home/working/FinalCorrected/demultiplexed/SC106_2.fastq SC106_forward.paired.fastq.gz SC106_forward.singletons.fastq.gz SC106_reverse.paired.fastq.gz SC106_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:14 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC106/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 1 Both Surviving: 0 (0.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 1 (100.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SC106_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC106_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC106_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC106_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC106_forward.paired.headclipped.fastq.gz SC106_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC106 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC106_forward.paired.headclipped.fastq.gz
[FLASH]     SC106_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC106.extendedFrags.fastq.gz
[FLASH]     ./SC106.notCombined_1.fastq.gz
[FLASH]     ./SC106.notCombined_2.fastq.gz
[FLASH]     ./SC106.hist
[FLASH]     ./SC106.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC106/SC106_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC106_centroids.fasta --uc SC106.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC107 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC107_1.fastq /home/working/FinalCorrected/demultiplexed/SC107_2.fastq SC107_forward.paired.fastq.gz SC107_forward.singletons.fastq.gz SC107_reverse.paired.fastq.gz SC107_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC107_1.fastq /home/working/FinalCorrected/demultiplexed/SC107_2.fastq SC107_forward.paired.fastq.gz SC107_forward.singletons.fastq.gz SC107_reverse.paired.fastq.gz SC107_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:15 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC107/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16670 Both Surviving: 3240 (19.44%) Forward Only Surviving: 944 (5.66%) Reverse Only Surviving: 191 (1.15%) Dropped: 12295 (73.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC107_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC107_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC107_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC107_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC107_forward.paired.headclipped.fastq.gz SC107_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC107 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC107_forward.paired.headclipped.fastq.gz
[FLASH]     SC107_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC107.extendedFrags.fastq.gz
[FLASH]     ./SC107.notCombined_1.fastq.gz
[FLASH]     ./SC107.notCombined_2.fastq.gz
[FLASH]     ./SC107.hist
[FLASH]     ./SC107.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3240 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3240
[FLASH]     Combined pairs:   268
[FLASH]     Uncombined pairs: 2972
[FLASH]     Percent combined: 8.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.020 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

268 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC107/SC107_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC107_centroids.fasta --uc SC107.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 268 sequences and identified 195 clusters (clustering threshold 1.00) - 1 clusters (representing 51 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC110 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC110_1.fastq /home/working/FinalCorrected/demultiplexed/SC110_2.fastq SC110_forward.paired.fastq.gz SC110_forward.singletons.fastq.gz SC110_reverse.paired.fastq.gz SC110_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC110_1.fastq /home/working/FinalCorrected/demultiplexed/SC110_2.fastq SC110_forward.paired.fastq.gz SC110_forward.singletons.fastq.gz SC110_reverse.paired.fastq.gz SC110_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:19 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC110/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 19004 Both Surviving: 3316 (17.45%) Forward Only Surviving: 1648 (8.67%) Reverse Only Surviving: 250 (1.32%) Dropped: 13790 (72.56%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC110_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC110_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC110_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC110_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC110_forward.paired.headclipped.fastq.gz SC110_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC110 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC110_forward.paired.headclipped.fastq.gz
[FLASH]     SC110_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC110.extendedFrags.fastq.gz
[FLASH]     ./SC110.notCombined_1.fastq.gz
[FLASH]     ./SC110.notCombined_2.fastq.gz
[FLASH]     ./SC110.hist
[FLASH]     ./SC110.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3316 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3316
[FLASH]     Combined pairs:   2849
[FLASH]     Uncombined pairs: 467
[FLASH]     Percent combined: 85.92%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.096 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2849 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC110/SC110_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC110_centroids.fasta --uc SC110.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2849 sequences and identified 1498 clusters (clustering threshold 1.00) - 27 clusters (representing 1109 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC113 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC113_1.fastq /home/working/FinalCorrected/demultiplexed/SC113_2.fastq SC113_forward.paired.fastq.gz SC113_forward.singletons.fastq.gz SC113_reverse.paired.fastq.gz SC113_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC113_1.fastq /home/working/FinalCorrected/demultiplexed/SC113_2.fastq SC113_forward.paired.fastq.gz SC113_forward.singletons.fastq.gz SC113_reverse.paired.fastq.gz SC113_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:25 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC113/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14610 Both Surviving: 351 (2.40%) Forward Only Surviving: 423 (2.90%) Reverse Only Surviving: 117 (0.80%) Dropped: 13719 (93.90%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC113_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC113_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC113_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC113_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC113_forward.paired.headclipped.fastq.gz SC113_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC113 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC113_forward.paired.headclipped.fastq.gz
[FLASH]     SC113_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC113.extendedFrags.fastq.gz
[FLASH]     ./SC113.notCombined_1.fastq.gz
[FLASH]     ./SC113.notCombined_2.fastq.gz
[FLASH]     ./SC113.hist
[FLASH]     ./SC113.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 351 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      351
[FLASH]     Combined pairs:   263
[FLASH]     Uncombined pairs: 88
[FLASH]     Percent combined: 74.93%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.013 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

263 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC113/SC113_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC113_centroids.fasta --uc SC113.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 263 sequences and identified 176 clusters (clustering threshold 1.00) - 4 clusters (representing 68 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC117 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC117_1.fastq /home/working/FinalCorrected/demultiplexed/SC117_2.fastq SC117_forward.paired.fastq.gz SC117_forward.singletons.fastq.gz SC117_reverse.paired.fastq.gz SC117_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC117_1.fastq /home/working/FinalCorrected/demultiplexed/SC117_2.fastq SC117_forward.paired.fastq.gz SC117_forward.singletons.fastq.gz SC117_reverse.paired.fastq.gz SC117_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:29 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC117/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 34241 Both Surviving: 7288 (21.28%) Forward Only Surviving: 1972 (5.76%) Reverse Only Surviving: 492 (1.44%) Dropped: 24489 (71.52%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC117_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC117_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SC117_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC117_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC117_forward.paired.headclipped.fastq.gz SC117_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC117 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC117_forward.paired.headclipped.fastq.gz
[FLASH]     SC117_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC117.extendedFrags.fastq.gz
[FLASH]     ./SC117.notCombined_1.fastq.gz
[FLASH]     ./SC117.notCombined_2.fastq.gz
[FLASH]     ./SC117.hist
[FLASH]     ./SC117.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 7288 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      7288
[FLASH]     Combined pairs:   423
[FLASH]     Uncombined pairs: 6865
[FLASH]     Percent combined: 5.80%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.046 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

423 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC117/SC117_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC117_centroids.fasta --uc SC117.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 423 sequences and identified 260 clusters (clustering threshold 1.00) - 2 clusters (representing 138 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC119 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC119_1.fastq /home/working/FinalCorrected/demultiplexed/SC119_2.fastq SC119_forward.paired.fastq.gz SC119_forward.singletons.fastq.gz SC119_reverse.paired.fastq.gz SC119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC119_1.fastq /home/working/FinalCorrected/demultiplexed/SC119_2.fastq SC119_forward.paired.fastq.gz SC119_forward.singletons.fastq.gz SC119_reverse.paired.fastq.gz SC119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:34 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC119/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16249 Both Surviving: 4595 (28.28%) Forward Only Surviving: 1155 (7.11%) Reverse Only Surviving: 242 (1.49%) Dropped: 10257 (63.12%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC119_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC119_forward.singletons.fastq.gz
Cropping the first 29 bases off reads in file: SC119_reverse.paired.fastq.gz
Cropping the first 29 bases off reads in file: SC119_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC119_forward.paired.headclipped.fastq.gz SC119_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC119 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC119_forward.paired.headclipped.fastq.gz
[FLASH]     SC119_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC119.extendedFrags.fastq.gz
[FLASH]     ./SC119.notCombined_1.fastq.gz
[FLASH]     ./SC119.notCombined_2.fastq.gz
[FLASH]     ./SC119.hist
[FLASH]     ./SC119.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4595 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4595
[FLASH]     Combined pairs:   827
[FLASH]     Uncombined pairs: 3768
[FLASH]     Percent combined: 18.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.039 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

827 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC119/SC119_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC119_centroids.fasta --uc SC119.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 827 sequences and identified 414 clusters (clustering threshold 1.00) - 7 clusters (representing 376 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC122 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC122_1.fastq /home/working/FinalCorrected/demultiplexed/SC122_2.fastq SC122_forward.paired.fastq.gz SC122_forward.singletons.fastq.gz SC122_reverse.paired.fastq.gz SC122_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC122_1.fastq /home/working/FinalCorrected/demultiplexed/SC122_2.fastq SC122_forward.paired.fastq.gz SC122_forward.singletons.fastq.gz SC122_reverse.paired.fastq.gz SC122_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:39 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC122/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 32336 Both Surviving: 1273 (3.94%) Forward Only Surviving: 724 (2.24%) Reverse Only Surviving: 364 (1.13%) Dropped: 29975 (92.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SC122_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SC122_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SC122_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SC122_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC122_forward.paired.headclipped.fastq.gz SC122_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC122 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC122_forward.paired.headclipped.fastq.gz
[FLASH]     SC122_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC122.extendedFrags.fastq.gz
[FLASH]     ./SC122.notCombined_1.fastq.gz
[FLASH]     ./SC122.notCombined_2.fastq.gz
[FLASH]     ./SC122.hist
[FLASH]     ./SC122.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1273 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1273
[FLASH]     Combined pairs:   1029
[FLASH]     Uncombined pairs: 244
[FLASH]     Percent combined: 80.83%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.037 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1029 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC122/SC122_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC122_centroids.fasta --uc SC122.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1029 sequences and identified 653 clusters (clustering threshold 1.00) - 17 clusters (representing 338 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC124 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC124_1.fastq /home/working/FinalCorrected/demultiplexed/SC124_2.fastq SC124_forward.paired.fastq.gz SC124_forward.singletons.fastq.gz SC124_reverse.paired.fastq.gz SC124_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC124_1.fastq /home/working/FinalCorrected/demultiplexed/SC124_2.fastq SC124_forward.paired.fastq.gz SC124_forward.singletons.fastq.gz SC124_reverse.paired.fastq.gz SC124_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:43 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC124/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16653 Both Surviving: 2962 (17.79%) Forward Only Surviving: 779 (4.68%) Reverse Only Surviving: 233 (1.40%) Dropped: 12679 (76.14%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC124_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC124_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC124_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC124_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC124_forward.paired.headclipped.fastq.gz SC124_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC124 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC124_forward.paired.headclipped.fastq.gz
[FLASH]     SC124_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC124.extendedFrags.fastq.gz
[FLASH]     ./SC124.notCombined_1.fastq.gz
[FLASH]     ./SC124.notCombined_2.fastq.gz
[FLASH]     ./SC124.hist
[FLASH]     ./SC124.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2962 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2962
[FLASH]     Combined pairs:   56
[FLASH]     Uncombined pairs: 2906
[FLASH]     Percent combined: 1.89%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.023 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

56 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC124/SC124_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC124_centroids.fasta --uc SC124.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 56 sequences and identified 54 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC125 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC125_1.fastq /home/working/FinalCorrected/demultiplexed/SC125_2.fastq SC125_forward.paired.fastq.gz SC125_forward.singletons.fastq.gz SC125_reverse.paired.fastq.gz SC125_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC125_1.fastq /home/working/FinalCorrected/demultiplexed/SC125_2.fastq SC125_forward.paired.fastq.gz SC125_forward.singletons.fastq.gz SC125_reverse.paired.fastq.gz SC125_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:46 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC125/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18973 Both Surviving: 2226 (11.73%) Forward Only Surviving: 628 (3.31%) Reverse Only Surviving: 189 (1.00%) Dropped: 15930 (83.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 23 bases off reads in file: SC125_forward.paired.fastq.gz
Cropping the first 23 bases off reads in file: SC125_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC125_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC125_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC125_forward.paired.headclipped.fastq.gz SC125_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC125 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC125_forward.paired.headclipped.fastq.gz
[FLASH]     SC125_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC125.extendedFrags.fastq.gz
[FLASH]     ./SC125.notCombined_1.fastq.gz
[FLASH]     ./SC125.notCombined_2.fastq.gz
[FLASH]     ./SC125.hist
[FLASH]     ./SC125.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2226 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2226
[FLASH]     Combined pairs:   37
[FLASH]     Uncombined pairs: 2189
[FLASH]     Percent combined: 1.66%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.015 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

37 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC125/SC125_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC125_centroids.fasta --uc SC125.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 37 sequences and identified 30 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC141 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC141_1.fastq /home/working/FinalCorrected/demultiplexed/SC141_2.fastq SC141_forward.paired.fastq.gz SC141_forward.singletons.fastq.gz SC141_reverse.paired.fastq.gz SC141_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC141_1.fastq /home/working/FinalCorrected/demultiplexed/SC141_2.fastq SC141_forward.paired.fastq.gz SC141_forward.singletons.fastq.gz SC141_reverse.paired.fastq.gz SC141_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:49 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC141/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 12531 Both Surviving: 2206 (17.60%) Forward Only Surviving: 549 (4.38%) Reverse Only Surviving: 337 (2.69%) Dropped: 9439 (75.33%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 27 bases off reads in file: SC141_forward.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC141_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC141_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC141_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC141_forward.paired.headclipped.fastq.gz SC141_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC141 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC141_forward.paired.headclipped.fastq.gz
[FLASH]     SC141_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC141.extendedFrags.fastq.gz
[FLASH]     ./SC141.notCombined_1.fastq.gz
[FLASH]     ./SC141.notCombined_2.fastq.gz
[FLASH]     ./SC141.hist
[FLASH]     ./SC141.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2206 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2206
[FLASH]     Combined pairs:   57
[FLASH]     Uncombined pairs: 2149
[FLASH]     Percent combined: 2.58%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.018 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

57 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC141/SC141_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC141_centroids.fasta --uc SC141.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 57 sequences and identified 45 clusters (clustering threshold 1.00) - 1 clusters (representing 6 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC142 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC142_1.fastq /home/working/FinalCorrected/demultiplexed/SC142_2.fastq SC142_forward.paired.fastq.gz SC142_forward.singletons.fastq.gz SC142_reverse.paired.fastq.gz SC142_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC142_1.fastq /home/working/FinalCorrected/demultiplexed/SC142_2.fastq SC142_forward.paired.fastq.gz SC142_forward.singletons.fastq.gz SC142_reverse.paired.fastq.gz SC142_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:52 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC142/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17791 Both Surviving: 3480 (19.56%) Forward Only Surviving: 964 (5.42%) Reverse Only Surviving: 223 (1.25%) Dropped: 13124 (73.77%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC142_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC142_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC142_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC142_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC142_forward.paired.headclipped.fastq.gz SC142_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC142 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC142_forward.paired.headclipped.fastq.gz
[FLASH]     SC142_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC142.extendedFrags.fastq.gz
[FLASH]     ./SC142.notCombined_1.fastq.gz
[FLASH]     ./SC142.notCombined_2.fastq.gz
[FLASH]     ./SC142.hist
[FLASH]     ./SC142.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3480 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3480
[FLASH]     Combined pairs:   55
[FLASH]     Uncombined pairs: 3425
[FLASH]     Percent combined: 1.58%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.023 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

55 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC142/SC142_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC142_centroids.fasta --uc SC142.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 55 sequences and identified 50 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC143 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC143_1.fastq /home/working/FinalCorrected/demultiplexed/SC143_2.fastq SC143_forward.paired.fastq.gz SC143_forward.singletons.fastq.gz SC143_reverse.paired.fastq.gz SC143_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC143_1.fastq /home/working/FinalCorrected/demultiplexed/SC143_2.fastq SC143_forward.paired.fastq.gz SC143_forward.singletons.fastq.gz SC143_reverse.paired.fastq.gz SC143_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:55 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC143/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 27590 Both Surviving: 2582 (9.36%) Forward Only Surviving: 805 (2.92%) Reverse Only Surviving: 426 (1.54%) Dropped: 23777 (86.18%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC143_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC143_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SC143_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC143_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC143_forward.paired.headclipped.fastq.gz SC143_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC143 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC143_forward.paired.headclipped.fastq.gz
[FLASH]     SC143_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC143.extendedFrags.fastq.gz
[FLASH]     ./SC143.notCombined_1.fastq.gz
[FLASH]     ./SC143.notCombined_2.fastq.gz
[FLASH]     ./SC143.hist
[FLASH]     ./SC143.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2582 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2582
[FLASH]     Combined pairs:   122
[FLASH]     Uncombined pairs: 2460
[FLASH]     Percent combined: 4.73%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.017 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

122 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC143/SC143_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC143_centroids.fasta --uc SC143.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 122 sequences and identified 91 clusters (clustering threshold 1.00) - 2 clusters (representing 20 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC157 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC157_1.fastq /home/working/FinalCorrected/demultiplexed/SC157_2.fastq SC157_forward.paired.fastq.gz SC157_forward.singletons.fastq.gz SC157_reverse.paired.fastq.gz SC157_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC157_1.fastq /home/working/FinalCorrected/demultiplexed/SC157_2.fastq SC157_forward.paired.fastq.gz SC157_forward.singletons.fastq.gz SC157_reverse.paired.fastq.gz SC157_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:20:58 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC157/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 4782 Both Surviving: 123 (2.57%) Forward Only Surviving: 143 (2.99%) Reverse Only Surviving: 56 (1.17%) Dropped: 4460 (93.27%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC157_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC157_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC157_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC157_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC157_forward.paired.headclipped.fastq.gz SC157_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC157 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC157_forward.paired.headclipped.fastq.gz
[FLASH]     SC157_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC157.extendedFrags.fastq.gz
[FLASH]     ./SC157.notCombined_1.fastq.gz
[FLASH]     ./SC157.notCombined_2.fastq.gz
[FLASH]     ./SC157.hist
[FLASH]     ./SC157.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 123 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      123
[FLASH]     Combined pairs:   34
[FLASH]     Uncombined pairs: 89
[FLASH]     Percent combined: 27.64%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

34 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC157/SC157_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC157_centroids.fasta --uc SC157.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 34 sequences and identified 30 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC16 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC16_1.fastq /home/working/FinalCorrected/demultiplexed/SC16_2.fastq SC16_forward.paired.fastq.gz SC16_forward.singletons.fastq.gz SC16_reverse.paired.fastq.gz SC16_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC16_1.fastq /home/working/FinalCorrected/demultiplexed/SC16_2.fastq SC16_forward.paired.fastq.gz SC16_forward.singletons.fastq.gz SC16_reverse.paired.fastq.gz SC16_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:21:00 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC16/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 33389 Both Surviving: 9086 (27.21%) Forward Only Surviving: 3296 (9.87%) Reverse Only Surviving: 356 (1.07%) Dropped: 20651 (61.85%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC16_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC16_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC16_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC16_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC16_forward.paired.headclipped.fastq.gz SC16_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC16 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC16_forward.paired.headclipped.fastq.gz
[FLASH]     SC16_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC16.extendedFrags.fastq.gz
[FLASH]     ./SC16.notCombined_1.fastq.gz
[FLASH]     ./SC16.notCombined_2.fastq.gz
[FLASH]     ./SC16.hist
[FLASH]     ./SC16.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 9086 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      9086
[FLASH]     Combined pairs:   8024
[FLASH]     Uncombined pairs: 1062
[FLASH]     Percent combined: 88.31%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.215 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

8024 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC16/SC16_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC16_centroids.fasta --uc SC16.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 8024 sequences and identified 3299 clusters (clustering threshold 1.00) - 194 clusters (representing 4308 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC168 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC168_1.fastq /home/working/FinalCorrected/demultiplexed/SC168_2.fastq SC168_forward.paired.fastq.gz SC168_forward.singletons.fastq.gz SC168_reverse.paired.fastq.gz SC168_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC168_1.fastq /home/working/FinalCorrected/demultiplexed/SC168_2.fastq SC168_forward.paired.fastq.gz SC168_forward.singletons.fastq.gz SC168_reverse.paired.fastq.gz SC168_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:21:18 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC168/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37551 Both Surviving: 5209 (13.87%) Forward Only Surviving: 1233 (3.28%) Reverse Only Surviving: 602 (1.60%) Dropped: 30507 (81.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC168_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC168_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC168_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC168_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC168_forward.paired.headclipped.fastq.gz SC168_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC168 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC168_forward.paired.headclipped.fastq.gz
[FLASH]     SC168_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC168.extendedFrags.fastq.gz
[FLASH]     ./SC168.notCombined_1.fastq.gz
[FLASH]     ./SC168.notCombined_2.fastq.gz
[FLASH]     ./SC168.hist
[FLASH]     ./SC168.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5209 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5209
[FLASH]     Combined pairs:   19
[FLASH]     Uncombined pairs: 5190
[FLASH]     Percent combined: 0.36%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.032 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

19 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC168/SC168_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC168_centroids.fasta --uc SC168.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 19 sequences and identified 19 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC178 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC178_1.fastq /home/working/FinalCorrected/demultiplexed/SC178_2.fastq SC178_forward.paired.fastq.gz SC178_forward.singletons.fastq.gz SC178_reverse.paired.fastq.gz SC178_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC178_1.fastq /home/working/FinalCorrected/demultiplexed/SC178_2.fastq SC178_forward.paired.fastq.gz SC178_forward.singletons.fastq.gz SC178_reverse.paired.fastq.gz SC178_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:21:21 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC178/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SC178_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC178_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC178_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC178_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC178_forward.paired.headclipped.fastq.gz SC178_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC178 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC178_forward.paired.headclipped.fastq.gz
[FLASH]     SC178_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC178.extendedFrags.fastq.gz
[FLASH]     ./SC178.notCombined_1.fastq.gz
[FLASH]     ./SC178.notCombined_2.fastq.gz
[FLASH]     ./SC178.hist
[FLASH]     ./SC178.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC178/SC178_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC178_centroids.fasta --uc SC178.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC187 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC187_1.fastq /home/working/FinalCorrected/demultiplexed/SC187_2.fastq SC187_forward.paired.fastq.gz SC187_forward.singletons.fastq.gz SC187_reverse.paired.fastq.gz SC187_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC187_1.fastq /home/working/FinalCorrected/demultiplexed/SC187_2.fastq SC187_forward.paired.fastq.gz SC187_forward.singletons.fastq.gz SC187_reverse.paired.fastq.gz SC187_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:21:22 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC187/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20627 Both Surviving: 1801 (8.73%) Forward Only Surviving: 617 (2.99%) Reverse Only Surviving: 312 (1.51%) Dropped: 17897 (86.76%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC187_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC187_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC187_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC187_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC187_forward.paired.headclipped.fastq.gz SC187_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC187 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC187_forward.paired.headclipped.fastq.gz
[FLASH]     SC187_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC187.extendedFrags.fastq.gz
[FLASH]     ./SC187.notCombined_1.fastq.gz
[FLASH]     ./SC187.notCombined_2.fastq.gz
[FLASH]     ./SC187.hist
[FLASH]     ./SC187.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1801 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1801
[FLASH]     Combined pairs:   22
[FLASH]     Uncombined pairs: 1779
[FLASH]     Percent combined: 1.22%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.013 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

22 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC187/SC187_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC187_centroids.fasta --uc SC187.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 22 sequences and identified 17 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC198 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC198_1.fastq /home/working/FinalCorrected/demultiplexed/SC198_2.fastq SC198_forward.paired.fastq.gz SC198_forward.singletons.fastq.gz SC198_reverse.paired.fastq.gz SC198_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC198_1.fastq /home/working/FinalCorrected/demultiplexed/SC198_2.fastq SC198_forward.paired.fastq.gz SC198_forward.singletons.fastq.gz SC198_reverse.paired.fastq.gz SC198_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:21:25 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC198/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 17339 Both Surviving: 3301 (19.04%) Forward Only Surviving: 903 (5.21%) Reverse Only Surviving: 242 (1.40%) Dropped: 12893 (74.36%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC198_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC198_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC198_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC198_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC198_forward.paired.headclipped.fastq.gz SC198_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC198 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC198_forward.paired.headclipped.fastq.gz
[FLASH]     SC198_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC198.extendedFrags.fastq.gz
[FLASH]     ./SC198.notCombined_1.fastq.gz
[FLASH]     ./SC198.notCombined_2.fastq.gz
[FLASH]     ./SC198.hist
[FLASH]     ./SC198.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3301 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3301
[FLASH]     Combined pairs:   51
[FLASH]     Uncombined pairs: 3250
[FLASH]     Percent combined: 1.54%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.019 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

51 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC198/SC198_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC198_centroids.fasta --uc SC198.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 51 sequences and identified 44 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC19A #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC19A_1.fastq /home/working/FinalCorrected/demultiplexed/SC19A_2.fastq SC19A_forward.paired.fastq.gz SC19A_forward.singletons.fastq.gz SC19A_reverse.paired.fastq.gz SC19A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC19A_1.fastq /home/working/FinalCorrected/demultiplexed/SC19A_2.fastq SC19A_forward.paired.fastq.gz SC19A_forward.singletons.fastq.gz SC19A_reverse.paired.fastq.gz SC19A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:21:28 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC19A/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31576 Both Surviving: 2907 (9.21%) Forward Only Surviving: 964 (3.05%) Reverse Only Surviving: 422 (1.34%) Dropped: 27283 (86.40%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SC19A_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SC19A_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SC19A_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SC19A_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC19A_forward.paired.headclipped.fastq.gz SC19A_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC19A -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC19A_forward.paired.headclipped.fastq.gz
[FLASH]     SC19A_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC19A.extendedFrags.fastq.gz
[FLASH]     ./SC19A.notCombined_1.fastq.gz
[FLASH]     ./SC19A.notCombined_2.fastq.gz
[FLASH]     ./SC19A.hist
[FLASH]     ./SC19A.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2907 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2907
[FLASH]     Combined pairs:   2641
[FLASH]     Uncombined pairs: 266
[FLASH]     Percent combined: 90.85%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.100 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2641 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC19A/SC19A_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC19A_centroids.fasta --uc SC19A.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2641 sequences and identified 1242 clusters (clustering threshold 1.00) - 23 clusters (representing 1210 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC202 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC202_1.fastq /home/working/FinalCorrected/demultiplexed/SC202_2.fastq SC202_forward.paired.fastq.gz SC202_forward.singletons.fastq.gz SC202_reverse.paired.fastq.gz SC202_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC202_1.fastq /home/working/FinalCorrected/demultiplexed/SC202_2.fastq SC202_forward.paired.fastq.gz SC202_forward.singletons.fastq.gz SC202_reverse.paired.fastq.gz SC202_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:21:35 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC202/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20207 Both Surviving: 377 (1.87%) Forward Only Surviving: 334 (1.65%) Reverse Only Surviving: 179 (0.89%) Dropped: 19317 (95.60%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC202_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC202_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC202_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC202_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC202_forward.paired.headclipped.fastq.gz SC202_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC202 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC202_forward.paired.headclipped.fastq.gz
[FLASH]     SC202_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC202.extendedFrags.fastq.gz
[FLASH]     ./SC202.notCombined_1.fastq.gz
[FLASH]     ./SC202.notCombined_2.fastq.gz
[FLASH]     ./SC202.hist
[FLASH]     ./SC202.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 377 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      377
[FLASH]     Combined pairs:   7
[FLASH]     Uncombined pairs: 370
[FLASH]     Percent combined: 1.86%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

7 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC202/SC202_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC202_centroids.fasta --uc SC202.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 7 sequences and identified 6 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC26 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC26_1.fastq /home/working/FinalCorrected/demultiplexed/SC26_2.fastq SC26_forward.paired.fastq.gz SC26_forward.singletons.fastq.gz SC26_reverse.paired.fastq.gz SC26_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC26_1.fastq /home/working/FinalCorrected/demultiplexed/SC26_2.fastq SC26_forward.paired.fastq.gz SC26_forward.singletons.fastq.gz SC26_reverse.paired.fastq.gz SC26_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:21:39 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC26/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 38586 Both Surviving: 14376 (37.26%) Forward Only Surviving: 3099 (8.03%) Reverse Only Surviving: 648 (1.68%) Dropped: 20463 (53.03%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC26_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC26_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SC26_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SC26_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC26_forward.paired.headclipped.fastq.gz SC26_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC26 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC26_forward.paired.headclipped.fastq.gz
[FLASH]     SC26_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC26.extendedFrags.fastq.gz
[FLASH]     ./SC26.notCombined_1.fastq.gz
[FLASH]     ./SC26.notCombined_2.fastq.gz
[FLASH]     ./SC26.hist
[FLASH]     ./SC26.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 14376 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      14376
[FLASH]     Combined pairs:   7773
[FLASH]     Uncombined pairs: 6603
[FLASH]     Percent combined: 54.07%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.206 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

7773 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC26/SC26_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC26_centroids.fasta --uc SC26.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 7773 sequences and identified 3818 clusters (clustering threshold 1.00) - 209 clusters (representing 3667 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC28 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC28_1.fastq /home/working/FinalCorrected/demultiplexed/SC28_2.fastq SC28_forward.paired.fastq.gz SC28_forward.singletons.fastq.gz SC28_reverse.paired.fastq.gz SC28_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC28_1.fastq /home/working/FinalCorrected/demultiplexed/SC28_2.fastq SC28_forward.paired.fastq.gz SC28_forward.singletons.fastq.gz SC28_reverse.paired.fastq.gz SC28_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:21:55 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC28/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 25262 Both Surviving: 3707 (14.67%) Forward Only Surviving: 1212 (4.80%) Reverse Only Surviving: 310 (1.23%) Dropped: 20033 (79.30%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC28_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC28_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC28_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC28_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC28_forward.paired.headclipped.fastq.gz SC28_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC28 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC28_forward.paired.headclipped.fastq.gz
[FLASH]     SC28_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC28.extendedFrags.fastq.gz
[FLASH]     ./SC28.notCombined_1.fastq.gz
[FLASH]     ./SC28.notCombined_2.fastq.gz
[FLASH]     ./SC28.hist
[FLASH]     ./SC28.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3707 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3707
[FLASH]     Combined pairs:   410
[FLASH]     Uncombined pairs: 3297
[FLASH]     Percent combined: 11.06%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.023 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

410 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC28/SC28_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC28_centroids.fasta --uc SC28.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 410 sequences and identified 243 clusters (clustering threshold 1.00) - 8 clusters (representing 159 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC34 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC34_1.fastq /home/working/FinalCorrected/demultiplexed/SC34_2.fastq SC34_forward.paired.fastq.gz SC34_forward.singletons.fastq.gz SC34_reverse.paired.fastq.gz SC34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC34_1.fastq /home/working/FinalCorrected/demultiplexed/SC34_2.fastq SC34_forward.paired.fastq.gz SC34_forward.singletons.fastq.gz SC34_reverse.paired.fastq.gz SC34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:21:59 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC34/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28305 Both Surviving: 36 (0.13%) Forward Only Surviving: 490 (1.73%) Reverse Only Surviving: 366 (1.29%) Dropped: 27413 (96.85%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SC34_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SC34_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC34_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC34_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC34_forward.paired.headclipped.fastq.gz SC34_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC34 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC34_forward.paired.headclipped.fastq.gz
[FLASH]     SC34_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC34.extendedFrags.fastq.gz
[FLASH]     ./SC34.notCombined_1.fastq.gz
[FLASH]     ./SC34.notCombined_2.fastq.gz
[FLASH]     ./SC34.hist
[FLASH]     ./SC34.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 36 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      36
[FLASH]     Combined pairs:   12
[FLASH]     Uncombined pairs: 24
[FLASH]     Percent combined: 33.33%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

12 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC34/SC34_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC34_centroids.fasta --uc SC34.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 12 sequences and identified 10 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC35 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC35_1.fastq /home/working/FinalCorrected/demultiplexed/SC35_2.fastq SC35_forward.paired.fastq.gz SC35_forward.singletons.fastq.gz SC35_reverse.paired.fastq.gz SC35_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC35_1.fastq /home/working/FinalCorrected/demultiplexed/SC35_2.fastq SC35_forward.paired.fastq.gz SC35_forward.singletons.fastq.gz SC35_reverse.paired.fastq.gz SC35_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:02 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC35/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22422 Both Surviving: 6879 (30.68%) Forward Only Surviving: 2026 (9.04%) Reverse Only Surviving: 280 (1.25%) Dropped: 13237 (59.04%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC35_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC35_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC35_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC35_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC35_forward.paired.headclipped.fastq.gz SC35_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC35 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC35_forward.paired.headclipped.fastq.gz
[FLASH]     SC35_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC35.extendedFrags.fastq.gz
[FLASH]     ./SC35.notCombined_1.fastq.gz
[FLASH]     ./SC35.notCombined_2.fastq.gz
[FLASH]     ./SC35.hist
[FLASH]     ./SC35.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6879 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6879
[FLASH]     Combined pairs:   1981
[FLASH]     Uncombined pairs: 4898
[FLASH]     Percent combined: 28.80%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.062 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1981 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC35/SC35_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC35_centroids.fasta --uc SC35.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1981 sequences and identified 1341 clusters (clustering threshold 1.00) - 41 clusters (representing 505 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC43 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC43_1.fastq /home/working/FinalCorrected/demultiplexed/SC43_2.fastq SC43_forward.paired.fastq.gz SC43_forward.singletons.fastq.gz SC43_reverse.paired.fastq.gz SC43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC43_1.fastq /home/working/FinalCorrected/demultiplexed/SC43_2.fastq SC43_forward.paired.fastq.gz SC43_forward.singletons.fastq.gz SC43_reverse.paired.fastq.gz SC43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:08 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC43/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 16887 Both Surviving: 490 (2.90%) Forward Only Surviving: 524 (3.10%) Reverse Only Surviving: 123 (0.73%) Dropped: 15750 (93.27%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC43_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC43_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC43_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC43_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC43_forward.paired.headclipped.fastq.gz SC43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC43 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC43_forward.paired.headclipped.fastq.gz
[FLASH]     SC43_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC43.extendedFrags.fastq.gz
[FLASH]     ./SC43.notCombined_1.fastq.gz
[FLASH]     ./SC43.notCombined_2.fastq.gz
[FLASH]     ./SC43.hist
[FLASH]     ./SC43.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 490 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      490
[FLASH]     Combined pairs:   330
[FLASH]     Uncombined pairs: 160
[FLASH]     Percent combined: 67.35%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.017 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

330 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC43/SC43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC43_centroids.fasta --uc SC43.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 330 sequences and identified 253 clusters (clustering threshold 1.00) - 10 clusters (representing 65 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC44 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC44_1.fastq /home/working/FinalCorrected/demultiplexed/SC44_2.fastq SC44_forward.paired.fastq.gz SC44_forward.singletons.fastq.gz SC44_reverse.paired.fastq.gz SC44_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC44_1.fastq /home/working/FinalCorrected/demultiplexed/SC44_2.fastq SC44_forward.paired.fastq.gz SC44_forward.singletons.fastq.gz SC44_reverse.paired.fastq.gz SC44_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:11 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC44/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18958 Both Surviving: 4960 (26.16%) Forward Only Surviving: 1556 (8.21%) Reverse Only Surviving: 276 (1.46%) Dropped: 12166 (64.17%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC44_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC44_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC44_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC44_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC44_forward.paired.headclipped.fastq.gz SC44_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC44 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC44_forward.paired.headclipped.fastq.gz
[FLASH]     SC44_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC44.extendedFrags.fastq.gz
[FLASH]     ./SC44.notCombined_1.fastq.gz
[FLASH]     ./SC44.notCombined_2.fastq.gz
[FLASH]     ./SC44.hist
[FLASH]     ./SC44.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4960 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4960
[FLASH]     Combined pairs:   146
[FLASH]     Uncombined pairs: 4814
[FLASH]     Percent combined: 2.94%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.034 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

146 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC44/SC44_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC44_centroids.fasta --uc SC44.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 146 sequences and identified 137 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC48 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC48_1.fastq /home/working/FinalCorrected/demultiplexed/SC48_2.fastq SC48_forward.paired.fastq.gz SC48_forward.singletons.fastq.gz SC48_reverse.paired.fastq.gz SC48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC48_1.fastq /home/working/FinalCorrected/demultiplexed/SC48_2.fastq SC48_forward.paired.fastq.gz SC48_forward.singletons.fastq.gz SC48_reverse.paired.fastq.gz SC48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:15 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC48/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 20800 Both Surviving: 72 (0.35%) Forward Only Surviving: 318 (1.53%) Reverse Only Surviving: 200 (0.96%) Dropped: 20210 (97.16%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SC48_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SC48_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC48_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC48_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC48_forward.paired.headclipped.fastq.gz SC48_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC48 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC48_forward.paired.headclipped.fastq.gz
[FLASH]     SC48_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC48.extendedFrags.fastq.gz
[FLASH]     ./SC48.notCombined_1.fastq.gz
[FLASH]     ./SC48.notCombined_2.fastq.gz
[FLASH]     ./SC48.hist
[FLASH]     ./SC48.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 72 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      72
[FLASH]     Combined pairs:   24
[FLASH]     Uncombined pairs: 48
[FLASH]     Percent combined: 33.33%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

24 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC48/SC48_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC48_centroids.fasta --uc SC48.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 24 sequences and identified 20 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC49 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC49_1.fastq /home/working/FinalCorrected/demultiplexed/SC49_2.fastq SC49_forward.paired.fastq.gz SC49_forward.singletons.fastq.gz SC49_reverse.paired.fastq.gz SC49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC49_1.fastq /home/working/FinalCorrected/demultiplexed/SC49_2.fastq SC49_forward.paired.fastq.gz SC49_forward.singletons.fastq.gz SC49_reverse.paired.fastq.gz SC49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:18 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC49/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 31250 Both Surviving: 6674 (21.36%) Forward Only Surviving: 2063 (6.60%) Reverse Only Surviving: 491 (1.57%) Dropped: 22022 (70.47%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC49_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC49_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SC49_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC49_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC49_forward.paired.headclipped.fastq.gz SC49_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC49 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC49_forward.paired.headclipped.fastq.gz
[FLASH]     SC49_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC49.extendedFrags.fastq.gz
[FLASH]     ./SC49.notCombined_1.fastq.gz
[FLASH]     ./SC49.notCombined_2.fastq.gz
[FLASH]     ./SC49.hist
[FLASH]     ./SC49.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6674 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6674
[FLASH]     Combined pairs:   899
[FLASH]     Uncombined pairs: 5775
[FLASH]     Percent combined: 13.47%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.042 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

899 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC49/SC49_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC49_centroids.fasta --uc SC49.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 899 sequences and identified 406 clusters (clustering threshold 1.00) - 9 clusters (representing 426 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC51 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC51_1.fastq /home/working/FinalCorrected/demultiplexed/SC51_2.fastq SC51_forward.paired.fastq.gz SC51_forward.singletons.fastq.gz SC51_reverse.paired.fastq.gz SC51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC51_1.fastq /home/working/FinalCorrected/demultiplexed/SC51_2.fastq SC51_forward.paired.fastq.gz SC51_forward.singletons.fastq.gz SC51_reverse.paired.fastq.gz SC51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:23 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC51/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23723 Both Surviving: 886 (3.73%) Forward Only Surviving: 537 (2.26%) Reverse Only Surviving: 282 (1.19%) Dropped: 22018 (92.81%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: SC51_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC51_forward.singletons.fastq.gz
Cropping the first 23 bases off reads in file: SC51_reverse.paired.fastq.gz
Cropping the first 23 bases off reads in file: SC51_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC51_forward.paired.headclipped.fastq.gz SC51_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC51 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC51_forward.paired.headclipped.fastq.gz
[FLASH]     SC51_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC51.extendedFrags.fastq.gz
[FLASH]     ./SC51.notCombined_1.fastq.gz
[FLASH]     ./SC51.notCombined_2.fastq.gz
[FLASH]     ./SC51.hist
[FLASH]     ./SC51.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 886 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      886
[FLASH]     Combined pairs:   396
[FLASH]     Uncombined pairs: 490
[FLASH]     Percent combined: 44.70%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.015 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

396 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC51/SC51_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC51_centroids.fasta --uc SC51.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 396 sequences and identified 187 clusters (clustering threshold 1.00) - 2 clusters (representing 184 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC51A #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC51A_1.fastq /home/working/FinalCorrected/demultiplexed/SC51A_2.fastq SC51A_forward.paired.fastq.gz SC51A_forward.singletons.fastq.gz SC51A_reverse.paired.fastq.gz SC51A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC51A_1.fastq /home/working/FinalCorrected/demultiplexed/SC51A_2.fastq SC51A_forward.paired.fastq.gz SC51A_forward.singletons.fastq.gz SC51A_reverse.paired.fastq.gz SC51A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:26 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC51A/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14869 Both Surviving: 83 (0.56%) Forward Only Surviving: 247 (1.66%) Reverse Only Surviving: 92 (0.62%) Dropped: 14447 (97.16%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC51A_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC51A_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC51A_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC51A_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC51A_forward.paired.headclipped.fastq.gz SC51A_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC51A -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC51A_forward.paired.headclipped.fastq.gz
[FLASH]     SC51A_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC51A.extendedFrags.fastq.gz
[FLASH]     ./SC51A.notCombined_1.fastq.gz
[FLASH]     ./SC51A.notCombined_2.fastq.gz
[FLASH]     ./SC51A.hist
[FLASH]     ./SC51A.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 83 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      83
[FLASH]     Combined pairs:   40
[FLASH]     Uncombined pairs: 43
[FLASH]     Percent combined: 48.19%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.003 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

40 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC51A/SC51A_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC51A_centroids.fasta --uc SC51A.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 40 sequences and identified 31 clusters (clustering threshold 1.00) - 2 clusters (representing 10 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC54A #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC54A_1.fastq /home/working/FinalCorrected/demultiplexed/SC54A_2.fastq SC54A_forward.paired.fastq.gz SC54A_forward.singletons.fastq.gz SC54A_reverse.paired.fastq.gz SC54A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC54A_1.fastq /home/working/FinalCorrected/demultiplexed/SC54A_2.fastq SC54A_forward.paired.fastq.gz SC54A_forward.singletons.fastq.gz SC54A_reverse.paired.fastq.gz SC54A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:28 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC54A/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 21305 Both Surviving: 2339 (10.98%) Forward Only Surviving: 692 (3.25%) Reverse Only Surviving: 326 (1.53%) Dropped: 17948 (84.24%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC54A_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC54A_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC54A_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC54A_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC54A_forward.paired.headclipped.fastq.gz SC54A_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC54A -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC54A_forward.paired.headclipped.fastq.gz
[FLASH]     SC54A_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC54A.extendedFrags.fastq.gz
[FLASH]     ./SC54A.notCombined_1.fastq.gz
[FLASH]     ./SC54A.notCombined_2.fastq.gz
[FLASH]     ./SC54A.hist
[FLASH]     ./SC54A.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2339 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2339
[FLASH]     Combined pairs:   270
[FLASH]     Uncombined pairs: 2069
[FLASH]     Percent combined: 11.54%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.014 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

270 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC54A/SC54A_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC54A_centroids.fasta --uc SC54A.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 270 sequences and identified 148 clusters (clustering threshold 1.00) - 3 clusters (representing 113 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC55 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC55_1.fastq /home/working/FinalCorrected/demultiplexed/SC55_2.fastq SC55_forward.paired.fastq.gz SC55_forward.singletons.fastq.gz SC55_reverse.paired.fastq.gz SC55_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC55_1.fastq /home/working/FinalCorrected/demultiplexed/SC55_2.fastq SC55_forward.paired.fastq.gz SC55_forward.singletons.fastq.gz SC55_reverse.paired.fastq.gz SC55_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:32 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC55/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22557 Both Surviving: 87 (0.39%) Forward Only Surviving: 392 (1.74%) Reverse Only Surviving: 196 (0.87%) Dropped: 21882 (97.01%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC55_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC55_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC55_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC55_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC55_forward.paired.headclipped.fastq.gz SC55_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC55 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC55_forward.paired.headclipped.fastq.gz
[FLASH]     SC55_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC55.extendedFrags.fastq.gz
[FLASH]     ./SC55.notCombined_1.fastq.gz
[FLASH]     ./SC55.notCombined_2.fastq.gz
[FLASH]     ./SC55.hist
[FLASH]     ./SC55.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 87 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      87
[FLASH]     Combined pairs:   27
[FLASH]     Uncombined pairs: 60
[FLASH]     Percent combined: 31.03%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

27 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC55/SC55_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC55_centroids.fasta --uc SC55.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 27 sequences and identified 22 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC57 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC57_1.fastq /home/working/FinalCorrected/demultiplexed/SC57_2.fastq SC57_forward.paired.fastq.gz SC57_forward.singletons.fastq.gz SC57_reverse.paired.fastq.gz SC57_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC57_1.fastq /home/working/FinalCorrected/demultiplexed/SC57_2.fastq SC57_forward.paired.fastq.gz SC57_forward.singletons.fastq.gz SC57_reverse.paired.fastq.gz SC57_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:35 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC57/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 41048 Both Surviving: 3355 (8.17%) Forward Only Surviving: 1317 (3.21%) Reverse Only Surviving: 345 (0.84%) Dropped: 36031 (87.78%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC57_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC57_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC57_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC57_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC57_forward.paired.headclipped.fastq.gz SC57_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC57 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC57_forward.paired.headclipped.fastq.gz
[FLASH]     SC57_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC57.extendedFrags.fastq.gz
[FLASH]     ./SC57.notCombined_1.fastq.gz
[FLASH]     ./SC57.notCombined_2.fastq.gz
[FLASH]     ./SC57.hist
[FLASH]     ./SC57.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3355 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3355
[FLASH]     Combined pairs:   2894
[FLASH]     Uncombined pairs: 461
[FLASH]     Percent combined: 86.26%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.094 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2894 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC57/SC57_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC57_centroids.fasta --uc SC57.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2894 sequences and identified 1559 clusters (clustering threshold 1.00) - 25 clusters (representing 1111 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC60 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC60_1.fastq /home/working/FinalCorrected/demultiplexed/SC60_2.fastq SC60_forward.paired.fastq.gz SC60_forward.singletons.fastq.gz SC60_reverse.paired.fastq.gz SC60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC60_1.fastq /home/working/FinalCorrected/demultiplexed/SC60_2.fastq SC60_forward.paired.fastq.gz SC60_forward.singletons.fastq.gz SC60_reverse.paired.fastq.gz SC60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:44 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC60/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 37624 Both Surviving: 4932 (13.11%) Forward Only Surviving: 1535 (4.08%) Reverse Only Surviving: 322 (0.86%) Dropped: 30835 (81.96%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC60_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC60_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC60_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC60_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC60_forward.paired.headclipped.fastq.gz SC60_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC60 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC60_forward.paired.headclipped.fastq.gz
[FLASH]     SC60_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC60.extendedFrags.fastq.gz
[FLASH]     ./SC60.notCombined_1.fastq.gz
[FLASH]     ./SC60.notCombined_2.fastq.gz
[FLASH]     ./SC60.hist
[FLASH]     ./SC60.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4932 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4932
[FLASH]     Combined pairs:   4428
[FLASH]     Uncombined pairs: 504
[FLASH]     Percent combined: 89.78%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.130 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4428 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC60/SC60_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC60_centroids.fasta --uc SC60.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4428 sequences and identified 2621 clusters (clustering threshold 1.00) - 51 clusters (representing 1597 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC61 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC61_1.fastq /home/working/FinalCorrected/demultiplexed/SC61_2.fastq SC61_forward.paired.fastq.gz SC61_forward.singletons.fastq.gz SC61_reverse.paired.fastq.gz SC61_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC61_1.fastq /home/working/FinalCorrected/demultiplexed/SC61_2.fastq SC61_forward.paired.fastq.gz SC61_forward.singletons.fastq.gz SC61_reverse.paired.fastq.gz SC61_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:54 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC61/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 18563 Both Surviving: 312 (1.68%) Forward Only Surviving: 310 (1.67%) Reverse Only Surviving: 307 (1.65%) Dropped: 17634 (95.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC61_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC61_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC61_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC61_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC61_forward.paired.headclipped.fastq.gz SC61_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC61 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC61_forward.paired.headclipped.fastq.gz
[FLASH]     SC61_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC61.extendedFrags.fastq.gz
[FLASH]     ./SC61.notCombined_1.fastq.gz
[FLASH]     ./SC61.notCombined_2.fastq.gz
[FLASH]     ./SC61.hist
[FLASH]     ./SC61.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 312 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      312
[FLASH]     Combined pairs:   4
[FLASH]     Uncombined pairs: 308
[FLASH]     Percent combined: 1.28%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.004 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

4 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC61/SC61_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC61_centroids.fasta --uc SC61.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 4 sequences and identified 4 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC66 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC66_1.fastq /home/working/FinalCorrected/demultiplexed/SC66_2.fastq SC66_forward.paired.fastq.gz SC66_forward.singletons.fastq.gz SC66_reverse.paired.fastq.gz SC66_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC66_1.fastq /home/working/FinalCorrected/demultiplexed/SC66_2.fastq SC66_forward.paired.fastq.gz SC66_forward.singletons.fastq.gz SC66_reverse.paired.fastq.gz SC66_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:55 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC66/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 8462 Both Surviving: 1017 (12.02%) Forward Only Surviving: 497 (5.87%) Reverse Only Surviving: 76 (0.90%) Dropped: 6872 (81.21%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC66_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC66_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC66_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC66_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC66_forward.paired.headclipped.fastq.gz SC66_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC66 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC66_forward.paired.headclipped.fastq.gz
[FLASH]     SC66_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC66.extendedFrags.fastq.gz
[FLASH]     ./SC66.notCombined_1.fastq.gz
[FLASH]     ./SC66.notCombined_2.fastq.gz
[FLASH]     ./SC66.hist
[FLASH]     ./SC66.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1017 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1017
[FLASH]     Combined pairs:   811
[FLASH]     Uncombined pairs: 206
[FLASH]     Percent combined: 79.74%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.031 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

811 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC66/SC66_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC66_centroids.fasta --uc SC66.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 811 sequences and identified 360 clusters (clustering threshold 1.00) - 7 clusters (representing 410 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC7 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC7_1.fastq /home/working/FinalCorrected/demultiplexed/SC7_2.fastq SC7_forward.paired.fastq.gz SC7_forward.singletons.fastq.gz SC7_reverse.paired.fastq.gz SC7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC7_1.fastq /home/working/FinalCorrected/demultiplexed/SC7_2.fastq SC7_forward.paired.fastq.gz SC7_forward.singletons.fastq.gz SC7_reverse.paired.fastq.gz SC7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:57 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC7/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 4 Both Surviving: 0 (0.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 1 (25.00%) Dropped: 3 (75.00%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 22 bases off reads in file: SC7_forward.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC7_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SC7_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC7_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC7_forward.paired.headclipped.fastq.gz SC7_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC7 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC7_forward.paired.headclipped.fastq.gz
[FLASH]     SC7_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC7.extendedFrags.fastq.gz
[FLASH]     ./SC7.notCombined_1.fastq.gz
[FLASH]     ./SC7.notCombined_2.fastq.gz
[FLASH]     ./SC7.hist
[FLASH]     ./SC7.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      0
[FLASH]     Combined pairs:   0
[FLASH]     Uncombined pairs: 0
[FLASH]     Percent combined: 0.00%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.001 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

0 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC7/SC7_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC7_centroids.fasta --uc SC7.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC72 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC72_1.fastq /home/working/FinalCorrected/demultiplexed/SC72_2.fastq SC72_forward.paired.fastq.gz SC72_forward.singletons.fastq.gz SC72_reverse.paired.fastq.gz SC72_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC72_1.fastq /home/working/FinalCorrected/demultiplexed/SC72_2.fastq SC72_forward.paired.fastq.gz SC72_forward.singletons.fastq.gz SC72_reverse.paired.fastq.gz SC72_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:22:58 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC72/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9942 Both Surviving: 120 (1.21%) Forward Only Surviving: 158 (1.59%) Reverse Only Surviving: 139 (1.40%) Dropped: 9525 (95.81%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC72_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC72_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: SC72_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC72_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC72_forward.paired.headclipped.fastq.gz SC72_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC72 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC72_forward.paired.headclipped.fastq.gz
[FLASH]     SC72_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC72.extendedFrags.fastq.gz
[FLASH]     ./SC72.notCombined_1.fastq.gz
[FLASH]     ./SC72.notCombined_2.fastq.gz
[FLASH]     ./SC72.hist
[FLASH]     ./SC72.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 120 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      120
[FLASH]     Combined pairs:   9
[FLASH]     Uncombined pairs: 111
[FLASH]     Percent combined: 7.50%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

9 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC72/SC72_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC72_centroids.fasta --uc SC72.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 9 sequences and identified 9 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC74 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC74_1.fastq /home/working/FinalCorrected/demultiplexed/SC74_2.fastq SC74_forward.paired.fastq.gz SC74_forward.singletons.fastq.gz SC74_reverse.paired.fastq.gz SC74_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC74_1.fastq /home/working/FinalCorrected/demultiplexed/SC74_2.fastq SC74_forward.paired.fastq.gz SC74_forward.singletons.fastq.gz SC74_reverse.paired.fastq.gz SC74_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:00 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC74/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14010 Both Surviving: 31 (0.22%) Forward Only Surviving: 250 (1.78%) Reverse Only Surviving: 64 (0.46%) Dropped: 13665 (97.54%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC74_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC74_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC74_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC74_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC74_forward.paired.headclipped.fastq.gz SC74_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC74 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC74_forward.paired.headclipped.fastq.gz
[FLASH]     SC74_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC74.extendedFrags.fastq.gz
[FLASH]     ./SC74.notCombined_1.fastq.gz
[FLASH]     ./SC74.notCombined_2.fastq.gz
[FLASH]     ./SC74.hist
[FLASH]     ./SC74.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 31 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      31
[FLASH]     Combined pairs:   20
[FLASH]     Uncombined pairs: 11
[FLASH]     Percent combined: 64.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

20 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC74/SC74_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC74_centroids.fasta --uc SC74.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 20 sequences and identified 19 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC77 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC77_1.fastq /home/working/FinalCorrected/demultiplexed/SC77_2.fastq SC77_forward.paired.fastq.gz SC77_forward.singletons.fastq.gz SC77_reverse.paired.fastq.gz SC77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC77_1.fastq /home/working/FinalCorrected/demultiplexed/SC77_2.fastq SC77_forward.paired.fastq.gz SC77_forward.singletons.fastq.gz SC77_reverse.paired.fastq.gz SC77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:02 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC77/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 28064 Both Surviving: 1951 (6.95%) Forward Only Surviving: 902 (3.21%) Reverse Only Surviving: 222 (0.79%) Dropped: 24989 (89.04%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC77_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC77_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SC77_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC77_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC77_forward.paired.headclipped.fastq.gz SC77_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC77 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC77_forward.paired.headclipped.fastq.gz
[FLASH]     SC77_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC77.extendedFrags.fastq.gz
[FLASH]     ./SC77.notCombined_1.fastq.gz
[FLASH]     ./SC77.notCombined_2.fastq.gz
[FLASH]     ./SC77.hist
[FLASH]     ./SC77.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1951 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1951
[FLASH]     Combined pairs:   1605
[FLASH]     Uncombined pairs: 346
[FLASH]     Percent combined: 82.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.042 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1605 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC77/SC77_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC77_centroids.fasta --uc SC77.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1605 sequences and identified 885 clusters (clustering threshold 1.00) - 18 clusters (representing 637 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC78 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC78_1.fastq /home/working/FinalCorrected/demultiplexed/SC78_2.fastq SC78_forward.paired.fastq.gz SC78_forward.singletons.fastq.gz SC78_reverse.paired.fastq.gz SC78_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC78_1.fastq /home/working/FinalCorrected/demultiplexed/SC78_2.fastq SC78_forward.paired.fastq.gz SC78_forward.singletons.fastq.gz SC78_reverse.paired.fastq.gz SC78_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:07 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC78/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 14701 Both Surviving: 2518 (17.13%) Forward Only Surviving: 803 (5.46%) Reverse Only Surviving: 238 (1.62%) Dropped: 11142 (75.79%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC78_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC78_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: SC78_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: SC78_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC78_forward.paired.headclipped.fastq.gz SC78_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC78 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC78_forward.paired.headclipped.fastq.gz
[FLASH]     SC78_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC78.extendedFrags.fastq.gz
[FLASH]     ./SC78.notCombined_1.fastq.gz
[FLASH]     ./SC78.notCombined_2.fastq.gz
[FLASH]     ./SC78.hist
[FLASH]     ./SC78.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 2518 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      2518
[FLASH]     Combined pairs:   102
[FLASH]     Uncombined pairs: 2416
[FLASH]     Percent combined: 4.05%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.017 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

102 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC78/SC78_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC78_centroids.fasta --uc SC78.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 102 sequences and identified 82 clusters (clustering threshold 1.00) - 1 clusters (representing 12 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC81 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC81_1.fastq /home/working/FinalCorrected/demultiplexed/SC81_2.fastq SC81_forward.paired.fastq.gz SC81_forward.singletons.fastq.gz SC81_reverse.paired.fastq.gz SC81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC81_1.fastq /home/working/FinalCorrected/demultiplexed/SC81_2.fastq SC81_forward.paired.fastq.gz SC81_forward.singletons.fastq.gz SC81_reverse.paired.fastq.gz SC81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:11 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC81/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 24476 Both Surviving: 4241 (17.33%) Forward Only Surviving: 1081 (4.42%) Reverse Only Surviving: 382 (1.56%) Dropped: 18772 (76.70%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: SC81_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: SC81_forward.singletons.fastq.gz
Cropping the first 25 bases off reads in file: SC81_reverse.paired.fastq.gz
Cropping the first 25 bases off reads in file: SC81_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC81_forward.paired.headclipped.fastq.gz SC81_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC81 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC81_forward.paired.headclipped.fastq.gz
[FLASH]     SC81_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC81.extendedFrags.fastq.gz
[FLASH]     ./SC81.notCombined_1.fastq.gz
[FLASH]     ./SC81.notCombined_2.fastq.gz
[FLASH]     ./SC81.hist
[FLASH]     ./SC81.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 4241 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      4241
[FLASH]     Combined pairs:   3838
[FLASH]     Uncombined pairs: 403
[FLASH]     Percent combined: 90.50%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.103 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

3838 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC81/SC81_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC81_centroids.fasta --uc SC81.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 3838 sequences and identified 2236 clusters (clustering threshold 1.00) - 47 clusters (representing 1401 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC82 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC82_1.fastq /home/working/FinalCorrected/demultiplexed/SC82_2.fastq SC82_forward.paired.fastq.gz SC82_forward.singletons.fastq.gz SC82_reverse.paired.fastq.gz SC82_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC82_1.fastq /home/working/FinalCorrected/demultiplexed/SC82_2.fastq SC82_forward.paired.fastq.gz SC82_forward.singletons.fastq.gz SC82_reverse.paired.fastq.gz SC82_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:19 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC82/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 11078 Both Surviving: 1658 (14.97%) Forward Only Surviving: 712 (6.43%) Reverse Only Surviving: 156 (1.41%) Dropped: 8552 (77.20%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC82_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC82_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC82_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC82_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC82_forward.paired.headclipped.fastq.gz SC82_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC82 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC82_forward.paired.headclipped.fastq.gz
[FLASH]     SC82_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC82.extendedFrags.fastq.gz
[FLASH]     ./SC82.notCombined_1.fastq.gz
[FLASH]     ./SC82.notCombined_2.fastq.gz
[FLASH]     ./SC82.hist
[FLASH]     ./SC82.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1658 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1658
[FLASH]     Combined pairs:   228
[FLASH]     Uncombined pairs: 1430
[FLASH]     Percent combined: 13.75%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.012 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

228 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC82/SC82_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC82_centroids.fasta --uc SC82.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 228 sequences and identified 168 clusters (clustering threshold 1.00) - 6 clusters (representing 56 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC86A #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC86A_1.fastq /home/working/FinalCorrected/demultiplexed/SC86A_2.fastq SC86A_forward.paired.fastq.gz SC86A_forward.singletons.fastq.gz SC86A_reverse.paired.fastq.gz SC86A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC86A_1.fastq /home/working/FinalCorrected/demultiplexed/SC86A_2.fastq SC86A_forward.paired.fastq.gz SC86A_forward.singletons.fastq.gz SC86A_reverse.paired.fastq.gz SC86A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:22 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC86A/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 33212 Both Surviving: 5816 (17.51%) Forward Only Surviving: 1743 (5.25%) Reverse Only Surviving: 506 (1.52%) Dropped: 25147 (75.72%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC86A_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC86A_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: SC86A_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC86A_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC86A_forward.paired.headclipped.fastq.gz SC86A_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC86A -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC86A_forward.paired.headclipped.fastq.gz
[FLASH]     SC86A_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC86A.extendedFrags.fastq.gz
[FLASH]     ./SC86A.notCombined_1.fastq.gz
[FLASH]     ./SC86A.notCombined_2.fastq.gz
[FLASH]     ./SC86A.hist
[FLASH]     ./SC86A.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 5816 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      5816
[FLASH]     Combined pairs:   1295
[FLASH]     Uncombined pairs: 4521
[FLASH]     Percent combined: 22.27%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.050 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1295 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC86A/SC86A_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC86A_centroids.fasta --uc SC86A.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1295 sequences and identified 585 clusters (clustering threshold 1.00) - 9 clusters (representing 583 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC88 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC88_1.fastq /home/working/FinalCorrected/demultiplexed/SC88_2.fastq SC88_forward.paired.fastq.gz SC88_forward.singletons.fastq.gz SC88_reverse.paired.fastq.gz SC88_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC88_1.fastq /home/working/FinalCorrected/demultiplexed/SC88_2.fastq SC88_forward.paired.fastq.gz SC88_forward.singletons.fastq.gz SC88_reverse.paired.fastq.gz SC88_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:29 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC88/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 22460 Both Surviving: 3099 (13.80%) Forward Only Surviving: 1398 (6.22%) Reverse Only Surviving: 318 (1.42%) Dropped: 17645 (78.56%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: SC88_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: SC88_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC88_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC88_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC88_forward.paired.headclipped.fastq.gz SC88_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC88 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC88_forward.paired.headclipped.fastq.gz
[FLASH]     SC88_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC88.extendedFrags.fastq.gz
[FLASH]     ./SC88.notCombined_1.fastq.gz
[FLASH]     ./SC88.notCombined_2.fastq.gz
[FLASH]     ./SC88.hist
[FLASH]     ./SC88.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3099 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3099
[FLASH]     Combined pairs:   1576
[FLASH]     Uncombined pairs: 1523
[FLASH]     Percent combined: 50.86%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.058 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

1576 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC88/SC88_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC88_centroids.fasta --uc SC88.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 1576 sequences and identified 873 clusters (clustering threshold 1.00) - 22 clusters (representing 660 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC9 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC9_1.fastq /home/working/FinalCorrected/demultiplexed/SC9_2.fastq SC9_forward.paired.fastq.gz SC9_forward.singletons.fastq.gz SC9_reverse.paired.fastq.gz SC9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC9_1.fastq /home/working/FinalCorrected/demultiplexed/SC9_2.fastq SC9_forward.paired.fastq.gz SC9_forward.singletons.fastq.gz SC9_reverse.paired.fastq.gz SC9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:34 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC9/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 19434 Both Surviving: 882 (4.54%) Forward Only Surviving: 689 (3.55%) Reverse Only Surviving: 234 (1.20%) Dropped: 17629 (90.71%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 24 bases off reads in file: SC9_forward.paired.fastq.gz
Cropping the first 24 bases off reads in file: SC9_forward.singletons.fastq.gz
Cropping the first 22 bases off reads in file: SC9_reverse.paired.fastq.gz
Cropping the first 22 bases off reads in file: SC9_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC9_forward.paired.headclipped.fastq.gz SC9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC9 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC9_forward.paired.headclipped.fastq.gz
[FLASH]     SC9_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC9.extendedFrags.fastq.gz
[FLASH]     ./SC9.notCombined_1.fastq.gz
[FLASH]     ./SC9.notCombined_2.fastq.gz
[FLASH]     ./SC9.hist
[FLASH]     ./SC9.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 882 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      882
[FLASH]     Combined pairs:   665
[FLASH]     Uncombined pairs: 217
[FLASH]     Percent combined: 75.40%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.028 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

665 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC9/SC9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC9_centroids.fasta --uc SC9.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 665 sequences and identified 445 clusters (clustering threshold 1.00) - 15 clusters (representing 211 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: SC93 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC93_1.fastq /home/working/FinalCorrected/demultiplexed/SC93_2.fastq SC93_forward.paired.fastq.gz SC93_forward.singletons.fastq.gz SC93_reverse.paired.fastq.gz SC93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/SC93_1.fastq /home/working/FinalCorrected/demultiplexed/SC93_2.fastq SC93_forward.paired.fastq.gz SC93_forward.singletons.fastq.gz SC93_reverse.paired.fastq.gz SC93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:37 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/SC93/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 13069 Both Surviving: 496 (3.80%) Forward Only Surviving: 247 (1.89%) Reverse Only Surviving: 106 (0.81%) Dropped: 12220 (93.50%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 21 bases off reads in file: SC93_forward.paired.fastq.gz
Cropping the first 21 bases off reads in file: SC93_forward.singletons.fastq.gz
Cropping the first 28 bases off reads in file: SC93_reverse.paired.fastq.gz
Cropping the first 28 bases off reads in file: SC93_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash SC93_forward.paired.headclipped.fastq.gz SC93_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC93 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     SC93_forward.paired.headclipped.fastq.gz
[FLASH]     SC93_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./SC93.extendedFrags.fastq.gz
[FLASH]     ./SC93.notCombined_1.fastq.gz
[FLASH]     ./SC93.notCombined_2.fastq.gz
[FLASH]     ./SC93.hist
[FLASH]     ./SC93.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 496 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      496
[FLASH]     Combined pairs:   37
[FLASH]     Uncombined pairs: 459
[FLASH]     Percent combined: 7.46%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.005 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

37 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/SC93/SC93_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC93_centroids.fasta --uc SC93.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 37 sequences and identified 31 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: T19 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/T19_1.fastq /home/working/FinalCorrected/demultiplexed/T19_2.fastq T19_forward.paired.fastq.gz T19_forward.singletons.fastq.gz T19_reverse.paired.fastq.gz T19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/T19_1.fastq /home/working/FinalCorrected/demultiplexed/T19_2.fastq T19_forward.paired.fastq.gz T19_forward.singletons.fastq.gz T19_reverse.paired.fastq.gz T19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:39 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/T19/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 29655 Both Surviving: 29 (0.10%) Forward Only Surviving: 265 (0.89%) Reverse Only Surviving: 90 (0.30%) Dropped: 29271 (98.71%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: T19_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: T19_forward.singletons.fastq.gz
Cropping the first 24 bases off reads in file: T19_reverse.paired.fastq.gz
Cropping the first 24 bases off reads in file: T19_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash T19_forward.paired.headclipped.fastq.gz T19_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T19 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     T19_forward.paired.headclipped.fastq.gz
[FLASH]     T19_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./T19.extendedFrags.fastq.gz
[FLASH]     ./T19.notCombined_1.fastq.gz
[FLASH]     ./T19.notCombined_2.fastq.gz
[FLASH]     ./T19.hist
[FLASH]     ./T19.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 29 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      29
[FLASH]     Combined pairs:   18
[FLASH]     Uncombined pairs: 11
[FLASH]     Percent combined: 62.07%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.002 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

18 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/T19/T19_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T19_centroids.fasta --uc T19.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 18 sequences and identified 16 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: T21 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/T21_1.fastq /home/working/FinalCorrected/demultiplexed/T21_2.fastq T21_forward.paired.fastq.gz T21_forward.singletons.fastq.gz T21_reverse.paired.fastq.gz T21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/T21_1.fastq /home/working/FinalCorrected/demultiplexed/T21_2.fastq T21_forward.paired.fastq.gz T21_forward.singletons.fastq.gz T21_reverse.paired.fastq.gz T21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:41 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/T21/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 8196 Both Surviving: 288 (3.51%) Forward Only Surviving: 162 (1.98%) Reverse Only Surviving: 140 (1.71%) Dropped: 7606 (92.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: T21_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: T21_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: T21_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: T21_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash T21_forward.paired.headclipped.fastq.gz T21_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T21 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     T21_forward.paired.headclipped.fastq.gz
[FLASH]     T21_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./T21.extendedFrags.fastq.gz
[FLASH]     ./T21.notCombined_1.fastq.gz
[FLASH]     ./T21.notCombined_2.fastq.gz
[FLASH]     ./T21.hist
[FLASH]     ./T21.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 288 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      288
[FLASH]     Combined pairs:   28
[FLASH]     Uncombined pairs: 260
[FLASH]     Percent combined: 9.72%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.005 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

28 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/T21/T21_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T21_centroids.fasta --uc T21.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 28 sequences and identified 24 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: T32 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/T32_1.fastq /home/working/FinalCorrected/demultiplexed/T32_2.fastq T32_forward.paired.fastq.gz T32_forward.singletons.fastq.gz T32_reverse.paired.fastq.gz T32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/T32_1.fastq /home/working/FinalCorrected/demultiplexed/T32_2.fastq T32_forward.paired.fastq.gz T32_forward.singletons.fastq.gz T32_reverse.paired.fastq.gz T32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:43 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/T32/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 13172 Both Surviving: 1614 (12.25%) Forward Only Surviving: 386 (2.93%) Reverse Only Surviving: 163 (1.24%) Dropped: 11009 (83.58%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: T32_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: T32_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: T32_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: T32_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash T32_forward.paired.headclipped.fastq.gz T32_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T32 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     T32_forward.paired.headclipped.fastq.gz
[FLASH]     T32_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./T32.extendedFrags.fastq.gz
[FLASH]     ./T32.notCombined_1.fastq.gz
[FLASH]     ./T32.notCombined_2.fastq.gz
[FLASH]     ./T32.hist
[FLASH]     ./T32.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 1614 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      1614
[FLASH]     Combined pairs:   285
[FLASH]     Uncombined pairs: 1329
[FLASH]     Percent combined: 17.66%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.018 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

285 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/T32/T32_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T32_centroids.fasta --uc T32.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 285 sequences and identified 187 clusters (clustering threshold 1.00) - 3 clusters (representing 87 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: T49 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/T49_1.fastq /home/working/FinalCorrected/demultiplexed/T49_2.fastq T49_forward.paired.fastq.gz T49_forward.singletons.fastq.gz T49_reverse.paired.fastq.gz T49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/T49_1.fastq /home/working/FinalCorrected/demultiplexed/T49_2.fastq T49_forward.paired.fastq.gz T49_forward.singletons.fastq.gz T49_reverse.paired.fastq.gz T49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:45 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/T49/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 9868 Both Surviving: 421 (4.27%) Forward Only Surviving: 275 (2.79%) Reverse Only Surviving: 113 (1.15%) Dropped: 9059 (91.80%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 20 bases off reads in file: T49_forward.paired.fastq.gz
Cropping the first 20 bases off reads in file: T49_forward.singletons.fastq.gz
Cropping the first 27 bases off reads in file: T49_reverse.paired.fastq.gz
Cropping the first 27 bases off reads in file: T49_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash T49_forward.paired.headclipped.fastq.gz T49_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T49 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     T49_forward.paired.headclipped.fastq.gz
[FLASH]     T49_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./T49.extendedFrags.fastq.gz
[FLASH]     ./T49.notCombined_1.fastq.gz
[FLASH]     ./T49.notCombined_2.fastq.gz
[FLASH]     ./T49.hist
[FLASH]     ./T49.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 421 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      421
[FLASH]     Combined pairs:   339
[FLASH]     Uncombined pairs: 82
[FLASH]     Percent combined: 80.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.018 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

339 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/T49/T49_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T49_centroids.fasta --uc T49.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 339 sequences and identified 210 clusters (clustering threshold 1.00) - 10 clusters (representing 127 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: T68 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/T68_1.fastq /home/working/FinalCorrected/demultiplexed/T68_2.fastq T68_forward.paired.fastq.gz T68_forward.singletons.fastq.gz T68_reverse.paired.fastq.gz T68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/T68_1.fastq /home/working/FinalCorrected/demultiplexed/T68_2.fastq T68_forward.paired.fastq.gz T68_forward.singletons.fastq.gz T68_reverse.paired.fastq.gz T68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:48 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/T68/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23250 Both Surviving: 3693 (15.88%) Forward Only Surviving: 778 (3.35%) Reverse Only Surviving: 388 (1.67%) Dropped: 18391 (79.10%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 25 bases off reads in file: T68_forward.paired.fastq.gz
Cropping the first 25 bases off reads in file: T68_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: T68_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: T68_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash T68_forward.paired.headclipped.fastq.gz T68_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T68 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     T68_forward.paired.headclipped.fastq.gz
[FLASH]     T68_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./T68.extendedFrags.fastq.gz
[FLASH]     ./T68.notCombined_1.fastq.gz
[FLASH]     ./T68.notCombined_2.fastq.gz
[FLASH]     ./T68.hist
[FLASH]     ./T68.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 3693 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      3693
[FLASH]     Combined pairs:   151
[FLASH]     Uncombined pairs: 3542
[FLASH]     Percent combined: 4.09%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.022 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

151 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/T68/T68_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T68_centroids.fasta --uc T68.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 151 sequences and identified 99 clusters (clustering threshold 1.00) - 2 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


##### processing query ID: T78 #####


### READ QUALITY TRIMMING ###


trimming PE reads with trimmomatic
java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/T78_1.fastq /home/working/FinalCorrected/demultiplexed/T78_2.fastq T78_forward.paired.fastq.gz T78_forward.singletons.fastq.gz T78_reverse.paired.fastq.gz T78_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/FinalCorrected/demultiplexed/T78_1.fastq /home/working/FinalCorrected/demultiplexed/T78_2.fastq T78_forward.paired.fastq.gz T78_forward.singletons.fastq.gz T78_reverse.paired.fastq.gz T78_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90
Nov 05, 2016 11:23:51 AM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer
SEVERE: null
java.io.FileNotFoundException: /home/working/FinalCorrected/T78/None (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:146)
	at org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)
	at org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)
	at org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)
	at org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)
	at org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)

Input Read Pairs: 23555 Both Surviving: 6269 (26.61%) Forward Only Surviving: 1325 (5.63%) Reverse Only Surviving: 408 (1.73%) Dropped: 15553 (66.03%)
TrimmomaticPE: Completed successfully


### READ CLIPPING ###

Cropping the first 26 bases off reads in file: T78_forward.paired.fastq.gz
Cropping the first 26 bases off reads in file: T78_forward.singletons.fastq.gz
Cropping the first 26 bases off reads in file: T78_reverse.paired.fastq.gz
Cropping the first 26 bases off reads in file: T78_reverse.singletons.fastq.gz

### MERGING READ PAIRS ###

merging paired-end reads with flash

flash T78_forward.paired.headclipped.fastq.gz T78_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T78 -z
[FLASH] Starting FLASH v1.2.11
[FLASH] Fast Length Adjustment of SHort reads
[FLASH]  
[FLASH] Input files:
[FLASH]     T78_forward.paired.headclipped.fastq.gz
[FLASH]     T78_reverse.paired.headclipped.fastq.gz
[FLASH]  
[FLASH] Output files:
[FLASH]     ./T78.extendedFrags.fastq.gz
[FLASH]     ./T78.notCombined_1.fastq.gz
[FLASH]     ./T78.notCombined_2.fastq.gz
[FLASH]     ./T78.hist
[FLASH]     ./T78.histogram
[FLASH]  
[FLASH] Parameters:
[FLASH]     Min overlap:           10
[FLASH]     Max overlap:           350
[FLASH]     Max mismatch density:  0.250000
[FLASH]     Allow &quot;outie&quot; pairs:   false
[FLASH]     Cap mismatch quals:    false
[FLASH]     Combiner threads:      1
[FLASH]     Input format:          FASTQ, phred_offset=33
[FLASH]     Output format:         FASTQ, phred_offset=33, gzip
[FLASH]  
[FLASH] Starting reader and writer threads
[FLASH] Starting 1 combiner threads
[FLASH] Processed 6269 read pairs
[FLASH]  
[FLASH] Read combination statistics:
[FLASH]     Total pairs:      6269
[FLASH]     Combined pairs:   2603
[FLASH]     Uncombined pairs: 3666
[FLASH]     Percent combined: 41.52%
[FLASH]  
[FLASH] Writing histogram files.
[FLASH]  
[FLASH] FLASH v1.2.11 complete!
[FLASH] 0.080 seconds elapsed



### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###


keeping only merged reads for subsequent analyses

2603 sequences survived filtering


### CLUSTERING ###


clustering using vsearch
vsearch --cluster_fast /home/working/FinalCorrected/T78/T78_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T78_centroids.fasta --uc T78.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


reduce cluster files - minimum coverage: 5

vsearch processed 2603 sequences and identified 1486 clusters (clustering threshold 1.00) - 60 clusters (representing 972 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses

WRITING BASIC READ STATS TO FILE


### GLOBAL CLUSTERING ###


Concatenating for global clustering


Performing global clustering

vsearch --cluster_fast /home/working/FinalCorrected/GLOBAL/global_queries.fasta --id 1.00 --strand both --threads 1 --centroids global_centroids.fasta --uc global.uc
vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores
https://github.com/torognes/vsearch



parse vsearch uc file


writing denovo OTU table


Please select a method for taxonomic assignment:
--blast
--pplace
--kraken


##### DONE! #####


Sat Nov  5 11:24:11 2016

</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Now we can run the final BLAST against the new files</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[13]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cd</span> <span class="n">FinalBlast</span><span class="o">/</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>/home/working/FinalCorrected/FinalBlast
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>We need an updated Querymap, based on the corrected version in the folder above.</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[14]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

<span class="k">for </span>sample in <span class="k">$(</span>cat ../Querymap_global_corrected.txt | cut -f 1<span class="k">)</span>
<span class="k">do</span>
<span class="k">    </span><span class="nv">fasta</span><span class="o">=</span><span class="k">$(</span>ls -1 ../<span class="nv">$sample</span>/<span class="nv">$sample</span><span class="se">\_</span>trimmed.fasta<span class="k">)</span>
    <span class="nb">echo</span> -e <span class="s2">&quot;$sample\tfasta\t$fasta&quot;</span>
<span class="k">done</span> &gt; Querymap.txt

head Querymap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>SA177	fasta	../SA177/SA177_trimmed.fasta
SB140	fasta	../SB140/SB140_trimmed.fasta
T19	fasta	../T19/T19_trimmed.fasta
Lg45	fasta	../Lg45/Lg45_trimmed.fasta
Q69	fasta	../Q69/Q69_trimmed.fasta
S72	fasta	../S72/S72_trimmed.fasta
SC66	fasta	../SC66/SC66_trimmed.fasta
S31	fasta	../S31/S31_trimmed.fasta
SC74	fasta	../SC74/SC74_trimmed.fasta
SA68	fasta	../SA68/SA68_trimmed.fasta
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[16]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span>grep <span class="s2">&quot;POS&quot;</span> Querymap.txt
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>POS2_1	fasta	../POS2_1/POS2_1_trimmed.fasta
POS4_1	fasta	../POS4_1/POS4_1_trimmed.fasta
POS1_1	fasta	../POS1_1/POS1_1_trimmed.fasta
POS3_1	fasta	../POS3_1/POS3_1_trimmed.fasta
POS2_2	fasta	../POS2_2/POS2_2_trimmed.fasta
POS4_2	fasta	../POS4_2/POS4_2_trimmed.fasta
POS1_2	fasta	../POS1_2/POS1_2_trimmed.fasta
POS3_2	fasta	../POS3_2/POS3_2_trimmed.fasta
POS2_3	fasta	../POS2_3/POS2_3_trimmed.fasta
POS4_3	fasta	../POS4_3/POS4_3_trimmed.fasta
POS1_3	fasta	../POS1_3/POS1_3_trimmed.fasta
POS3_3	fasta	../POS3_3/POS3_3_trimmed.fasta
POS2_4	fasta	../POS2_4/POS2_4_trimmed.fasta
POS4_4	fasta	../POS4_4/POS4_4_trimmed.fasta
POS1_4	fasta	../POS1_4/POS1_4_trimmed.fasta
POS3_4	fasta	../POS3_4/POS3_4_trimmed.fasta
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>And a REFmap and reference database (fortunately, the error had nothing to do with the reference database, so we can just copy this across from the previous "Final BLAST" folder!)</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[17]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="o">!</span><span class="nb">echo</span> <span class="s2">&quot;eyorks_flora_final.gb\tgb&quot;</span> &gt; REFmap.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[18]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">cp</span> <span class="o">../../</span><span class="n">AgainstRefDB</span><span class="o">/</span><span class="n">Curat1</span><span class="o">/</span><span class="n">FINAL</span><span class="o">/</span><span class="n">Sativa</span><span class="o">/</span><span class="n">eyorks_flora_final</span><span class="o">.</span><span class="n">gb</span> <span class="o">.</span>
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[19]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre><span class="n">ls</span>
</pre></div>

</div>
</div>
</div>

<div class="output_wrapper">
<div class="output">


<div class="output_area"><div class="prompt"></div>
<div class="output_subarea output_stream output_stdout output_text">
<pre>eyorks_flora_final.gb  Querymap.txt  REFmap.txt
</pre>
</div>
</div>

</div>
</div>

</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>Ready!</p>

</div>
</div>
</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[24]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash

metaBEAT_global.py <span class="se">\</span>
-Q Querymap.txt <span class="se">\</span>
--cluster <span class="se">\</span>
--clust_match 1 <span class="se">\</span>
--clust_cov 5 <span class="se">\</span>
--blast <span class="se">\</span>
--bitscore_skim_LCA 0.03 <span class="se">\</span>
--bitscore_skim_adjust_off <span class="se">\</span>
--REFlist REFmap.txt <span class="se">\</span>
--min_ident 0.95 <span class="se">\</span>
-n 5 <span class="se">\</span>
-v <span class="se">\</span>
-@ callumjmacgregor@gmail.com <span class="se">\</span>
-o MothPollenFinal &gt; log_final
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[25]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre>%%bash
blastn -query /home/working/FinalCorrected/FinalBlast/GLOBAL/global_centroids.fasta <span class="se">\</span>
-db /home/working/FinalCorrected/FinalBlast/GLOBAL/BLAST_0.95/marker_blast_db <span class="se">\</span>
-out /home/working/FinalCorrected/FinalBlast/GLOBAL/BLAST_0.95/global_blastn.out.txt
</pre></div>

</div>
</div>
</div>

</div>
<div class="cell border-box-sizing code_cell rendered">
<div class="input">
<div class="prompt input_prompt">In&nbsp;[&nbsp;]:</div>
<div class="inner_cell">
    <div class="input_area">
<div class=" highlight hl-ipython2"><pre> 
</pre></div>

</div>
</div>
</div>

</div>
    </div>
  </div>
</body>
</html>
